<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3598657">3598657</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23555009">23555009</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-12-39291</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0059270">10.1371/journal.pone.0059270</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span>: <span class="subject" title="subject">Crops</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis Tools</span>: <span class="subject" title="subject">Transcriptomes</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Developmental Biology</span>: <span class="subject" title="subject">Plant Growth and Development</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis Tools</span>: <span class="subject" title="subject">Transcriptomes</span> </div>: <span class="subject" title="subject">Genome Expression Analysis</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span>: <span class="subject" title="subject">Seeds</span> </div>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Using RNA-Seq to Profile Soybean Seed Development from Fertilization to Maturity</div> <div class="alt-title" title="alt-title">
Using RNA-Seq to Profile Soybean Seed Development</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">Sarah I.</span> </span> <a href="#aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">Lila O.</span> </span> <a href="#aff1" /> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[], <span class="addr-line" title="addr-line">Department of Crop Sciences, University of Illinois, Urbana, Illinois, United States of America </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellegrini</span> <span class="given-names" tagx="given-names" title="given-names">Matteo</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">UCLA-DOE Institute for Genomics and Proteomics, United States of America</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">l-vodkin@illinois.edu</span> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Obtained funding and edited manuscript: LV. Conceived and designed the experiments: LV SIJ. Performed the experiments: SIJ. Analyzed the data: SIJ. Wrote the paper: SIJ. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2013</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2013-3-3</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e59270</span> <span class="history" title="history"> <span class="received" title="received">received: 2012-12-12</span> <span class="accepted" title="accepted">accepted: 2013-2-13</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Jones, Vodkin</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2013</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>To understand gene expression networks leading to functional properties and compositional traits of the soybean seed, we have undertaken a detailed examination of soybean seed development from a few days post-fertilization to the mature seed using Illumina high-throughput transcriptome sequencing (RNA-Seq). RNA was sequenced from seven different stages of seed development, yielding between 12 million and 78 million sequenced transcripts. These have been aligned to the 79,000 gene models predicted from the soybean genome recently sequenced by the Department of Energy Joint Genome Institute. Over one hundred gene models were identified with high expression exclusively in young seed stages, starting at just four days after fertilization. These were annotated as being related to many basic components and processes such as histones and proline-rich proteins. Genes encoding storage proteins such as glycinin and beta-conglycinin had their highest expression levels at the stages of largest fresh weight, confirming previous knowledge that these storage products are being rapidly accumulated before the seed begins the desiccation process. Other gene models showed high expression in the dry, mature seeds, perhaps indicating the preparation of pathways needed later, in the early stages of imbibition. Many highly-expressed gene models at the dry seed stage are, as expected, annotated as hydrophilic proteins associated with low water conditions, such as late embryogenesis abundant (LEA) proteins and dehydrins, which help preserve the cellular structures and nutrients within the seed during desiccation. More significantly, the power of RNA-Seq to detect genes expressed at low levels revealed hundreds of transcription factors with notable expression in at least one stage of seed development. Results from a second biological replicate demonstrate high reproducibility of these data revealing a comprehensive view of the transciptome of seed development in the cultivar Williams, the reference cultivar for the first soybean genome sequence. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This research was supported by grants from the Illinois Soybean Association, the United Soybean Board, and the Illinois-Missouri Biotechnology Alliance. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Soybean is one of the most important oilseed crops in the world, used as food for both humans and animals as well as for a variety of industrial oil applications. The recent sequencing of the soybean genome <a href="#pone.0059270-Schmutz1">[1]</a> coupled with improvements in next-generation RNA sequencing technology allow a powerful way to study expression changes in thousands of genes throughout multiple stages of soybean seed development which influence such agronomically-important traits as oil and protein production, disease resistance, and stress tolerance. A recent increase in large scale soybean seed development studies using technologies such as microarrays <a href="#pone.0059270-Le1">[2]</a> and cDNA sequencing <a href="#pone.0059270-Sha1">[3]</a> emphasizes the importance of understanding the genetics underlying this crop's seed development pathway. </p> <p>Carlson and Lersten <a href="#pone.0059270-Carlson1">[4]</a> have reviewed the events in early soybean seed development, which may vary in exact timing depending on cultivar and environmental conditions. Soybeans are largely self-fertilizing and the first day the flower is fully open is usually considered the first day of fertilization. The stage when the soybean plant begins flowering is also known as R1 <a href="#pone.0059270-Ritchie1">[5]</a>. Around 3–5 DAF (days after flowering) the embryo has a spherical shape (known as the globe or globular stage) with an acellular endosperm and a rudimentary vascular system developing through the outer integument (which will become the seed coat). The cotyledons initiate during the heart stage at about 8–10 DAF, by which point the endosperm is almost entirely cellular; the inner and outer integuments have thickened to ten to fifteen cell layers each. </p> <p>By 12–14 DAF, approximately half of the available space in the seed is taken up by the endosperm, the cotyledons have begun to elongate, and the vascular system running through the outer integument becomes much more extensive. Around 17–19 DAF, the endosperm begins to degenerate and its cellular contents are assimilated by the cotyledons, where organelles as well as protein and lipid globules have begun to form. By 22–24 DAF, the cotyledons and primary leaves are well on their way towards their full size in the seed, and the primordium of the first trifoliate leaf begins to develop. The stages from 12 DAF to 24 DAF tend to occur between R3 and R4 in this experiment, with seeds found in pods between 5 mm and 15 mm long <a href="#pone.0059270-Ritchie1">[5]</a>. </p> <p>Later stages of soybean seed development have been well-described by <a href="#pone.0059270-Ritchie1">[5]</a>. Between the stages of R4 and R7 the seeds grow rapidly, accumulating nutrients and storage proteins. During R6, the vegetative parts of the parent plant start to turn yellow and leaf senescence begins. The accumulation of nutrients slows down in the seeds and by R7, virtually all the seed’s dry weight has been acquired. In the cultivar Williams used here, the total fresh weight of the seed peaks around 400–500 mg. After this point the seed starts to desiccate and turn yellow, and the total fresh weight of the seed decreases as water is lost. At this point the seed contains a large embryo with cotyledons, primary leaves, and a primordium for the first trifoliate leaf, all surrounded by a three-layer dry seed coat <a href="#pone.0059270-Carlson1">[4]</a>. By stage R8, almost all of the pods and seeds have turned brown and dry and the embryos are quiescent. Later, when the seed imbibes water and begins germination, metabolic activity can restart immediately using enzymes, initiation and elongation factors, and other compounds that were produced during development and stored <a href="#pone.0059270-Bewley1">[6]</a>. </p> <p>Previously, we examined gene expression changes during stages from mid-maturation to full maturation, the period of the most intensive seed fill, using microarrays <a href="#pone.0059270-Jones1">[7]</a>. Here, we use the power of high-throughput RNA sequencing technology, including very young stages starting at just a few days after flowering, as well as later mature seed stages, to elucidate expression patterns across a broad period of seed development revealing the entire transcriptome. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="stagesandreplicatesused" title="sec"> <div class="title" tagx="title" title="title">
Stages and Replicates Used</div> <p>Seven stages of soybean seeds were studied, encompassing major milestones in development such as tissue differentiation, nutrient accumulation, storage protein synthesis, and desiccation as shown in <a href="#pone-0059270-t001">Table 1</a>. Younger stages (4 DAF, 12–14 DAF, 22–24 DAF, 5–6 mg) were examined as whole seeds (combined seed coat and cotyledon) in order to preserve the liquid endosperm without damage. Cotyledons were examined in two older stages (100–200 mg and 400–500 mg) in order to study the expression of important storage protein genes during the period of seed fill. Dry seeds were also used whole, with combined cotyledon and seed coat. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t001">10.1371/journal.pone.0059270.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Stages of soybean seed development and RNA-Seq data.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Stage</td> <td>Age/Weight</td> <td>Tissue</td> <td>Developmental Events</td> <td>Bio Rep 1 Reads</td> <td>Bio Rep 2 Reads</td> </tr> </thead> <tbody> <tr> <td>1</td> <td>4 DAF</td> <td>Whole seed</td> <td>Spherical embryo; acellular endosperm</td> <td>11.5 M</td> <td>51.1 M</td> </tr> <tr> <td>2</td> <td>12–14 DAF</td> <td>Whole seed</td> <td>Cotyledons elongate; endosperm forms cell walls</td> <td>24.3 M</td> <td>183.5 M</td> </tr> <tr> <td>3</td> <td>22–24 DAF</td> <td>Whole seed</td> <td>Cotyledons have organelles; first trifoliate leaf initiates</td> <td>77.6 M</td> <td>181.9 M</td> </tr> <tr> <td>4</td> <td>5–6 mg</td> <td>Whole seed</td> <td>Endosperm depleted; rapid accumulation of water, nutrients</td> <td>30.5 M</td> <td>149.9 M</td> </tr> <tr> <td>5</td> <td>100–200 mg</td> <td>Cotyledon</td> <td>Nutrient accumulation continues; storage proteins synthesized</td> <td>18.1 M</td> <td>129.7 M</td> </tr> <tr> <td>6</td> <td>400–500 mg</td> <td>Cotyledon</td> <td>Peak of fresh weight; embryo prepares for desiccation</td> <td>34.1 M</td> <td>122.0 M</td> </tr> <tr> <td>7</td> <td>100–200 mg</td> <td>Whole seed, dry</td> <td>Quiescent embryo; low moisture content</td> <td>30.2 M</td> <td>64.0 M</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Shown are the seven stages of soybean seed development used for RNA-Seq. Days after flowering (DAF) or fresh weight in milligrams (mg) is shown along with tissue and the number of processed reads obtained from two biological replicates of RNA high-throughput sequencing (in millions, M). Key events in development are noted. </p> </div> </div> </div> <p>High-throughput next generation transcriptome sequencing using Illumina technology (RNA-Seq) was performed on seven stages of soybean seeds, with two biological replicates per stage, resulting in millions of reads per sample; see <a href="#pone-0059270-t001">Table 1</a> for totals. These reads were aligned to 78,773 soybean gene models determined by the Soybean Genome Project <a href="#pone.0059270-Schmutz1">[1]</a> using the program Bowtie <a href="#pone.0059270-Langmead1">[8]</a>. Thousands of genes were found to be significantly differently expressed between various stages ( <a href="#pone-0059270-t002">Table 2</a>). Data is given in RPKMs (reads per kilobase of gene model per million mapped reads; <a href="#pone.0059270-Mortazavi1">[9]</a>). Data from the two biological replicates are highly similar, with between 99.53% and 99.99% of gene models showing no significant difference in expression between the two biological replicates of a given stage ( <a href="#pone-0059270-t003">Table 3</a>). The RPKM filtering and data clustering were performed using values from Biological Replicate 1; the values from Biological Replicate 2 of the specific gene models in each cluster were graphed to show the overall similarity of the expression trends between the two replicates. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t002">10.1371/journal.pone.0059270.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Significantly different gene models between two stages of soybean seed development.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Stages Compared</td> <td>Number of Gene Models</td> <td>Percentage of Gene Models</td> </tr> </thead> <tbody> <tr> <td>4 DAF vs. 5–6 mg whole seed</td> <td>3435</td> <td>4.4</td> </tr> <tr> <td>100–200 mg cotyledon vs. dry whole seed</td> <td>10594</td> <td>13.5</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Number and percentage (of 78,773) of gene models significantly different in expression between two stages of soybean seed development (adjusted p-value ≤0.05). Data from two biological replicates were used for each stage compared. </p> </div> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t003">10.1371/journal.pone.0059270.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Gene models with no significant difference between biological replicates.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Stage</td> <td>Number of Gene Models</td> <td>Percentage of Gene Models</td> </tr> </thead> <tbody> <tr> <td>4 DAF</td> <td>78560</td> <td>99.73</td> </tr> <tr> <td>12–14 DAF</td> <td>78732</td> <td>99.95</td> </tr> <tr> <td>22–24 DAF</td> <td>78660</td> <td>99.86</td> </tr> <tr> <td>5–6 mg whole seed</td> <td>78402</td> <td>99.53</td> </tr> <tr> <td>100–200 mg cotyledon</td> <td>78762</td> <td>99.99</td> </tr> <tr> <td>400–500 mg cotyledon</td> <td>78760</td> <td>99.98</td> </tr> <tr> <td>Dry whole seed</td> <td>78717</td> <td>99.93</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Number and percentage (of 78,773) of gene models with no significant difference in expression between two biological replicates (adjusted p-value &amp;gt;0.05), for each stage of soybean seed development. </p> </div> </div> </div> </div> <div class="genemodelsexpressedhighlyinyoungseedsonly" title="sec"> <div class="title" tagx="title" title="title">
Gene Models Expressed Highly in Young Seeds Only</div> <p>The 78,773 soybean gene models were filtered to retain gene models with RPKM≥500 in at least one of the four young stages (4 DAF, 12–14 DAF, 22–24 DAF, 5–6 mg) and also RPKM &amp;lt;500 in all three older stages (100–200 mg cotyledon, 400–500 mg cotyledon, dry seed). This resulted in a list of 111 genes with a peak in expression in at least one of the young seed stages, and lower expression in all older seed stages. The minimum RPKM of 500 was chosen to highlight the most highly-expressed genes at these early seed development stages. These gene models were then grouped into 6 clusters with Multi-Experiment Viewer (MeV; <a href="#pone.0059270-Saeed1">[10]</a>). All six clusters are shown in <a href="#pone.0059270.s001">Figure S1</a> and the gene models in each cluster are listed in <a href="#pone.0059270.s004">Table S1</a>. </p> <p>The cluster in <a href="#pone-0059270-g001">Figure 1A</a> contains 41 gene models. In general these genes peak in RPKM at 12–14 DAF whole seed, with RPKMs ranging from about 500 to over 1800. There is also a secondary peak at 5–6 mg whole seed, with most RPKMs ranging from about 200 to 800. Throughout the three oldest stages of development RPKMs are &amp;lt;200. Of the 41 genes in <a href="#pone-0059270-g001">Figure 1A</a>, 26 models (63.4%) are annotated as histones ( <a href="#pone-0059270-t004">Table 4</a>). Other gene model annotations found in this cluster include chlorophyll A-B binding protein and protease inhibitor/seed storage/LTP family. RPKM data for these 41 genes from Biological Replicate 2 ( <a href="#pone-0059270-g001">Figure 1B</a>) also show higher expression in the younger seed stages than in the older. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0059270.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Clusters of gene models prominent at different developmental stages in soybean seeds.</div> <p>Clusters were created from Biological Replicate 1 data (A, C, E, G) and the same gene models were also graphed with Biological Replicate 2 data (B, D, F, H). A and C have gene models with RPKM≥500 in at least one of four young seed stages, and RPKM &amp;lt;500 in all three older stages. E and G have gene models with RPKM≥1000 in at least one of three older seed stages. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t004">10.1371/journal.pone.0059270.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
Annotations for gene models shown in <a href="#pone-0059270-g001">Figure 1</a>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Annotation</td> <td>Number of Gene Models</td> <td>Percentage of Gene Models</td> </tr> </thead> <tbody> <tr> <td> <a href="#pone-0059270-g001"> <i>Figure 1A</i> </a> <i> (high in young seeds)</i> </td> <td> <i>41 total</i> </td> <td /> </tr> <tr> <td>PF00125 Core histone H2A/H2B/H3/H4</td> <td>26</td> <td>63.4</td> </tr> <tr> <td>PF00504 Chlorophyll A-B binding protein</td> <td>3</td> <td>7.3</td> </tr> <tr> <td>PF00234 Protease inhibitor/seed storage/LTP family</td> <td>3</td> <td>7.3</td> </tr> <tr> <td>PF02704 Gibberellin regulated protein</td> <td>2</td> <td>4.9</td> </tr> <tr> <td>PF08069 Ribosomal S13/S15 N-terminal domain</td> <td>2</td> <td>4.9</td> </tr> <tr> <td>Other</td> <td>5</td> <td>12.2</td> </tr> <tr> <td> <a href="#pone-0059270-g001"> <i>Figure 1C</i> </a> <i> (high in young seeds)</i> </td> <td> <i>20 total</i> </td> <td /> </tr> <tr> <td>AAA66288.1 proline-rich protein</td> <td>6</td> <td>30.0</td> </tr> <tr> <td>Unknown function</td> <td>5</td> <td>25.0</td> </tr> <tr> <td>ACA04850.1 senescence-associated protein [Picea abies]</td> <td>3</td> <td>15.0</td> </tr> <tr> <td>CAB45653.1 putative tonoplast intrinsic protein [P. sativum]</td> <td>2</td> <td>10.0</td> </tr> <tr> <td>Other</td> <td>4</td> <td>20.0</td> </tr> <tr> <td> <a href="#pone-0059270-g001"> <i>Figure 1E</i> </a> <i> (high in mid-maturation seeds)</i> </td> <td> <i>30 total</i> </td> <td /> </tr> <tr> <td>P04405.2 GLYG2_SOYBN Glycinin; P25974.1 GLCB_SOYBN Beta-conglycinin</td> <td>10</td> <td>33.3</td> </tr> <tr> <td>CAA49340.1 ADR6 [Glycine max]</td> <td>5</td> <td>16.7</td> </tr> <tr> <td>PF01277 Oleosin</td> <td>5</td> <td>16.7</td> </tr> <tr> <td>PF00305 Lipoxygenase</td> <td>3</td> <td>10.0</td> </tr> <tr> <td>Other</td> <td>7</td> <td>23.3</td> </tr> <tr> <td> <a href="#pone-0059270-g001"> <i>Figure 1G</i> </a> <i> (high in dry seeds)</i> </td> <td> <i>40 total</i> </td> <td /> </tr> <tr> <td>PF03760 Late embryogenesis abundant (LEA) group 1</td> <td>14</td> <td>35.0</td> </tr> <tr> <td>PF04927 Seed maturation protein</td> <td>12</td> <td>30.0</td> </tr> <tr> <td>PF00257 Dehydrin</td> <td>4</td> <td>10.0</td> </tr> <tr> <td>S71562 drought-induced protein SDi-6</td> <td>2</td> <td>5.0</td> </tr> <tr> <td>PF00009 Elongation factor Tu GTP binding domain</td> <td>2</td> <td>5.0</td> </tr> <tr> <td>ACU18354.1 unknown [Glycine max]</td> <td>2</td> <td>5.0</td> </tr> <tr> <td>Other</td> <td>4</td> <td>10.0</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>The gene models in clusters shown in <a href="#pone-0059270-g001">Figure 1</a>, divided into representative PFAM and NCBI non-redundant database annotation groups along with the number and percentage (of the cluster’s total) of gene models in each group. The group "Other" represents a miscellaneous category of remaining annotations. Annotations may be shortened for clarity. </p> </div> </div> </div> <p>The cluster in <a href="#pone-0059270-g001">Figure 1C</a> contains 20 gene models whose common feature is a peak in RPKM at the 5–6mg whole seed stage. RPKMs at this stage range from about 500 to 1500. RPKMs are generally lower at the other stages of development; some gene models have a smaller peak at 12–14 DAF and/or increase slightly in RPKM at the dry seed stage, though the highest RPKM is still &amp;lt;500. Of the 20 gene models in <a href="#pone-0059270-g001">Figure 1C</a>, 6 models (30%) are annotated as proline-rich proteins ( <a href="#pone-0059270-t004">Table 4</a>). Another 25% of genes in this cluster have unknown functions. Other gene model annotations with this expression pattern include senescence-associated proteins and tonoplast intrinsic proteins. These genes show a similar pattern of expression in Biological Replicate 2 ( <a href="#pone-0059270-g001">Figure 1D</a>), with some displaying even higher RPKMs at the 4 DAF stage. </p> <p>These 20 gene models were further examined for their RPKM values in two additional samples, 5–6 mg seed coats and 5–6 mg cotyledons ( <a href="#pone-0059270-t005">Table 5</a>). Whereas the original 5–6 mg sample represented the combined seed coat and cotyledon, the two new samples separate those two major tissues to more precisely determine the location of gene expression. Nine of the gene models (including a 7-member splice variant family of extensin/proline-rich proteins) have RPKMs at least 100-fold higher in the 5–6 mg seed coat sample than in the cotyledon sample, with average RPKMs of about 580 and 3, respectively. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t005">10.1371/journal.pone.0059270.t005</a> <span class="label" tagx="label" title="label">Table 5</span>  <div class="title" tagx="title" title="title">
Expression patterns of gene models shown in <a href="#pone-0059270-g001">Figure 1C</a>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Expression Pattern</td> <td>Number of Gene Models</td> <td>5–6 mg Whole Seed RPKM Ave</td> <td>5–6 mg Seed Coat RPKM Ave</td> <td>5–6 mg Cotyledon RPKM Ave</td> <td>Predominant Annotation</td> </tr> </thead> <tbody> <tr> <td>Much higher in seed coat</td> <td>9</td> <td>906</td> <td>586</td> <td>2.5</td> <td>Extensin/proline-rich proteins</td> </tr> <tr> <td>Higher in seed coat</td> <td>4</td> <td>870</td> <td>407</td> <td>50</td> <td>Unknown</td> </tr> <tr> <td>High in both seed coat and cotyledon</td> <td>5</td> <td>1128</td> <td>1714</td> <td>1020</td> <td>Senescence-associated proteins</td> </tr> <tr> <td>Higher in cotyledon</td> <td>2</td> <td>641</td> <td>65</td> <td>638</td> <td>Membrane intrinsic proteins</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>The 20 gene models in <a href="#pone-0059270-g001">Figure 1C</a> (genes with high RPKM at 5–6 mg whole seed) divided into groups based on their expression patterns in the 5–6 mg separated seed coat and cotyledon samples. Data from 5–6 mg combined whole seed sample also shown. RPKMs have been averaged for each group of gene models. The predominant annotation of each expression pattern group is also shown. </p> </div> </div> </div> <p>Four genes, mostly with unknown functions, have higher RPKMs in the seed coat but the cotyledon RPKMs are not insignificant. The RPKMs in the seed coat average about 400, while those in the cotyledon average about 50, between 6- and 15-fold higher expression in the seed coats. </p> <p>Five genes, including those annotated as senescence-associated, have high RPKMs in both the seed coat and the cotyledon at the 5–6 mg stage, with the average RPKM in both tissues being over 1000. The RPKMs in each tissue tend to be at similar levels, with most less than 2-fold higher in the seed coat. Finally, a two-member splice variant family, annotated as intrinsic proteins, displays about 10-fold higher expression in the cotyledon than in the seed coat, with average RPKMs of about 640 and 65, respectively. </p> </div> <div class="genemodelsexpressedhighlyinolderseedstages" title="sec"> <div class="title" tagx="title" title="title">
Gene Models Expressed Highly In Older Seed Stages</div> <p>In another analysis, RPKMs from the 78,773 soybean gene models were filtered to include only those with RPKM≥1000 in at least one of the three older seed stages (100–200 mg cotyledon, 400–500 mg cotyledon, dry seed). A higher minimum RPKM was chosen due to the generally larger RPKMs associated with older seed stages, compared to the younger stages. This filtering resulted in a list of 162 gene models with high expression in at least one older seed stage. The genes were then grouped into 7 clusters with Multi-Experiment Viewer. All seven clusters are shown in <a href="#pone.0059270.s002">Figure S2</a> and the gene models in each cluster are listed in <a href="#pone.0059270.s004">Table S1</a>. </p> <p>The cluster in <a href="#pone-0059270-g001">Figure 1E</a> contains 30 genes with high RPKMs at 100–200 mg cotyledon and 400–500 mg cotyledon, and comparatively low RPKMs at all other stages. The peaks at the two cotyledon stages are roughly equal and the RPKMs range from about 1000 to 13,000. At dry seed, the RPKM may reach as high as 2000; however, throughout the four earliest stages of seed development, the highest RPKM is only 87. Of the 30 gene models in <a href="#pone-0059270-g001">Figure 1E</a>, 10 models (33.3%) are annotated as storage proteins such as glycinin and beta-conglycinin ( <a href="#pone-0059270-t004">Table 4</a>). Other gene model annotations found in this cluster include auxin down-regulated (ADR) proteins, oleosin, and lipoxygenase. These same genes have a very similar expression pattern in Biological Replicate 2 ( <a href="#pone-0059270-g001">Figure 1F</a>). </p> <p>Another cluster, <a href="#pone-0059270-g001">Figure 1G</a> <b>,</b> contains 40 gene models. These genes peak in expression at the final stage studied, that of dry, whole seed, with RPKMs ranging from about 1000 to 11,000 at this stage. Expression is comparatively low and flat at all other stages. Of the 40 genes in <a href="#pone-0059270-g001">Figure 1G</a>, 14 (35%) are annotated as late embryogenesis abundant (LEA) proteins ( <a href="#pone-0059270-t004">Table 4</a>). Another 30% are annotated as various seed proteins such as seed maturation proteins and small hydrophilic plant seed proteins. Other gene model annotations found in this cluster include dehydrins and drought-induced proteins. Data from Biological Replicate 2 ( <a href="#pone-0059270-g001">Figure 1H</a>) shows a similar expression pattern for these genes. </p> </div> <div class="genemodelsannotatedasstorageproteins" title="sec"> <div class="title" tagx="title" title="title">
Gene Models Annotated as Storage Proteins</div> <p>The NCBI non-redundant database annotations for the 78,773 soybean gene models were keyword-filtered to assemble lists of gene models annotated as various soybean seed and storage proteins such as glycinin and beta-conglycinin. These sublists were then filtered to retain gene models with RPKM≥5 in at least one of the seven seed development stages studied here. </p> <p>Glycinin and beta-conglycinin gene models with RPKM≥5 in at least one seed stage were found to have high RPKMs at both of the cotyledon stages, 100–200 mg and 400–500 mg ( <a href="#pone-0059270-g002">Figure 2A</a>). Beta-conglycinin genes tended to peak at 100–200 mg cotyledon with most RPKMs in the range of 1000 to over 25,000. Glycinin genes, in contrast, tended to peak at the later 400–500 mg cotyledon stage, with RPKMs from about 5000 to nearly 23,000. At all other stages, RPKMs were comparatively low, with a maximum RPKM of about 60 in all four young stages (4 DAF through 5–6 mg) and a maximum RPKM of about 800 in the dry seed stage. Data for these gene models from Biological Replicate 2 showed a similar expression pattern ( <a href="#pone-0059270-g002">Figure 2B</a>). Displaying this data on a log scale reveals variation in the expression of the genes at the younger stages of seed development ( <a href="#pone-0059270-g002">Figure 2C</a> and, with Biological Replicate 2 data, <a href="#pone-0059270-g002">Figure 2D</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0059270.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Storage protein gene models grouped by annotation.</div> <p>Annotations were derived from PFAM and/or the NCBI non-redundant database. Clusters were created from Biological Replicate 1 data (A, C, E) and the same gene models were also graphed with Biological Replicate 2 data (B, D, F). A shows glycinin (pink) and beta-conglycinin (black) storage proteins with RPKM≥5 in at least one of seven stages; C is the same graph using log scale. (An RPKM of 0 will not display on a log scale and may appear as a break in the line.) E shows omega-3-fatty acid desaturase (pink) and omega-6-fatty acid desaturase (black) with RPKM≥5 in at least one of seven stages, log scale. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p>   </div> <p>In contrast, of the 14 gene models annotated as omega-6-fatty acid desaturase with RPKM≥5 in at least one seed stage, most were steady throughout all stages of development with RPKMs between 50 and 100 ( <a href="#pone-0059270-g002">Figure 2E</a>). A few peaked dramatically at the two cotyledon stages, 100–200 mg and 400–500 mg, with RPKMs greater than 1000. In contrast genes annotated as omega-3-fatty acid desaturase tended to have lower RPKMs, less than 50 at most stages with a drop below 1 at the dry seed stage. A similar pattern is found in Biological Replicate 2 ( <a href="#pone-0059270-g002">Figure 2F</a>). The specific gene models depicted in each figure are listed in <a href="#pone.0059270.s005">Table S2</a>. </p> </div> <div class="genemodelsannotatedastranscriptionfactors" title="sec"> <div class="title" tagx="title" title="title">
Gene Models Annotated as Transcription Factors</div> <p>Transcription factors are key regulators of gene expression which are generally expressed at a lower level than the genes they influence. In order to explore the network of gene regulation during soybean seed development, a list was assembled of transcription factors among the 78,773 soybean gene models. The NCBI non-redundant database annotations were keyword-filtered to assemble a list of 5608 genes annotated as transcription factors, which was treated as a complete list of all transcription factor genes for the purposes of this analysis. </p> <p>The 22–24 DAF whole seed had the most transcription factors, with nearly 2000 having RPKMs≥5 ( <a href="#pone-0059270-g003">Figure 3</a>). Roughly 40% of these genes had RPKMs in the 5–10 range, with another 30% having an RPKM of at least 21. The young whole seed samples–4 DAF, 12–14 DAF, 22–24 DAF, and 5–6 mgWS–all had more transcription factor genes (with RPKM≥5) than the three older stages, though the dry whole seed stage had only slightly fewer than 5–6 mg whole seed with about 1000 and 1100 transcription factor genes, respectively. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0059270.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
A histogram dividing 5608 transcription factor gene models into bins based on their RPKMs. </div> <p>Nine different libraries are shown, including 5–6 mg separated seed coat and cotyledon. Gene models must have RPKM≥5 at a stage to be displayed. Gene models with RPKM≥51 are compiled together at the top (orange). 5–6 mgWS = 5–6 mg whole seed. </p>   </div> <p>The fewest number of transcription factor genes (with RPKM≥5) was found in the 5–6 mg separated seed coat sample; only about 500 genes met the criteria in this tissue. Twice as many transcription factor gene models were found in the 5–6 mg separated cotyledon, only slightly fewer than in the 5–6 mg whole seed (which combines the cotyledon and seed coat tissues). </p> <p>The list of 5608 transcription factor gene models was then filtered to retain genes with RPKM≥25 in at least one of the seven seed development stages studied here (not including the 5–6 mg separated seed coat and cotyledon). This minimum RPKM was chosen to highlight the transcription factor gene models with the highest expression levels throughout soybean seed development. This resulted in a list of 676 transcription factor genes with notable expression in at least one stage of seed development, about 12% of the total list of transcription factors. The 676 transcription factor gene models were then grouped into 7 clusters with Multi-Experiment Viewer. All seven clusters are shown in <a href="#pone.0059270.s003">Figure S3</a> and the gene models in each cluster are listed in <a href="#pone.0059270.s004">Table S1</a>. </p> <p>The cluster in <a href="#pone-0059270-g004">Figure 4A</a> contains 228 transcription factor genes. In general these gene models peak in RPKM at 22–24 DAF, with RPKMs ranging from about 25 to over 400. RPKMs may be notable at other young stages as well, but tend to be comparatively low at the three oldest stages. Of the 228 transcription factor genes in <a href="#pone-0059270-g004">Figure 4A</a>, 26 models (11.4%) are annotated as MYB-like DNA-binding domain ( <a href="#pone-0059270-t006">Table 6</a>). Almost 10% are annotated as zinc fingers. Other gene model annotations found in this cluster include auxin response factors, AP2/ERF transcription factors, and SRF-type transcription factors. A similar pattern of expression is found for these gene models in Biological Replicate 2 ( <a href="#pone-0059270-g004">Figure 4B</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0059270.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Transcription factor gene models.</div> <p>Clusters were created from Biological Replicate 1 data (A, C, E, G) and the same gene models were also graphed with Biological Replicate 2 data (B, D, F, H). A and C display transcription factors with RPKM≥25 in at least one of seven stages of seed development. E displays gene models annotated as AP2/ERF/EREBP (ethylene responsive) transcription factors with RPKM≥25 in at least one stage of seed development. The Y axis has been set at 400, cutting off the purple line on the right, which reaches an RPKM of over 1400 at the dry seed stage. G displays gene models annotated as No Apical Meristem (NAM/NAC) transcription factors with RPKM≥25 in at least one stage of seed development. Annotations were derived from PFAM and/or the NCBI non-redundant database. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0059270.t006">10.1371/journal.pone.0059270.t006</a> <span class="label" tagx="label" title="label">Table 6</span>  <div class="title" tagx="title" title="title">
Annotations for transcription factor gene models shown in <a href="#pone-0059270-g004">Figure 4</a>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Annotation</td> <td>Number of GeneModels</td> <td>Percentage of Gene Models</td> </tr> </thead> <tbody> <tr> <td> <a href="#pone-0059270-g004"> <i>Figure 4A</i> </a> <i> (TFs high in young seeds)</i> </td> <td> <i>228 total</i> </td> <td /> </tr> <tr> <td>PF00249 Myb-like DNA-binding domain</td> <td>26</td> <td>11.4</td> </tr> <tr> <td>PF00096 Zinc finger, C2H2 type</td> <td>22</td> <td>9.7</td> </tr> <tr> <td>PF02309 AUX/IAA family; NP_851244.1 AUXIN RESPONSE FACTOR 2 [A. thaliana]</td> <td>15</td> <td>6.6</td> </tr> <tr> <td>XP_002326298.1 AP2/ERF transcription factor</td> <td>13</td> <td>5.7</td> </tr> <tr> <td>PF00319 SRF-type transcription factor</td> <td>13</td> <td>5.7</td> </tr> <tr> <td>PF02362 B3 DNA binding domain</td> <td>11</td> <td>4.8</td> </tr> <tr> <td>XP_002300324.1 global transcription factor group</td> <td>11</td> <td>4.8</td> </tr> <tr> <td>PF02183 Homeobox associated leucine zipper</td> <td>8</td> <td>3.5</td> </tr> <tr> <td>PF06217 GAGA binding protein-like family</td> <td>7</td> <td>3.1</td> </tr> <tr> <td>AAY30856.1 MADS-box transcription factor [P. dulcis]</td> <td>7</td> <td>3.1</td> </tr> <tr> <td>Other</td> <td>95</td> <td>41.7</td> </tr> <tr> <td> <a href="#pone-0059270-g004"> <i>Figure 4C</i> </a> <i> (TFs high in dry seeds)</i> </td> <td> <i>108 total</i> </td> <td /> </tr> <tr> <td>XP_002307142.1 AP2/ERF transcription factor</td> <td>21</td> <td>19.4</td> </tr> <tr> <td>PF02045 CCAAT-binding transcription factor (CBF-B/NF-YA)</td> <td>11</td> <td>10.2</td> </tr> <tr> <td>PF00170 bZIP transcription factor</td> <td>10</td> <td>9.3</td> </tr> <tr> <td>PF02365 No apical meristem (NAM) protein</td> <td>10</td> <td>9.3</td> </tr> <tr> <td>PF00096 Zinc finger, C2H2 type</td> <td>10</td> <td>9.3</td> </tr> <tr> <td>PF00249 Myb-like DNA-binding domain</td> <td>5</td> <td>4.6</td> </tr> <tr> <td>CAA87076.1 heat shock transcription factor 21 [G. max]</td> <td>4</td> <td>3.7</td> </tr> <tr> <td>Other</td> <td>37</td> <td>34.3</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>The gene models in the clusters shown in <a href="#pone-0059270-g004">Figure 4</a> (transcription factors), divided into representative PFAM and NCBI non-redundant database annotation groups along with the number and percentage (of the cluster’s total) of gene models in each group. The group "Other" represents a miscellaneous category of remaining annotations. Annotations may be shortened for clarity. </p> </div> </div> </div> <p>The cluster in <a href="#pone-0059270-g004">Figure 4C</a> contains 108 transcription factor genes that generally peak in RPKM at the final, dry seed stage, with the highest RPKM being over 1600. Most of the gene models have comparatively low RPKMs at all the other stages of development, with a maximum RPKM of 125 throughout the first five stages. Of the 108 transcription factor genes in <a href="#pone-0059270-g004">Figure 4C</a>, 21 models (19.4%) are annotated as AP2/ERF transcription factors ( <a href="#pone-0059270-t006">Table 6</a>). Another 10% are annotated as CCAAT-binding transcription factors. Other transcription factor annotations found in this cluster included bZIP, no apical meristem (NAM), and zinc fingers. These gene models show similar expression in Biological Replicate 2 data ( <a href="#pone-0059270-g004">Figure 4D</a>). </p> <p>The transcription factor genes can also be grouped into families according to their annotations (gene models listed in <a href="#pone.0059270.s005">Table S2</a>). <a href="#pone-0059270-g004">Figure 4E</a> shows that transcription factors annotated as AP2/ethylene responsive factors (with RPKM≥25 in at least one stage) display a variety of shapes throughout seed development. Some members of the family are most highly expressed during the early stages, such as 12–14 DAF or 22–24 DAF whole seed, while others appear to peak in expression at the final, dry seed stage. A similar variety in expression patterns is found in data from Biological Replicate 2 ( <a href="#pone-0059270-g004">Figure 4F</a>). </p> <p>The no apical meristem (NAM) transcription factors have a much more uniform expression pattern, at least those gene models with RPKM≥25 in at least one stage of seed development ( <a href="#pone-0059270-g004">Figure 4G</a>). Most of these gene models clearly peak in expression at the dry seed stage, with the highest RPKM around 700. At the five earliest stages of seed development (4 DAF through 100–200 mg cotyledon), the highest RPKM is only about 37. This pattern of expression is repeated in Biological Replicate 2 ( <a href="#pone-0059270-g004">Figure 4H</a>). </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In the early stages of development, the seeds are undergoing rapid growth and differentiation as discrete structures such as the embryo, seed coat, cotyledons, endosperm, and primary leaves form. This can be seen in the annotations for genes in <a href="#pone-0059270-g001">Figures 1A and 1C</a>, which peak in RPKM at early stages such as 12–14 DAF and 5–6 mg ( <a href="#pone-0059270-t004">Table 4</a>). Almost two-thirds of the gene models in <a href="#pone-0059270-g001">Figure 1A</a> are annotated as histones, proteins around which DNA is wound for compact storage in cells. The large number of histone genes showing high expression in the young seed stages may reflect the rapid expansion in cell number at this point in development, as new histones are produced to organize the DNA strands into nucleosomes. Histones are highly over-represented in this cluster, as a keyword search of the annotations (NCBI non-redundant database and PFAM) for 78,773 soybean gene models reveals &amp;lt;1% described as encoding histones. </p> <p>Likewise, about 30% of the genes in <a href="#pone-0059270-g001">Figure 1C</a> are annotated as proline-rich proteins, which are involved in cell wall structure in plants. The high expression of these genes in early stages may be due to increased cell growth and development at this time. This annotation is also highly over-represented in this cluster, with &amp;lt;1% of the 78,773 soybean gene models described as encoding proline-rich proteins. Other studies have shown soybean proline-rich protein gene <i>PRP1</i> to be highly expressed in developing seed coats up to 50 mg fresh weight <a href="#pone.0059270-Hong1">[11]</a>, <a href="#pone.0059270-Lindstrom1">[12]</a>. </p> <p>Intriguingly, other gene models with high RPKM at these early stages are annotated as senescence-associated. For example, the genes with this annotation in <a href="#pone-0059270-g001">Figure 1C</a> were found to have extremely high RPKMs (&amp;gt;1000) in both the cotyledon/embryo and seed coat tissues of the 5–6 mg stage ( <a href="#pone-0059270-t005">Table 5</a>). Senescence-associated plant genes and proteins have mainly been studied in the context of dying petals and leaves, tissues which release their nutrients to other parts of the plant and degrade as part of the plant’s normal life cycle <a href="#pone.0059270-Panavas1">[13]</a>, <a href="#pone.0059270-Lim1">[14]</a>. Our data suggest these genes have an important role in early stages of seed development as yet unexplored, especially as &amp;lt;1% of the total number of gene models are annotated as senescence-associated. </p> <p>By the later stages of seed development, the seed is focused on accumulating nutrients and synthesizing storage proteins. The RNA-Seq data reflect this, with storage protein annotations over-represented in the gene models shown in <a href="#pone-0059270-g001">Figure 1E</a>, which peak at the two mid-maturation cotyledon stages. For example, common annotations ( <a href="#pone-0059270-t004">Table 4</a>) among these genes are well-known seed storage proteins such as glycinin, beta-conglycinin, and sucrose-binding proteins, which are generally found at &amp;lt;1% in the genome as a whole. Similar expression patterns for these storage protein genes have been found in other studies <a href="#pone.0059270-Meinke1">[15]</a>– <a href="#pone.0059270-Asakura1">[17]</a>. A study by <a href="#pone.0059270-Peng1">[18]</a> of developing <i>Arabidopsis</i> seeds, starting with the very young globular stage and culminating with mature embryos and green cotyledons, also found a similar expression pattern for genes encoding oleosins and seed storage proteins. </p> <p>Previous work done by our lab <a href="#pone.0059270-Jones1">[7]</a> using cDNA microarrays with the same older stages of development found that genes in the auxin down-related (ADR) and lipoxygenase families had high expression at the 400–500 mg cotyledon stage, in comparison to the 100–200 mg cotyledon stage. Lipoxygenase is involved in the storage of nitrogen and the oxidation of polyunsaturated fatty acids in seeds <a href="#pone.0059270-Stephenson1">[19]</a>, <a href="#pone.0059270-SkrzypczakJankun1">[20]</a> and may also be used later in reactions during early shoot growth <a href="#pone.0059270-IslasFlores1">[21]</a>. Auxin down-regulated (ADR) genes represent an intriguing family first described by <a href="#pone.0059270-Baulcombe1">[22]</a> as having reduced RNA concentration following auxin treatment of soybean hypocotyls. Members of this family have since been discovered to be highly expressed in soybean somatic embryos <a href="#pone.0059270-ThibaudNissen1">[23]</a> and soybean cotyledons both immature <a href="#pone.0059270-Jones1">[7]</a> and post-germination <a href="#pone.0059270-Gonzalez1">[24]</a>. Both ADR6 and lipoxygenase annotations are over-represented in this cluster, as both are found at &amp;lt;1% in the total list of 78,773 gene models. </p> <p>A recent study <a href="#pone.0059270-Asakura1">[17]</a> used Affymetrix microarrays to examine differentially expressed genes in stages of soybean seeds roughly corresponding to the middle stages of the current experiment, culminating in whole seeds at the stage of largest fresh weight for their cultivar. Genes with many of the same annotations, such as glycinin, beta-conglycinin, lipoxygenase, and oleosin, were found to peak in expression at the stage of largest fresh weight, coincident with our results. Interestingly, they also detected a number of genes annotated as seed maturation protein, late embryogenesis abundant (LEA) protein, and dehydrin with this same expression pattern; in our study such genes peaked in expression at the final, dry seed stage ( <a href="#pone-0059270-g001">Figure 1G</a>), which <a href="#pone.0059270-Asakura1">[17]</a> did not examine. </p> <p>Seed storage proteins such as glycinin and omega-6-fatty acid desaturase are of great agronomic importance and have been well-studied in soybean. As shown in <a href="#pone-0059270-g002">Figure 2A</a>, gene models annotated as glycinin and beta-conglycinin peak in RPKM at 400–500 mg cotyledon and 100–200 mg cotyledon, respectively. Other studies <a href="#pone.0059270-Meinke1">[15]</a>– <a href="#pone.0059270-Asakura1">[17]</a> have found that mRNAs encoding these two major soybean storage proteins show the same pattern of low expression at early stages, followed by beta-conglycinin increasing in expression before glycinin, with a dramatic decrease for both at the mature, dry seed stage. </p> <p>In contrast, most of the gene models annotated as fatty acid desaturase show steady expression throughout multiple stages of development ( <a href="#pone-0059270-g002">Figure 2E</a>). Genes annotated as omega-6 desaturase were found to have a similar expression pattern in equivalent soybean seed stages <a href="#pone.0059270-Asakura1">[17]</a>, with a peak at the stage of largest fresh weight. Both omega-6 and omega-3-fatty acid desaturase are involved in the conversion of oil in the soybean seed to more highly desaturated forms throughout development. </p> <p>At the final stage of development, the seed is undergoing desiccation in preparation for a quiescent period prior to imbibition and germination. Almost all of the highly-expressed gene models shown in <a href="#pone-0059270-g001">Figure 1G</a> are annotated as hydrophilic proteins associated with low water conditions in plants. The late embryogenesis abundant (LEA) proteins (of which dehydrins are a subfamily) are known to accumulate both in the desiccating stage of seed development as well as in vegetative tissues experiencing environmentally-induced water loss <a href="#pone.0059270-Bray1">[25]</a>, <a href="#pone.0059270-GarayArroyo1">[26]</a>. These proteins may protect cellular structures in a variety of ways, such as by stabilizing cell membranes, quarantining ions that have been concentrated by water loss, or binding the remaining available water <a href="#pone.0059270-Bray1">[25]</a>. LEA genes with a similar expression pattern were found in developing <i>Arabidopsis</i> seeds, peaking in expression at the mature embryo stage <a href="#pone.0059270-Peng1">[18]</a>. The LEA and dehydrin annotations are over-represented in this cluster, as they are found in &amp;lt;1% of annotations in the 78,773 gene models total. </p> <p>Transcription factors are crucial components of regulation systems which may initiate vital gene expression changes while displaying relatively low expression themselves. Thus when examining transcription factor gene models in this RNA-Seq data, an RPKM cut-off of just 25 was chosen. The wide variety of transcription factor families found to be highly expressed in our data reflects the complex network of regulation required during seed development, especially at the younger stages when seeds are undergoing a diverse array of morphological changes. </p> <p>The transcription factor gene models shown in <a href="#pone-0059270-g004">Figure 4A</a>, which peak at 22–24 DAF, have many annotations ( <a href="#pone-0059270-t006">Table 6</a>) which have also been found to be significantly expressed in developing <i>Arabidopsis</i> seeds with a similar pattern that peaks in early to middle development; such annotations include zinc finger proteins, GATA-binding factors, MYB domains, and members of the APETALA2-ethylene response factor family (AP2/ERF) <a href="#pone.0059270-Peng1">[18]</a>. MYB domain transcription factor <i>AtMYB123,</i> also known as <i>TRANSPARENT TESTA2</i>, is known to be involved in the regulation of proanthocyanidin production in developing <i>Arabidopsis</i> seed coats <a href="#pone.0059270-Nesi1">[27]</a>, <a href="#pone.0059270-Appelhagen1">[28]</a>. AUX/IAA family genes and auxin response factors, similar to those also found in this cluster, interact with the hormone auxin to regulate genes involved in the proper development of young embryos and cotyledons <a href="#pone.0059270-Jenik1">[29]</a>. Recently, certain MYB transcription factors as well as auxin response factors have been identified as the targets of microRNAs in multiple mature cotyledon stages of the soybean cultivar Williams (the same used in the present experiment), suggesting a mechanism by which some of these genes are down-regulated after a peak in expression at the early seed stages <a href="#pone.0059270-Shamimuzzaman1">[30]</a>. Small RNA profiles, including miRNAs for soybean, are available for certain stages of seed coat and cotyledon development <a href="#pone.0059270-Tuteja1">[31]</a>, <a href="#pone.0059270-Zabala1">[32]</a>. A keyword search was used to determine approximately what percentage of the 5608 transcription factor gene models fell into each family; about 14% of this larger group were annotated as Myb-like factors, suggesting this family is not over-represented in the young seed cluster with 11.4%. However, the auxin-related transcription factors comprise only about 2% of transcription factor gene models generally, but 6.6% of the cluster highly expressed at 22–24 DAF. Other families mentioned in <a href="#pone-0059270-t006">Table 6</a> which are over-represented in this young seed cluster include the global transcription factor group (4.8% of the cluster), the GAGA-binding protein family (3.1% of the cluster), and MADS-box transcription factors (3.1% of the cluster), each of which comprises &amp;lt;1% of the transcription factors found on the larger list. </p> <p>In contrast, the transcription factor gene models shown in <a href="#pone-0059270-g004">Figure 4C</a> peak in expression at the final, dry seed stage. Previous work from our lab <a href="#pone.0059270-Jones1">[7]</a> using cDNA and oligo microarrays with these same older stages of seed development found that an unexpectedly high percentage of transcription-related genes were expressed during the dry seed stage relative to the 100–200 mg cotyledon. The over-expressed transcription factor annotations reported included Nuclear Factor Y (NF-Y), zinc fingers, and AP2, which also appear on the current RNA-Seq list of highly expressed transcription factors at the dry seed stage ( <a href="#pone-0059270-t006">Table 6</a>). Some of these factors could be involved in preparing transcripts in the dry seed in anticipation of germination, decreasing the time and effort needed to produce essential proteins once imbibition begins. A study by <a href="#pone.0059270-Peng1">[18]</a> of developing <i>Arabidopsis</i> seeds also using microarray data noted that <i>ABSCISIC ACID INSENSITIVE 5</i> ( <i>ABI5</i>), a bZIP transcription factor, displayed a similar expression profile to <a href="#pone-0059270-g004">Figure 4C</a> and is known to regulate certain LEA genes, which also have this expression profile in both studies. Additionally, both NF-Y transcription factors and AP2/ERF family members have been implicated in drought responses in plants, suggesting a role they may play in desiccating seeds <a href="#pone.0059270-Li1">[33]</a>, <a href="#pone.0059270-Xu1">[34]</a>. According to the keyword classification of the list of 5608 transcription factor gene models, many of the annotations mentioned in <a href="#pone-0059270-t006">Table 6</a> which peak in expression at the dry seed stage are over-represented in this cluster. For example, AP2/ethylene responsive transcription factors and the CBF/NF-Y family comprise 19.4% and 10.2% of this cluster, respectively, but &amp;lt;10% and &amp;lt;5% of the larger list. The no apical meristem (NAM/NAC) family is found at 9.3% in the cluster compared to &amp;lt;4% overall, and heat shock transcription factors are found at 3.7% in the cluster versus &amp;lt;2% overall. </p> <p>It was noted that some transcription factor families, such as the AP2/ethylene responsive factors, have a variety of expression patterns throughout seed development, as different members regulate genes needed at different times ( <a href="#pone-0059270-g004">Figure 4E</a>). This large family of transcription factors, which also includes the dehydration-responsive element binding protein (DREB) and the related to ABI3/VP1 (RAV) genes, has been linked to a similarly wide variety of biotic and abiotic stresses and hormonal signaling, as well as certain developmental processes such as floral organ identity and root morphogenesis <a href="#pone.0059270-Dietz1">[35]</a>. MYB is another family in which different members are highly expressed at young stages ( <a href="#pone-0059270-g004">Figure 4A</a>) or dry seeds ( <a href="#pone-0059270-g004">Figure 4C</a>). </p> <p>However, other families, such as the no apical meristem transcription factors, have a much more specific expression pattern, in this case being highly expressed at the final, dry seed stage ( <a href="#pone-0059270-g004">Figure 4G</a>). NAM or NAC genes (so-called due to four early transcription factors found with the same new domain–petunia <i>NAM</i> plus <i>Arabidopsis ATAF1</i> and <i>2</i> and <i>CUC2)</i> are part of a large plant-specific family with a diverse array of functions, including biotic and abiotic stress responses, hormone signaling, leaf senescence, and meristem and primordia formation during embryogenesis <a href="#pone.0059270-Olsen1">[36]</a>. A large number of the NAC genes identified in soybean, including several found in this study to have high expression in the dry, mature seeds, are known to be involved in drought stress response in other tissues such as roots and shoots <a href="#pone.0059270-Le2">[37]</a>. This finding indicates these genes have important functions during tissue desiccation, whether brought on by environmental stress or normal developmental processes. </p> <p>In this study, seven stages of soybean seeds, from just a few days after fertilization to dry, mature seeds, were subjected to next-generation sequencing, resulting in millions of sequenced transcripts per stage. When aligned to nearly 79,000 predicted soybean gene models, this wealth of data provides a broad overview of the important processes shaping the development of one of the most important oilseed crops in the world. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialandrnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and RNA Extraction</div> <p>Immature soybean seeds ( <i>Glycine max</i> cv. Williams, maturity group III) were harvested from greenhouse-grown plants. The three earliest stages were harvested at the stated Days After Flowering (DAF) and the whole seeds were removed from the early pods under an Olympus SZ61 microscope (Melville, NY) and fresh-frozen in liquid nitrogen. For the next three stages (5–6 mg, 100–200 mg, 400–500 mg), whole seeds were sorted by the stated fresh weight ranges. 5–6 mg seeds were removed from the early pods and either the whole seeds or separated seed coats and cotyledons were fresh-frozen in liquid nitrogen. For 100–200 mg and 400–500 mg seeds the separated seed coats and cotyledons were lyophilized. Dry seeds were harvested at maturity (approximately 100–200 mg) and stored whole at room temperature. </p> <p>Total RNA was extracted from tissues using phenol:chloroform and a lithium chloride precipitation <a href="#pone.0059270-Gonzalez1">[24]</a>. RNA from the three oldest stages was further purified with a Qiagen RNeasy Midi kit (Valencia, CA). Soybean is highly inbred, but in order to minimize biological variation, RNA was extracted and combined from multiple seeds and plants, using between four and one hundred different seeds depending on the stage (younger stages required more seeds be extracted at the same time to yield usable amounts of RNA). RNA from separate biological samples was used for the two biological replicates. 5–6 mg separated seed coat and cotyledon data come from only a single biological replicate. </p> </div> <div class="high-throughputrnasequencingandalignment" title="sec"> <div class="title" tagx="title" title="title">
High-Throughput RNA Sequencing and Alignment</div> <p>Total RNA was sequenced by the Keck Center (University of Illinois) using an Illumina Genome Analyzer II for the Biological Replicate 1 samples (except for 22–24 DAF), and using an Illumina HiSeq 2000 for the Biological Replicate 2 samples (and also 22–24 DAF of Bio Rep 1, 5–6 mg separated seed coat, and 5–6 mg separated cotyledon). In Biological Replicate 1, the 22–24 DAF and 100–200 mg cotyledon samples were sequenced using the single read method while the other samples were sequenced with the paired-ends method; in Biological Replicate 2, all samples used the single read method. 5–6 mg separated seed coat and cotyledon also used the single read method. For both biological replicates, as well as 5–6 mg separated seed coat and cotyledon, all samples had reads 75 bp in length after processing, except the 4 DAF and dry whole seed samples of Biological Replicate 2, which had reads 100 bp in length. The technical differences between the sample sequencing methods reflect improvements in the standard high-throughput RNA sequencing techniques over time. Sequencing data was filtered through the standard Illumina pipeline including the CASSAVA program. </p> <p>The reads were aligned to 78,773 Glyma1 cDNA soybean gene models determined by the Soybean Genome Project, Department of Energy, Joint Genome Institute <a href="#pone.0059270-Schmutz1">[1]</a>, using the alignment program Bowtie <a href="#pone.0059270-Langmead1">[8]</a>. Bowtie parameters allowed up to three mismatches and up to 25 alignments per read; reads aligned more than 25 times were discarded, with none of their alignments being counted. </p> </div> <div class="sequencingdata,genemodelannotations,clustering,andp-values" title="sec"> <div class="title" tagx="title" title="title">
Sequencing Data, Gene Model Annotations, Clustering, and P-Values</div> <p>Data is given in RPKMs <a href="#pone.0059270-Mortazavi1">[9]</a>, which stands for reads per kilobase of gene model per million mapped reads. The total number of mapped reads takes into account reads that aligned up to 25 different times, including multiple matches within the same gene model. </p> <p>Gene model annotations by PFAM were from the Soybean Genome Project in the Phytozome database version 4.0 <a href="#pone.0059270-Goodstein1">[38]</a>. Annotations of gene models using BLASTX (e-value 1e-4 or better) against NCBI’s non-redundant database version 8.0.1 have been described <a href="#pone.0059270-Hunt1">[39]</a>. The top 5 hits, as well as five others from species of interest (such as <i>Glycine max</i>), were compiled for each gene model. Gene models were grouped into families by manual inspection using these annotations. </p> <p>Gene models were clustered using Multi-Experiment Viewer (MeV) <a href="#pone.0059270-Saeed1">[10]</a> using data from Biological Replicate 1. The k-means clustering technique was used with the Pearson correlation distance metric and 2500 maximum iterations. P-values (adjusted with the Benjamini-Hochberg procedure) both between biological replicates and between stages were generated using DESeq <a href="#pone.0059270-Anders1">[40]</a> in Bioconductor <a href="#pone.0059270-Gentleman1">[41]</a>. </p> <p>Raw and processed sequencing data are available in the Gene Expression Omnibus database at <a href="http://www.ncbi.nlm.nih.gov/geo/as">http://www.ncbi.nlm.nih.gov/geo/as</a> series accession number GSE42871. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p> <b>Clusters of gene models with high expression in young seed stages.</b> Clusters produced using Biological Replicate 1 data (left column) depict gene models with RPKM≥500 in at least one of four young seed stages, and RPKM &amp;lt;500 in all three older stages. The right column shows the same gene models with Biological Replicate 2 data. Cluster numbers (determined by MeV) are shown at the far right. A and B are repeated from <a href="#pone-0059270-g001">Figure 1C and 1D</a>; K and L are repeated from <a href="#pone-0059270-g001">Figure 1A and 1B</a>. Note the different scales used by E and F. The Y axis of J has been set at 1500, cutting off the blue line, which reaches an RPKM of over 2700 at 5–6 mg whole seed. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0059270.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p> <b>Clusters of gene models with high expression in older seed stages.</b> Clusters produced using Biological Replicate 1 data (left column) depict gene models with RPKM≥1000 in at least one of three older seed stages. The right column shows the same gene models with Biological Replicate 2 data. Cluster numbers (determined by MeV) are shown at the far right. C and D are repeated from <a href="#pone-0059270-g001">Figure 1G and 1H</a>; M and N are repeated from <a href="#pone-0059270-g001">Figure 1E and 1F</a>. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0059270.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p> <b>All clusters of transcription factor gene models.</b> Clusters produced using Biological Replicate 1 data (left column) depict gene models annotated as transcription factors with RPKM≥25 in at least one of seven stages of seed development. The right column shows the same gene models with Biological Replicate 2 data. Cluster numbers (determined by MeV) are shown at the far right. Annotations were derived from PFAM and/or the NCBI non-redundant database. Stages are numbered in order on the x axis: 4 DAF whole seed, 12–14 DAF whole seed, 22–24 DAF whole seed, 5–6 mg whole seed, 100–200 mg cotyledon, 400–500 mg cotyledon, dry whole seed. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0059270.s003.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p> <b>Gene models depicted in three sets of clusters.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0059270.s004.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p> <b>Selected gene models annotated as storage proteins and transcription factors.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0059270.s005.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We are grateful to Cameron Lowe and Achira Kulasekara for help with data analysis. We gratefully acknowledge the sequencing services of the Keck Center, University of Illinois. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0059270-Schmutz1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmutz</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlueter</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitros</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the palaeopolyploid soybean</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">463</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913">20075913</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Le1" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le</span> <span class="given-names" tagx="given-names" title="given-names">BH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagmaister</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bui</span> <span class="given-names" tagx="given-names" title="given-names">AQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harada</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Using genomics to study legume seed development</span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">144</span>: <span class="fpage" tagx="fpage" title="fpage">562</span>– <span class="lpage" tagx="lpage" title="lpage">574</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17556519">17556519</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Sha1" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sha</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shan</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">XA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Large-scale sequencing of normalized full-length cDNA library of soybean seed at different developmental stages and analysis of the gene expression profiles based on ESTs </span>. <span class="source" tagx="source" title="source">Mol Biol Rep</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">2867</span>– <span class="lpage" tagx="lpage" title="lpage">2874</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21667246">21667246</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Carlson1" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Carlson JB, Lersten NR (2004) Reproductive morphology. In: Boerma HR, Specht JE, editors. Soybeans: improvement, production, and uses. Madison WI: American Society of Agronomy. 59–95. </span> </li> <li tag="ref"> <a name="pone.0059270-Ritchie1" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ritchie SW, Hanway JJ, Thompson HE, Benson GO (1996) How a soybean plant develops. Special Report No. 53. Ames, IA: Iowa State University of Science and Technology Cooperative Extension Service. </span> </li> <li tag="ref"> <a name="pone.0059270-Bewley1" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bewley JD, Hempel FD, McCormick S, Zambryski P (2000) Reproductive development. In: Buchanan BB, Gruissem W, Jones RL, editors. Biochemistry and molecular biology of plants. Rockville MD: American Society of Plant Physiologists. 988–1043. </span> </li> <li tag="ref"> <a name="pone.0059270-Jones1" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">SI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">DO</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Flux of transcript patterns during soybean seed development</span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">136</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20181280">20181280</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Langmead1" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pop</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">R25</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174">19261174</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Mortazavi1" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mortazavi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">William</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCue</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaeffer</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wold</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">621</span>– <span class="lpage" tagx="lpage" title="lpage">628</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045">18516045</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Saeed1" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saeed</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharov</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">TM4: a free, open-source system for microarray data management and analysis</span>. <span class="source" tagx="source" title="source">Biotechniques</span> <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">374</span>– <span class="lpage" tagx="lpage" title="lpage">378</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12613259">12613259</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Hong1" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagao</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Key</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> ( <span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">Developmentally regulated expression of soybean proline-rich cell wall protein genes</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">937</span>– <span class="lpage" tagx="lpage" title="lpage">943</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2535533">2535533</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Lindstrom1" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindstrom</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1991</span>) <span class="mixed-article-title" title="mixed-article-title">A soybean cell wall protein is affected by seed color genotype</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">561</span>– <span class="lpage" tagx="lpage" title="lpage">571</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1841717">1841717</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Panavas1" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panavas</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pikula</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reid</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubinstein</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">EL</span> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Identification of senescence-associated genes from daylily petals</span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">237</span>– <span class="lpage" tagx="lpage" title="lpage">248</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10412903">10412903</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Lim1" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">PO</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nam</span> <span class="given-names" tagx="given-names" title="given-names">HG</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Leaf senescence</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">115</span>– <span class="lpage" tagx="lpage" title="lpage">136</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17177638">17177638</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Meinke1" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meinke</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beachy</span> <span class="given-names" tagx="given-names" title="given-names">RN</span> </span> ( <span class="year" tagx="year" title="year">1981</span>) <span class="mixed-article-title" title="mixed-article-title">Expression of storage-protein genes during soybean seed development</span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">153</span>: <span class="fpage" tagx="fpage" title="fpage">130</span>– <span class="lpage" tagx="lpage" title="lpage">139</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24276763">24276763</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Nielsen1" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">NC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dickinson</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thanh</span> <span class="given-names" tagx="given-names" title="given-names">VH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scallon</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization of the glycinin gene family in soybean</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">313</span>– <span class="lpage" tagx="lpage" title="lpage">328</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2485233">2485233</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Asakura1" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asakura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terauchi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narikawa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yagasaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Global gene expression profiles in developing soybean seeds</span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span> <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">147</span>– <span class="lpage" tagx="lpage" title="lpage">153</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22245912">22245912</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Peng1" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">FY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weselake</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Gene coexpression clusters and putative regulatory elements underlying seed storage reserve accumulation in <i>Arabidopsis.</i> </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">286</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21635767">21635767</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Stephenson1" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephenson</span> <span class="given-names" tagx="given-names" title="given-names">LC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bunker</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubbs</span> <span class="given-names" tagx="given-names" title="given-names">WE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimes</span> <span class="given-names" tagx="given-names" title="given-names">HD</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Specific soybean lipoxygenases localize to discrete subcellular compartments and their mRNAs are differentially regulated by source-sink status </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">116</span>: <span class="fpage" tagx="fpage" title="fpage">923</span>– <span class="lpage" tagx="lpage" title="lpage">933</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9501125">9501125</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-SkrzypczakJankun1" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skrzypczak-Jankun</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borbulevych</span> <span class="given-names" tagx="given-names" title="given-names">OY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jankun</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Soybean lipoxygenase-3 in complex with 4-nitrocatechol</span>. <span class="source" tagx="source" title="source">Acta Crystallogr D Biol Crystallogr</span> <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">613</span>– <span class="lpage" tagx="lpage" title="lpage">615</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14993710">14993710</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-IslasFlores1" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Islas-Flores</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corrales-Villamar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bearer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raya</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Villanueva</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Isolation of lipoxygenase isoforms from <i>Glycine max</i> embryo axes based on apparent cross-reactivity with anti-myosin antibodies </span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="volume" tagx="volume" title="volume">1571</span>: <span class="fpage" tagx="fpage" title="fpage">64</span>– <span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12031291">12031291</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Baulcombe1" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baulcombe</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Key</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> ( <span class="year" tagx="year" title="year">1980</span>) <span class="mixed-article-title" title="mixed-article-title">Polyadenylated RNA sequences which are reduced in concentration following auxin treatment of soybean hypocotyls </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">255</span>: <span class="fpage" tagx="fpage" title="fpage">8907</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. </span> </li> <li tag="ref"> <a name="pone.0059270-ThibaudNissen1" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thibaud-Nissen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shealy</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khanna</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Clustering of microarray data reveals transcript patterns associated with somatic embryogenesis in soybean </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">118</span>– <span class="lpage" tagx="lpage" title="lpage">136</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12746518">12746518</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Gonzalez1" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">DO</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Specific elements of the glyoxylate pathway play a significant role in the functional transition of the soybean cotyledon during seedling development </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">468</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18093333">18093333</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Bray1" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bray</span> <span class="given-names" tagx="given-names" title="given-names">EB</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular responses to water deficit</span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">1035</span>– <span class="lpage" tagx="lpage" title="lpage">1040</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12231998">12231998</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-GarayArroyo1" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garay-Arroyo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colmenero-Flores</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garciarrubio</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Covarrubias</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Highly hydrophilic proteins in prokaryotes and eukaryotes are common during conditions of water deficit </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">275</span>: <span class="fpage" tagx="fpage" title="fpage">5668</span>– <span class="lpage" tagx="lpage" title="lpage">5674</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10681550">10681550</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Nesi1" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nesi</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jond</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debeaujon</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caboche</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lepiniec</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> TT2 gene encodes an R2R3 MYB domain protein that acts as a key determinant for proanthocyanidin accumulation in developing seed </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">2099</span>– <span class="lpage" tagx="lpage" title="lpage">2114</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11549766">11549766</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Appelhagen1" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Appelhagen</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huep</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmelzer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weisshaar</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">TRANSPARENT TESTA1 interacts with R2R3-MYB factors and affects early and late steps of flavonoid biosynthesis in the endothelium of <i>Arabidopsis thaliana</i> seeds </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">67</span>: <span class="fpage" tagx="fpage" title="fpage">406</span>– <span class="lpage" tagx="lpage" title="lpage">419</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21477081">21477081</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Jenik1" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenik</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barton</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Surge and destroy: the role of auxin in plant embryogenesis</span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">3577</span>– <span class="lpage" tagx="lpage" title="lpage">85</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16077088">16077088</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Shamimuzzaman1" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shamimuzzaman</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Identification of soybean seed developmental stage-specific and tissue-specific miRNA targets by degradome sequencing </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">310</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22799740">22799740</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Tuteja1" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabala</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varala</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hudson</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Endogenous, tissue-specific short interfering RNAs silence the chalcone synthase gene family in <i>Glycine max</i> seed coats </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3063</span>– <span class="lpage" tagx="lpage" title="lpage">3077</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19820189">19820189</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Zabala1" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabala</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campos</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varala</span> <span class="given-names" tagx="given-names" title="given-names">KK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bloomfield</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">SI</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Divergent patterns of endogenous small RNA populations from seed and vegetative tissues of <i>Glycine max</i> </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">177</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23031057">23031057</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Li1" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">WX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oono</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> NFYA5 transcription factor is regulated transcriptionally and posttranscriptionally to promote drought resistance </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">2238</span>– <span class="lpage" tagx="lpage" title="lpage">2251</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18682547">18682547</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Xu1" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">ZS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">LC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">YZ</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Functions and application of the AP2/ERF transcription factor family in crop improvement</span>. <span class="source" tagx="source" title="source">J Integr Plant Biol</span> <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">570</span>– <span class="lpage" tagx="lpage" title="lpage">585</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21676172">21676172</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Dietz1" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dietz</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogel</span> <span class="given-names" tagx="given-names" title="given-names">MO</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Viehhauser</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">AP2/EREBP transcription factors are part of gene regulatory networks and integrate metabolic, hormonal and environmental signals in stress acclimation and retrograde signaling </span>. <span class="source" tagx="source" title="source">Protoplasma</span> <span class="volume" tagx="volume" title="volume">245</span>: <span class="fpage" tagx="fpage" title="fpage">3</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20411284">20411284</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Olsen1" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsen</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ernst</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leggio</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skriver</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors: structurally distinct, functionally diverse</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">87</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15708345">15708345</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Le2" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishiyama</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watanabe</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mochida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Genome-wide survey and expression analysis of the plant-specific NAC transcription factor family in soybean during development and dehydration stress </span>. <span class="source" tagx="source" title="source">DNA Res</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">263</span>– <span class="lpage" tagx="lpage" title="lpage">276</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21685489">21685489</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Goodstein1" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodstein</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howson</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neupane</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Phytozome: a comparative platform for green plant genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">D1178</span>– <span class="lpage" tagx="lpage" title="lpage">D1186</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026">22110026</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Hunt1" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strömvik</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Transcript profiling reveals expression differences in wild-type and glabrous soybean lines </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">145</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22029708">22029708</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Anders1" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anders</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Differential expression analysis for sequence count data</span>. <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">R106</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621">20979621</a> </span> </span> </li> <li tag="ref"> <a name="pone.0059270-Gentleman1" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gentleman</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carey</span> <span class="given-names" tagx="given-names" title="given-names">VJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bates</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bolstad</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dettling</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Bioconductor: open software development for computational biology and bioinformatics</span>. <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">R80</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15461798">15461798</a> </span> </span> </li> </ul> </div> </div>  </body></html>