<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4810360">4810360</a> </span> <span class="pii" title="pii">srep23609</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/srep23609">10.1038/srep23609</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
The NAC transcription factor ANAC046 is a positive regulator of chlorophyll degradation and senescence in Arabidopsis leaves </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda-Yamamizo</span> <span class="given-names" tagx="given-names" title="given-names">Chihiro</span> </span> <a href="#a1">1</a> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitsuda</span> <span class="given-names" tagx="given-names" title="given-names">Nobutaka</span> </span> <a href="#a3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakamoto</span> <span class="given-names" tagx="given-names" title="given-names">Shingo</span> </span> <a href="#a3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogawa</span> <span class="given-names" tagx="given-names" title="given-names">Daisuke</span> </span> <a href="#a4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohme-Takagi</span> <span class="given-names" tagx="given-names" title="given-names">Masaru</span> </span> <a href="#a5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohmiya</span> <span class="given-names" tagx="given-names" title="given-names">Akemi</span> </span> <a href="#c1">a</a> <a href="#a1">1</a> </span> <span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">National Agriculture and Food Research Organization (NARO), Institute of Floricultural Science </span> <span class="country" tagx="country" title="country">Japan</span> </span> <span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Research Fellow of Japanese Society for the Promotion of Science (JSPS)</span> <span class="country" tagx="country" title="country">Japan</span> </span> <span class="citation_author_institution" id="a3">[3], <span class="institution" title="institution">Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST) </span> <span class="country" tagx="country" title="country">Japan</span> </span> <span class="citation_author_institution" id="a4">[4], <span class="institution" title="institution">Horticultural Experiment Center, Wakayama Prefectural Agricultural Research Station, Gobo </span> <span class="country" tagx="country" title="country">Japan</span> </span> <span class="citation_author_institution" id="a5">[5], <span class="institution" title="institution">Institute for Environmental Science and Technology (IEST), Saitama University</span> <span class="country" tagx="country" title="country">Japan</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">a</span> <span class="email" tagx="email" title="email">ohmiya@affrc.go.jp</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-03-03</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">23609</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-01-18</span> <span class="accepted" title="accepted">accepted: 2016-03-03</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Macmillan Publishers Limited</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Chlorophyll (Chl) degradation occurs during leaf senescence, embryo degreening, bud breaking, and fruit ripening. The Chl catabolic pathway has been intensively studied and nearly all the enzymes involved are identified and characterized; however, the molecular regulatory mechanisms of this pathway are largely unknown. In this study, we performed yeast one-hybrid screening using a transcription factor cDNA library to search for factors controlling the expression of Chl catabolic genes. We identified ANAC046 as a common regulator that directly binds to the promoter regions of <i>NON-YELLOW COLORING1</i>, <i>STAY-GREEN1</i> ( <i>SGR1</i>), <i>SGR2</i>, and <i>PHEOPHORBIDE a OXYGENASE</i>. Transgenic plants overexpressing <i>ANAC046</i> exhibited an early-senescence phenotype and a lower Chl content in comparison with the wild-type plants, whereas loss-of-function mutants exhibited a delayed-senescence phenotype and a higher Chl content. Microarray analysis of <i>ANAC046</i> transgenic plants showed that not only Chl catabolic genes but also senescence-associated genes were positively regulated by ANAC046. We conclude that ANAC046 is a positive regulator of Arabidopsis leaf senescence and exerts its effect by controlling the expression of Chl catabolic genes and senescence-associated genes. </p> </div> </div> </div> <div class="body" title="body"> <p>Loss of green color, the most visually apparent phenomenon during leaf senescence, is caused by chlorophyll (Chl) degradation. This process facilitates redistribution of nutrients from senescent leaves to reproductive organs and increases reproductive success <a href="#b1">1</a>. In addition, loss of Chl in flowers and fruits is an important trait that enables them easily visible among leaves to attract pollinators and seed dispersers. </p> <p>During past three decades, identification of Chl catabolite structures <a href="#b2">2</a> and analysis of stay-green mutants, in which Chl degradation is impaired <a href="#b3">3</a> <a href="#b4">4</a>, have facilitated the elucidation of the Chl catabolic pathway. To date, nearly all enzymes involved in Chl degradation have been identified and characterized. In higher plants, reduction of Chl <i>b</i> to Chl <i>a</i> is the initial step of Chl breakdown ( <a href="#f1">Fig. 1</a>). Chl <i>b</i> reductase, encoded by <i>NON-YELLOW COLORING 1</i> ( <i>NYC1</i>) and <i>NYC1-like</i> ( <i>NOL</i>), catalyzes the conversion of Chl <i>b</i> to 7-hydroxymethyl Chl <i>a</i> <a href="#b5">5</a> <a href="#b6">6</a>, which is then converted into Chl <i>a</i> by 7-hydroxymethyl-chlorophyll <i>a</i> reductase (HCAR) <a href="#b7">7</a>. The central Mg atom of Chl <i>a</i> is removed by a metal-chelating substance (MCS), the molecular nature of which has not been identified, to form pheophytin <i>a</i>. Pheophytinase (PPH) then catalyzes the removal of the phytol tail of pheophytin <i>a</i> to form pheophorbide <i>a</i> <a href="#b8">8</a> <a href="#b9">9</a>. Subsequently, the porphyrin macrocycle ring of pheophorbide <i>a</i> is opened by pheophorbide <i>a</i> oxygenase (PaO) to produce red chlorophyll catabolite (RCC) <a href="#b10">10</a> <a href="#b11">11</a>. RCC is converted to nonfluorescent chlorophyll catabolite (NCC) or nonfluorescent dioxobilin-type chlorophyll catabolite (NDCC), end-products of Chl breakdown, via <i>primary</i> fluorescent Chl catabolite ( <i>p</i>FCC) by the actions of RCC reductase (RCCR), MES16, and CYP89A9 <a href="#b12">12</a> <a href="#b13">13</a> <a href="#b14">14</a>. Chl breakdown also requires a protein termed STAY GREEN (SGR). <i>Arabidopsis</i> has three SGR homologs, SGR1, SGR2, and SGR-like <a href="#b15">15</a>. SGR1 and SGR-like positively regulate Chl degradation in senescing and pre-senescing leaves, respectively, whereas SGR2 negatively regulates Chl degradation in senescing leaves <a href="#b16">16</a> <a href="#b17">17</a>. Many Chl catabolic genes, including <i>NYC1</i>, <i>PPH</i>, <i>PaO</i>, <i>MES16</i>, and <i>SGR1</i> are coordinately expressed and their transcript levels increase in senescing leaves <a href="#b4">4</a>. </p> <p>Senescence is accompanied by transcriptional reprogramming of a large number of genes. In senescent leaves of Arabidopsis, more than 100 transcription factors (TFs) from various families, including NAC (no apical meristem [NAM], ATAF1/2, and cup-shaped cotyledon [CUC2]), WRKY, zinc finger proteins, and AP2/EREBP are upregulated <a href="#b18">18</a>. In particular, plant-specific NAC superfamily constitutes a large portion of senescence-regulated genes in Arabidopsis <a href="#b18">18</a> <a href="#b19">19</a> <a href="#b20">20</a>. Several NAC TFs, including AtNAP/ANAC029 <a href="#b21">21</a>, ORESARA1/ANAC092 <a href="#b20">20</a> <a href="#b21">21</a> <a href="#b22">22</a> <a href="#b23">23</a>, ORE1 SISTER1/ANAC059 (ORS1) <a href="#b24">24</a>, and AtNAC016 <a href="#b25">25</a>, positively regulate leaf senescence. Their downstream target genes, including senescence-associated genes (SAGs), have been identified <a href="#b20">20</a>. Only limited information is available on whether the senescence-related NAC TFs are involved in the control of Chl catabolic gene expression. Recently, Qiu <i>et al</i>. <a href="#b26">26</a> reported that ORE1 directly binds to the promoter regions of <i>SGR1</i>, <i>NYC1</i>, and <i>PaO</i>. </p> <p>Towards an understanding the mechanism of Chl degradation, we performed comprehensive screening of TFs that bind to the promoter regions of Chl catabolic genes using an improved yeast one-hybrid (Y1H) screening system with an Arabidopsis TF cDNA library <a href="#b27">27</a>. Because the TF cDNA library is composed of 10% or less number of all Arabidopsis genes, the rate of false positives is much lower than in traditional screening. Another advantage of this approach is that we can directly employ the promoter region as bait. Here, we report identification of ANAC046, a previously uncharacterized member of the NAC superfamily, as a common regulator of the expression of <i>NYC1</i>, <i>SGR1</i>, <i>SGR2</i>, and <i>PaO</i>. ANAC046 belongs to the same subgroup as ORE1 and ORS1. Our analysis of mutant and transgenic <i>ANAC046</i> plants showed that ANAC046, ORE1, and ORS1 share common functions but also have distinct roles in leaf senescence. </p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="identificationoftranscriptionfactorsthatbindtothepromoterregionsofchlorophyllcatabolicgenes" title="sec"> <div class="title" tagx="title" title="title">
Identification of transcription factors that bind to the promoter regions of chlorophyll catabolic genes </div> <p>To identify the TFs that directly control Chl degradation, we first constructed yeast strains carrying the <i>HIS3</i> reporter gene driven by the promoter regions of the following Chl catabolic genes: <i>NYC1</i>, <i>NOL</i>, <i>HCAR</i>, <i>PPH</i>, <i>PaO</i>, <i>RCCR</i>, <i>CYP89A9</i>, <i>MES16</i>, <i>SGR1</i>, and <i>SGR2</i>. Then, we examined binding of ca. 1300 TFs (ca. 70% of all Arabidopsis TFs) to these regions. After the initial screening with 343 mini-pools, we performed a second screening and identified 93 interactions between the 10 promoters and 54 individual TFs ( <a href="#S1">Fig. S1</a>). Among them, we were interested in the particular group of NAC TFs because three genes namely ANAC046, ANAC087, and ANAC100 were detected in experiments employing the promoters of <i>NYC1</i>, <i>SGR1</i>, <i>SGR2</i>, and <i>PaO</i> ( <a href="#S1">Fig. S1</a>). Phylogenetic analysis clustered these three proteins into the same clade as ANAC079, ORE1, and ORS1 in the NAM subgroup of group I of the NAC superfamily ( <a href="#S1">Fig. S2</a>) <a href="#b28">28</a>. <i>ANAC079</i>, <i>ANAC100</i>, and <i>ORE1</i> have miR164-complementary sites with ≤3 mismatches to miR164 and are likely to be target genes for this miRNA ( <a href="#S1">Fig. S3</a>). </p> <p>Next, we examined the binding of six TFs to four promoters in a separate Y1H experiment ( <a href="#f2">Fig. 2a</a>). ANAC046 bound to all the promoters tested. For ANAC087, the TF most similar to ANAC046 ( <a href="#S1">Fig. S2</a>), the affinity to all of the promoters was low. ORE1 bound to the <i>SGR2</i> and <i>PaO</i> promoters, and ORS1 did not bind to any of the promoters tested. Therefore, we chose ANAC046 for further analysis. </p> <p>To examine the interactions between ANAC046 and promoters of the 12 chlorophyll catabolic genes, we employed transient effector–reporter analysis in leaf protoplasts. As shown in <a href="#f2">Fig. 2b</a>, the VP16-fused form of ANAC046 clearly activated the reporter driven by the promoters of <i>NYC1</i>, <i>SGR1</i>, <i>SGR2</i>, and <i>PaO</i>, suggesting <i>in planta</i> binding of ANAC046 to these promoters. Thus, the results obtained in leaf protoplasts were consistent with those obtained in yeast. ANAC046 also activated <i>MES16</i> promoter <i>in planta</i> with less extent, although this activation was not observed in yeast. </p> </div> <div class="phenotypicanalysisofanac046transgenicplants" title="sec"> <div class="title" tagx="title" title="title">
Phenotypic Analysis of <i>ANAC046</i> Transgenic Plants </div> <p>To elucidate the function of ANAC046 in Chl degradation, we produced transgenic Arabidopsis plants constitutively overexpressing <i>ANAC046</i> ( <i>35S:ANAC046</i>; OX) and plants expressing a chimeric repressor containing the SRDX domain at the C-terminus ( <i>35S:ANAC046-SRDX</i>; SRDX). The SRDX plant is expected to show loss-of-function phenotype of <i>ANAC046</i> and its functionally redundant transcription factors as a result of dominant suppression of its target genes <a href="#b29">29</a>. We also obtained a putative knockout mutant, SM_3_27647 (KO), which has a transposon insertion in the <i>ANAC046</i> coding region. </p> <p>The OX plants germinated on MS-agar slightly later than WT plants ( <a href="#f3">Fig. 3a</a>). At 40 days after sowing (DAS), the OX plants had wavy rosette leaves and were remarkably smaller than WT plants ( <a href="#f3">Fig. 3b</a>). The KO and SRDX plants were slightly larger than WT plants during the course of development. The dry weight of aerial parts and roots was significantly lower in OX plants and that of roots was significantly higher in KO and SRDX plants than in WT plants ( <a href="#f3">Fig. 3c</a>). Chl content in the 8–9th leaf at 40 DAS was slightly higher in the KO and SRDX plants and significantly lower in OX plants than in the WT plants ( <a href="#f3">Fig. 3d</a>). </p> <p>To compare the ultrastructure of chloroplasts, the leaves of WT, OX, KO, and SRDX plants at 40 DAS were analyzed under a transmission electron microscope (TEM). At the end of the light period, chloroplast structure was similar in all plants tested: all mesophyll chloroplasts contained several large starch granules ( <a href="#f3">Fig. 3e</a>), which were produced by photosynthesis during the day. At the end of the dark period, only small starch granules were visible in the chloroplasts of the WT, KO, and SRDX plants, whereas chloroplasts of the OX plants still contained several large starch granules ( <a href="#f3">Fig. 3e</a>). In line with these observations, iodine staining showed that the leaves of OX plants, but not those of the WT, KO, or SRDX plants, contained a high level of starch at the end of the dark period, whereas the starch level was high in the leaves of all plants at the end of the light period ( <a href="#f3">Fig. 3f</a>). </p> </div> <div class="involvementofanac046inleafsenescence" title="sec"> <div class="title" tagx="title" title="title">
Involvement of ANAC046 in leaf senescence</div> <p>To determine the involvement of ANAC046 in age-dependent senescence, we compared the plant phenotypes and Chl content in the leaves of OX, KO, SRDX, and WT plants at 35 and 70 DAS. Leaf yellowing was accelerated in the OX plants and delayed in the KO and SRDX plants in comparison with the WT plants ( <a href="#f4">Fig. 4a</a>). Leaf Chl content was significantly higher in the KO and SRDX plants than in the WT plants at both 35 and 70 DAS ( <a href="#f4">Fig. 4b</a>). At 70 DAS, Chl content in the leaves of the OX plants was approximately half that of the WT plants. CO <sub>2</sub> assimilation rate was not significantly different among WT, OX, KO, and SRDX plants at 35 DAS, but was significantly higher in the KO and SRDX plants and slightly (but not significantly) lower in the OX plants than in the WT plants at 70 DAS ( <a href="#f4">Fig. 4c</a>). Ion leakage rate (a senescence indicator) was similar in all plants until 56 DAS ( <a href="#f4">Fig. 4d</a>). At 70 DAS, it was significantly higher in the OX plants but significantly lower in the KO and SRDX plants than in the WT plants. </p> <p>The levels of photosynthetic proteins associated with PSI and PSII were assessed by western blotting with antibodies against the core proteins (PsaA and PsbA, respectively) and the antenna proteins of the light-harvesting complexes (Lhca1 and Lhcb1). The levels of the large subunit of Rubisco (RbcL), which is involved in carbon fixation, were also examined. At 35 DAS, we did not detect remarkable differences in the levels of the photosynthetic proteins among plants tested ( <a href="#f4">Fig. 4e</a>). At 70 DAS, much higher levels of all proteins were detected in the KO and SRDX plants than in the WT plants, whereas all the proteins tested were below the detection limit in the OX plants. </p> <p>We also analyzed the effect of ANAC046 on dark-induced senescence by transferring the plants at 35 DAS to continuous darkness. Leaves of the OX plants grown on MS-agar were pale green and somewhat smaller than WT plants at 35 DAS ( <a href="#f5">Fig. 5a</a>). Chl content decreased faster in the OX plants, but slower in the KO and SRDX plants, than in the WT plants ( <a href="#f5">Fig. 5b</a>). The SRDX plants retained a high level of Chl even at 13 days after dark treatment (DAD). Ion leakage rate was significantly higher in the OX plants at 4 and 13 DAD and significantly lower in SRDX plants at 8 and 13 DAD than in the WT plants ( <a href="#f5">Fig. 5c</a>). The levels of all the photosynthetic proteins tested drastically decreased at 4 DAD in the OX plants but remained constant in the WT, KO, and SRDX plants ( <a href="#f5">Fig. 5d</a>). They were below the detection limit at 8 DAD in the OX plants. In contrast, the KO and SRDX plants contained substantial amounts of photosynthetic proteins even at 13 DAD, when these proteins became undetectable in WT plants. </p> <p>To obtain a comprehensive view of transcriptomic changes in transgenic plants, we performed microarray experiments ( <a href="#f6">Fig. 6</a>). We used RNAs from three independent lines of the OX and SRDX plants for the analysis. The level of <i>ANAC046</i> expression in these transgenic plants was more than 17-fold higher than that of WT ( <a href="#S1">Fig. S4</a>). We also confirmed that <i>ANAC046</i> expression in the KO plants was extremely low. We found that SRDX and KO plants, which had similar phenotypes, also had similar transcriptomes (r = 0.71; <a href="#S1">Fig. S5</a>). The expression of Chl catabolic genes, including <i>NYC1</i>, <i>CLH1</i>, <i>PPH</i>, <i>PaO</i>, <i>SGR1</i>, and <i>SGR2</i> was markedly higher in the OX plants and lower in the KO and SRDX plants than in the WT plants ( <a href="#f6">Fig. 6a</a>). Likewise, the expression of marker genes for leaf senescence, including <i>SAG12</i> and <i>SAG13</i>, was also higher in the OX plants and lower in the KO and SRDX plants than in WT plants ( <a href="#f6">Fig. 6b</a>). We also examined age-dependent changes in the expression of genes upregulated in each of the three transgenic lines by using publicly available microarray data of wild type ( <a href="#f6">Fig. 6c</a>) <a href="#b30">30</a>. The expression of most genes upregulated in the OX plants increased with age, whereas the expression of most genes upregulated in the KO and SRDX plants decreased with age ( <a href="#f6">Fig. 6c</a>). These data clearly support accelerated senescence of OX plants and delayed senescence of KO and SRDX plants. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The Chl degradation pathway consists of six chloroplast-located enzymes, two cytoplasm-located enzymes, a metal-chelating substance, and the key regulator SGR, and produces nonfluorescent chlorophyll catabolites <a href="#b3">3</a> <a href="#b4">4</a> <a href="#b31">31</a>. In Arabidopsis leaves, the expression of Chl catabolic genes is highly coordinated during senescence <a href="#b4">4</a>, suggesting the existence of TFs that bind to the promoter regions and coordinate the expression of a number of Chl catabolic genes. In this study, we found that several NAC TFs, including ANAC046, ANAC087, and ANAC100, directly bind to the promoter regions of <i>NYC1</i>, <i>SGR1</i>, <i>SGR2</i>, and <i>PaO</i>. It is worthy of note that the expression of these Chl catabolic enzymes is well correlated with that of these three NAC TFs, with both groups showing a drastic increase during leaf senescence <a href="#b4">4</a>. Our analyses of mutant and transgenic ANAC046 plants indicate that ANAC046 is a positive regulator of leaf senescence. </p> <p>ANAC046 belongs to the same subgroup as ORE1 and ORS1 ( <a href="#S1">Fig. S2</a>), TFs associated with leaf senescence <a href="#b20">20</a> <a href="#b22">22</a> <a href="#b23">23</a> <a href="#b24">24</a>. Recently, Qiu <i>et al</i>. <a href="#b26">26</a> showed that ORE1 directly promote the expression of a similar set of Chl catabolic genes. Our microarray analysis of <i>ANAC046</i> mutant plants showed that many genes inducible by ORE1, ORS1, or both were also upregulated in <i>ANAC046</i> OX plants and downregulated in <i>ANAC046</i> SRDX and KO plants ( <a href="#S1">Fig. S6</a> and <a href="#S1">Table S1</a>) suggesting that these TFs have overlapping functions during age-dependent senescence. However, there may be no or only limited functional redundancy between ORE1 and ANAC046 in terms of age-dependent senescence because distinct delayed senescence phenotypes were observed in an single mutant of <i>ORE1</i> and that of <i>ANAC046</i>. Recent reports have shown that <i>ORE1</i> is a miR164 target gene <a href="#b22">22</a> and its expression is enhanced by EIN3 <a href="#b26">26</a>. In contrast, <i>ANAC046</i> has neither miR164 target site nor EIN3 binding site, suggesting that expression of these NAC TFs controlled by different regulatory mechanism. Public database (eFP browser; <a href="http://bbc.botany.utoronto.ca/efp/cgi-bin/efpWeb.cgi">http://bbc.botany.utoronto.ca/efp/cgi-bin/efpWeb.cgi</a>) showed that these NAC TFs have different expression profiles under various senescence-promoting conditions: e.g., the expression of <i>ORS1</i> and <i>ANAC046</i> is upregulated in the dark, whereas that of <i>ORE1</i> is only slightly affected. Conversely, <i>ORE1</i> expression is upregulated by abscisic acid (ABA) treatment, whereas the expression of <i>ANAC046</i> only slightly upregulated and that of <i>ORS1</i> is unchanged. Therefore, we conclude that ANAC046, ORE1, and ORS1 share common functions but also have distinct roles and expression patterns in leaf senescence. Very recently, Sakuraba <i>et al</i>. <a href="#b32">32</a> reported that ANAC016 directly binds to the <i>SGR1</i> promoter and activates its expression during leaf senescence, whereas it does not bind to the promoters of other Chl catabolic genes, including <i>NYC1</i>, <i>SGR2</i>, and <i>PAO</i>. The result also indicates that several NAC TFs are involved in the regulation of Chl catabolic gene expression, although each has distinct regulatory roles. </p> <p>As recent studies revealed, some NAC TFs form homo/hetero-dimer and influence the activity each other <a href="#b33">33</a>. In addition to the direct protein-protein interaction, some NAC TFs are known to activate expression of other NAC TFs <a href="#b34">34</a> <a href="#b35">35</a>. In our microarray data, some senescence-associated NAC TFs such as <i>ATAF1</i> <a href="#b36">36</a>, <i>ANAC055</i>, <i>ANAC019</i>, <i>RD26/ANAC072</i> <a href="#b37">37</a>, <i>ANAC087</i> <a href="#b38">38</a>, and <i>ANAC048</i> were upregulated in <i>ANAC046</i> OX plants and downregulated in <i>ANAC046</i> SRDX and/or KO plants while expression of <i>NAP</i> <a href="#b25">25</a>, <i>ORE1</i> <a href="#b22">22</a>, <i>JUB1/ANAC042</i> <a href="#b39">39</a> and some other senescence-associated NAC TFs were not affected in these transgenic plants ( <a href="#S1">Fig. S7</a>). At least the former NAC TFs could be downstream regulators of ANAC046 or co-regulated TFs with ANAC046. On the other hand, the latter NAC TFs could be upstream of ANAC046 or independent from ANAC046. The complicated NAC-NAC regulatory circuit is unique aspect of plants and remains to be addressed in future studies. </p> <p>In Arabidopsis, single mutants of <i>NYC1</i> <a href="#b6">6</a>, <i>SGR1</i> <a href="#b15">15</a> <a href="#b40">40</a>, <i>PPH</i> <a href="#b9">9</a>, and <i>PaO</i> <a href="#b10">10</a> <a href="#b11">11</a> <a href="#b40">40</a> exhibit stay-green phenotypes with decreased levels of photosynthetic activity (termed "nonfunctional stay-green"). In contrast, KO and SRDX plants retained both Chl and photosynthetic proteins longer than the WT plants did during both age-dependent and dark-induced senescence. As judged by the CO <sub>2</sub> assimilation rate, the decline in photosynthetic activity in the old leaves was much slower in KO and SRDX plants than in WT plants. We therefore consider the KO and SRDX plants to have functional stay-green phenotypes. Because a decline in photosynthetic activity is a typical symptom of leaf senescence, the results demonstrate that, besides regulating Chl degradation, ANAC046 plays important roles in other physiological events during leaf senescence, including regulation of photosynthetic activity. </p> <p>Overexpression of <i>ANAC046</i> severely impaired plant growth and enhanced leaf yellowing. These phenotypes were not observed in plants overexpressing <i>ORE1</i> and <i>ORS1</i>, again showing the functional difference between ORE1 and ANAC046 <a href="#b20">20</a> <a href="#b23">23</a> <a href="#b24">24</a>. Furthermore, mesophyll chloroplasts of the OX plants at 40 DAS contained large starch granules at the end of the dark period, whereas the WT plants did not ( <a href="#f4">Fig. 4</a>). This suggests that the OX plants cannot degrade as much starch during the dark period as WT plants can, which may reduce carbon supply and therefore plant size becomes small. It is noteworthy that the phenotypes of loss-of-function mutants defective in starch metabolism such as <i>maltose excess 1</i> ( <i>mex1</i>: maltose transporter) and <i>disproportionating enzyme 1</i> ( <i>dpe1</i>: debranching enzyme) <a href="#b41">41</a> <a href="#b42">42</a> are similar to that of <i>ANAC046</i> OX plants. However, there was no significant difference in the expression levels of <i>MEX1</i> and <i>DPE1</i> between the OX and WT plants in our microarray analysis ( <a href="#S1">Fig. S8</a>). Accumulation of starch generally occurs in old leaves, although the molecular mechanism of this selectivity remains unclear <a href="#b43">43</a> <a href="#b44">44</a>. It is therefore reasonable to speculate that starch accumulation is a senescence syndrome that occurs earlier in the OX plants than in the WT plants. </p> <p>Plant hormones positively or negatively regulate leaf senescence by integrating developmental and environmental cues <a href="#b45">45</a>. It is widely acknowledged that ABA, jasmonic acid (JA), salicylic acid (SA), and ethylene are positive regulators of leaf senescence, whereas cytokinins, gibberellins, and auxins are negative regulators. The signaling pathways of these plant hormones communicate with each other and cooperatively or antagonistically regulate senescence. To elucidate the role of each plant hormone in senescence, much effort has been made to analyze mutants defective in the signaling pathway of each hormone. Buchanan-Wollaston <i>et al</i>. <a href="#b46">46</a> have identified a set of senescence-related genes downregulated in the SA signaling–defective mutant <i>NahG</i>; many of them are upregulated during age-dependent senescence but are unaffected by dark treatment. We examined the microarray data of <i>ANAC046</i> mutants to see whether the expression of SA-dependent and developmentally regulated genes is affected by ANAC046). However, expression of these genes were only slightly affected in the OX and KO plants ( <a href="#S1">Fig. S9</a>). In contrast, nearly all senescence-related genes downregulated in both the JA signaling–defective mutant <i>coi1</i> and ethylene signaling–defective mutant <i>ein2</i> <a href="#b46">46</a> were upregulated in the OX plants and downregulated in the KO and SRDX plants ( <a href="#S1">Fig. S10</a>). Most genes dependent on the JA pathway, ethylene pathway, or both are also upregulated during dark-induced senescence <a href="#b46">46</a>. These results suggest that ANAC046 plays a role in controlling developmental and dark-induced senescence downstream of JA and ethylene signaling pathways. </p> <p>Recently, several TFs involved in hormonal signaling pathway have been identified as positive regulators of Chl catabolic gene expression. ABA INSENSITIVE5 (ABI5) and ENHANCED EM LEVEL (EEL) directly bind to the promoters of <i>SGR1</i>, <i>SGR2</i>, and <i>NYC1</i> in chromatin immunoprecipitation assays <a href="#b47">47</a>. In Arabidopsis embryos, SGRs play a key role in degreening, which is regulated by ABA through ABI3. Direct binding of ABI3 to the promoters of <i>SGR1</i> and <i>SGR2</i> was demonstrated by gel-shift assay <a href="#b48">48</a>. MYC2/3/4, a major JA signaling component, directly promote the expression of <i>SGR1</i>, <i>NYC1</i>, and <i>PaO</i> <a href="#b49">49</a>. However, these TFs involved in ABA- and JA-signaling did not bind to the promoters of either <i>SGR1</i> or <i>NYC1</i> in our Y1H screening. We found that several TFs (in addition to ANAC046, ANAC087, and ANAC100) bind to the promoters of <i>SGR1</i>, <i>SGR2</i>, <i>NYC1</i>, and <i>PaO</i>, although the precise functions of these TFs remain to be elucidated. It is possible that the Chl catabolic pathway is finely tuned by multiple TFs involved in various aspects of plant growth and development. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Our comprehensive Y1H screening for TFs that directly target Chl catabolic genes identified a previously uncharacterized TF, ANAC046. We showed that ANAC046 stimulates the expression not only of Chl catabolic genes but also of senescence-related genes. We also identified several TFs that directly bind the promoters of Chl catabolic genes. We assume that Chl degradation and leaf senescence are controlled by multiple TFs, which may function redundantly. Further study is needed to unravel the integration of these TFs into the molecular network that controls Chl degradation and leaf senescence. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialsandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and growth conditions</div> <p> <i>Arabidopsis thaliana</i> ecotype Columbia-0 was used throughout this study. The Arabidopsis transposon-tagged line SM_3_27647 from Nottingham Arabidopsis Stock Centre (NASC) was used as the <i>anac046</i> knockout mutant. Seedlings were grown on MS-agar plates in a climate chamber under fluorescent light (120 μmol m <sup>−2</sup> s <sup>−1</sup>; 16-h light/8-h dark) at 20 °C, transferred into soil at 15 DAS, and grown in controlled-growth cabinets under the same conditions except for the temperature, which was 22/16 °C day/night). For dark treatment, at 35 DAS plants were placed in a growth cabinet at 22 °C without light. </p> </div> <div class="plasmidconstructionandtransformationintoplants" title="sec"> <div class="title" tagx="title" title="title">
Plasmid construction and transformation into plants</div> <p>For the Y1H experiment, the promoter region of each gene was cloned into the pDONR_P4P1R vector by Gateway BP reaction to produce entry clone (Life Technologies). Primers used for amplification of the promoter regions are listed in <a href="#S1">Table S2</a>. The promoters were subcloned into the R4L1pDEST_HISi vectors by Gateway LR reaction (Life Technologies). Only <i>MES16</i> promoter was cloned into R4L1pDEST_HISi2 in which extra start codons between TATA-box and start codon of HIS3 are mutated. Yeast strains harboring bait constructs in their genomes were prepared from strain YM4271 by sequential insertion of promoter: <i>HIS3</i> into genomic <i>HIS3</i> loci. </p> <p>To produce reporter constructs for transient effector–reporter analysis, promoter regions in the above-described entry clones were transferred into the R4L1pDEST_LUC_HSP vector <a href="#b50">50</a> by Gateway LR reaction. To produce the effector constructs, coding region of each <i>ANAC046</i> and <i>VAMP722</i> (which localizes to vacuolar membrane; negative control) was cloned in to the pDONR207 vector by Gateway BP reaction and then subcloned into the pDEST35SVP16HSPG vector, which is modified from p35SSRDXHSPG <a href="#b51">51</a>, by Gateway LR reaction. </p> <p>To construct <i>35S:ANAC046-SRDX</i>, the protein-coding region of <i>ANAC046</i> without the stop codon was amplified by PCR and cloned into the <i>Sma</i>I site of p35SSRDXG <a href="#b52">52</a>. To construct <i>35S:ANAC046</i>, the coding region with the stop codon was amplified and cloned into the <i>Sma</i>I– <i>Sal</i>I site of p35SHSPG <a href="#b51">51</a>. The transgene cassette was transferred into the T-DNA destination vector pBCKH by Gateway LR reaction. The <i>35S:ANAC046-SRDX</i> and <i>35S:ANAC046</i> constructs were transformed into Arabidopsis plants by the floral dip method <a href="#b53">53</a>. </p> </div> <div class="y1hscreening" title="sec"> <div class="title" tagx="title" title="title">
Y1H screening</div> <p>Yeast strains harboring bait constructs in their genomes were tested for the background <i>HIS3</i> gene expression by spotting liquid-grown cultures onto media with various concentrations of 3-amino-1,2,4-triazole (3-AT). After determination of the range of appropriate 3-AT concentration to suppress prey-independent growth, liquid-grown yeast strains were transformed individually with 343 mini-pools (which cover 1304 TFs) in four 96-well plates as described previously <a href="#b27">27</a>. The transformed yeasts were spotted onto four sets of four square plates (three different 3-AT concentrations plus positive control) and cultured for several days. After identification of a positive mini-pool in the first screening, the positive TF in this mini-pool was identified in the second screening. </p> </div> <div class="transienteffector–reporteranalysisinplants" title="sec"> <div class="title" tagx="title" title="title">
Transient effector–reporter analysis in plants</div> <p>Leaf mesophyll cells were isolated from peeled rosette leaves of 3- to 4-week-old Arabidopsis plants using the Tape Arabidopsis Sandwich method <a href="#b54">54</a>. Protoplasts were prepared and transfected using the polyethylene glycol method described previously <a href="#b55">55</a>. Reporter constructs containing the firefly luciferase gene under the control of the promoters of chlorophyll catabolic genes were co-transfected with the effector construct containing <i>ANAC046</i> or <i>VAMP722</i> fused with the VP16 transcriptional activation domain driven by the CaMV 35S promoter. The modified Renilla luciferase gene driven by the CaMV 35S promoter (phRLHSP) was also co-transfected as an internal control. The transfected protoplasts were incubated at 22 °C for 16–18 h in the dark. The dual-luciferase assay was carried out using the Pikka Gene Dual Assay Kit (Toyo Ink, Inc.). Reporter activity was normalized to the activity of Renilla luciferase and expressed as relative luciferase activity. </p> </div> <div class="chlorophyllextractionandhplcanalysis" title="sec"> <div class="title" tagx="title" title="title">
Chlorophyll extraction and HPLC analysis</div> <p>Tissues were ground into powder in liquid nitrogen and extracted with acetone. The samples (80 μl) were mixed with 20 μl of water. Pigments were analyzed by high-performance liquid chromatography (HPLC; X-LC, Jasco) using a reversed-phase column (Symmetry C8, 150 × 4.6 mm; Waters) according to Zapata <i>et al</i>. <a href="#b56">56</a>. </p> </div> <div class="westernblotting" title="sec"> <div class="title" tagx="title" title="title">
Western blotting</div> <p>Proteins were extracted from rosette leaves according to Sakuraba <i>et al</i>. <a href="#b57">57</a>. Protein extracts equivalent to 1.0 mg of leaf tissue per lane were subjected to electrophoresis on a 10% polyacrylamide gel (PAGEL; ATTO) and blotted onto a nitrocellulose membrane (GE Healthcare). Primary antibodies against PsaA, PsbA, Lhca1, Lhcb1, and RbcL were purchased from Agrisera. Anti-rabbit IgG linked to horseradish peroxidase (GE Healthcare) was used as a secondary antibody. Horseradish peroxidase activity was detected using Immobilon Western Chemiluminescent HRP Substrate (Millipore) according to the manufacturer’s protocol. </p> </div> <div class="transmissionelectronmicroscopy(tem)" title="sec"> <div class="title" tagx="title" title="title">
Transmission electron microscopy (TEM)</div> <p>Leaves were cut into small pieces and rapidly immersed in 0.05 M cacodylate buffer (pH 7.4) containing 2% paraformaldehyde and 2% glutaraldehyde at 4 °C overnight. After fixation, samples were rinsed 3 times with 0.05 M cacodylate buffer, followed by post-fixation with 2% osmium tetroxide in 0.05 M cacodylate buffer at 4 °C for 4 h. Samples were dehydrated in a graded ethanol series, infiltrated with propylene oxide, and finally embedded in Quetol-651 (Nisshin EM Co.). Ultrathin sections were cut with a diamond knife using an ultramicrotome (Ultracut UCT; Leica Microsystems), picked up on copper grids, and stained with uranyl acetate and lead citrate. Sections were observed under a transmission electron microscope (JEM-1200EX; JEOL Ltd) at an acceleration voltage of 80 kV. Digital images were taken with a CCD camera (Veleta; Olympus Soft Imaging Solutions GmbH). </p> </div> <div class="ionleakagemeasurements" title="sec"> <div class="title" tagx="title" title="title">
Ion leakage measurements</div> <p>Rosette leaves were placed in a tube, immersed in 10 ml of 0.4 M mannitol solution, and shaken at room temperature for 3 h. Initial conductivity of the solution was measured using an electroconductivity meter (TOA DKK). Subsequently, the samples were boiled for 10 min, cooled down to room temperature, and measured for total conductivity. Conductivity was expressed as the percentage of the initial conductivity versus the total conductivity. </p> </div> <div class="co2assimilationrateofleaves" title="sec"> <div class="title" tagx="title" title="title">
CO <sub>2</sub> assimilation rate of leaves </div> <p>CO <sub>2</sub> assimilation rates were determined at 20 °C using a modified portable-gas-exchange system (LI6400; Li-Cor). Whole plants were exposed to an LED light source (100 μmol photons m <sup>−2</sup> s <sup>−1</sup>) in the chamber. </p> </div> <div class="microarrayanalysis" title="sec"> <div class="title" tagx="title" title="title">
Microarray analysis</div> <p>Microarray experiments were performed using Agilent Arabidopsis (V4; 4 × 44k) microarrays according to the manufacturer’s instructions. Total RNA was extracted from leaves of WT and OX plants at 50 DAS and from leaves of WT, KO, and SRDX plants at 5 DAD using TRIzol reagent (Life Technologies) and an RNeasy Mini Kit (Qiagen). RNA integrity was evaluated using an Agilent 2100 Bioanalyzer (Agilent Technologies). Total RNA (200 ng) was used as starting material. Three biological replicates were tested by the one-color method. RNA from three independent lines of the OX and SRDX plants was used for analysis. Spot signal values were calculated by the Feature Extraction software (version 9.1; Agilent). We defined the QC value as 1 when a spot passed the "FeatNonUnifOL" filter and as 2 when the spot further passed the "FeatPopnOL" filter. We defined the detection value as 1 when a spot passed the "IsPosAndSignif" filter and as 2 when the spot further passed the "IsWellAboveBG" filter. All signal values were divided by the median value among spots with QC = 2 followed by quantile normalization <a href="#b58">58</a> using all previously obtained microarray data in our group to make each signal distribution the same. Spot-to-gene conversion was accomplished based on a table provided by The Arabidopsis Information Resource ( <a href="ftp://ftp.Arabidopsis.org/home/tair/Microarrays/Agilent/agilent_array_elements-2010-12-20.txt">ftp://ftp.Arabidopsis.org/home/tair/Microarrays/Agilent/agilent_array_elements-2010-12-20.txt</a>). </p> <p>The upregulated genes in OX, KO, and SRDX plants were defined as top 200 upregulated genes with the false discovery rate &amp;lt;0.15; the latter was calculated using the QVALUE software with default settings <a href="#b59">59</a>. </p> </div> <div class="quantitativereal-timepcranalysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR analysis</div> <p>The transcript levels of <i>ANAC046</i> was analyzed by quantitative real-time PCR (RT-qPCR) with the SYBR Premix Ex Taq II polymerase (TaKaRa) and Thermal Cycler Dice Real Time System TP800 (TaKaRa), according to the manufacturers’ instructions. The expression level of <i>actin</i> was used to normalize the transcript levels of each sample. Primers used for RT-qPCR analysis were shown in <a href="#S1">Table S4</a>. </p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional Information</div> <p> <b>Accession codes:</b> Sequence data from this article can be found in The Arabidopsis Information Resource under the following accession numbers: <i>CLH1</i> (AT1G19670), <i>CLH2</i> (AT5G43860), <i>CYP89A9</i> (AT3G03470), <i>HCAR</i> (AT1G04620), <i>NOL</i> (AT5G04900), <i>NYC1</i> (AT4G13250), <i>PAO</i> (AT3G44880), <i>PPH</i> (AT5G13800), <i>RCCR</i> (AT4G37000), <i>SGR1/NYE1</i> (AT4G22920), <i>SGR2</i> (AT4G11910), <i>ANAC046</i> (AT3G04060), <i>ANAC087</i> (AT5G18270), <i>ORE1</i> (AT5G39610), <i>ORS1</i> (AT3G29035), <i>ANAC079</i> (AT5G07680), and <i>ANAC100</i> (AT5G61430). Microarray data were deposited in the National Center for Biotechnology Information Gene Expression Omnibus (http://www.ncbi.nlm.nih.gov/geo/) under accession number GSE71767. </p> <p> <b>How to cite this article</b>: Oda-Yamamizo, C. <i>et al</i>. The NAC transcription factor ANAC046 is a positive regulator of chlorophyll degradation and senescence in Arabidopsis leaves. <i>Sci. Rep</i>. <b>6</b>, 23609; doi: 10.1038/srep23609 (2016). </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div>  <div class="media" title="media"> <a href="srep23609-s1.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors appreciate the skillful technical assistance of Ms. Fumie Tobe (AIST) and the technical advice of Dr. Ayuko Ushio (NARO). This work was supported in part by a Grant-in-Aid from the National Agriculture and Food Research Organization (NARO), JSPS KAKENHI Grant Number 25292025, and JSPS Research Fellowships for Young Scientists to C.O.-Y. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="b1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ougham</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canter</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donnison</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="mixed-article-title" title="mixed-article-title">What stay-green mutants tell us about nitrogen remobilization in leaf senescence</span>. <span class="source" tagx="source" title="source">J Exp Bot.</span> <span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">801</span>– <span class="lpage" tagx="lpage" title="lpage">808</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11912223">11912223</a> </span> </span> </li> <li tag="ref"> <a name="b2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matile</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörtensteiner</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chlorophyll degradation</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Physiol. Plant Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">67</span>– <span class="lpage" tagx="lpage" title="lpage">95</span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012204">15012204</a> </span> </span> </li> <li tag="ref"> <a name="b3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörtensteiner</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chlorophyll degradation during senescence</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">77</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16669755">16669755</a> </span> </span> </li> <li tag="ref"> <a name="b4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörtensteiner</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Update on biochemistry of chlorophyll breakdown</span>. <span class="source" tagx="source" title="source">Plant Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">82</span>, <span class="fpage" tagx="fpage" title="fpage">505</span>– <span class="lpage" tagx="lpage" title="lpage">517</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22790503">22790503</a> </span> </span> </li> <li tag="ref"> <a name="b5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusaba</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Rice non-yellow coloring1 is involved in light-harvesting complex II and grana degradation during leaf senescence </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">1362</span>– <span class="lpage" tagx="lpage" title="lpage">1375</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17416733">17416733</a> </span> </span> </li> <li tag="ref"> <a name="b6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horie</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Participation of chlorophyll <i>b</i> reductase in the initial step of the degradation of light-harvesting chlorophyll <i>a/b</i>-protein complexes in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="volume" tagx="volume" title="volume">284</span>, <span class="fpage" tagx="fpage" title="fpage">17449</span>– <span class="lpage" tagx="lpage" title="lpage">17456</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19403948">19403948</a> </span> </span> </li> <li tag="ref"> <a name="b7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meguro</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of the 7-hydroxymethyl chlorophyll <i>a</i> reductase of the chlorophyll cycle in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">3442</span>– <span class="lpage" tagx="lpage" title="lpage">3453</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21934147">21934147</a> </span> </span> </li> <li tag="ref"> <a name="b8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morita</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Defect in non-yellow coloring 3, an alpha/beta hydrolase-fold family protein, causes a stay-green phenotype during leaf senescence in rice </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">940</span>– <span class="lpage" tagx="lpage" title="lpage">952</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19453447">19453447</a> </span> </span> </li> <li tag="ref"> <a name="b9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schelbert</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Pheophytin pheophorbide hydrolase (pheophytinase) is involved in chlorophyll breakdown during leaf senescence in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">767</span>– <span class="lpage" tagx="lpage" title="lpage">785</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19304936">19304936</a> </span> </span> </li> <li tag="ref"> <a name="b10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruzinska</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Chlorophyll breakdown: pheophorbide <i>a</i> oxygenase is a Rieske-type iron-sulfur protein, encoded by the <i>accelerated cell death 1</i> gene </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">15259</span>– <span class="lpage" tagx="lpage" title="lpage">15264</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14657372">14657372</a> </span> </span> </li> <li tag="ref"> <a name="b11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirashima</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satoh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis-accelerated cell death</i> gene <i>ACD1</i> is involved in oxygenation of pheophorbide <i>a</i>: inhibition of the pheophorbide <i>a</i> oxygenase activity does not lead to the "stay-green" phenotype in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">1266</span>– <span class="lpage" tagx="lpage" title="lpage">1274</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14701922">14701922</a> </span> </span> </li> <li tag="ref"> <a name="b12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wüthrich</span> <span class="given-names" tagx="given-names" title="given-names">K. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Molecular cloning, functional expression and characterization of RCC reductase involved in chlorophyll catabolism </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">189</span>– <span class="lpage" tagx="lpage" title="lpage">198</span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10743659">10743659</a> </span> </span> </li> <li tag="ref"> <a name="b13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christ</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">MES16, a member of the methylesterase protein family, specifically demethylates fluorescent chlorophyll catabolites during chlorophyll breakdown in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">158</span>, <span class="fpage" tagx="fpage" title="fpage">628</span>– <span class="lpage" tagx="lpage" title="lpage">641</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22147518">22147518</a> </span> </span> </li> <li tag="ref"> <a name="b14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christ</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Cytochrome P450 CYP89A9 is involved in the formation of major chlorophyll catabolites during leaf senescence in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1868</span>– <span class="lpage" tagx="lpage" title="lpage">1880</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23723324">23723324</a> </span> </span> </li> <li tag="ref"> <a name="b15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of a novel chloroplast protein AtNYE1 regulating chlorophyll degradation during leaf senescence in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">144</span>, <span class="fpage" tagx="fpage" title="fpage">1429</span>– <span class="lpage" tagx="lpage" title="lpage">1441</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17468209">17468209</a> </span> </span> </li> <li tag="ref"> <a name="b16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> stay-green2 is a negative regulator of chlorophyll degradation during leaf senescence </span>. <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1288</span>– <span class="lpage" tagx="lpage" title="lpage">1302</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24719469">24719469</a> </span> </span> </li> <li tag="ref"> <a name="b17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">S. Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paek</span> <span class="given-names" tagx="given-names" title="given-names">N. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The divergent roles of STAYGREEN (SGR) homologs in chlorophyll degradation</span>. <span class="source" tagx="source" title="source">Mol. Cells</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">390</span>– <span class="lpage" tagx="lpage" title="lpage">395</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25913011">25913011</a> </span> </span> </li> <li tag="ref"> <a name="b18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome of <i>Arabidopsis</i> leaf senescence </span>. <span class="source" tagx="source" title="source">Plant Cell Env.</span> <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">521</span>– <span class="lpage" tagx="lpage" title="lpage">549</span> ( <span class="year" tagx="year" title="year">2004</span>). </span> </li> <li tag="ref"> <a name="b19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balazadeh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riaño-Pachón</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mueller-Roeber</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcription factors regulating leaf senescence in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant Biol.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">63</span>– <span class="lpage" tagx="lpage" title="lpage">75</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18721312">18721312</a> </span> </span> </li> <li tag="ref"> <a name="b20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balazadeh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Gene regulatory network controlled by NAC transcription factor ANAC092/AtNAC2/ORE1 during salt-promoted senescence </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">62</span>, <span class="fpage" tagx="fpage" title="fpage">250</span>– <span class="lpage" tagx="lpage" title="lpage">264</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20113437">20113437</a> </span> </span> </li> <li tag="ref"> <a name="b21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">AtNAP, a NAC family transcription factor, has an important role in leaf senescence</span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">46</span>, <span class="fpage" tagx="fpage" title="fpage">601</span>– <span class="lpage" tagx="lpage" title="lpage">612</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16640597">16640597</a> </span> </span> </li> <li tag="ref"> <a name="b22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Trifurcate feed-forward regulation of age-dependent cell death involving <i>miR164</i> </span> in <span class="source" tagx="source" title="source">Arabidopsis. Science</span> <span class="volume" tagx="volume" title="volume">323</span>, <span class="fpage" tagx="fpage" title="fpage">1053</span>– <span class="lpage" tagx="lpage" title="lpage">1057</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19229035">19229035</a> </span> </span> </li> <li tag="ref"> <a name="b23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rauf</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">ORE1 balances leaf senescence against maintenance by antagonizing G2-like-mediated transcription </span>. <span class="source" tagx="source" title="source">EMBO Rep.</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">382</span>– <span class="lpage" tagx="lpage" title="lpage">388</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23459204">23459204</a> </span> </span> </li> <li tag="ref"> <a name="b24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balazadeh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">ORS1, an H <sub>2</sub>O <sub>2</sub>-responsive NAC transcription factor, controls senescence in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">346</span>– <span class="lpage" tagx="lpage" title="lpage">360</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21303842">21303842</a> </span> </span> </li> <li tag="ref"> <a name="b25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Y. S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Mutation of the <i>Arabidopsis</i> NAC016 transcription factor delays leaf senescence </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">1660</span>– <span class="lpage" tagx="lpage" title="lpage">1672</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23926065">23926065</a> </span> </span> </li> <li tag="ref"> <a name="b26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">EIN3 and ORE1 Accelerate degreening during ethylene-mediated leaf senescence by directly activating chlorophyll catabolic genes in Arabidopsis </span> <span class="source" tagx="source" title="source">PLOS Genet.</span> doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pgen.1005399">10.1371/journal.pgen.1005399</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitsuda</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Efficient yeast one-/two-hybrid screening using a library composed only of transcription factors in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">2145</span>– <span class="lpage" tagx="lpage" title="lpage">2151</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20980269">20980269</a> </span> </span> </li> <li tag="ref"> <a name="b28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ooka</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Comprehensive analysis of NAC family genes in <i>Oryza sativa</i> and <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">DNA Res.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>– <span class="lpage" tagx="lpage" title="lpage">247</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15029955">15029955</a> </span> </span> </li> <li tag="ref"> <a name="b29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiratsu</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsui</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koyama</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohme-Takagi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Dominant repression of target genes by chimeric repressors that include the EAR motif, a repression domain </span>, in <span class="source" tagx="source" title="source">Arabidopsis. Plant J.</span> <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">733</span>– <span class="lpage" tagx="lpage" title="lpage">739</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12787253">12787253</a> </span> </span> </li> <li tag="ref"> <a name="b30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Breeze</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">High-resolution temporal profiling of transcripts during <i>Arabidopsis</i> leaf senescence reveals a distinct chronology of processes and regulation </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">873</span>– <span class="lpage" tagx="lpage" title="lpage">894</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21447789">21447789</a> </span> </span> </li> <li tag="ref"> <a name="b31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hörtensteiner</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kräutler</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chlorophyll breakdown in higher plants</span>. <span class="source" tagx="source" title="source">Biochim. Biophys. Acta Bioenergetics</span> <span class="volume" tagx="volume" title="volume">1807</span>, <span class="fpage" tagx="fpage" title="fpage">977</span>– <span class="lpage" tagx="lpage" title="lpage">988</span> ( <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis NAC016 promotes chlorophyll breakdown by directly upregulating STAYGREEN1 transcription </span>. <span class="source" tagx="source" title="source">Plant Cell Rep.</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">155</span>– <span class="lpage" tagx="lpage" title="lpage">166</span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26441053">26441053</a> </span> </span> </li> <li tag="ref"> <a name="b33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">VND-INTERACTING2, a NAC domain transcription factor, negatively regulates xylem vessel formation in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1249</span>– <span class="lpage" tagx="lpage" title="lpage">1263</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20388856">20388856</a> </span> </span> </li> <li tag="ref"> <a name="b34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endo</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Multiple classes of transcription factors regulate the expression of VASCULAR-RELATED NAC-DOMAIN7, a master switch of xylem vessel differentiation </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">242</span>– <span class="lpage" tagx="lpage" title="lpage">254</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25265867">25265867</a> </span> </span> </li> <li tag="ref"> <a name="b35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Multiple classes of transcription factors regulate the expression of VASCULAR-RELATED NAC-DOMAIN7, a master switch of xylem vessel differentiation </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">68</span>, <span class="fpage" tagx="fpage" title="fpage">1104</span>– <span class="lpage" tagx="lpage" title="lpage">1114</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21883551">21883551</a> </span> </span> </li> <li tag="ref"> <a name="b36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garapati</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcription factor Arabidopsis Activating Factor1 integrates carbon starvation responses with trehalose metabolism </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">169</span>, <span class="fpage" tagx="fpage" title="fpage">379</span>– <span class="lpage" tagx="lpage" title="lpage">390</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26149570">26149570</a> </span> </span> </li> <li tag="ref"> <a name="b37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takasaki</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">SNAC-As, stress-responsive NAC transcription factors, mediate ABA-inducible leaf senescence</span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">84</span>, <span class="fpage" tagx="fpage" title="fpage">1114</span>– <span class="lpage" tagx="lpage" title="lpage">1123</span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26518251">26518251</a> </span> </span> </li> <li tag="ref"> <a name="b38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Gene regulatory cascade of senescence-associated NAC transcription factors activated by ETHYLENE-INSENSITIVE2-mediated leaf senescence signaling in Arabidopsis </span>. <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">4023</span>– <span class="lpage" tagx="lpage" title="lpage">4036</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24659488">24659488</a> </span> </span> </li> <li tag="ref"> <a name="b39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">JUNGBRUNNEN1, a reactive oxygen species-responsive NAC transcription factor, regulates longevity in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">482</span>– <span class="lpage" tagx="lpage" title="lpage">506</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22345491">22345491</a> </span> </span> </li> <li tag="ref"> <a name="b40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Stay-green and chlorophyll catabolic enzymes interact at light-harvesting complex II for chlorophyll detoxification during leaf senescence in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">507</span>– <span class="lpage" tagx="lpage" title="lpage">518</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22366162">22366162</a> </span> </span> </li> <li tag="ref"> <a name="b41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Critchley</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">A critical role for disproportionating enzyme in starch breakdown is revealed by a knock-out mutation in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">89</span>– <span class="lpage" tagx="lpage" title="lpage">100</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11359613">11359613</a> </span> </span> </li> <li tag="ref"> <a name="b42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niittylä</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">A previously unknown maltose transporter essential for starch degradation in leaves</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">303</span>, <span class="fpage" tagx="fpage" title="fpage">87</span>– <span class="lpage" tagx="lpage" title="lpage">89</span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14704427">14704427</a> </span> </span> </li> <li tag="ref"> <a name="b43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diaz</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of markers to determine the extent and variability of leaf senescence in Arabidopsis. A metabolic profiling approach </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">138</span>, <span class="fpage" tagx="fpage" title="fpage">898</span>– <span class="lpage" tagx="lpage" title="lpage">908</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15923326">15923326</a> </span> </span> </li> <li tag="ref"> <a name="b44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoyama</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Ubiquitin ligase ATL31 functions in leaf senescence in response to the balance between atmospheric CO <sub>2</sub> and nitrogen availability in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">293</span>– <span class="lpage" tagx="lpage" title="lpage">305</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24399238">24399238</a> </span> </span> </li> <li tag="ref"> <a name="b45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jibran</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunter</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dijkwel</span> <span class="given-names" tagx="given-names" title="given-names">P. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Hormonal regulation of leaf senescence through integration of developmental and stress signals </span>. <span class="source" tagx="source" title="source">Plant Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">82</span>, <span class="fpage" tagx="fpage" title="fpage">547</span>– <span class="lpage" tagx="lpage" title="lpage">561</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23504405">23504405</a> </span> </span> </li> <li tag="ref"> <a name="b46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchanan-Wollaston</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Comparative transcriptome analysis reveals significant differences in gene expression and signalling pathways between developmental and dark/starvation-induced senescence in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">567</span>– <span class="lpage" tagx="lpage" title="lpage">585</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15860015">15860015</a> </span> </span> </li> <li tag="ref"> <a name="b47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Phytochrome-interacting transcription factors PIF4 and PIF5 induce leaf senescence in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">4636</span> doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms5636">10.1038/ncomms5636</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25119965">25119965</a> </span> </span> </li> <li tag="ref"> <a name="b48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delmas</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">ABI3 controls embryo degreening through Mendel’s <i>I</i> locus </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="volume" tagx="volume" title="volume">110</span>, <span class="fpage" tagx="fpage" title="fpage">E3888</span>– <span class="lpage" tagx="lpage" title="lpage">E3894</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24043799">24043799</a> </span> </span> </li> <li tag="ref"> <a name="b49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Jasmonic acid promotes degreening via MYC2/3/4- and ANAC019/055/072-mediated regulation of major chlorophyll catabolic genes </span>. <span class="source" tagx="source" title="source">Plant J.</span> doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.13030">10.1111/tpj.13030</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oshima</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">MIXTA-Like transcription factors and WAX INDUCER1/SHINE1 coordinately regulate cuticle development in <i>Arabidopsis</i> and <i>Torenia fournieri</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1609</span>– <span class="lpage" tagx="lpage" title="lpage">1624</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23709630">23709630</a> </span> </span> </li> <li tag="ref"> <a name="b51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oshima</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Novel vector systems to accelerate functional analysis of transcription factors using chimeric repressor gene-silencing technology (CRES-T) </span>. <span class="source" tagx="source" title="source">Plant Biotechnol.</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">201</span>– <span class="lpage" tagx="lpage" title="lpage">210</span> ( <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitsuda</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Efficient production of male and female sterile plants by expression of a chimeric repressor in <i>Arabidopsis</i> and rice </span>. <span class="source" tagx="source" title="source">Plant Biotechnol. J.</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">325</span>– <span class="lpage" tagx="lpage" title="lpage">332</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17147638">17147638</a> </span> </span> </li> <li tag="ref"> <a name="b53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clough</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bent</span> <span class="given-names" tagx="given-names" title="given-names">A. F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Floral dip: A simplified method for <i>Agrobacterium</i>-mediated transformation of <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">735</span>– <span class="lpage" tagx="lpage" title="lpage">743</span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10069079">10069079</a> </span> </span> </li> <li tag="ref"> <a name="b54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">F. H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Tape- <i>Arabidopsis</i> sandwich - a simpler <i>Arabidopsis</i> protoplast isolation method </span>. <span class="source" tagx="source" title="source">Plant Methods</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">10</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19133153">19133153</a> </span> </span> </li> <li tag="ref"> <a name="b55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Engineering the <i>Oryza sativa</i> cell wall with rice NAC transcription factors regulating secondary wall formation </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="fpage" tagx="fpage" title="fpage">12</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23346092">23346092</a> </span> </span> </li> <li tag="ref"> <a name="b56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zapata</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodríguez</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrido</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Separation of chlorophylls and carotenoids from marine phytoplankton: a new HPLC method using a reversed phase C8 column and pyridine-containing mobile phases </span>. <span class="source" tagx="source" title="source">Mar. Ecol. Prog. Ser.</span> <span class="volume" tagx="volume" title="volume">195</span>, <span class="fpage" tagx="fpage" title="fpage">29</span>– <span class="lpage" tagx="lpage" title="lpage">45</span> ( <span class="year" tagx="year" title="year">2000</span>). </span> </li> <li tag="ref"> <a name="b57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuraba</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Overproduction of Chl <i>b</i> retards senescence through transcriptional reprogramming in <i>Arabidopsis</i> </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">505</span>– <span class="lpage" tagx="lpage" title="lpage">517</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22285931">22285931</a> </span> </span> </li> <li tag="ref"> <a name="b58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bolstad</span> <span class="given-names" tagx="given-names" title="given-names">B. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irizarry</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Astrand</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Speed</span> <span class="given-names" tagx="given-names" title="given-names">T. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparison of normalization methods for high density oligonucleotide array data based on variance and bias </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">185</span>– <span class="lpage" tagx="lpage" title="lpage">193</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12538238">12538238</a> </span> </span> </li> <li tag="ref"> <a name="b59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Storey</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tibshirani</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical significance for genome-wide studies</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">9440</span>– <span class="lpage" tagx="lpage" title="lpage">9445</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12883005">12883005</a> </span> </span> </li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Author Contributions</b> C.O.-Y. performed most of the experiments and wrote the article; M.N. conducted Y1H screening, plant transformation, microarray analysis, and wrote the article; S.S. conducted transient effector–reporter analysis; D.O. conducted plasmid construction; M.O.-T. contributed to the experimental design; A.O. conceived the project and wrote the article. </p> </div> </div> </div>  </body></html>