<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4729950">4729950</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2016.00011">10.3389/fpls.2016.00011</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Microarray Analysis of Rice d1 (RGA1) Mutant Reveals the Potential Role of G-Protein Alpha Subunit in Regulating Multiple Abiotic Stresses Such as Drought, Salinity, Heat, and Cold </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jangam</span> <span class="given-names" tagx="given-names" title="given-names">Annie P.</span> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pathak</span> <span class="given-names" tagx="given-names" title="given-names">Ravi R.</span> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghuram</span> <span class="given-names" tagx="given-names" title="given-names">Nandula</span> </span> <a href="#fn001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/198356/overview" /> </span> </span> </div> <span class="citation_author_institution" id="">[], <span class="institution" title="institution">University School of Biotechnology, Guru Gobind Singh Indraprastha University</span> <span class="country" tagx="country" title="country">Dwarka, India</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Girdhar Kumar Pandey, University of Delhi, India</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Hao Peng, Washington State University, USA; Ashverya Laxmi, National Institute of Plant Genome Research, India </p> </div> <div class="corresp" title="corresp">
*Correspondence: Nandula Raghuram <span class="email" tagx="email" title="email">raghuram@ipu.ac.in</span>; <span class="email" tagx="email" title="email">raghuram98@hotmail.com</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Plant Physiology, a section of the journal Frontiers in Plant Science </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">11</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-8-20</span> <span class="accepted" title="accepted">accepted: 2016-1-07</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Jangam, Pathak and Raghuram</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The genome-wide role of heterotrimeric G-proteins in abiotic stress response in rice has not been examined from a functional genomics perspective, despite the availability of mutants and evidences involving individual genes/processes/stresses. Our rice whole transcriptome microarray analysis (GSE 20925 at NCBI GEO) using the G-alpha subunit (RGA1) null mutant (Daikoku 1 or d1) and its corresponding wild type ( <i>Oryza sativa</i> Japonica Nipponbare) identified 2270 unique differentially expressed genes (DEGs). Out of them, we mined for all the potentially abiotic stress-responsive genes using Gene Ontology terms, STIFDB2.0 and Rice DB. The first two approaches produced smaller subsets of the 1886 genes found at Rice DB. The GO approach revealed similar regulation of several families of stress-responsive genes in RGA1 mutant. The Genevestigator analysis of the stress-responsive subset of the RGA1-regulated genes from STIFDB revealed cold and drought-responsive clusters. Meta data analysis at Rice DB revealed large stress-response categories such as cold (878 up/810 down), drought (882 up/837 down), heat (913 up/777 down), and salt stress (889 up/841 down). One thousand four hundred ninety-eight of them are common to all the four abiotic stresses, followed by fewer genes common to smaller groups of stresses. The RGA1-regulated genes that uniquely respond to individual stresses include 111 in heat stress, eight each in cold only and drought only stresses, and two genes in salt stress only. The common DEGs (1498) belong to pathways such as the synthesis of polyamine, glycine-betaine, proline, and trehalose. Some of the common DEGs belong to abiotic stress signaling pathways such as calcium-dependent pathway, ABA independent and dependent pathway, and MAP kinase pathway in the RGA1 mutant. Gene ontology of the common stress responsive DEGs revealed 62 unique molecular functions such as transporters, enzyme regulators, transferases, hydrolases, carbon and protein metabolism, binding to nucleotides, carbohydrates, receptors and lipids, morphogenesis, flower development, and cell homeostasis. We also mined 63 miRNAs that bind to the stress responsive transcripts identified in this study, indicating their post-transcriptional regulation. Overall, these results indicate the potentially extensive role of RGA1 in the regulation of multiple abiotic stresses in rice for further validation. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
G-protein</div> <div class="kwd" title="kwd">
heat</div> <div class="kwd" title="kwd">
cold</div> <div class="kwd" title="kwd">
salt</div> <div class="kwd" title="kwd">
drought</div> <div class="kwd" title="kwd">
stress</div> <div class="kwd" title="kwd">
rice</div> <div class="kwd" title="kwd">
RGA1</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Council of Scientific and Industrial Research <span class="named-content" title="named-content">10.13039/501100001412</span> </div> <span class="award-id" tagx="award-id" title="award-id">[60(0056)/02/EMRII]</span> <span class="award-id" tagx="award-id" title="award-id">[09/806(013) 2008-EMRI]</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Abiotic stress responses in plants are being increasingly addressed on a genome-wide scale to find newer gene targets for protecting crop yields in the era of climate change (Pandey et al., <a href="#B33">2015</a>). Rice has been a crop of particular interest in this regard, not only because of its popularity as a post-genomic model crop, but also its importance as a staple food for half of the world's population. In rice, transcriptome-wide analyses of abiotic stress response have been reported in terms of either specific stresses, or specific families of genes that respond to multiple stresses, or both. They include drought-responsive (Wang et al., <a href="#B45">2011</a>) and salinity-responsive (Jiang et al., <a href="#B20">2013</a>) rice transcriptomes spanning multiple gene families, pathways, and transcription factors. Studies that examined multiple stresses in parallel include transcriptome-wide response to water-deficit, cold, and salt stress in rice (Ray et al., <a href="#B38">2011</a>; Venu et al., <a href="#B44">2013</a>). </p> <p>There have been many other whole transcriptome microarray studies in rice under different abiotic stress conditions, but they reported only specific gene families that responded to various stresses. They include the MADS-box transcription factor family (Arora et al., <a href="#B2">2007</a>), F-Box Proteins (Jain et al., <a href="#B19">2007</a>), calcium-dependent protein kinase (CDPK) gene family (Ray et al., <a href="#B37">2007</a>), auxin-responsive genes (Jain and Khurana, <a href="#B18">2009</a>), protein phosphatase gene family (Singh et al., <a href="#B39">2010</a>), Sulfotransferase (SOT) gene family (Chen et al., <a href="#B10">2012</a>), thioredoxin gene family (Nuruzzaman et al., <a href="#B31">2012</a>), half-size ABC protein subgroup G (Matsuda et al., <a href="#B26">2012</a>), class III aminotransferase gene family (Sun et al., <a href="#B41">2013</a>), Ca <sup>2+</sup> ATPases gene family (Kamrul Huda et al., <a href="#B21">2013</a>), Rice RING E3 Ligase Family (Lim et al., <a href="#B23">2013</a>) etc. </p> <p>Hetetrotrimeric G-protein signaling components have often been implicated in stress response in plants. For example, in pea, Gα subunit was shown to be up-regulated by heat, as well as to impart heat and salt tolerance when overexpressed in transgenic tobacco, whereas the Gβ subunit imparted only heat tolerance (Misra et al., <a href="#B27">2007</a>). The role of α subunit in salt stress has also been shown in Arabidopsis (Colaneri et al., <a href="#B11">2014</a>), rice, and maize (Urano et al., <a href="#B43">2014</a>). Recently, we demonstrated that stress-related genes/pathways constitute the largest functional cluster of GPCR/G-protein-regulated genes in <i>Arabidopsis</i> using whole transcriptome analyses of knock-out mutants of GCR1 and GPA1 (Chakraborty et al., <a href="#B8">2015a</a>, <a href="#B7">b</a>). </p> <p>The rice G protein subunits are well characterized as RGA1 for Gα subunit (Ishikawa et al., <a href="#B17">1995</a>), RGB1 for Gβ subunit (Ishikawa et al., <a href="#B16">1996</a>) and RGG1 and RGG2 for the Gγ subunits (Kato et al., <a href="#B22">2004</a>). The expression of rice Gα subunit (RGA1) gene was reported to be up-regulated by salt, cold, and drought stresses, and down regulated by heat stress (Yadav et al., <a href="#B47">2013</a>). However, the regulation of the two Gγ subunits was different—while both RGG1 and RGG2 were up-regulated in salt, cold, heat, and ABA treatments, only RGG1 was up-regulated in drought stress (Yadav et al., <a href="#B46">2012</a>). While these two studies demonstrated that abiotic stresses regulate the expression of Gα and Gγ genes in rice, the role of G-proteins in mediating various stress responses in rice remains uncharacterized on a genome-wide scale. The availability of a natural mutant of RGA1 (D1) in rice (Ashikari et al., <a href="#B3">1999</a>) makes a functional genomic approach particularly attractive in this regard. We carried out a microarray analysis of this RGA1 mutant in comparison with the wild type in rice (GSE 20925 at NCBI GEO), which provided a convenient starting point for the present study, to examine the stress-related genes in the genome-wide response to the RGA1 null mutation in rice. In specific terms, we asked what proportion of the RGA1-regulated transcriptome corresponds to abiotic stress response in rice and how are these genes distributed in terms of major individual abiotic-stresses or in terms of their differential regulation in the RGA1 mutant or normal rice plants. We report here an integrative analysis of our experimental RGA1 mutant microarray data with the <i>in silico</i> meta data analysis of the known response of normal rice plants to various abiotic stresses. </p> </div> <div class="materials and methods" title="materials and methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="plantmaterialandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant material and growth conditions</div> <p>Seeds of the rice d1 mutant (devoid of Gα subunit or RGA1) and its corresponding wild type ( <i>Oryza sativa</i> japonica Nipponbare) were obtained from the Faculty of Agriculture, Kyushu University, Japan. They were surface-sterilized with 70% ethanol and 0.01% Triton-X 100 and grown on 0.5x B5 media containing 0.7% agar at 25 ± 1°C with fluorescent white light intensity of 1 kilo lux and a 12/12 photoperiod for 25 days till the emergence of the tertiary leaves and used for microarray analysis. </p> </div> <div class="rnaisolationandanalysis" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation and analysis</div> <p>Total RNA was isolated by hot phenol extraction and lithium chloride precipitation method as described (Pathak and Lochab, <a href="#B35">2010</a>). Total RNA was qualitatively and quantitatively analyzed by spectrophotometry and agarose gel electrophoresis. Prior to microarray experiments, RNA integrity values (RIN) of the total RNA samples were determined using the Agilent 2100 Bionalyzer equipment as per the manufacturer's instructions and only samples with RIN values higher than 5 were used for microarray experiments. </p> </div> <div class="wholetranscriptomemicroarray" title="sec"> <div class="title" tagx="title" title="title">
Whole transcriptome microarray</div> <p>cRNA labeling of total RNAs from the RGA1 mutant and its corresponding wild type was carried out using Agilent Low RNA Input Fluorescent Linear Amplification Kit (USA) as per the manufacturer's instructions, using Cy3 and Cy5 dyes (Perkin-Elmer, USA). Amplified samples were purified using Qiagen's RNeasy mini spin columns. The quantity and specific activity of cRNA was determined by using NanoDrop ND-1000 Spectrophotometer. Samples with specific activity &amp;gt;8 were hybridized with Agilent rice whole genome 60-mer microarrays (4 × 44 K, Ver 2) at 65°C for 17 h using Agilent Microarray Hybridization materials and equipment, as per the manufacturer's instructions. Slides were washed for 1 min each with Agilent Gene expression Wash Buffer I and II at RT and 37°C, respectively, and rinsed with acetonitrile for cleaning up and drying. They were scanned on an Agilent scanner (G2565B) at 100% laser power. Data extraction was carried out with Agilent Feature Extraction software (version 9.1). </p> <p>The raw data was normalized using the recommended "Per Chip and Per Gene Normalization" feature of the software GeneSpring GX Version 11.5. The correlation of replicates was checked using principal component analysis and correlation coefficients were obtained. The geometric mean (geomean) fold change values are represented as log <sub>2</sub>. The average data of biological replicates were used for final calculations. Log <sub>2</sub> fold change value of 1.0 with a <i>p</i>-value of 0.05 was taken as the cut-off to identify the differentially regulated genes (DEGs). </p> </div> <div class="dataminingandmeta-analysisofthestressrelatedgenes" title="sec"> <div class="title" tagx="title" title="title">
Data mining and meta-analysis of the stress related genes</div> <p>The stress-related genes were segregated from the above RGA1-regulated DEGs using the GO term "stress." This was done using rice genome annotation version 7 and also validated with the "manually curated database for rice proteins" (Gour et al., <a href="#B15">2013</a>). Further data mining was done using the genes corresponding to individual stresses downloaded from the stress responsive transcription factor database (STIFDB2.0, Naika et al., <a href="#B29">2013</a>), to find RGA1-regulated DEGs corresponding to heat, drought, salt, and cold. In order to identify additional stress-related genes among RGA1-responsive genes, our entire RGA1-regulated transcriptome was used as an input at the online database RiceDB (Narsai et al., <a href="#B30">2013</a>) to identify all the rice genes that responded to at least one of the four abiotic stresses i.e., cold, heat, drought, and salt. These genes were sorted into up-regulated and down-regulated sets and subjected to various Venn selections (Oliveros, <a href="#B32">2007–2015</a>) to generate a core list of 1498 stress-responsive genes common to all four stresses in rice. The core gene list was further classified into various functional categories, pathways and processes using a GO enrichment analysis tool, AGRIGO (Du et al., <a href="#B14">2010</a>) with binomial statistical test and cut-off for FDR-adjusted <i>P</i>-value of 0.05. Hierarchical clustering was done using average linkage based on Euclidean distance subsets of individual stress conditions such as heat, cold, drought/dehydration, salt, submergence, and shift from aerobic to anaerobic germination, cold, and drought. Biclustering was done with a threshold value of 1 and the largest bicluster was used for the analysis. Expression data were obtained for both the clustering analyses using Genevestigator (Zimmermann et al., <a href="#B49">2004</a>). </p> </div> <div class="rt-pcrvalidationofthestress-relatedgenes" title="sec"> <div class="title" tagx="title" title="title">
RT-PCR validation of the stress-related genes</div> <p>In order to validate the stress-responsive genes identified from the microarray results, quantitative RT-PCR experiments were carried out using total RNAs isolated from two biological replicates of the wild type and RGA1 mutant rice plants grown and harvested under similar conditions. Two technical replicates were used to set up RT-PCR from each of the biological replicates, using gene-specific primers designed in-house for the selected genes. The primer sequences are provided in Supplementary Table <a href="#SM1">2</a>. PCR amplifications were performed in 20 μl by using the KAPA SYBR FAST universal QPCR kit (KAPA BIOSYSTEMS) with 1.0 μl of sample cDNA prepared by using iScript cDNA synthesis kit (Cat#170-8891) from BIORAD and 100 n moles of each gene-specific primer. Actin (ACT) was used as an internal control for normalization. Quantification of the relative changes in gene expression was performed by using the 2 <sup>−ΔΔCT</sup> method (Pfaffl, <a href="#B36">2001</a>). </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>Whole transcriptome microarray analysis of the rice RGA1 (Gα) null mutant in comparison with its WT yielded a total of 2270 differentially expressed genes under MIAME compliant conditions, using stringent cut-off values (geomean 1.0 with <i>p</i>-value of 0.05) and removing redundancies. The raw data of this entire microarray experiment are reported at NCBI GEO (GSE 20925). Among these RGA1-regulated genes, a large number of abiotic stress-responsive genes have been identified using their annotation information or online databases for further bioinformatic analysis as detailed below. </p> <div class="stress-responsivegenesidentifiedbygo-terms" title="sec"> <div class="title" tagx="title" title="title">
Stress-responsive genes identified by GO-terms</div> <p>Our search for stress-related genes among these RGA1-regulated DEGs using the GO terms related to stress yielded 94 abiotic stress-related DEGs that are nearly equally distributed in terms of up/down regulation (49 up/45 down). A vast majority of these genes could be clustered into &amp;lt; 40 related families (20 up/20 down) showing identical mode of up/down regulation, despite wide variation in the extent of their regulation (Table <a href="#T1">1</a>). For example, all the RGA1-regulated members of gene families such as DREB seem to be uniformly up-regulated, albeit to varying extents, ranging between +3.99 and +1.18. In addition, there are 21 stress-related DEGs that are individually regulated in the RGA1 mutant with no other family member, including up-regulated genes such as CDPK, MAP kinase kinase 2, DnaJ like protein, and down-regulated genes such as Myb factor, phytochelatin synthetase, and water-stress inducible protein (RAB21). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p> <b>RGA1-regulated stress-responsive gene families with their fold changes in mutant</b>. </p>  <table> <thead> <tr> <th> <b>RGA1 up regulated stress responsive families</b> </th> <th> <b>Fold change</b> </th> <th> <b>RGA1 down regulated stress responsive families</b> </th> <th> <b>Fold change</b> </th> </tr> </thead> <tbody> <tr> <td>DRE-binding protein 1A,1B,1C</td> <td>3.99–1.18</td> <td>Peroxidase</td> <td>−6.09 to −1.42</td> </tr> <tr> <td>Metallothionein-like protein type 2</td> <td>3.88–3.47</td> <td>Catalase isozyme</td> <td>−3.33 to −3.09</td> </tr> <tr> <td>Catalase isozyme 2</td> <td>2.89–2.6</td> <td>Haem peroxidase family protein.</td> <td>−2.88 to −1.47</td> </tr> <tr> <td>Glutathione S-transferase GST 19,6</td> <td>2.49–1.55</td> <td>LRR-like protein</td> <td>−2.71 to −2.54</td> </tr> <tr> <td>Heat shock proteins (70,82,90)</td> <td>2.31–1.47</td> <td>Pathogen-inducible alpha dioxygenase</td> <td>−2.60 to −2.58</td> </tr> <tr> <td>MAP Kinase</td> <td>2.19–2.06</td> <td>Triosephosphate isomerase</td> <td>−1.89 to −1.63</td> </tr> <tr> <td>Endo-1, 3; 1, 4-beta-D-glucanase</td> <td>2.06–1.55</td> <td>Anth (Pollen-specific desiccation-associated LLA23 protein)</td> <td>−1.86 to −1.59</td> </tr> <tr> <td>Rossmann-like alpha/beta/alpha sandwich fold domain containing protein</td> <td>2.05–1.28</td> <td>Cytosolic 6-phosphogluconate dehydrogenase</td> <td>−1.81 to −1.72</td> </tr> <tr> <td>Zinc finger, domain containing proteins (AN1-type, TAZ-type)</td> <td>2.04–1.50</td> <td>Glycine-rich RNA-binding protein 1</td> <td>−1.79 to −1.72</td> </tr> <tr> <td>Heat stress transcription factor Spl7 (RHSF10)</td> <td>1.99–1.11</td> <td>Beta-glucanase precursor</td> <td>−1.72 to −1.64</td> </tr> <tr> <td>Sucrose synthase 2</td> <td>1.84–1.33</td> <td>Cytosolic Glyceraldehyde-3-phosphate dehydrogenase</td> <td>−1.70 to −1.50</td> </tr> <tr> <td>Anthranilate synthase beta chain</td> <td>1.75–1.44</td> <td>Superoxide dismutase</td> <td>−1.69 to −1.49</td> </tr> <tr> <td>Manganese-superoxide dismutase precursor</td> <td>1.52–1.26</td> <td>WW/Rsp5/WWP domain containing protein</td> <td>−1.59 to −1.33</td> </tr> <tr> <td>6-phosphogluconate dehydrogenase</td> <td>1.46–1.35</td> <td>Serine/threonine-protein kinase SAPK3 (Osmotic stress/abscisic acid-activated protein kinase 3) </td> <td>−1.46 to −1.36</td> </tr> <tr> <td>Serine/threonine-protein kinase SAPK2/9</td> <td>1.34–1.05</td> <td>Alcohol dehydrogenase 1.</td> <td>−1.42 to −1.23</td> </tr> <tr> <td>UspA domain containing protein</td> <td>1.30–1.18</td> <td>Class III peroxidase GvPx2b</td> <td>−1.39 to −1.08</td> </tr> <tr> <td>GTP-binding nuclear protein Ran1B</td> <td>1.15–1.04</td> <td>Nucleoside diphosphate kinase I</td> <td>−1.36 to −1. 33</td> </tr> <tr> <td>Multiple stress-responsive zinc-finger protein ISAP1 (Stress-associated protein 1)</td> <td>1.14–1.09</td> <td>Hypothetical conserved gene. Similar to peroxidase (Os03t0339300-02)</td> <td>−1.25 to −1. 07</td> </tr> <tr> <td>OSIGBa0145M07.4 protein</td> <td>1.83–1.14</td> <td>Aldehyde dehydrogenase</td> <td>−1.21 to −1.16</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>From the differentially regulated genes (DEGs) identified in our microarray analysis of the RGA1 mutant, 94 genes with GO terms related to stress were segregated and categorized into families </i>. </p> </div> </div> </div> <div class="stress-responsivegenesidentifiedatstifdb2.0" title="sec"> <div class="title" tagx="title" title="title">
Stress-responsive genes identified at STIFDB2.0</div> <p>Data mining for all abiotic stress-responsive genes of rice at STIFDB2.0 yielded 626 genes in all, corresponding to heat (522), drought (101), salt (37), and cold (15), as shown in the left panel of Figure <a href="#F1">1</a>. A Venn selection between these 626 stress responsive genes and the 2270 RGA1-regulated genes identified on our microarray yielded 106 genes (Figure <a href="#F1">1</a>, inset), indicating the role of RGA1 in mediating their stress regulation. A significant majority of them respond to heat (94), followed by drought (13), salt (6), and cold (4), with the 25 genes being common to salt and drought stresses (Figure <a href="#F1">1</a>, right panel). But this order becomes very different when seen in terms of what proportion of each of the stress responses was RGA1-regulated: With four out of all 15 cold-responsive genes listed at STIFDB2.0 being regulated by RGA1, cold-response has the highest proportion of genes under the regulation of RGA1 (27%), followed by heat (18%), salt (16%), and drought (13%). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p> <b>Stress responsive genes among RGA1-regulated genes in rice</b>. The left panel shows Venn selections between the subsets of all rice abiotic stress-responsive genes listed at STIFDB2.0. The inset shows Venn selection between all 626 abiotic stress-responsive genes listed at STIFDB2.0 and 2270 RGA-1-regulated DEGs identified on our microarray. The left panel shows the break-up of the 106 RGA-regulated stress-responsive genes identified in the inset in terms of individual stresses viz., heat (94), drought (13), salt (6), and cold (4). </p>   </div> </div> <div class="expressionprofilesofrga1-regulatedstress-responsivegenes" title="sec"> <div class="title" tagx="title" title="title">
Expression profiles of RGA1-regulated stress-responsive genes</div> <p>Hierarchical clustering of the transcripts of the 106 RGA1-regulated stress responsive genes using Genevestigator revealed their differential expression under 132 perturbations related to abiotic stress studies reported in literature. Out of them, the data in Figure <a href="#F2">2</a> include only 118 perturbations such as heat, cold, drought/dehydration, salt, submergence, and shift from aerobic to anaerobic germination, that have affected the expression of the vast majority of 106 genes queried based on our study. This revealed a prominent cluster of over 25 genes that are highly up-regulated (over 2.5-fold) and a similar number of highly down-regulated (over 2.5-fold) under 24 cold stress studies in literature on rice. There are an even larger number of genes that are differentially regulated under drought, of which the down-regulated genes are both predominant and better clustered, relative to the up-regulated genes. Though there are a smaller number of heat responsive genes, they are neither well clustered not consistent between different studies. With respect to salt, the results from nine studies show that very few of our 106 RGA-regulated genes respond to salt stress in rice. In view of these findings, further <i>in silico</i> analysis of transcript profiles was restricted to cold and drought stress conditions. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <b>Hierarchical clustering of the transcripts of 106 RGA1-regulated stress-responsive genes in various abiotic stress studies in rice (118 perturbations) </b>. The red and green colors indicate up-regulation (log2 [2.5]) and down-regulation (log2 [−2.5]), respectively as shown in the color bar. Hierarchical clustering was done using average linkage based on Euclidean distance subsets of individual stress conditions such as heat, cold, drought/dehydration, salt, submergence, and shift from aerobic to anaerobic germination cold and drought. The expression data were obtained using Genevestigator (Zimmermann et al., <a href="#B49">2004</a>). </p>   </div> <p>Biclustering analysis of the expression profiles of 106 RGA1-regulated, stress-responsive genes in various studies on cold stress revealed that 17 genes were up-regulated and 11 genes were down-regulated in 39 different perturbations/studies (Figure <a href="#F3">3</a>, left panel). Their comparison with the actual fold-change values of those genes on our microarray revealed that about half of them are similarly up-regulated in both RGA1 mutant (without stress) as well as in normal rice plants under cold stress. The remaining genes include seven up-regulated genes and six down-regulated genes in the RGA1 mutant with opposite pattern of regulation under cold stress in normal rice plants in literature (Figure <a href="#F3">3</a>, right panel). The genes up-regulated in the RGA1 mutant but down-regulated by cold stress in the normal plants include mitochondrial chaperonin-60, 4,5-DOPA dioxygenase extradiol-like protein, isoform 2 of heat stress transcription factor B-2c, cytochrome P450 family protein, calcyclin-binding protein, DnaJ-like protein. The genes down-regulated in the RGA1 mutant but up-regulated in normal plants include amino acid transporter-like protein and alpha-amylase isozyme 3D precursor. The opposite pattern of regulation of these genes could be due to the RGA1 mutation, which indicates that RGA1 may mediate the response of these genes to cold stress. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <b>Expression profiles of 106 RGA1-regulated stress-responsive genes in cold stress (39 perturbations from literature) </b>. The red and green colors indicate up-regulation (log2 [2.5]) and down-regulation (log2 [−2.5]), respectively, as shown in the color bar. The expression data in the left panel were obtained using Genevestigator. The table compares their regulation in normal plants under stress in literature with actual fold-change values in the RGA1 mutant. </p>   </div> <p>A similar biclustering analysis of the expression patterns of 106 RGA1-regulated stress-responsive genes in studies on drought stress revealed that 13 genes were up-regulated and 10 genes were down-regulated in 30 different perturbations/studies (Figure <a href="#F4">4</a>, left panel). When their up/down regulation was compared with the actual fold-change values obtained on our microarray, six of the up-regulated genes and one of the down-regulated genes from literature are similarly up-regulated in both RGA1 mutant (without stress) as well as in normal rice plants under drought stress. Among the rest, seven up-regulated and three down-regulated genes in the RGA1 mutant showed opposite pattern of regulation under drought stress in normal rice plants in literature (Figure <a href="#F4">4</a>, right panel). The genes up-regulated in the RGA1 mutant but down-regulated by drought stress in the normal plants include flavanone 3-hydroxylase-like protein, Isoform 2 of heat stress transcription factor, B-2cAlpha/beta hydrolase fold-3 domain containing protein, U box domain containing protein, and plant basic secretory protein family protein. The genes down-regulated in the RGA1 mutant by up-regulated in normal plants include Trehalose-6-phosphate synthase, MPI, and Ntdin. The opposite pattern of regulation of these genes could be due to the RGA1 mutation, which indicates that RGA1 may mediate the response of these genes to drought stress. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p> <b>Expression profiles of 106 RGA1-regulated stress- responsive genes in drought stress (30 perturbations from literature) </b>. The red and green colors indicate up-regulation (log2 [2.5]) and down-regulation (log2 [−2.5]), respectively, as shown in the color bar. The expression data in the left panel were obtained using Genevestigator. The table compares their regulation in normal plants under stress in literature with actual fold-change values in the RGA1 mutant. </p>   </div> <p>Interestingly, ribose phosphate pyrophosphokinase 3 is up-regulated in the RGA1 mutant as well as in response to cold and drought stress in literature, whereas isoform 2 of the heat stress transcription factor is up-regulated in the RGA1 mutant, but down-regulated in drought and cold stresses. </p> </div> <div class="meta-dataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Meta-data analysis</div> <p>Data mining using our entire non-redundant list of 2270 RGA1-regulated DEGs (1242 up and 1028 down) as input query at the Rice DB Oryza information portal revealed a much larger number of 1886 stress-related genes as differentially regulated in our RGA1 mutant. This prompted a comparison of various stress-responsive gene lists identified using different approaches in this study, such as gene ontology (94), STIFDB2.0 (106), and Rice DB (1886). A Venn selection of all three sets revealed that the former two are largely subsets of the 1886 DEGs identified using Rice DB (Figure <a href="#F5">5</a>). Therefore, the rest of the meta-data analysis was carried out using these 1886 genes. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p> <b>Venn selection of RGA1 regulated stress responsive genes mined from RiceDB, STIFDB, and GO term </b>. The overlap among the three sets revealed that the genes mined using GO term stress and stress responsive genes from STIFDB are largely subsets of the 1886 DEGs identified using Rice DB. </p>   </div> <p>The distribution of these 1886 RGA1-regulated, stress-responsive DEGs in terms of individual stresses was found to be 1730 DEGs in salt stress (889 up/841 down), 1719 DEGs in drought (882 up/837 down), 1690 DEGs in heat (913 up/777 down), and 1688 DEGs in cold (878 up/810) down with 1498 genes (773 up/725 down) common to all four stresses (Figure <a href="#F6">6</a>). In other words, as many as 1886 G-protein-regulated genes are responsive to one or more of these stresses, indicating their possible regulation through G-protein (RGA1) signaling. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p> <b>Meta-data analysis of RGA1-regulated genes regulated under various abiotic stresses</b>. The 2270 RGA1-regulated genes (1242 up and 1028 down) were used as input query at Rice DB to generate genes responsive to cold (878 up/810 down), drought (882 up/837 down), heat (913 up/777 down), and salt stress (889 up/841 down) with 1498 genes common to all four stresses and totaling 1886 unique genes. Their Venn selections are depicted as total <b>(A)</b>, up-regulated <b>(B)</b> and down-regulated <b>(C)</b> sets, using the online tool Venny (Oliveros, <a href="#B32">2007–2015</a>). </p>   </div> <p>Interestingly, the largest majority of 1886 with 1498 genes (or 80%) are common to all four abiotic stresses, followed by 137 genes common to cold, drought and salt stresses, followed by 38 genes common to drought, heat, and salt and so on, indicating their common regulation through G-proteins (Table <a href="#T2">2</a>). Even more interesting is the fact that as many as 111 heat-responsive genes are not common to any other stress and are uniquely regulated through G-proteins by heat only, followed by eight genes each in cold only and drought only, and two genes in salt stress only (Table <a href="#T2">2</a>). Some of the exclusively heat-responsive RGA1-regulated genes include superoxide dismutase, chitin-inducible gibberellin-responsive protein, brassinosteroid insensitive 1-associated receptor kinase 1, Hsp70 heat shock family protein, GTP-binding nuclear protein Ran1B (fragment), low affinity sulfate transporter 3, mitochondrial chaperonin-60, nucleoside diphosphate kinase I (EC 2.7.4.6) (NDPK I), wound responsive protein, and auxin response factor 2 (ARF1-binding protein). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p> <b>Distribution of RGA1-regulated genes among major abiotic stresses in Rice DB</b>. </p>  <table> <thead> <tr> <th> <b>Stress categories</b> </th> <th> <b>Up regulated in RGA1 mutant</b> </th> <th> <b>Down regulated in RGA1 mutant</b> </th> <th> <b>Total</b> </th> </tr> </thead> <tbody> <tr> <td>Cold, drought, heat, and salt</td> <td>773</td> <td>728</td> <td>1498 <a href="#TN1"> <sup>*</sup> </a> </td> </tr> <tr> <td>Cold, drought, and salt</td> <td>72</td> <td>65</td> <td>137</td> </tr> <tr> <td>Drought, heat, and salt</td> <td>17</td> <td>21</td> <td>38</td> </tr> <tr> <td>Cold, heat, and salt</td> <td>16</td> <td>8</td> <td>24</td> </tr> <tr> <td>Drought and salt</td> <td>7</td> <td>15</td> <td>22</td> </tr> <tr> <td>Cold and drought</td> <td>8</td> <td>2</td> <td>10</td> </tr> <tr> <td>Drought and heat</td> <td>6</td> <td>2</td> <td>8</td> </tr> <tr> <td>Heat and salt</td> <td>6</td> <td>1</td> <td>7</td> </tr> <tr> <td>Cold and heat</td> <td>3</td> <td>3</td> <td>6</td> </tr> <tr> <td>Cold and salt</td> <td>1</td> <td>4</td> <td>5</td> </tr> <tr> <td>Cold, drought, and heat</td> <td>2</td> <td>1</td> <td>3</td> </tr> <tr> <td>Heat only</td> <td>94</td> <td>17</td> <td>111</td> </tr> <tr> <td>Cold only</td> <td>6</td> <td>2</td> <td>8</td> </tr> <tr> <td>Drought only</td> <td>1</td> <td>7</td> <td>8</td> </tr> <tr> <td>Salt only</td> <td>0</td> <td>2</td> <td>2</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>The 1886 RGA1-regulated genes identified as responsive to abiotic stresses at Rice DB have been categorized in terms of shared/unique stress categories and their up/down regulation in the RGA1 mutant </i>. </p> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">*</span> <p> <i>Three genes out of 1501 were redundant or common to up/down categories, hence 1498</i>. </p> </div> </div> </div> <p>In order to validate the stress-responsive genes identified from the microarray results, quantitative RT-PCR experiments were carried out using total RNAs isolated from the wild type and RGA1 mutant rice plants grown and harvested under similar conditions. Out of the 1498 RGA1-regulated genes identified as common to multiple abiotic stresses on the microarray, 12 of the most up/downregulated genes were validated by qRT-PCR. Their fold change data are shown in Figure <a href="#F7">7</a> along with microarray results. The data clearly show the broad correspondence between the microarray data and RT-PCR results for both upregulated and downregulated sets of genes. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p> <b>qRT-PCR validation of RGA1-regulated genes identified as common to various abiotic stresses </b>. Out of the 1498 RGA1-regulated genes identified as common to multiple abiotic stresses on the microarray, 12 of the most up/down-regulated genes were validated by qRT-PCR. Their fold change data are shown based on averages of two biological replicates and two technical replicates of total RNA, along with microarray results. The left panel in red shows the up-regulated genes and the right panel in green shows the down-regulated genes. </p>   </div> <p>Gene ontology analysis of the core list of 1498 genes shared by all four stresses revealed 62 unique GO terms associated with molecular functions such as transporter activity, enzyme regulator activity, transferase activity, hydrolase activity, metabolic processes (carbon and protein), binding to nucleotides, carbohydrates, receptors and lipids, anatomical structure morphogenesis, flower development, and cell homeostasis (Supplementary Table <a href="#SM1">1</a>). Further analysis using AGRIGO showed that many of these 1498 shared stress-responsive genes also share many GO terms of biological process, such as response to stimuli (GO: 0050896) with 49 genes out of the 95 genes (or 51%) accepted by AGRIGO for the query; 29 genes (30%) in response to chemical stimulus (GO: 0042221), 49 genes (51%) in response to stress (GO:0006950); 25 genes (26%) belong to oxidation reduction (GO:0055114); five genes (5%) belong to the category cellular response to chemical stimulus (GO:0070887), and 25 genes (26%) belong to response to oxidative stress (GO:0006979; Figure <a href="#F8">8</a>). This reveals the role of RGA1 in regulating a diverse range of processes related to stress response. GO terms of molecular function such as electron carrier activity had 80 genes (4%) and 61 genes (3%) in calcium ion binding out of a total of 1942 genes, indicating the role of RGA1 in their regulation. Its role also seems to be important in regulating the products of diverse cellular locations, such as etioplasts (130 genes), mitochondria (33 genes), plastid (16 genes), nucleus (15 genes), chloroplast (12 genes), and three genes each in endoplasmic reticulum, vacuole, and golgi apparatus (Figure <a href="#F8">8</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p> <b>Gene Ontology enrichment of RGA1-regulated, stress responsive genes from Rice DB</b>. The 1498 genes common to all four major abiotic stresses were subjected to GO enrichment using AgriGO with default settings. <b>(A)</b> Biological process categorization of the RGA1-regulated genes shared by salt, heat, cold, and drought stresses. <b>(B)</b> Molecular function categorization and <b>(C)</b> Subcellular localization of the RGA1-regulated genes shared by all four abiotic-stresses. </p>   </div> </div> <div class="miningformirnastargetingrga1-regulated,stressresponsivegenes" title="sec"> <div class="title" tagx="title" title="title">
Mining for miRNAs targeting RGA1-regulated, stress responsive genes</div> <p>Data mining for miRNAs at Rice DB using the GO terms of 1498 RGA1-regulated genes shared by all four stresses revealed that 63 of them could be targets of miRNAs. This indicates the role of RGA1 in post-transcriptional regulation of 63 target genes for the first time. They include 38 up-regulated genes and 25 down-regulated genes identified in the RGA1 mutant (Table <a href="#T3">3</a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p> <b>miRNA targets among RGA1-regulated stress-responsive genes</b>. </p>  <table> <thead> <tr> <th> <b>Gene identifier</b> </th> <th> <b>miRNA_gene</b> </th> <th> <b>Rap_description</b> </th> <th> <b>GOID</b> </th> </tr> </thead> <tbody> <tr> <td>Os11g0119100</td> <td>OsSBS99</td> <td>Catalytic/hydrolase</td> <td>GO:0008152</td> </tr> <tr> <td>Os11g0115400</td> <td>OsSBS97</td> <td>Lipid transfer protein LPT IV</td> <td>NA</td> </tr> <tr> <td>Os01g0679600</td> <td>OsSBS87</td> <td>THAP domain-containing protein 4</td> <td>GO:0003674</td> </tr> <tr> <td>Os03g0206400</td> <td>OsSBS73</td> <td>Conserved hypothetical protein</td> <td>GO:0003674</td> </tr> <tr> <td>Os10g0181200</td> <td>OsSBS68</td> <td>Protein prenyltransferase</td> <td>GO:0006139</td> </tr> <tr> <td>Os08g0562600</td> <td>OsSBS67ab</td> <td>C2 calcium-dependent membrane targeting</td> <td>GO:0003674</td> </tr> <tr> <td>Os10g0391400</td> <td>OsSBS58</td> <td>Tify domain containing protein</td> <td>GO:0006950</td> </tr> <tr> <td>Os03g0818400</td> <td>OsSBS53</td> <td>40S ribosomal protein S23 (S12)</td> <td>GO:0005840</td> </tr> <tr> <td>Os05g0128200</td> <td>OsSBS45</td> <td>Hypothetical conserved gene</td> <td>GO:0009058</td> </tr> <tr> <td>Os04g0396800</td> <td>OsSBS4</td> <td>Serine carboxypeptidase K10B2.2</td> <td>GO:0005618</td> </tr> <tr> <td>Os03g0126000</td> <td>OsSBS27</td> <td>Phosphorybosylanthranilatetransferase 1</td> <td>GO:0008152</td> </tr> <tr> <td>Os08g0561700</td> <td>osa-MIR552</td> <td>Superoxide dismutase</td> <td>GO:0008152</td> </tr> <tr> <td>Os03g0738400</td> <td>osa-MIR530-5p′</td> <td>Serine hydroxymethyltransferase,)</td> <td>GO:0005739</td> </tr> <tr> <td>Os02g0761400</td> <td>osa-MIR530-3p</td> <td>ATPREP2</td> <td>GO:0005576</td> </tr> <tr> <td>Os08g0137400</td> <td>osa-MIR530</td> <td>Cupredoxin domain containing protein</td> <td>GO:0003674</td> </tr> <tr> <td>Os09g0428000</td> <td>osa-MIR529b</td> <td>Glycosyltransferase, family 2 domain protein</td> <td>GO:0016740</td> </tr> <tr> <td>Os01g0823600</td> <td>osa-MIR444f.17</td> <td>Conserved hypothetical protein</td> <td>GO:0003674</td> </tr> <tr> <td>Os02g0324400</td> <td>osa-MIR444c-5p</td> <td>FON2 SPARE1</td> <td>GO:0005576</td> </tr> <tr> <td>Os02g0274900</td> <td>osa-MIR444bc.9</td> <td>Metabolite transport protein csbC</td> <td>GO:0006810</td> </tr> <tr> <td>Os07g0583600</td> <td>osa-MIR444bc.25</td> <td>Chitin-inducible gibberellin-responsive protein</td> <td>GO:0007165</td> </tr> <tr> <td>Os03g0802500</td> <td>osa-MIR444ad.2,e</td> <td>ATPase, AAA-type, core domain</td> <td>GO:0005783</td> </tr> <tr> <td>Os08g0482700</td> <td>osa-MIR444</td> <td>Conserved hypothetical protein</td> <td>GO:0003674</td> </tr> <tr> <td>Os04g0459600</td> <td>osa-MIR442</td> <td>Mog1/PsbP, alpha/beta/alpha sandwich domain</td> <td>NA</td> </tr> <tr> <td>Os05g0414700</td> <td>osa-MIR403</td> <td>Brassinosteroid insensitive 1 receptor kinase 1</td> <td>GO:0005102</td> </tr> <tr> <td>Os05g0557700</td> <td>osa-MIR399j</td> <td>Conserved hypothetical protein</td> <td>GO:0005575</td> </tr> <tr> <td>Os01g0850700</td> <td>osa-MIR397b′</td> <td>Cupredoxin domain containing protein</td> <td>GO:0009056</td> </tr> <tr> <td>Os01g0121600</td> <td>osa-MIR396c-3p</td> <td>Conserved hypothetical protein</td> <td>GO:0006810</td> </tr> <tr> <td>Os01g0180800</td> <td>osa-MIR396c</td> <td>Heat shock protein Hsp70 family protein</td> <td>GO:0005634</td> </tr> <tr> <td>Os03g0225500</td> <td>osa-MIR395f ′</td> <td>Nucleoporin, Nup133/Nup155-</td> <td>GO:0005515</td> </tr> <tr> <td>Os05g0574500</td> <td>osa-MIR395c′o′</td> <td>GTP-binding nuclear protein Ran1B</td> <td>GO:0005515</td> </tr> <tr> <td>Os03g0195300</td> <td>osa-MIR395</td> <td>Low affinity sulfate transporter 3</td> <td>GO:0016020</td> </tr> <tr> <td>Os03g0559700</td> <td>osa-MIR393ab′</td> <td>Conserved hypothetical protein</td> <td>NA</td> </tr> <tr> <td>Os03g0388900</td> <td>osa-MIR319a.2</td> <td>Peptidase C14, caspase catalytic protein</td> <td>GO:0019538</td> </tr> <tr> <td>Os01g0323600</td> <td>osa-MIR2055′</td> <td>S-adenosylmethionine synthase 2</td> <td>GO:0016020</td> </tr> <tr> <td>Os03g0409100</td> <td>osa-MIR1884b-3p</td> <td>PUA-like domain domain protein</td> <td>GO:0019538</td> </tr> <tr> <td>Os08g0417000</td> <td>osa-MIR1884b</td> <td>2OG-Fe (II) oxygenase domain protein</td> <td>GO:0008152</td> </tr> <tr> <td>Os10g0462900</td> <td>osa-MIR1879′</td> <td>Mitochondrial chaperonin-60</td> <td>GO:0005739</td> </tr> <tr> <td>Os08g0512400</td> <td>osa-MIR1879</td> <td>Unknown function DUF296 domain protein</td> <td>GO:0003677</td> </tr> <tr> <td>Os07g0492000</td> <td>osa-MIR1862</td> <td>Nucleoside diphosphate kinase I (NDK I)</td> <td>GO:0005576</td> </tr> <tr> <td>Os01g0763200</td> <td>osa-MIR1861d</td> <td>Transcription factor PCF7 (Fragment)</td> <td>GO:0005575</td> </tr> <tr> <td>Os02g0830700</td> <td>osa-MIR1860-5p</td> <td>Leucine-rich repeat</td> <td>GO:0005623</td> </tr> <tr> <td>Os09g0530700</td> <td>osa-MIR1860-3p</td> <td>Hypothetical conserved gene</td> <td>GO:0003674</td> </tr> <tr> <td>Os02g0580900</td> <td>osa-MIR1858b</td> <td>TGF-beta receptor</td> <td>NA</td> </tr> <tr> <td>Os03g0667100</td> <td>osa-MIR1851</td> <td>BTB/POZ domain containing protein</td> <td>GO:0006950</td> </tr> <tr> <td>Os10g0510000</td> <td>osa-MIR1850</td> <td>Actin</td> <td>GO:0009719</td> </tr> <tr> <td>Os01g0720600</td> <td>osa-MIR1847.7</td> <td>Starch synthase IVa</td> <td>GO:0016740</td> </tr> <tr> <td>Os03g0101300</td> <td>osa-MIR1847.10</td> <td>Hexose transporter</td> <td>GO:0005829</td> </tr> <tr> <td>Os03g0821100</td> <td>osa-MIR1846d-5p</td> <td>Non-cell-autonomous HS cognate protein 70</td> <td>GO:0016020</td> </tr> <tr> <td>Os07g0684800</td> <td>osa-MIR1846c-5p</td> <td>NAM/CUC2-like protein</td> <td>GO:0005634</td> </tr> <tr> <td>Os08g0357000</td> <td>osa-MIR172c</td> <td>Wound responsive protein</td> <td>GO:0006950</td> </tr> <tr> <td>Os04g0442000</td> <td>osa-MIR172b</td> <td>Auxin response factor 2 (ARF1-BP)</td> <td>GO:0009058</td> </tr> <tr> <td>Os03g0784000</td> <td>osa-MIR172ad</td> <td>FAD dependent oxidoreductase family protein</td> <td>GO:0008152</td> </tr> <tr> <td>Os02g0662700</td> <td>osa-MIR171i</td> <td>Scl1 protein (Fragment)</td> <td>GO:0008150</td> </tr> <tr> <td>Os03g0696300</td> <td>osa-MIR169no</td> <td>Nuclear transcription factor Y subunit A-1</td> <td>GO:0007275</td> </tr> <tr> <td>Os03g0687000</td> <td>osa-MIR168a</td> <td>Predicted protein</td> <td>GO:0016020</td> </tr> <tr> <td>Os03g0640800</td> <td>osa-MIR166m</td> <td>(Homeodomain-leucine zipper protein 14)</td> <td>GO:0003677</td> </tr> <tr> <td>Os12g0147800</td> <td>osa-MIR164d</td> <td>Phytosulfokines 5 precursor</td> <td>NA</td> </tr> <tr> <td>Os05g0580000</td> <td>osa-MIR162b</td> <td>ADP-glucose pyrophosphorylase (EC 2.7.7.27)</td> <td>GO:0009058</td> </tr> <tr> <td>Os03g0140200</td> <td>osa-MIR1441</td> <td>Cytochrome P450 86A1</td> <td>GO:0005623</td> </tr> <tr> <td>Os02g0821200</td> <td>osa-MIR1428</td> <td>Ribosomal protein L28e domain protein</td> <td>GO:0005840</td> </tr> <tr> <td>Os01g0967800</td> <td>osa-MIR1328</td> <td>Conserved hypothetical protein</td> <td>NA</td> </tr> <tr> <td>Os10g0503800</td> <td>osa-MIR1322</td> <td>Remorin</td> <td>GO:0005575</td> </tr> <tr> <td>Os06g0195900</td> <td>NA</td> <td>NOG, C-terminal domain containing protein</td> <td>GO:0016020</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>Data mining at Rice DB using them are targets for miRNA regulation</i>. </p> </div> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Heterotrimeric G-protein subunits or their interacting partners have either been implicated in stress signal transduction or have been shown to respond to stress themselves (Urano et al., <a href="#B42">2013</a>). Experimental approaches, including genome-wide studies, were generally focused on the response to individual stresses or individual components of G-protein signaling. The role of the G-protein α subunit in individual abiotic stress responses has been in particular focus, in relation to heat/salt stress in pea (Misra et al., <a href="#B27">2007</a>) and salt stress in Arabidopsis (Colaneri et al., <a href="#B11">2014</a>), rice and maize (Urano et al., <a href="#B43">2014</a>), or indirectly in ABA signaling (Pandey et al., <a href="#B34">2010</a>; Alvarez et al., <a href="#B1">2011</a>) or oxidative stress (Booker et al., <a href="#B5">2012</a>). The expression of rice Gα subunit (RGA1) gene itself was reported to be up-regulated by salt, cold, and drought stresses, and down regulated by heat stress (Yadav et al., <a href="#B47">2013</a>). However, there are no comprehensive studies on the genome-wide involvement of any heterotrimeric G-protein subunit in all the main abiotic stresses in any plant, except Arabidopsis (Chakraborty et al., <a href="#B8">2015a</a>, <a href="#B7">b</a>). Comprehensive functional genomic analyses are particularly lacking on the genome-wide role of RGA1 or other G-protein subunits in multiple abiotic stress responses in rice. </p> <p>In view of our own recent findings reported elsewhere in this issue on the growing importance of G-protein signaling components in abiotic stress response in <i>Arabidopsis</i> (Chakraborty et al., <a href="#B9">2015c</a>), as well as the importance of abiotic stress in rice crop improvement, we sought to examine the abiotic stress component of our RGA1 transcriptome microarray data in detail. This was done by combining our experimental functional genomic data with <i>in silico</i> meta data analysis to answer the following questions: Does abiotic stress figure prominently in the genome-wide response to RGA1 null mutation in rice and if yes, what are the various genes involved and how are they distributed in terms of major individual abiotic-stresses or in terms of their differential regulation in the RGA1 mutant? How do they compare with the known genome-wide response of normal rice plants to various abiotic stresses? Can <i>in silico</i> transcriptome meta-data analyses provide adequate insights for integrative understanding on abiotic stress signaling components in rice as possible converging points for interventions? </p> <p>Our microarray experiments under MIAME compliant conditions using the Japonica rice RGA1 mutant and wild type (GSE 20925 at NCBI GEO) revealed 2270 differentially expressed genes, out of which the stress responsive data set was identified and analyzed using three approaches: Gene Ontology terms, data mining from STIFDB, and meta-data analysis from Rice DB. Firstly, segregation using Gene Ontology terms yielded 94 genes corresponding to various abiotic stress categories, most of which belonged to less than 40 families (Table <a href="#T1">1</a>), indicating their regulation by RGA1. The fact that majority of these families showed similar patterns of up/down regulation indicates that their regulation by RGA1 is also uniform, while there are a few families such as those related to oxidative stress response that show differential regulation of their members in the RGA1 mutant. The uniform mode of up/down regulation of multiple members of the same family of stress-responsive genes reveals the inherently coordinated pattern of gene regulation in response to a stress signal (e.g., DREB), whereas the varied extent of that regulation reveals the fine tuning of the signal/response flux through a regulatory cascade. Such patterns of regulation may be amenable to deeper network analysis. </p> <p>Secondly, data mining for genes specifically categorized as stress-responsive genes from Japonica rice at STIFDB yielded 626 genes, out which 106 genes belonging to various abiotic stresses—heat drought, salt cold, were RGA1-regulated (Figure <a href="#F1">1</a>). Together, these 106 abiotic stress-responsive genes constitute less than 5% of all the G-protein (RGA1) regulated genes. But they constitute a far higher proportion (17%) of the 626 abiotic stress-responsive genes, indicating the larger role for G-proteins in regulating them, even though mediating abiotic stress seems to be a smaller part of the genome-wide role of G-proteins. However, this difference may also be an artifact arising out of the relatively lesser coverage of 626 rice stress-responsive genes on the STIFDB, as compared to 3150 genes in <i>Arabidopsis</i>, as similar analysis on its GPA1 mutant produced more consistent ranking with cold&amp;gt;salt&amp;gt;drought (Chakraborty et al., <a href="#B9">2015c</a>). </p> <p>Hierarchical clustering of the 106 RGA1-regulated, stress-responsive genes mined from STIFDB2.0 using Genevestigator revealed prominent clusters of cold and drought responsive genes (Figure <a href="#F2">2</a>), which were subjected to further analysis by biclustering using the same software. While hierarchical clustering helps in grouping genes with similar profiles across all abiotic stress conditions, Biclustering identifies groups of genes that exhibit similarity only in a subset of conditions such as cold or drought, irrespective of their expression profiles in other conditions. The regulation of genes identified as highly differentially regulated by biclustering in both cold and drought conditions was compared with the fold-change values obtained on our microarray (Figures <a href="#F3">3</a>, <a href="#F4">4</a>). This revealed that some of the genes follow similar pattern of regulation between the stress response in normal rice plants and the RGA1-response in mutants unexposed to stress. While these may indicate independent regulation, the remaining genes that follow opposite pattern of regulation could be due to the RGA1 mutation, suggesting that RGA1 may mediate the response of these genes to cold or drought stresses. </p> <p>Thirdly, metadata analyses based on data mining at Rice DB using the 2270 genes we identified in the RGA1-transcriptome microarray revealed a much larger number of 1886 stress-related genes as differentially regulated in our RGA1 mutant. A Venn selection of the stress-responsive gene lists identified by all three approaches used in this study viz., gene ontology (94), STIFDB2.0 (106) and Rice DB (1886) revealed that the former two are largely subsets of the 1886 DEGs identified using Rice DB (Figure <a href="#F5">5</a>). Their Venn selections in terms of individual abiotic stress categories and by up/down regulation on our microarray (Figure <a href="#F6">6</a>) revealed 1498 genes as common to all four stresses, with fewer common genes in smaller combinations of stresses (Table <a href="#T2">2</a>). Out of them, 12 of the most up/down-regulated genes have been validated by qRT-PCR (Figure <a href="#F7">7</a>), confirming the broad trends of up/down regulated genes identified on the microarray. These include the well-known stress-responsive genes such as catalase and aquaporin. Among the individual stresses, the sheer number of RGA1-regulated genes that only respond to heat (and no other abiotic stress) is striking, and needs further analysis. Comparative microarray or RT-PCR profiling of the RGA1 mutant and wild type rice plants exposed to various abiotic stresses would reveal more details in this regard. </p> <p>Gene Ontology enrichment of the 1498 RGA1-regulated genes shared by all four abiotic stresses using AGRIGO revealed their molecular functions, cellular localizations, and biological processes (Figure <a href="#F8">8</a>). In terms of processes, genes from the various abiotic stress signaling pathways such as calcium-dependent pathways, ABA dependent or independent pathways, and MAP kinase pathways, as well as various pathways involved in the production of osmoprotectants, heat shock proteins, metallothioneins, antioxidants etc., were found to be differentially regulated in the RGA1 mutant as elaborated below. Together, they clearly indicate the crucial role of G-protein alpha subunit signaling in transducing/mediating the response of rice to multiple abiotic stresses. </p> <div class="calcium-dependentpathwaysingprotein-mediatedabioticstresssignaling" title="sec"> <div class="title" tagx="title" title="title">
Calcium-dependent pathways in G protein-mediated abiotic stress signaling</div> <p>Calcium is a well-known second messenger in abiotic signal transduction and various calcium binding proteins such as calmodulins, calcineurin, CDPKs, and calcineurin B-like interacting protein kinases (CIPK) play an important role in calcium-dependent abiotic stress (Batistič and Kudla, <a href="#B4">2012</a>). The CBL proteins form a complex network with their target kinases CIPKs and regulate target gene expression (Das and Pandey, <a href="#B13">2010</a>). Some genes related to calcium signaling were shown to be involved in stress signaling in rice (Batistič and Kudla, <a href="#B4">2012</a>), and transgenic manipulation of some such genes has been shown to improve stress-tolerance in rice (Campo et al., <a href="#B6">2014</a>). In our study, calcium dependent protein kinase (Os02g0685900) is up regulated while Calmodulin-like protein CaML3 (Os11g0141400) is down regulated in the RGA1 mutant. Their further validation could help determine their potential as candidate genes for development of rice plants tolerant to multiple abiotic stresses. </p> </div> <div class="mapkinasepathwaysing-protein-mediatedabioticstresssignaling" title="sec"> <div class="title" tagx="title" title="title">
Map kinase pathways in G-protein-mediated abiotic stress signaling</div> <p>Many MAPKs have been reported in rice for various abiotic stresses (Danquah et al., <a href="#B12">2014</a>). The MAPK gene OsMSRMK2 is highly induced by a variety of stresses including ABA, JA, SA, drought, and salt but not by cold (Danquah et al., <a href="#B12">2014</a>). In this study, for example, we found MAP kinase (Os03g0285800) to be up regulated. A receptor-like kinase, or <i>O. sativa</i> stress-induced protein kinase gene 1, which is known to be involved in drought and salt stress tolerance is also induced in the RGA1 mutant. </p> </div> <div class="abasignalinging-proteinmediatedabioticstressresponse" title="sec"> <div class="title" tagx="title" title="title">
ABA signaling in G-protein mediated abiotic stress response</div> <p>ABA is involved in the regulation of many aspects of plant growth and development and also is the major hormone that controls plant responses to abiotic stresses (Danquah et al., <a href="#B12">2014</a>), especially drought stress. ABA is also one of the most studied hormones in relation to G-protein signaling (Zhao et al., <a href="#B48">2010</a>). We found a related gene encoding the SNF1-related protein kinase regulatory gamma subunit 1 (AKIN gamma1Os04g0382300) to be suppressed in the RGA1 mutant. Similarly, drought- responsive element binding protein (DREB) is a part of ABA-independent pathway, from which both DREB2 and CRT/DRE binding protein were up regulated in our data. Among the ABA signaling pathway genes, we also found that MYB expression was enhanced in the RGA1 mutants as compared with wild-type plants. Abscisic acid responsive element-binding factors belong to the ABA dependent pathway, of which AREB2 was up regulated in our data. Members of TF families that are involved in both ABA-independent (AP2/ERF and WRKY) and ABA dependent pathways are also involved in stress tolerance (Song et al., <a href="#B40">2012</a>). </p> </div> <div class="transcriptionfactorsandmirnasingprotein-mediatedabioticstressresponse" title="sec"> <div class="title" tagx="title" title="title">
Transcription factors and miRNAs in G protein-mediated abiotic stress response</div> <p>The expression of many stress responsive genes is mediated by transcription factors that bind to specific cis-elements in the promoters of their target genes. We found various transcription factors such as ADH1, OsNAC5, OsWRKY45, bZIP23/72 to be differentially regulated in our RGA1 mutant. Further characterization and validation of the transcription factors identified in our study may reveal their potential as candidate genes to engineer tolerance to various abiotic stresses in rice. At the post-transcriptional level, miRNAs are also known to play important regulatory roles in plant development and stress. miRNAs, such as miR168, miR171, and miR396, are regulated by abiotic stresses such as salinity, drought, and cold in rice (Mal et al., <a href="#B24">2015</a>). So far, no study has reported RGA1-responsive miRNAs involved in stress. In this study, we have mined 63 RGA1-regulated target genes for miRNAs that are also stress responsive. Further validation of their role in stress-response could reveal if they have any potential in crop improvement. </p> </div> <div class="osmoprotectantgenes,leagenes,heatshockproteins,andothers" title="sec"> <div class="title" tagx="title" title="title">
Osmoprotectant genes, lea genes, heat shock proteins, and others</div> <p>Several genes found to be differentially regulated in the RGA1 mutant belong to biosynthetic pathways of osmoprotectants such as polyamine, glycine-betaine, proline, and trehalose. Three genes (Os08g0445700, Os02g0661100, and Os01g0749400) involved in the trehalose synthesis pathway were up regulated in the RGA1 mutant. A major pathway that is significantly down regulated is the betanidin degradation pathway with 21 genes being down regulated. Two genes (Os03g0738400, Os12g0409000) from the glycine betaine synthesis were also up regulated in the RGA1 mutant. These genes are known to be involved in various stress responses such as increased submergence tolerance, drought, and cold resistance (Marco et al., <a href="#B25">2015</a>). </p> <p>Heat shock proteins and molecular chaperone proteins like metallothionein proteins are involved in heat and drought tolerance. Their genes were highly up regulated in the RGA1 mutant with fold changes up to three and validated by qRT-PCR (Figure <a href="#F7">7</a>). Late embryogenesis abundant (LEA protein) genes are known to help in drought and salinity tolerance (Mondini and Pagnotta, <a href="#B28">2015</a>). In our study, Lea14-A was up-regulated in the RGA1 mutant, indicating its potential importance in rice stress. Similarly, among the hormone regulatory genes, we found RGA1-regulation of IPT and ABA hydroxylase, which delay senescence and yield under drought and reduce sterility under cold stress. Oxidative stress related genes such as Glutathione S-transferase and superoxide dismutase genes are involved in salt and cold stress (Marco et al., <a href="#B25">2015</a>). In our study, superoxide dismutase was down regulated in the RGA1 mutant, indicating the important role of G-protein alpha subunit in SOD-mediated regulation of oxidative stress. Genes encoding proton pumps, antiporters, and ion transporters like vacuolar Na+/H+ antiporter and aquaporins are also known to enhance salt and cold tolerance. Our data shows that aquaporins are down regulated 3.36 times in the RGA1 mutant and is validated by qRT-PCR (Figure <a href="#F7">7</a>). </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Overall, our results clearly indicate the potentially crucial role of the G-protein α subunit (RGA1) in regulating the response of the rice plant to multiple abiotic stresses for further experimental validation. The 1886 RGA1-regulated and stress-responsive genes we mined in this study may represent only a subset of overall-stress responsive genes in rice, but they do constitute a G-protein (RGA1)-regulated subset that was never described in any plant so far, except in Arabidopsis elsewhere in this issue (Chakraborty et al., <a href="#B9">2015c</a>). The fact that as many as 1498 RGA1-regulated, stress-responsive genes are common to the four abiotic stresses (drought, salt, heat, cold), and that relatively fewer genes are uniquely regulated by RGA1 in response to individual stresses indicates that RGA1-signaling could be a converging point for the regulation of multiple abiotic stress responses. Its experimental validation, as well as that of the exceptionally large number of 111 unique genes regulated by RGA1 in heat stress (unshared with the other three stresses) could offer glimpses into the commonalities and differences in heat stress signaling vis-à-vis other stresses. </p> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Toshihiro Kumamaru from the Plant Genetic Resources Lab, Faculty of Agriculture, Kyushu University, Fukuoka, Japan, for the seeds of the rice d1 mutant. We also thank Devapriya Choudhury, JNU, for bioinformatic training and Saurabh Raghuvanshi, DU, for discussions on StifDB; Aiyaz and others at Genotypic, for their help in the reanalysis of the microarray data; Sunila and Navjyoti chakraborty for critical proof-reading of the manuscript, and the reviewers for their constructive suggestions. This work was supported by research grant to NR [60(0056)/02/EMRII] and research fellowship to JAP [09/806(013) 2008-EMRI] from the Council of Scientific and Industrial Research (CSIR), Government of India. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <p>The Supplementary Material for this article can be found online at: <a href="http://journal.frontiersin.org/article/10.3389/fpls.2016.00011">http://journal.frontiersin.org/article/10.3389/fpls.2016.00011</a> </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="DataSheet1.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alvarez</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hicks</span> <span class="given-names" tagx="given-names" title="given-names">L. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">ABA-dependent and -independent G-protein signaling in Arabidopsis roots revealed through an iTRAQ </span>. <span class="source" tagx="source" title="source">J. Proteome Res.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">3107</span>– <span class="lpage" tagx="lpage" title="lpage">3122</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/pr2001786">10.1021/pr2001786</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21545083">21545083</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arora</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ray</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">V. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">MADS-box gene family in rice: genome-wide identification, organization and expression profiling during reproductive development and stress </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">242</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-8-242">10.1186/1471-2164-8-242</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17640358">17640358</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashikari</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yano</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sasaki</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshimura</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Rice gibberellin-insensitive dwarf mutant gene Dwarf 1 encodes the alpha-subunit of GTP-binding protein </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">96</span>, <span class="fpage" tagx="fpage" title="fpage">10284</span>– <span class="lpage" tagx="lpage" title="lpage">10289</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.96.18.10284">10.1073/pnas.96.18.10284</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10468600">10468600</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Batistič</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kudla</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of calcium signaling pathways in plants</span>. <span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span> <span class="volume" tagx="volume" title="volume">1820</span>, <span class="fpage" tagx="fpage" title="fpage">1283</span>– <span class="lpage" tagx="lpage" title="lpage">1293</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbagen.2011.10.012">10.1016/j.bbagen.2011.10.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22061997">22061997</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Booker</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burkey</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgan</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fiscus</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Minimal influence of G-protein null mutations on ozone-induced changes in gene expression, foliar injury, gas exchange and peroxidase activity in <i>Arabidopsis thaliana</i> L </span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">668</span>– <span class="lpage" tagx="lpage" title="lpage">681</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3040.2011.02443.x">10.1111/j.1365-3040.2011.02443.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21988569">21988569</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campo</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baldrich</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messeguer</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lalanne</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coca</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">San Segundo</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Overexpression of a calcium-dependent protein kinase confers salt and drought tolerance in Rice by preventing membrane lipid peroxidation </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">165</span>, <span class="fpage" tagx="fpage" title="fpage">688</span>– <span class="lpage" tagx="lpage" title="lpage">704</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.230268">10.1104/pp.113.230268</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24784760">24784760</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakraborty</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanyuka</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pathak</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choudhury</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hooley</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015b</span>). <span class="mixed-article-title" title="mixed-article-title">G-protein α-subunit (GPA1) regulates stress, nitrate and phosphate response, flavonoid biosynthesis, fruit/seed development and substantially shares GCR1 regulation in <i>A. thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">89</span>, <span class="fpage" tagx="fpage" title="fpage">559</span>– <span class="lpage" tagx="lpage" title="lpage">576</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-015-0374-2">10.1007/s11103-015-0374-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26346778">26346778</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakraborty</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanyuka</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pathak</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choudhury</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hooley</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015a</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of Arabidopsis GCR1 mutant reveals its roles in stress, hormones, secondary metabolism and phosphate starvation </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0117819</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0117819">10.1371/journal.pone.0117819</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25668726">25668726</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chakraborty</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghuram</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015c</span>). <span class="mixed-article-title" title="mixed-article-title">G-protein signalling components GCR1 and GPA1 mediate responses to multiple abiotic stresses in Arabidopsis </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="issue" tagx="issue" title="issue">1000</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2015.01000">10.3389/fpls.2015.01000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26635828">26635828</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis and environmental response profiling of SOT family genes in rice ( <i>Oryza sativa</i>) </span>. <span class="source" tagx="source" title="source">Genes Genomics</span> <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">549</span>– <span class="lpage" tagx="lpage" title="lpage">560</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13258-012-0053-5">10.1007/s13258-012-0053-5</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colaneri</span> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tunc-Ozdemir</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Growth attenuation under saline stress is mediated by the heterotrimeric G protein complex </span>. <span class="source" tagx="source" title="source">BMC Plant Biol.</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">129</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-14-129">10.1186/1471-2229-14-129</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24884438">24884438</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danquah</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Zelicourt</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colcombet</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirt</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The role of ABA and MAPK signaling pathways in plant abiotic stress responses</span>. <span class="source" tagx="source" title="source">Biotechnol. Adv.</span> <span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">40</span>– <span class="lpage" tagx="lpage" title="lpage">52</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2013.09.006">10.1016/j.biotechadv.2013.09.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24091291">24091291</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">G. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Expressional analysis and role of calcium regulated kinases in abiotic stress signaling</span>. <span class="source" tagx="source" title="source">Curr. Genomics</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">2</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2174/138920210790217981">10.2174/138920210790217981</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20808518">20808518</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ling</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">agriGO: a GO analysis toolkit for the agricultural community</span>. <span class="source" tagx="source" title="source">Nucl. Acids Res.</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">W64</span>– <span class="lpage" tagx="lpage" title="lpage">W70</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkq310">10.1093/nar/gkq310</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20435677">20435677</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gour</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garg</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joseph</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghuvanshi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Manually curated database of rice proteins</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">D1214</span>– <span class="lpage" tagx="lpage" title="lpage">D1221</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt1072">10.1093/nar/gkt1072</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24214963">24214963</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishikawa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isasaki</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asahi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and characterization of a cDNA for the beta subunit of a G protein from rice </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">228</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.pcp.a028935">10.1093/oxfordjournals.pcp.a028935</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8665097">8665097</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishikawa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsubouchi</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iwasaki</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asahi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and characterization of a cDNA for the alpha subunit of a G protein from rice </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">353</span>– <span class="lpage" tagx="lpage" title="lpage">359</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7767602">7767602</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khurana</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Transcript profiling reveals diverse roles of auxin-responsive genes during reproductive development and abiotic stress in rice </span>. <span class="source" tagx="source" title="source">FEBS J.</span> <span class="volume" tagx="volume" title="volume">276</span>, <span class="fpage" tagx="fpage" title="fpage">3148</span>– <span class="lpage" tagx="lpage" title="lpage">3162</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1742-4658.2009.07033.x">10.1111/j.1742-4658.2009.07033.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19490115">19490115</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nijhawan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arora</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ray</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">F-box proteins in rice. Genome-wide analysis, classification, temporal and spatial gene expression during panicle and seed development, and regulation by light and abiotic stress </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">143</span>, <span class="fpage" tagx="fpage" title="fpage">1467</span>– <span class="lpage" tagx="lpage" title="lpage">1483</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.106.091900">10.1104/pp.106.091900</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17293439">17293439</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">S.-Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramamoorthy</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramachandran</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide survey on genomic variation, expression divergence, and evolution in two contrasting rice genotypes under high salinity stress </span>. <span class="source" tagx="source" title="source">Genome Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">2032</span>– <span class="lpage" tagx="lpage" title="lpage">2050</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evt152">10.1093/gbe/evt152</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24121498">24121498</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamrul Huda</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akhter Banu</span> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trivedi</span> <span class="given-names" tagx="given-names" title="given-names">D. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of plant-type II Ca <sup>2+</sup> ATPases gene family from rice and Arabidopsis: potential role in abiotic stresses </span>. <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">32</span>– <span class="lpage" tagx="lpage" title="lpage">47</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2013.01.002">10.1016/j.plaphy.2013.01.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23416494">23416494</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizutani</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamaki</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumagai</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamiya</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirobe</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of heterotrimeric G protein complexes in rice plasma membrane</span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">320</span>– <span class="lpage" tagx="lpage" title="lpage">331</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2004.02046.x">10.1111/j.1365-313X.2004.02046.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15078334">15078334</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">S. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hwang</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">C. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hwang</span> <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moon</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Comprehensive analysis of the rice RING E3 ligase family reveals their functional diversity in response to abiotic stress </span>. <span class="source" tagx="source" title="source">DNA Res.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">299</span>– <span class="lpage" tagx="lpage" title="lpage">314</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/dst011">10.1093/dnares/dst011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23571674">23571674</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mal</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deb</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aftabuddin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kundu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">A network analysis of miRNA mediated gene regulation of rice: crosstalk among biological processes </span>. <span class="source" tagx="source" title="source">Mol. Biosyst.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">2273</span>– <span class="lpage" tagx="lpage" title="lpage">2280</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C5MB00222B">10.1039/C5MB00222B</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26066638">26066638</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marco</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bitrián</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrasco</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajam</span> <span class="given-names" tagx="given-names" title="given-names">M. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alcázar</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiburcio</span> <span class="given-names" tagx="given-names" title="given-names">A. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic engineering strategies for abiotic stress tolerance in plants</span>, in <span class="source" tagx="source" title="source">Plant Biology and Biotechnology</span>, eds <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahadur</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkat Rajam</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahijram</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krishnamurthy</span> <span class="given-names" tagx="given-names" title="given-names">K. V.</span> </span> </span> ( <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">579</span>– <span class="lpage" tagx="lpage" title="lpage">609</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-81-322-2283-5_29">10.1007/978-81-322-2283-5_29</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuda</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Funabiki</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furukawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Komori</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koike</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tokuji</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis and expression profiling of half-size ABC protein subgroup G in rice in response to abiotic stress and phytohormone treatments </span>. <span class="source" tagx="source" title="source">Mol. Genetics Genomics</span> <span class="volume" tagx="volume" title="volume">287</span>, <span class="fpage" tagx="fpage" title="fpage">819</span>– <span class="lpage" tagx="lpage" title="lpage">835</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-012-0719-3">10.1007/s00438-012-0719-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22996334">22996334</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misra</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkataraman</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sopory</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Heterotrimeric G-protein complex and G-protein-coupled receptor from a legume ( <i>Pisum sativum</i>): role in salinity and heat stress and cross-talk with phospholipase C </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">656</span>– <span class="lpage" tagx="lpage" title="lpage">669</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2007.03169.x">10.1111/j.1365-313X.2007.03169.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17587233">17587233</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mondini</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pagnotta</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Drought and salt stress in cereals</span>. <span class="source" tagx="source" title="source">Sust. Agric. Rev.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-3-319-16988-0_1">10.1007/978-3-319-16988-0_1</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naika</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shameer</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathew</span> <span class="given-names" tagx="given-names" title="given-names">O. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gowda</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sowdhamini</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">STIFDB2: an updated version of plant stress-responsive transcription factor database with additional stress signals, stress-responsive transcription factor binding sites and stress-responsive genes in Arabidopsis and rice </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcs185">10.1093/pcp/pcs185</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23300091">23300091</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narsai</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devenish</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castleden</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narsai</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shou</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Rice DB: an Oryza information portal linking annotation, subcellular location, function, expression, regulation and evolutionary information for rice and Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="volume" tagx="volume" title="volume">76</span>, <span class="fpage" tagx="fpage" title="fpage">1057</span>– <span class="lpage" tagx="lpage" title="lpage">1073</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12357">10.1111/tpj.12357</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24147765">24147765</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nuruzzaman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharoni</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satoh</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shammari</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimizu</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sasaya</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The thioredoxin gene family in rice: genome-wide identification and expression profiling under different biotic and abiotic treatments </span>. <span class="source" tagx="source" title="source">Biochem. Biophys. Res. Commun.</span> <span class="volume" tagx="volume" title="volume">423</span>, <span class="fpage" tagx="fpage" title="fpage">417</span>– <span class="lpage" tagx="lpage" title="lpage">423</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbrc.2012.05.142">10.1016/j.bbrc.2012.05.142</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22683629">22683629</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliveros</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007–2015</span>). <span class="source" tagx="source" title="source">Venny. An Interactive Tool for Comparing Lists with Venn's Diagrams</span>. <a href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a>. </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">G. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Small and large g proteins in biotic and abiotic stress responses in plants</span>, in <span class="source" tagx="source" title="source">Elucidation of Abiotic Stress Signaling in Plants</span>, ed <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">G. K.</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">231</span>– <span class="lpage" tagx="lpage" title="lpage">270</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-1-4939-2211-6_9">10.1007/978-1-4939-2211-6_9</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">R.-S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gookin</span> <span class="given-names" tagx="given-names" title="given-names">T. E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Boolean modeling of transcriptome data reveals novel modes of heterotrimeric G-protein action </span>. <span class="source" tagx="source" title="source">Mol. Syst. Biol.</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">372</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2010.28">10.1038/msb.2010.28</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20531402">20531402</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pathak</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lochab</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">A method for rapid isolation of total RNA of high purity and yield from <i>Arthrospira platensis</i> </span>. <span class="source" tagx="source" title="source">Can. J. Microbiol.</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">578</span>– <span class="lpage" tagx="lpage" title="lpage">584</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1139/W10-045">10.1139/W10-045</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20651857">20651857</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfaffl</span> <span class="given-names" tagx="given-names" title="given-names">M. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">A new mathematical model for relative quantification in real-time RT-PCR</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">e45</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/29.9.e45">10.1093/nar/29.9.e45</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11328886">11328886</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ray</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arora</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapoor</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Expression analysis of calcium-dependent protein kinase gene family during reproductive development and abiotic stress conditions in rice ( <i>Oryza sativa</i> L. ssp. indica) </span>. <span class="source" tagx="source" title="source">Mol. Genet. Genomics</span> <span class="volume" tagx="volume" title="volume">278</span>, <span class="fpage" tagx="fpage" title="fpage">493</span>– <span class="lpage" tagx="lpage" title="lpage">505</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-007-0267-4">10.1007/s00438-007-0267-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17636330">17636330</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ray</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dansana</span> <span class="given-names" tagx="given-names" title="given-names">P. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giri</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deveshwar</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arora</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Modulation of transcription factor and metabolic pathway genes in response to water-deficit stress in rice </span>. <span class="source" tagx="source" title="source">Funct. Integr. Genomics</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">157</span>– <span class="lpage" tagx="lpage" title="lpage">178</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-010-0187-y">10.1007/s10142-010-0187-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20821243">20821243</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giri</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapoor</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">G. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Protein phosphatase complement in rice: genome-wide identification and transcriptional analysis under abiotic stress conditions and reproductive development </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">435</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-11-435">10.1186/1471-2164-11-435</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20637108">20637108</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rocha</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Expression analysis of abiotic stress-responsive genes in two rice heterotic crosses under cold, heat and drought stresses </span>. <span class="source" tagx="source" title="source">Plant Breeding</span> <span class="volume" tagx="volume" title="volume">131</span>, <span class="fpage" tagx="fpage" title="fpage">267</span>– <span class="lpage" tagx="lpage" title="lpage">275</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1439-0523.2011.01928.x">10.1111/j.1439-0523.2011.01928.x</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">D.-W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">H.-W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">D.-T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification of the class III aminotransferase gene family in rice and expression analysis under abiotic stress </span>. <span class="source" tagx="source" title="source">Genes Genomics</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">597</span>– <span class="lpage" tagx="lpage" title="lpage">608</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13258-013-0108-2">10.1007/s13258-013-0108-2</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urano</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J.-G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Botella</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Heterotrimeric G protein signalling in the plant kingdom</span>. <span class="source" tagx="source" title="source">Open Biol.</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">120186</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rsob.120186">10.1098/rsob.120186</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23536550">23536550</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urano</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colaneri</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Gα modulates salt-induced cellular senescence and cell division in rice and maize</span>. <span class="source" tagx="source" title="source">J. Exp. Bot.</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">6553</span>– <span class="lpage" tagx="lpage" title="lpage">6561</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/eru372">10.1093/jxb/eru372</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25227951">25227951</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venu</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sreerekha</span> <span class="given-names" tagx="given-names" title="given-names">M. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheshu Madhav</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nobuta</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohan</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Deep transcriptome sequencing reveals the expression of key functional and regulatory genes involved in the abiotic stress signaling pathways in rice </span>. <span class="source" tagx="source" title="source">J. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">216</span>– <span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12374-013-0075-9">10.1007/s12374-013-0075-9</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide temporal-spatial gene expression profiling of drought responsiveness in rice </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">149</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-149">10.1186/1471-2164-12-149</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21406116">21406116</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">D. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Islam</span> <span class="given-names" tagx="given-names" title="given-names">S. M. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Rice heterotrimeric G-protein gamma subunits (RGG1 and RGG2) are differentially regulated under abiotic stress </span>. <span class="source" tagx="source" title="source">Plant Signal. Behav.</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">733</span>– <span class="lpage" tagx="lpage" title="lpage">740</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.20356">10.4161/psb.20356</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22751322">22751322</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">D. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Rice heterotrimeric G-protein alpha subunit (RGA1): <i>in silico</i> analysis of the gene and promoter and its up regulation under abiotic stress </span>. <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="volume" tagx="volume" title="volume">63</span>, <span class="fpage" tagx="fpage" title="fpage">262</span>– <span class="lpage" tagx="lpage" title="lpage">271</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2012.11.031">10.1016/j.plaphy.2012.11.031</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23313793">23313793</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanley</span> <span class="given-names" tagx="given-names" title="given-names">B. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Assmann</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">ABA-regulated G-protein signaling in Arabidopsis guard cells: a <i>proteomic</i> perspective </span>. <span class="source" tagx="source" title="source">J. Proteome Res.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">1637</span>– <span class="lpage" tagx="lpage" title="lpage">1647</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/pr901011h">10.1021/pr901011h</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20166762">20166762</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmermann</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch-Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hennig</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gruissem</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">GENEVESTIGATOR. Arabidopsis microarray database and analysis toolbox</span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">136</span>, <span class="fpage" tagx="fpage" title="fpage">2621</span>– <span class="lpage" tagx="lpage" title="lpage">2632</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.104.046367">10.1104/pp.104.046367</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15375207">15375207</a> </span> </span> </li> </ul> </div> </div>  </body></html>