<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4754947">4754947</a> </span> <span class="pii" title="pii">srep21625</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/srep21625">10.1038/srep21625</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genetic dissection of plant architecture and yield-related traits in <i>Brassica napus</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">Guangqin</span> </span> <a href="#a1">1</a> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Qingyong</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Hao</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Qian</span> </span> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Chunyu</span> </span> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">Chuchuan</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Yongming</span> </span> <a href="#c1">a</a> <a href="#a1">1</a> <a href="#a2">2</a> </span> <span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Key Laboratory of Rapeseed Genetics and Breeding of Agriculture Ministry of China, Huazhong Agricultural University </span> <span class="country" tagx="country" title="country">China</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">a</span> <span class="email" tagx="email" title="email">ymzhou@mail.hzau.edu.cn</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-02-02</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">21625</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-08-25</span> <span class="accepted" title="accepted">accepted: 2016-01-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Macmillan Publishers Limited</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>An optimized plant architecture (PA) is fundamental for high-yield breeding but the genetic control of the important trait is largely unknown in rapeseed. Here plant architecture factors (PAFs) were proposed to consist of main inflorescence length proportion (MILP), branch height proportion (BHP), and branch segment proportion (BSP). Comparison of different genotypes in a DH population grown in diverse environments showed that an optimized PAF performance with MILP and BHP between 0.3–0.4 was important for high yield potential. In total, 163 unique quantitative trait loci (QTLs) for PA- and plant yield (PY)-related traits were mapped onto a high-density genetic map. Furthermore, 190 PA-related candidate genes for 91 unique PA QTLs and 2350 PY epistatic interaction loci-pairs were identified, which explain 2.8–51.8% and 5.2–23.6% of phenotypic variation, respectively. Three gene categories, transcription factor, auxin/IAA, and gibberellin, comprise the largest proportions of candidate genes for PA-related QTLs. The effectiveness of QTL candidate genes prediction was demonstrated by cloning of three candidate genes, <i>Bna.A02.CLV2</i>, <i>Bna.A09.SLY2</i>, and <i>Bna.C07.AHK4</i>. The study thus outlines a gene network for control of PA-related traits and provides novel information for understanding the establishment of ideal PA and for developing effective breeding strategies for yield improvement in rapeseed and other crops. </p> </div> </div> </div> <div class="body" title="body"> <p>Plant architecture (PA) refers to the morphological characteristics and spatial arrangement of the plant corpus and is perhaps the most visible change that occurs during crop domestication. PA modifications have shown significant impacts on crop adaptation and yield potential <a href="#b1">1</a> <a href="#b2">2</a>. A well-known example is the introduction of dwarf genes into wheat and rice, which resulted in reduced plant height and modified PA and, thus, greatly enhanced seed yield <a href="#b3">3</a> <a href="#b4">4</a>. PA is also one of the most important and direct characteristics to distinguish various species. Therefore, a comprehensive understanding of the genetic control of PA in crops can not only provide insights into crop domestication history, morphogenesis and development, but also lay a foundation for designing efficient breeding strategies to improve crop yield <a href="#b5">5</a> <a href="#b6">6</a> <a href="#b7">7</a>. </p> <p>An optimal PA can improve leaf area index, photosynthetic efficiency, fertilizer tolerability, and harvest index, thus leading to increased seed yield. Therefore, breeding for an optimized PA has drawn great attention in major crops. For instance, improvements of maize PA, especially leaf angle, have significantly contributed to increase yield <a href="#b8">8</a> <a href="#b9">9</a> <a href="#b10">10</a>. An ideal plant architecture (IPA) in rice has been proposed to reduce unproductive tillers, and to yield thicker and more sturdy stems and more grains per panicle than current rice varieties <a href="#b6">6</a>. Various parameters used to quantitatively describe the suitable PA or IPA in different crops, but some common characteristics are often considered in breeding practices, such as (semi-)dwarfed and compact plant, rigid stem, and up-right and dark green leaves <a href="#b2">2</a>. The majority of PA-related traits are quantitative traits that are affected by environmental factors and growth conditions, such as light, temperature, nutrition, and plant density. The genetic control of PA is complicated as the final establishment of a PA in a given species involves a series of developmental processes, such as the initiation and development of shoot apical meristem (SAM) and axillary meristem (AM; tillers, branches), stem and internode elongation, and the differentiation of floral organs in inflorescences <a href="#b1">1</a> <a href="#b2">2</a>. </p> <p>Over the last two decades, a number of genes involved in PA formation have been identified through reverse genetics and map-based cloning, which can be categorized into a wide array of biological pathways. Strikingly, many of these genes are involved in hormone biosynthesis and regulation. Genes controlling plant height are related to gibberellin (GA) metabolism and signal transduction <a href="#b4">4</a> <a href="#b11">11</a>. Several genes that work as a master switch of SAM initiation and development, are related to auxin, indole acetic acid (Aux/IAA) biosynthesis and signal transduction <a href="#b12">12</a> <a href="#b13">13</a> <a href="#b14">14</a>. More recently, genes for cytokinin (CK) metabolic regulation and transport, brassinosteroid (BR) biosynthesis and signal transduction, and strigolactone (SL) metabolism regulation and signal transduction have been demonstrated to be important in various PA-related traits, such as <i>CKX</i> <a href="#b15">15</a>, <i>KN1</i> <a href="#b16">16</a>, <i>BKI1</i> <a href="#b17">17</a>, <i>BRI1</i> <a href="#b18">18</a>, <i>D14</i> <a href="#b19">19</a>, <i>D27</i> <a href="#b20">20</a>. Moreover, several transcription factors (TFs) and cell cycle (CC) genes affect PA establishment and maintenance <a href="#b5">5</a> <a href="#b21">21</a> <a href="#b22">22</a> <a href="#b23">23</a>. Although a large number of genes related to PA regulation and development have been identified in different species, our understanding of crop morphogenesis and developmental regulation is still limited. The lack of mutants linked to all aspects of plant morphogenesis is a bottleneck that must be relieved to fully define the network <a href="#b2">2</a> <a href="#b24">24</a>. </p> <p>Rapeseed ( <i>Brassica napus</i>, AACC, 2n = 4x = 38) is one of the three major oil crops in the world, and it provides edible oil and raw materials for bio-energy applications. <i>B. napus</i> was formed approximately 7,500 years ago through the natural hybridization of its two progenitor diploids, <i>B. rapa</i> (AA, 2n = 2x = 20) and <i>B. oleracea</i> (CC, 2n = 2x = 18) <a href="#b25">25</a> <a href="#b26">26</a> <a href="#b27">27</a>. These three species and Arabidopsis are thought to share a common ancestor <a href="#b25">25</a> <a href="#b27">27</a> <a href="#b28">28</a>. <i>B. napus</i>, a relatively young species in terms of its evolutionary age, has a short history of artificial domestication (Approximately 400–500 years) <a href="#b27">27</a> <a href="#b29">29</a> <a href="#b30">30</a>. To date, few studies on rapeseed PA have been reported, which described the composition <a href="#b31">31</a>, properties of PA characters <a href="#b32">32</a>, and genetic model and combining abilities of some PA-related traits <a href="#b33">33</a> <a href="#b34">34</a>. More recently, two genes related to plant height, <i>dwf2</i> in <i>B. rapa</i> <a href="#b35">35</a> and <i>Bnrga-ds</i> in <i>B. napus</i> <a href="#b36">36</a>, were identified as <i>AtRGA</i> homologs. However, the genetic control of PA in this important crop is largely unknown. </p> <p>The current study was aimed to systematically dissect the genetic architecture of PA in rapeseed using a doubled haploid (DH) population, which was derived from two parents with different PA structures genotyped using <i>B. napus</i> 6 K SNP (single nucleotide polymorphism) arrays. The parameters called plant architecture factors (PAFs) in rapeseed were introduced to dissect complex PA traits into simple components that are more stable across environments and genotypes. Quantitative trait loci (QTLs) and candidate genes involved in the genetic control of PA and their interactions with plant yield (PY) were identified. Our results provide novel information about the genetic architecture of PA in <i>B. napus</i>, which is important for improving yield by manipulating PA-related traits. </p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="quantitationofplantarchitecturewithplantarchitecturefactorsderivedfromprimarymorphologicaltraits" title="sec"> <div class="title" tagx="title" title="title">
Quantitation of plant architecture with plant architecture factors derived from primary morphological traits </div> <p>To quantify variations in PA-traits, we divided a plant into three segments ( <a href="#f1">Fig. 1</a>): main inflorescence length (MIL), branch height (BH), and branch segment (BS). These three segment measurements were simply divided by plant height (PH) so as to obtain three proportion values: MIL proportion (MILP), BH proportion (BHP) and BS proportion (BSP). These three proportion parameters were termed plant architecture factors (PAFs) and used them to describe the PA components in the following analyses ( <a href="#f1">Fig. 1</a>). Similarly, a series of secondary parameters were obtained based on the measurements of primary morphological traits, including branch average length (BAL = BTL/BN), total number of plant silique (PSN = MISN + BSN), PY (MIY + BY), MISN proportion (MISN/PSN), and MIY proportion (MIY/PY). Together, there are 12 primary traits (from direct measurement) and 8 secondary traits (calculated from primary traits), which can be grouped into 10 PA-related traits and 10 PY-related traits. </p> <p>The MIL, MILP, BN, BAL, SSN, BY, MIY, PY, and TSW between the two parents (Hua_5 and J7005) were significantly different in all environments examined ( <a href="#S1">Supplementary Table S1</a>). Furthermore, majority of 20 traits showed continuous distributions in the DH population in four environments ( <a href="#S1">Supplementary Table S1</a>, <a href="#S1">Figs S1–S4</a>). All of the traits were significantly different in terms of genotype (G), environment (E), and genotype by environment (G × E) based on ANOVA results ( <a href="#S1">Supplementary Table S2</a>). The broad-sense heritabilities ( <i>h</i> <sup> <i>2</i> </sup>) of all traits varied from 50–92.8%, with higher percentages associated with PA-related traits compared to PY-related traits ( <a href="#S1">Supplementary Table S2</a>). The correlations between PY- and PA-related traits based on different environments were highly significant ( <a href="#S1">Supplementary Table S3</a>). </p> </div> <div class="optimizedplantarchitecturefactorsareimportantcomponentsforhighyieldpotential" title="sec"> <div class="title" tagx="title" title="title">
Optimized plant architecture factors are important components for high yield potential</div> <p>To evaluate the effects of PAFs on PY and PY-related traits, the DH lines with significant differences in PY were grouped into two pools (i.e., high-yield and low-yield pools) in each environment ( <a href="#t1">Table 1</a>). The average performances of the other traits were compared between these two pools. The high-yield pools had similar PY levels as Hua_5, a high-yielding control cultivar grown under the same conditions. However, the yield components, i.e., PSN, SSN and TSW varied significantly. The high-yield pools in all or most of the environments had higher PSN and SSN but similar TSW compared to those in the low-yield pools, while Hua_5 exhibited a similar yield level to the high-yield pool but with more SSN and heavier TSW ( <a href="#t1">Table 1</a>). The results suggested that high-yield genotypes may have different combinations of yield components, which is consistent with previous studies <a href="#b37">37</a> <a href="#b38">38</a>. However, the MILPs of the high-yield pools and Hua_5 were stable, with a range of 0.35–0.4 in all winter-type rapeseed environments, while the MILPs of the low-yield pools were higher than 0.4 ( <a href="#t1">Table 1</a>). Similarly, the BHPs of high-yield pools were between 0.32–0.38 across environments, while the BHPs of the low-yield pools were higher than 0.4 or lower than 0.3 ( <a href="#t1">Table 1</a>). </p> <p>The population and control in spring-growing environment (GS11) performed differently due to dramatic differences in ecological and farming conditions and had a relatively low yield level compared to the same genotypes in winter-type rapeseed environments, suggesting that the DH population and Hua_5 are not optimized to be spring-growth ecotypes. Accordingly, the PA parameters did not distribute into the optimal norm as they did in winter-growing conditions, although we observed similar trends in the high- and low-yield pools ( <a href="#t1">Table 1</a>). </p> <p>Together, these results indicate that the MILP and BHP in different high-yield genotypes should be at an appropriate range to reach a certain yield level. Compared the three PY components (PSN, SSN, and TSW), PAFs may function as a stable and easy-to-use measurement in selecting high-yield genotypes. </p> <p>To verify the above results, we further analyzed the performance of three high-yield cultivars used as check (CK) varieties in the national rapeseed regional trial in China during 2006–2010, which was conducted for winter-type rapeseed across major growth regions <a href="#b37">37</a>. All cultivars were high-yield and widely grown in large scales in the corresponding ecological areas ( <a href="#t2">Table 2</a>). The PY components varied greatly from year to year in terms of growth density, but the PAFs were much more consistent ( <a href="#t2">Table 2</a>). For example, PSN and BN ranged from 222.7 to 483.5 and 5.9 to 9.6, respectively, while all MILPs were between 0.3–0.4 in different regions, years and growth densities ( <a href="#t2">Table 2</a>). The results further indicated that optimized PAF performance was important to achieve high yield potential. </p> </div> <div class="pa-relatedqtlsaremorestablethanpy-relatedqtls" title="sec"> <div class="title" tagx="title" title="title">
PA-related QTLs are more stable than PY-related QTLs</div> <p>A total of 346 QTLs for 20 examined traits were identified in four environments. These QTLs were evenly mapped in the A and C rapeseed genomes ( <a href="#t3">Table 3</a>, <a href="#f2">Fig. 2</a>, and <a href="#S1">Supplementary Table S4</a>) and could explain 2.8–51.8% (10.3% per QTL on average) of the phenotypic variation ( <i>R</i> <sup> <i>2</i> </sup>), with an average confidence interval of 4.19 cM. Among 167 PY-related trait QTLs identified, 99 (59.3%) exhibited additive effects from Hua_5, and 68 (40.7%) exhibited additive effects from J7005 ( <a href="#t4">Table 4</a>). Among the 179 PA-related trait QTLs identified, 73 (40.8%) exhibited additive effects from Hua_5, and 106 (59.2%) exhibited additive effects from J7005 ( <a href="#t4">Table 4</a>). Such distributions were consistent with the phenotypic performances of the two parental lines, i.e., Hua_5 being a high-yield cultivar and J7005 being a cultivar with special PA component traits ( <a href="#f1">Fig. 1 and</a> <a href="#S1">Supplementary Table S1</a>). </p> <p>Compared with PY-related trait QTLs, the log likelihood of the odds (LOD) scores of the PA-related trait QTLs (LOD = 7.45, <i>R</i> <sup> <i>2</i> </sup> = 10.3% on average) were higher than those of PY-related traits (LOD = 6.48, <i>R</i> <sup> <i>2</i> </sup> = 10.3% on average). Although the majority of the above QTLs showed minor effects on phenotypic variations with <i>R</i> <sup> <i>2</i> </sup> &amp;lt; 10% ( <a href="#S1">Supplementary Table S4</a>), 47 QTLs had <i>R</i> <sup> <i>2</i> </sup> &amp;gt; 20% in one environment (26 QTLs) or &amp;gt; 10% in any two environments (21 identified QTLs). We referred to such QTLs as major QTLs, and they accounted for 13.6% of all identified QTLs. There were 24 PA-related trait major QTLs (LOD = 19.39, <i>R</i> <sup> <i>2</i> </sup> = 27.4% on average) and 23 PY-related trait major QTLs (LOD = 11.57, <i>R</i> <sup> <i>2</i> </sup> = 20.1% on average). We identified a major QTL for the PA-related trait MIL, with an <i>R</i> <sup> <i>2</i> </sup> of up to 51.8% (LOD = 27.4) ( <a href="#S1">Supplementary Table S4</a>). We also repeatedly identified 41 QTLs in two or more environments, 22 of which were PA-related trait QTLs with an average LOD = 7.98 and <i>R</i> <sup> <i>2</i> </sup> = 10.0%, and 19 of which were PY-related trait QTLs with an average LOD = 6.83 and <i>R</i> <sup> <i>2</i> </sup> = 10.5%. The above results indicated that PA-related trait QTLs may be more stable than PY-related trait QTLs. </p> <p>The PAFs (BHP, MILP, and BSP) are secondary traits derived from the four primary traits PH, BH, MIL, and BS. Notably, there were 42.9% (9/21, BHP), 66.7% (14/21, MILP), and 86.7% (13/15, BSP) QTLs for secondary traits belonging to new QTLs not identified by the four primary traits ( <a href="#S1">Supplementary Table S4</a>). Moreover, 2 of the newly identified QTLs were major QTLs ( <i>R</i> <sup> <i>2</i> </sup> &amp;gt; 20%), and the <i>R</i> <sup> <i>2</i> </sup> values of 9 QTLs were &amp;gt; 10%. These results suggested that PAFs as PA measurements may be less affected by environment, and more suitable for the identification of PA-related QTLs. </p> </div> <div class="uniquepa-relatedqtlshavepleiotropiceffectsonyieldandyieldcomponents" title="sec"> <div class="title" tagx="title" title="title">
Unique PA-related QTLs have pleiotropic effects on yield and yield components</div> <p>The above identified QTLs were integrated into unique QTLs with a two-round strategy by QTL meta-analysis. First, 301 consensus QTLs (same traits in different environments) were generated by merging 346 identified QTLs. Second, the overlapping consensus QTLs of different traits were pooled into 163 unique QTLs ( <a href="#t3">Table 3</a> and <a href="#S1">Supplementary S5</a>). By doing so, the average confidence interval dropped to 3.19 cM. Among the unique QTLs, 69 ( <a href="#f2">Fig. 2</a>, blue sites on innermost circle; <a href="#S1">Supplementary Table S5</a>), 58 (red sites on innermost circle) and 36 (green sites on innermost circle) were related to PA-traits, PY-traits, or both, respectively. Moreover, 94 (57.7%) unique QTLs were responsible for a single trait, with 51 related to PA traits and 43 related to PY traits. The remaining 69 unique QTLs (42.3%) were pleiotropic, affecting two or more traits, with 18 (26.1%) and 15 (21.7%) related to PA- and PY-traits, respectively, and 36 (52.2%) related to both ( <a href="#S1">Table S5</a>). These 36 pleiotropic QTLs simultaneously controlled PA- and PY-related traits, which corresponded to 145 identified QTLs with 68 PA-related QTLs (LOD = 7.82, <i>R</i> <sup> <i>2</i> </sup> = 10.6% on average) and 77 PY-related QTLs (LOD = 6.54, <i>R</i> <sup> <i>2</i> </sup> = 10.3% on average). The results clearly indicate that PA can affect PY-related traits to a large extent. Moreover, the effect of PA-related traits on PY was similar to the effect of PY components (PSN, SSN, and PY) on PY. </p> <p>To estimate the influence and effect of the PAFs on PY, pleiotropic QTLs of PAFs, PY, and PY components traits were selected to analyze the relationship and effects between PAFs and PY. Four pleiotropic QTLs simultaneously controlled BHP and MILP traits ( <a href="#S1">Supplementary Table S5</a>), two of which were major QTLs and were repeatedly identified as QTLs for both BHP and MILP. Interestingly, the additive (A) effects of these four BHP QTLs were opposite of those of MILP. Meanwhile, three pleiotropic QTLs simultaneously controlled PAFs and PY, one of which was a major QTL that controlled PY ( <i>R</i> <sup> <i>2</i> </sup> = 21.9% in WH09, and <i>R</i> <sup> <i>2</i> </sup> = 14.6% in GS11). Similarly, 8 pleiotropic QTLs controlled both PAFs and PY components. These results clearly suggest that plants with optimized PAFs have better yield potential at the QTL level. </p> </div> <div class="identificationofcandidategenesforpa-relatedqtls" title="sec"> <div class="title" tagx="title" title="title">
Identification of candidate genes for PA-related QTLs</div> <p>To identify candidate genes for unique QTLs, we collected 256 genes that are involved in PA development and regulation in Arabidopsis and other species ( <a href="#t5">Table 5</a> and <a href="#S1">Supplementary S6</a>). These genes can arbitrarily be categorized into 8 groups based on their assigned functions and metabolic pathways, namely, Auxin/IAA- (40), CK- (22), GA- (42), BR- (25), and SL- (22) regulated pathways, TFs (37), CC genes (26), and other types of genes (42) ( <a href="#t5">Table 5</a> and <a href="#S1">Supplementary Table S6</a>). A total of 1080 homologous genes were identified in the <i>B. napus</i> genome. However, there were 40 plant architecture genes that could not be identified in <i>B. napus</i> genome ( <a href="#S1">Supplementary Table S6</a>). </p> <p>These homologous genes were mapped onto the <i>B. napus</i> genetic linkage map according to the relationship between the <i>B. napus</i> genetic map and the physical map <a href="#b39">39</a>. After the exclusion of homologous genes located on scaffolds and unplaced sequences in the reference genome, 826 homologous genes (76.5%) were anchored into the <i>B. napus</i> genetic map ( <a href="#t5">Table 5</a>, <a href="#f3">Fig. 3</a>, and <a href="#S1">Supplementary Table S8</a>). These genes were evenly distributed across all <i>B. napus</i> linkage groups ( <a href="#t5">Table 5</a>, <a href="#f3">Fig. 3</a>, and <a href="#S1">Supplementary Table S8</a>). </p> <p>Next, we predicted the candidate genes of 105 unique QTLs, except for 58 PY-related QTLs (69 unique QTLs only related to PA; 36 unique QTLs were pleiotropic QTLs related to both PA and PY). In total, 190 homologous genes were located in or near the confidence intervals of 91 unique QTLs ( <a href="#f3">Fig. 3</a> and <a href="#S1">Supplementary Table S9</a>), which were considered as candidate genes. Among the 91 unique QTLs with candidate genes, 61 were related to PA only, and the remaining 30 were pleiotropic QTLs. We did not identify any homologous genes near the remaining 14 unique QTLs, and considered these loci unknown or PA-specific loci in <i>B. napus</i> ( <a href="#S1">Supplementary Table S9</a>). </p> <p>To further verify the effectiveness and accuracy of this method in the identification of QTL candidate genes, three QTLs were chosen according to the importance and effectiveness to clone the candidate genes. DNA fragments containing three QTLs candidate genes were cloned in two parents (Hua_5 and J7005; <a href="#S1">Supplementary Figs S5–S7</a>): <i>CLV2</i> for <i>quPAY.A02_24.39</i> (ID_15 in <a href="#S1">Table S9</a>), <i>SLY2</i> for <i>quBSP.A09_26.7</i> (ID_77 in <a href="#S1">Table S9</a>), and <i>AHK4</i> for <i>quPAY.C07_21.9</i> (ID_139 in <a href="#S1">Table S9</a>). The cloned DNA sequences of the candidate genes were aligned against the <i>B. napus</i> reference genomic and coding sequences (CDS. Darmor- <i>bzh</i> <a href="#b27">27</a>), as well as the sequences of their respective Arabidopsis homologous genes ( <a href="#S1">Supplementary Figs S5–S7</a>). There were numbers of SNPs in the two parents in all three cloned genes at genomic DNA levels ( <a href="#S1">Supplementary Figs S5–S7</a>). At amino acid (aa) level, Bna.A02.CLV2 in J7005 and Bna.C07.AHK4 in Hua_5 harbored a precocious termination at the 382 aa residues and 383 aa residues, respectively ( <a href="#S1">Supplementary Figs S5,S7</a>), while four aa mutations for Bna.A09.SLY2 were identified between the two parents ( <a href="#S1">Supplementary Fig. S6</a>). The above results indicated that the candidate gene sequences between two parents had obvious differences, providing further evidence that the method for predication of the QTL candidate genes is effective and veracious. </p> <p>Because of the high homoeology between the A and C genomes of <i>B. napus</i>, some QTLs identified in A genome of <i>B. napus</i> also had corresponding orthologous QTLs in C genome (vice versa), and these QTLs also identified same candidate genes. For instances, a QTL for SSN was identified in A03 (identified QTL ID_103; unique QTL ID_30), and a corresponding orthologous QTL for SSN was also identified in the corresponding orthologous region of C03 (identified QTL ID_151; unique QTL ID_108). These two QTLs identified the same candidate genes <i>PsLP</i> and <i>LeSP</i> ( <a href="#S1">Table S9</a>). Similarly, two QTLs for AHP in A04 and C04 (identified QTL ID_110, 305; unique QTL ID_42, 68) also had two corresponding orthologous QTLs in C04 and C07 (identified QTL ID_60, 59, 332; unique QTL ID_118, 137). However, some of QTLs did not identify corresponding orthologous counterparts in the two subgenomes. For instance, <i>qPH.A01</i> (ID_1), <i>qMILP.A02</i> (ID_17), <i>qBS.C04</i> (ID_241), <i>qTSW.C05</i> (ID_65) were only identified in A or C genome of <i>B. napus</i>. Above results indicated that the A- and C-subgenomes of <i>B. napus</i> may experience the selections at both same and different direction during the process of the parallel evolution. </p> <p>To understand how pleiotropic QTLs may influence PY through PA, we examined 3 pleiotropic major QTLs for PY ( <a href="#S1">Supplementary Table S10</a>). There were 8 candidate genes for the 3 pleiotropic QTLs, which have been previously identified to be involved in regulating PA or PA-related traits. For instance, <i>AtCUC2</i> and <i>PhNAM</i> influence embryo, flora, and SAM development. High <i>CUC2</i> transcript levels have been reported to increase floral organ number and to shorten internodes <a href="#b40">40</a> <a href="#b41">41</a> <a href="#b42">42</a>; The <i>sp</i> tomato mutant disrupted the regularity of alternating vegetative and reproductive phases, and resulted in the characteristic scorpioid architecture of tomato inflorescence <a href="#b43">43</a>; The <i>PsLF</i> genes determines the flowering node in pea and also affects inflorescence structure <a href="#b44">44</a>. The above results indicate that the effects of loci on PY may largely be through the regulation of PA-related traits; therefore they exhibit pleiotropy on both PA- and PY-related traits. </p> </div> <div class="epistaticlocipairsforpa-andpy-relatedtraitsareequallyimportanttoplantyield" title="sec"> <div class="title" tagx="title" title="title">
Epistatic loci pairs for PA- and PY-related traits are equally important to plant yield </div> <p>To explore the influence of PA on PY, we analyzed all possible two-locus interactions between PA components and PY in the whole genome using a two-way ANOVA procedure <a href="#b45">45</a>. Because only two genotypes (A and B) exist in each locus of a DH population, we could only identify the interactions of additive effects of two different loci (AA). In total, 2350 interaction loci-pairs were identified in the four environments, with 690 for WH09, 746 for WH10, 311 for HG10, and 603 for GS11 ( <a href="#S1">Supplementary Table S11</a>). These interactions can explain 5.18–23.58% PY variation variances (Var; 8.65% on average). Furthermore, PY was affected not only by relevant QTLs but also by a large number of epistatic interaction loci-pairs. We identified 92% (2167) of the total interaction loci-pairs (2350) in a specific environment, and 183 interaction loci-pairs existed in two or more environments, with only 3 interaction loci-pairs detected in three environments. The results indicate that the interaction loci-pairs with PY were highly influenced by environment. </p> <p>To verify the authenticity and effectiveness of the interaction loci-pairs, eight interaction loci-pairs were randomly selected for multiple comparisons between two-locus genotypic combinations of PY ( <a href="#f4">Fig. 4</a>). The average PY values of the 4 different combinations were significantly different ( <a href="#f4">Fig. 4</a>), suggesting that the interaction loci-pairs indeed existed. </p> <p>To understand the relationship between the interaction loci-pairs and the identified QTLs, we aligned the positions of interaction loci-pairs and mapped QTLs. Approximately half of the interaction loci-pairs, including 342 (49.6%, WH09), 440 (59.0%, WH10), 160 (51.4%, HG10), and 261 (43.3%, GS11), were located within identified QTLs confidence intervals, or overlapped with the intervals ( <a href="#f5">Fig. 5</a> and <a href="#S1">Supplementary Table S11</a>), which contained 69 (95.8%), 81 (88.0%), 84 (98.8%), and 95 (97.9%) different QTLs, respectively, covering almost more than 95% QTLs (329/346) in the entire genome. Of these identified QTLs, 38.5% (463/1203) were affected by PA-related traits (average Var = 8.65) ( <a href="#f5">Fig. 5</a>), 37.7% (453/1203) were affected by PY-related traits (average Var = 8.93), and 23.9% (287/1203) were affected by both (average Var = 8.83). </p> <p>We further analyzed the effects of the interaction loci-pairs corresponding PAFs (MILP, BHP, and BSP) and PY components (PSN, SSN, and TSW) on PY. There were 290 and 363 interaction loci-pairs corresponding PAFs (average Var = 8.53) and PY components traits (average Var = 8.74). These findings further illustrated that PA has a significant influence on PY (accounting for 1/3 of the total effect), and the number and effects of PA-related traits were similar to those of PY-related traits on PY. The effects of the PY interaction loci-pairs corresponding PAFs were equally important to those of the PY components. Interestingly, the interaction loci-pairs were not directly related to QTL effect, and the interaction effect (Var) was also independent from the QTL effect. </p> <p>For the 183 interaction loci-pairs identified repeatedly in two or more environments, 97 were located in or overlapped with the confidence intervals of 100 identified QTLs ( <a href="#S1">Supplementary Table S12</a>). Of these loci-pairs, 33, 44, and 20 were affected by PA-related traits (average Var = 9.25), PY-related traits (average Var = 8.80), or both (average Var = 9.24), respectively. These results indicate that the contribution of PA-related traits to PY can be more stably identified because the traits are less influenced by the environment. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The improvements of PY in major crops over the last decades have primarily resulted from enhanced photosynthesis efficiency and increased energy conversion efficiency <a href="#b1">1</a> <a href="#b2">2</a> <a href="#b46">46</a>, which heavily rely on the continuous optimization of PA-related traits <a href="#b5">5</a> <a href="#b6">6</a>. Therefore, a better understanding of PA development and genetic regulation has a fundamental influence on the efficiency of high-yield improvement. </p> <p>As a very complicated trait, rapeseed yield is not only related to the yield-component traits (PSN, SSN, and TSW) <a href="#b47">47</a>, but also influenced by various abiotic and biotic stresses <a href="#b48">48</a>, flowering time <a href="#b49">49</a>, seed germination and seedling vigor <a href="#b50">50</a>. Although it has been recognized for long time that PA is one of the most important factors on crop adaptation and yield potential <a href="#b1">1</a> <a href="#b2">2</a>, there is a lack of effective measurements in rapeseed that has a relatively loose plant architecture with complex spatial structure. In this study, we proposed the concept of "PAFs" and investigated the relationship between PA and yield. Our results demonstrated that PY is significantly correlated with PA-related traits ( <a href="#S1">Supplementary Table S3</a>). It was showed that PAFs have a direct influence on PY and plot yield by analyzing data from high-yield and low-yield pools in the DH population, which was further verified with data from the China national rapeseed regional trial of winter-type rapeseed area. </p> <p>Our results and previous studies showed that the three PY components vary from one material to another, which can result in a number of combinations of traits that achieve similar yield levels. This finding imposes a huge challenge on yield breeding for selection of high-yield genotypes. It is important to identify a more universal measurement that can directly reflect yield level of a given crop rather than evaluation of all of a crop’s component traits for seed yield. In our case, PAF seems to be a useful indicator for such a purpose and can be used to predict rapeseed yield level. Compared to the three PY components, PAFs can be easily and accurately measured and can be screened on a large scale in the field. Therefore, PAFs may be widely applied to high-yield breeding in rapeseed and other crops. </p> <p>PA has been shown to be influenced by environmental factors, such as light, temperature, nutrition, and plant density. In this study, the heritability of PA-related traits exhibited higher than that of PY-related traits ( <a href="#S1">Supplementary Table S2</a>), indicating that PAFs are more stable measurements for genetic manipulation. </p> <p>Through extensive genetic analysis with a high-density genetic map, we identified 163 unique QTLs related to PA, PY, or both. Analyses of pleiotropic QTLs for controlling PA and yield further showed that PA and PA-related traits interact with PY. Approximately 60% of pleiotropic QTLs exhibited effects on PY and three components of PY through PA-related traits ( <a href="#S1">Supplementary Table S5</a>). Such a finding suggests that we can directly monitor pleiotropic QTLs to improve PY by selecting relevant PAF phenotypes with molecular markers on large scale in practical breeding programs. </p> <p>We collected 256 genes related to PA development and regulation and grouped the genes into 8 categories. Among the 190 predicted candidate genes corresponding to 91 unique QTLs ( <a href="#f3">Fig. 3</a> and <a href="#S1">Supplementary Table S9</a>), it was found that genes in the categories of TF (24.7%), Aux/IAA (22.6%), and GA (13.7%) accounted for the largest proportions, suggesting that these three gene categories are important for PA domestication and improvements in rapeseed, as well as in the process of yield improvement. For instance, <i>AtCUC2</i>, <i>PhNAM</i>, <i>LeSP</i>, and <i>PsLF</i> act as TFs in PA development and also indirectly influence seed yield <a href="#b40">40</a> <a href="#b41">41</a> <a href="#b42">42</a> <a href="#b43">43</a> <a href="#b44">44</a>. The "Green revolution genes" are involved in GA metabolism and signal transduction (e.g., <i>D8</i>, <i>RGA</i>, and <i>RHT1</i>) and play important roles in stem and internode elongation <a href="#b3">3</a> <a href="#b4">4</a>. </p> <p>To demonstrate the effectiveness of the prediction of QTL candidate genes, we cloned three candidate genes ( <a href="#S1">Supplementary Figs S5–S7</a>; <i>Bna.A02.CLV2</i> for <i>quPAY.A02_24.39</i>, <i>Bna.A09.SLY2</i> for <i>quBSP.A09_26.7</i>, and <i>Bna.C07.AHK4</i> for <i>quPAY.C07_21.9</i>) from the two parents. The sequences of the three candidate genes were aligned against <i>B. napus</i> reference genome and corresponding Arabidopsis genes. These three genes are all related to PA regulation in Arabidopsis <a href="#b1">1</a> <a href="#b2">2</a> <a href="#b51">51</a> <a href="#b52">52</a> <a href="#b53">53</a>. The <i>Atclv2</i> mutation results in fewer siliques, shorter main inflorescence and siliques <a href="#b51">51</a>. The <i>PsClv2</i> mutation ( <i>sym28</i>) has shorter internodes between flowering nodes, and fasciated stem <a href="#b54">54</a>, which is similar to the parent J7005 used in this study ( <a href="#S1">Supplementary Table S1</a>. Bna.A02.CLV2 in J7005 harbored a precocious termination at the 382 aa residues) compared with another parent Hua_5. Similarly, the <i>Atahk4</i> mutation leads to a more loose PA with higher branch height and fewer branches <a href="#b52">52</a>, which is similar to the performance of Hua_5 ( <a href="#S1">Supplementary Table S1</a>. Bna.C07.AHK4 in Hua_5 harbored a precocious termination at the 383 aa residues). <i>AtSLY2</i> was shown to be related to PA regulation through GA signal <a href="#b53">53</a>. Therefore, the mutations on these genes are consistent with the phenotypic performances of the two parental lines ( <a href="#f1">Fig. 1</a> and <a href="#S1">Supplementary Table S1</a>). </p> <p>PA may vary from one species from another, and such a difference is related to the genetic regulation of important PA-related genes. However, there has not been a systematic study on differentiation, the evolutionary relationship of these PA regulation genes in different species, and the similarities and differences of their regulation in <i>Brassica</i> species. Through the comparison of homologous genes in <i>B. napus</i>, many important genes for PA formation and regulation were identified, such as <i>SPY</i> and <i>CKI1</i> reported in Arabidopsis; <i>RA3</i> and <i>TD1</i> in maize; <i>TAD1</i>, <i>D14</i>, and <i>GA2ox1</i> in rice; and "Green Revolution" genes ( <i>D8</i>, <i>RGL1</i>, and <i>RHT1</i>), which had at least one homologous gene in <i>B. napus</i> ( <a href="#S1">Supplementary Table S6</a>). However, there were 40 PA-regulating genes, including some important genes, such as <i>BZR1</i> and <i>GAI</i> in Arabidopsis; <i>TB1</i>, <i>RA1</i>, and <i>RA2</i> in maize; and <i>DEP1</i>, <i>IPA1</i> ( <i>OsSPL14</i>), <i>MOC1</i>, <i>TAC1</i>, and <i>LA1</i> in rice, which did not have homologous genes in <i>B. napus</i> ( <a href="#S1">Supplementary Table S6</a>). Such a result may be attributed to a small number of homologous genes, low density of the genetic map, or the existence of specific PA-regulating loci in <i>Brassica</i>. </p> <p>In this study, a large number of AA interaction loci-pairs were identified ( <a href="#S1">Supplementary Table S11</a>), suggesting that yield formation is extremely complex. Interaction loci-pairs explained 5.18–23.18% of phenotypic variation. This finding suggests that yield improvement may be achieved by manipulating a few interaction loci, which could further broaden the gene pool for PY improvement in rapeseed. </p> <p> <i>B. napus</i> has a very complex genome structure. In this study, we identified a total of 1818 (77.4%) interaction loci-pairs with conserved blocks in Arabidopsis. Moreover, only 105 (5.78%) interaction loci-pairs have two loci that are both located on different copies of the same conserved block ( <a href="#S1">Supplementary Table S13</a>).There may be an indirect relationship between whole-genome epistatic interactions and different copies of the same conserved blocks in <i>B. napus</i>. </p> <p>Among the 183 interaction loci-pairs detected in two or more environments, 97 were located in or overlapped with confidence intervals of 100 different QTLs, and the QTLs were related to 257 candidate genes ( <a href="#S1">Supplementary Table S12</a>). Interestingly, more than half of the genes fell into three categories of genes, GA (20.2%), TF (17.1%), and Aux/IAA (13.2%), indicating the importance of these three gene categories in plant domestication and PA formation. These three gene categories are worth future in-depth studies in rapeseed. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterials,fieldtrialsandtraitevaluation" title="sec"> <div class="title" tagx="title" title="title">
Plant materials, field trials and trait evaluation</div> <p>A DH population consisting of 254 individual DH lines was generated from a microspore culture of F1 buds of the cross between cv. Huashuang 5 (Hua_5), a semi-winter <i>B. napus</i> variety, and J7005, a pure line with distinct PA from Hua_5. A random subset including 190 lines from the 254 DH lines was sampled as the mapping population (designated as the HJ-DH population thereafter) for linkage map construction and QTL mapping <a href="#b39">39</a> <a href="#b55">55</a> <a href="#b56">56</a>. </p> <p>The DH population together with two parents, F1 and RF1 were planted in Wuhan for two consecutive winter-growing seasons of 2009–2010 (WH09) and 2010–2011 (WH10), in Huanggang in the 2010–2011 (HG10) season, and in Gansu in the spring of 2011 (GS11). The field experiment followed a randomized complete block design with three replicates, including Hua_5 as a control for every 20 plots. Each line was planted in two rows with 12 plants in each row, with distances of 17 cm between plants within each row and 30 cm between rows. Field management followed regular breeding practices. </p> <p>Fifteen plants growing uniformly from each plot were chosen for trait evaluation. The mean of each trait calculated from the 15 sampled plants in one plot was used to determine plot performance. Traits examined in this study were grouped into two categories. The first category included plant yield (PY) related traits, i.e., silique seed number (SSN), thousand seed weight (TSW), main inflorescence silique number (MISN), branch silique number (BSN), main inflorescence yield (MIY), and branch yield (BY). The second category included PA-related traits, i.e. plant height (PH), branch height (BH), main inflorescence length (MIL), branch number (BN), and branch total length (BTL). </p> <p>The performance of three high-yield check (CK) cultivars included in the national rapeseed regional trial in China was used to verify the results from the DH population. Each of the CKs were grown in numbers of environment (N) with various plant density ( <a href="#t2">Table 2</a>) following a standard field experiment of a randomized complete block design with three replicates and 20 m <sup>2</sup> for each plot as outlined in the reports by National Agricultural Technology Extension and Service Center (NATESC) <a href="#b37">37</a> <a href="#b57">57</a>. </p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>The broad-sense heritability ( <i>h</i> <sup>2</sup>) of each trait in HJ-DH population was calculated as: </p> <p> </p><div class="disp-formula" title="disp-formula"> <span class="inline-graphic" title="inline-graphic" /> </div> <p /> <p>where σ <sup>2</sup> <sub>g</sub> is genotypic variance, σ <sup>2</sup> <sub>ge</sub> variance due to genotype by environment interaction, σ <sup>2</sup> <sub>e</sub> error variance, <i>n</i> number of environments; <i>r</i> number of replications. The estimates of σ <sup>2</sup> <sub>g</sub>, σ <sup>2</sup> <sub>ge</sub> and σ <sup>2</sup> <sub>e</sub>, were obtained from an ANOVA with environment considered as a random effect. </p> <p>The statistical analysis of the phenotypes (correlation coefficient, multiple comparison) was conducted through the Excel 2010 and SAS 7.0 software. </p> </div> <div class="molecularmarkerandsnparraygenotyping" title="sec"> <div class="title" tagx="title" title="title">
Molecular marker and SNP array genotyping</div> <p>Primer sequences of SSR (simple sequence repeat) markers for genetic mapping were described by Cai <i>et al.</i> <a href="#b39">39</a>. SNP genotyping was performed using <i>B. napus</i> 6 K Illumina Infinium HD Assay SNP arrays (Illumina Inc., San Diego, CA) developed by the University of Queensland. The analysis process and bi-filtering method were described previously by Cai <i>et al.</i> <a href="#b56">56</a>. </p> </div> <div class="geneticmapconstruction,qtlmappingandmeta-analysis" title="sec"> <div class="title" tagx="title" title="title">
Genetic map construction, QTL mapping and meta-analysis</div> <p>Linkage analysis with all markers was performed using MAPMAKER 3.0 <a href="#b58">58</a> and MSTmap <a href="#b59">59</a> software. Detailed information describing genetic map construction was provided previously by Cai <i>et al.</i> <a href="#b39">39</a>. QTLs were detected using the composite interval mapping (CIM) procedure in QTL Cartographer V2.5 software ( <a href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a>). A P = 0.05 significance threshold for QTLs was determined through permutation analysis with 1,000 repetitions. Other parameters and QTL mapping methods were described previously by Feng <i>et al.</i> <a href="#b60">60</a>. </p> <p>Identified QTLs were integrated using a "two-round" strategy of QTL meta-analysis <a href="#b60">60</a> <a href="#b61">61</a> with BioMercator2.1 software <a href="#b62">62</a> <a href="#b63">63</a>. In the first round, identified QTLs of a specific trait that were repeatedly detected in multiple environments were integrated and labeled as consensus QTLs (step 1). In the second round, overlapping consensus QTLs of different traits were integrated into unique QTLs (step 2). </p> </div> <div class="identificationofhomologouspa-relatedgenesinb.napus" title="sec"> <div class="title" tagx="title" title="title">
Identification of homologous PA-related genes in <i>B. napus</i> </div> <p>To identify and locate putative PA-related homologous genes in <i>B. napus</i>, the CDS sequences of PA-related genes in Arabidopsis were used as queries to search for homologous genes using BLASTn program, and the protein sequences of PA-related genes in other species were queried against the <i>B. napus</i> genome <a href="#b27">27</a> ( <a href="http://www.genoscope.cns.fr/brassicanapus/">http://www.genoscope.cns.fr/brassicanapus/</a>) using the BLASTp program with an E-value of 1E-50 and an Identity of 50% set as thresholds. </p> </div> <div class="identificationofphysicalssrandsnpmarkerpositionsinb.napus" title="sec"> <div class="title" tagx="title" title="title">
Identification of physical SSR and SNP marker positions in <i>B. napus</i> </div> <p>An SSR primer sequence was used to identify its physical position in the <i>B. napus</i> reference genome <a href="#b27">27</a> through e-PCR analysis. When an SSR marker had more than two amplicons, the accurate position for a particular locus was manually determined by referring to the physical positions of its upstream and downstream amplicons <a href="#b64">64</a>. </p> <p>The probe sequence of a SNP was used to identify its physical position in the <i>B. napus</i> reference genome using BLASTn analysis. The best-hit locus of BLASTn results was chosen as the potential physical position of the given SNP locus in <i>B. napus</i>. </p> <p>The method used to identify homoeologous exchanges described by Chalhoub <i>et al.</i> <a href="#b27">27</a> was performed as described previously by Cai <i>et al.</i> <a href="#b39">39</a>. The relationship between the <i>B. napus</i> genetic map and the physical map (reference genome) was established ( <a href="#S1">Supplementary Table S7</a>) by homoeologous exchanges identified in the HJ-DH population. </p> <p>The homologs of PA-related genes in <i>B. napus</i> were mapped onto <i>B. napus</i> genetic linkage groups based on the physical positions of the markers and homologous genes. </p> </div> <div class="genome-wideepistaticinteractionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide epistatic interaction analysis</div> <p>Genome-wide epistatic interaction analysis was conducted using two-way analysis of variance (ANOVA) as described by Zhou <i>et al.</i> <a href="#b45">45</a>. Briefly, a statistic for the F-test was first constructed as: </p> <p> </p><div class="disp-formula" title="disp-formula"> <span class="inline-graphic" title="inline-graphic" /> </div> <p /> <p>where <i>F</i> <sub> <i>A × B</i> </sub> is the F-test for the significance of interaction effect of locus-pair (A locus and B locus). <i>SS</i> <sub> <i>A × B</i> </sub> and <i>SS</i> <sub> <i>E</i> </sub> are the sum of squares of the AB locus-pair interaction effect and the random error effect, respectively. <i>df</i> <sub> <i>A × B</i> </sub> and <i>df</i> <sub> <i>E</i> </sub> are the degree of freedom of the AB locus-pair interaction effect and random error effect, respectively. A significance threshold for the F-test at P ≤ 0.001 was calculated through 10,000- repetition permutation analysis. The details of epistatic interaction loci detection have been described previously by Zhou <i>et al.</i> <a href="#b45">45</a>. </p> <p>The two-locus-interaction significant pairs that passed the permutation tests and significance on F-test, were used to merge the interaction effects of the neighboring molecular markers by using of point-to-point expansion method. The two interaction loci were ≥defined as <i>A</i> <sub> <i>x</i> </sub> and <i>B</i> <sub> <i>y</i> </sub>, and the F-value was calculated as <i>F(x, y)</i> by two-way ANOVA, and then a <i>M × N</i> matrix was constructed: </p> <p> </p><div class="disp-formula" title="disp-formula"> <span class="inline-graphic" title="inline-graphic" /> </div> <p /> <p>We used the <i>A</i> <sub> <i>x</i> </sub> and <i>B</i> <sub> <i>y</i> </sub> as the centers, and extended from these two centers in two directions by <i>m</i> and <i>n</i> marker loci ( <i>m</i> ≥ 10, <i>n</i> ≥ 10), respectively. The distance between <i>A</i> <sub> <i>1</i> </sub> and <i>A</i> <sub> <i>m</i> </sub>, <i>B</i> <sub> <i>1</i> </sub> and <i>B</i> <sub> <i>n</i> </sub> was not less than 2 cM. </p> <p>The largest effect locus <i>F(x, y)</i> was chosen in the matrix as the starting point to perform a merging process. We first fixed <i>A</i> <sub> <i>x</i> </sub>, and used <i>B</i> <sub> <i>y</i> </sub> as the center, extended towards the two ends to test the significance of the molecular markers within the region from <i>B</i> <sub> <i>1</i> </sub> to <i>B</i> <sub> <i>n</i> </sub> with <i>A</i> <sub> <i>x</i> </sub>. The molecular markers in the <i>B</i> region were merged using a threshold 1.5- <i>Lg</i> <sub> <i>10</i> </sub> <i>P</i>-drop confidence interval, and these merged markers were defined as <i>A</i> <sub> <i>x′</i> </sub>. And then we fixed <i>B</i> <sub> <i>y</i> </sub>, and extended the loci towards the two ends, centering on the <i>A</i> <sub> <i>x</i> </sub>, to test the significance of the molecular markers in the <i>A</i> <sub> <i>1</i> </sub> to <i>A</i> <sub> <i>m</i> </sub> region with <i>B</i> <sub> <i>y</i> </sub>. Similarly, the molecular markers in this <i>A</i> region were merged using a threshold 1.5- <i>Lg</i> <sub> <i>10</i> </sub> <i>P</i>-drop confidence interval, and they were defined as <i>B</i> <sub> <i>y′</i> </sub>. Finally, the combination <i>A</i> <sub> <i>x′</i> </sub> ×  <i>B</i> <sub> <i>y′</i> </sub> consisting of two groups of markers from regions <i>A</i> and <i>B</i> were regarded as one epistatic interaction pair, the effect of which come from the most significant pair <i>A</i> <sub> <i>x</i> </sub> ×  <i>B</i> <sub> <i>y</i> </sub> in all the corresponding pairs. </p> </div> <div class="genecloningandsequencealignment" title="sec"> <div class="title" tagx="title" title="title">
Gene cloning and sequence alignment</div> <p>Genomic DNA was prepared from young leaf tissues as described previously <a href="#b39">39</a>. The primers used for amplification of the genomic DNA fragments are listed in <a href="#S1">supplementary Table S14</a>. The PCR products were cloned into pMD18-T vector (Takara Corporation, Japan) according to the manufacturer’s instructions. The M13F and M13R universal primers and the BigDye Terminator Cycle Sequencing v3.1 (Applied Biosystems, Foster City, CA, USA) were used for sequencing. Sequences were aligned using SEQUENCHER 4.1.2 (Gene Codes Corporation, Ann Arbor, MI, USA). </p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional Information</div> <p> <b>How to cite this article</b>: Cai, G. <i>et al.</i> Genetic dissection of plant architecture and yield-related traits in <i>Brassica napus</i>. <i>Sci. Rep.</i> <b>6</b>, 21625; doi: 10.1038/srep21625 (2016). </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div>  <div class="media" title="media"> <a href="srep21625-s1.pdf">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 1</div>  <div class="media" title="media"> <a href="srep21625-s2.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 2</div>  <div class="media" title="media"> <a href="srep21625-s3.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 3</div>  <div class="media" title="media"> <a href="srep21625-s4.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 4</div>  <div class="media" title="media"> <a href="srep21625-s5.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 5</div>  <div class="media" title="media"> <a href="srep21625-s6.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 6</div>  <div class="media" title="media"> <a href="srep21625-s7.xls">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary dataset 7</div>  <div class="media" title="media"> <a href="srep21625-s8.xls">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The work is financially supported by the funding from Ministry of Science and Technology of China (2015CB150200, 2014DFA32210), Ministry of Agriculture of China (nycytx-00503 and 948 project (2011-G23)), Fundamental Research Funds for Central Universities (2662015PY128), China Postdoctoral Science Foundation (2015M572166), and Huazhong Agricultural University (STSIF2010YB05). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="b1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Busov</span> <span class="given-names" tagx="given-names" title="given-names">V. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunner</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strauss</span> <span class="given-names" tagx="given-names" title="given-names">S. H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genes for control of plant stature and form</span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">177</span>, <span class="fpage" tagx="fpage" title="fpage">589</span>– <span class="lpage" tagx="lpage" title="lpage">607</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18211474">18211474</a> </span> </span> </li> <li tag="ref"> <a name="b2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular basis of plant architecture</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">253</span>– <span class="lpage" tagx="lpage" title="lpage">279</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18444901">18444901</a> </span> </span> </li> <li tag="ref"> <a name="b3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evenson</span> <span class="given-names" tagx="given-names" title="given-names">R. E.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gollin</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Assessing the impact of the Green Revolution, 1960 to 2000</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">300</span>, <span class="fpage" tagx="fpage" title="fpage">758</span>– <span class="lpage" tagx="lpage" title="lpage">762</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12730592">12730592</a> </span> </span> </li> <li tag="ref"> <a name="b4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">‘Green revolution’ genes encode mutant gibberellin response modulators</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">400</span>, <span class="fpage" tagx="fpage" title="fpage">256</span>– <span class="lpage" tagx="lpage" title="lpage">261</span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10421366">10421366</a> </span> </span> </li> <li tag="ref"> <a name="b5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Genetic control of rice plant architecture under domestication</span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">1365</span>– <span class="lpage" tagx="lpage" title="lpage">1369</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18820696">18820696</a> </span> </span> </li> <li tag="ref"> <a name="b6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Regulation of <i>OsSPL14</i> by <i>OsmiR156</i> defines ideal plant architecture in rice </span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">541</span>– <span class="lpage" tagx="lpage" title="lpage">544</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20495565">20495565</a> </span> </span> </li> <li tag="ref"> <a name="b7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pickersgill</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Domestication of plants in the Americas: insights from Mendelian and molecular genetics</span>. <span class="source" tagx="source" title="source">Ann. Bot.</span> <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">925</span>– <span class="lpage" tagx="lpage" title="lpage">940</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17766847">17766847</a> </span> </span> </li> <li tag="ref"> <a name="b8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duvick</span> <span class="given-names" tagx="given-names" title="given-names">D. N.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The Contribution of Breeding to Yield Advances in maize ( <i>Zea mays</i> L.) </span>. <span class="source" tagx="source" title="source">Adv. Agron.</span> <span class="volume" tagx="volume" title="volume">86</span>, <span class="fpage" tagx="fpage" title="fpage">83</span>– <span class="lpage" tagx="lpage" title="lpage">145</span> ( <span class="year" tagx="year" title="year">2005</span>). </span> </li> <li tag="ref"> <a name="b9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population </span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">159</span>– <span class="lpage" tagx="lpage" title="lpage">162</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21217756">21217756</a> </span> </span> </li> <li tag="ref"> <a name="b10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ku</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Quantitative trait loci mapping of leaf angle and leaf orientation value in maize ( <i>Zea mays</i> L.) </span>. <span class="source" tagx="source" title="source">Theor. Appl. Genet.</span> <span class="volume" tagx="volume" title="volume">121</span>, <span class="fpage" tagx="fpage" title="fpage">951</span>– <span class="lpage" tagx="lpage" title="lpage">959</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20526576">20526576</a> </span> </span> </li> <li tag="ref"> <a name="b11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monna</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Positional cloning of rice semidwarfing gene, <i>sd-1</i>: rice "green revolution gene" encodes a mutant enzyme involved in gibberellin synthesis </span>. <span class="source" tagx="source" title="source">DNA Res.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">11</span>– <span class="lpage" tagx="lpage" title="lpage">17</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11939564">11939564</a> </span> </span> </li> <li tag="ref"> <a name="b12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrášek</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">PIN proteins perform a rate-limiting function in cellular auxin efflux</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">312</span>, <span class="fpage" tagx="fpage" title="fpage">914</span>– <span class="lpage" tagx="lpage" title="lpage">918</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16601150">16601150</a> </span> </span> </li> <li tag="ref"> <a name="b13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Komatsu</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maekawa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimamoto</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kyozuka</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The <i>LAX1</i> and <i>FRIZZY PANICLE 2</i> genes determine the inflorescence architecture of rice by controlling rachis-branch and spikelet development </span>. <span class="source" tagx="source" title="source">Dev. Biol.</span> <span class="volume" tagx="volume" title="volume">231</span>, <span class="fpage" tagx="fpage" title="fpage">364</span>– <span class="lpage" tagx="lpage" title="lpage">373</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11237465">11237465</a> </span> </span> </li> <li tag="ref"> <a name="b14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ulmasov</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hagen</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guilfoyle</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>ARF1</i>, a transcription factor that binds to auxin response elements </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">276</span>, <span class="fpage" tagx="fpage" title="fpage">1865</span>– <span class="lpage" tagx="lpage" title="lpage">1868</span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9188533">9188533</a> </span> </span> </li> <li tag="ref"> <a name="b15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Werner</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motyka</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strnad</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmülling</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulation of plant growth by cytokinin</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">10487</span>– <span class="lpage" tagx="lpage" title="lpage">10492</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11504909">11504909</a> </span> </span> </li> <li tag="ref"> <a name="b16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rupp</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frank</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Werner</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strnad</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmülling</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Increased steady state mRNA levels of the <i>STM</i> and <i>KNAT1</i> homeobox genes in cytokinin overproducing <i>Arabidopsis thaliana</i> indicate a role for cytokinins in the shoot apical meristem </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">557</span>– <span class="lpage" tagx="lpage" title="lpage">563</span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10417706">10417706</a> </span> </span> </li> <li tag="ref"> <a name="b17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chory</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Brassinosteroids regulate dissociation of <i>BKI1</i>, a negative regulator of <i>BRI1</i> signaling, from the plasma membrane </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">313</span>, <span class="fpage" tagx="fpage" title="fpage">1118</span>– <span class="lpage" tagx="lpage" title="lpage">1122</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16857903">16857903</a> </span> </span> </li> <li tag="ref"> <a name="b18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">BAK1, an Arabidopsis LRR Receptor-like Protein Kinase, Interacts with <i>BRI1</i> and Modulates Brassinosteroid Signaling </span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">110</span>, <span class="fpage" tagx="fpage" title="fpage">213</span>– <span class="lpage" tagx="lpage" title="lpage">222</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12150929">12150929</a> </span> </span> </li> <li tag="ref"> <a name="b19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arite</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title"> <i>d14</i>, a strigolactone-insensitive mutant of rice, shows an accelerated outgrowth of tillers </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">1416</span>– <span class="lpage" tagx="lpage" title="lpage">1424</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19542179">19542179</a> </span> </span> </li> <li tag="ref"> <a name="b20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title"> <i>DWARF27</i>, an iron-containing protein required for the biosynthesis of strigolactones, regulates rice tiller bud outgrowth </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">1512</span>– <span class="lpage" tagx="lpage" title="lpage">1525</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19470589">19470589</a> </span> </span> </li> <li tag="ref"> <a name="b21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Control of tillering in rice</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">422</span>, <span class="fpage" tagx="fpage" title="fpage">618</span>– <span class="lpage" tagx="lpage" title="lpage">621</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12687001">12687001</a> </span> </span> </li> <li tag="ref"> <a name="b22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Veylder</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Functional analysis of cyclin-dependent kinase inhibitors of Arabidopsis</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">1653</span>– <span class="lpage" tagx="lpage" title="lpage">1668</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11449057">11449057</a> </span> </span> </li> <li tag="ref"> <a name="b23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Cell cycle function of a rice B2-type cyclin interacting with a B-type cyclin-dependent kinase </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">417</span>– <span class="lpage" tagx="lpage" title="lpage">425</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12753582">12753582</a> </span> </span> </li> <li tag="ref"> <a name="b24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuo</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular dissection of complex agronomic traits of rice: a team effort by Chinese scientists in recent years </span>. <span class="source" tagx="source" title="source">Nat. Sci. Rev.</span> <span class="volume" tagx="volume" title="volume">00</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">24</span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The genome of the mesopolyploid crop species <i>Brassica rapa</i> </span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">1035</span>– <span class="lpage" tagx="lpage" title="lpage">1039</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998">21873998</a> </span> </span> </li> <li tag="ref"> <a name="b26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">U</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome analysis in <i>Brassica</i> with special reference to the experimental formation of <i>B. napus</i> and peculiar mode of fertilization </span>. <span class="source" tagx="source" title="source">Jap. J. Bot.</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">389</span>– <span class="lpage" tagx="lpage" title="lpage">452</span> ( <span class="year" tagx="year" title="year">1935</span>). </span> </li> <li tag="ref"> <a name="b27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chalhoub</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Early allopolyploid evolution in the post-Neolithic <i>Brassica napus</i> oilseed genome </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">345</span>, <span class="fpage" tagx="fpage" title="fpage">950</span>– <span class="lpage" tagx="lpage" title="lpage">953</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25146293">25146293</a> </span> </span> </li> <li tag="ref"> <a name="b28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schranz</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lysak</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitchell-Olds</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The ABC’s of comparative genomics in the Brassicaceae: building blocks of crucifer genomes </span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">535</span>– <span class="lpage" tagx="lpage" title="lpage">542</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17029932">17029932</a> </span> </span> </li> <li tag="ref"> <a name="b29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bancroft</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="source" tagx="source" title="source">Genetics and Genomics of the Brassicaceae</span> (ed. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bancroft</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>). <span class="volume" tagx="volume" title="volume">Vol. 9</span> (Springer, <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Campo</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="source" tagx="source" title="source">Biology of Brassica coenospecies</span> (ed. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Campo</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>). <span class="volume" tagx="volume" title="volume">Vol. 4</span> (Access Online via Elsevier, <span class="year" tagx="year" title="year">1999</span>). </span> </li> <li tag="ref"> <a name="b31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">LI</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Study on inheritance and heterosis of plant-type traits in compact rapeseed lines</span>. <span class="source" tagx="source" title="source">Acta Agriculturae Boreali-Occidentalis Sinica</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">36</span> ( <span class="year" tagx="year" title="year">2006</span>). </span> </li> <li tag="ref"> <a name="b32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Evaluation of lodging resistance in rapeseed ( <i>Brassica napus</i> L.) and relationship between plant architecture and lodging resistance </span>. <span class="source" tagx="source" title="source">Chinese J. Oil Crop Sci.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">54</span>– <span class="lpage" tagx="lpage" title="lpage">57</span> ( <span class="year" tagx="year" title="year">2007</span>). </span> </li> <li tag="ref"> <a name="b33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Genetic analysis of plant type in semi-dwarf new line (10D130) of rapeseed</span>. <span class="source" tagx="source" title="source">Acta Agronomica Sinica</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">207</span>– <span class="lpage" tagx="lpage" title="lpage">215</span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Study on plant-type structure and it’s ideotype in rapeseed III. Plant type and canopy structure of some high yield varieties </span>. <span class="source" tagx="source" title="source">Chinese J. Oil Crop Sci.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">36</span>– <span class="lpage" tagx="lpage" title="lpage">41</span> ( <span class="year" tagx="year" title="year">1998</span>). </span> </li> <li tag="ref"> <a name="b35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muangprom</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osborn</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of a dwarf gene in <i>Brassica rapa</i>, including the identification of a candidate gene </span>. <span class="source" tagx="source" title="source">Theor. Appl. Genet.</span> <span class="volume" tagx="volume" title="volume">108</span>, <span class="fpage" tagx="fpage" title="fpage">1378</span>– <span class="lpage" tagx="lpage" title="lpage">1384</span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14727029">14727029</a> </span> </span> </li> <li tag="ref"> <a name="b36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">A missense mutation in the VHYNP motif of a DELLA protein causes a semi-dwarf mutant phenotype in <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">Theor. Appl. Genet.</span> <span class="volume" tagx="volume" title="volume">121</span>, <span class="fpage" tagx="fpage" title="fpage">249</span>– <span class="lpage" tagx="lpage" title="lpage">258</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20221582">20221582</a> </span> </span> </li> <li tag="ref"> <a name="b37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Yield and agronomic traits of winter rapeseed cultivars registered in China from 2001 to 2010 </span>. <span class="source" tagx="source" title="source">Chinese J. Oil Crop Sci.</span> <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>– <span class="lpage" tagx="lpage" title="lpage">244</span> ( <span class="year" tagx="year" title="year">2012</span>). </span> </li> <li tag="ref"> <a name="b38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of yield and disease resistance traits of new winter rapeseed variety in the past twenty years in China </span>. <span class="source" tagx="source" title="source">Chinese Agricultural Science Bulletin</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">375</span>– <span class="lpage" tagx="lpage" title="lpage">380</span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">A complex recombination pattern in the genome of allotetraploid <i>Brassica napus</i> as revealed by a high-density genetic map </span>. <span class="source" tagx="source" title="source">PloS ONE</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e109910</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25356735">25356735</a> </span> </span> </li> <li tag="ref"> <a name="b40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nikovics</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The balance between the <i>MIR164A</i> and <i>CUC2</i> genes controls leaf margin serration in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">2929</span>– <span class="lpage" tagx="lpage" title="lpage">2945</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17098808">17098808</a> </span> </span> </li> <li tag="ref"> <a name="b41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peaucelle</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Traas</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laufs</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plants expressing a miR164-resistant <i>CUC2</i> gene reveal the importance of post-meristematic maintenance of phyllotaxy in Arabidopsis </span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">134</span>, <span class="fpage" tagx="fpage" title="fpage">1045</span>– <span class="lpage" tagx="lpage" title="lpage">1050</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17251269">17251269</a> </span> </span> </li> <li tag="ref"> <a name="b42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X. G.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Cytokinin overproduction-caused alteration of flower development is partially mediated by <i>CUC2</i> and <i>CUC3</i> in Arabidopsis </span>. <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">450</span>, <span class="fpage" tagx="fpage" title="fpage">109</span>– <span class="lpage" tagx="lpage" title="lpage">120</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19913077">19913077</a> </span> </span> </li> <li tag="ref"> <a name="b43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pnueli</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The <i>SELF-PRUNING</i> gene of tomato regulates vegetative to reproductive switching of sympodial meristems and is the ortholog of <i>CEN</i> and <i>TFL1</i> </span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">125</span>, <span class="fpage" tagx="fpage" title="fpage">1979</span>– <span class="lpage" tagx="lpage" title="lpage">1989</span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9570763">9570763</a> </span> </span> </li> <li tag="ref"> <a name="b44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foucher</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title"> <i>DETERMINATE</i> and <i>LATE FLOWERING</i> are two <i>TERMINAL FLOWER1/CENTRORADIALIS</i> homologs that control two distinct phases of flowering initiation and development in pea </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">2742</span>– <span class="lpage" tagx="lpage" title="lpage">2754</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14563931">14563931</a> </span> </span> </li> <li tag="ref"> <a name="b45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Genetic composition of yield heterosis in an elite rice hybrid</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="volume" tagx="volume" title="volume">109</span>, <span class="fpage" tagx="fpage" title="fpage">15847</span>– <span class="lpage" tagx="lpage" title="lpage">15852</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23019369">23019369</a> </span> </span> </li> <li tag="ref"> <a name="b46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doebley</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaut</span> <span class="given-names" tagx="given-names" title="given-names">B. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">B. D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The molecular genetics of crop domestication</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">127</span>, <span class="fpage" tagx="fpage" title="fpage">1309</span>– <span class="lpage" tagx="lpage" title="lpage">1321</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17190597">17190597</a> </span> </span> </li> <li tag="ref"> <a name="b47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Mapping of quantitative trait loci and development of allele-specific markers for seed weight in <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">Theor. Appl. Genet.</span> <span class="volume" tagx="volume" title="volume">121</span>, <span class="fpage" tagx="fpage" title="fpage">1289</span>– <span class="lpage" tagx="lpage" title="lpage">1301</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20574694">20574694</a> </span> </span> </li> <li tag="ref"> <a name="b48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzig</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Early osmotic adjustment responses in drought-resistant and drought-sensitive oilseed rape </span>. <span class="source" tagx="source" title="source">J. Intergr. Plant Biol.</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">797</span>– <span class="lpage" tagx="lpage" title="lpage">809</span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schiessl</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samans</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hüttel</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhard</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snowdon</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Capturing sequence variation among flowering-time regulatory gene homologs in the allopolyploid crop species <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">404</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25202314">25202314</a> </span> </span> </li> <li tag="ref"> <a name="b50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzig</span> <span class="given-names" tagx="given-names" title="given-names">S. V.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Genome-wide association mapping unravels the genetic control of seed germination and vigor in <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">221</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25914704">25914704</a> </span> </span> </li> <li tag="ref"> <a name="b51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishimura</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Histidine kinase homologs that act as cytokinin receptors possess overlapping functions in the regulation of shoot and root growth in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1365</span>– <span class="lpage" tagx="lpage" title="lpage">1377</span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15155880">15155880</a> </span> </span> </li> <li tag="ref"> <a name="b52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayes</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>CLAVATA2</i>, a regulator of meristem and organ development in Arabidopsis </span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">125</span>, <span class="fpage" tagx="fpage" title="fpage">3843</span>– <span class="lpage" tagx="lpage" title="lpage">3851</span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9729492">9729492</a> </span> </span> </li> <li tag="ref"> <a name="b53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krusell</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The <i>Clavata2</i> genes of pea and <i>Lotus japonicus</i> affect autoregulation of nodulation </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">861</span>– <span class="lpage" tagx="lpage" title="lpage">871</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21276104">21276104</a> </span> </span> </li> <li tag="ref"> <a name="b54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ariizumi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steber</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Mutations in the F-box gene <i>SNEEZY</i> result in decreased Arabidopsis GA signaling </span>. <span class="source" tagx="source" title="source">Plant Signaling &amp;amp; Behavior</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">831</span>– <span class="lpage" tagx="lpage" title="lpage">833</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21455025">21455025</a> </span> </span> </li> <li tag="ref"> <a name="b55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Identification of QTLs for resistance to Sclerotinia Stem Rot and <i>BnaC.IGMT5.a</i> as a candidate gene of the major resistant QTL <i>SRC6</i> in <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">PloS ONE</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e67740</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23844081">23844081</a> </span> </span> </li> <li tag="ref"> <a name="b56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">A bi-filtering method for processing single nucleotide polymorphism array data improves the quality of genetic map and accuracy of quantitative trait locus mapping in doubled haploid populations of polyploid <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">409</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26018616">26018616</a> </span> </span> </li> <li tag="ref"> <a name="b57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">National Agricultural Technology Extension and Service Center. <span class="source" tagx="source" title="source">Breeding Progress of Winter Rapeseed Varieties in China-Compiled reports of the national rapeseed regional trial in China </span> [In Chinese, NATESC. (ed.)] (China Agricultural Science and Technology Press, Beijing, <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lincoln</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Constructing genetic maps with MAPMAKER/EXP 3.0</span>. <span class="source" tagx="source" title="source">Whitehead Institute Technical Report</span>, <span class="fpage" tagx="fpage" title="fpage">78</span>– <span class="lpage" tagx="lpage" title="lpage">79</span> ( <span class="year" tagx="year" title="year">1993</span>). </span> </li> <li tag="ref"> <a name="b59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhat</span> <span class="given-names" tagx="given-names" title="given-names">P. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Close</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lonardi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph </span>. <span class="source" tagx="source" title="source">PLoS Genet.</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">e1000212</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18846212">18846212</a> </span> </span> </li> <li tag="ref"> <a name="b60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Characterization of metabolite quantitative trait loci and metabolic networks that control glucosinolate concentration in the seeds and leaves of <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">193</span>, <span class="fpage" tagx="fpage" title="fpage">96</span>– <span class="lpage" tagx="lpage" title="lpage">108</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21973035">21973035</a> </span> </span> </li> <li tag="ref"> <a name="b61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Unraveling the complex trait of crop yield with quantitative trait loci mapping in <i>Brassica napus</i> </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">182</span>, <span class="fpage" tagx="fpage" title="fpage">851</span>– <span class="lpage" tagx="lpage" title="lpage">861</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19414564">19414564</a> </span> </span> </li> <li tag="ref"> <a name="b62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goffinet</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerber</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative trait loci: a meta-analysis</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">155</span>, <span class="fpage" tagx="fpage" title="fpage">463</span>– <span class="lpage" tagx="lpage" title="lpage">473</span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10790417">10790417</a> </span> </span> </li> <li tag="ref"> <a name="b63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arcade</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">BioMercator: integrating genetic maps and QTL towards discovery of candidate genes</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">2324</span>– <span class="lpage" tagx="lpage" title="lpage">2326</span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15059820">15059820</a> </span> </span> </li> <li tag="ref"> <a name="b64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Identification of candidate genes of QTLs for seed weight in <i>Brassica napus</i> through comparative mapping among Arabidopsis and <i>Brassica</i> species </span>. <span class="source" tagx="source" title="source">BMC Genet.</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">105</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23216693">23216693</a> </span> </span> </li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Author Contributions</b> G.C. and Y.Z. conceived and performed experiments and wrote the manuscript; Q.Y.Y., C.Z. and C.F. were involved in data analysis; G.C., H.C. and Q.Y. performed the genetic map construction, QTL mapping, candidate gene and gene network analyses. All the authors have commented, read and approved the final manuscript. </p> </div> </div> </div>  </body></html>