<html >

   
  
      
  
                  
  
                  
  
            
  
            
            
    <title>BMC Evolutionary Biology | Full text | Gene conversion and 
purifying selection shape nucleotide variation in gibbon L/M opsin genes</title>


<body data-twttr-rendered="true" class=" theme    "><iframe id="sina_anywhere_iframe" style="display: none;"></iframe>


                                        



   
  
      
  
                  
  
                  
  
            
  
            
            

                    

<div id="oas-campaign" style="display:none">biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/11/312</div>
<div id="oas-positions" style="display:none">Bottom,Top</div>

<div id="branding" role="banner">
                			
		<dl class="google-ad wide  advert-text">
	<dt style="display: block;" class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">
		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="//oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/11/312/16709845129@Top?">
            
        </a>
    </dd>
			</dl>
		</noscript>
			            <div class="sup-panel-outer">
        <div class="sup-panel-inner">

            <style>
  #nojs li.greeting, #nojs li#loginLink {display:none;}
</style>


<ul id="login" class="login">

    <noscript><li><a href="/log"></a></li></noscript>
    
    <li id="welcome"><span id="username"></span></li>

    <li id="loginLink"><a href="http://www.biomedcentral.com/logon">Log on</a></li>
    
    
</ul>


            <ul class="nav-sup">

    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title"></span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">

        

    
<style>
.survey {
	margin: 10px 0;
}
.survey, .survey .survey-sentence {
	font-size: 13px;
}
.survey-button a, .survey-button a:active, .survey-button a:visited, .survey-button a:hover  {
    color: #FFFFFF;
    text-decoration: none;
}
.survey-button {
	background-color: #FFA500;
    border-radius: 5px;
    color: #FFFFFF;
    font-weight: bold;
    font-size: 13px;
    font-weight: bold;
    padding: 5px 13px;
    background: #ef8142; /* Old browsers */
	background: -moz-linear-gradient(top,  #ef8142 0%, #ef8142 50%, #a64510 100%, #a64510 100%); /* FF3.6+ */
	background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#ef8142), color-stop(50%,#ef8142), color-stop(100%,#a64510), color-stop(100%,#a64510)); /* Chrome,Safari4+ */
	background: -webkit-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Chrome10+,Safari5.1+ */
	background: -o-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Opera 11.10+ */
	background: -ms-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* IE10+ */
	background: linear-gradient(to bottom,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* W3C */
	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ef8142', endColorstr='#a64510',GradientType=0 ); /* IE6-9 */

}
</style>




          <div class="logo">
        <a href="http://www.biomedcentral.com/bmcevolbiol/"></a>     
      </div>    
        
                    



            <div id="impact-factor" class="official">
    
    <span id="impact-factor-value">3.41</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm" action=" /quickSearchRedirectURL " onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>
                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcevolbiol/search/results" selected="selected">this journal</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="http://www.biomedcentral.com/bmcevolbiol/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list">

	<ul class="primary-nav" role="navigation">
		<li><a href="http://www.biomedcentral.com/bmcevolbiol/"><span id="home-tab">Home</span></a></li>
														<li class="current"><a href="http://www.biomedcentral.com/bmcevolbiol/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li><a href="http://www.biomedcentral.com/bmcevolbiol/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li><a href="http://www.biomedcentral.com/bmcevolbiol/about"><span id="about-tab">About this journal</span></a></li>
	<li><a href="http://www.biomedcentral.com/bmcevolbiol/my"><span id="my-tab">My BMC Evolutionary Biology</span></a></li>
		
    		</ul>
</div>
            </div>
</div>

<div class="container">
	<div id="content" class="overflow-visible" role="main">
			    
	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcevolbiol/">
		    
		</a>
		</h1>
	</div><br/-->



			
	



<div style="position: absolute;" class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Methods</a>
   </li>
   <li>
      <a href="#sec3">Results</a>
   </li>
   <li>
      <a href="#sec4">Discussion</a>
   </li>
   <li>
      <a href="#sec5">Conclusions</a>
   </li>
   <li>
      <a href="#sec6">Authors' contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>

<dl id="box-outline1">
</dl>

				



</div>



<div class="mobile-hidden" id="right-panel">
 	
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcevolbiol"><strong>BMC Evolutionary Biology</strong></a></h5>
 		        <ul class="square normal" id="section-name">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/sections/speciation_and_evolutionary_genetics">Speciation and evolutionary genetics</a></li>
        </ul>
                <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/content/11">Volume 11</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="http://www.biomedcentral.com/1471-2148/11/312/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		<li class="pdfFileSize">
			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2148/11/312', 1, true]);" href="http://www.biomedcentral.com/content/pdf/1471-2148-11-312.pdf">PDF</a>
			<span>(442KB)</span>
		</li>

        
				<li>
						<a href="http://www.biomedcentral.com/1471-2148/11/312/additional">Additional files</a>
					</li>
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="http://www.biomedcentral.com/pubmed/22017819">PubMed record</a>
        </li>
                            <li><a href="http://www.biomedcentral.com/1471-2148/11/312/about">Article metrics</a></li>
                <li>
                        <a href="http://www.biomedcentral.com/1471-2148/11/312/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li><a href="http://www.biomedcentral.com/1471-2148/11/312/about#citations">Cited by</a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=%22Gene+conversion+and+purifying+selection+shape+nucleotide+variation+in+gibbon+L%2FM+opsin+genes%22&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i></i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22T+Hiwatashi%22">Hiwatashi T</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22A+Mikami%22">Mikami A</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22T+Katsumura%22">Katsumura T</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22B+Suryobroto%22">Suryobroto B</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22D+Perwitasari-Farajallah%22">Perwitasari-Farajallah D</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+Malaivijitnond%22">Malaivijitnond S</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22B+Siriaroonrat%22">Siriaroonrat B</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22H+Oota%22">Oota H</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+Goto%22">Goto S</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+Kawamura%22">Kawamura S</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i></i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Hiwatashi_T%20[Author]">Hiwatashi T</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Mikami_A%20[Author]">Mikami A</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Katsumura_T%20[Author]">Katsumura T</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Suryobroto_B%20[Author]">Suryobroto B</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Perwitasari-Farajallah_D%20[Author]">Perwitasari-Farajallah D</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Malaivijitnond_S%20[Author]">Malaivijitnond S</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Siriaroonrat_B%20[Author]">Siriaroonrat B</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Oota_H%20[Author]">Oota H</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Goto_S%20[Author]">Goto S</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Kawamura_S%20[Author]">Kawamura S</a>
				</li>
						</ul> 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2148/11/312">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2148/11/312">on Google Scholar</a></li>
                    <li><a href="http://www.biomedcentral.com/pubmed/related/22017819">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	






<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="http://www.biomedcentral.com/1471-2148/11/312/citation">Download references</a>
					</li>
				<li><a href="http://www.biomedcentral.com/content/download/xml/1471-2148-11-312.xml">Download XML</a></li>
				<li>
						<a href="http://www.biomedcentral.com/1471-2148/11/312/email?from=standard">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1471-2148-11-312&amp;volume=11&amp;issue=1&amp;title=Gene+conversion+and+purifying+selection+shape+nucleotide+variation+in+gibbon+L%2FM+opsin+genes&amp;author_name=Tomohide%20Hiwatashi&amp;start_page=1&amp;end_page=14">Order reprints</a></li>
		<li>
						<a href="http://www.biomedcentral.com/1471-2148/11/312/postcomment">Post a comment</a>
					</li>			
		<ul id="download-to-links">
			<a class="downloadto_title btn" href="#"><span class="icon"></span>Download to ...</a>
			
			<ul id="downloadto" class="iconlist hidebeforeload" style="display: none;">
				<li>
					<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F11%2F312" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
				</li>
				<li>
					<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/11/312" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
				</li>
			</ul>
		</ul>
		
		<ul id="mobile-downloadto" class="iconlist">
		
		<h6>Download to ...</h6>
			<li>
				<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F11%2F312" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
			</li>
			
			<li>
				<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/11/312" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
		</ul>

		
	</ul>
		




<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div fb-iframe-plugin-query="action=recommend&amp;app_id=&amp;href=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F11%2F312&amp;layout=button_count&amp;locale=en_GB&amp;sdk=joey&amp;send=false&amp;show_faces=false&amp;width=100" fb-xfbml-state="rendered" class="fb-like fb_iframe_widget" data-href="http://www.biomedcentral.com/1471-2148/11/312" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"><span style="vertical-align: bottom; width: 121px; height: 20px;"><iframe class="" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/like.html" style="border: medium none; visibility: visible; width: 121px; height: 20px;" title="fb:like Facebook Social Plugin" scrolling="no" allowtransparency="true" name="f1b44a3812757cc" width="100px" frameborder="0" height="1000px"></iframe></span></div>
	</li>
	<li>
		<iframe style="width: 107px; height: 20px;" data-twttr-rendered="true" title="Twitter Tweet Button" class="twitter-share-button twitter-tweet-button twitter-share-button twitter-count-horizontal" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/tweet_button.html" allowtransparency="true" scrolling="no" id="twitter-widget-0" frameborder="0"></iframe>
	</li>
	<li>
			<div id="___plusone_0" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 90px; height: 20px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/fastbutton.html" name="I0_1415658012257" id="I0_1415658012257" vspace="0" tabindex="0" style="position: static; top: 0px; width: 90px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 20px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" width="100%" frameborder="0"></iframe></div>
	</li>

        <li>
    </li>
    
		<a class="sharethis_title btn" href="#"><span class="icon"></span>More options...</a>
	<ul style="display: none;" class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2148/11/312"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
				<li>
	</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2148/11/312&amp;title=Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L/M%20opsin%20genes"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="http://www.biomedcentral.com/1471-2148/11/312/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2148/11/312"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div id="___plusone_1" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/fastbutton_002.html" name="I1_1415658012263" id="I1_1415658012263" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" width="100%" frameborder="0"></iframe></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div id="___plusone_2" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/fastbutton_003.html" name="I2_1415658012268" id="I2_1415658012268" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" width="100%" frameborder="0"></iframe></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/11/312"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
			</li>

			<li><a href="http://www.reddit.com/submit" onclick="window.location = 'http://www.reddit.com/submit?url=http://www.biomedcentral.com/1471-2148/11/312'; return false"> <span class="share-icons reddit"></span>Reddit</a>
    </li>






		</ul>
</ul>

    </div>

    
  
        
                
        


















                                                                                
    
    	<div id="biome-badge" style="width: 100%; display: none;">
              <script>

                window.onmessage = function(e) {
                    if(e.data == "biome-failed") {
                        console.log(e);
                        document.getElementById("biome-badge").style.display = "none";
                    }
                };

              </script>


                <iframe src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Gene%20conversion%20and%20purifying%20selection%20shape%20nucleotide%20variation%20in%20gibbon%20L_M%20opsin%20genes_files/biome-widget.html" width="100%" height="75px">Your browser does not support iframes</iframe>


    </div>
		
	

	    
   
  
      
  
                  
  
                  
  
            
  
            
            

<div id="signup-to-etoc" class="button-collection-margins block mobile-border-bottom">	
					<h3>Email updates</h3>
			<p id="receive_issue_alerts_journal_updates_text">Keep up to date with the latest news and content from BMC Evolutionary Biology and BioMed Central.</p>
			
				
<form id="articleSignupForUpdates" method="post" action="/bmcevolbiol/signuptoupdates.html">
	<div id="message-box"></div>
    <fieldset class="block-form">
        
        
        
    </fieldset>
</form>	
</div>

	


			<div style="position: absolute;" id="advert-offset">
    					
		<dl class="google-ad  ">
	<dt class="hide">
				
					<a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
			</dt>
		<dd>

		<!-- OAS AD 'Right3' begin -->
		<script type="text/javascript">
		OAS_AD('Right3');

		</script>
		<!-- OAS AD 'Right3' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    
    <dd>
        <a href="//oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/11/312/19397799820@Right3?">
            
        </a>
    </dd>
			</dl>
		</noscript>
					</div>
	

	
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

</div>








 </div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">            


    
   
  
      
  
                  
  
                  
  
            
  
            
            

<div id="topmatter">
	


	   










				<a href="http://www.biomedcentral.com/about/access"></a>
					<span class="articletype">Research article</span>

<h1>Gene conversion and purifying selection shape nucleotide variation in gibbon L/M opsin genes</h1>
        
			<div class="multipleins">
		
	<p class="authors">
        																																																																
																																																																								
																																																																								
																																																																								
																																									         <strong>Tomohide Hiwatashi</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins1">1</a></sup>, <strong>Akichika Mikami</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins2">2</a></sup><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins8">8</a></sup>, <strong>Takafumi Katsumura</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins1">1</a></sup>, <strong>Bambang Suryobroto</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins3">3</a></sup>, <strong>Dyah Perwitasari-Farajallah</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins3">3</a></sup><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins4">4</a></sup>, <strong>Suchinda Malaivijitnond</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins5">5</a></sup>, <strong>Boripat Siriaroonrat</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins6">6</a></sup>, <strong>Hiroki Oota</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins9">9</a></sup>, <strong>Shunji Goto</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins10">10</a></sup><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins7">7</a></sup> and <strong>Shoji Kawamura</strong><sup><a href="http://www.biomedcentral.com/1471-2148/11/312/#ins1">1</a></sup><sup>*</sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                                                                 
                                                                   
                            		<p class="authors">
							<span>*</span>
									 Corresponding author:										            Shoji  Kawamura <a href="mailto:kawamura@k.u-tokyo.ac.jp">kawamura@k.u-tokyo.ac.jp</a>    					</p>
      
</li>
			
                                                                        
                                                                        
                                                                        
                                                                        
        		</ul>

		
		 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow active"></i>Author Affiliations</a></p>            <div> 
						<div class="collapsible-content">
							
				<div id="ins_container" style="display: block;">
										    						    <p>
						        <sup id="ins1">1</sup>                                
Department of Integrated Biosciences, Graduate School of Frontier 
Sciences, The University of Tokyo, Kashiwa 277-8562, Japan
                            </p>
																    						    <p>
						        <sup id="ins2">2</sup>                                
Department of Behavioral and Brain Sciences, Primate Research Institute,
 Kanrin, Inuyama, Aichi 484-8506, Japan
                            </p>
																    						    <p>
						        <sup id="ins3">3</sup>                                
Department of Biology, Faculty of Mathematics and Natural Sciences, 
Bogor Agricultural University, Kampus IPB Darmaga, Bogor 16680, 
Indonesia
                            </p>
																    						    <p>
						        <sup id="ins4">4</sup>                                
Primate Research Center, Bogor Agricultural University, Jalan Lodaya 
II/5, Bogor 16151, Indonesia
                            </p>
																    						    <p>
						        <sup id="ins5">5</sup>                                
Primate Research Unit, Department of Biology, Faculty of Science, 
Chulalongkorn University, Bangkok 10330, Thailand
                            </p>
																    						    <p>
						        <sup id="ins6">6</sup>                                Research and Conservation Division; Zoological Park Organization (ZPO), Bangkok, Thailand
                            </p>
																    						    <p>
						        <sup id="ins7">7</sup>                                
Center for Human Evolution Modeling Research, Primate Research 
Institute, Kanrin, Inuyama, Aichi 484-8506, Japan
                            </p>
																    						    <p>
						        <sup id="ins8">8</sup>                                Department of Rehabilitaion, Chubu Gakuin University, Kirigaoka 2-1, Seki, Gifu 501-3993, Japan
                            </p>
																    						    <p>
						        <sup id="ins9">9</sup>                                
Department of Anatomy, Kitasato University School of Medicine, Kitasato 
1-15-1, Sagamihara, Kanagawa 252-0374, Japan
                            </p>
																    						    <p>
						        <sup id="ins10">10</sup>                                
Faculty of Veterinary Medicine, Azabu University, Fuchinobe 1-17-71, 
Sagamihara, Kanagawa 252-5301 Japan
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="http://www.biomedcentral.com/bmcevolbiol/logon">log on</a>.
    </p>
						</div>
			</div>
				</div>
</div>
		</div>
	  </div>
<div>
                        <div class="collapsible-content">
        <p><em>BMC Evolutionary Biology</em> 2011, <strong>11</strong>:312
			<span class="pseudotab">doi:10.1186/1471-2148-11-312</span></p>
	    				    
    
        		
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2148/11/312">http://www.biomedcentral.com/1471-2148/11/312</a></p>
		
                    <table cellpadding="0" cellspacing="0">
        <tbody>
        <tr><td>Received:</td><td>26 May 2011</td></tr>                <tr><td>Accepted:</td><td>22 October 2011</td></tr>        <tr><td>Published:</td><td>22 October 2011</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div style="line-height:140%">
        <p>
            © 2011 Hiwatashi et al; licensee BioMed Central Ltd. 
        </p>
					<p>
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</div></div>

</div>
<div id="article-body">
    <div><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height:160%">Routine trichromatic color vision is a characteristic feature of catarrhines (humans,
         apes and Old World monkeys). This is enabled by L and M opsin genes arrayed on the
         X chromosome and an autosomal S opsin gene. In non-human catarrhines, genetic variation
         affecting the color vision phenotype is reported to be absent or rare in both L and
         M opsin genes, despite the suggestion that gene conversion has homogenized the two
         genes. However, nucleotide variation of both introns and exons among catarrhines has
         only been examined in detail for the L opsin gene of humans and chimpanzees. In the
         present study, we examined the nucleotide variation of gibbon (Catarrhini, Hylobatidae)
         L and M opsin genes. Specifically, we focused on the 3.6~3.9-kb region that encompasses
         the centrally located exon 3 through exon 5, which encode the amino acid sites functional
         for the spectral tuning of the genes.
      </p>
      <h4>Results</h4>
      <p style="line-height:160%">Among 152 individuals representing three genera (<em>Hylobates</em>, <em>Nomascus </em>and <em>Symphalangus</em>), all had both L and M opsin genes and no L/M hybrid genes. Among 94 individuals
         subjected to the detailed DNA sequencing, the nucleotide divergence between L and
         M opsin genes in the exons was significantly higher than the divergence in introns
         in each species. The ratio of the inter-LM divergence to the intra-L/M polymorphism
         was significantly lower in the introns than that in synonymous sites. When we reconstructed
         the phylogenetic tree using the exon sequences, the L/M gene duplication was placed
         in the common ancestor of catarrhines, whereas when intron sequences were used, the
         gene duplications appeared multiple times in different species. Using the GENECONV
         program, we also detected that tracts of gene conversions between L and M opsin genes
         occurred mostly within the intron regions.
      </p>
      <h4>Conclusions</h4>
      <p style="line-height:160%">These results indicate the historical accumulation of gene conversions between L and
         M opsin genes in the introns in gibbons. Our study provides further support for the
         homogenizing role of gene conversion between the L and M opsin genes and for the purifying
         selection against such homogenization in the central exons to maintain the spectral
         difference between L and M opsins in non-human catarrhines.
      </p>
   </div>
</div>
<div><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">In catarrhine primates (humans, apes and Old World monkeys) the L and M opsin genes
         are closely juxtaposed on the X chromosome and, in combination with the autosomal
         S opsin gene, enable routinely trichromatic color vision <a name="d21221e224"></a><a name="d21221e226"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>,<a onclick="LoadInParent('#B2'); return false;" href="#B2">2</a>]. The L and M opsin genes have a close evolutionary relationship and are highly similar
         in nucleotide sequence (~96% identity). Among 15 amino acid differences between the
         human L and M opsin genes, three account for the main shifts in spectral sensitivities
         and tuning <a name="d21221e230"></a><a name="d21221e232"></a><a name="d21221e234"></a><a name="d21221e236"></a><a name="d21221e238"></a><a name="d21221e240"></a><a name="d21221e242"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>-<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>].
      </p>
      <p style="line-height:160%">The organization of the L and M opsin genes among humans is known to be variable and
         includes the absence of an L or M opsin gene or the presence of L/M hybrid genes with
         an intermediate spectral sensitivity. A high incidence (approximately 3-8%) of color
         vision "deficiencies" in males results as a consequence <a name="d21221e248"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>]. This variation is caused by unequal meiotic recombination between L and M opsin
         genes. When the nucleotide sequence of a part of one gene appears to be replaced with
         the corresponding sequence of the other, this type of recombination is often called
         gene conversion. Gene conversion is suggested to have occurred frequently between
         the human L and M opsin genes <a name="d21221e252"></a><a name="d21221e254"></a><a name="d21221e256"></a><a name="d21221e258"></a><a name="d21221e260"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>] and to have played a crucial role in generating hybrids of the two genes with altered
         spectral sensitivities <a name="d21221e264"></a><a name="d21221e266"></a><a name="d21221e268"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. Even among individuals with normal color vision, the allele frequency of the L opsin
         gene with Ala at the site 180 instead of Ser is reported to be 30-38% in non-African
         populations <a name="d21221e272"></a><a name="d21221e274"></a><a name="d21221e276"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>].
      </p>
      <p style="line-height:160%">Compared to humans, the incidence of color vision variation is reported to be rare
         in other catarrhines <a name="d21221e282"></a><a name="d21221e284"></a><a name="d21221e286"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>-<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]. Among 744 male long-tailed macaques (<em>Macaca fascicularis</em>) examined, only three were found to have a single L/M hybrid gene with an intermediate
         spectral sensitivity and to be dichromats <a name="d21221e293"></a><a name="d21221e295"></a><a name="d21221e297"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. Among 58 male chimpanzees (<em>Pan troglodytes</em>), one was found to have an L/M hybrid gene with an intermediate spectral sensitivity
         in addition to one normal M opsin gene on the X chromosome and to be a protanomalous
         trichromat <a name="d21221e304"></a><a name="d21221e306"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. Thus, frequencies of color vision variants in male long-tailed macaques and male
         chimpanzees can be calculated to be ~0.4% and ~1.7%, respectively. These frequencies
         could be overestimated because no variants were found in 455 male monkeys from other
         macaque species <a name="d21221e311"></a><a name="d21221e313"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>] and because the chimpanzees examined were from limited numbers of breeding colonies
         <a name="d21221e317"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]. Other studies have reported an absence of color vision defects in Old World monkeys
         and apes <a name="d21221e321"></a><a name="d21221e323"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>].
      </p>
      <p style="line-height:160%">Nevertheless, gene conversion is suggested to have occurred frequently between the
         L and M opsin genes in non-human catarrhines on the basis of the following observations:
         (1) the intraspecific nucleotide divergence between the L and M opsin genes (paralogous
         divergence) tends to be smaller than divergence of the same gene between species (orthologous
         divergence) <a name="d21221e329"></a><a name="d21221e331"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>], (2) allelic polymorphism is often shared between L and M opsin genes and between
         species <a name="d21221e335"></a><a name="d21221e337"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>], (3) paralogous nucleotide divergence in introns and peripheral exons (exons 1 and
         6) is significantly smaller than those in the centrally located exons (exons 2-5),
         which contain the amino acid sites affecting absorption spectra of the L and M photopigments
         <a name="d21221e341"></a><a name="d21221e343"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>,<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>]. These studies suggest that gene conversions at nucleotide sites relevant for the
         spectral difference between the L and M opsins have been effectively eliminated from
         the population by purifying natural selection.
      </p>
      <p style="line-height:160%">If gene conversion occurred frequently between the L and M opsin genes and if purifying
         selection was active in non-human catarrhines, we would also expect another intraspecific
         pattern of nucleotide variation: higher nucleotide divergence between the L and M
         opsin genes in central exons than in introns in addition to lower nucleotide diversity
         within these exons than within introns. However, the within-species nucleotide variation
         of both exons and introns has been evaluated for only the L opsin gene of two African
         hominoids, humans <a name="d21221e349"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>] and chimpanzees (primarily <em>P. t. verus</em>) <a name="d21221e356"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>]. In the present study, we focused on gibbons (Family Hylobatidae), commonly known
         as the lesser apes, for which normal trichromacy is reported <a name="d21221e360"></a>[<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>]. Gibbons occur in Asia and are the most diverse and speciose of all living apes <a name="d21221e364"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>], making them an ideal group with which to assess the range of L/M opsin genetic variation.
         We examined the nucleotide variation of both the L and M opsin genes by sequencing
         the 3.6~3.9-kb genomic region encompassing exon 3 to exon 5 from individuals in five
         species and three genera of gibbons.
      </p>
   </div>
</div>
<div><a name="sec2"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>Gibbon DNA samples</h4>
      <p style="line-height:160%">Blood samples were collected from a total of 157 individuals of the following species:
         Agile (<em>Hylobates agilis</em>; N = 37), Kloss' (<em>H. klossii</em>; N = 2), White-handed (<em>H. lar</em>; N = 40), Silvery Javan (<em>H. moloch</em>; N = 6), Mueller's Bornean gray (<em>H. muelleri</em>; N = 6), Pileated (<em>H. pileatus</em>; N = 19), Chinese White-cheeked (<em>Nomascus leucogenys</em>; N = 16) and Siamang (<em>Symphalangus syndactylus</em>; N = 31). Sampling was conducted at the Ragunan Zoo and the Pontianak Zoo in Indonesia,
         and the Chiang Mai Zoo, the Bangkok Zoo and the Khao Kheow Open Zoo in Thailand. We
         also sampled gibbons reared by local residents in Kalimantan, Indonesia. Genomic DNA
         was extracted from blood samples using the DNA Microextraction Kit (Stratagene, Santa
         Clara, CA) or the QIAamp DNA Blood Mini Kit (Qiagen, Duesseldorf, Germany). Research
         permissions were granted by each country and sampling was conducted according to the
         Guide for the Care and Use of Laboratory Animals by the National Institute of Health,
         U.S.A. (1985) and the Guide for the Care and Use of Laboratory Primates by the Primate
         Research Institute, Kyoto University (1986, 2002). All procedures were approved by
         the animal ethics committee of the Primate Research Institute, Kyoto University.
      </p>
      <p style="line-height:160%">Among the 157 individuals, 152 were subjected to the genotyping of the L/M opsin genes
         (Additional file <a name="d21221e404"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S1). The remaining 5 individuals (two <em>H. agilis</em>, one <em>H. lar</em>, and two <em>S. syndactylus</em>) were included in the analysis of the neutral reference genes. Among the 152 individuals,
         94 were subjected to DNA sequencing of the entire 3.6~3.9-kb region encompassing exon
         3 to exon 5 (Table <a name="d21221e416"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T1">1</a>).
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1"></a><strong>Additional file 1.</strong> <strong>Supplementary tables and figures</strong>. Tables S1, S2, S3, S4, S5 and S6 and Figures S1, S2 and S3.</p><p>
         	Format: PDF 
         	Size: 160KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s1.pdf">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;" href="http://www.adobe.com/products/acrobat/readstep.html">Adobe Acrobat Reader</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T1"><strong>Table 1.</strong></a> The number of gibbon individuals for which the L and M opsin genes were sequenced
      </p></div></div>
      <h4>Genotyping and sequencing of the gibbon L and M opsin genes</h4>
      <p style="line-height:160%">In primates, the L and M opsin genes are arrayed in the same orientation on the X-chromosome
         and separated by approximately 24 kb <a name="d21221e552"></a>[<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. Both genes consist of six exons that encode a protein 364 amino acids long, which
         spans approximately 15 kb <a name="d21221e556"></a><a name="d21221e558"></a><a name="d21221e560"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]. The first (most upstream) position in the gene array is typically occupied by the
         L opsin gene, followed by one or more M opsin genes downstream <a name="d21221e564"></a><a name="d21221e566"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>]. However, the long-range polymerase-chain reaction (PCR) necessary to determine the
         position of an L or M opsin gene in the array is not feasible for a large number of
         samples due to the paucity of gene-specific nucleotide sites and of gene-position
         specific nucleotide sites <a name="d21221e570"></a><a name="d21221e572"></a><a name="d21221e574"></a><a name="d21221e576"></a><a name="d21221e578"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>,<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>-<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>]. We thus identified a gene not by the position in the gene array, but rather by its
         inferred spectral property.
      </p>
      <p style="line-height:160%">The peak absorption spectra (λmax) of the catarrhine L and M opsin photopigments are
         approximately 560 nm and 530 nm, respectively <a name="d21221e584"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. This spectral difference is mainly attributed to the amino acid differences at three
         sites: the residue 180 encoded in the exon 3 and the residues 277 and 285 in the exon
         5 <a name="d21221e588"></a><a name="d21221e590"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>,<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>]. Amino acid changes from Ser to Ala at the site 180, (denoted Ser180Ala), Tyr277Phe,
         and Thr285Ala shift the λmax values by -7, -8, and -15 nm respectively, in nearly
         an additive manner, and the reverse amino acid changes cause opposite spectral shifts
         by approximately the same extent <a name="d21221e594"></a><a name="d21221e596"></a><a name="d21221e598"></a><a name="d21221e600"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>-<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a>]. Therefore, the contribution of exon 5 to the difference of absorption spectra between
         the L and M opsins is greater than that of exon 3.
      </p>
      <p style="line-height:160%">Based on the inferred absorption spectra, we defined the gibbon L opsin gene as having
         Tyr and Thr and the M opsin gene as having Phe and Ala at the residues 277 and 285,
         respectively. This differs slightly from previous studies in which gene identity is
         also based on other nucleotide differences with little or no spectral effects <a name="d21221e606"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>]. We defined the spectral hybrid as having Ala at the reside 180 in the exon 3 of
         the L opsin gene and as having Ser at the exon 3 of the M opsin gene.
      </p>
      <p style="line-height:160%">The average size of the amplified region among the M opsin sequences was approximately
         3.6 kb in each species, while that of the L opsin gene was approximately 3.9, 3.8
         and 3.6 kb in the <em>Hylobates </em>species, <em>N. leucogenys </em>and <em>S. syndactylus</em>, respectively. The size difference of the genomic region between the L and M opsin
         genes was due to an Alu element in the intron 3 of the L opsin gene. We isolated this
         genomic region by two sets of PCR schemes.
      </p>
      <p style="line-height:160%">For the first PCR set, we designed an L-specific and an M-specific reverse primer,
         Ex5L_Rev (5'- CCCCAGCAGACGCAGTACGCAAAGAT -3') and Ex5M_Rev (5'- CCCCAGCAGAAGCAGAATGCCAGGAC
         -3'), at a region including residue 277 and five surrounding nucleotide sites at which
         the human L (GenBank Accession number <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=Z68193">Z68193</a>) and M (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=AC092402">AC092402</a>) opsin genes differ. This primer site does not contain the residue 285, which is
         attributable to the relatively few nucleotide differences reported between L and M
         immediately downstream of residue 285. Combined with this, a forward primer common
         to the human L and M opsin genes was set in intron 2, just upstream of exon 3 (Ex3_For:
         5'-GGATCACAGGTCTCTGGTCTCTG-3'). The nucleotide sequence of exon 3 was then determined
         for the PCR products using a sequencing primer common to the L and M opsin genes (Ex3_Rev:
         5'-GAGCGTGCAATGTCTATCAA-3') located in intron 3. The nucleotide differences in the
         region used for the gene-specific reverse primers and in the residue 285 downstream
         of the primers in the exon 5 were confirmed by a second PCR set.
      </p>
      <p style="line-height:160%">Our second PCR set was comprised of a reverse primer common to the human L and M opsin
         genes. It was designed in intron 5 just downstream of exon 5 (Ex5_Rev: 5'-GAAATGACCGGGAAAGGCTC-3').
         An L- and an M-specific forward primer were designed in exon 3 at a region upstream
         of the site 180, Ex3L_For (5'-TTTCCTGGGAGAGGTGGCTGGTGGTG-3') and Ex3M_For (5'-TTTCCTGGGAGAGATGGATGGTGGTC-3'),
         so as to include three differing nucleotide sites between the published human L and
         M opsin genes. The nucleotide sequence of exon 5 was then determined for the PCR product
         using a common sequencing primer between the L and M opsin genes (Ex5_For: 5'-CACTCAGGGCTGGAAGATGGC-3')
         designed in intron 4. The nucleotide differences in the region used for the gene-specific
         forward primers in exon 3 were confirmed by the first PCR set described above.
      </p>
      <p style="line-height:160%">By using the combination of these two PCR sets, we avoided the possible misinterpretation
         of the presence or absence of a gene type by false-positive or false-negative amplification
         by the gene-specific primers. This is an improvement from previous studies relying
         on the success of PCR using gene-specific primers <a name="d21221e633"></a><a name="d21221e635"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>].
      </p>
      <p style="line-height:160%">PCRs were carried out in 25 μl containing 1.5 units of Ex Taq polymerase hot start
         version (Takara, Tokyo) with 1× Ex Taq Buffer, 0.2 mM each of dNTPs, 1 μM each of
         the forward and reverse primers, and ca. 5 ng of the gibbon genomic DNA at 94°C for
         10 min followed by 35 cycles at 94°C for 30 sec, 65°C for 30 sec, and 72°C for 4 min.
         Distilled water was used as the template for the negative control in every reaction.
         The amplification was confirmed by 0.5% agarose-gel electrophoresis and the amplified
         DNA fragments were purified using the Montage PCR Centrifugal Filter Device (Millipore,
         Tokyo).
      </p>
      <p style="line-height:160%">Pruified DNA samples were directly sequenced using Applied Biosystems model 3130 automatic
         sequencer with Big Dye Terminator v3.1 Cycle Sequencing kit (Applied Biosystems Japan,
         Tokyo) and sequencing primers (Ex3_Rev, Ex5_For and other primers set inside the amplified
         region). Operationally, we first sequenced the L and M opsin genes of both DNA strands
         from a sample of a male as only one X-chromosome is present. We sequenced only one
         strand for other samples if there was no nucleotide difference from the ones for which
         both strands were sequenced. When we found nucleotide differences in two or more individuals
         at the same site, we sequenced both strands of at least one of the samples to confirm
         the variation. When we found a different nucleotide in only one sample, <em>i.e</em>. as a singleton among the samples, we repeated the PCR for the sample and sequenced
         the region for both strands to rule out a PCR error and to confirm the variation.
      </p>
      <p style="line-height:160%">When a sample from a male showed two nucleotide peaks at one or more sites ("heterozygous"
         sites) in the L or M opsin gene, the individual should have two or more loci of the
         gene with different sequences. In this case, we conservatively inferred two loci for
         this gene. When a female showed heterozygous sites, the individual could have allelic
         differences or two or more copies of the gene. We regarded such females as having
         one locus with different alleles for this opsin type because this arrangement gives
         the minimum possible and the most conservative estimate of the number of sequences.
         Thus, the sequence number could be underrepresented in our results and one gene group
         (<em>i.e</em>. the L or M opsin group) does not necessarily consist of orthologous members. When
         we confined an analysis to the orthologous gene groups, we used only male samples
         with single L and M opsin genes.
      </p>
      <p style="line-height:160%">We did not attempt to separate the sequences by DNA cloning. In the alignment of the
         nucleotide sequences (Additional files <a name="d21221e653"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S2">2</a>, <a name="d21221e656"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S3">3</a>, <a name="d21221e659"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S4">4</a>, <a name="d21221e662"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S5">5</a> and <a name="d21221e665"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S6','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S6">6</a>), we treated these sequences as two alternate gene sequences and arbitrarily assigned
         the nucleotides at the heterozygous sites into the two sequences. For calculations
         of the nucleotide diversity within a gene group and the nucleotide divergence between
         gene groups (see below for details), the nucleotide combination at the heterozygous
         sites (haplotype) is not necessary. When we conducted an analysis requiring the haplotype
         information, such as the GENECONV procedure (described below) or the evaluation of
         linkage disequilibrium, we used only male samples with single L and M opsin genes.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S2"></a><strong>Additional file 2.</strong> <strong>Alignment of the gibbon L and M opsin gene exon 3</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon L and M
         opsin gene exon 3.</p><p>
         	Format: TXT 
         	Size: 53KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s2.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S3','S3',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S3"></a><strong>Additional file 3.</strong> <strong>Alignment of the gibbon L and M opsin gene intron 3</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon L and M
         opsin gene intron 3.</p><p>
         	Format: TXT 
         	Size: 508KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s3.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S4','S4',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S4"></a><strong>Additional file 4.</strong> <strong>Alignment of the gibbon L and M opsin gene exon 4</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon L and M
         opsin gene exon 4.</p><p>
         	Format: TXT 
         	Size: 52KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s4.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S5','S5',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S5"></a><strong>Additional file 5.</strong> <strong>Alignment of the gibbon L and M opsin gene intron 4</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon L and M
         opsin gene intron 4.</p><p>
         	Format: TXT 
         	Size: 441KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s5.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S6','S6',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S6"></a><strong>Additional file 6.</strong> <strong>Alignment of the gibbon L and M opsin gene exon 5</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon L and M
         opsin gene exon 5.</p><p>
         	Format: TXT 
         	Size: 72KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s6.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div></div>
      <h4>Nucleotide sequencing of the neutral reference regions</h4>
      <p style="line-height:160%">The eta-globin gene is suitable as a neutral reference because it is a single copy
         pseudogene well characterized for primates <a name="d21221e735"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>] and the ortholog for a gibbon (<em>H. lar</em>) has been reported (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=M54985">M54985</a>) <a name="d21221e745"></a>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>]. A pair of PCR primers was designed according to the gibbon sequence to amplify ca.
         0.5 kb in the 5'-flanking region that is free of repeat sequences: forward, 5'-AGAGGAGAAGTAAAAAGCCAC-3';
         reverse, 5'-ACATAACTCTCAAAATCCCAC-3'. The S opsin gene intron 4 was also used as a
         neutral reference. A pair of PCR primers was designed according to the human S opsin
         sequence (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=L32835">L32835</a>) to amplify ca. 0.5 kb that is free of repeat sequences: forward, 5'-CCCTGCCAACTTTTAGCTTGCAC-3';
         reverse, 5'-TTCCCGCACCATCTCCATGAT-3'.
      </p>
      <p style="line-height:160%">The PCRs were carried out with the same composition of the reaction mixture as used
         for the L/M opsin gene. Purification and sequencing were carried out as previously
         described for the L/M opsin gene. As was the case with the L/M opsin gene, we did
         not clone the neutral references and thus the two allelic DNA sequences of individuals
         were not separated. In the alignment (Additional files <a name="d21221e753"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S7','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S7">7</a>, <a name="d21221e756"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S8','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S8">8</a>), two heterozygous alleles were tentatively sorted into two allele sequences.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S7','S7',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S7"></a><strong>Additional file 7.</strong> <strong>Alignment of the gibbon eta-globin pseudogene 5' flanking region</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon eta-globin
         pseudogene 5' flanking region.</p><p>
         	Format: TXT 
         	Size: 168KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s7.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S8','S8',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S8"></a><strong>Additional file 8.</strong> <strong>Alignment of the gibbon S opsin gene intron 4</strong>. A "sequential" (fasta) format of the sequence alignments for the gibbon S opsin
         gene intron 4.</p><p>
         	Format: TXT 
         	Size: 148KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-11-312-s8.txt">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"></a></span></p></div></div>
      <p style="line-height:160%">In each of the five gibbon species, the nucleotide sequences of the L and M opsin
         genes from one male sample with single L and M opsin genes have been deposited to
         the GenBank/EMBL/DDBJ database. The nucleotide sequences of the eta-globin 5' flanking
         and the S opsin intron 4 sequences were also deposited to the database from one individual
         each from the five species who had no heterozygous sites. Their sequence ID in this
         study (see also Additional files <a name="d21221e785"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S2">2</a>, <a name="d21221e788"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S7','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S7">7</a>, <a name="d21221e791"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S8','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S8">8</a>) and the accession numbers in the database are listed in Table <a name="d21221e794"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T2">2</a>.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T2"><strong>Table 2.</strong></a> The GenBank/EMBL/DDBJ accession numbers of gibbon L and M opsin genes
      </p></div></div>
      <h4>Data analysis</h4>
      <p style="line-height:160%">Alignment of the nucleotide sequences was conducted using the Clustal W software <a name="d21221e1107"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>] and was adjusted manually. The average number of between-group nucleotide differences
         per nucleotide site was designated the nucleotide divergence (<em>d</em>). The <em>d </em>value between the L and M opsin genes within a species was calculated by eliminating
         all positions containing gaps from the alignment. The average number of within-group
         nucleotide differences per nucleotide site between two sequences was designated the
         nucleotide diversity (π) and was calculated by eliminating all positions containing
         gaps from the alignment. The average numbers of between- and within-group synonymous
         differences per synonymous site and non-synonymous differences per non-synonymous
         site were calculated using the Nei-Gojobori method <a name="d21221e1117"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. We connected the alignments of the eta-globin pseudogene and the S opsin gene intron
         4 to calculate the π value of the combined neutral references for the individuals
         for which both genes were sequenced. The standard errors of the π and <em>d </em>values including those of their synonymous and non-synonymous components were estimated
         by a bootstrap procedure (1000 replicates). All of these computations were conducted
         using MEGA version 5 <a name="d21221e1125"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>]. The statistical significance of the difference between two <em>d </em>value or two π values was evaluated by the one-tailed Z test.
      </p>
      <p style="line-height:160%">For constructing an among-group phylogenetic tree, we corrected the <em>d </em>values for multiple substitutions by the Jukes-Cantor formula <a name="d21221e1137"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>] and used the neighbor-joining method <a name="d21221e1141"></a>[<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>] by eliminating all positions containing gaps from the alignment. The exon and intron
         sequences of a human L (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=Z68193">Z68193</a>) and M (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=AC092402">AC092402</a>) opsin genes and the exon sequence of a crab-eating macaque L (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=AF158968">AF158968</a>) and M (GenBank <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=AF158975">AF158975</a>) opsin genes <a name="d21221e1158"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>] were also included as well as the exon sequence of a mouse M opsin gene (GenBank
         <a target="_blank" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=AF011389">AF011389</a>) as an outgroup. In the case of the human, macaque and mouse sequences, one gene
         sequence was regarded as one group. The reliability of the topology of the phylogenetic
         tree was evaluated by bootstrap resampling (1000x) using the "sendbs2" program <a name="d21221e1165"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>]. To compute the bootstrap values, the human, macaque and mouse sequences were input
         twice in the alignment because the sendbs2 does not accept one sequence as one group.
      </p>
      <p style="line-height:160%">For testing gene conversion, we applied the GENECONV 1.81 program <a name="d21221e1171"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>] implementing Sawyer's statistical method <a name="d21221e1175"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. GENECONV searches for a consecutively identical region between two sequences and
         evaluates the probability of achieving homogeneity when nucleotide differences are
         assumed to be randomly distributed throughout the sequence region.
      </p>
   </div>
</div>
<div><a name="sec3"></a><h3>Results</h3>
   <div class="collapsible-content">
      <h4>L/M opsin genotyping</h4>
      <p style="line-height:160%">On the basis of the amino acid composition at site 180 encoded in the exon 3 and 277
         and 285 encoded in exon 5, the L/M opsin genotype was determined for 152 individuals
         of gibbons from the eight species (Additional file <a name="d21221e1188"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S1). All individuals had standard L and M opsin genes. No L/M hybrid genes
         were found in terms of the three-site composition.
      </p>
      <h4>Multiple M opsin copies</h4>
      <p style="line-height:160%">The nucleotide variation was analyzed for five species, <em>H. agilis</em>, <em>H. lar, H. pileatus</em>, <em>N. leucogenys </em>and <em>S. syndactylus </em>for the genomic region encompassing the exon 3 through the exon 5, approximately 3.6~3.9
         kb. We excluded <em>H. klossii</em>, <em>H. moloch </em>and <em>H. muelleri </em>from the subsequent analysis because of their small sample size. A total of 94 individuals
         including 51 males were subjected for the analysis (Table <a name="d21221e1219"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T1">1</a>). Among the males, there were no individuals detected that had multiple L opsin genes.
         On the other hand, multiple M opsin genes were detected in 80.0% of <em>N. leucogenys</em>, 28.6% of <em>H. lar</em>, 16.7% of <em>H. agilis</em>, and 10.0% of <em>H. pileatus </em>and <em>S. syndactylus </em>males.
      </p>
      <h4>Nucleotide divergence between L and M opsin genes within species</h4>
      <p style="line-height:160%">Figure <a name="d21221e1244"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> summarizes the nucleotide divergence between the L and M opsin genes in the exons
         and introns in the five gibbon species (Additional file <a name="d21221e1247"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S2 for the values). Exons were generally more divergent than introns. In particular,
         the divergence of exons 3 and 5 was significantly higher than that in the introns.
         Within the exons, however, non-synonymous divergence was lower or not significantly
         higher than synonymous divergence in any of the three exons in all five species (Additional
         file <a name="d21221e1250"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S3). Both synonymous and non-synonymous divergences of exons were significantly
         larger than the intron divergence in all five species (Figure <a name="d21221e1253"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a>).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1"><strong>Figure 1.</strong></a> <strong>The nucleotide divergence (<em>d</em>) between the L and M opsin genes in five species of gibbons, <em>H. agilis</em>, <em>H. lar</em>, <em>H. pileatus</em>, <em>N. leucogenys </em>and <em>S. syndactylus</em></strong>. The <em>d </em>values of the exons are represented by black, the introns by white, the synonymous
         sites by light gray, and the non-synonymous sites by dark gray bars. The error bars
         indicate the estimated standard error of the <em>d </em>values based on the 1000 × bootstrap resampling. The asterisks indicate that the <em>d </em>values are significantly higher than the <em>d </em>value of the combined sequence of the introns 3 and 4 in each species. The single
         and double asterisks represent the statistical significance at 0.05 and 0.01 levels,
         respectively, based on the one-tailed Z test. Ex3, exon 3; Ex4, exon 4; Ex5, exon
         5, Int3, intron 3; Int4, intron 4; Int, the introns 3 and 4 combined; S, synonymous
         sites in the exons 3, 4 and 5; N, non-synonymous sites in the exons 3, 4 and 5.
      </p></div></div>
      <p style="line-height:160%">We also examined male samples with a single M as well as a single L opsin sequence
         types. There were ten <em>H. agilis</em>, ten <em>H. lar</em>, nine <em>H. pileatus</em>, one <em>N. leucogenys</em>, and nine <em>S. syndactylus </em>males that had single L and M opsin genes. We excluded the one <em>N. leucogenys </em>from the calculation because of its inapplicability to the population analysis. As
         shown in Additional file <a name="d21221e1338"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1, the pattern of nucleotide divergence between the L and M opsin genes
         in the exons and introns in these individuals was essentially the same as the pattern
         shown in Figure <a name="d21221e1341"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a>.
      </p>
      <h4>Nucleotide diversity of L and M opsin genes within species</h4>
      <p style="line-height:160%">Figure <a name="d21221e1350"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a> summarizes the nucleotide diversity of the L and M opsin exons and introns and of
         the neutral references (see Additional file <a name="d21221e1353"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Tables S4 and S5 for the values). As for the neutral references, we also incorporated
         the nucleotide diversity reported by Kim et al (2011) <a name="d21221e1357"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>] for the30~75 kb of autosomal regions and the 2~13 kb of X chromosomal regions of
         these gibbon species. In Figure <a name="d21221e1360"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a>, the nucleotide diversity values of the autosomal neutral references are adjusted
         (multiplied by 3/4) for comparison with X-chromosomal L/M opsin genes (see Discussion).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2"><strong>Figure 2.</strong></a> <strong>The nucleotide diversity (π) of the exons (black bar) and the introns (white bar)
            of the L and M opsin genes and the neutral references (gray bars) in the five species
            of gibbons</strong>. The gibbon genomic data reported for autosomal (G<sub>A</sub>) and X-chromosomal (G<sub>X</sub>) regions <a name="d21221e1382"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>] are also indicated as neutral references. The π values of the combined sequences
         of the eta globin pseudogene and the S opsin intron 4 (Eta+S) and the π values of
         G<sub>A </sub>are multiplied by 3/4. The single and double asterisks represent the statistical significance
         at 0.05 and 0.01 levels, respectively, based on the one-tailed Z test.
      </p></div></div>
      <p style="line-height:160%">In both the L and M opsin genes, contrary to the case with the nucleotide divergence,
         the nucleotide diversity of the exons was lower overall than that of the introns and
         in many cases was even zero (Additional file <a name="d21221e1391"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S4). Alternatively, introns were as variable as the neutral references (Figure
         <a name="d21221e1394"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a>). When the three exons and the two introns were combined respectively, the exons
         were significantly less variable than the introns (Figure <a name="d21221e1397"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a>). When using only males with single L and M opsin genes, the pattern of nucleotide
         diversity of L and M opsin genes (Additional file <a name="d21221e1400"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S2) was also the same with that shown in Figure <a name="d21221e1403"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a>.
      </p>
      <p style="line-height:160%">Within the exons, variation consisted mainly of synonymous variation (Additional file
         <a name="d21221e1408"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S3 and Table S6). The synonymous variation was as high as, or lower than,
         the intron variation. However, the synonymous variation showed a large stochastic
         error compared to the nucleotide diversity in the introns (Figure S3) and the synonymous
         nucleotide divergence (Figure <a name="d21221e1411"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a>). This is because of the much smaller number of the synonymous nucleotide sites (137~138
         bp) in the coding region than in the total length of the introns (3.0~3.3 kb) and
         the lower level of synonymous nucleotide diversity (0~0.4%) than the synonymous divergence
         between the L and M opsin genes (6.1~6.4%).
      </p>
      <h4>Comparison of inter-LM divergence and intra-L/M polymorphism between synonymous sites
         and introns
      </h4>
      <p style="line-height:160%">Using the male samples with single L and M opsin genes, we counted the number of sites
         fixed with different nucleotides between the L and M opsin genes (inter-LM divergence)
         and the number of polymorphic nucleotide sites in the L, M or both genes within a
         species (intra-L/M polymorphism). We then categorized these sites into non-synonymous
         and synonymous sites in the exons and intron sites. As shown in Table <a name="d21221e1420"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T3">3</a>, the relative amount of divergence to polymorphism was significantly smaller in introns
         than in exon synonymous sites in all of the four species examined (<em>P</em>&lt;0.02, Fisher's exact test). On the other hand, between non-synonymous and synonymous
         sites within exons, there was no significant difference detected in the relative amount
         of divergence to polymorphism (data not presented).
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T3"><strong>Table 3.</strong></a> Comparison of inter-LM divergence and intra-L/M polymorphism between synonymous sites
         and introns in gibbons
      </p></div></div>
      <h4>Among-group phylogenetic tree</h4>
      <p style="line-height:160%">By combining the three exons and regarding one gene type (<em>i.e</em>. L or M opsin) in a species as a group, an among-group phylogenetic tree was constructed
         based on the nucleotide divergence between the groups (Figure <a name="d21221e1646"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3A</a>). The L and M opsin genes were separated at the common ancestor of the human, gibbons
         and macaque. In contrast, when the two introns were combined and an among-group phylogenetic
         tree was constructed for gibbons and human, the L and M opsin groups were clustered
         in <em>N. leucogenys</em>, <em>S. syndactylus</em>, human, and in the genus <em>Hylobates </em>(Figure <a name="d21221e1659"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3B</a>). In both the exon and intron trees, <em>Nomascus </em>was placed at the most basal position among the gibbons (Figure <a name="d21221e1665"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3</a>), being consistent with the current knowledge of the gibbon molecular phylogeny <a name="d21221e1669"></a><a name="d21221e1671"></a><a name="d21221e1673"></a><a name="d21221e1675"></a><a name="d21221e1677"></a>[<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>-<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>].
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3"><strong>Figure 3.</strong></a> <strong>The among-group phylogenetic trees of exons (A) and introns (B) of the L/M opsin genes
            of gibbons</strong>. (A) The combined sequences of exons 3, 4 and 5 are considered. The human L and M,
         the crab-eating macaque L and M, and the mouse M opsin gene sequences are included.
         (B) The combined intron 3 and 4 sequences are considered. The human L and M opsin
         gene sequences are included. Bootstrap values over 80% are indicated at the branch
         nodes. Scale bars indicate the number of nucleotide substitution per site. Hag, <em>H. agilis</em>; Hla, <em>H. lar</em>; Hpi, <em>H. pileatus</em>; Nle, <em>N. leucogenys</em>; Ssy, <em>S. syndactylus</em>.
      </p></div></div>
      <h4>Test of gene conversion by GENECONV</h4>
      <p style="line-height:160%">For the GENECONV analysis we considered only male samples including the one <em>N. leucogenys </em>sample with single L and M opsin genes. We detected tracts of the gene conversion
         in each of the five species predominantly in intron regions with the global <em>P </em>values of less than 0.05 based on permutations for the entire alignment of the 78
         sequences and corrected for multiple comparisons (Figure <a name="d21221e1720"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F4">4</a>).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F4"><strong>Figure 4.</strong></a> <strong>Distribution of gene conversions between the L and M opsin genes in gibbons detected
            by GENECONV</strong>. The top line represents the exon-intron structure of the L/M opsin genes. The arrows
         represent the area covered by gene conversions detected for pairs of L and M opsin
         genes in the five gibbon species (see Figure 3 legend for the abbreviations of species
         names).
      </p></div></div>
   </div>
</div>
<div><a name="sec4"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">In the present study we examined intraspecific genetic variation of gibbon L and M
         opsin genes. This study also enabled comparison of the intraspecific nucleotide diversity
         between the L/M opsin gene regions and other autosomal regions as neutral references.
         In theory, when mutations are selectively neutral and population size is constant
         through generations, the nucleotide diversity is expected to be equal to the population
         mutation rate (4<em>N<sub>e</sub></em>μ for autosomal and 3 <em>N<sub>e</sub></em>μ for X-chromosomal genes of diploid organisms, where <em>N<sub>e </sub></em>is the effective population size and μ is the mutation rate per nucleotide site per
         chromosome per generation) <a name="d21221e1758"></a>[<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>]. The nucleotide diversity of the gibbon neutral references was in the range of 0.10~0.41%
         (0.28% on average) when the two references were combined (Additional file <a name="d21221e1761"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S5). These values are comparable to the nucleotide diversity of gibbon genomes
         recently reported by Kim et al. (2011) <a name="d21221e1766"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>]; 0.16~0.29% (0.25% on average) in the autosomal regions and 0.04~0.18% (0.11% on
         average) in the X chromosomal regions of the same five species examined in this study
         (Figure <a name="d21221e1769"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a> and Additional file <a name="d21221e1772"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S5). Consistent with Kim et al. (2011) <a name="d21221e1776"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>], they are also higher overall than reported nucleotide diversity of autosomal genes
         in humans (0.12%), chimpanzees (0.19%), bonobos (0.10%) and gorillas (0.15%) and are
         comparable with that in orangutans (0.36%) <a name="d21221e1780"></a><a name="d21221e1782"></a><a name="d21221e1784"></a>[<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>-<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>], suggesting a generally larger <em>N<sub>e </sub></em>in Asian apes than in African apes. Although, our samples were collected from zoos
         and privately owned animals, the consistency with the large genome data set <a name="d21221e1794"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>] implies that these samples could fairly represent the diversity of a natural population.
         The high nucleotide diversity of the reference genes also safely precludes the possibility
         that our samples are biased to a close kin group.
      </p>
      <p style="line-height:160%">All 152 individuals from the eight species examined had both of the normal L and M
         opsin genes. Based on the composition of the three amino acid sites relevant to absorption
         spectra, there was no L/M hybrid gene detected with an intermediate spectral sensitivity.
         This observation is consistent with previous studies on the genetic variation of the
         L and M opsin genes in non-human catarrhines reporting no or rare observation of the
         gene deletion or the hybrid genes <a name="d21221e1800"></a><a name="d21221e1802"></a><a name="d21221e1804"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>-<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>].
      </p>
      <p style="line-height:160%">Ninety four individuals from five species, including 51 males, were subjected to sequencing
         of the 3.6~3.9-kb region encompassing exon 3 through exon 5. Among them, plural M
         opsin genes were detected in 10-80% of males in each species (23.5% of all males).
         Only a single L opsin sequence was detected in all males. Although more data are desired,
         these results are comparable to humans where multiple M copies are found in 66% of
         males of European origin <a name="d21221e1810"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>] and 56% of Japanese males <a name="d21221e1814"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>] and more than one L copy is rarely found <a name="d21221e1818"></a>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a>]. Regarding non-human catarrhines, studies concur that a single L opsin copy is generally
         found in the genomes whereas some studies report that multiple M copies are rare <a name="d21221e1822"></a><a name="d21221e1824"></a><a name="d21221e1826"></a><a name="d21221e1828"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>] yet other studies report that they are common <a name="d21221e1832"></a><a name="d21221e1834"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Thus, among Old World monkeys and apes, there seems to be no clear trend on the
         copy number variation of M opsin gene. There also seems to be no clear correlation
         between the frequency of X chromosomes carrying multiple copies of the M opsin gene
         with and the frequency of L or M opsin gene deletion or the hybridization.
      </p>
      <p style="line-height:160%">In this study, we defined the L and M opsin genes on the basis of their deduced amino
         acid composition at residues 277 and 285 and not on the basis of their position in
         the L/M opsin gene array on the chromosome. As explained in the Methods, the sequence
         numbers of the L and M opsin gene groups could be underrepresented and the nucleotide
         diversity and divergence values for the gene groups could therefore be overestimated.
         Conceptually, the gene group in this study does not necessarily correspond to the
         "orthologous" group because it allows inclusion of duplicated copies. Nevertheless,
         when we only included the male samples with single L and M opsin genes to confine
         the analysis to the orthologous gene groups, the pattern of nucleotide divergence
         and diversity was indistinguishable from that where all samples were used (compare
         Figure <a name="d21221e1839"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> with Additional file <a name="d21221e1842"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1, and Figure <a name="d21221e1845"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a> with Additional file <a name="d21221e1848"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S2). Thus, there appears to be little or no distortion made in the pattern
         of nucleotide diversity and divergence by our collective treatment of the gene groups.
      </p>
      <p style="line-height:160%">The within-species nucleotide divergence between L and M opsin genes was significantly
         higher in the exons than in the introns (Figure <a name="d21221e1853"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1856"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1 and Table S2). This is opposite to what is expected for the general pattern
         of divergence: <em>i.e</em>. the protein-coding exons are generally more conservative than introns. Within the
         exons, however, non-synonymous divergence was not significantly higher than synonymous
         divergence in any of the three exons in the five species (Figure <a name="d21221e1862"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1865"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1 and Table S3). This is an orthodox pattern of nucleotide divergence in
         coding regions under a functional constraint <a name="d21221e1870"></a>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>]. It is hence unlikely that the higher divergence in exons than in introns is due
         to divergent natural selection between the two genes acting on the amino acid sequence.
         The intron divergence was significantly lower than both of the synonymous and non-synonymous
         divergences in all of the five species (Figure <a name="d21221e1873"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1876"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1). The low divergence in introns is likely due to a homogenizing process
         operating between the two genes, <em>i.e</em>. gene conversion, and the purifying selection against the gene conversion in the
         exons.
      </p>
      <p style="line-height:160%">Contrary to the pattern of nucleotide divergence, in both L and M opsin genes the
         within-species nucleotide diversity was lower overall in exons than in introns (Figure
         <a name="d21221e1884"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a> and Additional file <a name="d21221e1887"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S2 and Table S4). Within the exons, the variation was largely synonymous
         (Figure S3 and Additional file <a name="d21221e1890"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Table S6). This is a typical pattern of nucleotide variation in protein-coding exons
         and introns. The nucleotide diversity of the introns was about the same magnitude
         with the nucleotide diversity of the autosomal neutral references multiplied by 3/4
         and that of the X-chromosomal references in each species (Figure <a name="d21221e1893"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F2">2</a> and Additional file <a name="d21221e1896"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S2 and Table S5). Therefore, gene conversion in the introns does not seem
         to affect the nucleotide diversity, possibly because the numbers of introduced and
         erased mutations by the gene conversion are canceled out.
      </p>
      <p style="line-height:160%">The contrasting pattern of the nucleotide divergence and diversity between exons and
         introns was further manifested by comparing the relative amount of the inter-LM divergence
         to the intra-L/M polymorphism between the synonymous sites and the introns. The ratio
         in the introns was significantly lower than that in the synonymous sites (Table <a name="d21221e1901"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/table/T3">3</a>). Since the synonymous sites and introns are both expected to be under selective
         neutrality, this pattern is consistent with the homogenization between the L and M
         opsin genes by gene conversion in introns. Within the exons, the ratio was not significantly
         different between synonymous and non-synonymous sites: in both nucleotide divergence
         and diversity the synonymous variation was generally larger than the non-synonymous
         variation (Figure <a name="d21221e1904"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1907"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figures S1 and S3). This suggests that the non-synonymous sites are under selective
         constraint and the non-synonymous differences between the two genes are protected
         by purifying selection in conjunction with neighboring synonymous differences from
         the homogenization by exon gene conversions.
      </p>
      <p style="line-height:160%">The among-group phylogenetic tree for the L and M opsin genes showed a contrasting
         cluster pattern between cases using exon versus intron regions (Figure <a name="d21221e1912"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3</a>). In the exon tree, gene duplication appears to have occurred once in the common
         ancestor of hominoids (gibbons and human) and Old World monkeys (crab-eating macaque)
         (Figure <a name="d21221e1915"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3A</a>), whereas in the intron tree, gene duplication appears to have occurred independently
         in <em>N. leucogenys</em>, <em>S. syndactylus</em>, a common ancestor of the <em>Hylobates </em>species, and humans (Figure <a name="d21221e1928"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F3">3B</a>). If the phylogenetic representation of the exon tree reflects the true relationships
         among the gene groups, the topology of the intron tree can be explained by the gene
         conversion between the duplicated L and M opsin genes at the individual level and
         by natural selection against the gene conversion involving exons at the population
         level. On the other hand, if the intron tree reflects the true relationships, the
         topology of the exon tree is inexplicable.
      </p>
      <p style="line-height:160%">Tracts of gene conversions were assessed by the GENECONV statistical method (Figure
         <a name="d21221e1933"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F4">4</a>). This method assesses whether the nucleotide differences found between two sequences
         are randomly distributed along the length of these sequences; it cannot detect gene
         conversions if a given region has fewer differences than its flanking regions. It
         has been advised that other methods, such as phylogenetic trees, have to be used to
         identify gene conversions (<em>e.g</em>. see <a name="d21221e1940"></a>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>]). Our GENECONV result is regarded consistent with the other results since the detected
         gene conversions were located mostly in the intron areas (Figure <a name="d21221e1943"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F4">4</a>). In <em>S. syndactylus</em>, the detected gene conversion included the most part of the exon 4 (Figure <a name="d21221e1950"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F4">4</a>). Considering that exon 4 contributes much less to the spectral difference between
         the L and M opsins than the exons 3 and 5 <a name="d21221e1954"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>] and that the nucleotide divergence between the L and M opsin genes in the exon 4
         is smaller than those in the exons 3 and 5 (Figure <a name="d21221e1957"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1960"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1), the GENECONV result suggest that the purifying selection against gene
         conversion is weaker in the exons 4 than in the exons 3 and 5.
      </p>
      <p style="line-height:160%">Throughout this study, we found no signature of positive (either directional or balancing)
         selection acting on the gibbon L and M opsin genes. As shown in Figure <a name="d21221e1965"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/figure/F1">1</a> and Additional file <a name="d21221e1968"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/11/312/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/11/312/suppl/S1">1</a>, Figure S1, the nucleotide divergence between the L and M opsin genes at non-synonymous
         sites was always smaller than that at synonymous sites. The nucleotide divergence
         between species for the L opsin genes or the M opsin genes at non-synonymous sites
         was also smaller or not significantly higher than that at synonymous sites (data not
         presented). Using the male samples with single L and M opsin genes (<em>i.e</em>. the "orthologous" sequence groups of the L and M opsin genes) of <em>H. agilis</em>, <em>H. lar</em>, <em>H. pileatus </em>and <em>S. syndactylus</em>, we detected no significant difference in the relative amount of the inter-specific
         divergence to the intra-specific polymorphism between synonymous and non-synonymous
         sites in either the L or M opsin gene (the McDonald-Kreitman test <a name="d21221e1988"></a>[<a onclick="LoadInParent('#B66'); return false;" href="#B66">66</a>]) or between the L or M opsin gene and the neutral references (the HKA test <a name="d21221e1992"></a>[<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>]) (data not presented). Using these male samples, we did not find any characteristic
         haplotype structure in terms of linkage disequilibrium (LD) in the approximately 3.6~3.9-kb
         genomic region of the L and M opsin genes as was not found in a study of chimpanzee
         L opsin gene <a name="d21221e1996"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>]. A measure of LD, the D', which takes a range from 0 to 1 <a name="d21221e2000"></a>[<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>], was nearly 1 throughout the region in all the four species examined (data not presented).
         This indicates low incidence of recombination between alleles, which is not unusual
         within such a short region. While the high LD could simply result from a small sample
         size, it is not contradictory with the frequent incidence of the gene conversion between
         gene loci because the LD concerns recombination between alleles.
      </p>
      <p style="line-height:160%">The present study of within-species nucleotide variation of both the L and M opsin
         genes in gibbons provides additional support to previous studies suggesting that the
         gene conversion has homogenized the tandemly duplicated L and M opsin genes and their
         spectral difference is maintained by purifying selection on centrally-located exons
         in non-human catarrhines <a name="d21221e2007"></a><a name="d21221e2009"></a><a name="d21221e2011"></a><a name="d21221e2013"></a><a name="d21221e2015"></a><a name="d21221e2017"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>,<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>,<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>-<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. The strict conservation of trichromacy in non-human catarrhines underscores a general
         belief that trichromacy is selectively more advantageous than dichromacy or anomalous
         trichromacy. Primate trichromacy is hypothesized to be adaptive for detecting anything
         differing from the background foliage in red-green coloration, the proposed major
         objects being mature fruits, young leaves, pelage and skin <a name="d21221e2021"></a><a name="d21221e2023"></a><a name="d21221e2025"></a><a name="d21221e2027"></a><a name="d21221e2029"></a><a name="d21221e2031"></a><a name="d21221e2033"></a><a name="d21221e2035"></a><a name="d21221e2037"></a><a name="d21221e2039"></a><a name="d21221e2041"></a><a name="d21221e2044"></a>[<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a>-<a onclick="LoadInParent('#B79'); return false;" href="#B79">79</a>].
      </p>
      <p style="line-height:160%">The strict conservation of the trichromacy in non-human catarrhines is in sharp contrast
         to New World monkeys. In many species of New World monkeys, the spectral variation
         of the L/M opsin is achieved by an allelic differentiation of a single locus on the
         X chromosome. This results in polymorphic color vision, consisting of trichromacy
         in females and dichromacy in both sexes <a name="d21221e2050"></a>[<a onclick="LoadInParent('#B80'); return false;" href="#B80">80</a>]. Genetic studies have shown that the allelic difference has been maintained by balancing
         selection <a name="d21221e2054"></a><a name="d21221e2056"></a><a name="d21221e2058"></a>[<a onclick="LoadInParent('#B81'); return false;" href="#B81">81</a>-<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>]. It remains to be elucidated whether the polymorphic color vision is a consequence
         of selective advantage on trichromats through the overdominance of the L/M opsin heterozygotes
         or the consequence of other mechanisms of balancing selection including a mutual benefit
         among different vision types <a name="d21221e2062"></a><a name="d21221e2064"></a>[<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>,<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a>]. It has been demonstrated that dichromats are superior to trichromats in some visual
         tasks such as defeating cryptic coloration and motion detection <a name="d21221e2068"></a><a name="d21221e2070"></a><a name="d21221e2072"></a>[<a onclick="LoadInParent('#B70'); return false;" href="#B70">70</a>,<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>,<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a>] and have a foraging advantage for surface-dwelling insects <a name="d21221e2076"></a><a name="d21221e2078"></a><a name="d21221e2080"></a>[<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>-<a onclick="LoadInParent('#B89'); return false;" href="#B89">89</a>]. In non-human catarrhines, however, this benefit of dichromacy could have been overwhelmed
         by that of trichromacy. It is still an open question whether the difference between
         the non-human catarrhines (routine trichromacy) and New World monkeys (polymorphic
         color vision) is attributable to a phenological difference among continents, such
         as in severity of seasonality and availability of non-colorful and non-seasonal fig
         and palms <a name="d21221e2085"></a>[<a onclick="LoadInParent('#B90'); return false;" href="#B90">90</a>], to a dietary variation such as in dependency on insets or colorful fruits, or to
         variation of social color signals <a name="d21221e2089"></a><a name="d21221e2091"></a>[<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>,<a onclick="LoadInParent('#B79'); return false;" href="#B79">79</a>].
      </p>
      <p style="line-height:160%">The absence of L/M defects in gibbons and other non-human catarrhines reveals the
         uniqueness of human color vision, in which the deletion of L or M opsin genes and
         the presence of L/M hybrid genes causes relatively high incidence of dichromacy and
         anomalous trichromacy <a name="d21221e2097"></a><a name="d21221e2099"></a><a name="d21221e2101"></a><a name="d21221e2103"></a><a name="d21221e2105"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>,<a onclick="LoadInParent('#B91'); return false;" href="#B91">91</a>-<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>]. Color vision polymorphisms among humans could be explained by frequent gene conversions
         of exon 3, due to the presence of a recombination hot-spot chi element conserved in
         the exon <a name="d21221e2109"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>] and relaxation of the selective constraint to maintain the spectral difference between
         the L and M opsin genes. Alternatively, adaptive explanations are possible. Balancing
         selection may be acting on subtle changes in light absorption of L <a name="d21221e2113"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>] and M opsins among females, or a selective advantage could be experienced by dichromatic
         males during hunting activities or via an increased ability to detect predators via
         penetrating camouflage <a name="d21221e2117"></a>[<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]. Further population studies of the L/M opsin genes for terrestrial-savanna catarrhines,
         such as baboons, would be important for elucidating evolutionary forces acting on
         development and maintenance of primate trichromacy.
      </p>
   </div>
</div>
<div><a name="sec5"></a><h3>Conclusions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The present study of within-species nucleotide variation of the L and M opsin genes
         in gibbons supports the notion that gene conversion has homogenized the tandemly duplicated
         L and M opsin genes and that their spectral difference is maintained by the purifying
         selection on the centrally-located exons in non-human catarrhines. Growing knowledge
         on genetic variation of opsin genes and visual ecology of primates has led to a paradigm
         shift toward the view that the adaptive value of trichromacy is conditional rather
         than universal, depending on the specific ecological demands on animals in their environments.
         Further population genetic and molecular phylogenic studies of primate visual opsins
         including humans, as well as observation and experiments on vision-guided primate
         behaviors, and phenological studies of habitats of wild primates will elucidate the
         detailed conditions of color vision evolution in primates.
      </p>
   </div>
</div>
<div><a name="sec6"></a><h3>Authors' contributions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">SK and AM conceived of the study. SK designed and supervised the experiments and analyses.
         MK organized the sample collection and DNA extraction. BSu organized the sample collection
         in Indonesia. DP conducted DNA extractions in Indonesia. SM organized the sample collection
         and conducted DNA extractions in Thailand. BSi organized the sample collection in
         Thailand. SG conducted the veterinary care of the animals during the sampling. TH
         carried out experiments, data compiling, and analysis. SK and TK performed additional
         data analyses. SK and TH wrote and illustrated the manuscript. HO advised on the research
         design and manuscript preparation. All authors read and approved the final manuscript.
      </p>
   </div>
</div>
<div><a name="ack"></a><h3>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">This study was supported by Grants-in-Aid for Scientific Research A 19207018 and 22247036
         from the Japan Society for the Promotion of Science (JSPS) and Grants-in-Aid for Scientific
         Research on Priority Areas "Comparative Genomics" 20017008 and "Cellular Sensor" 21026007
         from the Ministry of Education, Culture, Sports, Science and Technology of Japan to
         S.K. and by a Grant-in-Aid for Scientific Research B 17405022 from JSPS to AM. We
         thank Dr. Amanda D. Melin for proofreading and commenting on the manuscript and Dr.
         Naoki Osada for helpful suggestions.
      </p>
   </div>
</div>
<div><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Nathans J,  Thomas D,  Hogness DS: <strong> Molecular genetics of human color vision: the genes encoding blue, green, and red
                  pigments. </strong></p><p><em>Science</em> 1986,  <strong>232</strong><strong>:</strong>193-202.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2937147">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2937147">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B1','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Jacobs GH: <strong> Primate photopigments and primate color vision. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1996,  <strong>93</strong><strong>:</strong>577-581.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8570598">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8570598">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=8570598">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B2','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Yokoyama R,  Yokoyama S: <strong> Convergent evolution of the red- and green-like visual pigment genes in fish, <em>Astyanax fasciatus</em>, and human. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1990,  <strong>87</strong><strong>:</strong>9315-9318.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2123554">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2123554">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2123554">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B3','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Neitz M,  Neitz J,  Jacobs GH: <strong> Spectral tuning of pigments underlying red-green color vision. </strong></p><p><em>Science</em> 1991,  <strong>252</strong><strong>:</strong>971-974.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1903559">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1903559">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B4','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Asenjo AB,  Rim J,  Oprian DD: <strong> Molecular determinants of human red/green color discrimination. </strong></p><p><em>Neuron</em> 1994,  <strong>12</strong><strong>:</strong>1131-1138.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8185948">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8185948">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B5','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Yokoyama S,  Radlwimmer FB: <strong> The "five-sites" rule and the evolution of red and green color vision in mammals. </strong></p><p><em>Mol Biol Evol</em> 1998,  <strong>15</strong><strong>:</strong>560-567.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9580985">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9580985">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B6','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Yokoyama S,  Radlwimmer FB: <strong> The molecular genetics of red and green color vision in mammals. </strong></p><p><em>Genetics</em> 1999,  <strong>153</strong><strong>:</strong>919-932.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10511567">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10511567">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10511567">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B7','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Yokoyama S,  Radlwimmer FB: <strong> The molecular genetics and evolution of red and green color vision in vertebrates. </strong></p><p><em>Genetics</em> 2001,  <strong>158</strong><strong>:</strong>1697-1710.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11545071">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11545071">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11545071">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B8','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Chan T,  Lee M,  Sakmar TP: <strong> Introduction of hydroxyl-bearing amino acids causes bathochromic spectral shifts in
                  rhodopsin. Amino acid substitutions responsible for red-green color pigment spectral
                  tuning. </strong></p><p><em>J Biol Chem</em> 1992,  <strong>267</strong><strong>:</strong>9478-9480.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1577792">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1577792">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B9','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Deeb SS: <strong> Genetics of variation in human color vision and the retinal cone mosaic. </strong></p><p><em>Curr Opin Genet Dev</em> 2006,  <strong>16</strong><strong>:</strong>301-307.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16647849">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16647849">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B10','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Drummond-Borg M,  Deeb SS,  Motulsky AG: <strong> Molecular patterns of X chromosome-linked color vision genes among 134 men of European
                  ancestry. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1989,  <strong>86</strong><strong>:</strong>983-987.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2915991">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2915991">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2915991">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B11','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Jorgensen AL,  Deeb SS,  Motulsky AG: <strong> Molecular genetics of X chromosome-linked color vision among populations of African
                  and Japanese ancestry: high frequency of a shortened red pigment gene among Afro-Americans. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1990,  <strong>87</strong><strong>:</strong>6512-6516.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2395857">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2395857">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2395857">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B12','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Shyue SK,  Li L,  Chang BH,  Li WH: <strong> Intronic gene conversion in the evolution of human X-linked color vision genes. </strong></p><p><em>Mol Biol Evol</em> 1994,  <strong>11</strong><strong>:</strong>548-551.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8015447">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8015447">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B13','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Zhou YH,  Li WH: <strong> Gene conversion and natural selection in the evolution of X-linked color vision genes
                  in higher primates. </strong></p><p><em>Mol Biol Evol</em> 1996,  <strong>13</strong><strong>:</strong>780-783.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8754214">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8754214">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B14','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Zhao Z,  Hewett-Emmett D,  Li WH: <strong> Frequent gene conversion between human red and green opsin genes. </strong></p><p><em>J Mol Evol</em> 1998,  <strong>46</strong><strong>:</strong>494-496.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9541545">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9541545">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B15','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Winderickx J,  Lindsey DT,  Sanocki E,  Teller DY,  Motulsky AG,  Deeb SS: <strong> Polymorphism in red photopigment underlies variation in colour matching. </strong></p><p><em>Nature</em> 1992,  <strong>356</strong><strong>:</strong>431-433.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1557123">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1557123">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B16','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Winderickx J,  Battisti L,  Hibiya Y,  Motulsky AG,  Deeb SS: <strong> Haplotype diversity in the human red and green opsin genes: evidence for frequent
                  sequence exchange in exon 3. </strong></p><p><em>Hum Mol Genet</em> 1993,  <strong>2</strong><strong>:</strong>1413-1421.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8242064">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8242064">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B17','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B18">
            <p><a name="B18"></a> Verrelli BC,  Tishkoff SA: <strong> Signatures of selection and gene conversion associated with human color vision variation. </strong></p><p><em>Am J Hum Genet</em> 2004,  <strong>75</strong><strong>:</strong>363-375.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15252758">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15252758">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15252758">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B18','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Onishi A,  Koike S,  Ida M,  Imai H,  Shichida Y,  Takenaka O,  Hanazawa A,  Komatsu H,  Mikami A,  Goto S, <em>et al</em>.: <strong> Dichromatism in macaque monkeys. </strong></p><p><em>Nature</em> 1999,  <strong>402</strong><strong>:</strong>139-140.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10647004">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10647004">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B19','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Jacobs GH,  Williams GA: <strong> The prevalence of defective color vision in Old World monkeys and apes. </strong></p><p><em>Col Res Appl</em> 2001,  <strong>26</strong>(Suppl)<strong>:</strong>S123-S127.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B20','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Terao K,  Mikami A,  Saito A,  Itoh S,  Ogawa H,  Takenaka O,  Sakai T,  Onishi A,  Teramoto M,  Udono T, <em>et al</em>.: <strong> Identification of a protanomalous chimpanzee by molecular genetic and electroretinogram
                  analyses. </strong></p><p><em>Vision Res</em> 2005,  <strong>45</strong><strong>:</strong>1225-1235.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15733956">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15733956">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B21','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Hanazawa A,  Mikami A,  Sulistyo Angelika P,  Takenaka O,  Goto S,  Onishi A,  Koike S,  Yamamori T,  Kato K,  Kondo A, <em>et al</em>.: <strong> Electroretinogram analysis of relative spectral sensitivity in genetically identified
                  dichromatic macaques. </strong></p><p><em>Proc Natl Acad Sci USA</em> 2001,  <strong>98</strong><strong>:</strong>8124-8127.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11427736">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11427736">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11427736">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B22','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Onishi A,  Koike S,  Ida-Hosonuma M,  Imai H,  Shichida Y,  Takenaka O,  Hanazawa A,  Komatsu H,  Mikami A,  Goto S, <em>et al</em>.: <strong> Variations in long- and middle-wavelength-sensitive opsin gene loci in crab-eating
                  monkeys. </strong></p><p><em>Vision Res</em> 2002,  <strong>42</strong><strong>:</strong>281-292.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11809481">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11809481">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B23','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Saito A,  Mikami A,  Hasegawa T,  Koida K,  Terao K,  Koike S,  Onishi A,  Takenaka O,  Teramoto M,  Mori Y: <strong> Behavioral evidence of color vision deficiency in a protanomalia chimpanzee (<em>Pan troglodytes</em>). </strong></p><p><em>Primates</em> 2003,  <strong>44</strong><strong>:</strong>171-176.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/12687482">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12687482">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B24','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Verrelli BC,  Lewis CM jr,  Stone AC,  Perry GH: <strong> Different selective pressures shape the molecular evolution of color vision in chimpanzee
                  and human populations. </strong></p><p><em>Mol Biol Evol</em> 2008,  <strong>25</strong><strong>:</strong>2735-2743.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/18832077">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18832077">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18832077">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B25','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> Ibbotson RE,  Hunt DM,  Bowmaker JK,  Mollon JD: <strong> Sequence divergence and copy number of the middle- and long-wave photopigment genes
                  in Old World monkeys. </strong></p><p><em>Proc R Soc Lond B</em> 1992,  <strong>247</strong><strong>:</strong>145-154.   <a target="_blank" href="http://dx.doi.org/10.1098/rspb.1992.0021">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B26','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Balding DJ,  Nichols RA,  Hunt DM: <strong> Detecting gene conversion: primate visual pigment genes. </strong></p><p><em>Proc R Soc B</em> 1992,  <strong>249</strong><strong>:</strong>275-280.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1359557">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1359557">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B27','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Deeb SS,  Jorgensen AL,  Battisti L,  Iwasaki L,  Motulsky AG: <strong> Sequence divergence of the red and green visual pigments in great apes and humans. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1994,  <strong>91</strong><strong>:</strong>7262-7266.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8041777">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8041777">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=8041777">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B28','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Dulai KS,  Bowmaker JK,  Mollon JD,  Hunt DM: <strong> Sequence divergence, polymorphism and evolution of the middle-wave and long-wave visual
                  pigment genes of great apes and Old World monkeys. </strong></p><p><em>Vision Res</em> 1994,  <strong>34</strong><strong>:</strong>2483-2491.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/7975287">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7975287">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B29','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Deegan JF,  Jacobs GH: <strong> Spectral sensitivity of gibbons: implications for photopigments and color vision. </strong></p><p><em>Folia Primatol</em> 2001,  <strong>72</strong><strong>:</strong>26-29.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11275745">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11275745">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B30','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Fleagle JG: <em>Primate Adaptation and Evolution</em>.  2nd edition.  San Diego: Academic Press;  1999. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B31','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Macke JP,  Nathans J: <strong> Individual variation in size of the human red and green visual pigment gene array. </strong></p><p><em>Invest Ophthalmol Vis Sci</em> 1997,  <strong>38</strong><strong>:</strong>1040-1043.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9113000">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9113000">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B32','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Kawamura S,  Hirai M,  Takenaka O,  Radlwimmer FB,  Yokoyama S: <strong> Genomic and spectral analyses of long to middle wavelength-sensitive visual pigments
                  of common marmoset (<em>Callithrix jacchus</em>). </strong></p><p><em>Gene</em> 2001,  <strong>269</strong><strong>:</strong>45-51.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11376936">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11376936">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B33','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Nagao K,  Takenaka N,  Hirai M,  Kawamura S: <strong> Coupling and decoupling of evolutionary mode between X- and Y-chromosomal red-green
                  opsin genes in owl monkeys. </strong></p><p><em>Gene</em> 2005,  <strong>352</strong><strong>:</strong>82-91.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15922519">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15922519">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B34','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Nathans J,  Piantanida TP,  Eddy RL,  Shows TB,  Hogness DS: <strong> Molecular genetics of inherited variation in human color vision. </strong></p><p><em>Science</em> 1986,  <strong>232</strong><strong>:</strong>203-210.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/3485310">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3485310">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B35','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B36">
            <p><a name="B36"></a> Vollrath D,  Nathans J,  Davis RW: <strong> Tandem array of human visual pigment genes at Xq28. </strong></p><p><em>Science</em> 1988,  <strong>240</strong><strong>:</strong>1669-1672.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2837827">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2837827">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B36','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> Neitz J,  Neitz M,  Kainz PM: <strong> Visual pigment gene structure and the severity of color vision defects. </strong></p><p><em>Science</em> 1996,  <strong>274</strong><strong>:</strong>801-804.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/8864125">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8864125">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B37','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Kainz PM,  Neitz M,  Neitz J: <strong> Molecular genetic detection of female carriers of protan defects. </strong></p><p><em>Vision Res</em> 1998,  <strong>38</strong><strong>:</strong>3365-3369.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9893850">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9893850">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B38','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B39">
            <p><a name="B39"></a> Hayashi T,  Motulsky AG,  Deeb SS: <strong> Position of a 'green-red' hybrid gene in the visual pigment array determines colour-vision
                  phenotype. </strong></p><p><em>Nat Genet</em> 1999,  <strong>22</strong><strong>:</strong>90-93.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10319869">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10319869">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B39','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> Oda S,  Ueyama H,  Nishida Y,  Tanabe S,  Yamade S: <strong> Analysis of L-cone/M-cone visual pigment gene arrays in females by long-range PCR. </strong></p><p><em>Vision Res</em> 2003,  <strong>43</strong><strong>:</strong>489-495.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/12594995">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12594995">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B40','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Oprian DD,  Asenjo AB,  Lee N,  Pelletier SL: <strong> Design, chemical synthesis, and expression of genes for the three human color vision
                  pigments. </strong></p><p><em>Biochemistry</em> 1991,  <strong>30</strong><strong>:</strong>11367-11372.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1742276">PubMedAbstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1021/bi00112a002">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B41','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Koop BF,  Goodman M,  Xu P,  Chan K,  Slightom JL: <strong> Primate eta-globin DNA sequences and man's place among the great apes. </strong></p><p><em>Nature</em> 1986,  <strong>319</strong><strong>:</strong>234-238.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/3945312">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3945312">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B42','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Bailey WJ,  Fitch DH,  Tagle DA,  Czelusniak J,  Slightom JL,  Goodman M: <strong> Molecular evolution of the psi eta-globin gene locus: gibbon phylogeny and the hominoid
                  slowdown. </strong></p><p><em>Mol Biol Evol</em> 1991,  <strong>8</strong><strong>:</strong>155-184.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2046542">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2046542">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B43','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Thompson JD,  Higgins DG,  Gibson TJ: <strong> CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through
                  sequence weighting, position-specific gap penalties and weight matrix choice. </strong></p><p><em>Nucleic Acids Res</em> 1994,  <strong>22</strong><strong>:</strong>4673-4680.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/7984417">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7984417">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7984417">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B44','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Nei M,  Gojobori T: <strong> Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide
                  substitutions. </strong></p><p><em>Mol Biol Evol</em> 1986,  <strong>3</strong><strong>:</strong>418-426.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/3444411">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3444411">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B45','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> Tamura K,  Peterson D,  Peterson N,  Stecher G,  Nei M,  Kumar S: <strong> MEGA5: Molecular Evolutionary Genetics Analysis using maximum likelihood, evolutionary
                  distance, and maximum parsimony methods. </strong></p><p><em>Mol Biol Evol</em> 2011,  <strong>28</strong><strong>:</strong>2731-2739.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21546353">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21546353">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B46','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Nei M,  Kumar S: <em>Molecular Evolution and Phylogenetics</em>.  New York: Oxford university press;  2000. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B47','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Saitou N,  Nei M: <strong> The neighbor-joining method: a new method for reconstructing phylogenetic trees. </strong></p><p><em>Mol Biol Evol</em> 1987,  <strong>4</strong><strong>:</strong>406-425.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/3447015">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3447015">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B48','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B49">
            <p><a name="B49"></a><strong> Evolutionary Genomics Lab </strong>[<a target="_blank" href="http://www.kms.ac.jp/%7Egenomelb/takezaki.eng.html">http://www.kms.ac.jp/~genomelb/takezaki.eng.html</a>] <a href="http://www.webcitation.org/query.php?url=http://www.kms.ac.jp/%7Egenomelb/takezaki.eng.html&amp;refdoi=10.1186/1471-2148-11-312" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B50">
            <p><a name="B50"></a><strong> GENECONV Molecular Biology Computer Program </strong>[<a target="_blank" href="http://www.math.wustl.edu/%7Esawyer/geneconv/index.html">http://www.math.wustl.edu/~sawyer/geneconv/index.html</a>] <a href="http://www.webcitation.org/query.php?url=http://www.math.wustl.edu/%7Esawyer/geneconv/index.html&amp;refdoi=10.1186/1471-2148-11-312" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Sawyer S: <strong> Statistical tests for detecting gene conversion. </strong></p><p><em>Mol Biol Evol</em> 1989,  <strong>6</strong><strong>:</strong>526-538.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/2677599">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2677599">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B51','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> Kim SK,  Carbone L,  Becquet C,  Mootnick AR,  Li DJ,  de Jong PJ,  Wall JD: <strong> Patterns of genetic variation within and between gibbon species. </strong></p><p><em>Mol Biol Evol</em> 2011,  <strong>28</strong><strong>:</strong>2211-2218.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21368318">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21368318">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21368318">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B52','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Roos C,  Geissmann T: <strong> Molecular phylogeny of the major hylobatid divisions. </strong></p><p><em>Mol Phylogenet Evol</em> 2001,  <strong>19</strong><strong>:</strong>486-494.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11399155">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11399155">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B53','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Chatterjee HJ: <strong> Phylogeny and biogeography of gibbons: A dispersal-vicariance analysis. </strong></p><p><em>Int J Primatol</em> 2006,  <strong>27</strong><strong>:</strong>699-712.   <a target="_blank" href="http://dx.doi.org/10.1007/s10764-006-9044-1">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B54','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Chan YC,  Roos C,  Inoue-Murayama M,  Inoue E,  Shih CC,  Pei KJ,  Vigilant L: <strong> Mitochondrial genome sequences effectively reveal the phylogeny of <em>Hylobates </em>gibbons. </strong></p><p><em>PLoS ONE</em> 2010,  <strong>5</strong><strong>:</strong>e14419.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21203450">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21203450">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21203450">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B55','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Matsudaira K,  Ishida T: <strong> Phylogenetic relationships and divergence dates of the whole mitochondrial genome
                  sequences among three gibbon genera. </strong></p><p><em>Mol Phylogenet Evol</em> 2010,  <strong>55</strong><strong>:</strong>454-459.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/20138221">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20138221">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B56" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B56','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Israfil H,  Zehr SM,  Mootnick AR,  Ruvolo M,  Steiper ME: <strong> Unresolved molecular phylogenies of gibbons and siamangs (Family: Hylobatidae) based
                  on mitochondrial, Y-linked, and X-linked loci indicate a rapid Miocene radiation or
                  sudden vicariance event. </strong></p><p><em>Mol Phylogenet Evol</em> 2011,  <strong>58</strong><strong>:</strong>447-455.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21074627">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21074627">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B57','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Hartl DL,  Clark AG: <em>Principles of Population Genetics</em>.  4th edition.  Sunderland: Sinauer Associates;  2007. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B58','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Nachman MW: <strong> Single nucleotide polymorphisms and recombination rate in humans. </strong></p><p><em>Trends Genet</em> 2001,  <strong>17</strong><strong>:</strong>481-485.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11525814">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11525814">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B59','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Tishkoff SA,  Verrelli BC: <strong> Patterns of human genetic diversity: implications for human evolutionary history and
                  disease. </strong></p><p><em>Annu Rev Genomics Hum Genet</em> 2003,  <strong>4</strong><strong>:</strong>293-340.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/14527305">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14527305">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B60','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Fischer A,  Pollack J,  Thalmann O,  Nickel B,  Paabo S: <strong> Demographic history and genetic differentiation in apes. </strong></p><p><em>Curr Biol</em> 2006,  <strong>16</strong><strong>:</strong>1133-1138.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16753568">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16753568">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B61','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Hayashi S,  Ueyama H,  Tanabe S,  Yamade S,  Kani K: <strong> Number and variations of the red and green visual pigment genes in Japanese men with
                  normal color vision. </strong></p><p><em>Jpn J Ophthalmol</em> 2001,  <strong>45</strong><strong>:</strong>60-67.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11163047">PubMedAbstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1016/S0021-5155%2800%2900298-7">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B62','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B63">
            <p><a name="B63"></a> Sharpe LT,  Stockman A,  Jagle H,  Nathans J: <strong> Opsin genes, cone photopigments, color vision, and color blindness. </strong>In <em>Color vision: from genes to perception</em>.  Edited by Gegenfurtner KR, Sharpe LT.  Cambridge: Cambridge University Press;  1999:3-51.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B63','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B64">
            <p><a name="B64"></a> Graur D,  Li W-H: <em>Fundamentals of Molecular Evolution</em>.  2nd edition.  Sunderland: Sinauer Publishers;  2000. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B64','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B65">
            <p><a name="B65"></a> Petronella N,  Drouin G: <strong> Gene conversions in the growth hormone gene family of primates: Stronger homogenizing
                  effects in the Hominidae lineage. </strong></p><p><em>Genomics</em> 2011,  <strong>98</strong><strong>:</strong>173-181.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21683133">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21683133">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B65" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B65','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B66">
            <p><a name="B66"></a> McDonald JH,  Kreitman M: <strong> Adaptive protein evolution at the <em>Adh </em>locus in <em>Drosophila</em>. </strong></p><p><em>Nature</em> 1991,  <strong>351</strong><strong>:</strong>652-654.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/1904993">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1904993">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B66" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B66','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B67">
            <p><a name="B67"></a> Hudson RR,  Kreitman M,  Aguade M: <strong> A test of neutral molecular evolution based on nucleotide data. </strong></p><p><em>Genetics</em> 1987,  <strong>116</strong><strong>:</strong>153-159.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/3110004">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3110004">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=3110004">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B67" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B67','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B68">
            <p><a name="B68"></a> Regan BC,  Julliot C,  Simmen B,  Vienot F,  Charles-Dominique P,  Mollon JD: <strong> Frugivory and colour vision in <em>Alouatta seniculus</em>, a trichromatic platyrrhine monkey. </strong></p><p><em>Vision Res</em> 1998,  <strong>38</strong><strong>:</strong>3321-3327.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9893844">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9893844">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B68" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B68','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B69">
            <p><a name="B69"></a> Sumner P,  Mollon JD: <strong> Catarrhine photopigments are optimized for detecting targets against a foliage background. </strong></p><p><em>J Exp Biol</em> 2000,  <strong>203</strong><strong>:</strong>1963-1986.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10851115">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10851115">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B69" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B69','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B70">
            <p><a name="B70"></a> Regan BC,  Julliot C,  Simmen B,  Vienot F,  Charles-Dominique P,  Mollon JD: <strong> Fruits, foliage and the evolution of primate colour vision. </strong></p><p><em>Phil Trans R Soc B</em> 2001,  <strong>356</strong><strong>:</strong>229-283.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11316480">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11316480">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11316480">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B70" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B70','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B71">
            <p><a name="B71"></a> Sumner P,  Mollon JD: <strong> Colors of primate pelage and skin: objective assessment of conspicuousness. </strong></p><p><em>Am J Primatol</em> 2003,  <strong>59</strong><strong>:</strong>67-91.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/12619048">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12619048">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B71" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B71','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B72">
            <p><a name="B72"></a> Lucas PW,  Darvell BW,  Lee PKD,  Yuen TDB,  Choong MF: <strong> Colour cues for leaf food selection by long-tailed macaques (<em>Macaca fascicularis</em>) with a new suggestion for the evolution of trichromatic colour vision. </strong></p><p><em>Folia Primatol</em> 1998,  <strong>69</strong><strong>:</strong>139-154.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9595683">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9595683">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B72" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B72','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B73">
            <p><a name="B73"></a> Dominy NJ,  Lucas PW: <strong> Ecological importance of trichromatic vision to primates. </strong></p><p><em>Nature</em> 2001,  <strong>410</strong><strong>:</strong>363-366.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11268211">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11268211">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B73" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B73','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B74">
            <p><a name="B74"></a> Lucas PW,  Dominy NJ,  Riba-Hernández 
P,  Stoner KE,  Yamashita N,  Loria-Calderon E,  Petersen-Pereira W,  
Rojas-Duran Y,  Salas-Pena R,  Solis-Madrigal S, <em>et al</em>.: <strong> Evolution and function of routine trichromatic vision in primates. </strong></p><p><em>Evolution</em> 2003,  <strong>57</strong><strong>:</strong>2636-2643.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/14686538">PubMedAbstract</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B74" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B74','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B75">
            <p><a name="B75"></a> Allen G: <em>The Color Sense: Its Origin and Development</em>.  London: Trubner &amp; Co;  1879. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B75" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B75','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B76">
            <p><a name="B76"></a> Surridge AK,  Osorio D,  Mundy NI: <strong> Evolution and selection of trichromatic vision in primates. </strong></p><p><em>Trends Ecol Evol</em> 2003,  <strong>18</strong><strong>:</strong>198-205.   <a target="_blank" href="http://dx.doi.org/10.1016/S0169-5347%2803%2900012-0">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B76" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B76','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B77">
            <p><a name="B77"></a> Vorobyev M: <strong> Ecology and evolution of primate colour vision. </strong></p><p><em>Clin Exp Optom</em> 2004,  <strong>87</strong><strong>:</strong>230-238.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15312027">PubMedAbstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1444-0938.2004.tb05053.x">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B77" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B77','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B78">
            <p><a name="B78"></a> Changizi MA,  Zhang Q,  Shimojo S: <strong> Bare skin, blood and the evolution of primate color vision. </strong></p><p><em>Biol Lett</em> 2006,  <strong>2</strong><strong>:</strong>217-221.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17148366">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17148366">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17148366">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B78" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B78','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B79">
            <p><a name="B79"></a> Fernandez AA,  Morris MR: <strong> Sexual selection and trichromatic color vision in primates: statistical support for
                  the preexisting-bias hypothesis. </strong></p><p><em>Am Nat</em> 2007,  <strong>170</strong><strong>:</strong>10-20.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17853988">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17853988">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B79" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B79','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B80">
            <p><a name="B80"></a> Jacobs GH: <strong> New World monkeys and color. </strong></p><p><em>Int J Primatol</em> 2007,  <strong>28</strong><strong>:</strong>729-759.   <a target="_blank" href="http://dx.doi.org/10.1007/s10764-007-9168-y">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B80" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B80','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B81">
            <p><a name="B81"></a> Boissinot S,  Tan Y,  Shyue SK,  Schneider H,  Sampaio I,  Neiswanger K,  Hewett-Emmett D,  Li WH: <strong> Origins and antiquity of X-linked triallelic color vision systems in New World monkeys. </strong></p><p><em>Proc Natl Acad Sci USA</em> 1998,  <strong>95</strong><strong>:</strong>13749-13754.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9811872">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9811872">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9811872">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B81" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B81','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B82">
            <p><a name="B82"></a> Surridge AK,  Mundy NI: <strong> Trans-specific evolution of opsin alleles and the maintenance of trichromatic colour
                  vision in Callitrichine primates. </strong></p><p><em>Mol Ecol</em> 2002,  <strong>11</strong><strong>:</strong>2157-2169.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/12296957">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12296957">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B82" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B82','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B83">
            <p><a name="B83"></a> Hiwatashi T,  Okabe Y,  Tsutsui T,  
Hiramatsu C,  Melin AD,  Oota H,  Schaffner CM,  Aureli F,  Fedigan LM, 
 Innan H,  Kawamura S: <strong> An explicit signature of balancing selection for color-vision variation in new world
                  monkeys. </strong></p><p><em>Mol Biol Evol</em> 2010,  <strong>27</strong><strong>:</strong>453-464.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19861643">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19861643">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B83" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B83','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B84">
            <p><a name="B84"></a> Mollon JD,  Bowmaker JK,  Jacobs GH: <strong> Variations of colour vision in a New World primate can be explained by polymorphism
                  of retinal photopigments. </strong></p><p><em>Proc R Soc Lond B</em> 1984,  <strong>222</strong><strong>:</strong>373-399.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/6149558">PubMedAbstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1098/rspb.1984.0071">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B84" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B84','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B85">
            <p><a name="B85"></a> Morgan MJ,  Adam A,  Mollon JD: <strong> Dichromats detect colour-camouflaged objects that are not detected by trichromats. </strong></p><p><em>Proc R Soc Lond B</em> 1992,  <strong>248</strong><strong>:</strong>291-295.   <a target="_blank" href="http://dx.doi.org/10.1098/rspb.1992.0074">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B85" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B85','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B86">
            <p><a name="B86"></a> Saito A,  Mikami A,  Kawamura S,  Ueno Y,  Hiramatsu C,  Widayati KA,  Suryobroto B,  Teramoto M,  Mori Y,  Nagano K, <em>et al</em>.: <strong> Advantage of dichromats over trichromats in discrimination of color-camouflaged stimuli
                  in nonhuman primates. </strong></p><p><em>Am J Primatol</em> 2005,  <strong>67</strong><strong>:</strong>425-436.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16342068">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16342068">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B86" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B86','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B87">
            <p><a name="B87"></a> Melin AD,  Fedigan LM,  Hiramatsu C,  Sendall C,  Kawamura S: <strong> Effects of colour vision phenotype on insect capture by a free-ranging population
                  of white-faced capuchins (<em>Cebus capucinus</em>). </strong></p><p><em>Anim Behav</em> 2007,  <strong>73</strong><strong>:</strong>205-214.   <a target="_blank" href="http://dx.doi.org/10.1016/j.anbehav.2006.07.003">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B87" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B87','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B88">
            <p><a name="B88"></a> Melin AD,  Fedigan LM,  Young HC,  Kawamura S: <strong> Can color vision variation explain sex differences in invertebrate foraging by capuchin
                  monkeys? </strong></p><p><em>Curr Zool</em> 2010,  <strong>56</strong><strong>:</strong>300-312.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B88" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B88','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B89">
            <p><a name="B89"></a> Caine NG,  Osorio D,  Mundy NI: <strong> A foraging advantage for dichromatic marmosets (<em>Callithrix geoffroyi</em>) at low light intensity. </strong></p><p><em>Biol Lett</em> 2010,  <strong>6</strong><strong>:</strong>36-38.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19740895">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19740895">PublisherFullText</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19740895">PubMedCentralFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B89" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B89','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B90">
            <p><a name="B90"></a> Dominy NJ,  Svenning JC,  Li WH: <strong> Historical contingency in the evolution of primate color vision. </strong></p><p><em>J Hum Evol</em> 2003,  <strong>44</strong><strong>:</strong>25-45.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/12604302">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12604302">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B90" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B90','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B91">
            <p><a name="B91"></a> Verhulst S,  Maes FW: <strong> Scotopic vision in colour-blinds. </strong></p><p><em>Vision Res</em> 1998,  <strong>38</strong><strong>:</strong>3387-3390.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/9893853">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9893853">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B91" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B91','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B92">
            <p><a name="B92"></a> Bosten JM,  Robinson JD,  Jordan G,  Mollon JD: <strong> Multidimensional scaling reveals a color dimension unique to 'color deficient' observers. </strong></p><p><em>Curr Biol</em> 2005,  <strong>15</strong><strong>:</strong>R950-R952.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16332521">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16332521">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B92" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B92','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B93">
            <p><a name="B93"></a> Hood SM,  Mollon JD,  Purves L,  Jordan G: <strong> Color discrimination in carriers of color deficiency. </strong></p><p><em>Vision Res</em> 2006,  <strong>46</strong><strong>:</strong>2894-2900.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16690099">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16690099">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B93" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B93','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B94">
            <p><a name="B94"></a> Sharpe LT,  de Luca E,  Hansen T,  Jagle H,  Gegenfurtner KR: <strong> Advantages and disadvantages of human dichromacy. </strong></p><p><em>J Vis</em> 2006,  <strong>6</strong><strong>:</strong>213-223.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16643091">PubMedAbstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16643091">PublisherFullText</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-11-312&amp;bibl=B94" onclick="popup('/sfx_links?ui=1471-2148-11-312&amp;bibl=B94','SFXMenu','460','420'); return false;"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
      </ol>
   </div>
</div>
<div class="article-alert-signup-div">

<a class="close png_bg" id="closeHelpBox">close</a>	

	
	
</div>
</div>
</div>
</div>
</div>
</div>


		</div>
</body>
</html>