<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5509165">5509165</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0180741">10.1371/journal.pone.0180741</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-17-01773</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Anatomy</span>: <span class="subject" title="subject">Leaves</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Databases</span>: <span class="subject" title="subject">Genomic Databases</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Databases</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Databases</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Prediction</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Prediction</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span>: <span class="subject" title="subject">Gene Mapping</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span>: <span class="subject" title="subject">Gene Mapping</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Artificial Gene Amplification and Extension</span>: <span class="subject" title="subject">Polymerase Chain Reaction</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Artificial Gene Amplification and Extension</span>: <span class="subject" title="subject">Polymerase Chain Reaction</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Genomics</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span>: <span class="subject" title="subject">Plant Biochemistry</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Plant Biochemistry</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Forms of DNA</span>: <span class="subject" title="subject">Complementary DNA</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Forms of DNA</span>: <span class="subject" title="subject">Complementary DNA</span> </div> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genetic mapping of the <i>LOBED LEAF 1 (ClLL1)</i> gene to a 127.6-kb region in watermelon ( <i>Citrullus lanatus</i> L.) </div> <div class="alt-title" title="alt-title">
Genetic mapping of the <i>LOBED LEAF 1 (ClLL1)</i> gene in watermelon ( <i>Citrullus lanatus</i> L.) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Chunhua</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Xiner</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Zhongyuan</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Qiyan</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Hao</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Yong</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Jianxiang</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Jianqiang</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9254-0730</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xian</span> </span> <a href="#cor001">*</a> <a href="#aff001" /> </span> </div> <span class="citation_author_institution" id="aff001">[], <span class="addr-line" title="addr-line">College of Horticulture, Northwest A&amp;amp;F University, Yangling, China</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Candela</span> <span class="given-names" tagx="given-names" title="given-names">Hector</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Universidad Miguel Hernández de Elche, SPAIN</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p> </p><ul> <li> <p> <b>Conceptualization:</b> CW XZ. </p> </li> <li> <p> <b>Data curation:</b> CW XC ZW QL. </p> </li> <li> <p> <b>Formal analysis:</b> CW XC. </p> </li> <li> <p> <b>Funding acquisition:</b> CW XZ. </p> </li> <li> <p> <b>Investigation:</b> CW XC ZW QL. </p> </li> <li> <p> <b>Methodology:</b> CW XZ. </p> </li> <li> <p> <b>Project administration:</b> CW XZ. </p> </li> <li> <p> <b>Resources:</b> JM YZ JY. </p> </li> <li> <p> <b>Software:</b> CW. </p> </li> <li> <p> <b>Supervision:</b> CW XZ. </p> </li> <li> <p> <b>Validation:</b> CW XZ. </p> </li> <li> <p> <b>Visualization:</b> CW XC. </p> </li> <li> <p> <b>Writing – original draft:</b> CW. </p> </li> <li> <p> <b>Writing – review &amp;amp; editing:</b> CW HL. </p> </li> </ul> <p /> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">zhangxian098@126.com</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-7-7</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0180741</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-1-13</span> <span class="accepted" title="accepted">accepted: 2017-6-20</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Wei et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0180741.pdf" title="self-uri">pone.0180741.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The lobed leaf character is a unique morphologic trait in crops, featuring many potential advantages for agricultural productivity. Although the majority of watermelon varieties feature lobed leaves, the genetic factors responsible for lobed leaf formation remain elusive. The F <sub>2:3</sub> leaf shape segregating population offers the opportunity to study the underlying mechanism of lobed leaf formation in watermelon. Genetic analysis revealed that a single dominant allele (designated <i>ClLL1</i>) controlled the lobed leaf trait. A large-sized F <sub>3:4</sub> population derived from F <sub>2:3</sub> individuals was used to map <i>ClLL1</i>. A total of 5,966 reliable SNPs and indels were identified genome-wide via a combination of BSA and RNA-seq. Using the validated SNP and indel markers, the location of <i>ClLL1</i> was narrowed down to a 127.6-kb region between markers W08314 and W07061, containing 23 putative ORFs. Expression analysis via qRT-PCR revealed differential expression patterns (fold-changes above 2-fold or below 0.5-fold) of three ORFs ( <i>ORF3</i>, <i>ORF11</i>, and <i>ORF18</i>) between lobed and non-lobed leaf plants. Based on gene annotation and expression analysis, <i>ORF18</i> (encoding an uncharacterized protein) and <i>ORF22</i> (encoding a homeobox-leucine zipper-like protein) were considered as most likely candidate genes. Furthermore, sequence analysis revealed no polymorphisms in cDNA sequences of <i>ORF18</i>; however, two notable deletions were identified in <i>ORF22</i>. This study is the first report to map a leaf shape gene in watermelon and will facilitate cloning and functional characterization of <i>ClLL1</i> in future studies. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Scientific Startup Foundation for Doctors of Northwest A and F University</span> </div> <span class="award-id" tagx="award-id" title="award-id">Z109021604</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Chunhua</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Basal Research Foundation of Northwest A and F University</span> </div> <span class="award-id" tagx="award-id" title="award-id">Z109021612</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">Chunhua</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Modern Agro-industry Technology Research System of China</span> </div> <span class="award-id" tagx="award-id" title="award-id">No. CARS-26-18</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9254-0730</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Xian</span> </span> </span> </div> <div class="funding-statement" title="funding-statement">
This work was supported by Scientific Startup Foundation for Doctors of Northwest A and F University (Z109021604), Basal Research Foundation of Northwest A and F University (Z109021612), and the Modern Agro-industry Technology Research System of China (No. CARS-26-18). </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Leaves are vitally important photosynthetic organs of flowering plants, determining the distribution of nutrients, gas exchange, and water transport. Leaves furthermore exhibit a remarkable variety in size, shape, and position on the stem [ <a href="#pone.0180741.ref001">1</a>– <a href="#pone.0180741.ref003">3</a>]. Leaf shapes reveal a clearly visible diversity among different species and even within the same species [ <a href="#pone.0180741.ref004">4</a>– <a href="#pone.0180741.ref008">8</a>]. In addition to molecular genetic regulators, leaf shapes can also be influenced by various environmental factors, such as severe fluctuations in temperature and light regimens [ <a href="#pone.0180741.ref009">9</a>, <a href="#pone.0180741.ref010">10</a>]. Leaf margin is an important trait of leaf shape and can be serrated, lobed, or entire (the latter phenotype is named non-lobed throughout this study) [ <a href="#pone.0180741.ref003">3</a>, <a href="#pone.0180741.ref011">11</a>]. In general, the leaf shape character can be easily identified at the seedling stage and thus, it can be used as an efficient morphological marker to distinguish hybrids from parental lines, consequently ensuring the purity of hybrid seeds [ <a href="#pone.0180741.ref003">3</a>, <a href="#pone.0180741.ref012">12</a>]. </p> <p>Leaf shape morphogenesis is a complex process and several genes have been identified that mediate its development [ <a href="#pone.0180741.ref013">13</a>]. In <i>Cardamine hirsuta</i>, the class I KNOTTED1-like homeobox (KNOX) proteins organize auxin maxima via the PINFORMED1 (PIN1) auxin efflux transporter, thus promoting leaflet initiation [ <a href="#pone.0180741.ref014">14</a>]; the transcriptional level of <i>KNOX</i> can be repressed by <i>CLAUSA</i> and <i>TRIPINNATE</i> gene products, thus affecting the leaflet number in tomato [ <a href="#pone.0180741.ref015">15</a>, <a href="#pone.0180741.ref016">16</a>]. In Arabidopsis, the transcription factor <i>CUP-SHAPED COTYLEDON</i> rendered the genes <i>CUC1</i>, <i>CUC2</i>, and <i>CUC3</i> redundant, but showed partially distinct functions in embryonic shoot meristem formation and cotyledon boundary specification [ <a href="#pone.0180741.ref017">17</a>]; gene <i>CUC2</i> can be targeted by miR164A, and the balance between <i>CUC2</i> and miR164A determines the extent of leaf serration [ <a href="#pone.0180741.ref018">18</a>]. Loss-of-function mutations of the gene <i>Mt-AGO7/LOBED LEAFLET1</i> result in lobed leaf margins in <i>Medicago truncatula</i> [ <a href="#pone.0180741.ref019">19</a>]. The hormone cytokinin (CK) has been reported to mediate the activity of KNOX1 proteins that aid the regulation of leaf shape in tomato [ <a href="#pone.0180741.ref020">20</a>]. The homeodomain leucine-zipper transcription factor <i>LMI1</i> ( <i>AT5G03790</i>) is a meristem identity regulator that interacts with LFY to activate <i>CAL</i> expression, providing additional functions in the formation of simple serrated leaves and in suppressing bract formation [ <a href="#pone.0180741.ref021">21</a>]. A previous study has demonstrated that the gene <i>RCO</i> ( <i>REDUCED COMPLEXITY</i>, a <i>LMI1-like</i> homologue) is also required for leaflet development in <i>C</i>. <i>hirsuta</i> [ <a href="#pone.0180741.ref022">22</a>]. </p> <p>To date, several lobed leaf genes have been genetically analyzed and mapped within different species. In rapeseed ( <i>Brassica napus</i> L), the <i>LOBED-LEAF 1</i> ( <i>BnLL1</i>) gene has been mapped to a 36.7-kb region [ <a href="#pone.0180741.ref003">3</a>]. The major leaf shape gene ( <i>L</i>) in cotton ( <i>Gossypium hirsutum</i> L.) has been mapped to a 5.4 cM interval at the distal region of the short-arm chromosome 15 and two <i>LMI1-like</i> genes have been identified as the most likely candidate genes [ <a href="#pone.0180741.ref006">6</a>]. A previous study reported the HD-zip transcription factor <i>GhLMI1-D1b</i> ( <i>Gorai</i>. <i>002G244000</i>) to be responsible for the majority of leaf shapes in cotton [ <a href="#pone.0180741.ref023">23</a>]. The semi-dominant allele <i>lma</i> has been mapped to a 376-kb syntenic region on chromosome 3 in the mungbean ( <i>Vigna radiata</i> L.) [ <a href="#pone.0180741.ref008">8</a>]. With the use of a combination of bulk segregant analysis (BSA) and next-generation sequencing, eight QTL traits have been detected in the <i>Mimulus guttatus</i> species complex [ <a href="#pone.0180741.ref024">24</a>]. In the Cucurbitaceae family, three lobed leaf genes have been reported. For example, the lobed leaf phenotype is controlled by a recessive gene in <i>Cucurbita maxima</i> (designated <i>lo-1</i>), while in <i>Cucurbita ecuadorensis</i>, the dominant allele <i>Lo-2</i> controls the phenotype [ <a href="#pone.0180741.ref025">25</a>, <a href="#pone.0180741.ref026">26</a>]. In melon ( <i>Cucumis melo</i> L.), a single recessive gene ( <i>pll</i>) contributes to the palmately lobed leaf trait and it has been located in a 14.6-kb region on CM3.5_scaffold00014 [ <a href="#pone.0180741.ref002">2</a>]. However, as far as we know, no lobed leaf gene has been cloned within any Cucurbitaceae species. </p> <p>Watermelon ( <i>Citrullus lanatus</i> L., 2n = 2x = 22) is a globally important cucurbit crop, accounting for 7% of the worldwide area devoted to vegetable production [ <a href="#pone.0180741.ref027">27</a>]. Currently, lobed leaf watermelon varieties dominate the market. In this study, we obtained a F <sub>2:3</sub> leaf shape segregating population during the breeding process. Subsequently, we constructed a larger F <sub>3:4</sub> population to fine map the lobed leaf trait in watermelon. Using a combination of BSA and the RNA-seq method, SNPs and indels were identified genome-wide. Finally, using validated SNP and indel markers, the gene <i>ClLL1</i> was mapped within a 127.6-kb interval, containing 23 putative open reading frames (ORFs). On the basis of functional annotation and qRT-PCR analysis, genes <i>ORF18</i> and <i>ORF22</i> were considered as potential <i>ClLL1</i> candidates. Further analysis revealed no polymorphisms within the cDNA sequences of <i>ORF18</i> among three watermelon genomes; however, two notable deletions were identified in <i>ORF22</i>. This study is the first to report a genetic map of the lobed leaf trait in watermelon and thus provides central information for further isolation and characterization of the gene <i>ClLL1</i>. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="plantmaterialsandphenotypicdatacollection" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and phenotypic data collection</div> <p>During the breeding process, a single plant of the commercial watermelon hybrid cultivar ‘Lingxiu’ was self-fertilized, thus obtaining F <sub>2</sub> seeds. Then, a F <sub>2</sub>-derived F <sub>3</sub> population (denoted as F <sub>2:3</sub>) was generated by self-pollinating F <sub>2</sub> plants, which exhibited leaf shape polymorphisms (lobed and non-lobed leaves). Due to seed number limitations, the F <sub>2:3</sub> plants were self-pollinated to produce a larger-sized F <sub>3:4</sub> population in July 2015, which was used to map the gene <i>ClLL1</i> in this study. </p> <p>The leaf shapes can clearly be distinguished after emergence of the sixth adult true leaf. Thus, to validate the inheritance pattern of the leaf shape, seeds of F <sub>3:4</sub> populations were germinated and cultured in plastic greenhouses at the Northwest A&amp;amp;F University under natural conditions from the autumn of 2015 to the summer of 2016. The leaf phenotype of each individual was recorded after the appearance of the sixth adult leaf. Then, the data were analyzed to evaluate the segregation ratio, using the Chi-square test. </p> </div> <div class="rnaisolationandrna-seq" title="sec"> <div class="title" tagx="title" title="title">
RNA isolation and RNA-seq</div> <p>A combined approach that utilized BSA and next generation sequencing of the transcriptome was applied in this study. Total RNA from young leaves of 31 and 23 lobed and non-lobed leaf seedlings at the sixth-leaf stage were extracted using the Trizol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer’s instructions. All contaminating genomic DNA was removed via RNase-free DNaseI (TaKaRa, Dalian, China). Then, the equivalent total RNA of 31 lobed leaf and 23 non-lobed leaf individuals were pooled, constituting the lobed leaf and non-lobed leaf bulks, respectively. RNA sequencing was performed on an Illumina HiSeq™ 2000 platform and 125 bp paired-end reads were generated by Gene Denovo Co. (Guangzhou, China). </p> </div> <div class="dataprocessingandanalysis" title="sec"> <div class="title" tagx="title" title="title">
Data processing and analysis</div> <p>Using an in-house Perl program, the raw data were filtered to remove all unusable reads, which included reads that contained the Illumina library construction adapters, reads with more than 10% unknown bases, and reads with more than 50% low quality bases (Q-value ≤ 10). An index file for the watermelon reference genome "97103" ( <a href="http://www.icugi.org/">http://www.icugi.org/</a>) was constructed with the software package Bowtie2 [ <a href="#pone.0180741.ref027">27</a>, <a href="#pone.0180741.ref028">28</a>]. Then, high quality (HQ) clean reads were aligned to the reference genome using the software package Tophat2 with default parameters [ <a href="#pone.0180741.ref029">29</a>]. Raw SNPs and indels calling were performed via SAMtools [ <a href="#pone.0180741.ref030">30</a>]. Reliable SNPs and indels information were obtained via filtering raw sets with the following criteria: 1) mapping quality &amp;gt; 40; 2) read depth of the variant position &amp;gt; 20. All mapped reads were visually investigated using the Integrative Genomic Viewer (IGV) [ <a href="#pone.0180741.ref031">31</a>]. </p> <p>The differentially expressed genes (DEGs) were identified with the R package edgeR [ <a href="#pone.0180741.ref032">32</a>]. Mapped clean reads of each gene were calculated and normalized into fragments per kilobase of exon per million mapped fragments (FPKM). The false discovery rate (FDR) was used to determine the P-value threshold in multiple tests. In this study, an FDR &amp;lt; 0.05 and a fold change &amp;gt; 2 were used to judge the significance of the gene expression differences. DEGs were used for GO and KEGG enrichment analyses, similar to a method described by Zhang [ <a href="#pone.0180741.ref033">33</a>]. Both GO terms and KEGG pathways with ≤ Q-values 0.05 were significantly enriched in DEGs. </p> </div> <div class="markerdevelopmentandpcrreaction" title="sec"> <div class="title" tagx="title" title="title">
Marker development and PCR reaction</div> <p>Informed by reliable SNPs and indels, the corresponding cleaved amplified polymorphic sequence (CAPS) markers were developed and the primers were designed with Primer Premier 5 ( <a href="http://www.premierbiosoft.com/">http://www.premierbiosoft.com/</a>). </p> <p>Genomic DNA was extracted from young leaves, using the CTAB method [ <a href="#pone.0180741.ref034">34</a>, <a href="#pone.0180741.ref035">35</a>]. Subsequent to determining the concentration of each DNA sample, approximately 200 ng genomic DNA was used as template for the PCR reaction. PCR amplification was done in a 20 μl reaction with 1 μl DNA, 10 μl PCR master mix (TSINGKE, Beijing, China), 0.5 μl of 10 μM per primer, and 8 μl distilled water. The following PCR protocol was utilized: initial denaturation at 94°C for 5 min, followed by 28 cycles at 94°C for 30 s, 57°C for 30 s, 72°C for 1 min, and a final extension at 72°C for 5 min. All PCR products were separated on 2% agarose gels. Markers exhibiting length polymorphisms were directly used for the genetic mapping, while the rest were digested with their specific endonuclease at 37°C or 65°C for 4 h, followed by analysis on agarose gel. </p> </div> <div class="mappingstrategy" title="sec"> <div class="title" tagx="title" title="title">
Mapping strategy</div> <p>For primary linkage analysis, we screened 93 F <sub>3:4</sub> individuals with 11 polymorphic markers, which were designed for each chromosome according to SNPs identified from RNA-seq data. Subsequent to obtaining the linked marker, new flanking markers were designed to screen the small F <sub>3:4</sub> population. After delimiting the target gene between two markers, a larger population consisting of 781 F <sub>3:4</sub> individuals, was used to identify recombinants. All recombinants identified from both populations were carefully transplanted into the field and used to extract a large amount of genomic DNA. To further localize the primary interval, a series of new markers were developed based on reliable SNPs and indels (after removing the low quality data, using the criteria mentioned above). These markers were subsequently used to screen the recombinants. Markers used in the gene mapping strategy are listed in <a href="#pone.0180741.t001">Table 1</a>. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0180741.t001">10.1371/journal.pone.0180741.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
List of all primers for 12 markers used in genetic mapping of <i>ClLL1</i>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Marker name</th> <th>Physical location</th> <th>Primer sequence (5-&amp;gt;3)</th> <th>PCR product (bp)</th> <th>Endonuclease</th> <th>Annealing temp (°C)</th> <th>Marker type</th> </tr> </thead> <tbody> <tr> <td>W01144</td> <td>Chr04:20818124</td> <td>F: <tt>ACCAAGCTACCCCAACCCACC</tt> </td> <td>932</td> <td>BamHI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>TCCAAGATTGGGAGGCGGTGC</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W01211</td> <td>Chr04:19907542</td> <td>F: <tt>AGGCCTGAGAATGCTCTGGGGA</tt> </td> <td>966</td> <td>XhoI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>TTCCTCGGGACCGACACGGAG</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W03041</td> <td>Chr04:20857870</td> <td>F: <tt>TAGACTGGGCGGAAGAGACGGC</tt> </td> <td>640</td> <td>TaqI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>ACTAATCCATCCCCGAGCACACCT</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W03042</td> <td>Chr04:20889818</td> <td>F: <tt>ACCATACAGCGCTGAAACTCTGCT</tt> </td> <td>664</td> <td>BstNI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>GCTTGCCTCCAGCTTCGCATGA</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W06252</td> <td>Chr04:20914978</td> <td>F: <tt>GCCAAACTAATCATACATACAG</tt> </td> <td>920</td> <td>EcoRV</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>GACATCAATAACATCCCAAGA</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W08314</td> <td>Chr04:21111771</td> <td>F: <tt>TTATTCTCAATAAACGCCCTTCCCTAGTA</tt> </td> <td>140</td> <td>EcoRV</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>CAGCGACATTTTGCAATATTTGAAGATAT</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W07061</td> <td>Chr04:21239403</td> <td>F: <tt>TGGTTGAGGCCGAAGAGGTTGGT</tt> </td> <td>844</td> <td>RsaI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>TTATGGGCATGCAGTGTGGGGC</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W07062</td> <td>Chr04:21262238</td> <td>F: <tt>GATTTGGCGTTACATCTGCTT</tt> </td> <td>789</td> <td>TaqI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>GTTACATTTGAGACATTTGGGT</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W07063</td> <td>Chr04:21294844</td> <td>F: <tt>TTTCATTTGGTCCCTATGTTT</tt> </td> <td>275</td> <td /> <td>57</td> <td>SCAR</td> </tr> <tr> <td /> <td /> <td>R: <tt>AACCTCAATTATTAACTAACTACTCAC</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W06253</td> <td>Chr04:21505058</td> <td>F: <tt>AAAGGCTTGGATTATGGAATT</tt> </td> <td>753</td> <td>PstI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>AAAAGTTGTGGGTTAGGGAAT</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W0130</td> <td>Chr04:22250767</td> <td>F: <tt>GCCCTGGCCGGACACAGGATA</tt> </td> <td>535</td> <td>XagI</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>CGACGTCGTCAAAGCCAGCATC</tt> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td>W01214</td> <td>Chr04:23498056</td> <td>F: <tt>TGGGCTCAAGACATAGAACATGCCA</tt> </td> <td>538</td> <td>HindIII</td> <td>57</td> <td>CAPS</td> </tr> <tr> <td /> <td /> <td>R: <tt>AGCTCATCCCACCGATTCATGTGT</tt> </td> <td /> <td /> <td /> <td /> </tr> </tbody> </table> </div> </div> </div> <div class="genepredictionandqtr-pcr" title="sec"> <div class="title" tagx="title" title="title">
Gene prediction and qTR-PCR</div> <p>The predicted genes in the mapping interval were downloaded from the watermelon database ( <a href="http://www.icugi.org/">http://www.icugi.org/</a>) [ <a href="#pone.0180741.ref027">27</a>]. The genomic sequence of the mapping interval was extracted from the watermelon reference genome using an in-house Perl program. The candidate gene prediction was further performed using FGENESH ( <a href="http://linux1.softberry.com/">http://linux1.softberry.com/</a>). The function of thus predicted genes was retrieved from NCBI using the BLASTp tool ( <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). </p> <p>Quantitative real-time PCR (qRT-PCR) was performed to identify all candidate genes. Gene-specific primers were designed based on the open reading frame sequence of predicted genes, using the software Primer Premier 5 with an annealing temperature range of 57–60°C and an amplification length between 200 and 300 bp. The housekeeping gene <i>Actin</i> was used as internal control [ <a href="#pone.0180741.ref036">36</a>]. The first-strand cDNA was synthesized using a FastQuant RT kit (TIANGEN, Beijing, China) and qRT-PCR was performed on a StepOnePlus Real-Time PCR platform (Applied Biosystems, Foster City, CA, USA). The values from reactions in triplicate were analyzed with the 2 <sup>-</sup>ΔΔ <sup>Ct</sup> method [ <a href="#pone.0180741.ref037">37</a>]. SPSS 21.0 software was used for statistical analysis, and the data are presented as mean values ± SD. The differential expression significance between lobed and non-lobed leaf individuals was verified via Student’s t-test. All primer pairs used in the qRT-PCR experiment are listed in <a href="#pone.0180741.s001">S1 Table</a>. </p> </div> <div class="sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence analysis</div> <p>Total RNA was extracted from leaves of both lobed and non-lobed leave plantlets, using the TriZol reagent (Invitrogen, Carlsbad, CA, USA) and first-strand cDNA was synthesized using a FastQuant RT kit (TIANGEN, Beijing, China). To amplify full-length sequences of candidate genes, gene-specific primers were designed using the software Primer Premier 5 and the results are listed in <a href="#pone.0180741.s001">S1 Table</a>. Nested PCR was performed to amplify the cDNA sequence of <i>ORF22</i>. PCR amplification was conducted in a 25 μl reaction vessel with one unit of Fast Pfu Taq (TransGen, Beijing, China), treated for 4 min at 95°C, followed by 30 cycles at 95°C for 30 s, 58 or 60°C for 20 s, and a final extension at 72°C for 2 min. The PCR products were purified and ligated into the vector pEASY-T1, using the TA clone kit (TransGen, Beijing, China). At least three positive clones per sample were sent for sequencing. The cDNA sequences of candidate genes <i>ORF18</i> ( <i>Cla018357</i>) and <i>ORF22</i> ( <i>Cla018360</i>) were downloaded from the watermelon genome database "97103" ( <a href="http://www.icugi.org/">http://www.icugi.org/</a>). Sequence analysis was performed using the software Geneious ( <a href="http://www.geneious.com/">http://www.geneious.com</a>). </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="geneticanalysisofthelobedleaftrait" title="sec"> <div class="title" tagx="title" title="title">
Genetic analysis of the lobed leaf trait</div> <p>Individuals from a F <sub>2:3</sub> population derived from the watermelon cultivar ‘Lingxiu’ during the breeding process, showed leaf form polymorphisms (lobed and non-lobed leaves). However, due to seed number limitation, it was impossible to perform the genetic mapping of the lobed leaf gene with this population. Therefore, the F <sub>2:3</sub> plants were self-pollinated to produce a sufficiently sized F <sub>3:4</sub> population. Consistent with F <sub>2:3</sub> plants, the leaves of F <sub>3:4</sub> individuals were classified as either lobed or non-lobed ( <a href="#pone.0180741.g001">Fig 1A</a>). Moreover, the margins of non-lobed leaves were not smooth and featured slight serrations and small symmetrical marginal protrusions ( <a href="#pone.0180741.g001">Fig 1B</a>). Therefore, to further observe the phenotypic variations, the first ten true leaves from both lobed and non-lobed seedlings were compared. As shown in <a href="#pone.0180741.g001">Fig 1C</a>, the leaf shapes can be visibly distinguished after the emergence of the third true leaf and differentiated at the sixth adult leaf stage. In addition, the symmetrical marginal protrusions mentioned above can clearly be observed on the sixth true leaf from non-lobed leaf plants. A small proportion of F <sub>3:4</sub> progenies was planted to analyze the genetic inheritance of the lobed leaf trait. As a result, 93 F <sub>3:4</sub> individuals presented two phenotypes, including 69 lobed and 24 non-lobed plants, with a 3:1 Mendelian ratio (χ <sup>2</sup> = 0.03, p = 0.86). Furthermore, leaves of seedlings derived from non-lobed F <sub>2:3</sub> individuals had a leaf form consistent with the parent plant. In summary, we inferred that a dominant allele, designated as <i>ClLL1</i>, controlled the lobed leaf trait in watermelon. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0180741.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Phenotypes of leaf shapes in watermelon.</div> <p>( <b>a)</b> Lobed and non-lobed leaf watermelon seedlings. ( <b>b)</b> Two different leaf shapes in watermelon. Small symmetrical marginal protrusions are marked with red arrows. ( <b>c)</b> Comparison of the first ten true leaves from lobed and non-lobed phenotype seedlings. </p>   </div> </div> <div class="genome-wideidentificationofsnpsandindels" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide identification of SNPs and indels</div> <p>To identify the SNPs and indels genome-wide, a method combining BSA and RNA-seq was used in this study. After removing low-quality sequences from the raw data, a total of 53 and 50 million clean reads were generated from lobed and non-lobed bulks, with approximately 6.59 and 6.19 Gbp data, respectively ( <a href="#pone.0180741.t002">Table 2</a>). The sequencing data has been submitted to NCBI and can be accessed in the respective short read archives (SRR5100272 and SRR5100273). Using strict criteria to filter the clean data, high quality (HQ) reads that accounted for 96% of the clean data were generated per bulk. The Q30 values of two samples were 89.02% and 88.31%, respectively. Then, these HQ clean reads were mapped to the watermelon reference genome using the software Bowtie2 and Tophat2 [ <a href="#pone.0180741.ref028">28</a>, <a href="#pone.0180741.ref029">29</a>]. As a result, approximately 85% of all reads could be mapped onto chromosomes and a total of 16,848 SNPs and indels were identified. Then, after removing the low quality sites (read counts &amp;lt; 20; quality &amp;lt; 40), 5,966 reliable SNPs and indels were selected and utilized to develop markers in the further analysis ( <a href="#pone.0180741.t003">Table 3</a> and <a href="#pone.0180741.s002">S2 Table</a>). The distribution of SNPs and indels on chromosomes showed considerable variation. E.g., chromosome 6 had the largest number (948) of SNPs and indels, while only 218 were located on chromosome 4. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0180741.t002">10.1371/journal.pone.0180741.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Detailed information of RNA-seq data of lobed and non-lobed leaf bulks.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th>Lobed Leaf</th> <th>Non-lobed Leaf</th> </tr> </thead> <tbody> <tr> <td> <b>Clean Reads Num</b> </td> <td>52698774</td> <td>49553160</td> </tr> <tr> <td> <b>Clean Data(bp)</b> </td> <td>6587346750</td> <td>6194145000</td> </tr> <tr> <td> <b>Q30 Percentage</b> </td> <td>87.87%</td> <td>86.77%</td> </tr> <tr> <td> <b>HQ Clean Reads Num</b> </td> <td>51015830</td> <td>47474006</td> </tr> <tr> <td> <b>HQ Clean Data(bp)</b> </td> <td>6376978750</td> <td>5934250750</td> </tr> <tr> <td> <b>Q30 Percentage</b> </td> <td>89.02%</td> <td>88.31%</td> </tr> <tr> <td> <b>Unique Mapped Reads</b> </td> <td>43340214 (84.95%)</td> <td>39878758 (84.00%)</td> </tr> <tr> <td> <b>Multiple Mapped reads</b> </td> <td>515006 (1.01%)</td> <td>455174 (0.96%)</td> </tr> <tr> <td> <b>Unmapped Reads</b> </td> <td>7134152 (13.98%)</td> <td>7108416 (14.97%)</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0180741.t003">10.1371/journal.pone.0180741.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Distribution of reliable SNPs and indels on watermelon chromosomes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th>Chr0</th> <th>Chr1</th> <th>Chr2</th> <th>Chr3</th> <th>Chr4</th> <th>Chr5</th> <th>Chr6</th> <th>Chr7</th> <th>Chr8</th> <th>Chr9</th> <th>Chr10</th> <th>Chr11</th> </tr> </thead> <tbody> <tr> <td> <b>SNP</b> </td> <td>77</td> <td>575</td> <td>363</td> <td>295</td> <td>197</td> <td>670</td> <td>881</td> <td>490</td> <td>334</td> <td>573</td> <td>435</td> <td>463</td> </tr> <tr> <td> <b>Indels</b> </td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> Insertion</td> <td>3</td> <td>30</td> <td>28</td> <td>20</td> <td>14</td> <td>39</td> <td>35</td> <td>28</td> <td>31</td> <td>31</td> <td>29</td> <td>27</td> </tr> <tr> <td> Deletion</td> <td>1</td> <td>28</td> <td>34</td> <td>22</td> <td>7</td> <td>32</td> <td>32</td> <td>34</td> <td>27</td> <td>31</td> <td>26</td> <td>24</td> </tr> <tr> <td> <b>Total</b> </td> <td>81</td> <td>633</td> <td>425</td> <td>337</td> <td>218</td> <td>741</td> <td>948</td> <td>552</td> <td>392</td> <td>635</td> <td>490</td> <td>514</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Chr0 represents unanchored supper-scaffolds</p> </div> </div> </div> </div> <div class="identificationofdegs" title="sec"> <div class="title" tagx="title" title="title">
Identification of DEGs</div> <p>The total mapped reads were used to analyze the DEGs with the criteria of FDR &amp;lt; 0.05 and fold change &amp;gt; 2. As a result, a total of 195 DEGs were identified within both bulked pools; among those, 133 were up-regulated and 62 were down-regulated (see <a href="#pone.0180741.s003">S3 Table</a>). Functional annotations of the DEGs, as well as the GO and KEGG enrichment analyses, revealed that a large proportion of transcription genes (such as the HD-zip transcription factor <i>Cla014193</i>) may be involved in watermelon leaf-shape morphogenesis. </p> </div> <div class="primarymappingofclll1" title="sec"> <div class="title" tagx="title" title="title">
Primary mapping of <i>ClLL1</i> </div> <p>To locate gene <i>ClLL1</i> on a chromosome, 11 markers were designed per chromosome (data not shown) and used to screen the F <sub>3:4</sub> population (93 individuals). Linkage analysis revealed that marker W01144 on chromosome 4 was linked to the lobed leaf trait. However, the polymorphic bands produced by W01144 were not easy to distinguish; therefore, a new marker W03041 was developed which is physically adjacent to W01144. Then, W03041 was used to genotype 93 mapping individuals and seven recombinants were detected with this technique ( <a href="#pone.0180741.g002">Fig 2A</a>). To determine the primary region for gene <i>ClLL1</i>, two flanking markers, W01211 (on the left side of W03041) and W01214 (on the right side of W03041), were developed to screen the population. Subsequent linkage analysis suggested that both markers W01211 and W03041 were located on the same side of gene <i>ClLL1</i>, with 11 and 7 recombinants and at distances of 5.9 and 3.8 cM, respectively. However, marker W01214 was situated on the other side, with 20 recombinants at a 12.9 cM genetic distance from <i>ClLL1</i> ( <a href="#pone.0180741.g002">Fig 2A</a>). Thus, we concluded gene <i>ClLL1</i> to be roughly delimited within a 2.64 Mb region between markers W03041 and W01214, with a 16.7 cM genetic distance. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0180741.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Genetic mapping of the lobed leaf trait gene <i>ClLL1</i> in watermelon. </div> <p>( <b>a)</b> Primary mapping of <i>ClLL1</i> using 93 F <sub>3:4</sub> individuals. Gene <i>ClLL1</i> was delimited to the region between markers W03041 and W01214. The numbers in brackets after the marker name indicate the numbers of recombinants. ( <b>b)</b> Fine mapping of <i>ClLL1</i>. The gene <i>ClLL1</i> was fine mapped in a l27.6-kb region between markers W08314 and W07061. ( <b>c</b>) Marker genotypes of the recombinants near the lobed leaf gene <i>ClLL1</i> between W08314 and W07061. The alleles are abbreviated according to their origin: A: Lobed leaf; B: Non-lobed leaf; H: Heterozygous. </p>   </div> </div> <div class="finemappingofclll1" title="sec"> <div class="title" tagx="title" title="title">
Fine mapping of <i>ClLL1</i> </div> <p>To precisely identify the genomic region surrounding gene <i>ClLL1</i>, a larger segregating population consisting of 781 individuals was utilized, which segregated 596 lobed and 185 non-lobed phenotypes, fitting a 3:1 ratio (χ <sup>2</sup> = 0.72, p = 0.40). The primary flanking markers W03041 and W01214 were utilized to screen this population and a further new 39 and 146 recombinants were identified. Combined with the recombinants obtained in the first population, a total of 212 recombinants were obtained. Consequently, seven new markers were designed for the primary region ( <a href="#pone.0180741.g002">Fig 2B</a>), and were used to phenotype 212 recombinant individuals. Finally, gene <i>ClLL1</i> was delimited to be between markers W08314 and W07061, with 18 and 2 recombinants and at genetic distances of 1.15 and 0.13 cM, respectively ( <a href="#pone.0180741.g002">Fig 2B and 2C</a>). Since we had no reliable SNPs or indels to develop new polymorphic markers, further localization of this mapping region was unfeasible. Based on the genetic distance between gene <i>ClLL1</i> and both flanking markers, we inferred the location of <i>ClLL1</i> to be closer to W07061. The physical distance between W08314 and W07061 was approximately 127.6-kb, according to the watermelon reference genome sequence. </p> </div> <div class="candidategenesforclll1" title="sec"> <div class="title" tagx="title" title="title">
Candidate genes for <i>ClLL1</i> </div> <p>According to the watermelon genome annotation database ( <a href="http://www.icugi.org/">http://www.icugi.org/</a>), 14 putative genes ( <i>Cla018348</i> to <i>Cla018361</i>) were annotated in the candidate region ( <a href="#pone.0180741.t004">Table 4</a>). Gene prediction was further conducted via FGENESH ( <a href="http://softberry.com/">http://softberry.com</a>), yielding nine additional putative ORFs. Then, all 23 putative amino acid sequences were subjected to BLASTP (NCBI), revealing that seven out of these nine ORFs ( <i>ORF2</i>, <i>4</i>, <i>8</i>, <i>12</i>, <i>15</i>, <i>17</i>, and <i>21</i>) identified by FGENESH possessed no effective annotations (E-value cutoff of 1e <sup>-10</sup>), and were consequently discarded for future analysis. <i>ORF5</i> ( <i>Cla018350</i>), <i>ORF9</i>, and <i>ORF18</i> ( <i>Cla018357</i>) encode unknown proteins ( <a href="#pone.0180741.t004">Table 4</a>). The <i>ORF1</i> ( <i>Cla018348</i>) protein exhibited 89% similarity with a predicted aspartic proteinase-like protein. <i>ORF3</i> ( <i>Cla018349</i>), <i>ORF6</i> ( <i>Cla018351</i>), <i>ORF7</i> ( <i>Cla018352</i>), and <i>ORF10</i> ( <i>Cla018353</i>) are homologues, sharing at least 55% amino acid sequence identity and encoding a putative threonine dehydratase. <i>ORF11</i> ( <i>Cla018354</i>) encodes a GDSL-motif lipase/hydrolase family protein. The remaining genes encode a pentatricopeptide repeat protein ( <i>ORF13</i>), beta-galactosidase-like protein ( <i>ORF14</i>), glycosyltransferase ( <i>ORF16</i>), serine/threonine protein kinase ( <i>ORF19</i>), 60S ribosomal protein L24 ( <i>ORF20</i>), homeobox-leucine zipper-like protein ( <i>ORF22</i>), and pyruvate kinase ( <i>ORF23</i>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0180741.t004">10.1371/journal.pone.0180741.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
Predicted genes between markers W08314 and W07061.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>ORF. no</th> <th>Position</th> <th>CDS</th> <th>Gene id</th> <th>NCBI BlastP Hit</th> </tr> </thead> <tbody> <tr> <td> <i>ORF1</i> </td> <td>Chr4:21115062..21120010</td> <td>1567</td> <td> <i>Cla018348</i> </td> <td>Aspartyl protease-like protein</td> </tr> <tr> <td> <i>ORF2</i> </td> <td>Chr4:21126988..21129534</td> <td>51</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF3</i> </td> <td>Chr4:21130256..21133784</td> <td>1845</td> <td> <i>Cla018349</i> </td> <td>Threonine dehydratase</td> </tr> <tr> <td> <i>ORF4</i> </td> <td>Chr4:21135471..21139884</td> <td>150</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF5</i> </td> <td>Chr4:21142412..21142771</td> <td>359</td> <td> <i>Cla018350</i> </td> <td>Unknown Protein</td> </tr> <tr> <td> <i>ORF6</i> </td> <td>Chr4:21143742..21147413</td> <td>1414</td> <td> <i>Cla018351</i> </td> <td>Threonine dehydratase</td> </tr> <tr> <td> <i>ORF7</i> </td> <td>Chr4:21156327..21160590</td> <td>1839</td> <td> <i>Cla018352</i> </td> <td>Threonine dehydratase</td> </tr> <tr> <td> <i>ORF8</i> </td> <td>Chr4:21161184..21165631</td> <td>198</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF9</i> </td> <td>Chr4:21166844..21167799</td> <td>267</td> <td /> <td>uncharacterized protein</td> </tr> <tr> <td> <i>ORF10</i> </td> <td>Chr4:21171749..21176612</td> <td>1887</td> <td> <i>Cla018353</i> </td> <td>Threonine dehydratase</td> </tr> <tr> <td> <i>ORF11</i> </td> <td>Chr4:21178315..21180439</td> <td>1051</td> <td> <i>Cla018354</i> </td> <td>GDSL-motif lipase/hydrolase family protein</td> </tr> <tr> <td> <i>ORF12</i> </td> <td>Chr4:21181007..21182362</td> <td>312</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF13</i> </td> <td>Chr4:21183583..21186210</td> <td>2627</td> <td> <i>Cla018355</i> </td> <td>Pentatricopeptide repeat protein</td> </tr> <tr> <td> <i>ORF14</i> </td> <td>Chr4:21187818..21189921</td> <td>480</td> <td /> <td>Beta-galactosidase-like</td> </tr> <tr> <td> <i>ORF15</i> </td> <td>Chr4:21191643..21193351</td> <td>87</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF16</i> </td> <td>Chr4:21194156..21197263</td> <td>1585</td> <td> <i>Cla018356</i> </td> <td>Glycosyltransferase</td> </tr> <tr> <td> <i>ORF17</i> </td> <td>Chr4:21200223..21205764</td> <td>165</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF18</i> </td> <td>Chr4:21208684..21212333</td> <td>1357</td> <td> <i>Cla018357</i> </td> <td>uncharacterized protein</td> </tr> <tr> <td> <i>ORF19</i> </td> <td>Chr4:21213450..21215972</td> <td>1637</td> <td> <i>Cla018358</i> </td> <td>Serine/threonine protein kinase</td> </tr> <tr> <td> <i>ORF20</i> </td> <td>Chr4:21220810..21222360</td> <td>490</td> <td> <i>Cla018359</i> </td> <td>60S ribosomal protein L24</td> </tr> <tr> <td> <i>ORF21</i> </td> <td>Chr4:21223829..21231305</td> <td>123</td> <td /> <td>No annotation</td> </tr> <tr> <td> <i>ORF22</i> </td> <td>Chr4:21232016..21234402</td> <td>699</td> <td> <i>Cla018360</i> </td> <td>Homeobox-leucine zipper-like protein</td> </tr> <tr> <td> <i>ORF23</i> </td> <td>Chr4:21242282..21257426</td> <td>1568</td> <td> <i>Cla018361</i> </td> <td>Pyruvate kinase</td> </tr> </tbody> </table> </div> </div> <p>To further determine possible candidates, we designed specific primers associated with 16 predicted genes (except for <i>ORF2</i>, <i>ORF4</i>, <i>ORF8</i>, <i>ORF12</i>, <i>ORF15</i>, <i>ORF17</i>, and <i>ORF21</i>) in the 127.6-kb region, and analyzed their expression levels in both lobed and non-lobed plants ( <a href="#pone.0180741.g003">Fig 3A</a>). The results indicated that the transcription of <i>ORF5</i>, <i>ORF6</i>, and <i>ORF9</i> could not be detected. <i>ORF1</i>, <i>ORF10</i>, <i>ORF14</i>, <i>ORF20</i>, <i>ORF22</i>, and <i>ORF23</i> revealed no expression differences between lobed and non-lobed plants ( <a href="#pone.0180741.g003">Fig 3B</a>). Using fold-changes above 2-fold or below 0.5-fold as thresholds, only <i>ORF11</i> was expressed at a significantly higher level (~2.9 fold) in non-lobed leaf plants compared to lobed leaf plants. <i>ORF11</i> was homologous to gene <i>AT5G03820</i> in Arabidopsis, which belongs to the GDSL-motif lipase/hydrolase family and may be involved in seed morphology [ <a href="#pone.0180741.ref038">38</a>]. The expression level of two genes ( <i>ORF3</i> and <i>ORF18</i>) was repressed in non-lobed plants, with 0.36 and 0.31 fold changes compared to lobed plants. <i>ORF3</i> encodes a threonine dehydratase protein, which is homologous to protein OMR1, catalysing the deamination and dehydration of threonine [ <a href="#pone.0180741.ref039">39</a>]. <i>ORF18</i> encodes an uncharacterized protein with unknown function. It is worth to note that <i>ORF22</i> with identical expression levels in both samples encodes a homeobox-leucine zipper-like protein ( <a href="#pone.0180741.g003">Fig 3B</a> and <a href="#pone.0180741.t004">Table 4</a>). It is a homologue of the gene <i>LMI1</i> ( <i>AT5G03790</i>), which has been reported to interact with LFY and CAL, functioning as a meristem identity regulator [ <a href="#pone.0180741.ref003">3</a>, <a href="#pone.0180741.ref021">21</a>]. A previous study confirmed that gene <i>RCO</i> (homologous to <i>LMI1</i>) was required for leaflet development in <i>C</i>. <i>hirsuta</i> [ <a href="#pone.0180741.ref022">22</a>]. Moreover, <i>LMI1</i> homologues <i>Bra009510</i> in rapeseed as well as <i>Gorai</i>. <i>002G244000</i> in cotton were identified as most likely candidate genes for leaf shape loci in these species [ <a href="#pone.0180741.ref003">3</a>, <a href="#pone.0180741.ref006">6</a>, <a href="#pone.0180741.ref023">23</a>]. In summary, according to gene annotation and expression analysis, as well as the genetic distance between gene <i>ClLL1</i> and two flanking markers, we inferred that both <i>ORF18</i> (encoding an unknown protein) and <i>ORF22</i> (homologous to <i>LMI1</i>) are two possible candidate genes for the lobed leaf shape trait of watermelon. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0180741.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Prediction and relative expression level of candidate genes in the <i>ClLL1</i> region. </div> <p>( <b>a)</b> 23 putative ORFs were predicted in a 127.6-kb region between makers W08314 and W07061. ( <b>b)</b> The relative expression level of candidate genes in both lobed and non-lobed leaf plants. The data are presented as average values of three replicates (mean value ± SD). "*, **" represent significant differences at <i>p</i> &amp;lt; 0.05 and <i>p</i> &amp;lt; 0.01, respectively, according to the Student’s t-test. <i>Actin</i> was used as an internal control. </p>   </div> <p>To verify this assumption, we analyzed the nucleotide sequence polymorphisms of the two possible candidate genes. Based on the RNA-seq data, the coverage of mapped reads on these predicted genes was visually investigated, using the Integrative Genomics Viewer (IGV) software ( <a href="#pone.0180741.s004">S1 Fig</a>). As a result, only the 14 ORFs annotated in the watermelon genome database were covered by sequenced reads. However, no reliable SNPs or indels were detected in these predicted genes. It is worth to note that <i>ORF5</i>, <i>ORF6</i>, and <i>ORF22</i> had only few mapped reads, which may affect the identification of nucleotide polymorphisms in these genes. To further identify DNA polymorphisms among leaf shapes, we designed gene specific primers for the two candidate genes ( <i>ORF18</i> and <i>ORF22</i>) and amplified full-length cDNA sequences from lobed and non-lobed leaf individuals. Additionally, cDNA sequences of both <i>ORF18</i> and <i>ORF22</i> were downloaded from the reference genome database of the watermelon cultivar "97103". Sequence alignment with <i>ORF18</i> showed no nucleotide polymorphisms among three genomes ( <a href="#pone.0180741.g004">Fig 4</a>). Sequence analysis of <i>ORF22</i> identified two prominent polymorphisms among three genomes. First, a 27-bp deletion (encoding nine amino acids) was found to be located at the start of the second exon in both lobed and non-lobed leaf genomes. The second notable polymorphism was a 24-bp deletion (encoding eight amino acids) at the end of the second exon was found only in the non-lobed leaf genome ( <a href="#pone.0180741.g004">Fig 4</a>). It is worth to note that neither of these two deletions resulted in a translation frameshift ( <a href="#pone.0180741.s005">S2 Fig</a>). However, the 24-bp deletion that encoded eight amino acids in the leucine zipper (LZ) motif may disturb the characteristic spacing of the leucine zipper and interfere with gene function. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0180741.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Nucleotide polymorphisms of <i>ORF18</i> and <i>ORF22</i> among three watermelon genomes. </div> <p>No nucleotide polymorphisms were identified in the cDNA sequences of <i>ORF18</i> among three genomes. Two deletions (27-bp and 24-bp) were found in the second exon of <i>ORF22</i> among three genomes. The homeodomain (HD) domain and leucine zipper (LZ) motif of <i>ORF22</i> were predicted by the software Pfam ( <a href="http://pfam.xfam.org/">http://pfam.xfam.org/</a>). </p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The watermelon is an important cucurbit crop planted widely throughout the world [ <a href="#pone.0180741.ref027">27</a>]. Currently, lobed leaf shape watermelon varieties dominate the market. However, both genetic control and underlying mechanisms that lead to the formation of lobed leaf shape in watermelon are still poorly understood. In this study, we revealed that the lobed leaf trait in watermelon was controlled by a single dominant allele, named <i>ClLL1</i>. Environmental factors, such as strong fluctuations in temperature or light regimens, were reported to adversely affect both leaf growth processes and leaf shape [ <a href="#pone.0180741.ref010">10</a>]. For example, the palmately lobed leaf trait in melon, another important cucurbit crop, was controlled by the single recessive gene <i>pll</i>, which is only expressed in individuals grown in the field [ <a href="#pone.0180741.ref002">2</a>]. However, the lobed leaf phenotype in watermelon can steadily be observed after the emergence of the sixth true leaf in plants either grown in the field or in artificial climate chambers and greenhouse conditions, indicating that the expression of gene <i>ClLL1</i> is not or only slightly affected by environmental factors. It has been reported that the cucurbit genome speciation event occurred 15–23 million years ago [ <a href="#pone.0180741.ref027">27</a>]. Therefore, it will be interesting to further investigate the underlying mechanism of leaf shape formation in these two cucurbit relatives. </p> <p>In the present study, a combined BSA and RNA-seq approach was utilized for genome-wide identification of SNPs and indels between two bulked pools, which has widely been used to develop molecular markers in gene mapping [ <a href="#pone.0180741.ref040">40</a>– <a href="#pone.0180741.ref042">42</a>]. Moreover, a total of 195 DEGs were identified, including 133 up-regulated and 62 down-regulated genes ( <a href="#pone.0180741.s003">S3 Table</a>). Using a F <sub>3:4</sub> population (N = 874) and 12 molecular markers, we successfully delimited <i>ClLL1</i> to a 127.6-kb interval between markers W08314 and W07061 ( <a href="#pone.0180741.g002">Fig 2</a>). The mapping region could not be further narrowed down, due to the limitations of reliable SNPs and indels within this interval. Please note that W08314 and W07061 located at 1.15 and 0.13 cM genetic distances from the lobed leaf trait, respectively, inferred gene <i>ClLL1</i> to be closer to W07061. Sequence annotation analysis showed that there are 23 putative ORFs in this region ( <a href="#pone.0180741.t004">Table 4</a>). According to SNPs and indels identified from RNA-seq data, we detected no reliable nucleotide polymorphisms in these predicted genes ( <a href="#pone.0180741.s004">S1 Fig</a>). Using qRT-PCR assays, we found that <i>ORF11</i> was expressed at a substantially higher level (more than twofold) in non-lobed leaf plants compared to lobed leaf plants, while the expression level of <i>ORF3</i> and <i>ORF18</i> had decreased (more than twofold) in the former ( <a href="#pone.0180741.g003">Fig 3</a>). <i>ORF11</i> encoding a GDSL-motif lipase/hydrolase family protein exhibited higher amino acid similarity to <i>AT5G03820</i> in Arabidopsis. Gene <i>AT5G03820</i> has been reported to be down-regulated by the bHLH transcription factor gene <i>RGE1</i>, possibly causing disordered hormone flux in the endosperm to function in the seed morphology [ <a href="#pone.0180741.ref038">38</a>]. <i>ORF3</i> is homologous to the threonine dehydratase protein OMR1. A previous study indicated that OMR1 catalyses the deamination and dehydration of threonine, which is the first and also the committed step in the biosynthesis of isoleucine [ <a href="#pone.0180741.ref039">39</a>]. <i>ORF18</i> encodes an uncharacterized protein, with significantly repressed expression level in non-lobed leaf plants. It is worth mentioning that <i>ORF22</i> is a <i>LMI1</i> ( <i>AT5G03790</i>) homologue in Arabidopsis, whose function is involved in the formation of simple serrated leaves [ <a href="#pone.0180741.ref021">21</a>]. It has been reported that the gene <i>RGO</i> (a homologue of <i>LMI1</i>) plays an important role in the leaflet development in <i>C</i>. <i>hirsuta</i> [ <a href="#pone.0180741.ref022">22</a>], and the most possible candidate genes of the lobed leaf trait in rapeseed and cotton are also characterized as <i>LMI1</i> homologues [ <a href="#pone.0180741.ref003">3</a>, <a href="#pone.0180741.ref006">6</a>, <a href="#pone.0180741.ref023">23</a>]. Using the software IGV and our RNA-seq data, we found that only few reads were mapped on gene <i>ORF22</i>, which may affect the detection of SNPs and indels. However, sequence alignment revealed that two notable deletions (27-bp and 24-bp) were identified in the cDNA sequences of <i>ORF22</i> among three genomes, and the second polymorphism (24-bp deletion encoding eight amino acids) in the LZ motif may disrupt the function of the allele. Therefore, based on gene annotation and sequence analysis, <i>ORF22</i> is still recognized as a candidate responsible for the lobed leaf phenotype in watermelon. </p> <p>Leaves are important photosynthetic organs of plants. To survive in different habitats, plants have the ability to adapt leaf position, size, and shape [ <a href="#pone.0180741.ref004">4</a>, <a href="#pone.0180741.ref024">24</a>]. A lobed leaf has numerous significant functional advantages in many aspects. For example, a lobed leaf has a lower hydraulic resistance, which may constitute a mechanism to improve the water balance under dry conditions [ <a href="#pone.0180741.ref043">43</a>]. The rate of heat transfer from lobed leaves is greater than in non-lobed leaves, thus reducing leaf temperature to prevent sunburn in plants [ <a href="#pone.0180741.ref002">2</a>, <a href="#pone.0180741.ref004">4</a>]. Furthermore, leaf shapes can be used in hybrid production as visible morphological markers [ <a href="#pone.0180741.ref003">3</a>]. Therefore, an improved understanding of genes and underlying regulator networks associated with lobed leaf shape may allow us to take full advantage of leaf shape in the breeding process. In this study, we performed genetic mapping of the lobed leaf gene <i>ClLL1</i> in watermelon and identified the candidate genes <i>ORF18</i> and <i>ORF22</i>, which will shed light on the molecular mechanism of lobed leaf formation in this important cucurbit crop. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Detailed information of primers used in the qRT-PCR assays and gene cloning.</div> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0180741.s001.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
All reliable SNPs and indels identified via RNA-seq data of lobe and non-lobed bulks.</div> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0180741.s002.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Table</span>  <div class="title" tagx="title" title="title">
Detailed information of differentially expressed genes.</div> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0180741.s003.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Coverage of mapped reads on the predicted genes within the <i>ClLL1</i> interval. </div> <p> <b>a</b> 23 putative ORFs in the 127.6-kb region. <b>b</b> Mapping results of RNA-seq data of lobed and non-lobed leaf bulks. Blue lines and rectangles indicate the 14 annotated genes in the watermelon genome database. Grey rectangles indicate the mapped reads from the RNA-seq data of each bulk. The visualization of mapping reads has been exhibited in IGV software. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0180741.s004.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Amino acid mutations caused by two deletions (27-bp and 24-bp) in ORF22 among three genomes. </div> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0180741.s005.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This work was supported by Scientific Startup Foundation for Doctors of Northwest A&amp;amp;F University (Z109021604), Basal Research Foundation of Northwest A&amp;amp;F University (Z109021612), and the Modern Agro-industry Technology Research System of China (No. CARS-26-18). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0180741.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsukaya</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Mechanism of leaf-shape determination</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">477</span>– <span class="lpage" tagx="lpage" title="lpage">496</span>. <span class="comment">doi: <a href="https://doi.org/10.1146/annurev.arplant.57.032905.105320">10.1146/annurev.arplant.57.032905.105320</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16669771">16669771</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">XW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ning</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">XL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">WL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">ZQ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Fine mapping of a gene that confers palmately lobed leaf ( <i>pll</i>) in melon ( <i>Cucumis melo</i> L.) </span>. <span class="source" tagx="source" title="source">Euphytica</span> <span class="volume" tagx="volume" title="volume">200</span>: <span class="fpage" tagx="fpage" title="fpage">337</span>– <span class="lpage" tagx="lpage" title="lpage">347</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">JX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic analysis and fine mapping of the <i>LOBED-LEAF 1</i> ( <i>BnLL1</i>) gene in rapeseed ( <i>Brassica napus</i> L.) </span>. <span class="source" tagx="source" title="source">Euphytica</span> <span class="volume" tagx="volume" title="volume">204</span>: <span class="fpage" tagx="fpage" title="fpage">29</span>– <span class="lpage" tagx="lpage" title="lpage">38</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nicotra</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leigh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boyce</span> <span class="given-names" tagx="given-names" title="given-names">CK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niklas</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Royer</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The evolution and functional significance of leaf shape in the angiosperms</span>. <span class="source" tagx="source" title="source">Functional Plant Biology</span> <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">535</span>– <span class="lpage" tagx="lpage" title="lpage">552</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsukaya</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Leaf development</span>. <span class="source" tagx="source" title="source">Arabidopsis Book</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e0163</span> <span class="comment">doi: <a href="https://doi.org/10.1199/tab.0163">10.1199/tab.0163</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23864837">23864837</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andres</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuraparthy</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping and genomic targeting of the major leaf shape gene ( <i>L</i>) in Upland cotton ( <i>Gossypium hirsutum</i> L.) </span>. <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="volume" tagx="volume" title="volume">127</span>: <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">177</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00122-013-2208-4">10.1007/s00122-013-2208-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24158249">24158249</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debernardi</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Morphogenesis of simple leaves: regulation of leaf size and shape</span>. <span class="source" tagx="source" title="source">Wiley Interdiscip Rev Dev Biol</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">41</span>– <span class="lpage" tagx="lpage" title="lpage">57</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/wdev.115">10.1002/wdev.115</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24902833">24902833</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Genome re-sequencing of two accessions and fine mapping the locus of lobed leaflet margins in mungbean </span>. <span class="source" tagx="source" title="source">Molecular Breeding</span> <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">128</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsukaya</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Leaf shape: genetic controls and environmental factors</span>. <span class="source" tagx="source" title="source">Int J Dev Biol</span> <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">547</span>– <span class="lpage" tagx="lpage" title="lpage">555</span>. <span class="comment">doi: <a href="https://doi.org/10.1387/ijdb.041921ht">10.1387/ijdb.041921ht</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16096964">16096964</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walter</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schurr</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Dynamics of leaf and root growth: endogenous control versus environmental impact</span>. <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">891</span>– <span class="lpage" tagx="lpage" title="lpage">900</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/aob/mci103">10.1093/aob/mci103</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15767269">15767269</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goliber</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kessler</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bharathan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sinha</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic, molecular, and morphological analysis of compound leaf development</span>. <span class="source" tagx="source" title="source">Current topics in developmental biology</span> <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">259</span>– <span class="lpage" tagx="lpage" title="lpage">290</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pu</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">SZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">CK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">JQ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Inheritance of divided leaf trait of rapeseed ( <i>Brassica napus</i>) and application in hybrid breeding </span>. <span class="source" tagx="source" title="source">Chinese Journal of Oil Crop Science</span> <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">60</span>– <span class="lpage" tagx="lpage" title="lpage">62</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sluis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hake</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Organogenesis in plants: initiation and elaboration of leaves</span>. <span class="source" tagx="source" title="source">Trends Genet</span> <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">300</span>– <span class="lpage" tagx="lpage" title="lpage">306</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.tig.2015.04.004">10.1016/j.tig.2015.04.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26003219">26003219</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barkoulas</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hay</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kougioumoutzi</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsiantis</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">A developmental framework for dissected leaf formation in the Arabidopsis relative <i>Cardamine hirsuta</i> </span>. <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">1136</span>– <span class="lpage" tagx="lpage" title="lpage">1141</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/ng.189">10.1038/ng.189</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19165928">19165928</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jasinski</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tattersall</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsiantis</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Negative regulation of KNOX expression in tomato leaves</span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">226</span>: <span class="fpage" tagx="fpage" title="fpage">1255</span>– <span class="lpage" tagx="lpage" title="lpage">1263</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00425-007-0572-5">10.1007/s00425-007-0572-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17628827">17628827</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hay</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsiantis</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">A KNOX family TALE</span>. <span class="source" tagx="source" title="source">Curr Opin Plant Biol</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">593</span>– <span class="lpage" tagx="lpage" title="lpage">598</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.pbi.2009.06.006">10.1016/j.pbi.2009.06.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19632142">19632142</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hibara</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karim</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takada</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taoka</span> <span class="given-names" tagx="given-names" title="given-names">KI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furutani</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Arabidopsis <i>CUP-SHAPED COTYLEDON3</i> regulates postembryonic shoot meristem and organ boundary formation </span>. <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">2946</span>– <span class="lpage" tagx="lpage" title="lpage">2957</span>. <span class="comment">doi: <a href="https://doi.org/10.1105/tpc.106.045716">10.1105/tpc.106.045716</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17122068">17122068</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nikovics</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blein</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peaucelle</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morin</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adia</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">The balance between the <i>MIR164A</i> and <i>CUC2</i> genes controls leaf margin serration in Arabidopsis </span>. <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">2929</span>– <span class="lpage" tagx="lpage" title="lpage">2945</span>. <span class="comment">doi: <a href="https://doi.org/10.1105/tpc.106.045617">10.1105/tpc.106.045617</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17098808">17098808</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">The trans-acting short interfering RNA3 pathway and no apical meristem antagonistically regulate leaf margin development and lateral organ separation, as revealed by analysis of an <i>argonaute7/lobed leaflet1</i> mutant in <i>Medicago truncatula</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">4845</span>– <span class="lpage" tagx="lpage" title="lpage">4862</span>. <span class="comment">doi: <a href="https://doi.org/10.1105/tpc.113.117788">10.1105/tpc.113.117788</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24368797">24368797</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shani</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben-Gera</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shleizer-Burko</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burko</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiss</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ori</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Cytokinin regulates compound leaf development in tomato</span>. <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">3206</span>– <span class="lpage" tagx="lpage" title="lpage">3217</span>. <span class="comment">doi: <a href="https://doi.org/10.1105/tpc.110.078253">10.1105/tpc.110.078253</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20959562">20959562</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saddic</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huvermann</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bezhani</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Winter</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwon</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">The LEAFY target <i>LMI1</i> is a meristem identity regulator and acts together with LEAFY to regulate expression of <i>CAULIFLOWER</i> </span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">1673</span>– <span class="lpage" tagx="lpage" title="lpage">1682</span>. <span class="comment">doi: <a href="https://doi.org/10.1242/dev.02331">10.1242/dev.02331</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16554366">16554366</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vlad</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kierzkowski</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rast</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vuolo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ioio</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galinha</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Leaf shape evolution through duplication, regulatory diversification, and loss of a homeobox gene </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">343</span>: <span class="fpage" tagx="fpage" title="fpage">780</span>– <span class="lpage" tagx="lpage" title="lpage">783</span>. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1248384">10.1126/science.1248384</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24531971">24531971</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andres</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coneva</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frank</span> <span class="given-names" tagx="given-names" title="given-names">MH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuttle</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samayoa</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Modifications to a <i>LATE MERISTEM IDENTITY1</i> gene are responsible for the major leaf shapes of Upland cotton ( <i>Gossypium hirsutum</i> L.) </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">114</span>: <span class="fpage" tagx="fpage" title="fpage">E57</span>– <span class="lpage" tagx="lpage" title="lpage">E66</span>. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.1613593114">10.1073/pnas.1613593114</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999177">27999177</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferris</span> <span class="given-names" tagx="given-names" title="given-names">KG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rushton</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenlee</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toll</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blackman</span> <span class="given-names" tagx="given-names" title="given-names">BK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willis</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Leaf shape evolution has a similar genetic architecture in three edaphic specialists within the Mimulus guttatus species complex </span>. <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">116</span>: <span class="fpage" tagx="fpage" title="fpage">213</span>– <span class="lpage" tagx="lpage" title="lpage">223</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/aob/mcv080">10.1093/aob/mcv080</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26070644">26070644</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">KE</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">1980</span>) <span class="mixed-article-title" title="mixed-article-title">Spontaneous mutant of <i>Cucurbita maxima</i> Duch. squash with lobed leaves </span>. <span class="source" tagx="source" title="source">Genetika</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">176</span>– <span class="lpage" tagx="lpage" title="lpage">178</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrington</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> ( <span class="year" tagx="year" title="year">1988</span>) <span class="mixed-article-title" title="mixed-article-title">Inheritance of leaf and fruit characteristics in <i>Cucurbita maxima</i> Duch. cv. Queensland Blue X <i>Cucurbita ecuadorensis</i> Cutler and Whitaker </span>. <span class="source" tagx="source" title="source">Queensland Journal of Agricultural and Animal sciences</span>. </span> </li> <li tag="ref"> <a name="pone.0180741.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salse</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lucas</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">The draft genome of watermelon ( <i>Citrullus lanatus</i>) and resequencing of 20 diverse accessions </span>. <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">51</span>– <span class="lpage" tagx="lpage" title="lpage">58</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/ng.2470">10.1038/ng.2470</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23179023">23179023</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">357</span>– <span class="lpage" tagx="lpage" title="lpage">359</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertea</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pimentel</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelley</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions </span>. <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">R36</span> <span class="comment">doi: <a href="https://doi.org/10.1186/gb-2013-14-4-r36">10.1186/gb-2013-14-4-r36</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408">23618408</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Handsaker</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wysoker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fennell</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Homer</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">The Sequence Alignment/Map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">2078</span>– <span class="lpage" tagx="lpage" title="lpage">2079</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thorvaldsdottir</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mesirov</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration </span>. <span class="source" tagx="source" title="source">Brief Bioinform</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>– <span class="lpage" tagx="lpage" title="lpage">192</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bib/bbs017">10.1093/bib/bbs017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22517427">22517427</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smyth</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">139</span>– <span class="lpage" tagx="lpage" title="lpage">140</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btp616">10.1093/bioinformatics/btp616</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teixeira da Silva</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of <i>Cymbidium sinense</i> and its application to the identification of genes associated with floral development </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">279</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1471-2164-14-279">10.1186/1471-2164-14-279</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23617896">23617896</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Fine genetic mapping of the white immature fruit color gene <i>w</i> to a 33.0-kb region in cucumber ( <i>Cucumis sativus</i> L.) </span>. <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="volume" tagx="volume" title="volume">128</span>: <span class="fpage" tagx="fpage" title="fpage">2375</span>– <span class="lpage" tagx="lpage" title="lpage">2385</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00122-015-2592-z">10.1007/s00122-015-2592-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26239410">26239410</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Fine mapping of the dialytic gene that controls multicellular trichome formation and stamen development in tomato </span>. <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="volume" tagx="volume" title="volume">129</span>: <span class="fpage" tagx="fpage" title="fpage">1531</span>– <span class="lpage" tagx="lpage" title="lpage">1539</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00122-016-2722-2">10.1007/s00122-016-2722-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27151537">27151537</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Identification of suitable reference genes for gene expression normalization in qRT-PCR analysis in watermelon </span>. <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e90612</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0090612">10.1371/journal.pone.0090612</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24587403">24587403</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2 <sup>−ΔΔCT</sup> method </span>. <span class="source" tagx="source" title="source">Methods</span> <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">402</span>– <span class="lpage" tagx="lpage" title="lpage">408</span>. <span class="comment">doi: <a href="https://doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kondou</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazawa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ichikawa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshizumi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">RETARDED GROWTH OF EMBRYO1, a new basic helix-loop-helix protein, expresses in endosperm to control embryo growth </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">147</span>: <span class="fpage" tagx="fpage" title="fpage">1924</span>– <span class="lpage" tagx="lpage" title="lpage">1935</span>. <span class="comment">doi: <a href="https://doi.org/10.1104/pp.108.118364">10.1104/pp.108.118364</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18567831">18567831</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Partial deficiency of isoleucine impairs root development and alters transcript levels of the genes involved in branched-chain amino acid and glucosinolate metabolism in Arabidopsis </span>. <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">599</span>– <span class="lpage" tagx="lpage" title="lpage">612</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/jxb/ers352">10.1093/jxb/ers352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23230023">23230023</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dietrich</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">The maize <i>glossy</i>13 gene, cloned via BSR-Seq and Seq-walking encodes a putative ABC transporter required for the normal accumulation of epicuticular waxes </span>. <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e82333</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0082333">10.1371/journal.pone.0082333</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24324772">24324772</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref041" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">YP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">A high-resolution linkage map of the <i>Rfd1</i>, a restorer-of-fertility locus for cytoplasmic male sterility in radish ( <i>Raphanus sativus</i> L.) produced by a combination of bulked segregant analysis and RNA-Seq </span>. <span class="source" tagx="source" title="source">Theoretical and Applied Genetics</span> <span class="volume" tagx="volume" title="volume">127</span>: <span class="fpage" tagx="fpage" title="fpage">2243</span>– <span class="lpage" tagx="lpage" title="lpage">2252</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00122-014-2376-x">10.1007/s00122-014-2376-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25119873">25119873</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref042" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramirez‐Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Segovia</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bird</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fenwick</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holdgate</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berry</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">RNA‐Seq bulked segregant analysis enables the identification of high‐resolution genetic markers for breeding in hexaploid wheat </span>. <span class="source" tagx="source" title="source">Plant biotechnology journal</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">613</span>– <span class="lpage" tagx="lpage" title="lpage">624</span>. <span class="comment">doi: <a href="https://doi.org/10.1111/pbi.12281">10.1111/pbi.12281</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25382230">25382230</a> </span> </span> </li> <li tag="ref"> <a name="pone.0180741.ref043" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sisó</span> <span class="given-names" tagx="given-names" title="given-names">S CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gil-Pelegrín</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Relationship between hydraulic resistance and leaf morphology in broadleaf <i>Quercus</i> species: a new interpretation of leaf lobation </span>. <span class="source" tagx="source" title="source">Trees</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">341</span>– <span class="lpage" tagx="lpage" title="lpage">345</span>. </span> </li> </ul> </div> </div>  </body></html>