<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3646847">3646847</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23667507">23667507</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-13-03046</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0062681">10.1371/journal.pone.0062681</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span>: <span class="subject" title="subject">Forestry</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span>: <span class="subject" title="subject">Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Genetics</span>: <span class="subject" title="subject">Gene Expression</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Developmental Biology</span>: <span class="subject" title="subject">Fertilization</span> <span class="subject" title="subject">Molecular Development</span> <span class="subject" title="subject">Organism Development</span> <span class="subject" title="subject">Plant Growth and Development</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epigenetics</span>: <span class="subject" title="subject">RNA interference</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span>: <span class="subject" title="subject">RNA interference</span> </div>: <span class="subject" title="subject">Gene Networks</span> <span class="subject" title="subject">Molecular Genetics</span> <span class="subject" title="subject">Plant Genetics</span> </div>: <span class="subject" title="subject">Genomics</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Sexual Dimorphism Floral MicroRNA Profiling and Target Gene Expression in Andromonoecious Poplar ( <i>Populus tomentosa</i>) </div> <div class="alt-title" title="alt-title">
MicroRNAome Profiling of Andromonoecious Poplar</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">Yuepeng</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Kaifeng</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ci</span> <span class="given-names" tagx="given-names" title="given-names">Dong</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Zhiyi</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Deqiang</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing P. R. China </span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, P. R. China </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Rongling</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Pennsylvania State University, United States of America</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">DeqiangZhang@bjfu.edu.cn</span> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: DZ. Performed the experiments: YS KM DC ZZ DZ. Analyzed the data: YS DC DZ. Wrote the paper: YS DC DZ. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2013</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2013-5-5</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e62681</span> <span class="history" title="history"> <span class="received" title="received">received: 2013-1-19</span> <span class="accepted" title="accepted">accepted: 2013-3-22</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Song et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2013</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Although the molecular basis of poplar sex-specific flower development remains largely unknown, increasing evidence indicates an essential role for microRNAs (miRNAs). The specific miRNA types and precise miRNA expression patterns in dioecious plant flower development remain unclear. Here, we used andromonoecious poplar, an exceptional model system, to eliminate the confounding effects of genetic background of dioecious plants. This system, combined with high-throughput sequencing and computational analysis, allowed us to characterize sex-specific miRNAomes from female and male flowers. Comparative miRNAome analysis combined with quantitative real-time PCR revealed the expression patterns of 27 miRNAs in poplar flower and showed that the targets of these miRNAs are involved in flower organogenesis, Ca <sup>2+</sup> transport, phytohormone synthesis and metabolism, and DNA methylation. This paper describes a complex regulatory network consisting of these miRNAs expressed in sex-specific flower development in a dioecious plant. The conserved and novel miRNA locations were annotated in the <i>Populus trichocarpa</i> genome. Among these, miRNA Pto-F70 and 4 targets are located in the sex-determination regions of chromosome XIX. Furthermore, two novel miRNAs, Pto-F47 and Pto-F68, were shown for the first time to be regulatory factors in phytohormone interactions. To our knowledge, this report is the first systematic investigation of sex-specific flower-related miRNAs and their targets in poplar, and it deepens our understanding of the important regulatory functions of miRNAs in female and male flower development in this dioecious plant. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was supported by the Introduction of Foreign Advanced Agricultural Science and Technology into China (No. 2009-4-22), the Project of the National Natural Science Foundation of China (No. 31170622, 30872042), and the Specific Programs in Graduate Science and Technology Innovation of Beijing Forestry University (No.BLYJ201202). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The genus <i>Populus</i> encompasses approximately 100 species divided into 5 sections and many of these species are important components of terrestrial ecosystems or important cultivated trees for wood, pulp, paper, and possibly biofuels uses. As dioecious trees, the genus <i>Populus</i> provides opportunities to study perennial woody plant biology and plant sexual dimorphism <a href="#pone.0062681-Tuskan1">[1]</a>. Moreover, the high-quality, annotated genome sequence of <i>P. trichocarpa</i> provides a key genomics tool, enabling genome-wide regulatory analyses <a href="#pone.0062681-Tuskan1">[1]</a>. Sexual dimorphism is a widely studied phenomenon in dioecious plants and likely results from different modes of selection operating in males and females <a href="#pone.0062681-Zluvova1">[2]</a>. As dioecious trees, poplar floral development is sexual dimorphism. The poplar flower is composed of two whorls, including a reduced perianth cup surrounding either the stamens or carpels, which begin to develop in spring (early May in Shandong province) of the year before the spring in which they will open <a href="#pone.0062681-Sheppard1">[3]</a>, <a href="#pone.0062681-Rottmann1">[4]</a>. Until late May or early June, structural development of the male inflorescence is virtually indistinguishable from development of the female inflorescence. After that, the flower morphology, bract morphology, and cell division patterns differ markedly between the sexes <a href="#pone.0062681-Boes1">[5]</a>. Pioneering experiments found that a series of genes involved in flowering pathways are differently expressed between female and male flowers <a href="#pone.0062681-An1">[6]</a>, <a href="#pone.0062681-Song1">[7]</a>. Although the molecular basis of poplar sex-specific flower development remains largely mysterious, increasing evidence suggests that miRNAs may play essential roles in this process. </p> <p>In plants, there are two classes of endogenous small RNAs: small interfering RNAs (siRNAs) and microRNAs (miRNAs). Both classes, ranging from 20–24 nt, are short non-coding RNA molecules that negatively regulate gene expression at the transcriptional and/or post-transcriptional levels <a href="#pone.0062681-Chen1">[8]</a>. siRNAs are generated from double- stranded RNA. By contrast, miRNAs are transcribed from a long precursor molecule that folds back on itself to form a hairpin. Dicer-Like1 (DCL1) protein cleaves this precursor molecule, resulting in a miRNA:miRNA* complex, which separates into miRNA and miRNA* after transport to the cytoplasm <a href="#pone.0062681-Bartel1">[9]</a>. In the cytoplasm, the miRNA is bound by ARGONAUTE proteins to form part of the RNA-induced silencing complex (RISC), which interacts with target mRNAs to cleave the RNA of target genes at the paired region <a href="#pone.0062681-Llave1">[10]</a>. Since the mature miRNA and its complementary target sequence have almost perfect complementarity, identifying a miRNA usually leads to the prediction and/or identification of its target. miRNAs have been shown to target genes during plant organ development, stress tolerance, phytohormone signaling, growth phase change, and disease resistance <a href="#pone.0062681-Navarro1">[11]</a>, <a href="#pone.0062681-Wu1">[12]</a>. </p> <p>miRNAs have been identified as regulators of flower development and other pathways. For example, in <i>Arabidopsis</i>, the miR164 family targets a subset of NAC transcription factors including <i>CUP-SHAPED COTYLEDON1</i> ( <i>CUC1</i>) and <i>CUC2</i>, which mediate organ boundary formation. Among these members, miRNA164c affects petal number in early flowers <a href="#pone.0062681-Baker1">[13]</a>. miRNA172 regulates <i>APETALA2</i> ( <i>AP2</i>) and several of its homologs that share two tandem AP2 DNA-binding domains and play an important role in regulating flower development <a href="#pone.0062681-Aukerman1">[14]</a>– <a href="#pone.0062681-Schmid1">[16]</a>. In maize, <i>TASSELSEED 4</i> ( <i>TS4</i>) and the <i>AP2</i> homolog <i>indeterminate spikelet 1</i> are confirmed miRNA172 targets. In <i>ts4</i> mutants, female organs develop in the male inflorescence. Moreover, branching is increased in <i>ts4</i> mutants indicating a link between sex determination and meristem fate <a href="#pone.0062681-Chuck1">[17]</a>. In tree species, the unknown functions of the majority of genes are still the greatest gap. miRNA studies provided a best chance for directing to identify functions for genes <a href="#pone.0062681-Sun1">[18]</a>. Small RNA libraries prepared from poplar leaves and vegetative buds have been analyzed by high-throughput sequencing, finding 48 <i>Populus</i>-specific miRNA families <a href="#pone.0062681-Barakat1">[19]</a>. However, sex-specific miRNA differences between female and male flower organs remain unclear, partly because of the effects of different genetic backgrounds of dioecious plants. For this work, we first used andromonoecious poplar, an exceptional model system that has both male and hermaphrodite flowers on a single plant, to eliminate confounding effects of genetic background of dioecious plants ( <a href="#pone.0062681.s001">Figure S1</a>). In this study, our aim is to screen sex-specific potential functional miRNAs in poplar floral tissue, investigate their defined expression patterns and demonstrate a possible functional network of sex-specific miRNAs in poplar flower organs. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="smallrnasequenceanalyses" title="sec"> <div class="title" tagx="title" title="title">
Small RNA Sequence Analyses</div> <p>Two small RNA (sRNA) libraries were constructed from female (F) and male (M) flowers of andromonoecious <i>P</i>. <i>tomentosa</i>. A total of 58,517,150 (F) and 47,918,351 (M) raw reads were generated from these libraries, respectively, using the Solexa sequencing technology ( <a href="#pone-0062681-t001">Table 1</a>). After removing contaminant reads, clean reads were obtained and screened against rRNA, tRNA, snRNA, snoRNA, and mRNA in the Rfam 9.1 and NCBI GenBank databases, resulting in 50,782,479 (F) and 42,837,470 (M) reads remaining for further analyses. Conserved miRNAs were identified by alignment to data in miRBase 19.0 with ±2 nt mismatch; a total of 4,789,550 (F) and 4,306,015 (M) reads that could form characteristic hairpin structures were identified. 4,459,932 (F) and 3,810,615 (M) unannotated unique sRNA sequences, which were further analyzed to predict novel miRNAs. The sequenced sRNAs were then mapped to the <i>Populus</i> genome, miRBase19.0, the NCBI GenBank database, and the Rfam database, and classified into six categories: rRNA, known miRNA (miRNAs in miRBase 19.0), exon, intron, repeat associated RNA, and unknown sRNA. More than 60% of the sequences were unknown sRNAs ( <a href="#pone-0062681-g001">Figure 1A</a>). The majority of total sRNA reads were either 21 or 24 nt for both libraries ( <a href="#pone-0062681-g001">Figure 1B</a>). In female flower libraries, the 21-nt sRNAs were the most abundant, making up 41.4% of total sequence reads. The 24-nt sRNAs were the second most abundant class at 17.3% of total sequence reads. In male flower libraries, the 21 and 24 nt sRNAs were not significantly different, making up 29.9% and 29.7%, respectively. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Small RNAs from female and male flower libraries in andromonoecious poplar.</div> <p>(A) Distribution of unique sRNA annotation categories. (B) Size distribution of small RNA reads. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t001">10.1371/journal.pone.0062681.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary of small RNA sequences from <i>P</i>. <i>tomentosa</i> female and male flower. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td /> <td>Female library (F)</td> <td>Male library (M)</td> </tr> </thead> <tbody> <tr> <td>Raw reads</td> <td>58,517,150</td> <td>47,918,351</td> </tr> <tr> <td>Clean reads</td> <td>50,782,479</td> <td>42,837,470</td> </tr> <tr> <td>Annotated sRNAs</td> <td>16,625,919</td> <td>16,030,471</td> </tr> <tr> <td>Un-annotated sRNAs</td> <td>34,156,560</td> <td>26,806,999</td> </tr> <tr> <td>Small RNAs</td> <td>4,789,550</td> <td>4,306,015</td> </tr> <tr> <td>Annotated sRNAs</td> <td>329,618</td> <td>495,400</td> </tr> <tr> <td>Un-annotated sRNAs</td> <td>4,459,932</td> <td>3,810,615</td> </tr> </tbody> </table> </div> </div> </div> <div class="conservedmirnasinandromonoeciousp.tomentosa" title="sec"> <div class="title" tagx="title" title="title">
Conserved miRNAs in Andromonoecious <i>P. tomentosa</i> </div> <p>To date, 46 conserved and non-conserved miRNA families have been discovered in the <i>Populus</i> genome <a href="#pone.0062681-GriffithsJones1">[20]</a>. In the two libraries sequenced in our study, 134 unique miRNA sequences were identified belonging to 38 conserved miRNA families ( <a href="#pone-0062681-t002">Table 2</a>). The expression levels of a few miRNA families, such as miR166 and miR472, were extraordinarily high in both libraries ( <a href="#pone-0062681-t002">Table 2</a>). miR166 was the most abundant, with 2,157,665 (F) and 142, 916 (M) reads accounting for 62.5% and 47.1% of all conserved miRNA reads, respectively. Several miRNA families, such as miR156, miR159, miR169, miR319, miR396, and miR1447, had moderate expression levels ( <a href="#pone-0062681-t002">Table 2</a>). By contrast, some miRNA families showed very low levels of expression, with fewer than 200 reads. Different members in the same miRNA family displayed drastically different expression levels. For example, miR166 members varied in abundance from 695 to 2,157,665 reads. Also, 38 unique miRNA sequences belonging to 38 conserved miRNA families were produced from 134 loci ( <a href="#pone.0062681.s003">Table S1</a>). Of these miRNA sequences, 11 were encoded by a single locus in the <i>Populus</i> genome, whereas the other 27 sequences had multiple loci. Most of these had 2–6 loci in the genome, and only a few had more than eight loci. For example, Pto-miR166a–l and Pto-miR169a–m had 12 and 13 loci, respectively. Thus, the size of miRNA families varies in andromonoecious <i>P. tomentosa</i>. Among these conserved miRNAs, miR6459 was only expressed in the female library. By contrast, two miRNAs, miRNA397 and miRNA6462, were detected as having male-specific expression ( <a href="#pone-0062681-g002">Figure 2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Summary of common and specific sequences between female and male libraries. (A) Conserved sRNAs and (B) Novel sRNAs. </div>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t002">10.1371/journal.pone.0062681.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Summary of conserved miRNAs found in andromonoecious <i>P. tomentosa</i>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>miRNAfamily</td> <td>Members</td> <td>Reads</td> <td>Ratios (female/male)</td> </tr> <tr> <td /> <td /> <td>Female</td> <td>Male</td> <td /> </tr> </thead> <tbody> <tr> <td>miR156</td> <td>6</td> <td>3629</td> <td>8330</td> <td>0.44</td> </tr> <tr> <td>miR159</td> <td>3</td> <td>808474</td> <td>20798</td> <td>38.87</td> </tr> <tr> <td>miR160</td> <td>4</td> <td>2695</td> <td>523</td> <td>5.15</td> </tr> <tr> <td>miR162</td> <td>2</td> <td>5094</td> <td>342</td> <td>14.89</td> </tr> <tr> <td>miR166</td> <td>12</td> <td>2157665</td> <td>142916</td> <td>15.10</td> </tr> <tr> <td>miR167</td> <td>8</td> <td>1995</td> <td>15568</td> <td>0.13</td> </tr> <tr> <td>miR168</td> <td>2</td> <td>38333</td> <td>5129</td> <td>7.47</td> </tr> <tr> <td>miR169</td> <td>13</td> <td>41555</td> <td>3514</td> <td>11.83</td> </tr> <tr> <td>miR171</td> <td>5</td> <td>820</td> <td>298</td> <td>2.75</td> </tr> <tr> <td>miR172</td> <td>2</td> <td>58</td> <td>105</td> <td>0.55</td> </tr> <tr> <td>miR319</td> <td>6</td> <td>115137</td> <td>2239</td> <td>51.42</td> </tr> <tr> <td>miR393</td> <td>5</td> <td>2498</td> <td>1911</td> <td>1.31</td> </tr> <tr> <td>miR394</td> <td>2</td> <td>4814</td> <td>1143</td> <td>4.21</td> </tr> <tr> <td>miR396</td> <td>9</td> <td>30598</td> <td>30786</td> <td>0.99</td> </tr> <tr> <td>miR397</td> <td>1</td> <td>0</td> <td>8</td> <td>0.00</td> </tr> <tr> <td>miR398</td> <td>3</td> <td>1783</td> <td>752</td> <td>2.37</td> </tr> <tr> <td>miR399</td> <td>10</td> <td>11180</td> <td>116</td> <td>96.38</td> </tr> <tr> <td>miR403</td> <td>3</td> <td>19815</td> <td>5087</td> <td>3.90</td> </tr> <tr> <td>miR408</td> <td>1</td> <td>157</td> <td>161</td> <td>0.98</td> </tr> <tr> <td>miR472</td> <td>2</td> <td>126848</td> <td>48300</td> <td>2.63</td> </tr> <tr> <td>miR475</td> <td>7</td> <td>1524</td> <td>394</td> <td>3.87</td> </tr> <tr> <td>miR476</td> <td>3</td> <td>1101</td> <td>637</td> <td>1.73</td> </tr> <tr> <td>miR482</td> <td>2</td> <td>1973</td> <td>312</td> <td>6.32</td> </tr> <tr> <td>miR530</td> <td>1</td> <td>337</td> <td>11</td> <td>30.64</td> </tr> <tr> <td>miR1444</td> <td>1</td> <td>29</td> <td>36</td> <td>0.81</td> </tr> <tr> <td>miR1446</td> <td>4</td> <td>22</td> <td>51</td> <td>0.43</td> </tr> <tr> <td>miR1447</td> <td>1</td> <td>22381</td> <td>2700</td> <td>8.29</td> </tr> <tr> <td>miR1448</td> <td>1</td> <td>16637</td> <td>4935</td> <td>3.37</td> </tr> <tr> <td>miR1450</td> <td>1</td> <td>16340</td> <td>826</td> <td>19.78</td> </tr> <tr> <td>miR6421</td> <td>1</td> <td>641</td> <td>711</td> <td>0.90</td> </tr> <tr> <td>miR6425</td> <td>4</td> <td>219</td> <td>245</td> <td>0.89</td> </tr> <tr> <td>miR6427</td> <td>1</td> <td>345</td> <td>112</td> <td>3.08</td> </tr> <tr> <td>miR6433</td> <td>2</td> <td>1026</td> <td>374</td> <td>2.74</td> </tr> <tr> <td>miR6445</td> <td>2</td> <td>9334</td> <td>3382</td> <td>2.76</td> </tr> <tr> <td>miR6447</td> <td>1</td> <td>2</td> <td>10</td> <td>0.20</td> </tr> <tr> <td>miR6459</td> <td>1</td> <td>603</td> <td>0</td> <td>0.00</td> </tr> <tr> <td>miR6462</td> <td>1</td> <td>0</td> <td>5</td> <td>0.00</td> </tr> <tr> <td>miR6478</td> <td>1</td> <td>1846</td> <td>1028</td> <td>1.80</td> </tr> <tr> <td>Total</td> <td>134</td> <td>3447508</td> <td>303795</td> <td>11.35</td> </tr> </tbody> </table> </div> </div> </div> <div class="identificationofnovelmirnasinandromonoeciousp.tomentosa" title="sec"> <div class="title" tagx="title" title="title">
Identification of Novel miRNAs in Andromonoecious <i>P. tomentosa</i> </div> <p>The characteristic hairpin structure of miRNA precursors was used to predict novel miRNAs. In total, 78 unique sRNA sequences were identified including 8 sRNAs with complementary miRNA*s ( <a href="#pone-0062681-t003">Table 3</a>, <a href="#pone-0062681-t004">4</a> and <a href="#pone-0062681-t005">5</a>). The majority of these novel miRNA candidates had lengths of 21 and 22 nt, and started with a 5′ U ( <a href="#pone-0062681-g003">Figure 3</a>). The 78 sRNA sequences were transcribed from 85 loci; most were only produced from one locus, but Pto-F2, Pto-F13, Pto-F31 and Pto-F31 were each generated from 3 loci. The length of the predicted novel miRNA precursors ranged from 76 to 303 nt, with an average of 140 nt. The average minimum free energy (MFE) value was −55.58 kcal/mol, with a range of −21.2 to −177.4 kcal/mol. The structures of the 78 novel miRNA precursors are shown in <a href="#pone.0062681.s002">Figure S2</a>. Most of them showed differential expression in the two libraries. For instance, the mature miRNA counts ranged from 63 to 8,480,860. Among those 78 miRNAs, only 6 sRNAs were found both in female and male libraries. 61 female-specific sRNAs and 11 male-specific sRNAs were detected, suggesting that more kinds of sRNA are involved in female floral development than in male development. To investigate whether these novel miRNA sequences were conserved across plant species, we used them as query sequences to perform BLASTN searches against the <i>Arabidopsis thaliana</i>, <i>Brassica rapa</i>, <i>Citrus sinensis</i>, <i>Medicago truncatula</i>, <i>Oryza sativa</i>, <i>Ricinus communis</i>, <i>Prunus persica</i>, <i>Vitis vinifera</i> and <i>Zea mays</i> genome databases in Phytozome v7.0. A large proportion of the novel sRNAs identified in andromonoecious <i>P. tomentosa</i> did not have perfect matches in different genome databases. Only 6 male-specific sRNA and 5 female-specific sRNAs were detected in other species ( <a href="#pone.0062681.s004">Table S2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
First nucleotide bias of novel candidate miRNAs in andromonoecious poplar female and male flower libraries. </div>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t003">10.1371/journal.pone.0062681.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Summary of common novel miRNAs in female and male flower libraries.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Name</td> <td>Sequence (5′-3′)</td> <td>LP</td> <td>MFE</td> <td>miRNA location</td> <td>miRNA reads</td> <td>Ratio(Female/Male)</td> <td>miR*reads</td> </tr> <tr> <td /> <td /> <td /> <td /> <td /> <td>Female</td> <td>Male</td> <td /> <td /> </tr> </thead> <tbody> <tr> <td>Pto-F1</td> <td> <span class="named-content" title="named-content">ATAGGTTTCCGATCATTCCTCC</span> </td> <td>169</td> <td>−81.7</td> <td>chr13∶103779–103800</td> <td>8990</td> <td>391</td> <td>22.99</td> <td>NO</td> </tr> <tr> <td>Pto-F2a</td> <td> <span class="named-content" title="named-content">TGAAATGCCATGAACAACGATC</span> </td> <td>125</td> <td>−53.4</td> <td>chr13∶11993451–11993472</td> <td>433</td> <td>8111</td> <td>0.05</td> <td>NO</td> </tr> <tr> <td>Pto-F2b</td> <td> <span class="named-content" title="named-content">TGAAATGCCATGAACAACGATC</span> </td> <td>115</td> <td>−47</td> <td>chr16∶4142967–4142987</td> <td>377</td> <td>942</td> <td>0.40</td> <td>NO</td> </tr> <tr> <td>Pto-F3</td> <td> <span class="named-content" title="named-content">ATGTAATCAAGGCTGCTAGTC</span> </td> <td>141</td> <td>−72.3</td> <td>chr17∶8682859–8682879</td> <td>71</td> <td>434</td> <td>0.16</td> <td>Yes</td> </tr> <tr> <td>Pto-F4</td> <td> <span class="named-content" title="named-content">TCTAAAGATCCGGCGGTCGGC</span> </td> <td>124</td> <td>−44</td> <td>chr18∶2804698–2804718</td> <td>63</td> <td>90</td> <td>0.70</td> <td>NO</td> </tr> <tr> <td>Pto-F5</td> <td> <span class="named-content" title="named-content">TATGTTTGACAGAAACCCCCT</span> </td> <td>171</td> <td>−89.2</td> <td>chr18∶8350113–8350132</td> <td>729</td> <td>1991</td> <td>0.37</td> <td>NO</td> </tr> <tr> <td>Pto-F6</td> <td> <span class="named-content" title="named-content">TGTGGTAGATATGTGAGGATT</span> </td> <td>148</td> <td>−65.2</td> <td>chr19∶10983127–10983147</td> <td>472</td> <td>233</td> <td>2.03</td> <td>NO</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t004">10.1371/journal.pone.0062681.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
Summary of male-specific novel miRNAs.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Name</td> <td>Sequence (5-3)</td> <td>LP</td> <td>MFE</td> <td>miRNA location</td> <td>miRNA reads</td> <td>miR*reads</td> </tr> </thead> <tbody> <tr> <td>Pto-F7</td> <td> <span class="named-content" title="named-content">CTCAAACTTTTCTCTTACTTC</span> </td> <td>283</td> <td>−168.1</td> <td>chr2∶1282971–1282991</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F8</td> <td> <span class="named-content" title="named-content">TACCGTTAGAAAGATCTCAAC</span> </td> <td>179</td> <td>−62</td> <td>chr2∶8266846–8266866</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F9</td> <td> <span class="named-content" title="named-content">ATAGGTTTCCGATCATTCCTCC</span> </td> <td>199</td> <td>−85.04</td> <td>chr5∶119707–119728</td> <td>819</td> <td>YES</td> </tr> <tr> <td>Pto-F10</td> <td> <span class="named-content" title="named-content">TTTTGCACGTAGAATTCAGGA</span> </td> <td>139</td> <td>−38</td> <td>chr5∶18398105–18398125</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F11</td> <td> <span class="named-content" title="named-content">ACGAGCCATCATAACTGTAGG</span> </td> <td>125</td> <td>−81.92</td> <td>chr6∶6767480–6767500</td> <td>714</td> <td>YES</td> </tr> <tr> <td>Pto-F12</td> <td> <span class="named-content" title="named-content">TAGTTCGAGGGACCAAAATCA</span> </td> <td>219</td> <td>−111.4</td> <td>chr6∶8063061–8063081</td> <td>357</td> <td>YES</td> </tr> <tr> <td>Pto-F13</td> <td> <span class="named-content" title="named-content">ATCTGCTTTCTGCGACTCCTC</span> </td> <td>183</td> <td>−68.8</td> <td>chr8∶8028138–8028158</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F14</td> <td> <span class="named-content" title="named-content">GCATTTGGACGTCGGGGAACT</span> </td> <td>90</td> <td>−23.7</td> <td>chr9∶7157296–7157316</td> <td>357</td> <td>NO</td> </tr> <tr> <td>Pto-F15a</td> <td> <span class="named-content" title="named-content">AGGATTGGAGGGAATTAAACA</span> </td> <td>135</td> <td>−40.55</td> <td>chr10∶20020250–20020270</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F15b</td> <td> <span class="named-content" title="named-content">AGGATTGGAGGGAATTAAACA</span> </td> <td>129</td> <td>−41.7</td> <td>chr12∶11845073–11845093</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F16</td> <td> <span class="named-content" title="named-content">TAATTCCATGACTGTGTACAG</span> </td> <td>156</td> <td>−37.82</td> <td>chr15∶10265155–10265175</td> <td>189</td> <td>NO</td> </tr> <tr> <td>Pto-F17a</td> <td> <span class="named-content" title="named-content">TTTAATTTCCTCCAATATCTTA</span> </td> <td>133</td> <td>−45.54</td> <td>chr18∶14578403–14578424</td> <td>210</td> <td>NO</td> </tr> <tr> <td>Pto-F17b</td> <td> <span class="named-content" title="named-content">TTTAATTTCCTCCAATATCTTA</span> </td> <td>136</td> <td>−45.54</td> <td>scaffold_269∶8974–8995</td> <td>210</td> <td>NO</td> </tr> <tr> <td>Pto-F17c</td> <td> <span class="named-content" title="named-content">TTTAGTTTCCTCCAATATCTTA</span> </td> <td>136</td> <td>−41.2</td> <td>scaffold_269∶9454–9475</td> <td>189</td> <td>NO</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t005">10.1371/journal.pone.0062681.t005</a> <span class="label" tagx="label" title="label">Table 5</span>  <div class="title" tagx="title" title="title">
Summary of female-specific novel miRNAs.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Name</td> <td>Sequence (5-3)</td> <td>LP</td> <td>MFE</td> <td>miRNA location</td> <td>miRNA reads</td> <td>miR*reads</td> </tr> </thead> <tbody> <tr> <td>Pto-F18</td> <td> <span class="named-content" title="named-content">TTGGTTAAACTCCCATCAGGA</span> </td> <td>133</td> <td>−65.6</td> <td>chr1∶7117690–7117710</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F19</td> <td> <span class="named-content" title="named-content">CTCAGATTAGCCAGGTGCCT</span> </td> <td>117</td> <td>−44.3</td> <td>chr1∶10983482–10983501</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F20</td> <td> <span class="named-content" title="named-content">TTATTAAACCCGGACCGGCCT</span> </td> <td>122</td> <td>−54.62</td> <td>chr1∶18312617–18312637</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F21</td> <td> <span class="named-content" title="named-content">TACCCGGTTACTGTGCATGTGC</span> </td> <td>76</td> <td>−33.3</td> <td>chr1∶38996098–38996119</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F22</td> <td> <span class="named-content" title="named-content">TATCCTTGGATCTGACGACCA</span> </td> <td>220</td> <td>−123.4</td> <td>chr1∶39819960–39819980</td> <td>294</td> <td>NO</td> </tr> <tr> <td>Pto-F23</td> <td> <span class="named-content" title="named-content">TTTTCGATCCAAGCTTCGGGT</span> </td> <td>80</td> <td>−27.1</td> <td>chr1∶42987729–42987749</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F24</td> <td> <span class="named-content" title="named-content">TTACTTCCTTTTGTCCCTCTC</span> </td> <td>303</td> <td>−177.4</td> <td>chr2∶1282957–1282977</td> <td>357</td> <td>NO</td> </tr> <tr> <td>Pto-F25</td> <td> <span class="named-content" title="named-content">TATAGATTGCAGAGGGAACC</span> </td> <td>203</td> <td>−71.04</td> <td>chr2∶12233873–12233892</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F26</td> <td> <span class="named-content" title="named-content">TTATTAAACCCGGACCGGCCT</span> </td> <td>78</td> <td>−42.8</td> <td>chr2∶19952785–19952805</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F27</td> <td> <span class="named-content" title="named-content">TCCGAGCTCTAATTATGTGGG</span> </td> <td>182</td> <td>−92</td> <td>chr2∶21648214–21648234</td> <td>798</td> <td>NO</td> </tr> <tr> <td>Pto-F28</td> <td> <span class="named-content" title="named-content">TTGGGCTGGCAGTTGTGATGAC</span> </td> <td>161</td> <td>−45.87</td> <td>chr3∶19076220–19076241</td> <td>231</td> <td>NO</td> </tr> <tr> <td>Pto-F29</td> <td> <span class="named-content" title="named-content">TTCCGGTTTTTGGGACTCCGAT</span> </td> <td>103</td> <td>−35.1</td> <td>chr4∶20319258–20319279</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F30</td> <td> <span class="named-content" title="named-content">TTCTTTCCTATCAAACTTCAAC</span> </td> <td>191</td> <td>−92.4</td> <td>chr4∶22210258–22210279</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F31</td> <td> <span class="named-content" title="named-content">TGAAGATAAGAGCTTGTTTGG</span> </td> <td>169</td> <td>−75.86</td> <td>chr5∶6548077–6548097</td> <td>651</td> <td>NO</td> </tr> <tr> <td>Pto-F32</td> <td> <span class="named-content" title="named-content">TTTGCACGTAGAATTCAGGATT</span> </td> <td>139</td> <td>−38</td> <td>chr5∶18398103–18398124</td> <td>588</td> <td>NO</td> </tr> <tr> <td>Pto-F33</td> <td> <span class="named-content" title="named-content">ATGGATTGATGAAGATGAGGA</span> </td> <td>92</td> <td>−26.3</td> <td>chr5∶21111319–21111339</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F34</td> <td> <span class="named-content" title="named-content">TTATTTCCTCAAACTTTCCTC</span> </td> <td>256</td> <td>−159.8</td> <td>chr5∶24262419–24262439</td> <td>273</td> <td>NO</td> </tr> <tr> <td>Pto-F35</td> <td> <span class="named-content" title="named-content">TAGAACCTGACCAGTTGAGCT</span> </td> <td>116</td> <td>−30</td> <td>chr6∶2313141–2313161</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F36</td> <td> <span class="named-content" title="named-content">TGGCCCATGATCTTCATTGTG</span> </td> <td>115</td> <td>−78.02</td> <td>chr6∶6767559–6767579</td> <td>357</td> <td>YES</td> </tr> <tr> <td>Pto-F37</td> <td> <span class="named-content" title="named-content">TTTTTGTTGTTTAAGAACCCT</span> </td> <td>301</td> <td>−128.6</td> <td>chr6∶21307737–21307757</td> <td>315</td> <td>NO</td> </tr> <tr> <td>Pto-F38</td> <td> <span class="named-content" title="named-content">TTGGGCTGGCAGTTGTGATGAC</span> </td> <td>161</td> <td>−46.2</td> <td>chr7∶301518–301539</td> <td>231</td> <td>NO</td> </tr> <tr> <td>Pto-F39</td> <td> <span class="named-content" title="named-content">TGTGGGACTCGAATTAGTGAT</span> </td> <td>96</td> <td>−26.43</td> <td>chr7∶13307862–13307882</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F40a</td> <td> <span class="named-content" title="named-content">GAGGAGGGAGAGAGATCTGT</span> </td> <td>136</td> <td>−38.3</td> <td>chr8∶5498368–5498387</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F40b</td> <td> <span class="named-content" title="named-content">GAGGAGGGAGAGAGATCTGT</span> </td> <td>114</td> <td>−34.3</td> <td>chr8∶5522852–5522871</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F41</td> <td> <span class="named-content" title="named-content">ATCCAACGGTTAGATCTCCCT</span> </td> <td>296</td> <td>−162.7</td> <td>chr8∶8443078–8443098</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F42</td> <td> <span class="named-content" title="named-content">CTTGAAACAATGTTTGGTGCAG</span> </td> <td>121</td> <td>−31.9</td> <td>chr8∶10699331–10699352</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F43</td> <td> <span class="named-content" title="named-content">TAGAGTATAAGAGGATCGACT</span> </td> <td>124</td> <td>−32.4</td> <td>chr8∶15869891–15869911</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F44</td> <td> <span class="named-content" title="named-content">ACTTTGACCTGAACTTGCCC</span> </td> <td>79</td> <td>−40</td> <td>chr9∶8255922–8255941</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F45</td> <td> <span class="named-content" title="named-content">TGTCCTGACTCGAACTCGAGA</span> </td> <td>82</td> <td>−21.2</td> <td>chr10∶7102937–7102957</td> <td>189</td> <td>NO</td> </tr> <tr> <td>Pto-F46</td> <td> <span class="named-content" title="named-content">TGAAGATAAGAGCTTGTTTGG</span> </td> <td>169</td> <td>−77.06</td> <td>chr10∶7227906–7227926</td> <td>651</td> <td>NO</td> </tr> <tr> <td>Pto-F47</td> <td> <span class="named-content" title="named-content">GGGACTGCTGTAGATGCTTGG</span> </td> <td>90</td> <td>−34.7</td> <td>chr10∶13462743–13462763</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F48</td> <td> <span class="named-content" title="named-content">TTGATGACTGATCTTGAGCAT</span> </td> <td>99</td> <td>−62.8</td> <td>chr10∶16376128–16376148</td> <td>189</td> <td>YES</td> </tr> <tr> <td>Pto-F49</td> <td> <span class="named-content" title="named-content">TTGTAAGAGCCTGAGACAGCC</span> </td> <td>143</td> <td>−51.8</td> <td>chr10∶18334850–18334870</td> <td>714</td> <td>NO</td> </tr> <tr> <td>Pto-F50</td> <td> <span class="named-content" title="named-content">TTTAATTTCCTCCAATATCTCA</span> </td> <td>169</td> <td>−45.91</td> <td>chr10∶20020296–20020317</td> <td>399</td> <td>NO</td> </tr> <tr> <td>Pto-F51</td> <td> <span class="named-content" title="named-content">TTGGAATCCTCTCTGATAATGC</span> </td> <td>105</td> <td>−30.5</td> <td>chr10∶20322598–20322619</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F52</td> <td> <span class="named-content" title="named-content">TTTAGGACCCGTGGGATTAGTC</span> </td> <td>102</td> <td>−27</td> <td>chr11∶8039096–8039117</td> <td>189</td> <td>NO</td> </tr> <tr> <td>Pto-F53</td> <td> <span class="named-content" title="named-content">TCAATGGGTAAAAATTCTGACC</span> </td> <td>96</td> <td>−24.4</td> <td>chr12∶3834076–3834097</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F54</td> <td> <span class="named-content" title="named-content">TTGGAGCTCGAGACTTGGCAC</span> </td> <td>90</td> <td>−25.74</td> <td>chr13∶1347925–1347945</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F55</td> <td> <span class="named-content" title="named-content">ATCACTGTGTCTATTAGGATGG</span> </td> <td>82</td> <td>−28.1</td> <td>chr13∶6333415–6333436</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F56</td> <td> <span class="named-content" title="named-content">TGCTAGGACCAAGTTTTCTGG</span> </td> <td>103</td> <td>−53.34</td> <td>chr13∶9970875–9970895</td> <td>609</td> <td>YES</td> </tr> <tr> <td>Pto-F57</td> <td> <span class="named-content" title="named-content">TTCAGGACCTGTAGAATTAGTC</span> </td> <td>92</td> <td>−24.2</td> <td>chr13∶12124499–12124520</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F58a</td> <td> <span class="named-content" title="named-content">TGAATGGCTGTTGAACTTGGC</span> </td> <td>156</td> <td>−49.5</td> <td>chr14∶293794–293814</td> <td>168</td> <td>NO</td> </tr> <tr> <td>Pto-F58b</td> <td> <span class="named-content" title="named-content">TGAATGGCTGTTGAACTTGGC</span> </td> <td>158</td> <td>−45.51</td> <td>chr14∶726835–726855</td> <td>168</td> <td>NO</td> </tr> <tr> <td>Pto-F58c</td> <td> <span class="named-content" title="named-content">TGAATGGCTGTTGAACTTGGC</span> </td> <td>158</td> <td>−52.6</td> <td>chr14∶1080042–1080062</td> <td>168</td> <td>NO</td> </tr> <tr> <td>Pto-F59</td> <td> <span class="named-content" title="named-content">TGATGGGTCTCATTTAGTAGA</span> </td> <td>215</td> <td>−86.36</td> <td>chr14∶5877228–5877248</td> <td>441</td> <td>NO</td> </tr> <tr> <td>Pto-F60</td> <td> <span class="named-content" title="named-content">AACCGAACCGAACTGAACCGA</span> </td> <td>98</td> <td>−33.4</td> <td>chr14∶16267732–16267752</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F61</td> <td> <span class="named-content" title="named-content">TTATAGTTTTGAAATCCGGCC</span> </td> <td>95</td> <td>−47.2</td> <td>chr15∶5715046–5715066</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F62</td> <td> <span class="named-content" title="named-content">AACCCAAGAACTCAACTCTGT</span> </td> <td>171</td> <td>−68.37</td> <td>chr15∶6997081–6997101</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F63</td> <td> <span class="named-content" title="named-content">GTACGATCCTATAGAACAGAT</span> </td> <td>116</td> <td>−34.5</td> <td>chr15∶15015652–15015672</td> <td>189</td> <td>YES</td> </tr> <tr> <td>Pto-F64</td> <td> <span class="named-content" title="named-content">CAACACCAGACCCAAGAGCTT</span> </td> <td>117</td> <td>−44.51</td> <td>chr16∶5544896–5544916</td> <td>210</td> <td>NO</td> </tr> <tr> <td>Pto-F65</td> <td> <span class="named-content" title="named-content">TTAGGGTTTAGGGTTTAGAA</span> </td> <td>157</td> <td>−41.6</td> <td>chr17∶210550–210569</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F66</td> <td> <span class="named-content" title="named-content">TCGAGATTGTACTGTTCATA</span> </td> <td>77</td> <td>−23.6</td> <td>chr17∶4120360–4120379</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F67</td> <td> <span class="named-content" title="named-content">TGGCAAATGATCTGCATCTGC</span> </td> <td>90</td> <td>−24.52</td> <td>chr17∶13666346–13666366</td> <td>105</td> <td>NO</td> </tr> <tr> <td>Pto-F68</td> <td> <span class="named-content" title="named-content">GTGGGTGGGTCTGGGTGGCA</span> </td> <td>118</td> <td>−47.7</td> <td>chr18∶10507789–10507808</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F69</td> <td> <span class="named-content" title="named-content">TTTGCTCTTCGTTTTCTCATG</span> </td> <td>170</td> <td>−77.8</td> <td>chr18∶14336738–14336758</td> <td>294</td> <td>NO</td> </tr> <tr> <td>Pto-F70</td> <td> <span class="named-content" title="named-content">TTTTATTCCTGACTCTGGCATC</span> </td> <td>166</td> <td>−43.31</td> <td>chr19∶907650–907671</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F71</td> <td> <span class="named-content" title="named-content">TTGGGAATTATTACAATGGCA</span> </td> <td>106</td> <td>−31.8</td> <td>chr19∶1460855–1460875</td> <td>147</td> <td>NO</td> </tr> <tr> <td>Pto-F72</td> <td> <span class="named-content" title="named-content">TTATTAAACCCGGACCGGCCT</span> </td> <td>87</td> <td>−39.6</td> <td>chr19∶2013039–2013059</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F73</td> <td> <span class="named-content" title="named-content">TGTTTTCCGGAAAGTAGTTTC</span> </td> <td>99</td> <td>−27.1</td> <td>chr19∶8890465–8890485</td> <td>378</td> <td>NO</td> </tr> <tr> <td>Pto-F74</td> <td> <span class="named-content" title="named-content">TAGGGCAGAGAATTGAATGACT</span> </td> <td>119</td> <td>−45.4</td> <td>chr19∶10646670–10646691</td> <td>252</td> <td>NO</td> </tr> <tr> <td>Pto-F75</td> <td> <span class="named-content" title="named-content">TCGAGATTTCTTTGTTGGAGCT</span> </td> <td>125</td> <td>−42.72</td> <td>scaffold_1233∶2309–2330</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F76</td> <td> <span class="named-content" title="named-content">TCGGCGTTGATGTAGAATGGC</span> </td> <td>153</td> <td>−53.1</td> <td>scaffold_401∶10122–10142</td> <td>126</td> <td>NO</td> </tr> <tr> <td>Pto-F77</td> <td> <span class="named-content" title="named-content">TTTTGAGACAATGCTGAAAAT</span> </td> <td>99</td> <td>−28.4</td> <td>scaffold_65∶42511–42531</td> <td>1953</td> <td>NO</td> </tr> <tr> <td>Pto-F78</td> <td> <span class="named-content" title="named-content">ACATGAAACGGCGTCGTTTTG</span> </td> <td>91</td> <td>−26</td> <td>scaffold_76∶39681–39701</td> <td>126</td> <td>NO</td> </tr> </tbody> </table> </div> </div> </div> <div class="novelmirnaandtargetswerelocatedingenderdeterminationregions" title="sec"> <div class="title" tagx="title" title="title">
Novel miRNA and Targets were Located in Gender Determination Regions</div> <p>It was reported that Chromosome XIX is an incipient sex chromosome in <i>Populus</i> <a href="#pone.0062681-Yin1">[21]</a>, <a href="#pone.0062681-Tuskan2">[22]</a>. In the present study, 5 novel miRNAs located in chromosome XIX were identified, including Pto-F6, Pto-F70, Pto-F71, Pto-F72 and Pto-F73 ( <a href="#pone-0062681-t006">Table 6</a>; <a href="#pone-0062681-g004">Figure 4A</a>). Pto-F6 was expressed both in female and male flower libraries, but expressed at higher levels in female flowers than in male ( <a href="#pone-0062681-t003">Table 3</a>). The Pto-70, Pto-71, Pto-72 and Pto-73 female-specific miRNAs were weakly expressed in female flowers ( <a href="#pone-0062681-t005">Table 5</a>). A total of 33 targets located on chromosome XIX were identified for these novel miRNAs. Among these, Pto-F6 and Pto-73 were predicted to match 14 targets respectively that are located on chromosome XIX ( <a href="#pone-0062681-t006">Table 6</a>). By contrast, no targets were identified for Pto-F72. Putative functions of these targets genes were annotated by BLAST searches against the TAIR, NCBI and JGI protein datasets. The results revealed that target genes are involved in disease resistance, response to auxin stimulus, decreased DNA methylation and membrane transport. Yin et al (2008) indicated that a 2.45 Mb segment in the peritelomeric region of chromosome XIX and is associated with gender determination <a href="#pone.0062681-Yin1">[21]</a>. In our study, Pto-70 and 4 targets genes were found in sex-determination regions ( <a href="#pone-0062681-g004">Figure 4B</a>). Among the 4 target genes, 3 targets were predicted to match Pto-73 miRNAs and one was predicted to match Pto-F6 miRNA. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Distribution of novel miRNAs on chromosome XIX.</div> <p>A. 6 novel miRNAs and their targets located on chromosome XIX. B. Schematic diagram of the order of 6 novel miRNAs and their targets on chromosome XIX. Each miRNA and corresponding targets are marked the same color. Putative sex determination region in a 2.45-Mb region at the peritelomeric end of chromosome XIX, based on Yin et al (2008). </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t006">10.1371/journal.pone.0062681.t006</a> <span class="label" tagx="label" title="label">Table 6</span>  <div class="title" tagx="title" title="title">
Targets of novel <i>P. tomentosa</i> miRNAs verified in XIX chromosome of poplar. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>miRNA</td> <td>Target Gene Model <sup>a</sup> </td> <td>Putative function <sup>b</sup> </td> <td>Location</td> <td>TAIR Gene Model <sup>c</sup> </td> </tr> </thead> <tbody> <tr> <td>Pto-F6</td> <td>POPTR_0019s00970</td> <td>negative regulation of catalytic activity, proteolysis,</td> <td>Chr19∶737356–739497</td> <td>AT1G04110</td> </tr> <tr> <td /> <td>POPTR_0019s01610</td> <td>response to auxin stimulus</td> <td>Chr19∶1429616–1430026</td> <td>AT5G53590</td> </tr> <tr> <td /> <td>POPTR_0019s02760</td> <td>protein phosphorylation</td> <td>Chr19∶2561092–2568350</td> <td>AT1G53430</td> </tr> <tr> <td /> <td>POPTR_0019s02880</td> <td>Pentatricopeptide repeat</td> <td>Chr19∶2750320–2751951</td> <td>AT2G34400</td> </tr> <tr> <td /> <td>POPTR_0019s03110</td> <td>Encodes a lipid acyl hydrolase with wide substrate specificity</td> <td>Chr19∶3122643–3124999</td> <td>AT2G26560</td> </tr> <tr> <td /> <td>POPTR_0019s03700</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶3775544–3779105</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s03970</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶4034540–4038218</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s06080</td> <td>KAR-UP F-box 1 (KUF1); CONTAINS InterPro DOMAIN/s</td> <td>Chr19∶6589704–6590843</td> <td>AT1G31350</td> </tr> <tr> <td /> <td>POPTR_0019s10290</td> <td>LRR and NB-ARC domains-containing disease resistance protein</td> <td>Chr19∶11867256–1870843</td> <td>AT3G14460</td> </tr> <tr> <td /> <td>POPTR_0019s10300</td> <td>LRR and NB-ARC domains-containing disease resistance protein</td> <td>Chr19∶11884201–11888700</td> <td>AT3G14460</td> </tr> <tr> <td /> <td>POPTR_0019s10890</td> <td>similar to expression supported by MPSS</td> <td>Chr19∶12374345–12375027</td> <td>AT1G35430</td> </tr> <tr> <td /> <td>POPTR_0019s12450</td> <td>similar to chloroplast precursor (TRX-M1)</td> <td>Chr19∶13509839–13510150</td> <td>AT4G03520</td> </tr> <tr> <td /> <td>POPTR_0019s13380</td> <td /> <td>Chr19∶14290118–14294196</td> <td>AT5G08415</td> </tr> <tr> <td /> <td>POPTR_0019s15030</td> <td>similar to decreased DNA methylation 1</td> <td>Chr19∶15655586–15661763</td> <td>AT5G66750</td> </tr> <tr> <td>Pto-F70</td> <td>POPTR_0019s02280</td> <td>member of Myosin-like proteins</td> <td>Chr19∶2112309–2122774</td> <td>AT1G54560</td> </tr> <tr> <td /> <td>POPTR_0019s10800</td> <td>Encodes a cyclase</td> <td>Chr19∶12293003–122983961</td> <td>AT1G78955</td> </tr> <tr> <td /> <td>POPTR_0019s15290</td> <td>unknown protein</td> <td>Chr19∶15825895–15830273</td> <td>AT5G01970</td> </tr> <tr> <td>Pto-F71</td> <td>POPTR_0019s03120</td> <td>Encodes a lipid acyl hydrolase with wide substrate specificity</td> <td>Chr19∶3134481–3137841</td> <td>AT2G26560</td> </tr> <tr> <td /> <td>POPTR_0019s08530</td> <td>ATP phosphoribosyl transferase</td> <td>Chr19∶10043845–10045971</td> <td>AT1G09795</td> </tr> <tr> <td>Pto-F73</td> <td>POPTR_0019s00820</td> <td>ENTH/VHS/GAT family protein</td> <td>Chr19∶545449–549077</td> <td>AT5G63640</td> </tr> <tr> <td /> <td>POPTR_0019s01010</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶774829–779781</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s01080</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶820958–823839</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s02030</td> <td /> <td>Chr19∶1889627–1891189</td> <td /> </tr> <tr> <td /> <td>POPTR_0019s02060</td> <td>LRR and NB-ARC domains-containing disease resistance protein</td> <td>Chr19∶1922498–1924769</td> <td>AT4G10780</td> </tr> <tr> <td /> <td>POPTR_0019s02180</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶2029692–2032970</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s02200</td> <td>Disease resistance protein (CC-NBS-LRR class) family</td> <td>Chr19∶2058206–2062218</td> <td>AT1G12290</td> </tr> <tr> <td /> <td>POPTR_0019s02410</td> <td /> <td>Chr19∶2215592–2216219</td> <td /> </tr> <tr> <td /> <td>POPTR_0019s03080</td> <td>NB-ARC domain-containing disease resistance protein</td> <td>Chr19∶3094719–3100126</td> <td>AT4G27220</td> </tr> <tr> <td /> <td>POPTR_0019s03480</td> <td>similar to phosphoenolpyruvate-carboxylase kinase</td> <td>Chr19∶3561751–3563659</td> <td>AT3G04530</td> </tr> <tr> <td /> <td>POPTR_0019s03800</td> <td>similar to SNF2 domain-containing protein</td> <td>Chr19∶3871132–3874229</td> <td>AT1G08600</td> </tr> <tr> <td /> <td>POPTR_0019s05300</td> <td>similar to putative disease resistance gene analog NBS-LRR.</td> <td>Chr19∶5594198–5607481</td> <td>AT4G27190</td> </tr> <tr> <td /> <td>POPTR_0019s05990</td> <td>integral membrane transporter family protein</td> <td>Chr19∶6422952–6425657</td> <td>AT1G64890</td> </tr> <tr> <td /> <td>POPTR_0019s12230</td> <td>similar to GTP-binding protein</td> <td>Chr19∶13325801–13327862</td> <td>AT5G60860</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Target Gene Model <sup>a</sup> and Putative function <sup>b</sup> were derived from Joint Genome Institute (JGI: <a href="http://www.jgi.doe.gov/">http://www.jgi.doe.gov/</a>) and (TAIR : <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>). </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>TAIR Gene Model <sup>c</sup> were derived from TAIR (TAIR : <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>). </p> </div> </div> </div> </div> <div class="targetpredictionforpto-mir156,pto-mir159,pto-mir172andpto-mir319andnovelmirnasinandromonoeciousp.tomentosa" title="sec"> <div class="title" tagx="title" title="title">
Target Prediction for Pto-miR156, Pto-miR159, Pto-miR172 and Pto-miR319 and Novel miRNAs in Andromonoecious <i>P. tomentosa</i> </div> <p>To understand the functions of sex-specific flower development related miRNAs, the first step is to predict and experimentally validate their targets. Consequently, we predicted 25 target genes for Pto-miR156, Pto-miR159, Pto-miR172 and Pto-miR319 that were different from the previously defined <i>Populus</i> miRNA targets (miRBase 19.0), and a total of 464 targets were identified for the 78 novel miRNAs ( <a href="#pone.0062681.s005">Table S3</a>). The number of predicted targets varied from 1 to 15 per miRNA. The categorization of female and male-specific miRNA target genes according to biological process is shown in <a href="#pone-0062681-g005">Figure 5</a>. After putative functional annotation by BLAST searches against the TAIR, NCBI and JGI protein datasets, all targets of female and male-specific miRNAs that had a greater than twofold difference in expression between the sexes (Ratio≧2or≦0.5) were identified as candidate genes for gene functional enrichment analysis. All the target genes between the sexes were functionally characterized using enrichment analysis of gene sets, mostly as described by Gene Ontology terms ( <a href="#pone-0062681-g005">Figure 5</a>). GO analysis showed that response to hormone stimulus, negative regulation of gene expression, regulation of flower development, and organ formation were significantly enriched terms for biological processes. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g005"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
AgriGO analysis of target genes for statistically enriched GO terms in the ‘Biological process’ ontology. </div> <p>Coloring of GO term nodes is proportional to their significance as indicated by the scale. </p>   </div> </div> <div class="expressionprofilingofmirnasandtargetsinandromonoeciousp.tomentosa" title="sec"> <div class="title" tagx="title" title="title">
Expression Profiling of miRNAs and Targets in Andromonoecious <i>P. tomentosa</i> </div> <p>Andromonoecious poplar, an exceptional model system, eliminates confounding effects of genetic background of dioecious plants and thus is a powerful system in which to identify accurate sex-specific factors related to flower development. Therefore, the comparison of the small RNA profiles of female and male flowers of andromonoecious poplar was informative for characterization of functional miRNAs involved in sex-specific flower development. We sequenced 134 conserved miRNA sequences and 78 predicted novel miRNA sequences. 70.1% (94/134) of conserved miRNA sequences were up-regulated in female flower, and only 40 conserved miRNAs were up-regulated in male flower. A total of 104 conserved and 2 novel miRNAs were verified as differentially expressed by greater than two-fold between the two libraries (ratio &amp;gt;2 or &amp;lt;0.5 and p&amp;lt;0.01), including 94 conserved and 2 novel miRNAs that were up-regulated in female flower and 37 conserved and 4 novel miRNAs that were up-regulated in male flower ( <a href="#pone-0062681-t002">Table 2</a> and <a href="#pone-0062681-t003">3</a>). Moreover, 3 conserved miRNAs (miRNA 397, miRNA 6459 and miRNA6462) and 72 novel miRNAs, including 11 male-specific and 61 female-specific, were expressed in a sex-specific manner ( <a href="#pone-0062681-t004">Table 4</a> and <a href="#pone-0062681-t005">5</a>). To examine the detailed expression pattern of these potential miRNAs in andromonoecious poplar flowers, the 27 differentially expressed miRNAs, including four conserved and 23 novel miRNAs, were verified by real-time quantitative PCR ( <a href="#pone-0062681-g006">Figure 6A</a>). Overall, comparisons of these data correspond to the deep sequencing results, indicating that the miRNA expression levels determined by high-throughput sequencing are reliable. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g006"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Pearson correlation Coefficient (PCC) heat map representing expression of miRNAs and their targets in andromonoecious poplar flower organs. </div> <p>Red and yellow indicate higher levels of transcriot and green and blue indicate lower levels of transcript. The names of samples are shown at the top. The gene model and gene annotation are shown on the right side. A: Heatmap of conserved and novel miRNAs. B: Heatmap of target genes. </p>   </div> <p>Base on the results of gene enrichment analysis, 31 putative target genes were selected as candidate genes for gene expression analysis ( <a href="#pone-0062681-t007">Table 7</a> and <a href="#pone-0062681-g006">Figure 6B</a>). The expression of miRNA target genes was detected using real-time quantitative PCR. For most target genes, expression was negatively correlated with the levels of a given miRNA, which is in accordance with the gene silencing function of miRNAs. It should be noted that expression of <i>CONSTANS-LIKE 6</i> ( <i>COL6</i>), the target of PtoF25, was not correlated with the abundance of PtoF25 ( <a href="#pone-0062681-g006">Figure 6B</a>), indicating that <i>COL6</i> expression might be regulated by other mechanisms. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0062681.t007">10.1371/journal.pone.0062681.t007</a> <span class="label" tagx="label" title="label">Table 7</span>  <div class="title" tagx="title" title="title">
Candidate targets of novel and conserved miRNAs in andromonoecious <i>P. tomentosa</i>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>miRNA</td> <td>Cleave sites</td> <td>Target Gene Model <sup>a</sup> </td> <td>Putative function <sup>b</sup> </td> <td>Accession number</td> <td>TAIR Gene Model</td> </tr> </thead> <tbody> <tr> <td>miRNA167</td> <td>392</td> <td>POPTR_0013s00750</td> <td>similar to MAINTENANCE OF METHYLATION</td> <td>KC477290</td> <td>AT5G66750</td> </tr> <tr> <td>Pto-F6</td> <td>235</td> <td>POPTR_0005s14640</td> <td>similar to gibberellin-responsive protein 5</td> <td>KC477281</td> <td>AT3G02380</td> </tr> <tr> <td>Pto-F6</td> <td>305</td> <td>POPTR_0004s18020</td> <td>similar to myb family transcription factor (MYB4);</td> <td>KC477284</td> <td>AT1G74670</td> </tr> <tr> <td>Pto-F6</td> <td>394</td> <td>POPTR_0019s15030</td> <td>similar to DECREASED DNA METHYLATION 1</td> <td>KC297686</td> <td>AT1G08060</td> </tr> <tr> <td>Pto-F7</td> <td>193</td> <td>POPTR_0004s10800</td> <td>similar to CONSTANS-like protein CO2.</td> <td>KC477292</td> <td>AT4G38620</td> </tr> <tr> <td>Pto-F11</td> <td>228</td> <td>POPTR_0012s03530</td> <td>similar to EARLY-PHYTOCHROME-RESPONSIVE1;</td> <td>KC477285</td> <td>AT1G18330</td> </tr> <tr> <td>Pto-F14</td> <td>336</td> <td>POPTR_0010s13870</td> <td>similar to PHYTOCHROME AND FLOWERING TIME 1</td> <td>KC477286</td> <td>AT3G60460</td> </tr> <tr> <td>Pto-F16</td> <td>310</td> <td>POPTR_0015s07740</td> <td>calcium-dependent protein kinase family protein</td> <td>KC477266</td> <td>AT1G68520</td> </tr> <tr> <td>Pto-F19</td> <td>548</td> <td>POPTR_0002s02970</td> <td>similar to Zinc finger protein CONSTANS-LIKE 6</td> <td>KC477293</td> <td>AT1G25540</td> </tr> <tr> <td>Pto-F19</td> <td>270</td> <td>POPTR_0014s05370</td> <td>similar to myb family transcription factor</td> <td>KC297692</td> <td>AT1G75540</td> </tr> <tr> <td>Pto-F25</td> <td>139</td> <td>POPTR_0010s13490</td> <td>similar to Zinc finger protein CONSTANS-LIKE 6</td> <td>KC477294</td> <td>AT1G19870</td> </tr> <tr> <td>Pto-F28</td> <td>293</td> <td>POPTR_0002s02680</td> <td>calmodulin-binding family protein</td> <td>KC477283</td> <td>AT4G36920</td> </tr> <tr> <td>Pto-F36</td> <td>283</td> <td>POPTR_0005s18400</td> <td>similar to APETAL2-like protein</td> <td>KC477268</td> <td>AT3G10300</td> </tr> <tr> <td>Pto-F45</td> <td>481</td> <td>POPTR_0006s04590</td> <td>calcium-binding EF hand family protein</td> <td>KC477282</td> <td>AT3G15210</td> </tr> <tr> <td>Pto-F47</td> <td>125</td> <td>POPTR_0001s40770</td> <td>similar to Ethylene responsive element binding factor 4</td> <td>KC477274</td> <td>AT4G34760</td> </tr> <tr> <td>Pto-F47</td> <td>99</td> <td>POPTR_0004s17150</td> <td>similar to auxin-responsive protein</td> <td>KC477279</td> <td>AT4G36920</td> </tr> <tr> <td>Pto-F51</td> <td>815</td> <td>POPTR_0007s10780</td> <td>similar to APETAL2-like protein</td> <td>KC477295</td> <td>AT3G16990</td> </tr> <tr> <td>Pto-F54</td> <td>224</td> <td>POPTR_0008s10610</td> <td>similar to seed maturation protein PM36</td> <td>KC477289</td> <td>AT1G74740</td> </tr> <tr> <td>Pto-F54</td> <td>180</td> <td>POPTR_0001s16100</td> <td>similar to UNUSUAL FLORAL ORGANS protein</td> <td>KC477287</td> <td>AT1G30950</td> </tr> <tr> <td>Pto-F55</td> <td>321</td> <td>POPTR_0007s08330</td> <td>similar to ABA DEFICIENT 3</td> <td>KC477272</td> <td>AT5G66750</td> </tr> <tr> <td>Pto-F56</td> <td>513</td> <td>POPTR_0010s11350</td> <td>similar to CYTOKININ RESPONSE 1</td> <td>KC477269</td> <td>AT2G01830</td> </tr> <tr> <td>Pto-F56</td> <td>167</td> <td>POPTR_0007s12710</td> <td>similar to DECREASED DNA METHYLATION 1</td> <td>KC477291</td> <td>AT1G16540</td> </tr> <tr> <td>Pto-F58a</td> <td>311</td> <td>POPTR_0008s13720</td> <td>similar to CYTOKININ RESPONSE 1</td> <td>KC477270</td> <td>AT2G01830</td> </tr> <tr> <td>Pto-F64</td> <td>168</td> <td>POPTR_0018s06080</td> <td>similar to ethylene-responsive transcriptional coactivator</td> <td>KC477275</td> <td>AT3G24500</td> </tr> <tr> <td>Pto-F65</td> <td>152</td> <td>POPTR_0002s25330</td> <td>similar to ABA insensitive 3</td> <td>KC477273</td> <td>AT3G02380</td> </tr> <tr> <td>Pto-F66</td> <td>71</td> <td>POPTR_0004s10800</td> <td>similar to CONSTANS-like protein 1</td> <td>KC477292</td> <td>AT3G24650</td> </tr> <tr> <td>Pto-F66</td> <td>187</td> <td>POPTR_0017s14410</td> <td>similar to CONSTANS-like protein 1</td> <td>KC477267</td> <td>AT5G43700</td> </tr> <tr> <td>Pto-F68</td> <td>253</td> <td>POPTR_0012s04470</td> <td>similar to Auxin efflux carrier component 1</td> <td>KC477277</td> <td>AT3G02380</td> </tr> <tr> <td>Pto-F68</td> <td>166</td> <td>POPTR_0005s05550</td> <td>similar to Auxin-induced protein IAA4</td> <td>KC477278</td> <td>AT1G73590</td> </tr> <tr> <td>Pto-F68</td> <td>89</td> <td>POPTR_0008s10100</td> <td>similar to gibberellin 2-oxidase</td> <td>KC477280</td> <td>AT1G47990</td> </tr> <tr> <td>Pto-F73</td> <td>421</td> <td>POPTR_0013s01640</td> <td>dormancy/auxin associated family protein</td> <td>KC477276</td> <td>AT5G21482</td> </tr> <tr> <td>Pto-F76</td> <td>221</td> <td>POPTR_0006s23760</td> <td>similar to cytokinin oxidase; similar to putative (CKX5)</td> <td>KC477271</td> <td>AT1G56220</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Target Gene Model <sup>a</sup> and Putative function <sup>b</sup> were derived from Joint Genome Institute (JGI: <a href="http://www.jgi.doe.gov/">http://www.jgi.doe.gov/</a>) and (TAIR : <a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>). </p> </div> </div> </div> </div> <div class="apossiblefunctionalnetworkofsex-specificflowerdevelopmentrelatedmirnasinandromonoeciousp.tomentosa" title="sec"> <div class="title" tagx="title" title="title">
A Possible Functional Network of Sex-specific Flower Development Related miRNAs in Andromonoecious <i>P. tomentosa</i> </div> <p>The idea that miRNAs could play important roles in flower development has been suggested in previous studies. However, no systematic investigation has been conducted into how flower development may be regulated via miRNA-mediated processes. In the present study, a possible functional network, which connects flower development related miRNAs with their targets were first reported ( <a href="#pone-0062681-t007">Table 7</a> and <a href="#pone-0062681-g007">Figure 7</a>). Base on the gene functional annotation, all target genes were divided into four subgroups as shown in <a href="#pone-0062681-g007">Figure 7</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0062681.g007"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <div class="title" tagx="title" title="title">
A possible functional network of flower development related miRNAs in andromonoecious <i>P. tomentosa</i>. </div> <p>Black line represent repression of target gene expression.</p>   </div> <p>The first subgroup related to floral development was regulated by 9 miRNAs including Pto-F6, Pto-F11, Pto-F14, Pto-F19, Pto-25, Pto-36, Pto-F51, Pto-F54 and Pto-F66. Four members of the <i>CONSTANS-LIKE</i> gene family were negatively regulated by three female-specific miRNAs including Pto-F19, Pto-F25 and Pto-F66. Of these, Pto-F66 was found to target 2 members of the <i>CONSTANS-LIKE2</i> gene family. Also, <i>EARLY-PHYTOCHROME-RESPONSIVE1</i> ( <i>EPR1</i>) and <i>PHYTOCHROME AND FLOWERING TIME 1</i> ( <i>PFT1</i>) were negatively regulated by two male-specific miRNAs, Pto-F11 and Pto-F14, respectively. In our study, we detected two novel miRNAs, Pto-F36 and Pto-51, targeting <i>APETALA 2</i> ( <i>AP2</i>). Female flower specific Pto-F54 was found to target two genes, <i>UNUSUAL FLORAL ORGANS</i> ( <i>UFO</i>) and <i>SEED MATURATION PROTEIN</i> ( <i>PM36</i>). </p> <p>Three miRNAs are involved in regulation of the second subgroup, genes that relate to Ca <sup>2+</sup> transport. Two female flower specific miRNAs, Pto-F28 and Pto-F45, were found to be negative factors for the expression of <i>CALMODULIN-BINDING FAMILY PROTEIN</i> and <i>CALCIUM-BINDING EF HAND FAMILY PROTEIN</i>. By contrast, <i>CALCIUM-DEPENDENT PROTEIN KINASE FAMILY PROTEIN</i> ( <i>CDPK</i>) gene expression was repressed by Pto-F16, a male-specific miRNA. </p> <p>In the third subgroup, there were 10 miRNAs involved in hormone related gene regulation. All of these miRNAs were female flower specific, except Pto-F6, which had expression significantly higher in female flower than in male. Pto-F68 has three target genes, including <i>AUXIN EFFLUX CARRIER COMPONENT 1</i>, <i>AUXIN-INDUCED PROTEIN</i> ( <i>IAA4</i>) and <i>GIBBERELLIN 2-OXIDASE</i>. By contrast, <i>ETHYLENE RESPONSIVE ELEMENT BINDING FACTOR 4</i> ( <i>EREBF4</i>) and <i>AUXIN-RESPONSIVE PROTEIN</i> ( <i>SAUR29</i>) were targeted by the same novel miRNA, Pto-F47. Two genes related to ABA, <i>ABA DEFICIENT 3</i> ( <i>ABA3</i>) and <i>ABA INSENSITIVE 3</i> ( <i>ABI3</i>), were targeted by female flower specific miRNAs Pto-F55 and Pto-F65, respectively. </p> <p>In the last subgroup, genes that relate to DNA methylation were regulated by 3 miRNAs. <i>MAINTENANCE OF METHYLATION1</i> ( <i>MET1</i>) was negatively regulated by miRNA167, which had expression significant higher in male flower than in female. By contrast, two members of <i>DECREASED DNA METHYLATION 1</i> ( <i>DDM1</i>) were negatively regulated by Pto-F6, which had higher expression in female flower and by female flower specific Pto-F56, respectively. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="overviewofthedeepsequencingdatasets" title="sec"> <div class="title" tagx="title" title="title">
Overview of the Deep Sequencing Datasets</div> <p>Although an increasing number of studies about miRNAs and their expression patterns from different tissues have been reported, there are still few studies on sex-specific miRNAs between female and male flowers of poplar. In this study, andromonoecious poplar, an exceptional model system, was used to eliminate confounding effects of genetic background of dioecious plants. We generated sRNA libraries from female and male flower of andromonoecious poplar and obtained over 40 million reads of 18–30 nt per library, substantially increasing the available data on poplar flower sRNAs <a href="#pone.0062681-Pantaleo1">[23]</a>, <a href="#pone.0062681-Zhao1">[24]</a>. Mapping analysis of unique sRNAs revealed that over 60% of the identified sRNAs were previously unknown, suggesting that even more sRNAs remain to be identified. The size distribution of small RNAs indicated that the numbers of 21-nt sRNAs were significantly higher than 24-nt sRNAs in female flower, suggesting that female flower development may require more sRNAs in this size class. By contrast, the numbers of 21-nt sRNAs and 24-nt sRNAs were not significantly different in male flower libraries. The 24-nt sRNAs mainly comprised siRNAs associated with repeats and transposons <a href="#pone.0062681-Lippman1">[25]</a>, <a href="#pone.0062681-Lu1">[26]</a>. 24-nt sRNA numbers in male flower were significantly higher than in female, suggesting that a number of siRNAs related to repeats and transposons are involved in male flower development. </p> </div> <div class="conservedmirnasexpressedinflower" title="sec"> <div class="title" tagx="title" title="title">
Conserved miRNAs Expressed in Flower</div> <p>We detected dramatically different expression levels among different members of the same miRNA family, suggesting that expression of conserved miRNAs varies in andromonoecious <i>P. tomentosa</i>. These conserved miRNAs have been identified in multiple flowering pathways. For example, three miRNA families (miR172, miR159/miR319 and miR156) are involved in flowering-time regulation <a href="#pone.0062681-JonesRhoades1">[27]</a>. In this study, miRNA159 and miRNA319 expression in female flower was 38.8-fold and 51.42-fold higher, respectively, than in male flower. miRNA172 was weakly expressed in female and male flower, with no significant difference. By contrast, miRNA156 was expressed in male flower at higher levels than in female flower. This result suggested that different miRNAs are involved in female and male flowering time regulation. miRNA159 and miRNA319 also have been reported to act with miRNA164 and miRNA167 in specifying particular cell types during the later stages of flower development <a href="#pone.0062681-Nag1">[28]</a>. The present study revealed that miRNA164 was not expressed in female or male flower, indicating miRNA164 might show stage-specific expression in flower development. miRNA167 was up-regulated dramatically in male flower, indicating that it may play an important role in specifying male flower cell types. miRNA156, miRNA157 and miRNA172 may be components of a regulatory pathway mediating the transition between the vegetative and reproductive phases in plants <a href="#pone.0062681-Nag1">[28]</a>. miRNA166, targeting <i>APETALA2</i> and type III homeodomain-leucine zipper (HD-Zip) family genes, was dramatically up-regulated in female flower, suggesting <i>AP2</i> and HD-ZiP genes might be negatively regulated in female flower <a href="#pone.0062681-Ji1">[29]</a>. miR167 and miR160 are thought to control transcription in response to the phytohormone auxin by targeting mRNAs coding for ARF DNA-binding proteins <a href="#pone.0062681-Rajagopalan1">[30]</a>, <a href="#pone.0062681-Reinhart1">[31]</a>. These miRNAs were detected in this study as having opposite expression patterns between female and male flower, suggesting that two miRNAs involved in regulation of auxin responses in female flower are different compare with in male flower. </p> </div> <div class="novelmirnasdisplayeddifferentexpressionpatternsbetweenfemaleandmaleflower" title="sec"> <div class="title" tagx="title" title="title">
Novel miRNAs Displayed Different Expression Patterns between Female and Male Flower</div> <p>The 78 novel miRNAs identified here spanned a size range of 20–22 nt, and the majority started with a 5′ U, consistent with results in Japanese apricot <a href="#pone.0062681-Gao1">[32]</a>. The majority of novel miRNAs showed relatively low expression levels, which was consistent with previous studies predicting that novel miRNAs are often expressed at lower levels than conserved miRNAs <a href="#pone.0062681-Fahlgren1">[33]</a>. The miRNA*s were generally less abundant than the corresponding mature miRNAs. It has been reported that miRNA*s have important functions through regulation of the miRNA precursors themselves when miRNA*s were more abundant than their corresponding miRNAs <a href="#pone.0062681-German1">[34]</a>. miRNA*s identified were used as extremely strict criteria to identify novel miRNAs <a href="#pone.0062681-Meyers1">[35]</a>. In our study, only 8 unique miRNAs were detected with complementary miRNA*s. Most novel miRNAs do not have identified miRNAs*, likely because of the depth of sequencing, which results in a low probability of identifying low-frequency miRNA*s. The numbers of female-specific novel miRNAs was 6-fold higher than male-specific novel miRNAs, suggesting that abundant novel miRNAs are involved in female flower development. A large proportion of the novel sRNAs identified in andromonoecious <i>P. tomentosa</i> did not have perfect matches in different genome databases, providing more evidence to support our conclusion that these novel miRNAs might be poplar-specific and were first reported in present study. Among all of the novel miRNAs, 11 miRNAs were found with perfect matches in other species, suggesting that some of these newly identified miRNAs may be broadly conserved within angiosperms. </p> </div> <div class="novelmirnaslocatedinsexdeterminationregions" title="sec"> <div class="title" tagx="title" title="title">
Novel miRNAs Located in Sex Determination Regions</div> <p>Although studies on determination of gender indicate that dioecy in <i>Populus</i> is genetically controlled, the precise gender-determining systems remain unclear. Based on genetic mapping results, a gender determination locus mapped to different positions on chromosome XIX, depending upon the <i>Populus</i> species <a href="#pone.0062681-Gaudet1">[36]</a>– <a href="#pone.0062681-Paolucci1">[38]</a>. Klevebring et al. (2009) found that the proposed sex-determining peritelomeric region of chromosome XIX showed a distinctive pattern of sRNA occurrence that differed significantly from the rest of the genome <a href="#pone.0062681-Klevebring1">[39]</a>. In our study, 5 novel miRNAs and 33 targets genes were located on chromosome XIX, indicating that these miRNAs might be involved in sex determination of poplar. The annotated results revealed that over one-third of targets were annotated as disease resistance genes. It has been reported that a NBS–LRR gene supercluster was located on <i>Populus</i> chromosome XIX. These results are consistent with the hypothesis that resistance to a floral pathogen and regulation of gender determination coevolved in a region of the genome that experiences reduced recombination <a href="#pone.0062681-Tuskan2">[22]</a>. Interestingly, Pto-F6 was predicted to match a target that encodes DNA (cytosine-5) -methyltransferase implying that DNA methylation might act in flower development. </p> </div> <div class="mirnatargetsandsub-networkanalysis" title="sec"> <div class="title" tagx="title" title="title">
miRNA Targets and Sub-network Analysis</div> <p>To better understand the functions of identified miRNAs, we sought to identify potential targets in andromonoecious <i>P. tomentosa</i> using computational analyses. Many targets of conserved miRNAs have been predicted and their functions annotated <a href="#pone.0062681-Lu2">[40]</a>, <a href="#pone.0062681-Lu3">[41]</a>. Integrating target genes and miRNA expression data identified sub-networks for further investigation. <i>COL1</i> encodes a zinc finger protein belonging to the first groupings of the flowering-time gene <i>CONSTANS</i> ( <i>CO</i>) <a href="#pone.0062681-Griffiths1">[42]</a>. Over-expression of <i>COL1</i> can shorten the period of two distinct circadian rhythms by affecting a light input pathway <a href="#pone.0062681-Ledger1">[43]</a>. In our study, <i>COL1</i> was targeted by female flower specific Pto-F66, which was expressed in female flowers suggesting that the novel miRNA Pto-F66 likely plays a negative role in the light input pathway in female flower. By contrast, <i>COL6</i> is part of the third groupings of <i>CO</i>, and is a target of Pto-F19 and Pto-F25. However, these two miRNAs were not correlated with <i>COL6</i> transcript abundance, indicating that <i>COL6</i> expression might be regulated by other mechanisms. The target of male specific miRNA Pto-F11, <i>EPR1</i> encodes a nuclear-localized MYB protein showing rapid induction in response to red light irradiation. <i>EPR1</i> was significantly down-regulated in male flower, suggesting that this primary phytochrome-responsive factor is likely repressed by the male flower specific novel miRNA Pto-F11 <a href="#pone.0062681-Kuno1">[44]</a>. Another gene involved in the photoperiodic pathway, <i>PFT1</i> transcript abundances were significantly decreased in male flower. <i>PFT1</i> is a target of PtoF14 and functions downstream of phyB to regulate the expression of <i>FLOWERING LOCUS T</i> ( <i>FT</i>), providing evidence that Pto-F14 might be involved in regulation of flowering time by affecting the light quality pathway <a href="#pone.0062681-Cerdn1">[45]</a>. </p> <p>The <i>AP2</i> gene family contains 144 members encoding at least one AP2 DNA-binding domain and the varied biological functions of AP2 family members range from development to stress and defense responses <a href="#pone.0062681-Weigel1">[46]</a>– <a href="#pone.0062681-Riechmann1">[48]</a>. Conserved miRNA172 has been identified as an important factor regulating AP2 transcription factors <a href="#pone.0062681-Chuck1">[17]</a>. In our study, miRNA172 was equally expressed in female and male flower, but two novel miRNAs, Pto-F36 and Pto-F51, were detected with female-specific expression. <i>AP2</i> transcript abundance decreased in female flower, providing evidence that these two miRNAs act in regulation of <i>AP2</i>. <i>UFO</i> encodes an F-box protein required for the determination of floral-organ and floral-meristem identity <a href="#pone.0062681-Hepworth1">[49]</a>. <i>PM36</i> is involved in reproductive system development. These genes were targeted by the female-specific novel miRNA Pto-F54 and <i>UFO</i> and <i>PM36</i> expression were down-regulated in female flower, indicating the broad biological functions of Pto-F54. Calcium is a ubiquitous second messenger in plant signal transduction cascades <a href="#pone.0062681-Cheng1">[50]</a>. Three novel miRNAs were identified as related to Ca <sup>2+</sup> transport, including one male flower specific and two female flower specific miRNAs, suggesting that Pto-F16, Pto-28 and Pto-45 are likely play different roles in regulating calcium transport in female and male flower development. </p> <p>Hormones play a central role in the coordination of internal developmental processes with external environmental signals <a href="#pone.0062681-Stepanova1">[51]</a>. A series of genes related to phytohormone synthesis and metabolism are differentially expressed between female and male during flower development (unpublished data). In our study, 13 genes related to five important phytohormones were found to be targeted by 12 novel miRNAs, suggesting that these miRNAs act broadly in regulation of phytohormone synthesis and metabolism. Previous work reported that auxin is necessary for GA-mediated control of root growth, and that attenuation of auxin transport or signaling delays the GA-induced disappearance of RGA (repressor of gal-3) <a href="#pone.0062681-Fu1">[52]</a>. Among these miRNAs, Pto-F68 targets three genes involved in auxin transport and gibberellin synthesis suggesting that Pto-F68 may regulate the interaction of auxin and gibberellin. Also, a series of physiological studies indicate that ethylene and auxin interact in a number of different ways depending on the cell type, developmental stage, and environmental conditions <a href="#pone.0062681-Stepanova1">[51]</a>. <i>EREBF4</i> and <i>SAUR2</i> expression were downregulated in female flower and targeted by Pto-F47 miRNA. This result may provide a clue that deepens our understanding of the important regulatory functions of miRNA Pto-F47 in phytohormone interactions. Poplar male flowers abscise earlier than female flowers, suggesting the ABA accumulation and regulation of related genes might be different between female and male flowers. <i>ABA3</i> encodes an enzyme that catalyzes the generation of the sulfurylated form of MoCo, a cofactor required by aldehyde oxidase, which functions in the last step of ABA biosynthesis in plants <a href="#pone.0062681-Xiong1">[53]</a>. <i>ABI3</i> encodes a DNA binding factor targeted to promoters responsive to ABA and auxin <a href="#pone.0062681-Nag2">[54]</a>. These two genes related to ABA biosynthesis and responses are targeted by novel miRNAs Pto-F55 and Pto-F65, which have female flower specific expression, suggesting these miRNAs have negative roles in flower abscission through regulation of ABA metabolism. </p> <p> <i>MET1</i>, encoding DNA (cytosine-5) –methyltransferase, which maintains CG methylation, was down-regulated in male flower <a href="#pone.0062681-Finnegan1">[55]</a>. <i>DDM1</i>, encoding a chromatin remodeling complex with ATPase activity that can remodel nucleosomes, was up-regulated in male flower <a href="#pone.0062681-Brzeski1">[56]</a>. <i>MET1</i> and <i>DDM1</i> play opposite roles in the regulation of DNA methylation levels; therefore, the different expression of these genes in female and male flowers might alter DNA methylation levels, which are higher in female flower than male (unpublished). Our study indicated that miRNA167, Pto-F6 and Pto-F56 might play a positive regulatory role in maintaining DNA methylation levels of female flower through regulation of <i>MET 1</i> and <i>DDM1</i>. </p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterials" title="sec"> <div class="title" tagx="title" title="title">
Plant Materials</div> <p>Four-hundred and sixty unrelated individuals, representing almost the whole geographic distribution of <i>P. tomentosa</i>, were collected in the national nursery of Guan Xian County (Shandong Province, China). After observation of flower morphology, three 29-year-old unrelated andromonoecious clones (‘2–14’, ‘3605’, and ‘5103’) were found. </p> <p>All three individuals were used in our study. Sampling that female and male flowers were dissected from the andromonoecious flowers was performed at the last phase of flower development, before pollination. All materials were collected from poplar individuals and immediately frozen in liquid nitrogen, and stored at −80°C for isolation of RNA. To eliminate genetic backgroud, the two bulks were made by mixing equal amounts of RNA extracted from three andromonoecious poplar female and male flower, respectively. </p> <p>This study was carried out in strict accordance with the recommendations in the Guide for Observational and field studies ( <a href="http://www.plosone.org/static/publication">http://www.plosone.org/static/publication</a> <b>)</b>. All necessary permits were obtained for the described field studies. The sampling of all individuals of <i>P</i>. <i>tomentosa</i> was approved by Youhui Zhang, director of National Garden of <i>P</i>. <i>tomentosa</i> in Guan Xian County, Shandong Province. </p> </div> <div class="totalrnaisolation,srnalibraryconstruction,andsolexasequencing" title="sec"> <div class="title" tagx="title" title="title">
Total RNA Isolation, sRNA Library Construction, and Solexa Sequencing</div> <p>Total RNA was isolated from female and male floral tissue by a modified CTAB method <a href="#pone.0062681-Chang1">[57]</a> with isopropanol instead of lithium chloride for RNA precipitation. Extracted RNA was used for sRNA library construction. Small RNAs were sequenced using an Illumina HiSeq 2000 at the Shanghai Bio Institute. </p> <p>Sequence data from this article have been deposited in the GenBank Data Library under the accession Nos: KC477266–KC477295. </p> </div> <div class="bioinformaticanalysesofsequencingdataandtargetpredictionofmirnas" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic Analyses of Sequencing Data and Target Prediction of miRNAs</div> <p>Clean reads were screened from raw reads by removing contaminating reads, including sequences with adapters, without the insert tag, with poly-A tails, shorter than 18 nt, or longer than 30 nt. Clean reads were then aligned with the <i>Populus</i> genome ( <a href="http://www.phytozome.net/poplar">http://www.phytozome.net/poplar</a>) using SOAP <a href="#pone.0062681-Li1">[58]</a>, with no mismatches allowed. Small RNA tags matching exons and introns of mRNAs were excluded from further analysis. Tags matching non-coding RNAs, including rRNAs, scRNA, tRNAs, snRNAs, and snoRNAs, deposited at the Rfam ( <a href="http://www.sanger.ac.uk/Software/Rfam">http://www.sanger.ac.uk/Software/Rfam</a>) and NCBI GenBank databases ( <a href="http://www.ncbi.nlm.nih.gov/blast/Blast.cgi">http://www.ncbi.nlm.nih.gov/blast/Blast.cgi</a>) <a href="#pone.0062681-Benson1">[59]</a> were also excluded from further analysis. The remaining unannotated sRNAs were searched against miRBase 19.0 with a maximum of two mismatches allowed <a href="#pone.0062681-GriffithsJones1">[20]</a> to identify conserved miRNAs in <i>P</i>. <i>beijingensis</i> and then the resulting sequences were screened for the presence of the characteristic hairpin structures using the program RNAfold ( <a href="http://www.tbi.univie.ac.at/ivo/RNA/ViennaRNA-1.8.1.tar.gz">http://www.tbi.univie.ac.at/ivo/RNA/ViennaRNA-1.8.1.tar.gz</a>) <a href="#pone.0062681-Mathews1">[60]</a>. We used the prediction software Mireap ( <a href="https://sourceforge.net/projects/mireap/">https://sourceforge.net/projects/mireap/</a>) to predict novel miRNAs by exploring the secondary structures, DL1 cleavage sites, and minimum free energies of the unannotated sRNA tags that could be mapped to the <i>Populus</i> genome. Basic criteria <a href="#pone.0062681-Meyers1">[35]</a> were used for selecting the potential novel miRNAs. Target genes were predicted in accordance with previous research <a href="#pone.0062681-Allen1">[61]</a>, <a href="#pone.0062681-Schwab1">[62]</a> but with higher stringency; only three mismatches were allowed in the miRNA/target duplex. Target searches were performed using <i>Populus</i> transcripts deposited in the Phytozome v7.0 ( <a href="http://www.phytozome">http://www.phytozome</a>. net/) and Pfam databases ( <a href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>), and the results were used to annotate the functions of potential targets. We compared miRNA expression between the two libraries (control and treatment) to determine which miRNAs were differentially expressed. We normalized the expression levels of miRNAs within each library to get the expression of transcripts per million. </p> </div> <div class="5′-race" title="sec"> <div class="title" tagx="title" title="title">
5′-RACE</div> <p>RNA Ligase-Mediated 5′-RACE (RLM-RACE) was performed with the First Choice RLM-RACE Kit (Ambion) according to the manufacturer’s instructions, with slight modifications. Briefly, 10 <i />µg of total RNA was directly ligated to the 5′ adaptor followed by reverse transcription with an oligo (dT) primer. PCR was performed with 5′ adaptor primers and 3′ gene-specific primers ( <a href="#pone.0062681.s006">Table S4</a>) using cDNA as the template. The RACE products were gel-purified, cloned, and sequenced. </p> </div> <div class="qrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
qRT-PCR</div> <p>Small RNAs (&amp;lt;200 nt) were isolated using the mirVana miRNA Isolation Kit (Ambion, USA) following the manufacturer’s instructions. Quantitative real-time PCR (qRT-PCR) was carried out as previously described <a href="#pone.0062681-Shi1">[63]</a> in an ABI PRISM 7500 Fast Real-time PCR System (Ambion, USA) using the SYBR Premix Ex TaqTM Kit (TaKaRa, Japan). The reactions were carried out in a 20 µl volume containing 2 µl of diluted cDNA, 200 nM of each primer, and 19 PCR Master Mix with the following conditions: 95°C for 30 s, and 45 cycles of 95°C for 5 s, 58°C for 15 s, and 72°C for 20 s. Then, a thermal denaturing cycle of 95°C for 15 s and 60°C for 1 min was applied to determine the dissociation curves, which were used to verify the specificity of PCR amplifications. All reactions were run in triplicate for each sample. Twelve miRNAs, including six conserved and six novel miRNAs were validated, and 5.8S rRNA was selected as a reference gene for normalization <a href="#pone.0062681-Lu3">[41]</a> ( <a href="#pone.0062681.s007">Table S5</a> and <a href="#pone.0062681.s008">Table S6</a>). </p> </div> <div class="networkanalysis" title="sec"> <div class="title" tagx="title" title="title">
Network Analysis</div> <p>The details of sex-specific flower development related miRNAs functional network methods were done according to previously described <a href="#pone.0062681-Wang1">[64]</a>. Comparative miRNAome analysis combined with realtime quantitative PCR and RACE–PCR revealed 23 miRNAs expressed in andromonoecious poplar flower and each of them has functional targets. Negative correlation between the miRNA and target expression profiles was validated using real-time quantitative PCR. These results indicated that miRNA-mediated regulation does occur in flower development and that these miRNAs with dynamic changes ultimately down-regulate their targets. All miRNA and functional targets were connected for network analysis. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p> <b>Morphological and microscopic observation of andromonoecious poplar.</b> A:Morphological characteristics of andromonoecious poplar flowers. B Pollen tubes stained with aniline blue, showing that pollen viability is high and that pollen grows well during selfing. Pt pollen tube, Sti stigma, Pg pollen grain. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0062681.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p> <b>Mature and precursor sequences and the predicted stem-loop structures of novel miRNAs from andromonoecious poplar flower tissue. </b> The mature miRNAs are in green, and the miRNA*s are in red. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0062681.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p>134 loci of 38 unique miRNAs.</p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0062681.s003.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p>11 poplar novel miRNAs were detected in other species.</p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0062681.s004.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S3</span>  <p>miRNAs and their targets.</p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0062681.s005.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S4</span>  <p>Primers for 5′ RACE mapping of miRNA cleavage sites.</p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0062681.s006.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S5</span>  <p>Real-time PCR primer sequences for candidate miRNAs.</p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0062681.s007.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S6</span>  <p>Real-time PCR primer sequences for candidate target genes.</p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0062681.s008.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0062681-Tuskan1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuskan</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DiFazio</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teichmann</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Poplar genomics is getting popular: The impact of the poplar genome project on tree research </span>. <span class="source" tagx="source" title="source">Plant Biol</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">2</span>– <span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15095128">15095128</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Zluvova1" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zluvova</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zak</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janousek</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vyskot</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Dioecious Silene latifolia plants show sexual dimorphism in the vegetative stage</span>. <span class="source" tagx="source" title="source">BMC Plant Bio</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">208</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20854681">20854681</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Sheppard1" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheppard</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunner</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krutovskii</span> <span class="given-names" tagx="given-names" title="given-names">KV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rottmann</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skinner</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">A <i>DEFICIENS</i> homolog from the dioecious tree black cottonwood is expressed in female and male floral meristems of the two-whorled, unisexual flowers </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">124</span>: <span class="fpage" tagx="fpage" title="fpage">627</span>– <span class="lpage" tagx="lpage" title="lpage">639</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11027713">11027713</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Rottmann1" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rottmann</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meilan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheppard</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunner</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skinner</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Diverse effects of overexpression of LEAFY and PTLF, a poplar ( <i>Populus</i>) homolog of LEAFY/FLORICAULA, in transgenic poplar and Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">235</span>– <span class="lpage" tagx="lpage" title="lpage">245</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10849341">10849341</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Boes1" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boes</span> <span class="given-names" tagx="given-names" title="given-names">TK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strauss</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> ( <span class="year" tagx="year" title="year">1994</span>) <span class="mixed-article-title" title="mixed-article-title">Floral phenology and morphology of black cottonwood, <i>Populus trichocarpa</i> ( <i>Salicaceae</i>) AM J BOT </span>. <span class="volume" tagx="volume" title="volume">81</span>: <span class="fpage" tagx="fpage" title="fpage">562</span>– <span class="lpage" tagx="lpage" title="lpage">567</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-An1" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">XM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">ZL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bo</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Isolation of a <i>LEAFY</i> homolog from <i>Populus tomentosa</i>: expression of <i>PtLFY</i> in <i>P. tomentosa</i> floral buds and <i>PtLFY</i>-IR-mediated gene silencing in tobacco ( <i>Nicotiana tabacum</i>) </span>. <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">89</span>– <span class="lpage" tagx="lpage" title="lpage">100</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21104255">21104255</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Song1" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">YP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">KF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bo</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">ZY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">DQ</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Sex-specific DNA methylation and gene expression in andromonoecious poplar</span>. <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">1393</span>– <span class="lpage" tagx="lpage" title="lpage">1405</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22476437">22476437</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Chen1" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Small RNAs-secrets and surprises of the genome. Plant J</span>. <span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">941</span>– <span class="lpage" tagx="lpage" title="lpage">958</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Bartel1" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">MicroRNAs: genomics, biogenesis, mechanism, and function</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">116</span>: <span class="fpage" tagx="fpage" title="fpage">281</span>– <span class="lpage" tagx="lpage" title="lpage">297</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14744438">14744438</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Llave1" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llave</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrington</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Cleavage of Scarecrow like mRNA targets directed by a class of Arabidopsis miRNA</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">297</span>: <span class="fpage" tagx="fpage" title="fpage">2053</span>– <span class="lpage" tagx="lpage" title="lpage">2056</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12242443">12242443</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Navarro1" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Navarro</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunoyer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jay</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnold</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dharmasiri</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">A plant miRNA contributes to antibacterial resistance by repressing auxin signaling</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">312</span>: <span class="fpage" tagx="fpage" title="fpage">436</span>– <span class="lpage" tagx="lpage" title="lpage">439</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16627744">16627744</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Wu1" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">MY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conway</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">The sequential action of miR156 and miR172 regulates developmental timing in Arabidopsis</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">138</span>: <span class="fpage" tagx="fpage" title="fpage">750</span>– <span class="lpage" tagx="lpage" title="lpage">759</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19703400">19703400</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Baker1" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baker</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sieber</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wellmer</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyerowitz</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">The early extra petals1 mutant uncovers a role for microRNA miR164c in regulating petal number in Arabidopsis </span>. <span class="source" tagx="source" title="source">Curr Biol</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">303</span>– <span class="lpage" tagx="lpage" title="lpage">315</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15723790">15723790</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Aukerman1" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aukerman</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakai</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Regulation of flowering time and floral organ identity by a MicroRNA and its APETALA2-like target genes </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">2730</span>– <span class="lpage" tagx="lpage" title="lpage">2741</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14555699">14555699</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Chen2" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">A microRNA as a translational repressor of APETALA2 in Arabidopsis flower development</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">303</span>: <span class="fpage" tagx="fpage" title="fpage">2022</span>– <span class="lpage" tagx="lpage" title="lpage">2025</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12893888">12893888</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Schmid1" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmid</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uhlenhaut</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godard</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bressan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Dissection of floral induction pathways using global expression analysis</span>. <span class="source" tagx="source" title="source">Development</span> <span class="volume" tagx="volume" title="volume">130</span>: <span class="fpage" tagx="fpage" title="fpage">6001</span>– <span class="lpage" tagx="lpage" title="lpage">6012</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14573523">14573523</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Chuck1" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chuck</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meeley</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irish</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakai</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hake</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Themaize tasselseed4 microRNA controls sex determination and meristem cell fate by targeting Tasselseed6/indeterminate spikelet1 </span>. <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">1517</span>– <span class="lpage" tagx="lpage" title="lpage">1521</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18026103">18026103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Sun1" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sederoff</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in trees Plant Mol Biol</span>. <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">37</span>– <span class="lpage" tagx="lpage" title="lpage">53</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Barakat1" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barakat</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diloreto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Depamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlson</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Conservation and divergence of microRNAs in <i>Populus</i> </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">481</span>– <span class="lpage" tagx="lpage" title="lpage">496</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18166134">18166134</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-GriffithsJones1" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffiths-Jones</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saini</span> <span class="given-names" tagx="given-names" title="given-names">HK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Dongen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enright</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">miRBase: tools for microRNA genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">36</span>: <span class="fpage" tagx="fpage" title="fpage">154</span>– <span class="lpage" tagx="lpage" title="lpage">158</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Yin1" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeFazio</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Counter</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sewell</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Genome structure and emerging evidence of an incipient sex chromosome in <i>Populus</i> </span>. <span class="source" tagx="source" title="source">Genome Res</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">422</span>– <span class="lpage" tagx="lpage" title="lpage">430</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18256239">18256239</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Tuskan2" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuskan</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DiFazio</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faivre-Rampant</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaudet</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harfouche</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">The obscure events contributing to the evolution of an incipient sex chromosome in <i>Populus</i>: a retrospective working hypothesis </span>. <span class="source" tagx="source" title="source">Tree Genet Genomes</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">559</span>– <span class="lpage" tagx="lpage" title="lpage">571</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Pantaleo1" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pantaleo</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szittya</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moxon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miozzi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moulton</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Identification of grapevine microRNAs and their targets using high-throughput sequencing and degradome analysis </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">960</span>– <span class="lpage" tagx="lpage" title="lpage">976</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20230504">20230504</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Zhao1" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">CZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frazier</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">YY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bi</span> <span class="given-names" tagx="given-names" title="given-names">YP</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Deep sequencing identifies novel and conserved microRNAs in peanuts ( <i>Arachis hypogaea</i> L.) </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">3</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20047695">20047695</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Lippman1" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lippman</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martienssen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">The role of RNA interference in heterochromatic silencing</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">431</span>: <span class="fpage" tagx="fpage" title="fpage">364</span>– <span class="lpage" tagx="lpage" title="lpage">370</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15372044">15372044</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Lu1" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tej</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haudenschild</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Elucidation of the small RNA component of the transcriptome</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">309</span>: <span class="fpage" tagx="fpage" title="fpage">1567</span>– <span class="lpage" tagx="lpage" title="lpage">1569</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16141074">16141074</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-JonesRhoades1" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">MicroRNAs and their regulatory roles in plants</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>– <span class="lpage" tagx="lpage" title="lpage">53</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16669754">16669754</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Nag1" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nag</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jack</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Sculpting the flower: the role of microRNAs in flower development</span>. <span class="source" tagx="source" title="source">Curr Top Dev Biol</span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">349</span>– <span class="lpage" tagx="lpage" title="lpage">378</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20705188">20705188</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Ji1" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yumul</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">ARGONAUTE10 and ARGONAUTE1 regulate the termination of floral stem cells through two microRNAs in Arabidopsis </span>. <span class="source" tagx="source" title="source">PLoS Genet</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1001358</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21483759">21483759</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Rajagopalan1" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajagopalan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaucheret</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trejo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">A diverse and evolutionarily fluid set of microRNAs in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">3407</span>– <span class="lpage" tagx="lpage" title="lpage">3425</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17182867">17182867</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Reinhart1" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weinstein</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in plants</span>. <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">1616</span>– <span class="lpage" tagx="lpage" title="lpage">1626</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12101121">12101121</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Gao1" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhuang</span> <span class="given-names" tagx="given-names" title="given-names">WB</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">High-throughput sequencing of small RNAs and analysis of differentially expressed microRNAs associated with pistil development in Japanese apricot </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">371</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22863067">22863067</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Fahlgren1" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahlgren</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapman</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sullivan</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">High-throughput sequencing of Arabidopsis micro RNAs: evidence for frequent birth and death of MIRNA genes PLoS One </span>. <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">e219</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-German1" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">German</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pillay</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeong</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hetawal</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Global identification of microRNA-target RNA pairs by parallel analysis of RNA ends</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">941</span>– <span class="lpage" tagx="lpage" title="lpage">946</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18542052">18542052</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Meyers1" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baulcombe</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Criteria for annotation of plant microRNAs</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">3186</span>– <span class="lpage" tagx="lpage" title="lpage">3190</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19074682">19074682</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Gaudet1" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaudet</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jorge</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paolucci</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beritognolo</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scarascia Mugnozza</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic linkage maps of <i>Populus nigra</i> L. including AFLPs, SSRs, SNPs and sex trait </span>. <span class="source" tagx="source" title="source">Tree Genet Genomes</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">36</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Pakull1" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pakull</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groppe</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Markussen</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fladung</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic linkage mapping in aspen ( <i>Populus tremula</i> L. and <i>Populus tremuloides</i> Michx.) </span>. <span class="source" tagx="source" title="source">Tree Genet Genomes</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">505</span>– <span class="lpage" tagx="lpage" title="lpage">515</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Paolucci1" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paolucci</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaudet</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jorge</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beritognolo</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terzoli</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic linkage maps of <i>Populus alba</i> L. and comparative mapping analysis of sex determination across <i>Populus</i> species </span>. <span class="source" tagx="source" title="source">Tree Genet Genomes</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">863</span>– <span class="lpage" tagx="lpage" title="lpage">875</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Klevebring1" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klevebring</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Street</span> <span class="given-names" tagx="given-names" title="given-names">NR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahlgren</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrington</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">A genome-wide profiling of <i>Populus</i> small RNAs </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">620</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20021695">20021695</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Lu2" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Stress-responsive microRNAs in <i>Populus</i> </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">131</span>– <span class="lpage" tagx="lpage" title="lpage">151</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18363789">18363789</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Lu3" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Novel and mechanical stress- responsive microRNAs in <i>Populus trichocarpa</i> that are absent from Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">2186</span>– <span class="lpage" tagx="lpage" title="lpage">2203</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15994906">15994906</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Griffiths1" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffiths</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunford</span> <span class="given-names" tagx="given-names" title="given-names">RP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coupland</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laurie</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">The evolution of CONSTANS-like gene families in barley, rice, and Arabidopsis</span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">131</span>: <span class="fpage" tagx="fpage" title="fpage">1855</span>– <span class="lpage" tagx="lpage" title="lpage">1867</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12692345">12692345</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Ledger1" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ledger</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strayer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashton</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kay</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putterill</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of the function of two circadian-regulated CONSTANS-LIKE genes</span>. <span class="source" tagx="source" title="source">The Plant J</span> <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">15</span>– <span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11359606">11359606</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Kuno1" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuno</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Møller</span> <span class="given-names" tagx="given-names" title="given-names">SG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinomura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">XM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">The novel MYB protein EARLY-PHYTOCHROME-RESPONSIVE1 is a component of a slave circadian oscillator in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">2476</span>– <span class="lpage" tagx="lpage" title="lpage">2488</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14523250">14523250</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Cerdn1" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerdán</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chory</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Regulation of flowering time by light quality</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">423</span>: <span class="fpage" tagx="fpage" title="fpage">881</span>– <span class="lpage" tagx="lpage" title="lpage">885</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12815435">12815435</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Weigel1" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">1995</span>) <span class="mixed-article-title" title="mixed-article-title">The APETALA2 domain is related to a novel type of DNA binding domain</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">388</span>– <span class="lpage" tagx="lpage" title="lpage">389</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7773013">7773013</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Okamuro1" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okamuro</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caster</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Villarroel</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Montagu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jofuku</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span> ( <span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">The AP2 domain of APETALA2 defines a large new family of DNA binding proteins in Arabidopsis</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">7076</span>– <span class="lpage" tagx="lpage" title="lpage">7081</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9192694">9192694</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Riechmann1" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riechmann</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyerowitz</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">The AP2/EREBP family of plant transcription factors Biological chemistry</span>. <span class="volume" tagx="volume" title="volume">379</span>: <span class="fpage" tagx="fpage" title="fpage">633</span>– <span class="lpage" tagx="lpage" title="lpage">646</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Hepworth1" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hepworth</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klenz</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haughn</span> <span class="given-names" tagx="given-names" title="given-names">GW</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>UFO</i> in the Arabidopsis inflorescence apex is required for floral-meristem identity and bract suppression </span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">223</span>: <span class="fpage" tagx="fpage" title="fpage">769</span>– <span class="lpage" tagx="lpage" title="lpage">778</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16244866">16244866</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Cheng1" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willmann</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Calcium Signaling through Protein Kinases. The Arabidopsis calcium-dependent protein kinase gene family </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">129</span>: <span class="fpage" tagx="fpage" title="fpage">469</span>– <span class="lpage" tagx="lpage" title="lpage">485</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12068094">12068094</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Stepanova1" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stepanova</span> <span class="given-names" tagx="given-names" title="given-names">AN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yun</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Likhacheva</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Multilevel interactions between ethylene and auxin in Arabidopsis roots</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">2169</span>– <span class="lpage" tagx="lpage" title="lpage">2185</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17630276">17630276</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Fu1" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">XD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harberd</span> <span class="given-names" tagx="given-names" title="given-names">NP</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Auxin promotes Arabidopsis root growth by modulating gibberellin response</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">421</span>: <span class="fpage" tagx="fpage" title="fpage">740</span>– <span class="lpage" tagx="lpage" title="lpage">743</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12610625">12610625</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Xiong1" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishitani</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis LOS5/ABA3 locus encodes a molybdenum cofactor sulfurase and modulates cold stress and osmotic stress– responsive gene expression </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">2063</span>– <span class="lpage" tagx="lpage" title="lpage">2083</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11549764">11549764</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Nag2" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nag</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maity</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DasGupta</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Dual DNA binding property of ABA insensitive 3 like factors targeted to promoters responsive to ABA and auxin </span>. <span class="source" tagx="source" title="source">Plant Mol Bio</span> <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">821</span>– <span class="lpage" tagx="lpage" title="lpage">838</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16270233">16270233</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Finnegan1" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finnegan</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Genger</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peacock</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">DNA methylation in plants</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Physiol Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012234">15012234</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Brzeski1" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brzeski</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jerzmanowski</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Deficient in DNA methylation 1 ( <i>DDM1</i>) defines a novel family of chromatin-remodeling factors </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">278</span>: <span class="fpage" tagx="fpage" title="fpage">823</span>– <span class="lpage" tagx="lpage" title="lpage">828</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12403775">12403775</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Chang1" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puryear</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cairney</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">A simple and efficient method for isolating RNA from pine trees</span>. <span class="source" tagx="source" title="source">Plant Mol Biol Rep</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">113</span>– <span class="lpage" tagx="lpage" title="lpage">116</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Li1" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kristiansen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">SOAP: short oligonucleotide alignment program</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">713</span>– <span class="lpage" tagx="lpage" title="lpage">714</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18227114">18227114</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Benson1" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benson</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karsch-Mizrachi</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ostell</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sayers</span> <span class="given-names" tagx="given-names" title="given-names">EW</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">GenBank</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">32</span>– <span class="lpage" tagx="lpage" title="lpage">37</span>. </span> </li> <li tag="ref"> <a name="pone.0062681-Mathews1" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mathews</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabina</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuker</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure </span>. <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">288</span>: <span class="fpage" tagx="fpage" title="fpage">911</span>– <span class="lpage" tagx="lpage" title="lpage">940</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10329189">10329189</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Allen1" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gustafson</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrington</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">microRNA directed phasing during trans-acting siRNA biogenesis in plants</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">121</span>: <span class="fpage" tagx="fpage" title="fpage">207</span>– <span class="lpage" tagx="lpage" title="lpage">221</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15851028">15851028</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Schwab1" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riester</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schommer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmid</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Specific effects of microRNAs on the plant transcriptome</span>. <span class="source" tagx="source" title="source">Dev Cell</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">517</span>– <span class="lpage" tagx="lpage" title="lpage">527</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15809034">15809034</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Shi1" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Facile means for quantifying microRNA expression by real-time PCR</span>. <span class="source" tagx="source" title="source">Biotechniques</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">519</span>– <span class="lpage" tagx="lpage" title="lpage">525</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16235564">16235564</a> </span> </span> </li> <li tag="ref"> <a name="pone.0062681-Wang1" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">ZM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">LG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bian</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">A Comparative miRNAome Analysis Reveals Seven Fiber Initiation-Related and 36 Novel miRNAs in Developing Cotton Ovules </span>. <span class="source" tagx="source" title="source">Molecular Plant</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">889</span>– <span class="lpage" tagx="lpage" title="lpage">900</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22138860">22138860</a> </span> </span> </li> </ul> </div> </div>  </body></html>