<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Genomics</span> <span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4700669">4700669</a> </span> <span class="publisher-id" title="publisher-id">2330</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-015-2330-6">10.1186/s12864-015-2330-6</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Transcriptome analysis of secondary cell wall development in <i>Medicago truncatula</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Huanzhong</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">huanzhong.wang@uconn.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Jung Hyun</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jung.yang@uconn.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Fang</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">Fang.Chen@unt.edu</span> </span> <a href="#Aff2" /> <a href="#Aff3" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torres-Jerez</span> <span class="given-names" tagx="given-names" title="given-names">Ivone</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">itjerez@noble.org</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Yuhong</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">ytang@noble.org</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Mingyi</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mingy.wang@gmail.com</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Qian</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">qian.du@uconn.edu</span> </span> <a href="#Aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">Xiaofei</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">xcheng@noble.org</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">Jiangqi</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jwen@noble.org</span> </span> <a href="#Aff2" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">Richard</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">Richard.Dixon@unt.edu</span> </span> <a href="#Aff2" /> <a href="#Aff3" /> </span> <span class="citation_author_institution" id="Aff1">[], </span> <span class="citation_author_institution" id="Aff2">[], </span> <span class="citation_author_institution" id="Aff3">[], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-1-1</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2016-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">17</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">23</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-4-29</span> <span class="accepted" title="accepted">accepted: 2015-12-17</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Legumes are important to humans by providing food, feed and raw materials for industrial utilizations. Some legumes, such as alfalfa, are potential bioenergy crops due to their high biomass productivity. Global transcriptional profiling has been successfully used to identify genes and regulatory pathways in secondary cell wall thickening in <i>Arabidopsis</i>, but such transcriptome data is lacking in legumes. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>A systematic microarray assay and high through-put real time PCR analysis of secondary cell wall development were performed along stem maturation in <i>Medicago truncatula</i>. More than 11,000 genes were differentially expressed during stem maturation, and were categorized into 10 expression clusters. Among these, 279 transcription factor genes were correlated with lignin/cellulose biosynthesis, therefore representing putative regulators of secondary wall development. The b-ZIP, NAC, WRKY, C2H2 zinc finger (ZF), homeobox, and HSF gene families were over-represented. Gene co-expression network analysis was employed to identify transcription factors that may regulate the biosynthesis of lignin, cellulose and hemicellulose. As a complementary approach to microarray, real-time PCR analysis was used to characterize the expression of 1,045 transcription factors in the stem samples, and 64 of these were upregulated more than 5-fold during stem maturation. Reverse genetics characterization of a cellulose synthase gene in cluster 10 confirmed its function in xylem development. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>This study provides a useful transcriptome and expression resource for understanding cell wall development, which is pivotal to enhance biomass production in legumes. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (doi:10.1186/s12864-015-2330-6) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Transcriptome</div> <div class="kwd" title="kwd">
Expression</div> <div class="kwd" title="kwd">
Secondary cell wall</div> <div class="kwd" title="kwd">
Development</div> <div class="kwd" title="kwd">
Medicago truncatula</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000199</span> <span class="institution" title="institution">U.S. Department of Agriculture</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">ID0E3PBG6788</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Huanzhong</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">UConn Research Excellence Program award</span> </div> <span class="award-id" tagx="award-id" title="award-id">ID0ECXBG6789</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Huanzhong</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Oklahoma Department of Energy Bioenergy Center</span> </div> <span class="award-id" tagx="award-id" title="award-id">ID0EE5BG6790</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">Richard</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">the Samuel Roberts Noble Foundation</span> </div> <span class="award-id" tagx="award-id" title="award-id">ID0EO6BG6791</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">Richard</span> </span> </span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2016" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Background</div> <p>Legumes include many agriculture important grain, pasture and agroforestry species that are well known for their ability to fix nitrogen through root nodules [ <a href="#CR1">1</a>, <a href="#CR2">2</a>]. Alfalfa ( <i>Medicago sativa</i>) is a major forage legume, with an average annual value of more than $8 billion in the USA alone [ <a href="#CR3">3</a>]. It’s high biomass productivity also makes alfalfa a potential bioenergy legume [ <a href="#CR4">4</a>]. Most cultivated legumes, including alfalfa, are multiploid with complex segregation and inheritance patterns [ <a href="#CR2">2</a>, <a href="#CR5">5</a>]. Barrel medic ( <i>Medicago truncatula</i>) is diploid with a relatively small genome, and has been adopted as a model species for legume genomics [ <a href="#CR6">6</a>– <a href="#CR8">8</a>]. In addition, <i>M. truncatula</i> is an important self-regenerating annual pasture species, especially in southern Australia [ <a href="#CR9">9</a>]. <i>M. truncatula</i> is also grown in rotation with cereal crops for improving soil quality [ <a href="#CR10">10</a>]. </p> <p>Modulation of secondary cell wall composition, such as reducing lignin content, improves alfalfa quality with better digestibility and higher fermentable sugar yields for biofuel production [ <a href="#CR11">11</a>– <a href="#CR13">13</a>], and reduced lignin transgenic alfalfa with increased digestibility has recently been de-regulated in the US by USDA-APHIS [ <a href="#CR14">14</a>] and is in commercial production. The bioengineering applications in alfalfa have largely relied on the identification of secondary wall biosynthetic and/or regulatory genes from the model legume <i>M. truncatula</i> [ <a href="#CR12">12</a>]. Secondary cell wall development in vascular and interfascicular tissues involves a large number of biosynthetic genes and is regulated at the transcriptional level [ <a href="#CR15">15</a>, <a href="#CR16">16</a>]. The NAM, ATAF1/2, and CUC2 (NAC) domain and MYB domain transcription factors (TFs) function as master regulators. The NAC domain TFs include VASCULAR-RELATED NAC-DOMAIN6 (VND6), VND7, NAC SECONDARY WALL THICKENING PROMOTING (NST1), NST2 and SECONDARY WALL-ASSOCIATED NAC DOMAIN 1 (SND1) [ <a href="#CR17">17</a>– <a href="#CR19">19</a>]. MYB domain TFs, i.e. MYB46 and MYB83, also function as master regulators for secondary cell wall development, but are downstream of the NAC domain TFs [ <a href="#CR20">20</a>, <a href="#CR21">21</a>]. Many other TFs are further downstream of the NAC and MYB domain master regulators, and form the hierarchical and non-hierarchical regulation networks. The regulatory pathways orchestrate the biosynthesis of cellulose, hemicelluloses and lignin [ <a href="#CR22">22</a>]. In <i>M. tuncatula</i>, a NAC domain TF was identified as an important regulator of secondary cell wall development, but with distinct regulatory mechanism [ <a href="#CR23">23</a>, <a href="#CR24">24</a>]. </p> <p>Global transcriptional profiling has been successfully used to identify genes and regulatory pathways in secondary cell wall biosynthesis in <i>Arabidopsis thaliana</i>, but such transcriptome data and systems analysis of TF functions in stem maturation is still lacking in legumes. The composition of secondary cell walls significantly affects the quality of forage legumes [ <a href="#CR12">12</a>, <a href="#CR13">13</a>, <a href="#CR25">25</a>]. In addition, cell wall biosynthesis is regulated differently in legumes compared to Arabidopsis [ <a href="#CR24">24</a>, <a href="#CR26">26</a>]. The Affymetrix Medicago genome array has been a critical tool for functional analysis and gene expression studies in legumes [ <a href="#CR7">7</a>, <a href="#CR27">27</a>, <a href="#CR28">28</a>]. To better understand secondary cell wall development and its regulation in <i>M. truncatula</i>, we have performed transcriptome microarray assay and high through-put quantitative real time PCR analysis. A large of number of genes were differentially expressed during stem maturation, and were placed into 10 expression clusters. TFs putatively functioning in secondary wall development were identified based on their expression patterns. Gene co-expression network analysis was used to identify TFs that may regulate the biosynthesis of individual components of the secondary cell wall. We further performed real-time PCR analysis to characterize the expression of 1,045 TFs, and 64 of these were upregulated more than 5-fold during stem maturation. This research provide a useful resource for molecular characterization of secondary cell wall development in legumes. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="analysisofsecondarycellwalldevelopmentinthemedicagostem" title="sec"> <div class="title" tagx="title" title="title">
Analysis of secondary cell wall development in the <i>Medicago</i> stem </div> <p>To understand the secondary cell wall development in vascular bundles and interfascicular fibers in <i>M. truncatula</i>, we characterized the developing primary stems by UV microscopy of cross sections. Stems of 7-week old <i>Medicago</i> plants develop 10-11 internodes under greenhouse conditions. Internodes at the middle of the stem grow much longer than the young internodes at the top, or the old internodes at the bottom. In order to collect representative and consistent samples, we chose only the center portion of each internode for histological analysis (Fig. <a href="#Fig1">1a</a>). In internode 2, which is located just below the growing apex, a few primary vascular vessels were observed and showed weak blue autofluorescence due to lignin deposition (Fig. <a href="#Fig1">1b</a>). In the third internode, more vessel elements developed in the vascular region, but no interfascicular fibers were observed (Fig. <a href="#Fig1">1c</a>). In internode 5, vascular bundles were well developed. Interfascicular fibers accumulated a considerable amount of secondary cell wall material, although the autofluorescence signal was still weak compared to the vascular bundle regions (Fig. <a href="#Fig1">1d</a>). In internode 7, both vascular bundles and interfascicular fibers accumulated large amounts of secondary wall material (Fig. <a href="#Fig1">1e</a>). In the mature internode 9, both vascular and interfascicular regions expanded in width, and secondary cell wall development was almost complete (Fig. <a href="#Fig1">1f</a>). Similar to stem development in <i>Arabidopsis</i> [ <a href="#CR29">29</a>], our histological analysis indicated that the most prominent developmental change during Medicago stem maturation was secondary cell wall differentiation and accumulation of lignocellulosic compounds. The stem maturation analysis in this research is consistent with the results of previous cell wall composition and digestibility assays in Medicago stems [ <a href="#CR30">30</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Secondary cell wall development is correlated with stem maturation. <b>a</b> A representative stem of 7-week old <i>M. truncatula</i> plants. Medicago plants normally have ten to eleven internodes at this stage. <b>b</b> to <b>f</b> Stem cross sections observed under UV light. The blue color is autofluorescence from lignified secondary cell walls. From the plant apex to the bottom, the internodes become more mature and the stems accumulate more secondary cell walls </p>   </div> <p /> </div> <div class="microarrayanalysisofsecondarywalldevelopmentandstemmaturation" title="sec"> <div class="title" tagx="title" title="title">
Microarray analysis of secondary wall development and stem maturation</div> <p>To characterize the transcriptome profile during <i>Medicago</i> stem maturation, we collected stem samples for RNA extraction and subsequent microarray expression analyses. Five internodes, i.e. the aforementioned internodes 2, 3, 5, 7, and 9 from the <i>Medicago</i> primary stem, were collected in three biological replicates. Each sample was a pool of 10 segments harvested from the central 2 cm of each internode. These samples represented the different secondary cell wall developmental stages along the stem maturation process (Fig. <a href="#Fig1">1</a>). </p> <p>The Affymetrix Medicago Genechip genome array contains 61,281 probe sets, most of which (about 50,900) are from <i>M. truncatula</i> gene sequences. Genechip analysis has been instrumental in identifying biologically meaningful genes and characterizing gene expression patterns in <i>M. truncatula</i> and <i>M. sativa</i> [ <a href="#CR7">7</a>, <a href="#CR27">27</a>]. In this study, we used 15 arrays to investigate the transcriptome change during Medicago stem maturation. RNA samples from internode 2 were used as the reference for the remaining internode samples. Genes with expression levels significantly changed between the control (Internode 2) and the other four older internodes were identified using associative analysis [ <a href="#CR31">31</a>]. Analysis of the microarray results indicated that 11,380 genes were significantly differentially expressed ( <i>p</i> &amp;lt; 8.16 × 10 <sup>-7</sup> and fold change ≥ 2) in the relatively more mature internodes. The differentially expressed genes are listed in supplemental data (Additional file <a href="#MOESM1">1</a>). </p> <p>The large number of differentially expressed genes indicates that stem development and secondary cell wall biosynthesis are complex processes that involve many biosynthetic pathways. It is also possible that many of these genes may not be directly involved in cell wall development. To identify which of these genes are important for secondary cell wall development, we performed hierarchical clustering analysis. The differentially expressed genes were placed into 10 clusters based on their expression patterns in the five investigated internodes (Fig. <a href="#Fig2">2</a> and Additional file <a href="#MOESM1">1</a>: Table S1). Interestingly, most of these genes were categorized in either cluster 1 or cluster 10 (Additional file <a href="#MOESM1">1</a>: Table S1). In cluster 10, gene expression increased and reached highest expression level in internode 5, and then decreased to a moderate level in the two order internodes (Fig. <a href="#Fig3">3</a>). As shown in Fig. <a href="#Fig1">1</a>, cells in the vascular bundle and interfascicular regions started actively accumulating secondary wall materials in internode 5, indicating that the expression of cluster 10 genes coincides with secondary wall accumulation. We reasoned that genes in cluster 10 are likely have a role in secondary cell wall development, which is supported by analyzing the expression of lignin biosynthetic genes (this research). Genes in clusters 5 and 8 also showed highest expression in internode 5, while their expression pattern slightly different from cluster 10. Genes in these two clusters already showed increased expression in the 3rd internode, in which secondary cell wall biosynthesis is not very active. Despite the noticeable difference in expression pattern, we can’t rule out the possibility that genes in cluster 5 and 8 may also have functions in secondary cell wall development. We therefore include genes from clusters 5, 8 and 10 in our further analysis. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Gene cluster analysis based on expression patterns. Among the 61,281 probe sets in the Affymetrix <i>Medicago</i> genome array, 11,380 probe sets showed increased or decreased expression (≥2-fold) in older internodes compared to the second internode. Genes with changed expression were clustered into 10 different categories based on their expression pattern. Red dots indicate the branch point of each cluster. Black indicates no change; green indicates down-regulation and red indicates up-regulation of expression. IN refers to internode number </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Expression patterns of the 10 gene clusters. Genes in the 10 clusters showed distinct expression patterns in the five characterized internodes. The horizontal axes represent the relative gene expression in internode 2 (IN2), internode 3 (IN3), internode 5 (IN5), internode 7 (IN7) and internode 9 (IN9), respectively </p>   </div> <p /> <p>In cluster 1, genes had highest expression in internode 2, and the expression levels progressively decreased along the stem maturation gradient until the fifth internode, and kept at certain levels in the even older internodes (Fig. <a href="#Fig3">3</a>). In contrast, expression of the genes in clusters 2, 3, 4 6 fluctuated along the stem maturation gradient without obvious correlation with secondary wall accumulation. Genes in cluster 7 are down regulated in the 3rd internode, which contradicts with the microscopic observation on secondary cell wall accumulation. Therefore, genes in this cluster are not likely function in secondary cell wall biosynthesis. The expression of genes in cluster 9 increase along stem maturation gradient, and reach the highest level until the oldest 9th internode, in which secondary wall accumulation should have almost completed. We therefore hypothesize that the functions of genes in cluster 9 may not be in secondary cell wall biosynthesis, but rather in the senescence process. Genes in these seven clusters were therefore excluded from further analysis. </p> </div> <div class="expressionofsecondarywallrelatedgenesduringstemmaturation" title="sec"> <div class="title" tagx="title" title="title">
Expression of secondary wall related genes during stem maturation</div> <p>Cellulose and lignin are two of the major components of the secondary cell wall. We reasoned that genes responsible for the biosynthesis of cellulose and lignin should be correlated with the stem maturation process. Genes responsible for monolignol biosynthesis have been identified using a comparative genomic approach in <i>M. truncatula</i> [ <a href="#CR12">12</a>]. Probe sets corresponding to these monolignol synthetic genes were identified through sequence analysis (Additional file <a href="#MOESM1">1</a>: Table S2). Expression patterns of these genes were extracted from our transcriptome data and are presented in Fig. <a href="#Fig4">4a</a>. Hierarchical clustering analysis indicated that most of the monolignol biosynthesis genes were upregulated along the stem maturation gradient and were categorized in cluster 10 (red colored, Fig. <a href="#Fig4">4a</a>), but some members were down-regulated and clustered in cluster 1 (green colored, Fig. <a href="#Fig4">4a</a>). This indicates that the annotated monolignol biosynthetic genes in cluster 1 may not be responsible for secondary cell wall biosynthesis during normal plant development. We also analyzed the expression of genes responsible for cellulose biosynthesis. Twenty six probe sets were identified as either putative cellulose synthase genes or cellulose synthase-like genes. Similar to the monolignol biosynthesis genes, these probe sets were categorized either in cluster 1 or in cluster 10 (Fig. <a href="#Fig4">4b</a>). Cellulose synthase genes in cluster 10 are likely to function in secondary wall biosynthesis (see below), while the functions of the putative cellulose synthases categorized in cluster 1 are still unclear. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Expression of genes with putative functions in secondary cell wall synthesis. Most of the lignin biosynthetic genes ( <b>a</b>) and cellulose synthase and cellulose synthase-like genes ( <b>b</b>) are assigned in cluster 10 (red), but some lignin and cellulose synthetic genes are assigned in cluster 1 (green). Cinnamyl alcohol dehydrogenase (CAD); cinnamoyl CoA reductase (CCR); hydroxycinnamoyl CoA shikimate:quinate hydroxycinnamoyl transferase (HCT); L-phenylalanine ammonia-lyase (PAL); 4-coumaroyl CoA ligase (4CL); ferulic acid/ coniferaldehyde/ coniferyl alcohol 5-hydroxylase (F5H); coumaroylshikimate 3′-hydroxylase (C3′H); caffeoyl CoA 3- <i>O</i>-methyl transferase (CCOAOMT); caffeic acid/5-hydroxyferulic acid 3- <i>O</i>-methyltransferase (COMT) </p>   </div> <p /> </div> <div class="transcriptionfactorscorrelatedwithsecondarycellwalldevelopment" title="sec"> <div class="title" tagx="title" title="title">
Transcription factors correlated with secondary cell wall development</div> <p>Genes responsible for secondary cell wall development are well known to be under transcriptional regulation in <i>Arabidopsis</i>. A large number of transcription factors (TFs) play important roles in regulating secondary wall biosynthetic genes [ <a href="#CR18">18</a>, <a href="#CR21">21</a>, <a href="#CR22">22</a>, <a href="#CR32">32</a>– <a href="#CR34">34</a>]. In <i>M. truncatula</i>, we previously reported the identification of two secondary cell wall related TFs in the NAC domain and WRKY domain gene families [ <a href="#CR24">24</a>, <a href="#CR35">35</a>, <a href="#CR36">36</a>], but the regulatory pathways in secondary wall biosynthesis are still largely unknown in <i>M. truncatula</i>. To identify novel regulatory genes in secondary cell wall development, we focused our transcriptome data analysis on TFs. In cluster 10 alone, there are 279 TFs that are correlated with secondary cell wall development. There are 8 and 4 TFs in cluster 5 and cluster 8, respectively. Among these 291 transcription factors, the b-ZIP, NAC, WRKY, C2H2 ZF, homeobox, and HSF gene families were over-represented as determined by hypergeometric probability analysis (Table <a href="#Tab1">1</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Categorization of differentially expressed transcription factors in cluster 5, cluster 8 and cluster 10 </p>  <table> <thead> <tr> <th>Category</th> <th>Occurrence</th> <th> <i>P</i>-value* </th> </tr> </thead> <tbody> <tr> <td>MYB</td> <td>30</td> <td>0.921</td> </tr> <tr> <td>b-ZIP</td> <td>28</td> <td>0.014</td> </tr> <tr> <td>NAC</td> <td>24</td> <td>0.004</td> </tr> <tr> <td>WRKY</td> <td>24</td> <td>0.06</td> </tr> <tr> <td>C2H2 (ZF)</td> <td>21</td> <td>0.115</td> </tr> <tr> <td>AP2/EREBP</td> <td>20</td> <td>0.687</td> </tr> <tr> <td>HB</td> <td>19</td> <td>0.122</td> </tr> <tr> <td>bHLH</td> <td>17</td> <td>0.869</td> </tr> <tr> <td>C2C2 (ZF)</td> <td>8</td> <td>0.97</td> </tr> <tr> <td>HSF</td> <td>8</td> <td>0.13</td> </tr> <tr> <td>C3H (ZF)</td> <td>6</td> <td>0.901</td> </tr> <tr> <td>MADS</td> <td>6</td> <td>0.949</td> </tr> <tr> <td>AUX/IAA</td> <td>5</td> <td>0.186</td> </tr> <tr> <td>CCAAT-binding</td> <td>5</td> <td>0.727</td> </tr> <tr> <td>Scarecrow</td> <td>5</td> <td>-</td> </tr> <tr> <td>RING (ZF)</td> <td>4</td> <td>-</td> </tr> <tr> <td>ARF</td> <td>3</td> <td>0.994</td> </tr> <tr> <td>SPF1</td> <td>3</td> <td>-</td> </tr> <tr> <td>AOPBP</td> <td>2</td> <td>-</td> </tr> <tr> <td>E2F</td> <td>2</td> <td>0.194</td> </tr> <tr> <td>GARP</td> <td>2</td> <td>0.989</td> </tr> <tr> <td>SREBP</td> <td>2</td> <td>-</td> </tr> <tr> <td>BES1/BZR1</td> <td>1</td> <td>0.374</td> </tr> <tr> <td>DELLA</td> <td>1</td> <td>-</td> </tr> <tr> <td>DnaJ protein</td> <td>1</td> <td>-</td> </tr> <tr> <td>EIL protein</td> <td>1</td> <td>0.69</td> </tr> <tr> <td>GT-1</td> <td>1</td> <td>-</td> </tr> <tr> <td>LIM</td> <td>1</td> <td>0.374</td> </tr> <tr> <td>LOB</td> <td>1</td> <td>-</td> </tr> <tr> <td>Phaseolin G-box binding</td> <td>1</td> <td>-</td> </tr> <tr> <td>PHD (ZF)</td> <td>1</td> <td>0.999</td> </tr> <tr> <td>TCP</td> <td>1</td> <td>-</td> </tr> <tr> <td>TFIIIB</td> <td>1</td> <td>-</td> </tr> <tr> <td>TGACG-BP</td> <td>1</td> <td>-</td> </tr> <tr> <td>Trihelix</td> <td>1</td> <td>0.209</td> </tr> <tr> <td>Others</td> <td>34</td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*A total of 291 putative transcription factor genes from cluster 5, 8, and 10 were categorized based on functional domains. The distribution as frequencies of occurrence was presented in column 2. * <i>P</i> values were calculated in hypergeometric distribution analysis as the following: The proportion of transcription factors of a given class was compared with the proportion of that class in the whole set of transcription factors. AOPBP, ascorbate oxidase promoter-binding protein. SREBP, sucrose-responsive element binding factor. The classification of the transcription factors can be found at <a href="http://mtgea.noble.org/">http://mtgea.noble.org/</a> </p> </div>  <p /> <p>The large number of TFs (279, <i>P</i> = 8.81 × 10 <sup>-42</sup>) in cluster 10 may reflect the fact that many coordinated biological pathways are involved in secondary wall development. To understand which TFs are specific to secondary wall biosynthesis, we further analyzed the expression patterns of these genes using the available microarray data from the Medicago Gene Expression Atlas [ <a href="#CR27">27</a>]. We reasoned that specific or enhanced expression of TFs in secondary wall bearing tissues would be a strong indication for involvement in secondary wall biosynthesis. We collected expression data of the TFs in 18 different tissues from the gene expression Atlas [ <a href="#CR27">27</a>]. Genes with similar expression patterns are clustered together using hierarchical clustering analysis (Additional file <a href="#MOESM1">1</a>: Figure S1). TFs in one of the sub-clades are highlighted with a red bar (Additional file <a href="#MOESM1">1</a>: Figure S1). These genes are highly expressed in petiole, stem, root and pod, tissues which accumulate secondary cell walls (Fig. <a href="#Fig5">5</a>). This sub-clade included the NAC domain transcription factor <i>MtNST1</i> that functions as a master regulator of secondary cell wall development [ <a href="#CR23">23</a>, <a href="#CR24">24</a>]. Other TFs clustering in the same clade may also participate in secondary wall development, and serve as good candidates for reverse genetic analysis. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Expression pattern analysis of transcription factor genes highly expressed in secondary wall thickening. Expression of all selected 279 transcription factors was analyzed based on expression patterns in different tissues. Transcription factors highly expressed in tissues with secondary wall thickening was presented in this figure </p>   </div> <p /> <p>Gene co-expression network analysis has been a powerful tool for addressing gene function, and we therefore applied this approach to identify TFs that may regulate the biosynthesis of individual secondary cell wall components. Expression patterns of all putative TFs and genes responsible for lignin, cellulose and hemicellulose synthesis were collected from the Medicago Gene Expression Atlas [ <a href="#CR27">27</a>]. Data for leaf, petiole, stem, shoot (split-root experiment), sufficient N <sub>2</sub>, flower, pod, root and split root (nodulating) sufficient N <sub>2</sub> and the present stem experiment (37 chips in total) were used. In this analysis, only probe sets corresponding to Medicago genes were utilized, while those labeled as "AFFX", "Sme" or "RPTR" were removed. The resulting 12,576 probe sets were used to calculate the Pearson correlation coefficient values (PCC). TFs connected (|PCC| &amp;gt; 0.8) with lignin, cellulose and hemicellulose synthesis genes are presented in Fig. <a href="#Fig6">6</a> We included 11 monolignol synthetic genes in this analysis. A large number of the TFs are connected with monolignol biosynthesis, but most of them only connected with 1 or two of the lignin genes. TFs that are highlighted in the red oval are connected to as many as 5 monolignol synthetic genes. These TFs are likely important regulators of the whole monolignol synthetic pathway (Fig. <a href="#Fig6">6a</a>). In contrast, fewer TFs were connected to secondary wall related cellulose and hemicellulose synthesis genes (Fig. <a href="#Fig6">6b</a> and <a href="#Fig6">c</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Co-expression network analysis of transcription factors with secondary wall biosynthetic genes. <b>a</b>- <b>c</b> Networks showing transcription factors co-expressed with 11 lignin biosynthetic genes ( <b>a</b>), three cellulose synthase genes ( <b>b</b>), and four hemi-cellulose synthase genes. Transcription factor genes are in brown color and wall biosynthetic genes are in green color. Red oval circles transcription factors connected with four or more lignin synthetic genes. Nodes represent genes, and edges represent significant positive co-expression relationships between genes </p>   </div> <p /> </div> <div class="highthrough-putquantitativereal-timepcranalyses" title="sec"> <div class="title" tagx="title" title="title">
High through-put quantitative real-time PCR analyses</div> <p>The Affymetrix Genechip array includes a large portion of the transcriptome from <i>M. truncatula</i>, but some genes are missing from the Genechip. Furthermore, the differentially expressed transcripts identified from microarray analysis were not absolutely quantified and need to be confirmed through independent analysis. As a complementary approach to the microarray analysis, we investigated the expression of TFs along stem maturation using large scale real-time PCR. A community resource has been previously established to study transcription factors in <i>M. truncatula</i> [ <a href="#CR37">37</a>]. This community resource included gene specific oligonucleotide primers for 1045 TFs. Among these, 916 TFs were expressed in the internode samples. The expression of 64 TFs was increased more than 5-fold along the stem maturation gradient (Additional file <a href="#MOESM1">1</a>: Table S3). Twenty of them were already detected in the microarray experiment, therefore the expression of these TFs validated the microarray data. Interestingly, 18 of the TFs were present on the Genechip, but their expression changed less than 2-fold in the microarray data, indicating that real-time PCR is the more sensitive analytical method. Furthermore, there were 26 significantly differentially expressed TFs that were not present on the Genechip. Hypergeometric probability analysis indicated that C2H2 Zinc-finger and MYB domain transcription factors are over represented among the 64 TFs (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Distribution of transcription factors with increased expression along stem maturation gradient as determined by high-throughput q-RT PCR </p>  <table> <thead> <tr> <th>Category</th> <th>Frequencies of occurrence</th> <th>* <i>P</i>-value </th> </tr> </thead> <tbody> <tr> <td>MYB</td> <td>10</td> <td>0.276</td> </tr> <tr> <td>AP2/EREBP</td> <td>5</td> <td>0.52</td> </tr> <tr> <td>C2H2 (ZF)</td> <td>5</td> <td>0.285</td> </tr> <tr> <td>CCHC (ZF)</td> <td>4</td> <td>-</td> </tr> <tr> <td>bHLH</td> <td>3</td> <td>0.853</td> </tr> <tr> <td>b-ZIP</td> <td>3</td> <td>0.803</td> </tr> <tr> <td>HB</td> <td>3</td> <td>0.573</td> </tr> <tr> <td>NAC</td> <td>3</td> <td>0.625</td> </tr> <tr> <td>WRKY</td> <td>3</td> <td>0.763</td> </tr> <tr> <td>B-box (ZF)</td> <td>2</td> <td>-</td> </tr> <tr> <td>GRAS</td> <td>2</td> <td>0.691</td> </tr> <tr> <td>MADS</td> <td>2</td> <td>0.647</td> </tr> <tr> <td>RING (ZF)</td> <td>2</td> <td>-</td> </tr> <tr> <td>C2C2 (ZF)</td> <td>1</td> <td>0.952</td> </tr> <tr> <td>CCAAT</td> <td>1</td> <td>0.735</td> </tr> <tr> <td>DHHC (ZF)</td> <td>1</td> <td>-</td> </tr> <tr> <td>Dof (ZF)</td> <td>1</td> <td>-</td> </tr> <tr> <td>GRF (ZF)</td> <td>1</td> <td>-</td> </tr> <tr> <td>LOB</td> <td>1</td> <td>-</td> </tr> <tr> <td>PHD (ZF)</td> <td>1</td> <td>-</td> </tr> <tr> <td>SBP</td> <td>1</td> <td>-</td> </tr> <tr> <td>Others</td> <td>9</td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>* <i>P</i>-value was calculated similar to Table <a href="#Tab1">1</a> </p> </div>  <p /> </div> <div class="reversegeneticscharacterizationoftheirx1geneinm.truncatula" title="sec"> <div class="title" tagx="title" title="title">
Reverse genetics characterization of the <i>IRX1</i> gene in <i>M. truncatula</i> </div> <p>Most genes in the lignin biosynthetic pathway have been identified from <i>Medicago</i>, and several have been used to improve digestibility and sugar yield in alfalfa [ <a href="#CR11">11</a>– <a href="#CR13">13</a>]. Much less is known about the functions of the cellulose synthase genes in legumes. Our microarray analysis indicated that 19 CesA probe sets were categorized in cluster 10 (Fig. <a href="#Fig4">4b</a> red colored probe sets), and may therefore represent bona fide secondary cell wall specific cellulose synthase genes. In <i>Arabidopsis</i>, three cellulose synthases form a complex to catalyze cellulose synthesis in the secondary cell well; these are AtCesA4/IRX5, AtCesA7/IRX3/FRA5 and AtCesA8/IRX1/ FRA6 [ <a href="#CR38">38</a>– <a href="#CR41">41</a>]. Using blast analysis, we found that one of the probe sets in cluster 10, Mtr.5123.1.S1_at, has highest identity with AtIRX1/AtCesA8. We named the corresponding <i>Medicago</i> homolog gene <i>MtIRX1</i>. To investigate the biological function of <i>MtIRX1</i>, we performed a reverse genetic analysis. Using <i>MtIRX1</i> gene-specific primers, three mutant lines were recovered from the Medicago <i>Tnt1</i> insertional mutant population. In a previous mutant screening experiment, we had already identified a <i>Tnt1</i> insertion in the <i>MtIRX1</i> gene and named the mutant <i>mtirx1</i>- <i>1</i> (NF3892). We therefore renamed the newly identified mutants NF7461, NF8667 and NF17907 as <i>mtirx1</i>- <i>2</i>, <i>mtirx1</i>- <i>3 and mtirx1</i>- <i>4</i>, respectively. These four mutant lines showed similar irregular xylem phenotypes, and the phenotypes of <i>mtirx1</i>- <i>1</i> were examined in detail. </p> <p>In wild type plants, cross sections of the sixth internodes showed well organized and regular shaped vascular vessel and fiber cells (Fig. <a href="#Fig7">7a</a>), while most of the vessel cells were collapsed in the <i>mtirx1</i>- <i>1</i> mutant plants (Fig. <a href="#Fig7">7b</a> and <a href="#Fig7">c</a>). Furthermore, the stems of the mutant plants were thinner than the wild type (Fig. <a href="#Fig7">7a</a> to <a href="#Fig7">c</a>). To better characterize the mutant phenotype, we examined the stem cross sections with phloroglucinol staining. Phloroglucinol stains the lignin a bright red color, therefore enabling the visualization of cells with secondary cell walls. Compared to the wild type (Fig. <a href="#Fig7">7d</a>), the xylem vessel cells were totally distorted and collapsed in the <i>mtirx1</i>- <i>1</i> plants (Fig. <a href="#Fig7">7e</a> and <a href="#Fig7">f</a>). Furthermore, the cell walls of the vascular and interfascicular fiber cells in <i>mtirx1</i>- <i>1</i> plants were visually thinner compared to the wild type plants (Fig. <a href="#Fig7">7b</a> and <a href="#Fig7">e</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>Characterization of the <i>mtirx1</i>- <i>1</i> mutant by UV microscopy and phloroglucinol staining. <b>a</b> to <b>c</b>, stem cross sections of wild type ( <b>a</b>), and the <i>mtirx1</i> mutant ( <b>b</b>) were observed under UV light. <b>c</b> A higher magnification of the rectangle marked area in ( <b>b</b>). Collapsed xylem phenotype is evident. The blue color is the auto-fluorescence of lignin under UV light, and the red color is the auto-fluorescence of chloroplasts. <b>d</b> to <b>f</b>, cross sections of wild type ( <b>d</b>) and the <i>mtirx1</i> mutant ( <b>e</b>) after phloroglucinol staining. <b>f</b>, Staining of the <i>mtirx1</i> mutant at a higher magnification. Bars = 10 μm </p>   </div> <p /> <p>In addition to the collapsed xylem, the <i>mtirx1</i>- <i>1</i> mutant plants also showed obvious growth phenotypes. As shown in Fig. <a href="#Fig8">8</a>, the <i>mtirx1</i>- <i>1</i> mutants are extremely small (Fig. <a href="#Fig8">8a</a> and <a href="#Fig8">b</a>). The mutant stems are much thinner, with excessive pigment accumulation in the epidermal cells (Fig. <a href="#Fig8">8c</a> and <a href="#Fig8">d</a>). The mutant leaves are also significantly smaller than the wild type. At higher magnification, the mutant leaves showed a dark green color with evident trichomes covering on the leaf surface (Fig. <a href="#Fig8">8e</a> and <a href="#Fig8">f</a>). In <i>Arabidopsis</i>, mutations of secondary wall related cellulose synthase genes showed similar irregular xylem phenotypes, as well as retarded plant growth [ <a href="#CR38">38</a>– <a href="#CR41">41</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Growth phenotype of <i>mtirx1</i> mutant plants. <b>a</b> A wild type <i>M. truncatula</i> plant. <b>b</b>, A representative <i>mtirx1</i>- <i>1</i> mutant, which is extremely small compared to the wild type. <b>c</b> and <b>d</b>, Representative stems of wild type ( <b>c</b>) and the mutant ( <b>d</b>). <b>e</b> Leaves of wild type (left) and mutant (right). <b>f</b> The mutant leaf at a higher magnification </p>   </div> <p /> <p>The <i>mtirx1</i>- <i>1</i> mutant plants could not set seeds under our glasshouse conditions. To investigate what caused the infertile phenotype, we characterized the flower structure in detail. As shown in Fig. <a href="#Fig9">9a</a>, the flowers from the mutant plants are slightly smaller than the wild type; and the stigma of the mutant sticks out of the flower. Unlike the wild type (Fig. <a href="#Fig9">9b</a>), the mutant stigma has a smooth surface with no pollen attached (Fig. <a href="#Fig9">9c</a>). We further characterized the viability of pollen grains using Alexander’s staining. The results indicated that the mutant pollen was as viable as that of the wild type (Fig. <a href="#Fig9">9d</a> and <a href="#Fig9">e</a>). Therefore, the infertile phenotype of the <i>mtirx1</i>- <i>1</i> mutant may be due to defective anther dehiscence. In both <i>Arabidopsis</i> and <i>Medicago</i>, disruption of secondary cell wall synthesis can result in the prevention of anther dehiscence [ <a href="#CR26">26</a>, <a href="#CR42">42</a>, <a href="#CR43">43</a>]. Reverse genetic characterization of <i>MTIRX1</i> provides proof of concept that the transcriptome data may serve as a valuable resource for further secondary cell wall gene discovery in <i>M. truncatula</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 9</span> </a> </div>   <p>Defects of the reproductive organs in <i>mtirx1</i>- <i>1</i> mutant plants. <b>a</b>, Flowers of the wild type (left) and <i>mtirx1</i>- <i>1</i> mutant (right), showing the slightly smaller flower of the mutant plants. <b>b</b> and <b>c</b>, Close-up of the stigmas of wild type ( <b>b</b>) and <i>mtirx1</i>- <i>1</i> mutants ( <b>c</b>) showing the smooth surface of mutant stigma with no pollen attaching on the surface. <b>d</b> and <b>e</b>, Alexander’s staining showing that pollen is viable in both wild type ( <b>d</b>) and mutant plants ( <b>e</b>) </p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The inflorescence stem of the herbaceous model plant <i>Arabidopsis</i> forms vascular xylem and undergoes secondary growth, and has therefore been used to study secondary cell wall development [ <a href="#CR44">44</a>, <a href="#CR45">45</a>]. The advantages of studying <i>Arabidopsis</i> include its known genome, easy transformation and available mutant resources for functional studies. For alfalfa and other legume species, <i>M. truncatula</i> may be a better model system due to the high homology of their genomics sequences. In some cases, genes identified from <i>M. truncatula</i> can be directly utilized in alfalfa [ <a href="#CR13">13</a>]. A high quality genome sequence of <i>M. truncatula</i> is available [ <a href="#CR8">8</a>]. A gene expression atlas with a global view of gene expression in all major organs, and a large collection of <i>Tnt1</i> retrotransposon insertional mutant lines available for gene functional analysis [ <a href="#CR27">27</a>, <a href="#CR46">46</a>, <a href="#CR47">47</a>] make <i>Medicago</i> an excellent model for functional genomics of cell wall development in legumes. </p> <p>The cluster analysis of lignin and cellulose biosynthetic genes indicates that most of the genes are correlated with secondary cell wall development as supported by characterization of the cellulose synthase gene <i>MtIRX1</i>. However, some of the putative lignin and cellulose synthase genes were categorized in cluster 1, which showed opposite expression patterns compared to genes in secondary cell wall development (Fig. <a href="#Fig4">4a</a> and <a href="#Fig4">b</a>). The functions of these gene are still unclear. One possibility is that these genes may function in cell wall development in certain conditions other than normal growth conditions. This is supported by functional analysis of the cinnamoyl CoA reductases (CCR) genes in <i>Medicago</i> [ <a href="#CR48">48</a>]. Mt <i>CCR1</i>, identified in this previous study as the primary <i>CCR</i> gene in the monolignol biosynthetic pathway, is categorized in cluster 10. Mutation of <i>MtCCR1</i> resulted in drastic inhibition of plant growth. In contrast <i>MtCCR2</i> is in cluster 1, and its mutation has no effect on plant growth [ <a href="#CR48">48</a>]. </p> <p>The high-throughput real time qRT-PCR analysis identified 64 transcription factor genes that significantly up-regulated along the stem maturation gradient. Among these, twenty were already detected in the microarray experiment. The overlapping transcription factors between Genechip and qRT-PCR is low mainly because these two methods have obvious difference in sensitivity. We found that eighteen TFs changed less than 2-fold in the microarray assay, but changed over 5 times in the qRT-PCR analysis, indicating that qRT-PCR is much more sensitive than the Genechip assay. In addition, the annotation of <i>M. truncatula</i> genome sequence is still far from complete. Even though the <i>M. truncatula</i> Genechip have 1394 probe sets corresponding to transcription factor genes, we found 26 TFs significantly differentially expressed in qRT-PCR analysis, but were not present on the Genechip. The GeneChip approach heavily depends on chip design. For example, the current <i>M. Truncatula</i> chip contains only about 70-75 % of the genome contents [ <a href="#CR27">27</a>]. In addition, the Genechip can’t differentiate closely related genes, which can be addressed by the complementary real time qRT-PCR approach. </p> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>In this study, we found that over 11,000 genes are differentially expressed along the maturation axis of <i>Medicago</i> primary stems. It was previously estimated that about 15 % of the genes in <i>Arabidopsis</i> may be involved in cell wall synthesis, remodeling, or turnover [ <a href="#CR49">49</a>]. It is likely that many of the differentially expressed genes play important roles in secondary cell wall development. However, stem development along the maturation axis involves many other developmental processes, including the maturation of ground tissues. The transcriptome expression pattern should also reflect the development of these tissues. To identify genes with specific function in secondary cell wall development, we have here analyzed the transcriptome data with further consideration of expression specificity. Genes with enhanced expression in secondary cell wall bearing tissues, or co-expressed with wall biosynthetic genes were selected for further characterization. These analyses should help to reduce the false discovery rate. Bioinformatics analysis together with reverse genetics studies are especially helpful to pinpoint the genes involved in secondary cell wall formation [ <a href="#CR50">50</a>], and the present confirmation of the function of <i>MtIRX1</i> highlights the potential of the present dataset for further cell wall gene discovery. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialsandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and growth conditions</div> <p>Wild type <i>M. truncatula</i> plants and <i>Nicotiana tabacum</i> (tobacco) <i>Tnt1</i> retrotransposon tagged Medicago mutants [ <a href="#CR47">47</a>] were grown in the glasshouse, and used for collecting tissues for molecular and chemical characterization. Plants were grown at 24 °C day/20 °C night, with a 16-h day/8-h night photoperiod (150 μmol m <sup>-2</sup> sec <sup>-1</sup>) and 70–80 % relative humidity. </p> </div> <div class="microarrayanalysis" title="sec"> <div class="title" tagx="title" title="title">
Microarray analysis</div> <p>Stems of 7 weeks old <i>M. truncatula</i> plants (totally 10-11 internodes) were used for sampling. Internodes 2, 3, 5, 7 and 9 counting from the plant tip were collected. Total RNAs were extracted using tri-reagent according to the manufacturer’s protocol (Invitrogen). RNA was cleaned and concentrated using the RNeasy MinElute Cleanup kit (Qiagen, <a href="http://www.qiagen.com">http://www.qiagen.com</a>). Ten micrograms of purified RNA were used for microarray analysis. The microarray experiment included five internodes with three biological replicates, used 15 Affymetrix Genechips in total. Probe labeling, hybridization and scanning were conducted according to the manufacturer’s instructions (Affymetrix, <a href="http://www.affymetrix.com">http://www.affymetrix.com</a>). The normalization of data was achieved by the robust multi-chip average (RMA) procedure [ <a href="#CR51">51</a>]. The presence/absence call for each probe set was obtained from dCHIP [ <a href="#CR52">52</a>]. Genes significantly differentially expressed between controls and mutants were selected using Associative Analysis [ <a href="#CR31">31</a>]. The type-I family-wise error rate was reduced using a Bonferroni corrected <i>P</i>-value threshold of 0.05/n, where n represents the number of genes present on the chip. The false discovery rate was monitored and controlled by Q-value (false discovery rate), calculated using EDGE (extraction of differential gene expression; <a href="http://www.bioconductor.org/packages/release/bioc/html/edge.html">http://www.bioconductor.org/packages/release/bioc/html/edge.html</a>) [ <a href="#CR53">53</a>, <a href="#CR54">54</a>]. Hierarchical clustering analysis was conducted with Spotfire DecisionSite 8.1 (Spotfire Inc., <a href="http://spotfire.tibco.com/">http://spotfire.tibco.com/</a>). For clustering analysis, data from different internode were expressed as relative to the level of IN2 just prior constructing clusters using the Pearson correlation coefficient. </p> </div> <div class="real-timepcr" title="sec"> <div class="title" tagx="title" title="title">
Real-time PCR</div> <p>Real-time PCR analysis of 1045 transcription factor genes were carried out at the Genomics/Microarray core facility at the Samuel Roberts Noble Foundation. Quantitative real-time PCR (qRT-PCR) and the calculation of relative expression were performed as described previously [ <a href="#CR35">35</a>]. In brief, cDNA samples prepared from the aforementioned five internodes were used for real time RT-PCR with technical duplicates. The 10 μl reaction included 2 μl of primers (0.5 μM of each primer), 5 μl of Power Sybr (Applied Biosystems, <a href="http://www.appliedbiosystems.com/absite/us/en/home.html">http://www.appliedbiosystems.com/absite/us/en/home.html</a>), 2 μl 1:20 diluted cDNA from the reverse transcription step, and 1 μl of water. Real-time RT-PCR data were analyzed using SDS 2.2.1 (Applied Biosystems). The community high-throughput qRT-PCR uses a 384-well plate format with eight reference genes included on each plate. Information of primer sequences, accession numbers and stability test were described previously [ <a href="#CR37">37</a>]. Transcript levels were determined by relative quantification using the Medicago Ubiquitin gene (TC102473, primers: UbiFw, GCAGATAGACACGCTGGGA; UbiRe, AACTCTTGGGCAGGCAATAA) as a reference. Amplification efficiency (E) was determined from three biological replicates of each of the five internode samples using LinRegPCR [ <a href="#CR55">55</a>]. The relative expression of all internodes to internode 2 was calculated using the mean of three biological replicates for each organ. A TF gene was considered increased or decreased only if transcript levels for that gene were changed 5-fold or more than those of internode 2. </p> </div> <div class="geneco-expressionnetworkanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gene co-expression network analysis</div> <p>Expression data were collected from the Medicago Gene Expression Atlas [ <a href="#CR27">27</a>]. Data for leaf, petiole, stem, shoot (split-root exp) sufficient N <sub>2</sub>, flower, pod, root and split root (nodulating) sufficient N <sub>2</sub> and the present stem experiment (37 chips in total) were used. Only the probe sets with at least two-fold change in comparing the maximal value to the minimal value in stem experiments were selected. We also removed the probe sets "AFFX", "Sme" and "RPTR". Thus, 12,576 probe sets remained. To generate a gene co-expression gene network for those selected probe sets, we used our in-house R script to calculate the Pearson correlation coefficient (PCC) values for all 79,071,600 (=12576 × (12576-1)/2) probe set pairs in the above dataset. Two probe sets (genes) are considered linked if their observed correlation level (|PCC|) exceeds a significant threshold value. To estimate this threshold, we permutated the expression values for each probe set independently to generate a random dataset following an approach proposed by Carter et al. [ <a href="#CR56">56</a>]. We calculated PCCs for all probe set pairs in this random dataset, and obtained the distribution of PCC frequency from -1 to 1 at a 0.1 interval. From this distribution, we observed that no probe set pair that can reach |PCC| &amp;gt; 0.8 in the random dataset (Additional file <a href="#MOESM1">1</a>: Figure S2), indicating that 0.8 is a statistical significant level to determine whether there is a connection between each probe set pairs in the network. As a result, a co-expression gene network is produced for further analysis. The TFs that are connected with lignin, cellulose and hemicellulose biosynthetic genes are presented as three sub-networks in the results (Fig. <a href="#Fig6">6</a>). The visualization of gene networks is implemented by an open source software Cytoscape [ <a href="#CR57">57</a>]. </p> </div> <div class="cellimagingandhistochemicalstaining" title="sec"> <div class="title" tagx="title" title="title">
Cell imaging and histochemical staining</div> <p>To characterize the <i>Tnt1</i> insertional mutants, the sixth internodes counting from the top of the plant were collected from plants grown in the glasshouse and immediately frozen in liquid nitrogen. Cross-sections (100 μm) of the sixth internodes were cut with a Leica CM 1850 cryostat (Leica Microsystems Inc., Buffalo Grove, IL, USA) at-20 °C and prepared for microscopy as previously described [ <a href="#CR58">58</a>]. Phloroglucinol-HCl staining was carried out as previously described [ <a href="#CR25">25</a>]. Photographs were taken with a Nikon Micophot-FX system ( <a href="http://www.nikon.com">http://www.nikon.com</a>) with a Nikon DXM 1200 color camera with consistent settings. </p> </div> <div class="availabilityofsupportingdata" title="sec"> <div class="title" tagx="title" title="title">
Availability of supporting data</div> <p>The Microarray data sets supporting the results of this article is available at ArrayExpress with a accession number: E-MTAB-3909 </p> </div> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="additionalfile" title="sec"> <div class="title" tagx="title" title="title">
Additional file</div> <p> </p><div class="media" title="media"> <a href="12864_2015_2330_MOESM1_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p>Supplemental Figure 1. Expression pattern analysis of cluster 10 transcription factor genes. Supplemental Figure 2. The PCC distributions in the real and random datasets. Supplemental Table 1. Distribution of differentially expressed genes in the 10 clusters. Supplemental Table 2. Probe sets and corresponding names of lignin biosynthetic genes. Supplemental Table 3. Transcription factor genes upregulated during stem maturation. </p>  </div> <p /> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Competing interests</b> </p> <p>The authors declare that they have no competing interests.</p> </div> <div class="fn-type-" title=""> <p> <b>Authors’ contributions</b> </p> <p>HW, FC and RAD conceived the study. HW, JY, IT, QD, XC and JW carried out the experiments. HW, YT, MW and RAD analyzed the data. HW and RAD wrote the manuscript. All authors read and approved the final manuscript. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Michael K. Udvardi and Klementina Kakar for use of the primer sets of the transcription factors, and for providing information about the corresponding genes. This research was supported, in part, by the USDA National Institute of Food and Agriculture, Hatch project number CONS00925, and the UConn Research Excellence Program award (to HW); by the Oklahoma Center for the Advancement of Science and Technology, Oklahoma Department of Energy Bioenergy Center (OBC) (to RAD); and by the Samuel Roberts Noble Foundation. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ane</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frugoli</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Recent advances in Medicago truncatula genomics</span> <span class="source" tagx="source" title="source">Int J Plant Genomics</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">2008</span> <span class="fpage" tagx="fpage" title="fpage">256597</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2008/256597">10.1155/2008/256597</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18288239">18288239</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vance</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Legumes: importance and constraints to greater use</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">131</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">872</span> <span class="lpage" tagx="lpage" title="lpage">877</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.017004">10.1104/pp.017004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12644639">12644639</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouton</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The economic benefits of forage improvement in the United States</span> <span class="source" tagx="source" title="source">Euphytica</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">154</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">263</span> <span class="lpage" tagx="lpage" title="lpage">270</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10681-006-9220-6">10.1007/s10681-006-9220-6</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Downing</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timothy</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of new generation cooperatives in agriculture for renewable energy research, development, and demonstration projects </span> <span class="source" tagx="source" title="source">Biomass Bioenergy</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">425</span> <span class="lpage" tagx="lpage" title="lpage">434</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biombioe.2004.09.004">10.1016/j.biombioe.2004.09.004</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Rourke</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bolon</span> <span class="given-names" tagx="given-names" title="given-names">YT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bucciarelli</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vance</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Legume genomics: understanding biology through DNA and RNA sequencing</span> <span class="source" tagx="source" title="source">Ann Bot</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">113</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1107</span> <span class="lpage" tagx="lpage" title="lpage">1120</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/aob/mcu072">10.1093/aob/mcu072</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24769535">24769535</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">HK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mun</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baek</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mudge</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimating genome conservation between crop and model legume species</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="issue" tagx="issue" title="issue">43</span> <span class="fpage" tagx="fpage" title="fpage">15289</span> <span class="lpage" tagx="lpage" title="lpage">15294</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0402251101">10.1073/pnas.0402251101</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15489274">15489274</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tesfaye</span> <span class="given-names" tagx="given-names" title="given-names">MY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samuel</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamb</span> <span class="given-names" tagx="given-names" title="given-names">FSJA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">GH-J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samac</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vance</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Medicago truncatula as a model for dicot cell wall development</span> <span class="source" tagx="source" title="source">Bioenerg Res</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">59</span> <span class="lpage" tagx="lpage" title="lpage">76</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12155-009-9034-1">10.1007/s12155-009-9034-1</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debelle</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oldroyd</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geurts</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Udvardi</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Medicago genome provides insight into the evolution of rhizobial symbioses</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">480</span> <span class="issue" tagx="issue" title="issue">7378</span> <span class="fpage" tagx="fpage" title="fpage">520</span> <span class="lpage" tagx="lpage" title="lpage">524</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132">22089132</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickery</span> <span class="given-names" tagx="given-names" title="given-names">PJH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">J</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donald</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Satellite derived maps of pasture growth status: association of classification with botanical composition </span> <span class="source" tagx="source" title="source">Aust J Exp Agric</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">547</span> <span class="lpage" tagx="lpage" title="lpage">562</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1071/EA97014">10.1071/EA97014</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jense</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">H-N</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">How can increased use of biological N2 fixation in agriculture benefit the environment?</span> <span class="source" tagx="source" title="source">Plant Soil</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">252</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">177</span> <span class="lpage" tagx="lpage" title="lpage">186</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1024189029226">10.1023/A:1024189029226</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Lignin modification improves fermentable sugar yields for biofuel production</span> <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">759</span> <span class="lpage" tagx="lpage" title="lpage">761</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt1316">10.1038/nbt1316</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17572667">17572667</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RAR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasa</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of monolignols. Genomic and reverse genetic approaches</span> <span class="source" tagx="source" title="source">Phytochem Rev</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">289</span> <span class="lpage" tagx="lpage" title="lpage">306</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/B:PHYT.0000045486.50637.37">10.1023/B:PHYT.0000045486.50637.37</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reddy</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shadle</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aljoe</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Targeted down-regulation of cytochrome P450 enzymes for forage quality improvement in alfalfa (Medicago sativa L.) </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="issue" tagx="issue" title="issue">46</span> <span class="fpage" tagx="fpage" title="fpage">16573</span> <span class="lpage" tagx="lpage" title="lpage">16578</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0505749102">10.1073/pnas.0505749102</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16263933">16263933</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="collab'">collab: USDA-APHIS</span> </span> <span class="person-group'"> <span class="collab'">collab: Agriculture Do, vol. 79</span> </span> <span class="mixed-article-title" title="mixed-article-title">Monsanto company and Forage Genetics International; determination of nonregulatory status of genetically engineered alfalfa </span> <span class="source" tagx="source" title="source">Federal Register</span> <span class="year" tagx="year" title="year">2014</span> <span class="fpage" tagx="fpage" title="fpage">66686</span> <span class="lpage" tagx="lpage" title="lpage">66687</span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Carpita NC. Progress in the biological synthesis of the plant cell wall: new ideas for improving biomass for bioenergy. Curr Opin Biotechnol. 2011. </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulation of plant biomass production</span> <span class="source" tagx="source" title="source">Curr Opin Plant Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">299</span> <span class="lpage" tagx="lpage" title="lpage">304</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2010.03.002">10.1016/j.pbi.2010.03.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20381410">20381410</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Udagawa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishikubo</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horiguchi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcription switches for protoxylem and metaxylem vessel formation</span> <span class="source" tagx="source" title="source">Genes Dev</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">1855</span> <span class="lpage" tagx="lpage" title="lpage">1860</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1331305">10.1101/gad.1331305</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16103214">16103214</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitsuda</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iwase</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamamoto</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors, NST1 and NST3, are key regulators of the formation of secondary walls in woody tissues of Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">270</span> <span class="lpage" tagx="lpage" title="lpage">280</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.047043">10.1105/tpc.106.047043</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17237351">17237351</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Two NAC domain transcription factors, SND1 and NST1, function redundantly in regulation of secondary wall synthesis in fibers of Arabidopsis </span> <span class="source" tagx="source" title="source">Planta</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">225</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1603</span> <span class="lpage" tagx="lpage" title="lpage">1611</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-007-0498-y">10.1007/s00425-007-0498-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17333250">17333250</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MYB83 is a direct target of SND1 and acts redundantly with MYB46 in the regulation of secondary cell wall biosynthesis in Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">1950</span> <span class="lpage" tagx="lpage" title="lpage">1964</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcp139">10.1093/pcp/pcp139</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19808805">19808805</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richardson</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The MYB46 transcription factor is a direct target of SND1 and regulates secondary wall biosynthesis in Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">2776</span> <span class="lpage" tagx="lpage" title="lpage">2792</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.107.053678">10.1105/tpc.107.053678</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17890373">17890373</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A battery of transcription factors involved in the regulation of secondary cell wall biosynthesis in Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">2763</span> <span class="lpage" tagx="lpage" title="lpage">2782</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.108.061325">10.1105/tpc.108.061325</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18952777">18952777</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NAC domain function and transcriptional control of a secondary cell wall master switch</span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">68</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1104</span> <span class="lpage" tagx="lpage" title="lpage">1114</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2011.04764.x">10.1111/j.1365-313X.2011.04764.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21883551">21883551</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">QA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">YB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Syringyl lignin biosynthesis is directly regulated by a secondary cell wall master switch </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="issue" tagx="issue" title="issue">32</span> <span class="fpage" tagx="fpage" title="fpage">14496</span> <span class="lpage" tagx="lpage" title="lpage">14501</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1009170107">10.1073/pnas.1009170107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20660755">20660755</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inoue</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blount</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Downregulation of caffeic acid 3-O-methyltransferase and caffeoyl CoA 3-O-methyltransferase in transgenic alfalfa. impacts on lignin structure and implications for the biosynthesis of G and S lignin </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">73</span> <span class="lpage" tagx="lpage" title="lpage">88</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.13.1.73">10.1105/tpc.13.1.73</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11158530">11158530</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego-Giraldo</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An NAC transcription factor orchestrates multiple features of cell wall development in Medicago truncatula </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">100</span> <span class="lpage" tagx="lpage" title="lpage">114</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20408998">20408998</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benedito</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torres-Jerez</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murray</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andriankaja</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kakar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A gene expression atlas of the model legume Medicago truncatula</span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">504</span> <span class="lpage" tagx="lpage" title="lpage">513</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03519.x">10.1111/j.1365-313X.2008.03519.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18410479">18410479</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holmes</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goffard</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiller</span> <span class="given-names" tagx="given-names" title="given-names">GF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rolfe</span> <span class="given-names" tagx="given-names" title="given-names">BG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imin</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptional profiling of Medicago truncatula meristematic root cells</span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">21</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-8-21">10.1186/1471-2229-8-21</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18302802">18302802</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minic</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jamet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">San-Clemente</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pelletier</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renou</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rihouey</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptomic analysis of Arabidopsis developing stems: a close-up on cell wall genes</span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-9-6">10.1186/1471-2229-9-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19149885">19149885</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schnurr</span> <span class="given-names" tagx="given-names" title="given-names">JAJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hans-Joachim</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samac</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparative study of alfalfa and <i>Medicago truncatula</i> stem traits: morphology, chemical composition, and ruminal digestibility </span> <span class="source" tagx="source" title="source">Crop Sci</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">1672</span> <span class="lpage" tagx="lpage" title="lpage">1680</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2135/cropsci2006.12.0762">10.2135/cropsci2006.12.0762</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dozmorov</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Centola</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An associative analysis of gene expression array data</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">204</span> <span class="lpage" tagx="lpage" title="lpage">211</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/19.2.204">10.1093/bioinformatics/19.2.204</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12538240">12538240</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukuda</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptional regulation in wood formation</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">64</span> <span class="lpage" tagx="lpage" title="lpage">70</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2006.12.006">10.1016/j.tplants.2006.12.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17224301">17224301</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avci</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grant</span> <span class="given-names" tagx="given-names" title="given-names">EH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haigler</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beers</span> <span class="given-names" tagx="given-names" title="given-names">EP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">XND1, a member of the NAC domain family in Arabidopsis thaliana, negatively regulates lignocellulose synthesis and programmed cell death in xylem </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">425</span> <span class="lpage" tagx="lpage" title="lpage">436</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2007.03350.x">10.1111/j.1365-313X.2007.03350.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18069942">18069942</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">SND1, a NAC domain transcription factor, is a key regulator of secondary wall synthesis in fibers of Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">3158</span> <span class="lpage" tagx="lpage" title="lpage">3170</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.047399">10.1105/tpc.106.047399</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17114348">17114348</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avci</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mutation of WRKY transcription factors initiates pith secondary wall formation and increases stem biomass in dicotyledonous plants </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="issue" tagx="issue" title="issue">51</span> <span class="fpage" tagx="fpage" title="fpage">22338</span> <span class="lpage" tagx="lpage" title="lpage">22343</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1016436107">10.1073/pnas.1016436107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21135241">21135241</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">On-off switches for secondary cell wall biosynthesis</span> <span class="source" tagx="source" title="source">Mol Plant</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">297</span> <span class="lpage" tagx="lpage" title="lpage">303</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/mp/ssr098">10.1093/mp/ssr098</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22138968">22138968</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kakar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wandrey</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Czechowski</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaertner</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scheible</span> <span class="given-names" tagx="given-names" title="given-names">WR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stitt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A community resource for high-throughput quantitative RT-PCR analysis of transcription factor gene expression in Medicago truncatula </span> <span class="source" tagx="source" title="source">Plant Methods</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1746-4811-4-18">10.1186/1746-4811-4-18</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18611268">18611268</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cellulose biosynthesis and deposition in higher plants</span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">178</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">239</span> <span class="lpage" tagx="lpage" title="lpage">252</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2008.02385.x">10.1111/j.1469-8137.2008.02385.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18298430">18298430</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howells</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huttly</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickers</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interactions among three distinct CesA proteins essential for cellulose synthesis</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1450</span> <span class="lpage" tagx="lpage" title="lpage">1455</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0337628100">10.1073/pnas.0337628100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12538856">12538856</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laurie</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multiple cellulose synthase catalytic subunits are required for cellulose synthesis in Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2529</span> <span class="lpage" tagx="lpage" title="lpage">2540</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.12.12.2529">10.1105/tpc.12.12.2529</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11148295">11148295</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morrison</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> <span class="suffix" tagx="suffix" title="suffix">3rd</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freshour</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Expression of a mutant form of cellulose synthase AtCesA7 causes dominant negative effect on cellulose biosynthesis </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">132</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">786</span> <span class="lpage" tagx="lpage" title="lpage">795</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.102.019331">10.1104/pp.102.019331</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12805608">12805608</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steiner-Lange</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Unte</span> <span class="given-names" tagx="given-names" title="given-names">US</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eckstein</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">ZA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmelzer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Disruption of Arabidopsis thaliana MYB26 results in male sterility due to non-dehiscent anthers </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">519</span> <span class="lpage" tagx="lpage" title="lpage">528</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-313X.2003.01745.x">10.1046/j.1365-313X.2003.01745.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12753590">12753590</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conner</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vizcay Barrena</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">ZA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Arabidopsis MYB26/MALE STERILE35 regulates secondary thickening in the endothecium and is essential for anther dehiscence </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">534</span> <span class="lpage" tagx="lpage" title="lpage">548</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.046391">10.1105/tpc.106.046391</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17329564">17329564</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaffey</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cholewa</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Regan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sundberg</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Secondary xylem development in Arabidopsis: a model for wood formation</span> <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">114</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">594</span> <span class="lpage" tagx="lpage" title="lpage">600</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1034/j.1399-3054.2002.1140413.x">10.1034/j.1399-3054.2002.1140413.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11975734">11975734</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nieminen</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kauppinen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Helariutta</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A weed for wood? Arabidopsis as a genetic model for xylem development</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">135</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">653</span> <span class="lpage" tagx="lpage" title="lpage">659</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.104.040212">10.1104/pp.104.040212</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15208411">15208411</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">HK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tadege</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ratet</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Udvardi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An efficient reverse genetics platform in the model legume Medicago truncatula</span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">201</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1065</span> <span class="lpage" tagx="lpage" title="lpage">1076</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.12575">10.1111/nph.12575</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24206427">24206427</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tadege</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwak</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eschstruth</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Large-scale insertional mutagenesis using the Tnt1 retrotransposon in the model legume Medicago truncatula </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">335</span> <span class="lpage" tagx="lpage" title="lpage">347</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03418.x">10.1111/j.1365-313X.2008.03418.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18208518">18208518</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shadle</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Temple</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Distinct cinnamoyl CoA reductases involved in parallel routes to lignin in Medicago truncatula </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="issue" tagx="issue" title="issue">41</span> <span class="fpage" tagx="fpage" title="fpage">17803</span> <span class="lpage" tagx="lpage" title="lpage">17808</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1012900107">10.1073/pnas.1012900107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20876124">20876124</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carpita</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tierney</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular biology of the plant cell wall: searching for the genes that define structure, architecture and dynamics </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="issue" tagx="issue" title="issue">1-2</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">5</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1010603527077">10.1023/A:1010603527077</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11554466">11554466</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeef</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodacre</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of novel genes in Arabidopsis involved in secondary cell wall formation using expression profiling and reverse genetics </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">2281</span> <span class="lpage" tagx="lpage" title="lpage">2295</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.105.031542">10.1105/tpc.105.031542</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980264">15980264</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irizarry</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bolstad</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collin</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cope</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hobbs</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Speed</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Summaries of Affymetrix GeneChip probe level data</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gng015">10.1093/nar/gng015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12582260">12582260</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">31</span> <span class="lpage" tagx="lpage" title="lpage">36</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.98.1.31">10.1073/pnas.98.1.31</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11134512">11134512</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leek</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monsen</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dabney</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Storey</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">EDGE: extraction and analysis of differential gene expression</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">507</span> <span class="lpage" tagx="lpage" title="lpage">508</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btk005">10.1093/bioinformatics/btk005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16357033">16357033</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Storey</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tibshirani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Statistical significance for genomewide studies</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">9440</span> <span class="lpage" tagx="lpage" title="lpage">9445</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1530509100">10.1073/pnas.1530509100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12883005">12883005</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakers</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruijter</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deprez</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moorman</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data </span> <span class="source" tagx="source" title="source">Neurosci Lett</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">339</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">62</span> <span class="lpage" tagx="lpage" title="lpage">66</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0304-3940(02)01423-4">10.1016/S0304-3940(02)01423-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12618301">12618301</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carter</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brechbuhler</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bond</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene co-expression network topology provides a framework for molecular characterization of cellular state </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">14</span> <span class="fpage" tagx="fpage" title="fpage">2242</span> <span class="lpage" tagx="lpage" title="lpage">2250</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bth234">10.1093/bioinformatics/bth234</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15130938">15130938</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shannon</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Markiel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ozier</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baliga</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramage</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks </span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">2498</span> <span class="lpage" tagx="lpage" title="lpage">2504</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shadle</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multi-site genetic modification of monolignol biosynthesis in alfalfa (Medicago sativa): effects on lignin composition in specific cell types </span> <span class="source" tagx="source" title="source">New Phytol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">179</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">738</span> <span class="lpage" tagx="lpage" title="lpage">750</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2008.02502.x">10.1111/j.1469-8137.2008.02502.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18547377">18547377</a> </span> </span> </li> </ul> </div> </div>  </body></html>