<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Plant Biol</span> <span class="iso-abbrev" title="iso-abbrev">BMC Plant Biol</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Plant Biology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2229</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3502329">3502329</a> </span> <span class="publisher-id" title="publisher-id">1471-2229-12-183</span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23043463">23043463</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/1471-2229-12-183">10.1186/1471-2229-12-183</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
microRNAs targeting DEAD-box helicases are involved in salinity stress response in rice ( <i>Oryza sativa</i> L.) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macovei</span> <span class="given-names" tagx="given-names" title="given-names">Anca</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">anca.macovei@unipv.it</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">Narendra</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">narendra@icgeb.res.in</span> </span> </div> <span class="citation_author_institution" id="I1">[1], </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2012</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2012-10-10</span> </span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">183</span> <span class="lpage" tagx="lpage" title="lpage">183</span> <span class="history" title="history"> <span class="received" title="received">received: 2012-6-20</span> <span class="accepted" title="accepted">accepted: 2012-10-5</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Macovei and Tuteja; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2012</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="http://www.biomedcentral.com/1471-2229/12/183" title="self-uri">http://www.biomedcentral.com/1471-2229/12/183</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Rice ( <i>Oryza sativa</i> L.), one of the most important food crop in the world, is considered to be a salt-sensitive crop. Excess levels of salt adversely affect all the major metabolic activities, including cell wall damage, cytoplasmic lysis and genomic stability. In order to cope with salt stress, plants have evolved high degrees of developmental plasticity, including adaptation via cascades of molecular networks and changes in gene expression profiles. Posttranscriptional regulation, through the activity of microRNAs, also plays an important role in the plant response to salinity conditions. MicroRNAs are small endogenous RNAs that modulate gene expression and are involved in the most essential physiological processes, including plant development and adaptation to environmental changes. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>In the present study, we investigated the expression profiles of osa-MIR414, osa-MIR408 and osa-MIR164e along with their targeted genes, under salinity stress conditions in wild type and transgenic rice plants ectopically expressing the <i>PDH45</i> (Pea DNA Helicase) gene. The present miRNAs were predicted to target the <i>OsABP</i> (ATP-Binding Protein), <i>OsDSHCT</i> (DOB1/SK12/helY-like DEAD-box Helicase) and <i>OsDBH</i> (DEAD-Box Helicase) genes, included in the DEAD-box helicase family. An <i>in silico</i> characterization of the proteins was performed and the miRNAs predicted targets were validated by RLM-5′RACE. The qRT-PCR analysis showed that the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes were up-regulated in response to 100 and 200 mM NaCl treatments. The present study also highlighted an increased accumulation of the gene transcripts in wild type plants, with the exception of the <i>OsABP</i> mRNA which showed the highest level (15.1-fold change compared to control) in the transgenic plants treated with 200 mM NaCl. Salinity treatments also affected the expression of osa-MIR414, osa-MIR164e and osa-MIR408, found to be significantly down-regulated, although the changes in miRNA expression were limited. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Osa-MIR414, osa-MIR164e and osa-MIR408 were experimentally validated for the first time in plants as targeting the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes. Our data showed that that the genes were up-regulated and the miRNAs were down-regulated in relation to salt stress. The negative correlation between the miRNAs and their targets was proven. </p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
microRNAs</div> <div class="kwd" title="kwd">
Helicases</div> <div class="kwd" title="kwd">
Salinity stress</div> <div class="kwd" title="kwd">
Experimental validation</div> <div class="kwd" title="kwd">
Expression profiles</div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Salinity stress negatively affects the quality and quantity of crop production. Rice is considered to be a salt-sensitive crop, and excess salt levels adversely affect all the major metabolic activities, causing cell wall damage, cytoplasmic lysis or genomic instability [ <a href="#B1">1</a>]. In order to cope with salt stress, plants have evolved high degrees of developmental plasticity, including adaptation via cascades of molecular networks [ <a href="#B2">2</a>] and changes in gene expression profiles [ <a href="#B3">3</a>]. Post-transcriptional regulation also plays an important role in the plant response to salinity conditions [ <a href="#B4">4</a>]. </p> <p>The microRNAs (miRNAs) are a large family of endogenous, non-coding, small RNAs (21–24 nucleotides in length) that were demonstrated to play crucial roles in the modulation of gene expression [ <a href="#B5">5</a>]. Plant miRNAs originate from hairpin precursors (pre-miRNA) after two sequential cleavages performed by DICER-LIKE1 (DCL1). Mature miRNAs are then incorporated into the RISC complex (RNA-induced silencing complex) and negatively regulate the expression of specific mRNA targets through mRNA cleavage, decay or translational repression by base-pairing with its targets [ <a href="#B6">6</a>]. The first miRNA (lin-4) discovered in 1993 was reported to be involved in the regulation of <i>Caenorhabditis elegans</i> larval development [ <a href="#B7">7</a>]. Since then, an increasing interest has been devoted for the study of miRNAs both in animals and plants. After the first identification of miRNAs in <i>Arabidopis</i>[ <a href="#B8">8</a>], a great number of plant miRNAs have been identified by computational means [ <a href="#B9">9</a>]. Many miRNA genes are conserved in rice and <i>Arabidopsis</i>, indicating that their origin forfeit the divergence between monocotiledonate and dicotiledonate plants. In plants, miRNAs generally interact with their targets through near-perfect complementarity and leads to target mRNA cleavage. The high degree of sequence complementarity between miRNAs and their target mRNAs allowed the prediction of targets using algorithms that scan the genome for mRNA-miRNA complementarity [ <a href="#B10">10</a>]. However, the predicted targets must be experimentally validated in order to confirm their function. </p> <p>Plant miRNAs have been predicted or confirmed to regulate a wide variety of developmental and physiological processes, such as cell proliferation, plant development, signal transduction, stress response and pathogen invasion [ <a href="#B11">11</a>]. Recent findings have demonstrated that many plant miRNAs can be induced by biotic or abiotic stress and they may play an important role in the process of adaptation to adverse environmental conditions [ <a href="#B12">12</a>, <a href="#B13">13</a>]. For example, miR398 and miR408 were up-regulated in response to water deficit in <i>Medicago truncatula</i>[ <a href="#B12">12</a>], miR393 was found to be strongly up-regulated by cold, dehydration, NaCl and ABA treatments, while miR389a proven to be down-regulated in response to these treatments in <i>Arabidopsis</i> plants [ <a href="#B14">14</a>]. In <i>Vitis vinifera</i>, miR171e was demonstrated to be involved in the resistance to blight while the <i>Arabidopsis</i> miR399 was induced by low-phosphorus, and miR160 and miR167 are UV-B-responsive [ <a href="#B15">15</a>]. </p> <p>In rice, a set of miRNAs that putatively control different helicase genes were recently identified by <i>in silico</i> analysis [ <a href="#B16">16</a>]. Helicases are enzymes that catalize the separation of double-stranded nucleic acids in an energy-dependent manner. Among helicases, the DEAD-box family is by far the largest family and it is characterized by the presence of nine conserved motifs that were demonstrated to be involved in the ATPase and helicase activities [ <a href="#B17">17</a>]. Helicases are implicated in a wide range of processes such as recombination, replication and translation initiation, double-strand break repair, maintenance of telomere length, nucleotide excision repair, cell division and proliferation [ <a href="#B18">18</a>]. Recent reports indicate that the expression of genes encoding helicases is also regulated in response to changes in specific environmental conditions, including temperature, light, oxygen or osmolarity [ <a href="#B19">19</a>, <a href="#B20">20</a>]. For example, the ectopical expression of PDH45 (Pea DNA Helicase 45) helicase, a member of the DEAD-box protein family, resulted into salinity stress tolerance both in tobacco and rice plants [ <a href="#B21">21</a>- <a href="#B23">23</a>]. <i>PDH45</i> gene was shown to confer high salinity stress tolerance also in bacteria ( <i>Escherichia coli</i>, BL2 cells) [ <a href="#B24">24</a>]. </p> <p>In the present study, the expression profiles of osa-MIR414, osa-MIR408 and osa-MIR164e along with their targeted genes, were investigated in relation with the early response to salinity stress. The present miRNAs were predicted to target the putative helicases <i>OsABP</i> (ATP-Binding Protein), <i>OsDSHCT</i> (DOB1/SK12/helY-like DEAD-box Helicase) and <i>OsDBH</i> (DEAD-Box Helicase) included in the DEAD-box family. No information concerning the activity of these helicases are currently available in plants. An <i>in silico</i> characterization of the proteins was performed and the miRNAs predicted targets were validated by RLM-5′RACE. Wild type and transgenic rice plants ectopically expressing the <i>PDH45</i> gene were used in the present study in order to assess if there are differences between sensitive and tolerant rice plants to early salinity stress conditions. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialsandtreatments" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and treatments</div> <p>Rice ( <i>Oryza sativa</i> subsp <i>. indica</i>) cultivar IR64 was used in the present study. Transgenic rice plants ectopically expressing the <i>PDH45</i> gene, as well as the empty vector control were raised as previously described [ <a href="#B23">23</a>]. Seeds were sowed in pots containing a mixture of vermiculite, sand, and peat moss in 1:1:1 ratio and kept under greenhouse conditions (30/20°C day/night temperature and 12 h photoperiod with 75-80% relative humidity) for two weeks. In order to test the early response to salt stress, rice plants were dipped in 100 and 200 mM NaCl solution for 3 and 6 h, respectively. </p> </div> <div class="leafdiskassay" title="sec"> <div class="title" tagx="title" title="title">
Leaf disk assay</div> <p>Leaf disks were excised from wild type, empty vector control and transgenic plants ectopically expressing the <i>PDH45</i> gene, grown for two weeks in greenhouse and submitted to salinity stress treatments. The disks were floated in 5 ml solution of NaCl (100 and 200 mM) or water as control for 72 h and the chlorophyll contend was measured. The leaf disks were homogenized using a pestle and mortar in 3 mL of 80% acetone. The homogenate was centrifuged (1,957 <i>g</i>) and the resulting supernatant was used for chlorophyll estimation. Absorbance of the solution was measured at 645 and 663 nm, using an Ultraspec 2100 pro spectrophotometer (Amersham Biosciences). Chlorophyll <i>a</i>, chlorophyll <i>b</i> and total content were calculated according to the following formulas: C <sub>a</sub> = 12.72A <sub>663</sub> − 2.59A <sub>645</sub>, C <sub>b</sub> = 22.88A <sub>645</sub> − 4.67A <sub>663</sub>, C <sub>T</sub> = 20.29A <sub>645</sub> + 8.05A <sub>663</sub>[ <a href="#B25">25</a>]. </p> </div> <div class="prolineestimation" title="sec"> <div class="title" tagx="title" title="title">
Proline estimation</div> <p>Proline was determined according to previously described procedure [ <a href="#B26">26</a>]. Plant material (500 mg) from 14-days old untreated and treated plants were extracted with 3% aqueous 5-sulphosalicylic acid and filtered through Whatman No. 2 filter paper. Glacial acetic acid and acid ninhydrine was added to the resulting supernatant and heated at 100°C for 1h. After the reaction was terminated, toluene was added. The toluene layer was then separated and the red color intensity was measured at 520nm. The assays were done in triplicates using corrected weight calculated for the actual moisture content of tissue at each treatment [ <a href="#B27">27</a>]. Proline content was expressed as mg proline per g of fresh weight. </p> </div> <div class="rnaextractionandcdnasynthesis" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction and cDNA synthesis</div> <p>Total RNA was extracted from whole plant by using TRIZOL reagent (Invitrogen) according to manufacturer’s instructions. A total quantity of 100 mg plant tissue was used. The RNA samples were treated with RNase-free DNaseI (Promega) to eliminate DNA contamination. The RNA quantification was performed using the PicoGene Spectrophotometer (Genetix Biotech). The absorbance ratios of the RNA samples at 260/280 nm and 260/230 nm were between 1.9 and 2.0. The quality of RNA samples was verified on 1% agarose gel. cDNA synthesis was performed using two different strategies. In order to study the expression levels of <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes, cDNA was synthesized by using the AccuScript High Fidelity 1 <sup>st</sup> Strand cDNA Synthesis Kit (Aegilent Technologies, Stratagene), according to manufacturer’s instructions. As for the miRNAs expression level, cDNA was obtained by using the NCode™ VILO™ miRNA cDNA Synthesis Kit (Invitrogen) as instructed by the supplier. </p> </div> <div class="micrornavalidationthrough5′rnaligasemediatedrapidamplificationofcdnaends" title="sec"> <div class="title" tagx="title" title="title">
microRNA validation through 5′ RNA ligase mediated rapid amplification of cDNA ends</div> <p>In order to get the cleavage transcripts, a modified procedure for RNA ligase-mediated 5′ RACE was performed using the FirstChoice RLM-RACE Kit (Ambion, Invitrogen). Total RNA (1 μg) from two weeks old rice plants, was ligated to 5′ RACE adaptor without calf intestine alkaline phosphatase treatment. The gene-specific outer primers were then used for cDNA synthesis. Initial PCR was carried out using the 5′ RACE outer primer and gene-specific outer primer. Nested PCR was then carried out using 1/50 of the initial PCR reaction, with the 5′ RACE inner primer and gene specific inner primer. The sequences of the gene-specific primers are shown in Table <a href="#T1">1</a>. The amplified PCR products were cloned in the pCR II – TOPO vector (Invitrogen) by using the TOPO TA Cloning Kit for Sequencing (Invitrogen), according to the supplier’s instructions. Subsequently, the purified plasmids were sequenced by Eurofins Genomics, India. Three different experiments were performed for each combination. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Sequences of oligonucleotide primers</p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th> <b>Cleavage site mapping</b> <sup> <b>a</b> </sup> </th> </tr> </thead> <tbody> <tr> <td> <i>OsABP</i> <hr /> </td> <td>1 <sup>st</sup> PCR <hr /> </td> <td>5′-ATCAAAATCATATCCCCTCCGC-3′ <hr /> </td> <td>258 bp <hr /> </td> </tr> <tr> <td>Nested PCR <hr /> </td> <td>5′-TCCTCCCTCCATACGAAAATTCTG-3′ <hr /> </td> <td>144 bp <hr /> </td> </tr> <tr> <td> <i>OsDBH</i> <hr /> </td> <td>1 <sup>st</sup> PCR <hr /> </td> <td>5′-GAAAAGGGGAACAGTGCTAG-3′ <hr /> </td> <td>150 bp <hr /> </td> </tr> <tr> <td>Nested PCR <hr /> </td> <td>5′-AACCAGTTCAGCTTTGGCTGCT-3′ <hr /> </td> <td>100 bp <hr /> </td> </tr> <tr> <td> <i>OsDSHCT</i> <hr /> </td> <td>1 <sup>st</sup> PCR <hr /> </td> <td>5′-GGCAGGTGGCAGGAAGATTATA-3′ <hr /> </td> <td>490 bp <hr /> </td> </tr> <tr> <td>Nested PCR <hr /> </td> <td>5′-ACATCGCCGGTCATCAATCCGACAT-3′ <hr /> </td> <td>310 bp <hr /> </td> </tr> <tr> <td> <b>QRT-PCR for gene amplification</b> <hr /> </td> </tr> <tr> <td> <i>OsABP</i> <hr /> </td> <td>Forward <hr /> </td> <td>5′-GGGGGAGGATGGTGAGTAAT-3′ <hr /> </td> <td>211 bp <hr /> </td> </tr> <tr> <td>Reverse <hr /> </td> <td>5′-CCACCACCATCATCATCAAA-3′ <hr /> </td> </tr> <tr> <td> <i>OsDBH</i> <hr /> </td> <td>Forward <hr /> </td> <td>5′-ACGATGGCACCTCAGAGAAT-3′ <hr /> </td> <td>177 bp <hr /> </td> </tr> <tr> <td>Reverse <hr /> </td> <td>5′-TGGACCAAGTGACAGATGGA-3′ <hr /> </td> </tr> <tr> <td> <i>OsDSHCT</i> <hr /> </td> <td>Forward <hr /> </td> <td>5′-GAGGAGGTGGAGAACACGAG-3′ <hr /> </td> <td>237 bp <hr /> </td> </tr> <tr> <td>Reverse <hr /> </td> <td>5′-GTAGGCCTTGGCCATCTC-3′ <hr /> </td> </tr> <tr> <td> <i>α-Tubulin</i> <hr /> </td> <td>Forward <hr /> </td> <td>5′-GGTGGAGGTGATGATGCTTT-3′ <hr /> </td> <td>200 bp <hr /> </td> </tr> <tr> <td>Reverse <hr /> </td> <td>5′-ACCACGGGCAAAGTTGTTAG-3′ <hr /> </td> </tr> <tr> <td> <b>QRT-PCR for miRNA amplification</b> <sup> <b>b</b> </sup> <hr /> </td> </tr> <tr> <td>osa-MIR414 <hr /> </td> <td>Forward <hr /> </td> <td>5′-TCATCCTCATCATCATCGTCC-3′ <hr /> </td> <td>100 bp <hr /> </td> </tr> <tr> <td>osa-MIR164e <hr /> </td> <td>Forward <hr /> </td> <td>5′-GGGAGTTCTGTGATTGGAGAG-3′ <hr /> </td> <td>100 bp <hr /> </td> </tr> <tr> <td>osa-MIR408 <hr /> </td> <td>Forward <hr /> </td> <td>5′-TGGAGAAGCAGGGCACGTGA-3′ <hr /> </td> <td>100 bp <hr /> </td> </tr> <tr> <td>U6</td> <td>Forward <hr /> </td> <td>5′-CGATAAAATTGGAACGATACAGA-3′ <hr /> </td> <td>150 bp</td> </tr> <tr> <td>Reverse</td> <td>5′-ATTTGGACCATTTCTCGATTTGT-3′</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <sup>a</sup> Gene-specific reverse primers used in combination with the FirstChoise RLM-RACE Kit (Ambion, Invitrogen) forward and nested primers. </p> <p> <sup>b</sup> Specific reverse primers for miRNAs, used in combination with the Universal qPCR forward primer provided in the NCode VILO kit (Invitrogen). </p> </div> </div> </div> <div class="quantitativereal-timepcr" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR</div> <p>qRT-PCR reactions were performed in a 7500 Real Time PCR System apparatus (Applied Biosystems). For the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> gene expression, qRT-PCR primers were design by using the GeneScript Primer Design Program ( <a href="https://www.genscript.com/ssl-bin/app/primer">https://www.genscript.com/ssl-bin/app/primer</a>) (Table <a href="#T1">1</a>). The primer pairs were design to amplify a region containing the cleavage site of osa-miMI14, osa-MIR168e and osa-MIR408, respectively. Predicted fragment size ranged between 150 and 250 bp. The <i>α-Tubulin</i> gene was used as endogenous control. The homogeneity of <i>α-Tubulin</i> gene expression under normal and salinity stress conditions was previously tested [ <a href="#B28">28</a>]. Brilliant III Ultra-Fast SYBR Green QPCR Master Mix (Aegilent Technologies, Stratagene) was used for qRT-PCR reaction as indicated by the supplier. The amplification conditions were as follows: initial incubation step at 95°C for 10 min, denaturation step at 95°C for 15 s, annealing step at 57°C for 30 s and extension step at 72°C for 30 s, for 40 cycles. Fluorescence data was collected during the extension step and the specificity of qRT-PCR products was confirmed by performing a melting temperature analysis at temperatures ranging from 55°C to 95°C in intervals of 0.5°C. PCR fragments were run in a 2.5% agarose gel to confirm the existence of a unique band with the expected size. </p> <p>In the case of miRNAs expression profiles, the EXPRESS SYBR GreenER miRNA qRT-PCR Kit (Invitrogen) was used, according to the manufacturer’s instructions. To design the miRNA-specific forward primers, the qPCR Primer Design Program provided in the NCode miRNA Database ( <a href="http://escience.invitrogen.com/ncode">http://escience.invitrogen.com/ncode</a>), was used. The Universal qPCR Primer provided in the NCode VILO kit was used as a reverse primer in the qRT-PCR reaction. The small nuclear RNA U6 was used as reference control [ <a href="#B29">29</a>]. The U6 expression was uniform under normal and salinity stress conditions (data not shown). The primer sequences are shown in Table <a href="#T1">1</a>. The amplification conditions were as follows: UDG incubation step at 50°C for 2 min, initial incubation step at 95°C for 2 min, denaturation step at 95°C for 15 s, annealing step at 60°C for 1 min, for 40 cycles. The melting temperature analysis (60 - 95°C) was also performed. </p> <p>Quantification of gene expression was performed by using the 2 <sup>-ΔΔCt</sup> method [ <a href="#B30">30</a>]. All reactions were performed in triplicates and graphically represented as fold change to control. The results are shown by their mean ± standard deviation. </p> </div> <div class="bioinformaticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatic analysis</div> <p>The genomic sequences of <i>OsABP</i> (LOC_Os06g33520), <i>OsDBH</i> (LOC_Os04g40970) and <i>OsDSHCT</i> (LOC_Os11g07500) helicases were obtained from the Rice Genome Annotation Project funded by NSF ( <a href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>). The protein domain search was performed in the NCBI Conserved Domain Database (NCBI-CDD; <a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml">http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml</a>). Comparison of amino acid sequences was performed using the NCBI Blast Service ( <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). In order to determine the predicted miRNAs, the individual gene sequences were imported in psRNA Target: A Plant Small RNA Regulator Target Analysis Server ( <a href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>). </p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>Three replicated plants from each treatment combination were selected for analysis. Results were subjected to Analysis of Variance (ANOVA) and the means were compared by Tukey’s test, which provides confidence intervals (p-values) as test results for evaluating whether the means of data sets are different. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="salinitystressinducesmodificationsinchlorophyllcontentandprolinelevelsinriceplants" title="sec"> <div class="title" tagx="title" title="title">
Salinity stress induces modifications in chlorophyll content and proline levels in rice plants </div> <p>In order to study the salt stress effects on wild type (WT), empty vector control (EV) and transgenic rice plants ectopically expressing the <i>PDH45</i> gene (TR), two different NaCl concentrations (100 mM and 200 mM) were used. The rice plants were grown under greenhouse conditions for two weeks and subsequently submitted to salt stress treatments. When leaf discs were floated separately on 100 and 200 mM NaCl for 72 h, the damage caused by salt stress was evident from the degree of bleaching observed in the leaf tissues (Additional file <a href="#S1">1</a>). Chlorophyll estimation was performed to confirm the leaf disc assay results. When the 100 mM NaCl solution was used, reduction in total chlorophyll content in the WT (2.1-fold), EV (2.4-fold) and TR (1.6-fold) plants, compared to the non-treated controls, was observed (Figure <a href="#F1">1</a>A). As for the treatment with 200 mM NaCl, the decrease in total chlorophyll content was even higher, resulting in 3.7-fold reduction in the case of WT plants, 4.1-fold for EV and 2.1-fold for TR plants. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p> <b>Total chlorophyll (A) and free proline level (B) in two weeks old rice plantlets grown in greenhouse in absence/presence of salinity stress (100 and 200 mM NaCl). </b> WT, wild type; EV, emply vector control; TR, transgenic rice plants ectopically expressing the <i>PDH45</i> gene; NT, non-treated control. </p>   </div> <p>The free proline content was also quantified, considering that this compound is a well-known osmoprotectant. The estimated free proline content in the untreated control samples was 62.4 ± 3.6 mg/g FW in WT, 61.1 ± 2.5 mg/g FW in EV and 66.4 ± 5.7 mg/g FW in TR plants. At 3 h after treatment, no significant changes in proline levels were observed (data not shown). The exposure of rice plants to salt stress for 6 h resulted into a significant ( <i>p</i> &amp;lt; 0.001) accumulation in free proline since the 100 mM NaCl solution induced a 1.2-fold, 1.3-fold and 1.5-fold increase in the WT, EV and TR plants, compared to the untreated control (Figure <a href="#F1">1</a>B). A further increase in the amount of free proline was observed using the 200 mM NaCl treatments. The estimated free proline content was 137.5 ± 11.0 mg/g FW (2.1-fold compared to control) in WT, 140.6 ± 9.6 mg/g FW (2.3-fold compared to control) in EV and 159.4 ± 16.9 mg/g FW (2.4-fold compared to control) in TR plants. </p> <p>The reported data show that the loss of chlorophyll induced by salinity stress was lower in TR rice plants compared to WT plants while the free proline content was slightly higher in the transgenic lines compared with the wild type plants. </p> </div> <div class="predictionofosa-mir414,osa-mir408andosa-mir164etargetsandexperimentalvalidation" title="sec"> <div class="title" tagx="title" title="title">
Prediction of osa-MIR414, osa-MIR408 and osa-MIR164e targets and experimental validation</div> <p>The computational miRNA target searches were performed on the psRNA Target Analysis Server ( <a href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>). From the bioinformatic search it was evident that the osa-MIR414, osa-MIR408 and osa-MIR164e possess multiple predicted targets. The miRNAs with their sequences, predicted targets and accession numbers are presented in Table <a href="#T2">2</a>. Umate and Tuteja (2010) [ <a href="#B16">16</a>] predicted that osa-MIR414 targeted six different helicases: the ATP-dependent RNA helicase DDX46 and DDX47, the Pre-mRNA-processing ATP-dependent RNA helicase PRP5, the RNA helicase A (RNAhA), an ATP-dependent helicase required to maintain repression (CTD2) and the ATP-Binding Protein (OsABP). The only predicted target that showed 100% sequence complementarity with osa-MIR414 was OsABP. The predicted targets for osa-MIR408 were DSHCT (DOB1/SK12/helY-like DEAD-box Helicase), MIP1 (Major Intrinsic Protein 1) and a basic blue copper protein (plastocyanin-like). The osa-MIR164e was predicted to target two different mRNAs: a DEAD-Box Helicase (DBH) and a member of the NAC (No Apical Meristem) gene family. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Prediction of conserved miRNA targets and experimental validation by 5′-RACE</p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th> <b>miRNA</b> </th> <th> <b>miRNA sequence</b> </th> <th> <b>Predicted targets</b> </th> <th> <b>Accession no</b> </th> <th> <b>5′-RACE validation</b> </th> </tr> </thead> <tbody> <tr> <td> <b>osa-MIR414</b> <hr /> </td> <td>3′- cugcuacuacuacuccuacu – 5′ <hr /> </td> <td>ABP <hr /> </td> <td>LOC_Os06g33520 <hr /> </td> <td>Yes <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>DDX46 <hr /> </td> <td>LOC_Os08g05810 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>DDX47 <hr /> </td> <td>LOC_Os03g46610 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>PRP5 <hr /> </td> <td>LOC_Os08g06344 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>RNAhA <hr /> </td> <td>LOC_Os01g56190 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>CTD2 <hr /> </td> <td>LOC_Os02g43460 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <b>osa-MIR408</b> <hr /> </td> <td>3′ - cggucccuucuccgucacguc – 5′ <hr /> </td> <td>DSHCT <hr /> </td> <td>LOC_Os11g07500 <hr /> </td> <td>Yes <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>MIP1 <hr /> </td> <td>LOC_Os10g41041 <hr /> </td> <td>n.d <hr /> </td> </tr> <tr> <td> <hr /> </td> <td> <hr /> </td> <td>Plastocyanin <hr /> </td> <td>LOC_Os02g43660 <hr /> </td> <td>Yes [ <a href="#B12">12</a>, <a href="#B31">31</a>] <hr /> </td> </tr> <tr> <td> <b>osa-MIR164e</b> <hr /> </td> <td>3′ - agugcacgggacgaagaggu – 5′ <hr /> </td> <td>DBH <hr /> </td> <td>LOC_Os04g40970 <hr /> </td> <td>Yes <hr /> </td> </tr> <tr> <td> </td> <td> </td> <td>NAC</td> <td>LOC_Os06g23650</td> <td>Yes [ <a href="#B32">32</a>] </td> </tr> </tbody> </table> </div> <p>To determine if the miRNAs can direct cleavage of their predicted helicase targets, a modified 5′ rapid amplification of cDNA ends (5′-RACE) protocol was performed. This technique has been used to validate miRNA targets as cleavage occurs opposite to around the tenth position from the 5′-end of the miRNA [ <a href="#B31">31</a>]. The results presented in Figure <a href="#F2">2</a> indicate that the 5′-RACE analysis could clearly identify the miRNA-mRNA cleavage for the osa-MIR414 target OsABP, the osa-MIR408 target OsDSHCT and the osa-MIR164e target OsDBH in the canonical 10 <sup>th</sup>-11 <sup>th</sup> positions. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <b>Target validation of conserved </b> <b> <i>Oryza sativa </i> </b> <b>miRNAs.</b> Experimental validation of miRNA-mRNA cleavage site was performed using a modified 5′RACE assay. Each miRNA is shown base-paired to its predicted target mRNA. The sequences of miRNAs are shown in 3′ to 5′ direction. The two points indicate a Watson-Crick base-pair, one point indicates a Wooble base-pair and the space indicates a mismatch. The arrow above the target mRNA indicates the 5′ end of cleavage products. Three different experiments were performed for each combination. </p>   </div> </div> <div class="osabp,osdbhandosdshctareputativelydead-boxhelicases" title="sec"> <div class="title" tagx="title" title="title">
OsABP, OsDBH and OsDSHCT are putatively DEAD-box helicases</div> <p>No information on <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> are currently available. Their helicase activity was not yet validated. The only information available come form bioinformatic approaches. The OsABP (LOC_Os06g33520), OsDBH (LOC_Os04g40970) and OsDSHCT (LOC_Os11g07500) nucleotide and amino acid sequences were obtained from the Rice Genome Annotation Project funded by NSF. The <i>OsABP</i> gene contains an open reading frame (ORF) of 2772 nt, encoding a protein of 923 aa, <i>OsDBH</i> possesses an ORF of 2781 nt that encodes for a protein of 926 aa, while <i>OsDSHCT</i> is characterised by an ORF (3012 nt) encoding a protein of 1003 aa. The search for protein domains, carried out using the CDD software from NCBI, revealed the presence of the DEAD and Helicase C-terminal domains in all the three sequences (Additional file <a href="#S2">2</a>). The DEAD domain, specific to the DEAD-like helicase superfamily, contains several ATP-binding sites and it is involved in ATP-dependent RNA or DNA unwinding. The helicase superfamily C-terminal domain is associated with DEXD-, DEAD- and DEAH-box proteins. This domain is found in a wide variety of helicases and helicase related proteins and it is characterized by the presence of a P-loop containing Nucleoside Triphosphate Hydrolases. Members of the P-loop NTPase domain superfamily are defined by a conserved nucleotide phosphate-binding motif, also referred as Walker A and Walker B motif that bind the beta-gamma phosphate moiety of the bound nucleotides and the Mg <sup>2+</sup> cation [ <a href="#B33">33</a>]. </p> <p>Additionally, the OsDSHCT protein included a RNA-processing arch domain and the DUF1181/NUC185 domain (Additional file <a href="#S2">2</a>). The RNA-processing arch domain, specific to DOB1 and SKI2 helicases, is required for the proper processing of 5.8S rRNA and it appears to function independently of canonical helicase activity [ <a href="#B34">34</a>]. The DUF1181/NUC185 domain is also found in DOB1/SK12/helY-like DEAD box helicases, involved in 3′ end formation of rRNA and mRNA transport [ <a href="#B35">35</a>]. </p> <p>Comparison of amino acid sequences was performed using the NCBI BlastP Service (Additional file <a href="#S2">2</a>). The rice OsABP protein shared 48% similarity with the <i>A. thaliana</i> DEAD-box ATP-dependent RNA helicase 31 (NP_201168) and approximately 50% similarity with the <i>Glicyne max</i> (XP_003548422) and <i>Vitis vinifera</i> (XP_002277120) proteins. The OsDBH protein revealed 50% similarity with the ATP-dependent DNA helicase Q-like 5-like form <i>A. thaliana</i> (NP_174109) and 53% similarity with the proteins from <i>G. max</i> (XP_003532718) and <i>V. vinifera</i> (XP_002266225). As for the OsDSHCT protein, 75% and 80% similarity to the <i>A. thaliana</i> (NP_565338) and <i>V. vinifera</i> (CBI24057) RNA helicase DOB1/SKI2 proteins was observed. </p> </div> <div class="expressionprofilesofosabp,osdbhandosdshctgenesinresponsetoearlysalinitystress" title="sec"> <div class="title" tagx="title" title="title">
Expression profiles of <i>OsABP</i>, Os <i>DBH</i> and Os <i>DSHCT</i> genes in response to early salinity stress </div> <p>The expression profiles of <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes were evaluated in two weeks-old rice plantlets subjected to 100 and 200 mM NaCl treatments in order to determine the early response (3 and 6 h) to salt stress conditions. The investigation was carried out using the wild type rice cultivar IR64 (WT) and the empty vector control (EV) that were compared with transgenic plants ectopically expressing the PDH45 helicase (TR). The TR plants, previously demonstrated to be tolerant to salinity stress, were used to estimate the potential differences between the mRNA levels of the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes. The results of the qRT-PCR analysis are shown in Figure <a href="#F3">3</a>. All the tested genes were significantly ( <i>p</i> &amp;lt; 0.001) up-regulated in response to treatments. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <b>Expression levels of </b> <b> <i>OsABP </i> </b> <b>(A), </b> <b> <i>OsDBH </i> </b> <b>(B) and </b> <b> <i>OsDSHCT </i> </b> <b>(C) genes in wild type (WT), empty vector control (EV) and transgenic rice plants ectopically expressing </b> <b> <i>PDH45 </i> </b> <b>gene (TR), grown in greenhouse and challenged with 100 and 200 mM NaCl for 3 and 6 h, respectively. </b> The relative expression is presented as fold change to control. Values are expressed as means ± SD of three independent replicated plants; Tuckey test <i>p</i> &amp;gt; 0.001 between treatments; a, no statistical differences between the samples inside each treatment; NT, non-treated control. </p>   </div> <p>The <i>OsABP</i> gene was highly up-regulated at 3 h after treatment when 100 mM NaCl was used. The WT and EV plants showed 8.2- and 7.7-fold up-regulation, while in the TR plants the observed up-regulation of the <i>OsABP</i> gene was only 2.2-fold. At 6 h after treatment the level of <i>OsABP</i> mRNA level decreased in the TR plants, compared to controls. When the 200 mM NaCl concentration was used, a change in the expression profile was observed. At 3 h after treatment only a slight increase (2.4-, 2.5- and 2.7-fold, respectively) in the <i>OsABP</i> mRNA levels was detected, while at 6 h the highest expression level (15.1-fold change compared to control) was evidenced in TR plants (Figure <a href="#F3">3</a>A). </p> <p>When the <i>OsDBH</i> gene was analysed, the 100 mM NaCl treatment induced a slight increase in the transcript levels at 3 h following treatment was detected in all the tested samples while a consistent accumulation was evident in WT and EV plants at 6 h (3.9- and 2.9-fold, respectively). The TR plants revealed only a 0.5-fold increase in the <i>OsDBH</i> transcript level. When the 200 mM NaCl solution was used, the highest transcript accumulation was observed at 3 h after treatment in WT and EV plants (5.4- and 6.6-fold, respectively), while after 6 h the <i>OsDBH</i> transcript levels decreased (1.2- and 1.3-fold, respectively). The TR plants showed similar expression patterns (1.4- and 1.7-fold up-regulation, respectively) at both time points (Figure <a href="#F3">3</a>B). </p> <p>As for the expression levels of the <i>OsDSHCT</i> gene, the 100 mM NaCl treatment induced a 2.3-fold increase in the transcript level in WT plants at 3 h after treatment and only 0.9-fold increase in TR plants. At 6 h after treatment, the amount of <i>OsDSHCT</i> mRNA decreased in WT plants (1.5-fold) and slightly increased in the TR plants (1.1-fold). When the 200 mM NaCl solution was used, only a slight accumulation in the transcript level was observed both at 3 h (0.7-, 0.7- and 0.9-fold) and 6 h (1.7-, 1.3- and 1.6-fold) in all the tested samples (Figure <a href="#F3">3</a>C). </p> <p>The reported data show enhanced accumulation of the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> gene transcripts in WT plants, with the exception of the <i>OsABP</i> mRNA that peaked in TR plants treated with 200 mM NaCl. </p> </div> <div class="expressionanalysisofosa-mir414,osa-mir408andosa-mir164eundersalinitystress" title="sec"> <div class="title" tagx="title" title="title">
Expression analysis of osa-MIR414, osa-MIR408 and osa-MIR164e under salinity stress</div> <p>Salinity treatments also affected the expression of the miRNAs in rice plants. The results of the qRT-PCR analysis are shown in Figure <a href="#F4">4</a>. Osa-MIR414, osa-MIR164e and osa-MIR408 were significantly ( <i>p</i> &amp;lt; 0.05) down-regulated in response to salinity stress. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p> <b>Expression levels of osa-MIR414 (A), osa-MIR164e (B) and osa-MIR408 (C) in wild type (WT), empty vector control (EV) and transgenic rice plants ectopically expressing </b> <b> <i>PDH45 </i> </b> <b>gene (TR), grown in greenhouse and challenged with 100 and 200 mM NaCl for 3 and 6 h, respectivelly. </b> The relative expression is presented as fold change to control. Values are expressed as means ± SD of three independent replicated plants; Tuckey test <i>p</i> &amp;gt; 0.05 between treatments; a, no statistical differences between the samples inside each treatment; NT, non-treated control. </p>   </div> <p>The expression of osa-MIR414 significantly decreased (0.8-fold) in WT and EV plants at 3 h after treatment with 100 mM NaCl while in TR plants, only a 0.3-fold decrease was observed under the same conditions. At 6 h after treatments, a 0.4- and 0.3-fold decrease in the amount of osa-MIR414 was observed in WT and TR plants, respectively. Exposure to 200 mM NaCl caused a decrease of approximately 0.6- and 0.3-fold in the level of osa-MIR414 transcript after 3 and 6 h, respectively, while in the TR plants the observed reduction was around 0.8-fold (Figure <a href="#F4">4</a>A). </p> <p>In the case of osa-MIR164e expression, the use of the 100 mM NaCl solution resulted into 0.5-fold decrease at 3 h after treatment and 0.6-fold decrease at 6 h after treatment. In the TR plants the estimated decrease of osa-MIR164e transcript was approximately 0.3-fold. When 200 mM NaCl treatment was used, a 0.7-fold decrease was measured in WT plants at 3 h after treatment, while at 6 h after treatment the WT, ER and TR plants showed a similar expression level, with a 0.4-fold reduction compared to the control (Figure <a href="#F4">4</a>B). </p> <p>As for the osa-MIR408 expression, the amount of mRNA decreased (approximately 0.9-fold) in WT, EV and TR plants at 3 h after treatment with both the 100 and 200 mM NaCl solutions. In this case the transcript levels were similar in WT, EV and TR plants also at 6 h after treatment, when a 0.5- and 0.7-fold reduction was evidenced in response to 100 and 200 mM NaCl, respectively (Figure <a href="#F4">4</a>C). </p> <p>Although salinity treatment affected the expression of all tested miRNAs, the reported changes in miRNA levels were small. </p> </div> </div> <div class="discussions" title="sec"> <div class="title" tagx="title" title="title">
Discussions</div> <p>In the present study, transgenic rice plants ectopically expressing the <i>PDH45</i> gene (TR), were tested under early-induced salinity stress conditions and compared with the wild type rice genotype IR64 (WT) and the empty vector control (EV). The TR plants are resistant to salinity stress and represent a valid system for testing the genes and miRNAs expression patters in comparison with the WT plants. The short-term response to salinity stress may be particularly relevant to better understand the biological significance of stress status in plant responses to salinity. </p> <p>A preliminary characterization of the stress response in rice was carried out by measuring the chlorophyll content and the free proline levels. The damage caused by salt stress was evident from the degree of bleaching observed in the leaf tissues, along with a decrease in chlorophyll content. However, the reduction in chlorophyll content was significantly higher in WT plants, compared with TR plants. Since the goal of this study was to test the plant response to early-induced salinity stress, the free proline levels were measured at 3 and 6 h post-treatment. Significant increase in free proline was observed in response to both NaCl concentrations (100 and 200 mM) at 6 h after treatment. The TR plants were able to accumulate higher amounts of free proline compared with the WT. This finding is in agreement with the recent study on rice plants ectopically expressing the <i>PDH45</i> gene, where the authors demonstrated that the plants were tolerant to salinity stress [ <a href="#B22">22</a>, <a href="#B23">23</a>]. </p> <p>The role of miRNAs as important players in the plant stress response has been highlighted in recent years, opening new intriguing perspectives for crop improvement. </p> <p>The main goal of the present work was to assess the role of three different rice miRNAs, osa-MIR414, osa-MIR164e and osa-MIR408, as regulators of genes encoding DEAD-box helicases in response to salinity stress. The miRNA hereby investigated have been described in a previous work [ <a href="#B16">16</a>] where an <i>in silico</i> analysis was used to identify a set of miRNAs involved in the regulation of rice helicases expression. A modified RLM -5′RACE protocol was used to validate osa-MIR414, osa-MIR164e, and osa-MIR408 predicted targets and our results clearly demonstrate that osa-MIR414 targets the OsABP mRNA, osa-MIR408 targets OsDSHCT and osa-MIR164e targets the OsDBH mRNA sequence, cleaving them in the canonical 10 <sup>th</sup>-11 <sup>th</sup> positions. In plants, there are only few reports concerning miRNAs that target specific helicase genes. In a recent study it demonstrated that the DCL1 helicase domain, targeted by miR168, is responsible for the ATP-dependence of miRNA processing, suggesting that this helicase domain is critical for miRNA biogenesis [ <a href="#B36">36</a>]. Besides the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> DEAD-box helicase genes, other predicted targets of the above cited miRNAs have been reported, as shown in Table <a href="#T2">2</a>. Some of these targets were already validated while others still need to be studied. Out of the six helicase that were predicted to be targets of osa-MIR414, only the OsABP protein was experimentally validated in this study. In the case of osa-MIR408, the predicted target corresponding to a plastocyanin-like protein has been previously validated in <i>Arabidposis</i>[ <a href="#B31">31</a>] and <i>Medicago truncatula</i>[ <a href="#B12">12</a>]. Other miR408 targets include some members of Major Intrinsic Proteins (MIPs) involved in plant osmoregulation, drought resistance, salt tolerance and seed germination [ <a href="#B37">37</a>]. In this case, the computational predicted target was not yet validated. As for osa-MIR164, the subset of the <i>Arabidopsis</i> NAC-domain transcription factor genes that include the CUP-SHAPED COTYLEDON ( <i>CUC1</i>, <i>CUC2</i> and <i>CUC3</i>) genes were already validate as targets [ <a href="#B32">32</a>]. The NAC-domain genes are required for organ separation. When miR164 was ectopically expressed in <i>Arabidopsis</i> plants, it resulted into floral organ fusion and cotyledon fusion [ <a href="#B38">38</a>]. </p> <p>No information regarding the activity of OsABP, OsDBH and OsDSHCT helicases are currently available in plants. In consequence, a thorough bioinformatic investigation was carried out to analyze and compare in detail the structural features of the OsABP, OsDBH and OsDSHCT proteins. The three proteins harbour the DEAD domain and the helicase C-terminal domain, typically conserved motifs found in helicase superfamily. The similarity search showed that OsABP and OsDBH were close to RNA helicase 31 and DNA helicase RecQl5, respectively, while the OsDSHCT revealed high similarity with the SKI2/DOB1 protein. RNA helicase 31 is a member of the DEAD-box family, possessing ATP-dependent helicase activity and RNA-binding property [ <a href="#B33">33</a>]. The RecQ helicases have been shown to play important roles in DNA repair, recombination and replication, consistent with their intrinsic DNA helicase activities [ <a href="#B39">39</a>]. Recently it was proved that the RecQl5 helicase promotes genome stabilization through interaction with RNA polymerase II [ <a href="#B40">40</a>]. The SKI/DOB1 proteins are essential ATP-dependent RNA helicases that play crucial roles in ribosome synthesis and the degradation of cytoplasmic mRNAs following poly (A) [ <a href="#B41">41</a>]. A recent report on OsABP revealed that the gene is up-regulated in response e to multiple abiotic stress treatments including NaCl, dehydration, ABA, blue and red light [ <a href="#B42">42</a>]. Although a large number of DEAD box proteins were identified by <i>in silico</i> analysis as putative predicted helicases, the confirmation of helicase activity was demonstrated only for few of them [ <a href="#B33">33</a>]. In plants, two DEAD-box-related helicases termed PDH47 and PDH45 were proven to be induced by a variety of abiotic stresses including salinity, cold, heat and ABA treatments, suggesting that they are components of a general stress response mechanism [ <a href="#B21">21</a>- <a href="#B23">23</a>, <a href="#B43">43</a>]. Recently, a single subunit of minichromosome maintenance 6 (MCM6) from pea was reported to function as DNA helicase and to promote salinity stress tolerance without affecting yield [ <a href="#B44">44</a>- <a href="#B46">46</a>]. Nonetheless the few references, genes encoding helicases were proven to be induced in response to salinity stress [ <a href="#B47">47</a>]. </p> <p>The stress-responsive gene expression results in metabolites accumulation, alteration of biochemical and physiological pathways, vital for the plant adaptation to stress conditions [ <a href="#B14">14</a>]. In the present study, qRT-PCR analysis showed that the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes are up-regulated in response to salinity stress in both WT and TR plants. However, a higher accumulation of the gene transcripts was observed in WT plants, except for the <i>OsABP</i> transcript that showed the highest level in TR plants treated with 200 mM NaCl. The differences observed between the experimental conditions and the control might be due to changes in transcriptional activity. However, some differences might result from the action of miRNAs [ <a href="#B48">48</a>]. The effects of miRNA directed mRNA degradation can be detectable through changes in the expression profiles of miRNA targeted genes. Our results shown that osa-MIR414, osa-MIR164e and osa-MIR408 were down-regulated in WT and TR rice plants challenged with salinity stress. Considering the fact that the targeted helicases were up-regulated and the miRNAs were down-regulated in relation to early-induced salt stress, we can state that the negative correlation between these miRNAs and their validated targets is certified. The levels of accumulation or decrease in the transcript levels is most visible in the case of TR plants, where the 0.9-fold decrease in the osa-MIR414 expression resulted into 15.1-fold up-regulation of <i>OsABP</i> gene. </p> <p>Several recent papers have described the miRNAs involvement in the response to various stress conditions [ <a href="#B49">49</a>]. miR408 was shown to accumulate at low levels in plants under standard conditions and its accumulation was highly induced by drought and heavy-metals [ <a href="#B31">31</a>, <a href="#B32">32</a>]. Since miRNAs can be targeted by more than one gene, a recent study demonstrated that AGO1 and AGO2 could act redundantly in the miR408-mediated plastocyanin regulation [ <a href="#B50">50</a>]. The <i>Arabidopsis</i> miR164 family comprises three members, miR164a, miR164b and miR164c, which negatively regulate several genes that encode NAC-like transcription factors [ <a href="#B32">32</a>]. In the present study, osa-MIR164e was identified and its target, a DEAD-box helicase, validated. Both osa-MIR408 and osa-MIR164e are responsive to early salinity stress conditions. miR164 and miR408 were proven to be responsive also to cold and drought conditions in Euphorbiaceae plants [ <a href="#B10">10</a>]. Several miRNAs that displayed different activities during salt stress were recently identified in <i>Zea mais</i>[ <a href="#B51">51</a>]. The authors shown that miR164 was down-regulated after 24 h of salt shock, while its target NAC1 (an early auxin-responsive gene) was up-regulated. A similar study showed that miR156, miR166, miR171, miR172, miR319, miR164 along with their target genes, were differentially expressed in stress-tolerant maize hybrids compared with stress-sensitive lines [ <a href="#B52">52</a>]. As for osa-MIR414, no other information was available in the literature. It was predicted to target six different helicases in rice plants [ <a href="#B16">16</a>], we validated the <i>OsABP</i> gene as one of its targets and shown the changes in their expression profiles under early salt stress conditions. However, further analysis is still needed in order to better characterize the miRNAs involvement in salinity stress as well as the function of their validated targets. </p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Osa-MIR414, osa-MIR164e and osa-MIR408 were experimentally validated for the first time in plants as targeting the <i>OsABP</i>, <i>OsDBH</i> and <i>OsDSHCT</i> genes. OsABP, OsDBH and OsDSHCT represent newly identified DEAD-box helicases in rice. Their function still needs to be experimentally validated. The expression profiles of the miRNAs and their validated targets were tested under early-induced salinity stress conditions both in wild type and transgenic rice plants ectopically expressing the PDH45 helicase. Our data showed that that the genes were up-regulated and the miRNAs were down-regulated in relation to salt stress. The negative correlation between the miRNAs and their targets was proven. The present results also highlights higher accumulation of the gene transcripts in WT plants when compared with TR plant, correlated with decrease in the miRNAs expression. </p> </div> <div class="abbreviations" title="sec"> <div class="title" tagx="title" title="title">
Abbreviations</div> <p>miRNA: microRNA; RLM-5′RACE: RNA ligase-mediated 5′ rapid amplification of cDNA ends; qRT-PCR: quantitative Real-Time Polymerase Chain Reaction; OsABP: ATP-Binding Protein; OsDBH: DEAD-Box Helicase; OsDSHCT: DOB1/SK12/helY-like DEAD-box Helicase; PDH45: Pea DNA Helicase 45; NaCl: Sodium chloride; WT: Wild type; EV: Empty vector control; TR: Transgenic plants. </p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>Authors declare that they have no competing interests.</p> </div> <div class="authors’contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>AM planned and performed all the experimental work. NT supervised the work and helped in the preparation of the final draft of the manuscript. Both authors read and approved the final manuscript. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 1</div> <p>Leaf disk assay.</p>  <div class="media" title="media"> <a href="1471-2229-12-183-S1.docx">LINK</a>  <p>Click here for file</p>  </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 2</div> <p>Domain organization of OsABP, OsDBH and OsDSHCT proteins.</p>  <div class="media" title="media"> <a href="1471-2229-12-183-S2.docx">LINK</a>  <p>Click here for file</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by the Short-Term Fellowship (Ref. No. F/ROM11-02) Program awarded through the International Center of Genetic Engineering and Biotechnology (ICGEB). Work on plant helicases, plant stress signaling and rice transformation in NT’s laboratory is partially supported by Department of Biotechnology (DBT), and Department of Science and Technology (DST), Government of India. Thanks to Dr. Ranjan Kumar Sahoo for providing the rice seeds, Dr Sunil Mukherjee for his advises during the initial stage of the work and Dr. Alma Balestrazzi for her useful suggestions concerning the preparation of the manuscript. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahi</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blumwald</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grover</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">Salt stress response in rice: genetics, molecular biology and comparative genomics</span> <span class="source" tagx="source" title="source">Funct Integr Genomics</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">263</span> <span class="lpage" tagx="lpage" title="lpage">284</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-006-0032-5">10.1007/s10142-006-0032-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16819623">16819623</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scandalios</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="mixed-article-title" title="mixed-article-title">Oxidative stress: molecular perception and transduction of signals triggering antioxidant gene defenses </span> <span class="source" tagx="source" title="source">Braz J Med Biol Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">995</span> <span class="lpage" tagx="lpage" title="lpage">1014</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1590/S0100-879X2005000700003">10.1590/S0100-879X2005000700003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16007271">16007271</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawasaki</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deyholos</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brazille</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawai</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galbraith</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohnert</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene expression profiles during initial phase of salt stress in rice</span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">889</span> <span class="lpage" tagx="lpage" title="lpage">905</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11283343">11283343</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mazzucotelli</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mastragelo</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crosatti</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guerra</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanca</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cattivelli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="mixed-article-title" title="mixed-article-title">Abiotic stress response in plants: When post-transcriptional and post-translational regulations control transcription </span> <span class="source" tagx="source" title="source">Plant Sci</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">174</span> <span class="fpage" tagx="fpage" title="fpage">420</span> <span class="lpage" tagx="lpage" title="lpage">431</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2008.02.005">10.1016/j.plantsci.2008.02.005</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNAs: Genomics, biogenesis, mechanism and function</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">116</span> <span class="fpage" tagx="fpage" title="fpage">281</span> <span class="lpage" tagx="lpage" title="lpage">297</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(04)00045-5">10.1016/S0092-8674(04)00045-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14744438">14744438</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNAs and their regulatory roles in plants</span> <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">19</span> <span class="lpage" tagx="lpage" title="lpage">53</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.57.032905.105218">10.1146/annurev.arplant.57.032905.105218</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16669754">16669754</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feinbaum</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ambros</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="mixed-article-title" title="mixed-article-title">The C. elegans heterochormic gene lin-4 encodes small RNAs with antisense complementarity to lin-14 </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">843</span> <span class="lpage" tagx="lpage" title="lpage">854</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0092-8674(93)90529-Y">10.1016/0092-8674(93)90529-Y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8252621">8252621</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weinstein</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in plants</span> <span class="source" tagx="source" title="source">Genes Dev</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">1616</span> <span class="lpage" tagx="lpage" title="lpage">1626</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1004402">10.1101/gad.1004402</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12101121">12101121</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffiths-Jones</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grocock</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dongen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bateman</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enright</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">miRBase: microRNA sequences, targets and gene nomenclature</span> <span class="source" tagx="source" title="source">Nucl Acid Res</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">140</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkj430">10.1093/nar/gkj430</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bo</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="mixed-article-title" title="mixed-article-title">Conservation and divergence of microRNAs and their functions in Euphorbiaceous plants</span> <span class="source" tagx="source" title="source">Nucl Acid Res</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">981</span> <span class="lpage" tagx="lpage" title="lpage">995</span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional diversity of miRNA in plants</span> <span class="source" tagx="source" title="source">Plant Sci</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">172</span> <span class="fpage" tagx="fpage" title="fpage">423</span> <span class="lpage" tagx="lpage" title="lpage">432</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2006.10.009">10.1016/j.plantsci.2006.10.009</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trindade</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Capitão</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dalmay</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fevereiro</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santos</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="mixed-article-title" title="mixed-article-title">miR398 and miR408 are up-regulated in response to water deficit in Medicago truncatula</span> <span class="source" tagx="source" title="source">Planta</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">231</span> <span class="fpage" tagx="fpage" title="fpage">705</span> <span class="lpage" tagx="lpage" title="lpage">716</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-009-1078-0">10.1007/s00425-009-1078-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20012085">20012085</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Capitão</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paiva</span> <span class="given-names" tagx="given-names" title="given-names">JAP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santos</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fevereiro</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="mixed-article-title" title="mixed-article-title">In Medicago truncatula, water deficit modulates the transcript accumulation of components of small RNA pathways </span> <span class="source" tagx="source" title="source">BMC Plat Biol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">79</span> <span class="lpage" tagx="lpage" title="lpage">93</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-11-79">10.1186/1471-2229-11-79</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunkar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> <span class="mixed-article-title" title="mixed-article-title">Novel and stress regulated micro-RNAs and other small RNAs from Arabidopsis</span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">2001</span> <span class="lpage" tagx="lpage" title="lpage">2019</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.104.022830">10.1105/tpc.104.022830</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15258262">15258262</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">ZM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">YO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">GT</span> </span> <span class="mixed-article-title" title="mixed-article-title">Review of plant miRNAs in environmental stressed conditions</span> <span class="source" tagx="source" title="source">Res J Agr Biol Sci</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">803</span> <span class="lpage" tagx="lpage" title="lpage">814</span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Umate</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">microRNA access to the target helicases from rice</span> <span class="source" tagx="source" title="source">Plant Signal Behav</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1171</span> <span class="lpage" tagx="lpage" title="lpage">1175</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.5.10.12801">10.4161/psb.5.10.12801</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20953156">20953156</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Umate</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Architectures of the unique domains associated with the DEAD-box helicase motif</span> <span class="source" tagx="source" title="source">Cell Cycle</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">4228</span> <span class="lpage" tagx="lpage" title="lpage">4235</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/cc.9.20.13635">10.4161/cc.9.20.13635</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20935500">20935500</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant DNA helicases: the long unwinding road</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">2201</span> <span class="lpage" tagx="lpage" title="lpage">2214</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/erg246">10.1093/jxb/erg246</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14504296">14504296</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Owttrim</span> <span class="given-names" tagx="given-names" title="given-names">GW</span> </span> <span class="mixed-article-title" title="mixed-article-title">RNA helicases and abiotic stress</span> <span class="source" tagx="source" title="source">Nucl Acid Res</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">3220</span> <span class="lpage" tagx="lpage" title="lpage">3230</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkl408">10.1093/nar/gkl408</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Stress responsive DEAD-box helicases: a new pathway to engineer plant stress tolerance</span> <span class="source" tagx="source" title="source">J Photochem Photobiol Biology</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">150</span> <span class="lpage" tagx="lpage" title="lpage">160</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jphotobiol.2006.02.010">10.1016/j.jphotobiol.2006.02.010</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phan</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sopory</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Pea DNA helicase 45 overexpression in tobacco confers high salinity tolerance without affecting yield </span> <span class="source" tagx="source" title="source">PNAS</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">509</span> <span class="lpage" tagx="lpage" title="lpage">514</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0406485102">10.1073/pnas.0406485102</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15630095">15630095</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elias</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hossain</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferdousi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahman</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seraj</span> <span class="given-names" tagx="given-names" title="given-names">ZI</span> </span> <span class="mixed-article-title" title="mixed-article-title">Over-expression of a Dead-box helicase, PDH45, confers both seedling and reproductive stage salinity tolerance to rice (Oryza sativa L.) </span> <span class="source" tagx="source" title="source">Mol Breeding</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">345</span> <span class="lpage" tagx="lpage" title="lpage">354</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11032-011-9625-3">10.1007/s11032-011-9625-3</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahoo</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Pea DNA helicase 45 promotes salinity stress tolerance in IR64 rice with improved yield </span> <span class="source" tagx="source" title="source">Plant Sign Behav</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="comment">in press</span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tajrishi</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaid</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Overexpression of a pea DNA helicase 45 in bacteria confers salinity stress tolerance</span> <span class="source" tagx="source" title="source">Plant Sign Behav</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1271</span> <span class="lpage" tagx="lpage" title="lpage">1275</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.6.9.16726">10.4161/psb.6.9.16726</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wellburn</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> <span class="mixed-article-title" title="mixed-article-title">The spectral determination of chlorophylls a and b, as well as total carotenoids, using various solvents with spectrophotometers of different resolution </span> <span class="source" tagx="source" title="source">J Plant Physiol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">144</span> <span class="fpage" tagx="fpage" title="fpage">307</span> <span class="lpage" tagx="lpage" title="lpage">313</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0176-1617(11)81192-2">10.1016/S0176-1617(11)81192-2</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bates</span> <span class="given-names" tagx="given-names" title="given-names">LS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waldren</span> <span class="given-names" tagx="given-names" title="given-names">RP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teare</span> <span class="given-names" tagx="given-names" title="given-names">ID</span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid determination of free proline for water-stress studies</span> <span class="source" tagx="source" title="source">Plant Soil</span> <span class="year" tagx="year" title="year">1973</span> <span class="volume" tagx="volume" title="volume">270</span> <span class="fpage" tagx="fpage" title="fpage">343</span> <span class="lpage" tagx="lpage" title="lpage">353</span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chattopadhyay</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of gene expression in response to water deficit of chickpea (Cicer arietinum L.) varieties differing in drought tolerance </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">24</span> <span class="lpage" tagx="lpage" title="lpage">38</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-10-24">10.1186/1471-2229-10-24</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20144227">20144227</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Islam</span> <span class="given-names" tagx="given-names" title="given-names">SMS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Rice heterodimeric G-protein Gamma subunits (RGG1 and RGG2) are differentially regulated under abiotic stress </span> <span class="source" tagx="source" title="source">Plant Sign Behav</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.7.1.18574">10.4161/psb.7.1.18574</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="mixed-article-title" title="mixed-article-title">Global expression profiling of rice microRNAs by one-tube stem-loop reverse transcription quantitative PCR revealed important roles of microRNAs in abiotic stress responses </span> <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="year" tagx="year" title="year">2012</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-010-0581-0">10.1007/s00438-010-0581-0</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and 2 <sup>-ΔΔCt</sup> method </span> <span class="source" tagx="source" title="source">Methods</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">402</span> <span class="lpage" tagx="lpage" title="lpage">408</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="mixed-article-title" title="mixed-article-title">Computational identification of plant microRNAs and their targets, including a stress induced miRNA </span> <span class="source" tagx="source" title="source">Mol Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">787</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molcel.2004.05.027">10.1016/j.molcel.2004.05.027</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15200956">15200956</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">LP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burge</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="mixed-article-title" title="mixed-article-title">Prediction of plant qsmicroRNA targets</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">110</span> <span class="fpage" tagx="fpage" title="fpage">513</span> <span class="lpage" tagx="lpage" title="lpage">520</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(02)00863-2">10.1016/S0092-8674(02)00863-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12202040">12202040</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aubourg</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kreis</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lecharny</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">The DEAD box RNA helicase family in Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Nucl Acid Res</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">628</span> <span class="lpage" tagx="lpage" title="lpage">636</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/27.2.628">10.1093/nar/27.2.628</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">RN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klauer</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hintze</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Hoof</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">he crystal structure of Mtr4 reveals a novel arch domain required for rRNA processing</span> <span class="source" tagx="source" title="source">EMBO J</span> <span class="year" tagx="year" title="year">2010</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">12</span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Staub</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fiziev</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosenthal</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinzmann</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="mixed-article-title" title="mixed-article-title">Insights into the evolution of the nucleolus by analysis of its protein domain repertoire</span> <span class="source" tagx="source" title="source">Bioessays</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">567</span> <span class="lpage" tagx="lpage" title="lpage">581</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.20032">10.1002/bies.20032</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15112237">15112237</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fedoroff</span> <span class="given-names" tagx="given-names" title="given-names">NV</span> </span> <span class="mixed-article-title" title="mixed-article-title">The helicase and RNaseIII domains of Arabidopsis DCL1 modulate catalytic parameters during microRNA biogenesis </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2012</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.112.193508">10.1104/pp.112.193508</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sankararamakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of major intrinsic proteins in the tree plant Populs trichocarpa: characterization of XIP subfamily of aquaporins from evolutionary perspective </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">134</span> <span class="lpage" tagx="lpage" title="lpage">162</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19930558">19930558</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mallory</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dugas</span> <span class="given-names" tagx="given-names" title="given-names">DV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNA regulation of NAC-domain targets is required for proper formation and separation of adjacent embryonic, vegetative and floral organs </span> <span class="source" tagx="source" title="source">Curr Biol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">1035</span> <span class="lpage" tagx="lpage" title="lpage">1046</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2004.06.022">10.1016/j.cub.2004.06.022</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15202996">15202996</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachrati</span> <span class="given-names" tagx="given-names" title="given-names">CZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hickson</span> <span class="given-names" tagx="given-names" title="given-names">ID</span> </span> <span class="mixed-article-title" title="mixed-article-title">RecQ helicases: guardian angels of the DNA replication fork</span> <span class="source" tagx="source" title="source">Chromosoma</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">117</span> <span class="fpage" tagx="fpage" title="fpage">219</span> <span class="lpage" tagx="lpage" title="lpage">233</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00412-007-0142-4">10.1007/s00412-007-0142-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18188578">18188578</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Islam</span> <span class="given-names" tagx="given-names" title="given-names">MN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fox</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Enomoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="mixed-article-title" title="mixed-article-title">RecQL5 promotes genome stabilization through two parallel mechanisms – interacting with RNA polymerase II and acting as a helicase </span> <span class="source" tagx="source" title="source">Mol Cell Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">2460</span> <span class="lpage" tagx="lpage" title="lpage">2472</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MCB.01583-09">10.1128/MCB.01583-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20231364">20231364</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de la Cruz</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kressler</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tollervey</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Linder</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="mixed-article-title" title="mixed-article-title">Dob1 (Mtr4p) is a putative ATP-dependent RNA helicase required for the 39 end formation of 5.8S rRNA in Saccharomices cerevisiae </span> <span class="source" tagx="source" title="source">EMBO J</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">1128</span> <span class="lpage" tagx="lpage" title="lpage">1140</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/emboj/17.4.1128">10.1093/emboj/17.4.1128</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9463390">9463390</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macovei</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaid</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tula</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">A new DEAD-box helicase ATP binding protein (OsABP) from rice is responsive to abiotic stress </span> <span class="source" tagx="source" title="source">Plant Signal Behav</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.7.1.18574">10.4161/psb.7.1.18574</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22301955">22301955</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pradhan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">Cold- and salinity stress-induced bipolar pea DNA helicase 47 is involved in protein synthesis and stimulated by phosphorylation with protein kinase </span> <span class="source" tagx="source" title="source">C Plant J</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">76</span> <span class="lpage" tagx="lpage" title="lpage">87</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02511.x">10.1111/j.1365-313X.2005.02511.x</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">NQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dang</span> <span class="given-names" tagx="given-names" title="given-names">HQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">A single subunit MCM6 from pea forms homohexamer and functions as DNA helicase</span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">327</span> <span class="lpage" tagx="lpage" title="lpage">336</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-010-9675-7">10.1007/s11103-010-9675-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20730596">20730596</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dang</span> <span class="given-names" tagx="given-names" title="given-names">HQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">NQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">A single subunit MCM6 from pea promotes salinity stress tolerance without affecting yield </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">19</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-011-9758-0">10.1007/s11103-011-9758-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21365356">21365356</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">NQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dang</span> <span class="given-names" tagx="given-names" title="given-names">HQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant MCM proteins: role in DNA replication and beyond</span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">537</span> <span class="lpage" tagx="lpage" title="lpage">545</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-011-9836-3">10.1007/s11103-011-9836-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22038093">22038093</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">A method to confer salinity stress tolerance to plants by helicase overexpression</span> <span class="source" tagx="source" title="source">Methods Mol Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">587</span> <span class="fpage" tagx="fpage" title="fpage">377</span> <span class="lpage" tagx="lpage" title="lpage">387</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20225163">20225163</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cartenter</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeffery</span> <span class="given-names" tagx="given-names" title="given-names">IB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Björkbacka</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fitzgerald</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Neill</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="mixed-article-title" title="mixed-article-title">Detecting microRNA activity from gene expression data</span> <span class="source" tagx="source" title="source">BMC Bioinf</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">257</span> <span class="lpage" tagx="lpage" title="lpage">271</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-11-257">10.1186/1471-2105-11-257</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macovei</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="mixed-article-title" title="mixed-article-title">microRNAs as promising tools for improving stress tolerance in rice</span> <span class="source" tagx="source" title="source">Plant Signal Behav</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.7.1.18574">10.4161/psb.7.1.18574</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22301955">22301955</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maunoury</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaucheret</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="mixed-article-title" title="mixed-article-title">AGO1 and AGO2 act redundantly in miR408-mediated plantacyanin regulation</span> <span class="source" tagx="source" title="source">PlosOne</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">e28729</span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="mixed-article-title" title="mixed-article-title">Differential expression of miRNAs in response to salt stress in maize roots</span> <span class="source" tagx="source" title="source">Ann Bot</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">29</span> <span class="lpage" tagx="lpage" title="lpage">38</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18952624">18952624</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elling</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">XW</span> </span> <span class="mixed-article-title" title="mixed-article-title">Differential expression of microRNAs in maize inbread and hybrid lines during salt and drought stress </span> <span class="source" tagx="source" title="source">Am J Plant Sci</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">69</span> <span class="lpage" tagx="lpage" title="lpage">76</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4236/ajps.2010.12009">10.4236/ajps.2010.12009</a> </span> </span> </li> </ul> </div> </div>  </body></html>