<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4683037">4683037</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26681181">26681181</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0145424">10.1371/journal.pone.0145424</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-15-33806</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genome-Wide Dissection of the MicroRNA Expression Profile in Rice Embryo during Early Stages of Seed Germination </div> <div class="alt-title" title="alt-title">
MicroRNA Analysis of Rice Seed Germination</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">Dongli</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Qiong</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Kun</span> </span> <a href="#aff003"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Pingfang</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#cor001">*</a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Key Laboratory of Plant Germplasm Enhancement and Speciality Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China </span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">University of Chinese Academy of Sciences, Beijing 100049, China</span> </span> <span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">College of life science, Wuhan University, Wuhan 430072, China</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Zhi Min</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Nanjing Agricultural University, CHINA</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: PY DH. Performed the experiments: DH QW. Analyzed the data: DH KW. Contributed reagents/materials/analysis tools: DH QW. Wrote the paper: DH PY. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">yangpf@wbgcas.cn</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-12-12</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0145424</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-8-1</span> <span class="accepted" title="accepted">accepted: 2015-12-3</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">He et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <a class="self-uri" href="pone.0145424.pdf" title="self-uri">pone.0145424.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The first 24 hours after imbibition (HAI) is pivotal for rice seed germination, during which embryo cells switch from a quiescent state to a metabolically active state rapidly. MicroRNAs (miRNAs) have increasingly been shown to play important roles in rice development. Nevertheless, limited knowledge about miRNA regulation has been obtained in the early stages of rice seed germination. In this study, the small RNAs (sRNAs) from embryos of 0, 12, and 24 HAI rice seeds were sequenced to investigate the composition and expression patterns of miRNAs. The bioinformatics analysis identified 289 miRNA loci, including 59 known and 230 novel miRNAs, and 35 selected miRNAs were confirmed by stem-loop real-time RT-PCR. Expression analysis revealed that the dry and imbibed seeds have unique miRNA expression patterns compared with other tissues, particularly for the dry seeds. Using three methods, Mireap, psRNATarget and degradome analyses, 1197 potential target genes of identified miRNAs involved in various molecular functions were predicted. Among these target genes, 39 had significantly negative correlations with their corresponding miRNAs as inferred from published transcriptome data, and 6 inversely expressed miRNA-target pairs were confirmed by 5ʹ-RACE assay. Our work provides an inventory of miRNA expression profiles and miRNA-target interactions in rice embryos, and lays a foundation for further studies of miRNA-mediated regulation in initial seed germination. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was supported by the the National Science Foundation of China (31101213 and 31271805), <a href="http://www.nsfc.gov.cn/">http://www.nsfc.gov.cn/</a>. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="The related sequence data has been uploaded to the NCBI GEO database with accession number GSE73657 (http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73657)." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The related sequence data has been uploaded to the NCBI GEO database with accession number GSE73657 ( <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73657">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73657</a>). </p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Rice is not only the staple food for over half of the world population, but also an ideal plant model for monocot seed germination as it has an annotated genome and a well-developed germination system. A series of physiological and biochemical events were involved in seed germination, during which embryo cells transit from a quiescent state to a metabolically active state. In rice, classical studies have defined seed germination as a sequential process following a triphasic model based on water uptake, namely, rapid water uptake in the first 20 hours after imbibition (HAI) (phase I), lag phase for metabolism reactivation (phase II, 20–48 HAI), and radicle emergence for seedling establishment (phase III, after 48 HAI) [ <a href="#pone.0145424.ref001">1</a>, <a href="#pone.0145424.ref002">2</a>]. Albeit no significant change has been observed in phenotypes of the 24 HAI seed, the oxygen uptake was found to be enhanced in parallel with seed wet weight increase [ <a href="#pone.0145424.ref003">3</a>]. During this period, the pro-mitochondria are converted to the typical cristae-rich mitochondrial structures following recovery of the aerobic respiration for seedling establishment [ <a href="#pone.0145424.ref002">2</a>, <a href="#pone.0145424.ref004">4</a>]. Previous transcriptome study has shown that a significant proportion of transcripts changed in the first 24 HAI, with most changes being observed between 3 and 12 HAI (5,396 up-regulated and 4,935 down-regulated transcripts) [ <a href="#pone.0145424.ref005">5</a>]. In addition, a phosphorproteome study also suggests that the first 12 HAI is a potentially important signal transduction phase for the initial rice seed germination [ <a href="#pone.0145424.ref006">6</a>]. Therefore, the first 24 HAI greatly determines the recovery of metabolism activity in rice seed during the germination progress. </p> <p>Plant microRNAs (miRNAs), generated from stem-loop regions of long primary transcripts by a Dicer-like (DCL) enzyme, are a class of non-coding sRNAs that are abundant and best-characterized with 20-24-nucleotide (nt) length [ <a href="#pone.0145424.ref007">7</a>]. MicroRNAs can guide the RNA-induced silencing complex (RISC) to cleave mRNA or remodel chromatin through fully or partly complementary base pairing, resulting in mRNA slicing and degradation, translational repression, or chromatin modifications [ <a href="#pone.0145424.ref008">8</a>, <a href="#pone.0145424.ref009">9</a>]. MicroRNAs have been shown to play important roles in plant growth and development, phyto-hormone homeostasis, and stress response. The ancient miRNAs often functionally target the evolutionarily conserved genes. For example, miR167 is involved in ovule and anther development by repressing Auxin Responsive Factor 6 ( <i>ARF6</i>) [ <a href="#pone.0145424.ref010">10</a>]; Arabidopsis seeds expressing the miR160-resistant form of <i>ARF10</i> are hypersensitive to abscisic acid (ABA) in a dose-dependent manner [ <a href="#pone.0145424.ref011">11</a>]. Additionally, miR164 and miR319 are important for organ morphogenesis by targeting NAM, ATAF1/2 and CUC2 domain-containing proteins ( <i>NAC</i>) and TEOSINTE BRANCHED1/CYCLOIDEA/PROLIFERATING ( <i>TCP</i>) transcription factor families, respectively [ <a href="#pone.0145424.ref012">12</a>, <a href="#pone.0145424.ref013">13</a>]. Some new miRNA-target pairs have been confirmed experimentally, i.e., miR402 targets DEMETER-LIKE protein3 ( <i>DML</i>3), a putative DNA glycosylase involved in DNA demethylation. Overexpression of miR402 and deficiency of <i>dml3</i> accelerates seed germination under stress conditions [ <a href="#pone.0145424.ref014">14</a>]. As a key regulator, miRNA has been extensively studied in rice for its role in root development, seed maturation, and pollen development [ <a href="#pone.0145424.ref015">15</a>– <a href="#pone.0145424.ref018">18</a>]. The recently released rice dataset of miRBase (V21.0) has documented 332 miRNA families. These identified miRNAs are important for understanding small RNA (sRNA)-mediated gene regulations in rice. However, to date, limited knowledge about miRNA regulation has been obtained in the early stage of rice seed germination. </p> <p>Rice seed is composed of a dominant starchy endosperm and a genetically vigorous embryo. Proteomic analysis of the dissected endosperms has shown that the biological processes operating in the endosperm are heavily regulated by the embryo during seed germination [ <a href="#pone.0145424.ref019">19</a>]. Therefore, embryo is an important tissue in seed germination control. In this study, we sequenced the sRNA populations from embryos of 0, 12, and 24 HAI rice seeds using next-generation deep sequencing technology. A series of miRNAs were identified, including both known and novel miRNAs. We also predicted the potential targets for the miRNAs. Real-time RT-PCR and 5ʹ-RACE assay were performed to confirm some deep sequencing and target prediction results. This study provides the unique composition and expression profiles of miRNAs and their potential regulations in the embryo at the early stages of rice seed germination. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialandseedgermination" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and Seed Germination</div> <p>The de-hulled rice ( <i>Oryza sativa</i> Nipponbare) seeds were washed three times with distilled water, then placed in 9-cm-diameter plates and imbibed in distilled water at 30°C in a dark biochemical incubator. Fifty seeds were placed in one plate and set up as a biological replicate. Four replicates were prepared for each time-point sample. After 0 h, 12 h, and 24 h of imbibition, embryos were dissected manually and collected for RNA extraction, respectively. For phytohormone treatment, seeds were incubated with 200 μM GA <sub>3</sub> and 50 mM ABA (Sigma, St. Louis, MO, USA) for 24 h, after which the embryos were collected. </p> </div> <div class="smallrnalibrariesconstructionandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Small RNA Libraries Construction and Sequencing</div> <p>Total RNA was extracted with TRIzol Reagent (Invitrogen, Carlsbad, CA, USA) according to the manufacturer’s instructions. Equal amounts (about 25 μg) of total RNA from four replicates of each time-point sample were mixed together for sRNA libraries construction. The sRNA libraries were constructed following the methods described by Lu et al [ <a href="#pone.0145424.ref020">20</a>]. Briefly, the total RNA was fractionated by 15% denaturing polyacrylamide gel (8 M urea) electrophoresis, and sRNAs in the range of 18–30 nt were excised and purified with a Spin-X cellulose acetate filter (2 mL, 0.45 um; Thermo Fisher, Waltham, MA, USA). After dephosphorylation and sequential ligation of 5ʹ- and 3ʹ- Solexa adaptors, the sRNAs were reverse-transcribed using superscript <sup>TM</sup>II reverse transcription kit (Invitrogen), and then amplified by PCR with phusion High-Fidelity PCR kit (Finnzymes, Espoo, Finland) to produce sRNA sequencing libraries. Next generation sequencing was performed on an Illumina platform (Nextomics Science and Technology Limited, Wuhan, China). All the sequence data have been deposited as a series with the accession number GSE73657 at NCBI’ GEO database ( <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73657">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73657</a>). </p> </div> <div class="deepsequencingandprocessing" title="sec"> <div class="title" tagx="title" title="title">
Deep Sequencing and Processing</div> <p>The raw sequencing data were filtered with FastQC ( <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) to delete low quality reads, adapters, contamination, and false sequences (such as reads with poly A/T/C/G). After filtering, sRNAs with size ranging from 18 to 30 nt were collected and mapped to the rice genome (MSU Rice Annotation Release 7.0) using Bowtie software, and then the sequences matching known rice rRNA, tRNA, snRNA, and snoRNA in Rfam11 ( <a href="https://www.sanger.ac.uk/resources/databases/rfam.html">https://www.sanger.ac.uk/resources/databases/rfam.html</a>) were discarded. Subsequently, the unique sRNAs were aligned with the data in miRBase 21 ( <a href="http://www.mirbase.org">www.mirbase.org</a>) to search for the conserved miRNAs. </p> <p> <i>De novo</i> prediction of novel miRNA was performed by miRDeep2. Genomic sequences with 300 bp length, surrounding the sRNAs (150 nt upstream and 150 nt downstream), were extracted to assess the potential secondary structures of pre-miRNA. The candidates were first filtered with default criteria, and then with the stringent criterion of minimal folding free energy index (MFEI) &amp;gt; 0.85 (MFEI = [(MFE/length of the sRNA sequence) × 100] / (G+C) %) [ <a href="#pone.0145424.ref021">21</a>]. </p> </div> <div class="targetpredictionofidentifiedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Target Prediction of Identified miRNAs</div> <p>The prediction of putative targets for identified miRNAs was performed with Mireap, PsRNATarget and degradome analyses based on the rice genome (MSU 7.0) [ <a href="#pone.0145424.ref022">22</a>– <a href="#pone.0145424.ref024">24</a>]. For the Mireap software analysis, the following modified parameters were used: 1) no more than four mismatches between miRNA and the target, the G-U base pair was counted as 0.5 mismatches; 2) no more than two adjacent mismatches in the miRNA/target duplex; 3) no adjacent mismatches in positions 2–12 (5ʹ of miRNA); 4) perfect matches in positions 10–11; 5) no more than 2.5 mismatches in positions 1–12; 6) MFE between miRNA and target gene not less than 75% of the MFE for a perfect matching. PsRNATarget prediction software ( <a href="http://plantgrn.noble.org/psRNATarget">http://plantgrn.noble.org/psRNATarget</a>) was used with default parameters. Degradome analysis was performed based on GSE18251, GSE17398, and GSE19050 dataset as previously described with p &amp;lt; 0.05 [ <a href="#pone.0145424.ref015">15</a>, <a href="#pone.0145424.ref022">22</a>]. WEGO software ( <a href="http://wego.genomics.org.cn/">http://wego.genomics.org.cn/</a>) was used for plotting gene ontology annotation results, and Agrigo software ( <a href="http://bioinfo.cau.edu.cn/agriGO">http://bioinfo.cau.edu.cn/agriGO</a>) was used to enrich the GO terms. </p> </div> <div class="expressionanalysisofmirnas,targetgenes,andhostgenes" title="sec"> <div class="title" tagx="title" title="title">
Expression Analysis of miRNAs, Target Genes, and Host Genes</div> <p>The miRNA expression profiles in multiple tissues were obtained from the published dataset [ <a href="#pone.0145424.ref016">16</a>]. To compare the miRNA expression in multiple tissues, the Log <sub>2</sub>TPM (Transcripts per million = number of actual reads/total number of clean reads × 1,000,000) value was presented. MeV4.9 was used for hierachical clustering (HCL) and principal component analysis (PCA). The expression patterns of target genes were extracted from the published microarray dataset (GSE43780) [ <a href="#pone.0145424.ref005">5</a>]. To analyze the expression relevance between a miRNA and its corresponding target gene(s), the TPM value of miRNA or the genechip robust multi-array average (GCRMA) value of the target gene was divided by its maximum to reach abundance between 0 and 1. </p> </div> <div class="stem-loopreversetranscriptionpolymerasechainreaction(rt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Stem-Loop Reverse Transcription Polymerase Chain Reaction (RT-PCR)</div> <p>RNA reverse transcription was performed with specific stem-loop primers as previously described [ <a href="#pone.0145424.ref025">25</a>] ( <a href="#pone.0145424.s008">S1 Table</a>). The stem-loop RT-PCR reactions contained 25 ng of RNA samples, 50 nM stem-loop RT primer, 1 × RT buffer, 0.25 mM of each dNTP, 5 U/μL SuperScript II reverse transcriptase (Invitrogen, USA) and 0.25 U/μL RNase inhibitor. The 10 μL reactions were incubated in a 96-well plate for 30 min at 16°C, 30 min at 42°C and 5 min at 85°C, and then held at 4°C in a Bio-Rad professional thermocycler (Bio-Rad, Hercules, California, USA). The real-time PCR amplification program consisted of initial denaturation at 95°C for 3 min, followed by 40 cycles of 95°C for 10 s, 58°C for 15 s, and 72°C for 15 s. Three biological replicates were performed on the CFX96 Real-time PCR system (Bio-Rad, California, USA). The melting curves (65°C to 95°C) were analyzed to check the specificity of PCR products. PCR efficiency was calculated by LinRegPCR ( <a href="http://www.hartfaalcentrum.nl/">http://www.hartfaalcentrum.nl/</a>; subject: LinRegPCR) considering an ideal value range (1.8–2.05) with correlation R <sup>2</sup>&amp;gt; 0.995 [ <a href="#pone.0145424.ref026">26</a>]. The relative miRNA expression was calculated using the following formula: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Ration=E</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
target</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
target</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mtext_UNKNOWN">
Ct(Control-sample)</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
/ E</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
ref.</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
ref.</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mtext_UNKNOWN">
Ct(Control-sample)</div> </div> </div> </div> </div> </div> </div> <p /> <p>In which E <sub>target</sub> represents PCR efficiency of measured miRNA/gene, E <sub>ref</sub> represents PCR efficiency of reference gene 18S RNA, and ΔCt represents the difference in threshold cycles between the control and sample. </p> </div> <div class="rnaligase-mediatedrapid-amplificationof5’-cdnaends(5’-race)" title="sec"> <div class="title" tagx="title" title="title">
RNA Ligase-Mediated Rapid-Amplification of 5’-cDNA Ends (5’-RACE)</div> <p>To investigate the cleavage sites occurring in the targeted mRNAs, 5ʹ-RACE was performed using the FirstChoice <sup>®</sup> RLM-RACE kit (Ambion, USA) according to the manufacturer’s instructions. Briefly, high quality mRNAs were directly ligated to the 5ʹ-RACE RNA adapter (44 nt), and the ligation reaction was then reverse-transcribed into cDNA with random primers. Two 5ʹ-RACE gene-specific outer and inner primers were used for nested PCR. The PCR products with the anticipated size were cloned and sequenced. The detailed primer sequences are listed in <a href="#pone.0145424.s008">S1 Table</a>. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="deepsequencingofsmallrnas" title="sec"> <div class="title" tagx="title" title="title">
Deep Sequencing of Small RNAs</div> <p>Previous study has shown that large-scale rearrangement of transcripts mediated by RNA synthesis and degradation occurs in the first 24 h of rice seed germination [ <a href="#pone.0145424.ref005">5</a>]. Using an experimental system similar to that utilized in the previous study [ <a href="#pone.0145424.ref005">5</a>], the miRNA composition and dynamic expression patterns were investigated in the early stages of rice seed germination. About 200 embryos from 0, 12, and 24 HAI rice seeds were collected for total RNA extraction and sRNA libraries construction, respectively ( <a href="#pone.0145424.g001">Fig 1</a>). RNA concentration and integrity were detected using Agilent 2100 Bioanalyzer and Agilent RNA 6000 Nano kit. About 100 μg of high-quality RNA from each sample was used for sRNAs isolation and sequencing. In total, 29230963, 34302280, and 33901503 raw reads with high quality (Q30 &amp;gt; 94%) were obtained for 0, 12, and 24 HAI sRNA libraries, respectively. After removal of low quality sequences and adapter contaminants, 19203513, 22332607, and 17359816 reads for each library were perfectly mapped to the reference rice genome (MSU 7.0) with size ranging from 18 to 30 nt ( <a href="#pone.0145424.t001">Table 1</a>). Consistent with the representative size range of DCLs cleavage products [ <a href="#pone.0145424.ref027">27</a>], most of these sRNAs were 21 or 24 nt in length. The 24-nt sRNAs occurred in a particularly high abundance, accounting for 25.3%, 34.7%, and 37.5% in the three libraries, respectively ( <a href="#pone.0145424.g002">Fig 2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Schematic representation of the miRNA analysis in rice embryos at the early stages of seed germination. </div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Size distribution of the sRNAs in libraries from 0, 12, and 24 HAI rice seed embryos.</div>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0145424.t001">10.1371/journal.pone.0145424.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Statistics for sRNA sequencing from three libraries.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Category</th> <th>0 HAI</th> <th>12 HAI</th> <th>24 HAI</th> </tr> <tr> <th /> <th>Reads <a href="#t001fn002"> <sup>b</sup> </a> </th> <th>Unique <a href="#t001fn002"> <sup>b</sup> </a> </th> <th>Reads <a href="#t001fn002"> <sup>b</sup> </a> </th> <th>Unique <a href="#t001fn002"> <sup>b</sup> </a> </th> <th>Reads <a href="#t001fn002"> <sup>b</sup> </a> </th> <th>Unique <a href="#t001fn002"> <sup>b</sup> </a> </th> </tr> </thead> <tbody> <tr> <td>high quality raw Reads <a href="#t001fn001"> <sup>a</sup> </a> </td> <td>29230963 (Q30 = 94.74%)</td> <td>-</td> <td>34302280 (Q30 = 95.29%)</td> <td>-</td> <td>33901503 (Q30 = 94.58%)</td> <td>-</td> </tr> <tr> <td>sequences of 18–30 nt</td> <td>22265577(100%)</td> <td>6596317(100%)</td> <td>27075645(100%)</td> <td>10712075(100%)</td> <td>21236500(100%)</td> <td>9420781(100%)</td> </tr> <tr> <td>mapping to genome</td> <td>19203513(86.3%)</td> <td>4856925(73.6%)</td> <td>22332607(82.5%)</td> <td>7892704(73.7%)</td> <td>17359816(81.8%)</td> <td>7115379(75.5%)</td> </tr> <tr> <td>CDS matched</td> <td>600910(2.7%)</td> <td>211949(3.2%)</td> <td>1570839(5.8%)</td> <td>338298(3.2%)</td> <td>1322306(6.2%)</td> <td>243265(2.6%)</td> </tr> <tr> <td> <b>Non-coding RNA</b> </td> <td /> <td /> <td /> </tr> <tr> <td>total</td> <td>17642144(100%)</td> <td> <b>-</b> <a href="#t001fn003"> <sup>c</sup> </a> </td> <td>19201655(100%)</td> <td> <b>-</b> <a href="#t001fn003"> <sup>c</sup> </a> </td> <td>14737124(100%)</td> <td> <b>-</b> <a href="#t001fn003"> <sup>c</sup> </a> </td> </tr> <tr> <td>known miRNA</td> <td>400047(2.3%)</td> <td>1724(0.4%)</td> <td>873690(4.6%)</td> <td>2506(2.9%)</td> <td>781907(5.3%)</td> <td>2314(29.6%)</td> </tr> <tr> <td>rRNA</td> <td>2289143(13.0%)</td> <td>24748(1.1%)</td> <td>1388914(7.2%)</td> <td>23099(1.7%)</td> <td>747877(5.1%)</td> <td>19579(2.6%)</td> </tr> <tr> <td>tRNA</td> <td>277627(1.6%)</td> <td>8121(2.9%)</td> <td>274039(1.4%)</td> <td>7349(2.7%)</td> <td>421275(2.9%)</td> <td>6878(1.6%)</td> </tr> <tr> <td>snoRNA</td> <td>40642(0.2%)</td> <td>8790(21.6%)</td> <td>44927(0.2%)</td> <td>6001(13.4%)</td> <td>28636(0.2%)</td> <td>4755(16.6%)</td> </tr> <tr> <td>snRNA</td> <td>5956(0.34%)</td> <td>1679(28.2%)</td> <td>6005(0.32%)</td> <td>1390(23.1%)</td> <td>3623(0.25%)</td> <td>1079(29.8%)</td> </tr> <tr> <td>others</td> <td>14628729(82.9%)</td> <td>4345729(29.7%)</td> <td>16614080(86.5%)</td> <td>7138503(43.0%)</td> <td>12753806(86.5%)</td> <td>6489573(50.9%)</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>a</sup>: Q30 is the percentage of base with correct recognition rate over than 99.9%. </p> </div> <div class="fn-type-" title=""> <p> <sup>b</sup>: the percentage to the 100% reads. </p> </div> <div class="fn-type-" title=""> <p> <sup>c</sup>: the percentage to the sequences of corresponding ‘reads’. </p> </div> </div> </div> <p>The composition of the genome-matched sRNAs of rice seed was complex, which included protein-coding RNA fragments (3.13–7.62%) and other non-coding transcripts. The non-coding sRNAs contained various categories, including known miRNAs (2.09–4.5%), rRNAs (4.31–11.9%), tRNAs (1.45–2.43%), snoRNAs (0.16–0.21%), snRNAs (0.02–0.03%) and unannotated sRNAs, (80.9–84.31%). The unannotated sRNAs were further used to predict the novel miRNAs. </p> </div> <div class="identificationofknownandnovelmirnasindryandimbibedriceseeds" title="sec"> <div class="title" tagx="title" title="title">
Identification of Known and Novel miRNAs in Dry and Imbibed Rice Seeds</div> <p>To scan the known miRNAs, the unique sRNAs were aligned to the rice miRNA datasets in miRBase 21 ( <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>), allowing up to two mismatches. In the three libraries, 59 known rice miRNAs homologs from 83 precursors corresponding to 42 miRNA families were matched ( <a href="#pone.0145424.s009">S2 Table</a>). Of which, 14 families had two or more miRNA members, and the other 28 families contained only one miRNA member. In this study, 17 conserved families were detected in the imbibed rice seeds, most of them were found in relatively high abundance. In particular, miR319, miR168, miR156, miR166, and miR159 were mapped with more than 10,000 reads ( <a href="#pone.0145424.s001">S1 Fig</a>). However, some conserved miRNAs, i.e., miR164, mi395, and miR393, were lowly expressed in rice seeds. Nineteen families are known rice-specific miRNAs, and most of them had only one member and were in relatively low abundance. A homolog of ptc-miR6478, previously reported only in <i>Populus trichocarpa</i>, was also identified in this study. Moreover, we detected a new precursor with stringent criteria for osa-miR1862 family (named osa-MIR1862d-2). These results show that our sequencing depth was sufficient to reflect the expression profiles of miRNAs during seed imbibition. </p> <p>For novel miRNAs identification, the 300-bp flanking sequences of unannotated sRNAs were initially extracted, and then subjected to MIRDeep2 to scan potential miRNA precursors with classical stem-loop structures using the default criteria. A total of 382 novel miRNA candidates with length of 18–26 nt processed from 481 putative precursors were obtained from the three libraries. MFEI has been identified to be useful in evaluating genuine miRNAs [ <a href="#pone.0145424.ref021">21</a>], which depends on the MFE, length and G+C content of the potential precursors. These criteria were used in our investigation of miRNAs in rice seeds. Low MFE is an important characteristic of miRNA precursors. In this study, the miRNA precursor’s MFE values varied from -180.6 to 5.2 kcal·mol <sup>-1</sup> according to Mfold 3.2 (Figure A in <a href="#pone.0145424.s005">S1 File</a>). In animal, the size of miRNA precursor ranges typically from 70 to 80 nt, but it is more variable in plant.[ <a href="#pone.0145424.ref028">28</a>]. In rice seeds, the size of miRNA precursors varied from 37 to 268 nt, with most being 51–110 nt(Figure B in <a href="#pone.0145424.s005">S1 File</a>). In addition, the high G+C content can make the pre-miRNA secondary structure more stable and thus difficult to be processed into mature miRNA by the RISC complex [ <a href="#pone.0145424.ref021">21</a>]. Here, the G+C content ranged from 14.03% to 88% (Figure C in <a href="#pone.0145424.s005">S1 File</a>). With all these factors in consideration, we calculated the MFEI value of all identified miRNA precursors as previously described [ <a href="#pone.0145424.ref021">21</a>]. Previous research has shown that more than 90% of known plant pre-miRNAs have an MFEI value greater than 0.85 [ <a href="#pone.0145424.ref021">21</a>]. In this study, the MFEI values ranged from 0.20 to 3.75, with an average of 1.26. To minimize the false positive miRNAs, we further set the novel miRNA criteria as follows: 1) the mature miRNA length ranged from 20 to 24 nt; 2) precursor had a classical fold-back structure and the length was no less than 45 nt; 3) the number of reads was greater than five in at least one library; 4) MFEI value was greater than 0.85 or the complementary miRNA* could be found. Finally, 230 novel miRNAs processed from 324 precursors were obtained ( <a href="#pone.0145424.g001">Fig 1</a>). The detailed information was listed in <a href="#pone.0145424.s010">S3 Table</a>. Of these novel miRNAs, 218 were shared by all three libraries. The novel miRNA was denoted with OsmiR plus a serial number, such as OsmiR-1. Different miRNA precursors that produced the same mature sequence were noted with -1, -2, and -3. </p> <p>In plants, most of the canonical miRNAs (∼21 nt) are in high abundance [ <a href="#pone.0145424.ref029">29</a>]. Similarly, in this study, the 21-nt miRNAs occupied an overwhelming majority of the reads ( <a href="#pone.0145424.g003">Fig 3A</a>, <a href="#pone.0145424.s009">S2 Table</a>). Moreover, a great number of long miRNAs (24 nt) with low abundance existed in the early stages of rice seed germination ( <a href="#pone.0145424.s010">S3 Table</a> and <a href="#pone.0145424.g003">Fig 3B</a>). Specifically, of the 230 novel miRNAs, 176 were 24-nt. The miRNAs function through interacting with AGO proteins, which bind to the miRNA with particular 5ʹ end nucleotide. AGO1 mainly recruits miRNAs that begin with a 5ʹ-U/T; AGO2 prefers to bind sRNAs with 5ʹ -A [ <a href="#pone.0145424.ref030">30</a>, <a href="#pone.0145424.ref031">31</a>]. The first base distribution of all 289 identified miRNAs showed that the first base of 24-nt miRNAs were tended to be A (66.2%), suggesting these miRNAs function with AGO2; while that of the shorter miRNAs were inclined to be U/T (69.3%) ( <a href="#pone.0145424.g003">Fig 3C</a>). Genome mapping revealed that most known and novel miRNAs were located in the intergenic regions, and fewer in the intragenic regions (i.e., exon, intron). In addition, the miRNAs were located randomly on the 5p or 3p arm of their precursors ( <a href="#pone.0145424.g003">Fig 3D and 3E</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Characteristics of all identified miRNAs in libraries from 0, 12 and 24 HAI rice seed embryos. </div> <p>Distribution of sequencing reads <b>(A)</b>, number of miRNAs <b>(B)</b> and the first base of miRNAs <b>(C)</b>. <b>(D)</b> Genome and <b>(E)</b> precursor arm locations of all identified miRNAs. </p>   </div> <p>To validate the deep sequencing results, 35 (17 known and 18 novel) miRNAs with sequence reads higher than 100 were randomly selected for stem-loop RT-PCR analysis. All the 35 amplified miRNAs produced a single PCR band with the length of about 60 bp on the gel (The amplified PCR product included the 20–24 bp miRNA sequence, part of the 44 bp stem-loop primer sequence (that is 34 bp) and several primer balance bases) ( <a href="#pone.0145424.s002">S2 Fig</a>). The PCR products were cloned and sequenced. All the tested miRNAs sequences were verified to be quite correct to that of the sequencing, demonstrating a high reliability of the deep sequencing results. </p> </div> <div class="theexpressionpatternsofknownandnovelmirnasduringriceseedimbibition" title="sec"> <div class="title" tagx="title" title="title">
The Expression Patterns of Known and Novel miRNAs During Rice Seed Imbibition</div> <p>To investigate the tissue expression pattern of the miRNAs, the expression data of known miRNAs in bicellular pollen (BCP), leaf, root and callus of rice were obtained from the previous report (Table A in <a href="#pone.0145424.s006">S2 File</a>) [ <a href="#pone.0145424.ref016">16</a>]. HCL analysis and PCA were performed with Log <sub>2</sub>TPM value of known miRNAs from BCP, leaf, root, callus, and embryos of 0, 12, and 24 HAI rice seeds. Results indicated that 7 samples were significantly clustered into two clades, and the seed embryo samples were separated from the others ( <a href="#pone.0145424.g004">Fig 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Analyses of tissue-specific expression pattern of known miRNAs.</div> <p>Hierarchical clustering analyses <b>(A)</b> and principal component analysis <b>(B)</b> of known miRNAs in various tissues. Clustering of known miRNAs based on Log <sub>2</sub>TPM. The bar represents the scale of miRNAs expression levels of miRNAs. The detailed information is listed in Table A in <a href="#pone.0145424.s006">S2 File</a>. </p>   </div> <p>In total, 286499, 636508, and 592581 miRNA reads were obtained from 0, 12, and 24 HAI libraries, respectively. Although a significantly smaller amount of miRNAs were mapped in the dry seeds (0 HAI) than that in the imbibed seeds (12 HAI and 24 HAI), the amount of genome-mapped sRNAs reads was substantial ( <a href="#pone.0145424.t001">Table 1</a>), which implied the existence of a large number of other categories of sRNAs, i.e., siRNAs, in dry seeds. HCL analysis using the LOG <sub>2</sub>TPM value of miRNAs showed that 0 HAI was separated from the other two time points ( <a href="#pone.0145424.s003">S3 Fig</a>, Table B in <a href="#pone.0145424.s006">S2 File</a>). In order to directly plot the miRNA expression patterns in germinating rice seed, we normalized the TPM values through dividing each miRNA TPM by its corresponding maximum and reaching the abundance between 0 and 1. After normalization, one miRNA that was detected in at least one library with the normalized value lower than 0.5 (ratio &amp;gt;2 fold) was regarded as differentially expressed. A total of 178 differentially expressed miRNAs were obtained in germination initiation stage (Table C in <a href="#pone.0145424.s006">S2 File</a>), most (128 miRNAs) of which were enriched in the 24 HAI. Only four miRNAs were enriched in the dry seeds, especially for ptc-miR6478 and a novel miRNA OsmiR-201. </p> <p>To validate the miRNAs expression patterns obtained by deep sequencing, the 35 selected miRNAs were further verified by stem-loop real-time RT-PCR. The results showed that 27 miRNAs (about 77%) had expression patterns consistent to those in initial sequencing ( <a href="#pone.0145424.g005">Fig 5</a>). However, some inconsistent results were also observed, such as OsmiR-220 and OsmiR-26, which were identified to be up-regulated remarkably with imbibition by sequencing but down-regulated by RT-PCR analysis, reflecting the dynamic change of the miRNAs expressions. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Heatmaps of sequencing data and stem-loop real-time RT-PCR.</div> <p>The bar represents the scale of the expression levels of miRNAs. The stars indicate inconsistent results between sequencing and RT-PCR. </p>   </div> <p>The intragenic miRNA, especially for the intronic miRNA, may be derived from a common precursor transcript and display coincidental expression pattern with the host gene [ <a href="#pone.0145424.ref032">32</a>, <a href="#pone.0145424.ref033">33</a>]. We also investigated the expression relationship between miRNAs and the host genes inferred from the transcriptome dataset (GSE43780) [ <a href="#pone.0145424.ref005">5</a>]. However, neither the exonic nor intronic miRNAs had significant co-expression with their host genes, suggesting that these miRNAs were controlled by different promoters with their host genes ( <a href="#pone.0145424.s004">S4 Fig</a>, <a href="#pone.0145424.s011">S4 Table</a>). </p> </div> <div class="targetpredictionandfunctionanalysisofidentifiedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Target Prediction and Function Analysis of Identified miRNAs</div> <p>The target genes of all identified miRNAs were predicted using three classical methods: Mireap, degradome and psRNATarget analysis. After searching the rice genome dataset MSU7.0, 132, 163, 1011 target genes were obtained for 72, 133, and 242 miRNAs by the three methods, which were assembled into 207, 217, and 1508 miRNA-target pairs, respectively. Finally, a total of 1197 target genes for 259 miRNAs were found, which constituted 1784 miRNA-target pairs ( <a href="#pone.0145424.s012">S5 Table</a>). Unexpectedly, only 25 pairs (21 genes targeted by 15 known miRNAs and one novel miRNA) were shared by all three prediction methods, and 123 (82 genes targeted by 26 known miRNAs and 24 novel miRNAs) pairs were shared by at least two methods ( <a href="#pone.0145424.g006">Fig 6A</a>). Of the 21 common targets, most were conserved transcription factors for ancient miRNAs, including SQUAMOSA PROMOTER BINDING PROTEIN-LIKE ( <i>SPL</i>s) protein coding genes targeted by miR156/miR529, <i>ARF</i>s targeted by miR160, <i>TCP</i>s targeted by miR319, and <i>MYB</i>s targeted by miR159. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Gene ontology (GO) analysis of miRNAs’ target genes.</div> <p> <b>(A)</b> Venn diagram shows the overlapping relationship of the target prediction results among the three methods. <b>(B)</b> Categorization of miRNAs target genes with WEGO ( <a href="http://wego.genomics.org.cn/cgi-bin/wego/index.pl">http://wego.genomics.org.cn/cgi-bin/wego/index.pl</a>). <b>(C)</b> GO enrichment of target genes using Agrigo ( <a href="http://bioinfo.cau.edu.cn/agriGO/">http://bioinfo.cau.edu.cn/agriGO/</a>). The color scale shows the P-value cutoff levels for each biological process. The more statistically significant it is, the darker or redder the color is. </p>   </div> <p>All the putative target genes were categorized through gene ontology (GO) analysis. The results showed that these genes were involved in various molecular functions, i.e., binding, catalytic activity, and transcription regulation, and played roles in many biological processes, i.e., metabolic and cellular processes and signaling ( <a href="#pone.0145424.g006">Fig 6B</a>). The Singular Enrichment Analysis (SEA) results of Agrigo software indicated that target genes related to the anatomical structure development (p = 2*10 <sup>−6</sup>) and multicellular organismal development (P = 5.7*10 <sup>−6</sup>) were significantly enriched in the biological process category ( <a href="#pone.0145424.g006">Fig 6C</a>). In contrast, no GO term was enriched for the categories of cellular component and molecular function. </p> <p>Furthermore, the expression profiles of target genes were extracted from the transcriptome dataset (GSE43780) [ <a href="#pone.0145424.ref005">5</a>]. In general, the mRNA expression profiles were negatively correlated with their corresponding miRNAs (Table A in <a href="#pone.0145424.s007">S3 File</a>). Of the 178 differentially expressed miRNAs, 28 (16 known and 12 novel) miRNAs with sequencing reads over 100 were significantly expressed opposite to their targets (Folds &amp;gt; 2, Correlation index &amp;lt; -0.5, Table B in <a href="#pone.0145424.s007">S3 File</a>). These targets included 39 genes encoding proteins for major and minor metabolism, protein degradation and post-translational modification, transportation, and transcriptional regulation, which participated in cellular, metabolic, growth, developmental, and other biological processes. </p> </div> <div class="verificationofmirnas-inducedcleavageby5’-race" title="sec"> <div class="title" tagx="title" title="title">
Verification of miRNAs-Induced Cleavage by 5’-RACE</div> <p>To validate the authentic cleavage event of the known or novel miRNAs in the germinating rice seed, some inversely expressed miRNA-target pairs were selected to verify the cleavage site through 5ʹ-RACE. </p> <p>GAMYB is well documented as the gibberellin (GA) signaling regulator in cereal aleurone cells [ <a href="#pone.0145424.ref034">34</a>]. The transcriptome data of germinating rice seeds indicated that both <i>OsGAMYBL1</i> ( <i>LOC_Os06g40330</i>) and <i>OsGAMYB</i> ( <i>LOC_Os01g59660</i>) were remarkably down-regulated in the first 24 HAI. Our RT-PCR results showed that the mRNA level of <i>OsGAMYBL1</i> was constant while the level of <i>OsGAMYB</i> decreased in the first 24 h of imbibition ( <a href="#pone.0145424.g007">Fig 7A</a>). ABA and GA are two vital phytohormones in regulating seed germination. We detected ABA and GA responses of the selected miRNAs and their targets in this study. RT-PCR results showed that both miR159 and <i>GAMYB</i>s were insensitive to ABA and GA in rice embryos. RNA ligase-mediated 5ʹ-RACE indicated that <i>OsGAMYBL1</i> and <i>OsGAMYB</i> produced PCR products with canonical cleavage sites of miR159 ( <a href="#pone.0145424.g007">Fig 7B</a>). Unexpectedly, we also found some fragments mapped down-stream of the canonical cleavage sites, which reflected the existence of RNA degradation after cleavage, and hence the post-transcriptional modification. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0145424.g007"> <span class="label" tagx="label" title="label">Fig 7</span> </a> </div>   <div class="title" tagx="title" title="title">
Real-time RT-PCR and 5’-RACE analysis.</div> <p> <b>(A)</b> Real-time RT-PCR analysis of the relative expression of osa-miR159b, osa-miR159f, osa-miR1428e and OsmiR203 and their corresponding targets in embryos of 0, 12, and 24 HAI and ABA- or GA- treated seeds. The data represent the mean values ± SD of three replicates. The rice 18S rRNA fragment was amplified as internal control. Asterisks (* p &amp;lt; 0.01, Student’s t-test) represent significant differences from the expression level in 24 HAI seed embryos. <b>(B)</b> Validation of target mRNA cleavage sites by RNA ligase mediated 5ʹ -RACE. The rectangles on the gel show the bands of nested PCR products. The arrows indicate the cleavage sites, the numbers under the arrows show the frequency of RACE clones sequenced, and the numbers in the brackets show the position and frequency of cleavage sites downstream of the canonical sites. </p>   </div> <p>Real-time RT-PCR results indicated that rice special miRNA osa-miR1428e-3p was increased during seed imbibition and induced greatly by GA. Inversely, its target <i>OSK3</i> (a carbon catabolite depressing protein kinase encoded by <i>LOC_Os03g17980</i>) was decreased during seed imbibition and insensitive to GA and ABA. OsmiR-203 was induced by GA as well, and its target <i>LOC_Os11g37540</i> (encoding a protein phosphatase 2C family protein) displayed consistent expression under all detected conditions ( <a href="#pone.0145424.g007">Fig 7A</a>). The 5ʹ-RACE also detected the canonical cleavage sites on the <i>OSK3</i> and <i>LOC_Os11g37540</i> for osa-miR1428e-3p and OsmiR-203, while most clones carried fragments matched to the downstream sequences, suggesting that mRNA fragments induced by miRNA-mediated cleavage were easily degraded from the 5ʹ terminal ( <a href="#pone.0145424.g007">Fig 7B</a>). </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="specificmirnaprofilesandexpressionpatternsindryandimbibedriceseeds" title="sec"> <div class="title" tagx="title" title="title">
Specific miRNA Profiles and Expression Patterns in Dry and Imbibed Rice Seeds</div> <p>The rice dataset of the miRBase (version 21) has recorded 688 precursors for mature miRNAs from 332 families. However, we identified only 83 precursors processed for 59 known miRNAs belonging to 42 families in rice seeds; this ratio is relatively lower than that in leaf, root and pollen [ <a href="#pone.0145424.ref016">16</a>]. Moreover, the HCL analysis and PCA indicated that the known miRNA expression profiles in the embryo of rice seeds were significantly separated from that of the leaf, root, callus and developing pollen ( <a href="#pone.0145424.g004">Fig 4</a>). Particularly, some known miRNAs, i.e., osa-miR1876 and osa-miR5150, were abundantly expressed in rice seed but were not detected in other tissues, and osa-miR159f, osa-miR166l, and osa-miR319b were predominantly expressed in the seed, which suggested the pivotal regulatory roles of these miRNAs in seed germination. The size distribution of miRNAs has been revealed to be tissue-specific in Arabidopsis and rice [ <a href="#pone.0145424.ref016">16</a>, <a href="#pone.0145424.ref035">35</a>]. Most of the conserved miRNAs are short sRNAs with size ranging from 20 to 22 nt [ <a href="#pone.0145424.ref007">7</a>]. The 24-nt miRNAs are involved in DNA methylation or histone modifications leading to transcriptional gene silencing and chromatin remodeling [ <a href="#pone.0145424.ref036">36</a>, <a href="#pone.0145424.ref037">37</a>]. In imbibed seeds, many 24-nt miRNAs with low abundance and fluctuating expression patterns were identified ( <a href="#pone.0145424.s009">S2</a> and <a href="#pone.0145424.s010">S3</a> Tables), and most of them were novel. From the above analysis, we concluded that the unique miRNAs composition and expression profiles existed in dry and imbibed rice seeds. </p> <p>The composition of miRNAs in germinating rice seed was relatively stable as 277 of 289 identified miRNAs were shared by three libraries ( <a href="#pone.0145424.g001">Fig 1</a>), whereas their expression pattern varied greatly. Altogether, 178 differentially expressed miRNAs were identified, and only few miRNAs had higher TPM values in dry seeds, such as ptc-miR6478 and OsmiR-201 (Table C in <a href="#pone.0145424.s006">S2 File</a>). However, the miRNAs expression patterns between 12 and 24 HAI were similar. HCL analysis showed that the expression profile of miRNAs in 0 HAI seed embryos was separated from that of the other two time points ( <a href="#pone.0145424.s004">S4 Fig</a>). The drastic turnover of the miRNAs profile from dry seeds to imbibed seeds suggested that miRNA-mediated regulatory metabolisms were rapidly activated after imbibition. </p> </div> <div class="rolesofmirnasininitiationofriceseedgermination" title="sec"> <div class="title" tagx="title" title="title">
Roles of miRNAs in Initiation of Rice Seed Germination</div> <p>Since protein translation from the stored long-lived mRNAs is indispensable for seed germination, the miRNA-mediated post-transcription regulation can play key roles in the mRNA selective translation and protein turnover during this process [ <a href="#pone.0145424.ref038">38</a>, <a href="#pone.0145424.ref039">39</a>]. To comprehensively explore the possible roles of identified miRNAs during seed imbibition, two widely accepted softwares, Mireap and PsRNATarget, were used to predict the miRNA targets. According to the miRNA-target sequence complementation, it is relatively easy to predict the plant miRNA targets with the reference genomic or transcriptomic sequence data [ <a href="#pone.0145424.ref023">23</a>]. However, the computational prediction is challenged by a high level of false positives that exist in miRNA-target pairing [ <a href="#pone.0145424.ref040">40</a>]. Degradome sequencing is a recently established, efficient strategy to identify miRNA targets on a large scale [ <a href="#pone.0145424.ref022">22</a>, <a href="#pone.0145424.ref041">41</a>]. In this study, degradome analysis was also used for targets prediction based on the published datasets. In total, 1197 target genes involved in various functions for 259 miRNAs were obtained by three prediction methods. However, the remaining 30 miRNAs were failed to find potential targets. These miRNAs might regulate gene expression through non-stringent complementary paring, such as translational repression [ <a href="#pone.0145424.ref042">42</a>, <a href="#pone.0145424.ref043">43</a>], or guide the non-coding locus cleavage and produce phasi/tasiRNA [ <a href="#pone.0145424.ref008">8</a>]. GO enrichment results indicated that target genes involved in anatomical structure development and multicellular organismal development, i.e., wall-associated protein kinases ( <i>OsWAK</i>s), expansin precursors, and <i>OsSPL</i>s, were over-represented. This is consistent with the development process of germination initiation, during which many cell structures need to be repaired or synthesized <i>de novo</i>. Unexpectedly, only 25 miRNA-target pairs overlapped in the three methods of analysis. The ultra-low overlap among these target prediction methods reflected a difference in the parameter setting or prediction principle. This is also strongly suggestive of the predicted targets which need to be confirmed experimentally since the result varied with different prediction methods. </p> <p>Transcription factors such as <i>TCP</i>s, <i>ARF</i>s and <i>SPL</i>s have been confirmed to be easily targeted by conserved miRNAs. By repressing <i>TCP</i>s, miR319 played a central role in coordinating multiple miRNAs (i.e., miR396 and miR164) and phytohormones (including auxin, ABA and GA) pathways to control lateral organ development [ <a href="#pone.0145424.ref044">44</a>, <a href="#pone.0145424.ref045">45</a>]. Auxin alone is not generally considered as a typical seed germination hormone, but it was revealed to interact with ABA through <i>ARF10</i> targeted by miR160 and to be involved in seed germination and post-germination control [ <a href="#pone.0145424.ref011">11</a>]. The auxin signal affects rice root development through regulation of miR156-SPLs [ <a href="#pone.0145424.ref046">46</a>]. SPL proteins have also been revealed to restrain the transition from post-germination to autotrophic growth and affect seedling establishment [ <a href="#pone.0145424.ref031">31</a>, <a href="#pone.0145424.ref047">47</a>]. In this study, osa-miR319 and osa-miR156abcdejl showed high abundance, but miR160abe was relatively low-expressed in all three libraries. Since the complex biological process of seed germination is regulated by a great interaction network of phytohormones, these conserved miRNAs are devoted to the germination initiation and the following seedling establishment through targeting the related signaling regulation factors. However, until now, the mechanisms of miRNAs-mediated phytohormone crosstalk involving in seed germination are largely unknown. </p> <p>The first 24 HAI is critical for initiation of rice seed germination, during which RNA transcription, metabolism resumption and cellular repair are highly activated [ <a href="#pone.0145424.ref019">19</a>, <a href="#pone.0145424.ref038">38</a>, <a href="#pone.0145424.ref048">48</a>, <a href="#pone.0145424.ref049">49</a>]. According to the published transcriptome data, 28 miRNAs were found to be expressed remarkably opposite to their targeted genes (Table B in <a href="#pone.0145424.s007">S3 File</a>), including conserved miRNAs, i.e., miR169ijk targeted nuclear factors, miR390 targeted ATP binding proteins, and miR166e targeted SGT1 protein, which are known to participate in plant development and disease resistance [ <a href="#pone.0145424.ref050">50</a>]. Novel miRNA-target pairs provide useful information to explore the new functions of miRNAs in rice seed germination. For example, OsmiR-129 sharply increased during seed imbibition, it might reduce the phytic acid synthesis in rice seeds by targeting a multi-drug resistance-associated ABC transporter and release micronutrients and organic phosphorus for seed germination [ <a href="#pone.0145424.ref051">51</a>]. Notably, among those inversely expressed miRNA-target pairs, several miRNAs were found to target genes with unknown function, i.e., OsmiR-100 (LOC_Os07g07030), OsmiR-217 (LOC_Os07g08669), OsmiR-232 (LOC_Os05g06910). It is worth in-depth investigation on the roles of those miRNA-target pairs in the seed germination regulation. </p> <p>The expression level of miRNAs are not only inversely but sometimes also positively correlated with their target genes for feedback regulation or other interference factors [ <a href="#pone.0145424.ref016">16</a>]. In this study, many differentially expressed miRNAs were identified to be positively correlated with their target genes, such as OsmiR-201. OsmiR-201 is highly expressed in dry seeds but the expression decreases rapidly during imbibition. Three potential targets of OsmiR-201, LOC_Os05g50660 (encoding a PX domain containing protein), LOC_Os01g69940 (encoding an F-box domain containing protein), and LOC_Os01g62800 (encoding a methyltransferase) decrease during seed imbibition. Exploring whether the feedback regulations exist in these kinds of miRNA-target pairs will help to understand the miRNA-mediated regulations from a new angle. </p> </div> <div class="micrornamediatesthecrosstalkbetweenabaandgaincontrollingriceseedgermination" title="sec"> <div class="title" tagx="title" title="title">
MicroRNA Mediates the Crosstalk between ABA and GA in Controlling Rice Seed Germination</div> <p>ABA is well documented as a seed dormancy maintainer that functions antagonistically to GA. MicroRNA159 has been reported extensively to be involved in the crosstalk between ABA and GA through cleavage of <i>GAMYB</i>-like genes in leaves, flowers, seed maturation and germination process [ <a href="#pone.0145424.ref034">34</a>, <a href="#pone.0145424.ref052">52</a>, <a href="#pone.0145424.ref053">53</a>]. In Arabidopsis seeds, exogenous ABA results in the accumulation of miR159 that participates in the regulation of seed germination by targeting MYB33 and MYB101, two positive regulators of ABA response [ <a href="#pone.0145424.ref053">53</a>]. This negative feedback might contribute to the developmental switch from seed dormancy to germination in Arabidopsis [ <a href="#pone.0145424.ref054">54</a>]. In rice, expression of miR159 was detected in various organs except in seeds. However, its target genes, <i>OsGAMYBL1</i> and <i>OsGAMYB</i>, which regulate almost all GA-regulated genes in aleuronic cells, are highly expressed in the seed aleurone layer and promote storage hydrolysis during seed germination [ <a href="#pone.0145424.ref034">34</a>]. The expression level of miR159 is found to be very low in the embryo of non-dormant maize seeds [ <a href="#pone.0145424.ref055">55</a>]. Unexpectedly, in this study, we identified abundant miR159, especially miR159f, in the embryos of germinating rice seeds. In addition, we observed that miR519 was insensitive to ABA or GA. Thus, the plentiful miR159 in rice embryo cannot be induced by ABA as it is in Arabidopsis. RACE analysis verified that <i>OsGAMYBL1</i> and <i>OsGAMYB</i> were both cleaved by miR159 at the canonical positions. Meanwhile, <i>OsGAMYB</i> (the ortholog of Arabidopsis MYB33) was detected to be down-regulated by RT-PCR and microarray analysis in the embryos of germinating rice seeds ( <a href="#pone.0145424.s007">S3 File</a> and <a href="#pone.0145424.g007">Fig 7A</a>). It is of interest to reveal the distinctive expression patterns of osa-miR159 and the role of osa-miR159 mediated crosstalk between ABA and GA in the embryo of germinating rice seed in future studies. </p> <p>Two other targets potentially involved in the ABA signaling were also verified by 5ʹ-RACE. One is <i>OSK3</i>, targeted by rice known miRNA osa-miR1428e-3p. <i>OSK3</i> has been identified as an important gene for starch accumulation in the early stages of rice endosperm development [ <a href="#pone.0145424.ref056">56</a>]. During seed germination, most proteins related to rice endosperm development are rapidly degraded [ <a href="#pone.0145424.ref019">19</a>, <a href="#pone.0145424.ref057">57</a>]. Similarly, <i>OSK3</i> was detected to decrease drastically by RT-PCR in seed imbibition. <i>OSK3</i> belongs to the sucrose non-fermenting related kinase 1 ( <i>SnRK1</i>) family, which is well documented as a central component of the regulatory response to glucose starvation and participates in ABA signaling pathways [ <a href="#pone.0145424.ref058">58</a>, <a href="#pone.0145424.ref059">59</a>]. However, RT-PCR results showed that <i>OSK3</i> was insensitive to ABA or GA in the embryo of germinating rice seed, suggesting that the function of <i>OSK3</i> might be regulated by ABA at the post-translational level. In contrast, osa-miR1428e-3p was induced significantly by GA, implying that GA displayed antagonistic role to ABA through up-regulating the inhibitor of the positive regulator in ABA signaling. Gene <i>LOC_Os11g37540</i> encoding a PP2C domain containing protein was predicted to be targeted by a novel miRNA OsmiR-203. Rice has 90 PP2C proteins, and most of them participate in ABA signaling by interacting with SnRKs and suppressing SnRKs activation through dephosphorylation of Ser/Thr residues in the activation loop [ <a href="#pone.0145424.ref058">58</a>, <a href="#pone.0145424.ref060">60</a>]. However, the RT-PCR results indicated that <i>LOC_Os11g37540</i> was insensitive to ABA or GA, and OsmiR-203 was slightly increased by GA but inhibited by ABA. These observations reflect the complex crosstalk between ABA and GA mediated by miRNAs in controlling rice seed germination. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Abundance of conserved miRNAs in three miRNA datasets of 0, 12 and 24 HAI rice seed embryos. </div> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0145424.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Verification of 35 identified miRNAs by stem-loop RT-PCR.</div> <p>Lanes 3–19 are the 17 known rice miRNAs: osa-miR156abcei, osa-miR156dj, osa-miR160e, osa-miR166j, osa-miR167abc, osa-miR168a, osa-miR171bcde, osa-miR171h, osa-miR319b, osa-miR390, osa-miR535, osa-miR820abc, osa-miR1428e, osa-miR1862abc, osa-miR1882e, osa-miR1883a, and osa-miR5150. Lanes 20–37 are the 18 novel miRNAs OsmiR-9, OsmiR-18, OsmiR-20, OsmiR-26, OsmiR-38, OsmiR-50, OsmiR-86, OsmiR-95, OsmiR-122, OsmiR-124, OsmiR-136, OsmiR-187, OsmiR-201, OsmiR-203, OsmiR-217, OsmiR-220, OsmiR-225, and OsmiR-230. Lane M1, 100 bp DNA ladder size marker. Lane M2, 50 bp DNA size marker. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0145424.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Fig</span>  <div class="title" tagx="title" title="title">
Hierarchical clustering analyses of all miRNAs in 0, 12 and 24 HAI rice seeds embryos.</div> <p>The bar represents the scale of the miRNAs expression levels. The detailed expression information is listed in the <a href="#pone.0145424.s011">S4B Table</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0145424.s003.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Fig</span>  <div class="title" tagx="title" title="title">
Correlation of expression of intronic or exonic miRNAs and their corresponding host genes. </div> <p>The bar represents the scale of the miRNAs expression levels, TPM values of miRNAs and GCRMA values of mRNAs were normalized between 0 and 1. The detailed information is listed in <a href="#pone.0145424.s012">S5 Table</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0145424.s004.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 File</span>  <div class="title" tagx="title" title="title">
Characteristics of the miRNA precursor candidates.</div> <p>Minimal folding free energy <b>(Figure A)</b>, size <b>(Figure B)</b> and GC% content <b>(Figure C)</b> distributions of the miRNA precursor candidates. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0145424.s005.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 File</span>  <div class="title" tagx="title" title="title">
Expression analysis of identified miRNAs.</div> <p>Expression profiling of known miRNAs in seven libraries ( <b>Table A</b>). Normalizating the expression of all miRNAs by Log2TPM for heatmap ( <b>Table B</b>). All 178 differentially expressed miRNAs ( <b>Table C</b>). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s006.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 File</span>  <div class="title" tagx="title" title="title">
Expression profiles of miRNAs and their corresponding targets.</div> <p>Expression profiles of all predicted miRNA-target pairs ( <b>Table A</b>). Oppositely expressed miRNA-target pairs ( <b>Table B</b>). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s007.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Primers used in this study.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s008.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
All identified known miRNAs.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s009.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Table</span>  <div class="title" tagx="title" title="title">
All predicted novel miRNAs.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s010.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Table</span>  <div class="title" tagx="title" title="title">
Expression profiles of intronic or exonic miRNAs and their host genes.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s011.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S5 Table</span>  <div class="title" tagx="title" title="title">
Target gene prediction for all identified miRNAs using three methods.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0145424.s012.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We appreciate Rebecca Njeri Damaris’s help in English editing.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0145424.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">XJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">XQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of rice ( <i>Oryza sativa</i>) seeds during germination </span>. <span class="source" tagx="source" title="source">Proteomics</span> <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">3358</span>– <span class="lpage" tagx="lpage" title="lpage">3368</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17849412">17849412</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millar</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whelan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Ordered assembly of mitochondria during rice germination begins with promitochondrial structures rich in components of the protein import apparatus </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">201</span>– <span class="lpage" tagx="lpage" title="lpage">223</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16429260">16429260</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murcha</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenkin</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millar</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whelan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Oxygen initiation of respiration and mitochondrial biogenesis in rice</span>. <span class="source" tagx="source" title="source">J Biolog Chem</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">282</span>: <span class="fpage" tagx="fpage" title="fpage">15619</span>– <span class="lpage" tagx="lpage" title="lpage">15631</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Constructing the metabolic and regulatory pathways in germinating rice seeds through proteomic approach </span>. <span class="source" tagx="source" title="source">Proteomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">2693</span>– <span class="lpage" tagx="lpage" title="lpage">2713</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1002/pmic.201000598">10.1002/pmic.201000598</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21630451">21630451</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narsai</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carroll</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ivanova</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Usadel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Mapping metabolic and transcript temporal switches during germination in rice highlights specific transcription factors and the role of RNA instability in the germination process </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">149</span>: <span class="fpage" tagx="fpage" title="fpage">961</span>– <span class="lpage" tagx="lpage" title="lpage">980</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.108.129874">10.1104/pp.108.129874</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19074628">19074628</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakata</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Komatsu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Quantitative proteomics reveals the role of protein phosphorylation in rice embryos during early stages of germination </span>. <span class="source" tagx="source" title="source">J Proteome Res</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">1766</span>– <span class="lpage" tagx="lpage" title="lpage">1782</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1021/pr401295c">10.1021/pr401295c</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24460219">24460219</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voinnet</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Origin, biogenesis, and activity of plant microRNAs</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">669</span>– <span class="lpage" tagx="lpage" title="lpage">687</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.cell.2009.01.046">10.1016/j.cell.2009.01.046</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19239888">19239888</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eckardt</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The plant cell reviews aspects of microRNA and PhasiRNA regulatory function</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">2382</span>– <span class="lpage" tagx="lpage" title="lpage">2391</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.113.250713">10.1105/tpc.113.250713</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23897925">23897925</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hannon</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">RNA interference</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">418</span>: <span class="fpage" tagx="fpage" title="fpage">244</span>– <span class="lpage" tagx="lpage" title="lpage">251</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12110901">12110901</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">MF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reed</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis microRNA167 controls patterns of ARF6 and ARF8 expression, and regulates both female and male reproduction </span>. <span class="source" tagx="source" title="source">Development</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">4211</span>– <span class="lpage" tagx="lpage" title="lpage">4218</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17021043">17021043</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">PP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montgomery</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahlgren</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nonogaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrington</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Repression of AUXIN RESPONSE FACTOR10 by microRNA160 is critical for seed germination and post-germination stages </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2007</span>; l <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">133</span>– <span class="lpage" tagx="lpage" title="lpage">146</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17672844">17672844</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wollmann</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schommer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boisbouvier</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Sequence and expression differences underlie functional specialization of Arabidopsis microRNAs miR159 and miR319 </span>. <span class="source" tagx="source" title="source">Dev. Cell</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">115</span>– <span class="lpage" tagx="lpage" title="lpage">125</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17609114">17609114</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koyama</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitsuda</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohme-Takagi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">TCP Transcription Factors Regulate the Activities of ASYMMETRIC LEAVES1 and miR164, as Well as the Auxin Response, during Differentiation of Leaves in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">3574</span>– <span class="lpage" tagx="lpage" title="lpage">3588</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.110.075598">10.1105/tpc.110.075598</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21119060">21119060</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNA402 affects seed germination of Arabidopsis thaliana under stress conditions via targeting DEMETER-LIKE Protein3 mRNA </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">1079</span>– <span class="lpage" tagx="lpage" title="lpage">1083</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/pcp/pcq072">10.1093/pcp/pcq072</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20460498">20460498</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">XX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">YF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Construction of small RNA-mediated gene regulatory networks in the roots of rice (Oryza sativa) </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23323973">23323973</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">LQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Deep sequencing on genome-wide scale reveals the unique composition and expression patterns of microRNAs in developing pollen of Oryza sativa </span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Characterization and expression profiles of miRNAs in rice seeds</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">916</span>– <span class="lpage" tagx="lpage" title="lpage">930</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkn998">10.1093/nar/gkn998</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19103661">19103661</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watanabe</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringler</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gu</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">QXJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">RNA-sequencing reveals previously unannotated protein- and microRNA-coding genes expressed in aleurone cells of rice seeds </span>. <span class="source" tagx="source" title="source">Genomics</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">122</span>– <span class="lpage" tagx="lpage" title="lpage">134</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.ygeno.2013.10.007">10.1016/j.ygeno.2013.10.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24200500">24200500</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">In-depth proteomic analysis of rice embryo reveals its important roles in seed germination</span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">1826</span>– <span class="lpage" tagx="lpage" title="lpage">1847</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/pcp/pcu114">10.1093/pcp/pcu114</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25231964">25231964</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">L</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>. <span class="source" tagx="source" title="source">Construction of small RNA cDNA libraries for deep sequencingMethods</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">110</span>– <span class="lpage" tagx="lpage" title="lpage">117</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">BH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">XP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cobb</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Evidence that miRNAs are different from other RNAs</span>. <span class="source" tagx="source" title="source">Cell Molr Life Sci</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">63</span>: <span class="fpage" tagx="fpage" title="fpage">246</span>– <span class="lpage" tagx="lpage" title="lpage">254</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jie</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dan</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jia</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization and Identification of Novel Serum Micrornas in Sepsis Patients with Different Outcomes </span>. <span class="source" tagx="source" title="source">Shock</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">480</span>– <span class="lpage" tagx="lpage" title="lpage">487</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1097/SHK.0b013e3182940cb8">10.1097/SHK.0b013e3182940cb8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23612084">23612084</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">XB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">PX</span> </span>. <span class="mixed-article-title" title="mixed-article-title">psRNATarget: a plant small RNA target analysis server</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">155</span>– <span class="lpage" tagx="lpage" title="lpage">159</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20817928">20817928</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Addo-Quaye</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">CleaveLand: a pipeline for using degradome data to find cleaved small RNA targets</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">130</span>– <span class="lpage" tagx="lpage" title="lpage">131</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btn604">10.1093/bioinformatics/btn604</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19017659">19017659</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ridzon</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broomer</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nguyen</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Real-time quantification of microRNAs by stem-loop RT-PCR</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15640442">15640442</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakers</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruijter</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deprez</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moorman</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data </span>. <span class="source" tagx="source" title="source">Neurosci Lett</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">339</span>: <span class="fpage" tagx="fpage" title="fpage">62</span>– <span class="lpage" tagx="lpage" title="lpage">66</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12618301">12618301</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henderson</span> <span class="given-names" tagx="given-names" title="given-names">IR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Dissecting Arabidopsis thaliana DICER function in small RNA processing, gene silencing and DNA methylation patterning </span>. <span class="source" tagx="source" title="source">Nat Genet</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">721</span>– <span class="lpage" tagx="lpage" title="lpage">725</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16699516">16699516</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">BH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">XP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">QL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cobb</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Computational identification of microRNAs and their targets</span>. <span class="source" tagx="source" title="source">Comput Biol Chem</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">395</span>– <span class="lpage" tagx="lpage" title="lpage">407</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17123865">17123865</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">HY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">QQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">DNA methylation mediated by a MicroRNA pathway</span>. <span class="source" tagx="source" title="source">Mol Cell</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">465</span>– <span class="lpage" tagx="lpage" title="lpage">475</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.molcel.2010.03.008">10.1016/j.molcel.2010.03.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20381393">20381393</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mi</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">YG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodges</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Sorting of small RNAs into Arabidopsis argonaute complexes is directed by the 5 ' terminal nucleotide </span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">116</span>– <span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.cell.2008.02.034">10.1016/j.cell.2008.02.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18342361">18342361</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">QQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">HY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Rice microRNA effector complexes and targets</span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3421</span>– <span class="lpage" tagx="lpage" title="lpage">3435</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.109.070938">10.1105/tpc.109.070938</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19903869">19903869</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baskerville</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Microarray profiling of microRNAs reveals frequent coexpression with neighboring miRNAs and host genes </span>. <span class="source" tagx="source" title="source">RNA</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">241</span>– <span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15701730">15701730</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">YX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genomewide analysis of intronic microRNAs in rice and Arabidopsis</span>. <span class="source" tagx="source" title="source">J Genetics</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">313</span>– <span class="lpage" tagx="lpage" title="lpage">324</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23271017">23271017</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuji</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aya</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueguchi-Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimada</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazono</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watanabe</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">GAMYB controls different sets of genes and is differentially regulated by microRNA in aleurone cells and anthers </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">427</span>– <span class="lpage" tagx="lpage" title="lpage">444</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16792694">16792694</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slotkin</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaughn</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borges</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanurdzic</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Becker</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feijo</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Epigenetic reprogramming and small RNA silencing of transposable elements in pollen</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">461</span>– <span class="lpage" tagx="lpage" title="lpage">472</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.cell.2008.12.038">10.1016/j.cell.2008.12.038</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19203581">19203581</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elling</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">ZY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide and organ-specific landscapes of epigenetic modifications and their relationships to mRNA and small RNA transcriptomes in maize </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1053</span>– <span class="lpage" tagx="lpage" title="lpage">1069</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.109.065714">10.1105/tpc.109.065714</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19376930">19376930</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groszmann</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greaves</span> <span class="given-names" tagx="given-names" title="given-names">IK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albertyn</span> <span class="given-names" tagx="given-names" title="given-names">ZI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scofield</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peacock</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">ES</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Changes in 24-nt siRNA levels in Arabidopsis hybrids suggest an epigenetic contribution to hybrid vigor </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">2617</span>– <span class="lpage" tagx="lpage" title="lpage">2622</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1073/pnas.1019217108">10.1073/pnas.1019217108</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21266545">21266545</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galland</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huguet</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arc</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cueff</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Job</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajjou</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Dynamic proteomics emphasizes the importance of selective mRNA translation and protein turnover during arabidopsis seed germination </span>. <span class="source" tagx="source" title="source">Mol &amp;amp; Cell Proteomics</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">252</span>– <span class="lpage" tagx="lpage" title="lpage">268</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallandt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajjou</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Regulation of mRNA translation controls seed germination and is critical for seedling vigor </span>. <span class="source" tagx="source" title="source">Frontiers Plant Sci</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Computational identification of plant MicroRNAs and their targets, including a stress-induced miRNA </span>. <span class="source" tagx="source" title="source">Mol Cell</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">787</span>– <span class="lpage" tagx="lpage" title="lpage">799</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15200956">15200956</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref041" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nobuta</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venu</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vemaraju</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kulkarni</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">An expression atlas of rice mRNAs and small RNAs</span>. <span class="source" tagx="source" title="source">Nature Biotech</span>. <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">473</span>– <span class="lpage" tagx="lpage" title="lpage">477</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref042" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braun</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huntzinger</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Izaurralde</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A molecular link between miRISCs and deadenylases provides new insight into the mechanism of gene silencing by microRNAs </span>. <span class="source" tagx="source" title="source">CSH Perspect Biol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref043" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iwakawa</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomari</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Molecular Insights into microRNA-mediated translational repression in plants</span>. <span class="source" tagx="source" title="source">Mol Cell</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">591</span>– <span class="lpage" tagx="lpage" title="lpage">601</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.molcel.2013.10.033">10.1016/j.molcel.2013.10.033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24267452">24267452</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref044" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mecchia</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debernardi</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schommer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Control of cell proliferation in Arabidopsis thaliana by microRNA miR396</span>. <span class="source" tagx="source" title="source">Development</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">137</span>: <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">112</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1242/dev.043067">10.1242/dev.043067</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20023165">20023165</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref045" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curaba</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhalla</span> <span class="given-names" tagx="given-names" title="given-names">PL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">miRNAs in the crosstalk between phytohormone signalling pathways</span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">1425</span>– <span class="lpage" tagx="lpage" title="lpage">1438</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/jxb/eru002">10.1093/jxb/eru002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24523503">24523503</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref046" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">KB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">JQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Gradual increase of miR156 regulates temporal expression changes of numerous genes during leaf development in rice </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">158</span>: <span class="fpage" tagx="fpage" title="fpage">1382</span>– <span class="lpage" tagx="lpage" title="lpage">1394</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.111.190488">10.1104/pp.111.190488</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22271747">22271747</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref047" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klein</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saedler</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huijser</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A new family of DNA binding proteins includes putative transcriptional regulators of the Antirrhinum majus floral meristem identity gene SQUAMOSA </span>. <span class="source" tagx="source" title="source">Mol Gen Genet</span>. <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">250</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8569690">8569690</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref048" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Proteomics of rice seed germination</span>. <span class="source" tagx="source" title="source">Front Plant Sci</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">246</span>– <span class="lpage" tagx="lpage" title="lpage">256</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.3389/fpls.2013.00246">10.3389/fpls.2013.00246</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23847647">23847647</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref049" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holdsworth</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finch-Savage</span> <span class="given-names" tagx="given-names" title="given-names">WE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grappin</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Job</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Post-genomics dissection of seed dormancy and germination</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18160329">18160329</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref050" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Azevedo</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Betsuyaku</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peart</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noel</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sadanandom</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Role of SGT1 in resistance protein accumulation in plant immunity</span>. <span class="source" tagx="source" title="source">Embo J</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">2007</span>– <span class="lpage" tagx="lpage" title="lpage">2016</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16619029">16619029</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref051" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">QL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frank</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engel</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Mutations of the multi-drug resistance-associated protein ABC transporter gene 5 result in reduction of phytic acid in rice seeds </span>. <span class="source" tagx="source" title="source">Theor and Appl Genet</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">119</span>: <span class="fpage" tagx="fpage" title="fpage">75</span>– <span class="lpage" tagx="lpage" title="lpage">83</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19370321">19370321</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref052" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millar</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gubler</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis GAMYB-like genes, MYB33 and MYB65, are MicroRNA-regulated genes that redundantly facilitate anther development </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">705</span>– <span class="lpage" tagx="lpage" title="lpage">721</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15722475">15722475</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref053" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span>. <span class="mixed-article-title" title="mixed-article-title">ABA induction of miR159 controls transcript levels of two MYB factors during Arabidopsis seed germination </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">592</span>– <span class="lpage" tagx="lpage" title="lpage">606</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17217461">17217461</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref054" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nonogaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNA gene regulation cascades during early stages of plant development</span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">1840</span>– <span class="lpage" tagx="lpage" title="lpage">1846</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/pcp/pcq154">10.1093/pcp/pcq154</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20937608">20937608</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref055" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">LW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HB</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of maize microRNAs involved in the very early stage of seed germination </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. </span> </li> <li tag="ref"> <a name="pone.0145424.ref056" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanegae</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyoshi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuchimoto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mori</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagato</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Expressions of rice sucrose non-fermenting-1 related protein kinase 1 genes are differently regulated during the caryopsis development </span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">669</span>– <span class="lpage" tagx="lpage" title="lpage">679</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16087344">16087344</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref057" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">PF</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Gene Expression profile changes in germinating rice</span>. <span class="source" tagx="source" title="source">J Integr Plant Biol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">835</span>– <span class="lpage" tagx="lpage" title="lpage">844</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1744-7909.2011.01074.x">10.1111/j.1744-7909.2011.01074.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21910826">21910826</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref058" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghavendra</span> <span class="given-names" tagx="given-names" title="given-names">AS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonugunta</span> <span class="given-names" tagx="given-names" title="given-names">VK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christmann</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grill</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">ABA perception and signalling</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">395</span>– <span class="lpage" tagx="lpage" title="lpage">401</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.tplants.2010.04.006">10.1016/j.tplants.2010.04.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20493758">20493758</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref059" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jossier</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouly</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meimoun</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arjmand</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lessard</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hawley</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">SnRK1 (SNF1-related kinase 1) has a central role in sugar and ABA signalling in Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">316</span>– <span class="lpage" tagx="lpage" title="lpage">328</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313X.2009.03871.x">10.1111/j.1365-313X.2009.03871.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19302419">19302419</a> </span> </span> </li> <li tag="ref"> <a name="pone.0145424.ref060" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giri</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapoor</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Protein phosphatase complement in rice: genome-wide identification and transcriptional analysis under abiotic stress conditions and reproductive development </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">435</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2164-11-435">10.1186/1471-2164-11-435</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20637108">20637108</a> </span> </span> </li> </ul> </div> </div>  </body></html>