<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5384091">5384091</a> </span> <span class="pii" title="pii">srep45981</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/srep45981">10.1038/srep45981</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genome-wide identification and functional prediction of cold and/or drought-responsive lncRNAs in cassava </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Shuxia</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Xiang</span> </span> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lei</span> <span class="given-names" tagx="given-names" title="given-names">Ning</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">Zhihao</span> </span> <a href="#a3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Pingjuan</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">Yuke</span> </span> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Wenquan</span> </span> <a href="#a1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">Ming</span> </span> <a href="#c1">a</a> <a href="#a1">1</a> </span> <span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">Institute of Tropical Bioscience and Biotechnology, Chinese Academy of Tropical Agricultural Sciences </span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">National Key Laboratory of Plant Molecular Genetics and National Center for Plant Gene Research (Shanghai), Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences </span> <span class="country" tagx="country" title="country">China</span> </span> <span class="citation_author_institution" id="a3">[3], <span class="institution" title="institution">Haikou Experimental Station, Chinese Academy of Tropical Agricultural Sciences</span> <span class="country" tagx="country" title="country">China</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">a</span> <span class="email" tagx="email" title="email">pengming@itbb.org.cn</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-04-04</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">45981</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-12-06</span> <span class="accepted" title="accepted">accepted: 2017-03-07</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Cold and drought stresses seriously affect cassava ( <i>Manihot esculenta</i>) plant growth and yield. Recently, long noncoding RNAs (lncRNAs) have emerged as key regulators of diverse cellular processes in mammals and plants. To date, no systematic screening of lncRNAs under abiotic stress and their regulatory roles in cassava has been reported. In this study, we present the first reference catalog of 682 high-confidence lncRNAs based on analysis of strand-specific RNA-seq data from cassava shoot apices and young leaves under cold, drought stress and control conditions. Among them, 16 lncRNAs were identified as putative target mimics of cassava known miRNAs. Additionally, by comparing with small RNA-seq data, we found 42 lncNATs and sense gene pairs can generate nat-siRNAs. We identified 318 lncRNAs responsive to cold and/or drought stress, which were typically co-expressed concordantly or discordantly with their neighboring genes. Trans-regulatory network analysis suggested that many lncRNAs were associated with hormone signal transduction, secondary metabolites biosynthesis, and sucrose metabolism pathway. The study provides an opportunity for future computational and experimental studies to uncover the functions of lncRNAs in cassava. </p> </div> </div> </div> <div class="body" title="body"> <p>Plants are sessile organisms and are constantly exposed to a wide range of environmental stresses during their life cycle. Cold and drought are the most severe abiotic stresses that seriously influence plant growth and development, and are major limiters of crop productivity worldwide <a href="#b1">1</a>. To overcome the stress-induced loss of crop yield, it is imperative to develop crop cultivars that are stress-tolerant. Plants respond and adapt to cold and/or drought stresses through complex physiological and biochemical processes that include altered gene expression levels, signal transition pathways, and cellular metabolic rate, thus acquiring resistance <a href="#b2">2</a>. Currently, numerous studies have been conducted to investigate the cold and/or drought-responsive transcriptome and regulatory networks in multiple plant species, including <i>Arabidopsis</i> <a href="#b3">3</a>, rice <a href="#b4">4</a>, maize <a href="#b5">5</a> and wheat <a href="#b6">6</a>. These studies have identified a large number of genes that are induced by cold and/or drought stresses, and are thought to function as key regulators in stress response and tolerance, such as <i>DEHYDRATION RESPONSE ELEMENT-BINDING PROTEINs</i> (DREBs) and COLD REGULATED (CORs) genes <a href="#b7">7</a> <a href="#b8">8</a>. </p> <p>Long noncoding RNAs (lncRNAs) are generally defined as RNA transcripts that length are more than 200 nucleotide (nt) but lack a coding sequence (CDS) or open reading frame (ORF) <a href="#b9">9</a>. In plants, although some lncRNAs are transcribed by RNA polymerase III or produced by plant-specific RNA polymerase IV/V, the majority of lncRNAs shows a clear signature of RNA polymerase II transcription <a href="#b10">10</a> <a href="#b11">11</a>. Some lncRNAs are transcribed from intergenic regions (lincRNAs) or introns, while others are long noncoding nature antisense transcripts (lncNATs) that overlap with protein-coding regions on the opposite strand <a href="#b12">12</a>. Early studies attributed lncRNAs to transcriptional noise, because of their low expression and primary sequence conservation compared with mRNAs. However, they have emerged as an important class of regulators in diverse biological processes across eukaryotes <a href="#b13">13</a> <a href="#b14">14</a>. Studies on the plant lncRNAs have revealed that they play key roles in almost all developmental process, including flowering time <a href="#b15">15</a>, root organogenesis <a href="#b16">16</a>, photo morphogenesis <a href="#b17">17</a>, and reproduction <a href="#b18">18</a>. In particular, plant lncRNAs are recognized as emerging regulatory components in response to abiotic stresses <a href="#b19">19</a>. However, to date, only a few plant lncRNAs with potential roles in abiotic stresses have been characterized. For examples, in <i>Arabidopsis</i>, more than 300 lncRNAs that were regulated by various stress stimuli including cold, drought, salt, heat and highlight have been detected, many of which contained conserved elements that might be responsible for the stress-responsive functions of lncRNAs <a href="#b20">20</a> <a href="#b21">21</a>. Overexpression of npc536 lead to obvious visible differences compared with wild-type plants under salt stress conditions <a href="#b21">21</a>. </p> <p>Functional analyses of lncRNAs have shown that they are involved in the transcriptional and post-transcriptional regulation of gene expression <i>via</i> a number of complex mechanisms. They can function in either cis or trans by sequence complementarity or homology with RNAs or DNAs, and/or by structure, forming molecular frames and scaffolds for assembly of macromolecular complexes <a href="#b9">9</a>. Thus far, although lncRNAs have received more attention in recent years, only a few lncRNAs have been sufficiently described in plant. Interestingly, target mimicry was a recently identified regulatory mechanism for microRNAs (miRNAs) functions and first found in plants. As an endogenous lncRNA, <i>INDUCED BY PHOSPHATE STARVATION 1 (IPS1</i>) interacts with miRNAs, which usually post-transcriptionally regulate the abundance of their mRNA targets through cleavage in plants, and function as miRNA target mimics. Pairing with a three-nucleotide bulge, <i>IPS1</i> binds to miR399 and destroys the miR399-mediated cleavage of its targets <i>PHO2</i> gene <a href="#b22">22</a>. Many endogenous miRNA target mimics have also been predicted by bioinformatics approaches <a href="#b23">23</a>. Some of target mimics of miR160 and miR166 have been experimentally confirmed for their roles in the regulation of plant development <a href="#b18">18</a> <a href="#b24">24</a>. </p> <p>To detect and discover novel lncRNAs, several strategies have been employed including both computational and experimental screenings <a href="#b25">25</a>. Recent genome-wide transcriptome analysis approaches, such as tiling arrays and next generation sequencing of full-length cDNA libraries, in model organisms have revealed that much larger portions of non-coding transcripts than previously believed. To date, by analyzing RNA-sequencing (RNA-seq) data, thousands of lncRNAs have been identified in a number of species. For examples, more than 120,000 lncRNAs were identified and annotated across 37 plant species and six algae in the Green Non-Coding Database (GreeNC Database) <a href="#b26">26</a>. In <i>Arabidopsis</i>, approximately 6500 lncRNAs were uncovered from 200 transcriptomic data sets, with either organ-specific or stress-induced expression patterns <a href="#b19">19</a>. Wang <i>et al</i>. discovered a total of 37,238 lncNATs, which associated with 70% of annotated mRNAs <a href="#b27">27</a>. In rice, by performing whole transcriptome strand-specific RNA sequencing (ssRNA-seq), 2224 lncRNAs involved in the reproductive process were verified <a href="#b18">18</a>. Similarly, by exploiting EST and RNA-seq datasets from 30 different experiments, 1,704 high-confidence lncRNAs were identified in maize <a href="#b28">28</a>. </p> <p>Cassava is one of the important root crops with stronger stress resistance, while the regulation mechanism response to cold and/or drought stress was largely unknown. The improved RNA-seq technologies make it possible for us to obtain a global view of various RNAs in response to stresses. Thus far, using molecular technologies and ‘omics’ tools, much work like identification of mRNAs and miRNAs has been performed and numbers of RNA data sets were available from various experiments conducted by different laboratories. For examples, the expression patterns of miRNAs were identified in cassava under condition of chilling, heat or drought treatment <a href="#b29">29</a> <a href="#b30">30</a>. By analyzing oligonucleotide microarray and RNA-seq data, the transcriptomic identification of candidate genes involved cold or drought stress responses have also been reported <a href="#b31">31</a> <a href="#b32">32</a> <a href="#b33">33</a> <a href="#b34">34</a> <a href="#b35">35</a>. iTRAQ-based proteomic analysis was employed to study different strategies of cassava in response to drought stress <a href="#b36">36</a>. However, these data sets have not yet been utilized to explore and study lncRNAs in cassava, and the mechanism of lncRNAs participating cold- and/or drought-tolerance remains obscure, and thus needs to be further explored. </p> <p>Here, we used a strand-specific RNA sequencing approach to investigate the genome-wide transcriptome reconfiguration of cassava challenged by low-temperature and polyethylene glycol-simulated drought stress in a high-throughput manner. We systematically identified first landscape of novel lncRNAs with a specific focus on the lncRNAs exhibited stress-regulated expression patterns. The potential function of lncRNAs and their target genes were also predicted and analyzed. Our results indicated that a number of cassava lncRNAs response to cold and/or drought stress, and enhanced our understanding of putative regulatory functions of lncRNAs in plants. </p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="experimentstoexplorecoldand/ordroughtresponseandtranscriptomesequencingofcassavaseedlings" title="sec"> <div class="title" tagx="title" title="title">
Experiments to explore cold and/or drought response and transcriptome sequencing of cassava seedlings </div> <p>To systematically identify lncRNAs related to cold and/or drought stress in cassava, we performed whole transcriptome ssRNA-seq of cassava shoot apices and young leaves from 15-day-old seedlings under low temperature (4 °C, 24 hour), PEG-simulated drought conditions (20% PEG, 6 hour) and control conditions. This protocol was chosen for two reasons: (i) we quantified the levels of transcripts of cold and/or drought responsive genes (known DREB/CBFs and CORs/RD29) using RNA derived from samples exposed to cold stress at 4 °C or drought stress for different time points, the expression of the stress-induced genes is maximal at 24 hour after cold stress and at 6 hour after PEG treatment, respectively ( <a href="#S1">Supplemental Figure S1</a>). (ii) We observed visible stem bending and leave wilting in PEG-induced drought stress treatments ( <a href="#S1">Supplemental Figure S1</a>). The phenotypic changes under cold stress were less severe than those under drought stress. These physiological differences indicated that plants under stress conditions may show significant changes in gene expression (including lncRNAs). Moreover, according to the precious study, soluble protein content of cassava leaves showed steady increase during the first 6 to 8 h 20%PEG treatment and decreased in prolonged drought stress conditions <a href="#b32">32</a>, indicating the accumulation of the stress-induced transcripts reached a high peak at 6 h PEG treatment. Hence, 24 hour cold stress treatment and 6 hour PEG-induced drought stress treatments were the sole conditions that allowed a clear discrimination of expression of stress responded transcripts. We generated three independent biological replicates in these cases. </p> </div> <div class="genome-wideidentificationofnovellncrnasincassava" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide identification of novel lncRNAs in cassava</div> <p>In total, 140 gigabases (Gb) raw reads of 125-bp length were generated of 9 samples by paired end sequencing with Illumina HiSeq 2500 machine. After trimming adapters and filtering out low quality reads, approximately 116 million clean reads with 62–64% mapping to the <i>Manihot esculenta</i> genome were obtained and were used for further analysis. To characterize cassava lncRNAs, we developed a computational identification pipeline based on whole transcriptome ssRNA-seq data ( <a href="#f1">Fig. 1</a>). The cassava transcriptome was reconstructed from all of the RNA-seq datasets using cufflink 2.0 <a href="#b37">37</a>. A total of ~7.2 million transcripts were obtained among 9 samples. Four filter processes were applied to distinguish lncRNAs from protein-coding transcript units. First, we removed transcripts that were overlapping with known protein-coding genes in sense. We totally discovered 76,069 transcripts and most of the transcripts (63.08%) were mRNAs. Second, we filtered transcripts with length &amp;lt;200nt. Then, we evaluated the coding potential of the remaining transcripts and obtained novel expressed lncRNAs. We used the Coding Potential Calculator (CPC) <a href="#b38">38</a>, Coding-Potential Assessment Tool (CPAT) <a href="#b39">39</a> and Coding-Non-Coding Index (CNCI) <a href="#b40">40</a> to predict the coding potential of each transcript. All transcripts with scores &amp;gt;0 were discarded. To guarantee the thorough elimination of protein-coding transcripts, we also employed HMMER to scan each transcript to exclude transcripts that encoded any of the known protein domains cataloged in the Pfam protein family database <a href="#b41">41</a> <a href="#b42">42</a>. Finally, after filtering out those FPKM (fragments per kilobase of transcript per million mapped reads) <a href="#b43">43</a> scores &amp;lt;0.5, which indicated infrequently expressed transcripts, and transcripts contained in only one sample, we obtained 682 reliably expressed novel lncRNAs, including 453 lincRNAs and 229 lncNATs ( <a href="#S1">Supplemental Data S1</a>). In addition, we aligned the 682 lncRNAs with GreeNC database, and found all these lncRNAs have not been annotated in cassava. </p> </div> <div class="characterizationofcassavalncrnas" title="sec"> <div class="title" tagx="title" title="title">
Characterization of cassava lncRNAs</div> <p>We characterized the basic genomic features of the obtained lncRNAs and compared these features with cassava protein-coding genes where appropriate. We found that the pericentromeric regions of most cassava chromosomes have lower density of mRNAs and lncNATs than chromosome ‘arms’, while cassava lincRNAs are more evenly distributed across chromosomes ( <a href="#f2">Fig. 2A</a>). Most (~55%) of the cassava lincRNAs and lncNATs contains two exons, while the majority of mRNAs consist of more than 10 exons ( <a href="#f2">Fig. 2B</a>). Full-length cassava lincRNAs are generally shorter than protein-coding transcripts, while the majority of lncNATs are relatively longer than mRNAs and lincRNAs ( <a href="#f2">Fig. 2C</a>). We further measured the conservation of lncRNAs through genomic alignment between the genomes of cassava and <i>Arabidopsis</i>, and obtained the conservation score of cassava lincRNAs and lncNATs ( <a href="#f2">Fig. 2D,E</a>). By comparing phastCons <a href="#b44">44</a> scores of mRNA and lncRNAs, as expected, the conservation of lncRNAs is substantially lower than that of protein coding genes. However, we found lncNATs were more conserved than lincRNAs ( <a href="#f2">Fig. 2F</a>). </p> </div> <div class="variationinlncrnaexpressionamongstresses" title="sec"> <div class="title" tagx="title" title="title">
Variation in lncRNA expression among stresses</div> <p>We then systematically estimated the expression level of all transcripts identified including lncRNAs and mRNAs using FPKM. The results showed that the majority of lincRNAs and lncNATs were expressed at similar levels and were relatively lower than mRNAs ( <a href="#f3">Fig. 3A</a>). The lower expression level and highly differentiated expression pattern of lncRNAs were also found in <i>Arabidopsis</i> and rice <a href="#b18">18</a> <a href="#b19">19</a>, suggesting that both of these characteristics are conserved for lncRNAs. To explore the transcriptional activity of lncRNAs from all samples in response to different stresses, we compared global expression levels of lncRNAs among normal, cold and drought conditions ( <a href="#S1">Supplemental Data S2</a>). Intriguingly, we found that average abundance of lncRNAs under cold stress tend to be lower as compared to control condition, while lncRNAs under drought stress exhibited a similar level of expression to that of control ( <a href="#f3">Fig. 3B</a>). Next, we identified significantly differentially expressed lncRNAs (DE-lncRNAs) under cold or drought stress as compared to normal condition ( <a href="#S1">Supplemental Data S2</a>). The results indicated that approximately 24.05% lncRNAs presented a significantly up-regulated pattern while only about 14.81% lncRNAs were significantly down-regulated under cold treatment ( <a href="#f3">Fig. 3C</a>). Similarly, nearly 9.82% lncRNAs were strongly induced and 7.78% lncRNAs were repressed by drought treatment ( <a href="#f3">Fig. 3D</a>). We further estimated the degree of the differential expression of lncRNAs based on the JS (Jensen-Shannon) score <a href="#b45">45</a>. We observed that lncRNAs have highly stress-specific expression pattern ( <a href="#f3">Fig. 3E</a>). We clustered the lncRNAs based on their expression patterns under cold stress, drought stress and control conditions. Remarkably, various expression differences of these lncRNAs were found among stress and control conditions, suggesting that these DE-lncRNAs may play an important role in cold and/or drought stress response ( <a href="#f3">Fig. 3E</a>). Finally, we surveyed the union of DE-lncRNAs response to both cold and drought stress. We found that there were 69 DE-lncRNAs under both stresses compared to control, among which, 40 lncRNAs showed an acquired expression pattern (up-regulated) for the induction of both cold and drought; and there were 198 cold-specific DE-lncRNAs and 51 drought-specific DE-lncRNAs respectively ( <a href="#f3">Fig. 3F</a>, <a href="#S1">Supplemental Data S2</a>). Taken together, we identified a total of 318 lncRNAs that respond to cold and/or drought stress. </p> </div> <div class="validationoflncrnaexpressionusingqrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
Validation of lncRNA expression using qRT-PCR</div> <p>To further characterize cold- and/or drought- responsive lncRNAs listed in <a href="#S1">supplemental data S2</a> in cassava, 12 lncRNAs under cold and/or drought conditions were randomly selected and analyzed by real-time quantitative PCR (qRT-PCR). As shown in <a href="#f4">Fig. 4</a>, the expression patterns of the stress-responsive lncRNAs as investigated by RNA-seq and qRT-PCR were relatively consistent with similar trends, suggesting that the lncRNA expression patterns based on RNA-seq data are reliable. The qRT-PCR results confirmed that 85% of lncRNAs (8 of the 12 detected lncRNAs) were cold- or drought- inducible in cassava. Among them, we found that three lncRNAs (lincRNA419, 207 and 234) were specifically up-regulated under the cold condition ( <a href="#f4">Fig. 4A,B and C</a>), while lincRNA101, 391 and 356 were uniquely up-regulated under the drought condition ( <a href="#f4">Fig. 4D,E and F</a>). The striking differences in the response pattern between the two treatments suggested that these lncRNAs may play different roles in cassava under the two types of stresses. Moreover, lincRNA28 and lincRNA105 were up-regulated after both cold and drought treatment ( <a href="#f4">Fig. 4G,H</a>), while four lncRNAs (lincRNA64, 350,182 and 392) were down-regulated under cold and/or drought treatment ( <a href="#f4">Fig. 4I–L</a>). The action mode of these lncRNAs in cassava will be further studied. </p> </div> <div class="lncrnasaspotentialmirnaprecursors" title="sec"> <div class="title" tagx="title" title="title">
LncRNAs as potential miRNA precursors</div> <p>MiRNAs are small RNAs of 20–22 nt in length, which generated via sequential cleavage of long precursor transcripts with stem-loop by Dicer-like 1 (DCL1), and play important roles in the regulation of gene expression at both transcriptional and post-transcriptional levels <a href="#b46">46</a>. By aligning miRNA precursors to the 682 lncRNAs, we identified 12 lncRNAs as 11 known cassava miRNA precursors, including miR156g, miR160d, miR166h, miR167g and miR169d ( <a href="#S1">Supplemental Data S3</a>). Among these lncRNAs, 7 lncRNAs are differentially expressed in response to cold and/or drought stress, suggesting these miRNAs may be involved in stress responses. </p> </div> <div class="functionalpredictionofseverallncrnasactingasmirnatargetmimics" title="sec"> <div class="title" tagx="title" title="title">
Functional prediction of several lncRNAs acting as miRNA target mimics</div> <p>It has been shown that lncRNAs can serve as miRNA target mimicry to interfere with the miRNA-mediated regulation of their targets in plants. Similar to the interactions of miRNAs with their targets, miRNA target mimicry also rely on the sequence-dependent interaction of miRNAs with lncRNAs, except for the bulges in the middle of miRNA-lincRNA duplexes <a href="#b22">22</a>. We predicted lncRNAs that might act as miRNA target mimicry using the algorithm developed by Wu <i>et al</i>. <a href="#b24">24</a>. In total, we found that 16 of the identified lncRNAs that may act as miRNA mimics bound by conserved miRNAs, such as miR164, miR169, miR2275 and miR1446 ( <a href="#f5">Fig. 5A</a>, <a href="#S1">Supplemental Data S4</a>). Considering the possible role of these lncRNAs in regulating the transcript levels of miRNA targets, we further investigated coordination of lncRNAs and miRNA-targeted genes expression after cold or drought treatment using RNA-seq data. Interestingly, we found lncRNAs was positively correlated with miRNA-targeted genes expression. For example, we found a cold-repressive lincRNA159, binding miRNA164 with three mismatches, coordinated with decrease of the expression of miR164-targeted <i>NAC ( <div class="u_UNKNOWN">N</div>AM, <div class="u_UNKNOWN">A</div>TAF1 </i>/ <i>2, <div class="u_UNKNOWN">C</div>UC2 </i>) genes under cold treatment ( <a href="#f5">Fig. 5B</a>). On the other hand, expression of lincRNA340 was induced by drought stress, accompanied by an increase of miR169-targeted NUCLEAR FACTOR Y (NF-Y) genes after drought treatment ( <a href="#f5">Fig. 5C</a>). It is known that miR164 and miR169 participate in regulating cold and/or drought stress responses in plants <a href="#b47">47</a> <a href="#b48">48</a>. Thus, these two miRNA-lncRNAs functional pairs might be important regulators of abiotic stresses responses. In addition, in response to both cold and drought stress, induction of lincRNA119 was positively correlated with dramatically increased the mRNA abundance of corresponding miR2275 targets ( <a href="#f5">Fig. 5D,E</a>). To further understand the relationship of miRNA target mimics and correlated miRNAs, we analyzed the expression level of miR164, miR169 and miR2275 under stress conditions by qRT-PCR, respectively. We found miR164 was significantly up-regulated by cold and drought stress, especially after cold treatment, while miR169 and miR2275 exhibited decreased expression pattern during all stress conditions ( <a href="#f5">Fig. 5F</a>). This result is consistent with the expression pattern of miRNA targets and miRNA target mimics. Taken together, these results strengthen our assumption that lncRNAs may be involved in inhibiting the functions of miRNAs. </p> </div> <div class="identificationofstressesresponsivelncnat-sirnasincassava" title="sec"> <div class="title" tagx="title" title="title">
Identification of stresses responsive lncNAT-siRNAs in cassava</div> <p>Previous research has suggested that NAT, which pairs of endogenous coding or non-coding RNAs with perfect complementarity to each other, may generate a double stranded RNA duplex to produce siRNAs <a href="#b49">49</a> <a href="#b50">50</a>. Nat-siRNAs has been documented to regulate gene expression in plants <a href="#b51">51</a>. Since nat-siRNAs generated from pairs of endogenous coding RNAs has been investigated in cassava <a href="#b52">52</a>, the question arises whether the lncNAT-siRNAs from lncNAT and sense RNA pairs are indeed produced in response to cold and/or drought stress. To investigate abiotic stress effects on siRNA generation from lncNAT pairs, we used public small RNA-seq data of three kinds of cold treatment, including chilling acclimation (CA, 24 to 14 °C), chilling stress after chilling acclimation (CCA, 14 to 4 °C), chilling shock (CS, 24 °C to 4 °C) and normal control (NC, 24 °C) <a href="#b53">53</a>. This small RNA-seq data set was derived from cassava leaves exposed to low temperature, using experimental conditions similar to those used for our ssRNA-seq data. Therefore, we aligned siRNA enrichments at the gene body regions of lncNATs and sense genes. All 19- to 25-nt siRNAs mapped to lncNATs and their complementary regions were taken as potential lncNAT-siRNAs. Our results showed that a large number of lncNATs (18.34%) may serve as precursors of lncNAT-siRNAs under CCA and/or CS conditions ( <a href="#S1">Supplemental Data S5</a>). Among them, 45.2% of lncNATs were significantly regulated by cold and/or drought stress ( <a href="#t1">Table 1</a>). The total and unique mapped lncNAT-siRNAs reads had lengths peaked at 21-nt, 22 nt and 24-nt ( <a href="#f6">Fig. 6A,B</a>), which is well consistent with previous studies <a href="#b52">52</a>. Focusing on cold-regulated lncNATs, for example, a total of 2127 siRNAs reads aligned to the overlap region of the two lncNAT pairs, lncNAT14-Manes.05G207400 and lncNAT179-Manes.14G040500, arranged in a convergent fashion ( <a href="#f6">Fig. 6C,D</a>, <a href="#t1">Table 1</a>). Specifically, siRNAs from lncNAT14 pairs were induced under CCA treatment compared to NC ( <a href="#f6">Fig. 6C</a>). These results suggest that siRNA-mediated regulation of lncNATs pairs may contribute to cold signaling pathways. </p> </div> <div class="functionalpredictionofstress-responsivelincrnatargetsincis-regulatoryrelationships" title="sec"> <div class="title" tagx="title" title="title">
Functional prediction of stress-responsive lincRNA targets in cis-regulatory relationships</div> <p>There are relatively few functionally characterized lncRNAs in cassava. Some mammalian lncRNAs can function in cis as scaffolds to recruit co-activator complexes that mediate chromosome looping between enhancer and promoter regions, controlling chromatin topology and modulating gene expression at the transcriptional level <a href="#b54">54</a>. Emerging evidence showed that lncRNAs tend to influence the expression of adjacent genes or overlapping genes <a href="#b55">55</a> <a href="#b56">56</a>. To investigate the possible functions of cassava lncRNAs, we predicted the potential targets of lincRNAs in cis-regulatory relationships. We searched for known protein-coding genes located within 10 Kb upstream and downstream of all the identified cassava lincRNAs. Interestingly, we found most of lincRNAs (76.6%) that were transcribed close to 524 protein-coding neighbors ( <a href="#S1">Supplemental Data S6</a>). We classified stress-responsive lincRNAs-neighbors pairs into two groups: concordant regulated group and discordant regulated group. For concordant lincRNAs-neighbors pairs, the transcripts abundance of neighbor genes should change in the same direction with its lincRNAs partner no less than 1.5 fold. For discordant pairs, the expression of adjacent genes should change in the opposite direction no less than 1.5 fold. We identified 98 and 81 concordant lincRNAs-neighbors pairs potentially involved in cold and drought stress response pathways, respectively. Interestingly, we also found 40 and 35 cold- and drought-responsive discordant lincRNAs-neighbors pairs, respectively ( <a href="#f7">Fig. 7A,B</a>, <a href="#S1">Supplemental Data S6</a>). We carried out qRT-PCR to confirm the transcription of five lincRNAs and their adjacent protein-coding genes under cold and/or drought stress conditions ( <a href="#f7">Fig. 7C</a>). The results showed that the regulation of these discordant lincRNAs pairs displayed a similar pattern as RNA-seq data ( <a href="#f7">Fig. 7D–I</a>). For examples, the expression of Manes.09G081500, coding a protein tyrosine kinase, was specifically down-regulated after 6 to 72 h of drought treatment accompanied by its up-regulated adjacent lincRNA356 ( <a href="#f7">Fig. 7C,G</a>). During cold treatment, this discordant pair did not show more than two fold expression differences. Moreover, Manes.15G059900 and Manes.15G060000 were transcribed oppositely with their adjacent lincRNA105 under cold and drought stress ( <a href="#f7">Fig. 7C,F and I</a>). It will be interesting to check the function of this lincRNAs in abiotic stress responses. Therefore, a number of discordant lincRNA-neighbor pairs that were identified suggested a potentially widespread occurrence of negative regulation by lincRNAs. </p> </div> <div class="functionalpredictionofprotein-codinggenesco-expressedwithde-lncrnasintrans-regulatoryrelationships" title="sec"> <div class="title" tagx="title" title="title">
Functional prediction of protein-coding genes co-expressed with DE-lncRNAs in trans-regulatory relationships </div> <p>To further understand the function of stress-responsive lncRNAs, we performed co-expression analysis to identify trans-regulatory network of lncRNAs. In total, there were 12651 and 11133 protein-coding genes that were co-expressed with cold and drought responsive lncRNAs, respectively ( <a href="#S1">Supplemental Data S7</a>). GO (Gene Ontology) analysis of the genes in lncRNA trans-regulatory network was performed to explore their functions. We found 45 GO terms that were significantly enriched (P &amp;lt; 0.05), and 6 of these terms were associated with both cold and drought stress. For example, the top 10 enriched terms included sequence-specific DNA binding transcription factor activity, monooxygenase activity, tetrapyrrole binding activity, protein kinase activity, and phosphotransferase activity ( <a href="#S1">Supplemental Figure S2</a>). We also analyze the statistical enrichment of differentially expressed lncRNA target genes in KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway. Based on the results of significantly differentially expressed lncRNAs analysis, 8 and 6 pathways were found responsive to cold and drought stress, respectively ( <a href="#f8">Fig. 8A,B</a>, <a href="#S1">Supplemental Data S8</a>). The most enriched pathways including plant hormone signal transduction, starch and sucrose metabolism, plant-pathogen interaction, biosynthesis of secondary metabolites and metabolic pathways. These results suggest that one of the principal roles of lncRNAs may be transcriptional regulation of gene expression, especially genes that involved in hormone signal transduction and secondary metabolic pathways. These findings provide a comprehensive view of cassava lncRNAs, which will enable in-depth functional analysis. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Understanding the mechanism of gene regulation will provide molecular basis for the resistance research of cassava, contributing to make cassava better adapt to cold and/or drought stress. Over the past decade, emerging evidence reflected that the existence of long non-coding RNAs revealed the complexity of eukaryotic genome expression <a href="#b9">9</a>. Recently, with the advance of next-generation sequencing technology, many novel non-coding RNA transcripts have been found in different species <a href="#b18">18</a> <a href="#b19">19</a> <a href="#b26">26</a> <a href="#b28">28</a> <a href="#b57">57</a> <a href="#b58">58</a> <a href="#b59">59</a>. However, although plants exhibit complicated physiological, biochemical and molecular responses to cold and/or drought stress, a genome-wide identification and characterization of known and novel lncRNAs involved in these responses is still lacking, especially in cassava. In the research, we systematically identified and analyzed cassava lncRNAs to find novel lncRNAs dynamically regulated by cold and/or drought stress with ssRNA-seq method. This is the first work to globally identify lncRNAs that respond to cold and/or drought stress in cassava. Based on strict screen criteria, we totally identified 682 high-confidence lncRNAs, of which 453 were lincRNAs and 229 were lncNATs. The number of lncRNAs was far less than that of lncRNAs identified in <i>Arabidopsis</i> or rice, which may be due to the rigorous filtration criteria we used to identify lncRNAs. These lncRNAs have a median length of 1899 nucleotides, and usually have 2–3 exons. By analyzing the expression patterns of lncRNAs, we found most lncRNAs were expressed at low levels compared with mRNAs. Our analysis generated a relatively robust list of potential lncRNAs for cassava, which will be useful for functional genomics research. </p> <p>To understand the crosstalk and specificity of the responses of cassava to cold and drought stress, we performed the integrated analysis of the effects of the two stresses on the levels of lncRNA transcripts. Finally, a total of 318 cold and/or drought stress responsive lncRNAs, which were significantly up-regulated or down-regulated, were identified from RNA-seq data. We further identified 69 common differentially expressed lncRNAs, 198 cold-specific DE-lncRNAs and 51 drought-specific DE-lncRNAs, respectively. For example, lincRNA419 were specifically up-regulated under cold stress and unchanged under drought stress, whereas the lincRNA101 was specifically identified under drought stress. Unexpectedly, we also noticed that the drought responsive lncRNAs was less than the cold responsive ones. These differences in expression of lncRNAs indicated that different lncRNAs members executed different functions to cope with various stresses. </p> <p>One important class of noncoding RNAs is lncRNAs that act as target mimics of miRNAs, which were first studied in plant <a href="#b22">22</a>. Consequently, several studies found that lncRNAs function as putative miRNAs target mimics by using computational methods in plant <a href="#b23">23</a> <a href="#b24">24</a>. In our study, we globally analyzed the regulatory network of miRNAs. Using bioinformatics, we found that 12 lncRNAs act as the precursor of miRNAs, and a portion of lncRNAs could not be directly cleaved by the miRNAs due to the existence of three nucleotide bulges at the 10th nucleotide positions of the miRNA-lncRNA pairing site. We have predicted 16 lncRNAs act as potential target mimics of conserved miRNAs in cassava. By analyzing the expression of lincRNAs candidates and miRNAs targets, two of these cold and/or drought stress-related lincRNAs (linRNA159 and 340) were predicted to be target mimics of miR164 and miR169, respectively. It has been reported that a decrease in miR164-targeted NAC genes cause drought tolerance in plant and that miR169-targeted NF-Y genes also plays a role in abiotic stress responses <a href="#b47">47</a> <a href="#b48">48</a>. Considering the coordinate expression pattern of linRNAs and miR164/169 targets, it is intriguing to associate these two lincRNAs with the important functions of miR164/miR169 in regulating cold and/or drought stress responses. </p> <p>The sensitive strand-specific RNA library construction protocol enabled us to identify many novel lncRNAs transcribed from antisense strand of protein-coding genes. Compared with previous genomic studies, our results represent more comprehensive findings to date and extend our knowledge of antisense transcripts in plants. Among a total of 229 lncNAT candidates, we found 153 cold and/or drought responsive lncNATs. This observation indicated these lncNATs may be very important to the gene regulation in response to cold and/or drought stress. Meanwhile, by comparing small RNAs associated with lncNAT pairs, we observed dozens of cold and/or drought responsive lncNAT pairs may serve as precursors of siRNAs ( <a href="#t1">Table 1</a>, <a href="#f6">Fig. 6</a>, <a href="#S1">Supplemental Data S5</a>), suggesting that these lncNATs may function through short siRNAs. </p> <p>Emerging evidence revealed that lncRNAs could regulate various stages of gene expression in cis. Cis-acting lncRNAs were reported to control the expression of genes that are positioned in the vicinity of their transcription sites55. Further analyses provided evidence for the importance of lncRNA also act as long-distance regulatory elements. In maize, the conserved non-coding RNA <i>Vgt1 (Vegetative to generative transition1</i>), located 70 kb upstream of the <i>ZmRap2</i> gene and influenced the expression of <i>ZmRap2</i> <a href="#b28">28</a>. In this study, we conducted a genome-wide investigation of the lincRNAs and protein-coding transcripts which adjacent to lincRNAs (&amp;lt;10 kb). We found that the expression of many lincRNA was correlated with their neighboring protein-coding genes, which showed differentially expression pattern under stress conditions. This phenomenon has also been proved by qRT-PCR. For example, lincRNA356 is up-regulated by drought treatment, while its neighboring genes (Manes.09G081500) was down-regulated, suggesting the potential for antagonistic expression of this lincRNA and the nearby coding gene. Our findings, together with previous reports in plant, provide a starting point for future investigation into the molecular mechanisms and regulatory functions of lncRNAs. Considering that more and more trans-acting lncRNAs have also been discovered, which can regulate gene expression at independent loci, we constructed a gene co-expression network of stress-responsive lncRNAs and coding genes. Based on the GO and KEGG enrichment analyses, we found that many genes encoded involved in hormone signal transduction and secondary metabolic pathways were differentially expressed under cold and/or drought stress. However, we noticed that a great number of DE-lncRNAs and correlated DEGs involved in the plant-pathogen interaction and starch and sucrose metabolism pathways were specifically enriched in the cold treatment, while lncRNAs-correlated DEGs belonged to metabolic pathways were specifically modulated by drought stress. These findings support the above viewpoint regarding the functional specialization of lncRNAs when encountering different stresses. Taken together, our results suggest that cassava implements divergent mechanisms to modulate the response to various environmental stimuli. The function of these lncRNAs should be further investigated. </p> <p>In summary, our study provides a comprehensive resource for functional investigation of lncRNAs under stress conditions, and also complements the reference genome annotation of cassava, which might further aid the gene cloning and trigger more functional studies on lncRNAs in cassava. The discovery of lncRNAs has filled gaps in our knowledge of cold and/or drought regulatory pathways. Although increasing number of the lncRNAs was identified and characterized, their roles in regulation of targeted genes were far from being understood in many plant species. By predicting lncRNA targets, we provided a set of research clues concerning the potential roles of the lncRNAs related to the signaling regulatory network under cold and/or drought stress condition. Therefore, the lncRNA investigation may provide new insights into complicate regulations of abiotic responses and thus could be conducted for improvements in cassava yield. </p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialsandstresstreatment" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and stress treatment</div> <p>Cassava ( <i>Manihot esculenta</i>) cultivar (TMS60444) was used in this study. Their stems were cut into approximately 1.5 cm in length with one buds and planted in MS plate for two weeks in a greenhouse at 26 ± 2 °C, with a photoperiod of 16 h light and 8 h dark. Plants with a uniform growth status (about 5 cm height and with 2–3 leaves) were transferred to a chamber for cold treatment at 4 °C under light. The shoot apices and youngest expanded leaf were collected after 1 h, 6 h, 12 h, 24 h, 48 h and 72 h of treatment, and then were frozen in liquid nitrogen and held at −80 °C for RNA extraction. For drought treatment, seedlings were treated with 20% PEG6000 and harvested at 1 h, 3 h, 6 h, 12 h, 24 h and 48 h after treatment. Cassava plants used for RNA-seq sequencing were planted as above, but cold and drought stress treated only at 24 h and 6 h time point, respectively. In all cases, parallel and untreated plants at the same stage were used as controls. More than ten plants were harvested and pooled for each time point, and the collection was repeated three times as biological replicates. </p> </div> <div class="wholetranscriptomelibraryconstructionandhigh-throughputsequencing" title="sec"> <div class="title" tagx="title" title="title">
Whole transcriptome library construction and high-throughput sequencing</div> <p>The total RNA isolation, whole transcriptome libraries preparation and deep sequencing were performed by the Annoroad Gene Technology Corporation (Beijing, PR China). Whole transcriptome libraries were constructed using TruSeq Stranded Total RNA with Ribo-Zero Gold (Illumina, San Diego, CA, USA) for rRNA depletion according to the manufacturer’s instructions. The total RNA-seq libraries were sequenced initially on a HiSeq 2500 instrument that generated paired-end reads of 125 nucleotides. </p> </div> <div class="bioinformaticsidentificationofcassavalncrnas" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics identification of cassava lncRNAs</div> <p>Reads with more than 10% N (Unable to determine base information), with adapter sequence, or low quality were removed from the raw reads to obtain clean reads for further analysis. Cleaned reads were aligned to the cassava reference genome using Tophat 2.0 program with parameter "–library-type fr-firststrand" <a href="#b60">60</a>. After the alignment, Cufflinks <a href="#b37">37</a> was employed to assemble reads into transcripts according to the instructions provided. The assembled transcripts detected in two or more samples were selected for further analysis. The number of fragments per kilobase per million mapped reads (FPKM) per gene was calculated <a href="#b43">43</a>. Next, we discarded transcripts that overlapped with known protein-coding genes on the same strand, transcripts with FPKM scores &amp;lt;0.5 and transcripts shorter than 200 nt. We used the CPC <a href="#b38">38</a>, CPAT <a href="#b39">39</a> and CNCI <a href="#b40">40</a> to filter transcripts with coding potential. The remaining transcripts with known protein domains were excluded by Pfam Scan according to Pfam HMM <a href="#b41">41</a> <a href="#b42">42</a>. The transcripts that remained were considered reliably expressed lncRNAs. Signifiant differently expressed lncRNAs between cold or drought stresses and normal condition were extracted. The cassava mRNA, lincRNAs, lncNATs were aligned to the cassava genome separately to obtain the distribution of all these kinds of transcripts along chromosome with a summarized size of every 1 Mb. </p> </div> <div class="quantitativerealtimepcr(qrt-pcr)validationoflncrnasandgenes" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real time PCR (qRT-PCR) validation of lncRNAs and genes</div> <p>Total RNA were isolated respectively from cassava shoot apices and youngest expanded leaf after stress treatments for qRT-PCR using the RNAiso reagent (OMEGA). First-strand cDNA was reverse transcribed by PrimeScriptTM RT reagent Kit with gDNA Eraser (Takara). The qRT-PCR was performed using SYBR Premix Ex TaqTM (Takara). The <i>MeACTIN</i> was used as the reference gene and all the primers used were as listed in <a href="#S1">Supplemental Data S9</a>. Quantification of lncRNA and gene expression was performed using the comparative Ct method, and the specificity of the amplified product was evaluated by melting curve. This experiment was performed three technical replicates and three biological replicates. </p> </div> <div class="predictionoflncrnasasmirnatargetmimics" title="sec"> <div class="title" tagx="title" title="title">
Prediction of lncRNAs as miRNA target mimics</div> <p>All lincRNAs and lncNATs candidates were used to predict miRNAs mimic target sites according to the standard of miR399 target mimic sites in <i>IPS1</i>. That is, the pairing is less than 4 mismatches between miRNA and lncRNA and is interrupted by a 3 nt mismatched loop at the expected miRNA cleavage site <a href="#b22">22</a> <a href="#b24">24</a>. Mature cassava miRNAs were downloaded from miRBase database ( <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>). </p> </div> <div class="analysisofsmallrnasderivedfromlncnats" title="sec"> <div class="title" tagx="title" title="title">
Analysis of small RNAs derived from lncNATs</div> <p>The 4 small RNAs dataset from samples under NC, CA, CCA and CS condition were downloaded from GEO under accession number GSE52178. The small RNAs were mapped to lncNATs using STAR program <a href="#b61">61</a>. Abundance of small RNAs was normalized to reads per 6 million (RP6M). The length distribution of total and unique small RNAs from lncNATs and sense genes was plotted respectively, and the small RNAs distribution along given pairs of lncNATs and sense genes were visualized using customized script. </p> </div> <div class="geneontology(go)andkyotoencyclopediaofgenesandgenomes(kegg)enrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis</div> <p>GO enrichment analysis of protein-coding genes correlated with lincRNAs was implemented by the GO seq R package <a href="#b62">62</a>. KOBAS software was used for testing the statistical enrichment of differential expressed lncRNA target genes in KEGG pathways ( <a href="http://www.genome.ad.jp/kegg/">http://www.genome.ad.jp/kegg/</a>) <a href="#b63">63</a>. </p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional Information</div> <p> <b>How to cite this article:</b> Li, S. <i>et al</i>. Genome-wide identification and functional prediction of cold and/or drought-responsive lncRNAs in cassava. <i>Sci. Rep.</i> <b>7</b>, 45981; doi: 10.1038/srep45981 (2017). </p> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div>  <div class="media" title="media"> <a href="srep45981-s1.pdf">LINK</a> </div> </div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Figure</div>  <div class="media" title="media"> <a href="srep45981-s2.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This work was supported by the grants from the National Natural Science Foundation of China (31561143012), the Program of Hainan Association for Science and Technology Plans to Youth R &amp;amp; D Innovation (HAST201627), and the Hainan Province Innovative Research Team Foundation (2016CXTD013). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="b1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptional regulatory networks in cellular responses and tolerance to dehydration and cold stresses </span>. <span class="source" tagx="source" title="source">Annual review of plant biology</span> <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">781</span>– <span class="lpage" tagx="lpage" title="lpage">803</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.57.032905.105444">10.1146/annurev.arplant.57.032905.105444</a> </span> ( <span class="year" tagx="year" title="year">2006</span>). </span> </li> <li tag="ref"> <a name="b2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptional regulatory networks in response to abiotic stresses in Arabidopsis and grasses </span>. <span class="source" tagx="source" title="source">Plant physiology</span> <span class="volume" tagx="volume" title="volume">149</span>, <span class="fpage" tagx="fpage" title="fpage">88</span>– <span class="lpage" tagx="lpage" title="lpage">95</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.108.129791">10.1104/pp.108.129791</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19126699">19126699</a> </span> </span> </li> <li tag="ref"> <a name="b3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray </span>. <span class="source" tagx="source" title="source">The Plant journal: for cell and molecular biology</span> <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">292</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12164808">12164808</a> </span> </span> </li> <li tag="ref"> <a name="b4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rabbani</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Monitoring expression profiles of rice genes under cold, drought, and high-salinity stresses and abscisic acid application using cDNA microarray and RNA gel-blot analyses </span>. <span class="source" tagx="source" title="source">Plant physiology</span> <span class="volume" tagx="volume" title="volume">133</span>, <span class="fpage" tagx="fpage" title="fpage">1755</span>– <span class="lpage" tagx="lpage" title="lpage">1767</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.103.025742">10.1104/pp.103.025742</a> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14645724">14645724</a> </span> </span> </li> <li tag="ref"> <a name="b5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernandes</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morrow</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casati</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walbot</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Distinctive transcriptome responses to adverse environmental conditions in Zea mays L </span>. <span class="source" tagx="source" title="source">Plant biotechnology journal</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">782</span>– <span class="lpage" tagx="lpage" title="lpage">798</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1467-7652.2008.00360.x">10.1111/j.1467-7652.2008.00360.x</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18643947">18643947</a> </span> </span> </li> <li tag="ref"> <a name="b6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gulick</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome comparison of winter and spring wheat responding to low temperature</span>. <span class="source" tagx="source" title="source">Genome</span> <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">913</span>– <span class="lpage" tagx="lpage" title="lpage">923</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1139/g05-039">10.1139/g05-039</a> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16391697">16391697</a> </span> </span> </li> <li tag="ref"> <a name="b7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarwal</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reddy</span> <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sopory</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Role of DREB transcription factors in abiotic and biotic stress tolerance in plants</span>. <span class="source" tagx="source" title="source">Plant cell reports</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1263</span>– <span class="lpage" tagx="lpage" title="lpage">1274</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-006-0204-8">10.1007/s00299-006-0204-8</a> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16858552">16858552</a> </span> </span> </li> <li tag="ref"> <a name="b8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Msanne</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stone</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Awada</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of abiotic stress-responsive Arabidopsis thaliana RD29A and RD29B genes and evaluation of transgenes </span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">234</span>, <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">107</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-011-1387-y">10.1007/s00425-011-1387-y</a> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21374086">21374086</a> </span> </span> </li> <li tag="ref"> <a name="b9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chekanova</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Long non-coding RNAs and their functions in plants</span>. <span class="source" tagx="source" title="source">Current opinion in plant biology</span> <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">207</span>– <span class="lpage" tagx="lpage" title="lpage">216</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pbi.2015.08.003">10.1016/j.pbi.2015.08.003</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26342908">26342908</a> </span> </span> </li> <li tag="ref"> <a name="b10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wierzbicki</span> <span class="given-names" tagx="given-names" title="given-names">A. T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haag</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pikaard</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Noncoding transcription by RNA polymerase Pol IVb/Pol V mediates transcriptional silencing of overlapping and adjacent genes </span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">135</span>, <span class="fpage" tagx="fpage" title="fpage">635</span>– <span class="lpage" tagx="lpage" title="lpage">648</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2008.09.035">10.1016/j.cell.2008.09.035</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19013275">19013275</a> </span> </span> </li> <li tag="ref"> <a name="b11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. C.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y. Q.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Long noncoding RNAs: new regulators in plant development</span>. <span class="source" tagx="source" title="source">Biochemical and biophysical research communications</span> <span class="volume" tagx="volume" title="volume">436</span>, <span class="fpage" tagx="fpage" title="fpage">111</span>– <span class="lpage" tagx="lpage" title="lpage">114</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbrc.2013.05.086">10.1016/j.bbrc.2013.05.086</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23726911">23726911</a> </span> </span> </li> <li tag="ref"> <a name="b12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mattick</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rinn</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Discovery and annotation of long noncoding RNAs</span>. <span class="source" tagx="source" title="source">Nature structural &amp;amp; molecular biology</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">5</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nsmb.2942">10.1038/nsmb.2942</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Werven</span> <span class="given-names" tagx="given-names" title="given-names">F. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcription of two long noncoding RNAs mediates mating-type control of gametogenesis in budding yeast </span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">150</span>, <span class="fpage" tagx="fpage" title="fpage">1170</span>– <span class="lpage" tagx="lpage" title="lpage">1181</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2012.06.049">10.1016/j.cell.2012.06.049</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22959267">22959267</a> </span> </span> </li> <li tag="ref"> <a name="b14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flynn</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">H. Y.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Long noncoding RNAs in cell-fate programming and reprogramming</span>. <span class="source" tagx="source" title="source">Cell stem cell</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">752</span>– <span class="lpage" tagx="lpage" title="lpage">761</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.stem.2014.05.014">10.1016/j.stem.2014.05.014</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24905165">24905165</a> </span> </span> </li> <li tag="ref"> <a name="b15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heo</span> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sung</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Vernalization-mediated epigenetic silencing by a long intronic noncoding RNA</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">331</span>, <span class="fpage" tagx="fpage" title="fpage">76</span>– <span class="lpage" tagx="lpage" title="lpage">79</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1197349">10.1126/science.1197349</a> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21127216">21127216</a> </span> </span> </li> <li tag="ref"> <a name="b16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campalans</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kondorosi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crespi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Enod40, a short open reading frame-containing mRNA, induces cytoplasmic localization of a nuclear RNA binding protein in Medicago truncatula </span>. <span class="source" tagx="source" title="source">The Plant cell</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1047</span>– <span class="lpage" tagx="lpage" title="lpage">1059</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.019406">10.1105/tpc.019406</a> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15037734">15037734</a> </span> </span> </li> <li tag="ref"> <a name="b17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis noncoding RNA mediates control of photomorphogenesis by red light</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span> <span class="volume" tagx="volume" title="volume">111</span>, <span class="fpage" tagx="fpage" title="fpage">10359</span>– <span class="lpage" tagx="lpage" title="lpage">10364</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1409457111">10.1073/pnas.1409457111</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24982146">24982146</a> </span> </span> </li> <li tag="ref"> <a name="b18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide screening and functional analysis identify a large number of long noncoding RNAs involved in the sexual reproduction of rice </span>. <span class="source" tagx="source" title="source">Genome biology</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">512</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13059-014-0512-1">10.1186/s13059-014-0512-1</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25517485">25517485</a> </span> </span> </li> <li tag="ref"> <a name="b19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis uncovers regulation of long intergenic noncoding RNAs in Arabidopsis</span>. <span class="source" tagx="source" title="source">The Plant cell</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">4333</span>– <span class="lpage" tagx="lpage" title="lpage">4345</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.112.102855">10.1105/tpc.112.102855</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23136377">23136377</a> </span> </span> </li> <li tag="ref"> <a name="b20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of stress-responsive lncRNAs in Arabidopsis thaliana by integrating expression, epigenetic and structural features </span>. <span class="source" tagx="source" title="source">The Plant journal: for cell and molecular biology</span> <span class="volume" tagx="volume" title="volume">80</span>, <span class="fpage" tagx="fpage" title="fpage">848</span>– <span class="lpage" tagx="lpage" title="lpage">861</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12679">10.1111/tpj.12679</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25256571">25256571</a> </span> </span> </li> <li tag="ref"> <a name="b21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben Amor</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Novel long non-protein coding RNAs involved in Arabidopsis differentiation and stress responses </span>. <span class="source" tagx="source" title="source">Genome research</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">57</span>– <span class="lpage" tagx="lpage" title="lpage">69</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.080275.108">10.1101/gr.080275.108</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18997003">18997003</a> </span> </span> </li> <li tag="ref"> <a name="b22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franco-Zorrilla</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Target mimicry provides a new mechanism for regulation of microRNA activity</span>. <span class="source" tagx="source" title="source">Nature genetics</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">1033</span>– <span class="lpage" tagx="lpage" title="lpage">1037</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ng2079">10.1038/ng2079</a> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17643101">17643101</a> </span> </span> </li> <li tag="ref"> <a name="b23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Target mimics: an embedded layer of microRNA-involved gene regulatory networks in plants </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">197</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-13-197">10.1186/1471-2164-13-197</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22613869">22613869</a> </span> </span> </li> <li tag="ref"> <a name="b24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Widespread long noncoding RNAs as endogenous target mimics for microRNAs in plants</span>. <span class="source" tagx="source" title="source">Plant physiology</span> <span class="volume" tagx="volume" title="volume">161</span>, <span class="fpage" tagx="fpage" title="fpage">1875</span>– <span class="lpage" tagx="lpage" title="lpage">1884</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.215962">10.1104/pp.113.215962</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23429259">23429259</a> </span> </span> </li> <li tag="ref"> <a name="b25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marker</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Experimental RNomics: identification of 140 candidates for small non-messenger RNAs in the plant Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">Current biology: CB</span> <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">2002</span>– <span class="lpage" tagx="lpage" title="lpage">2013</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12477388">12477388</a> </span> </span> </li> <li tag="ref"> <a name="b26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paytuvi Gallart</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermoso Pulido</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anzar Martinez de Lagran</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanseverino</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aiese Cigliano</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">GREENC: a Wiki-based database of plant lncRNAs</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">D1161</span>– <span class="lpage" tagx="lpage" title="lpage">1166</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1215">10.1093/nar/gkv1215</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26578586">26578586</a> </span> </span> </li> <li tag="ref"> <a name="b27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification of long noncoding natural antisense transcripts and their responses to light in Arabidopsis </span>. <span class="source" tagx="source" title="source">Genome research</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">444</span>– <span class="lpage" tagx="lpage" title="lpage">453</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.165555.113">10.1101/gr.165555.113</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24402519">24402519</a> </span> </span> </li> <li tag="ref"> <a name="b28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide discovery and characterization of maize long non-coding RNAs</span>. <span class="source" tagx="source" title="source">Genome biology</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">R40</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2014-15-2-r40">10.1186/gb-2014-15-2-r40</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24576388">24576388</a> </span> </span> </li> <li tag="ref"> <a name="b29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ballen-Taborda</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of Cassava MicroRNAs under Abiotic Stress</span>. <span class="source" tagx="source" title="source">International journal of genomics</span> <span class="volume" tagx="volume" title="volume">2013</span>, <span class="fpage" tagx="fpage" title="fpage">857986</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1155/2013/857986">10.1155/2013/857986</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24328029">24328029</a> </span> </span> </li> <li tag="ref"> <a name="b30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Conservation and divergence of microRNAs and their functions in Euphorbiaceous plants</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">981</span>– <span class="lpage" tagx="lpage" title="lpage">995</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkp1035">10.1093/nar/gkp1035</a> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19942686">19942686</a> </span> </span> </li> <li tag="ref"> <a name="b31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling of low temperature-treated cassava apical shoots showed dynamic responses of tropical plant to cold stress </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">64</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-13-64">10.1186/1471-2164-13-64</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22321773">22321773</a> </span> </span> </li> <li tag="ref"> <a name="b32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Physiological Investigation and Transcriptome Analysis of Polyethylene Glycol (PEG)-Induced Dehydration Stress in Cassava </span>. <span class="source" tagx="source" title="source">International journal of molecular sciences</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.3390/ijms17030283">10.3390/ijms17030283</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26927071">26927071</a> </span> </span> </li> <li tag="ref"> <a name="b33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Reactive oxygen species regulate leaf pulvinus abscission zone cell separation in response to water-deficit stress in cassava </span>. <span class="source" tagx="source" title="source">Scientific reports</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">21542</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep21542">10.1038/srep21542</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26899473">26899473</a> </span> </span> </li> <li tag="ref"> <a name="b34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lokko</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of an 18,166 EST dataset for cassava (Manihot esculenta Crantz) enriched for drought-responsive genes </span>. <span class="source" tagx="source" title="source">Plant cell reports</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">1605</span>– <span class="lpage" tagx="lpage" title="lpage">1618</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-007-0378-8">10.1007/s00299-007-0378-8</a> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17541599">17541599</a> </span> </span> </li> <li tag="ref"> <a name="b35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Utsumi</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis using a high-density oligomicroarray under drought stress in various genotypes of cassava: an important tropical crop </span>. <span class="source" tagx="source" title="source">DNA research: an international journal for rapid publication of reports on genes and genomes </span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">335</span>– <span class="lpage" tagx="lpage" title="lpage">345</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/dss016">10.1093/dnares/dss016</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22619309">22619309</a> </span> </span> </li> <li tag="ref"> <a name="b36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of different strategies adapted by two cassava cultivars in response to drought stress: ensuring survival or continuing growth </span>. <span class="source" tagx="source" title="source">Journal of experimental botany</span> <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">1477</span>– <span class="lpage" tagx="lpage" title="lpage">1488</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/eru507">10.1093/jxb/eru507</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25547914">25547914</a> </span> </span> </li> <li tag="ref"> <a name="b37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation </span>. <span class="source" tagx="source" title="source">Nature biotechnology</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">511</span>– <span class="lpage" tagx="lpage" title="lpage">515</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.1621">10.1038/nbt.1621</a> </span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine </span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">W345</span>– <span class="lpage" tagx="lpage" title="lpage">349</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm391">10.1093/nar/gkm391</a> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17631615">17631615</a> </span> </span> </li> <li tag="ref"> <a name="b39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">CPAT: Coding-Potential Assessment Tool using an alignment-free logistic regression model </span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">e74</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt006">10.1093/nar/gkt006</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23335781">23335781</a> </span> </span> </li> <li tag="ref"> <a name="b40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Utilizing sequence intrinsic composition to classify protein-coding and long non-coding transcripts </span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">e166</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt646">10.1093/nar/gkt646</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23892401">23892401</a> </span> </span> </li> <li tag="ref"> <a name="b41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Pfam: the protein families database</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">D222</span>– <span class="lpage" tagx="lpage" title="lpage">230</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt1223">10.1093/nar/gkt1223</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371">24288371</a> </span> </span> </li> <li tag="ref"> <a name="b42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A new generation of homology search tools based on probabilistic inference. <i>Genome informatics</i> </span>. <span class="source" tagx="source" title="source">International Conference on Genome Informatics</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">205</span>– <span class="lpage" tagx="lpage" title="lpage">211</span> ( <span class="year" tagx="year" title="year">2009</span>). </span> </li> <li tag="ref"> <a name="b43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">G. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kin</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">V. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Measurement of mRNA abundance using RNA-seq data: RPKM measure is inconsistent among samples </span>. <span class="source" tagx="source" title="source">Theory in biosciences = Theorie in den Biowissenschaften</span> <span class="volume" tagx="volume" title="volume">131</span>, <span class="fpage" tagx="fpage" title="fpage">281</span>– <span class="lpage" tagx="lpage" title="lpage">285</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12064-012-0162-3">10.1007/s12064-012-0162-3</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22872506">22872506</a> </span> </span> </li> <li tag="ref"> <a name="b44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siepel</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</span>. <span class="source" tagx="source" title="source">Genome research</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">1034</span>– <span class="lpage" tagx="lpage" title="lpage">1050</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.3715005">10.1101/gr.3715005</a> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16024819">16024819</a> </span> </span> </li> <li tag="ref"> <a name="b45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cabili</span> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses </span>. <span class="source" tagx="source" title="source">Genes &amp;amp; development</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">1915</span>– <span class="lpage" tagx="lpage" title="lpage">1927</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.17446611">10.1101/gad.17446611</a> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21890647">21890647</a> </span> </span> </li> <li tag="ref"> <a name="b46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hake</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNAs: a role in plant development</span>. <span class="source" tagx="source" title="source">Current biology: CB</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">R851</span>– <span class="lpage" tagx="lpage" title="lpage">852</span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14588265">14588265</a> </span> </span> </li> <li tag="ref"> <a name="b47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Conserved miR164-targeted NAC genes negatively regulate drought resistance in rice</span>. <span class="source" tagx="source" title="source">Journal of experimental botany</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">2119</span>– <span class="lpage" tagx="lpage" title="lpage">2135</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/eru072">10.1093/jxb/eru072</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24604734">24604734</a> </span> </span> </li> <li tag="ref"> <a name="b48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W. X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis NFYA5 transcription factor is regulated transcriptionally and posttranscriptionally to promote drought resistance </span>. <span class="source" tagx="source" title="source">The Plant cell</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">2238</span>– <span class="lpage" tagx="lpage" title="lpage">2251</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.108.059444">10.1105/tpc.108.059444</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18682547">18682547</a> </span> </span> </li> <li tag="ref"> <a name="b49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vacic</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Girke</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lonardi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Small RNAs and the regulation of cis-natural antisense transcripts in Arabidopsis</span>. <span class="source" tagx="source" title="source">BMC molecular biology</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">6</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2199-9-6">10.1186/1471-2199-9-6</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18194570">18194570</a> </span> </span> </li> <li tag="ref"> <a name="b50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of plant nat-siRNAs reveals insights into their distribution, biogenesis and function </span>. <span class="source" tagx="source" title="source">Genome biology</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">R20</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2012-13-3-r20">10.1186/gb-2012-13-3-r20</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22439910">22439910</a> </span> </span> </li> <li tag="ref"> <a name="b51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunkar</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification and analysis of small RNAs originated from natural antisense transcripts in Oryza sativa </span>. <span class="source" tagx="source" title="source">Genome research</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">70</span>– <span class="lpage" tagx="lpage" title="lpage">78</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.084806.108">10.1101/gr.084806.108</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18971307">18971307</a> </span> </span> </li> <li tag="ref"> <a name="b52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quintero</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perez-Quintero</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of ta-siRNAs and cis-nat-siRNAs in cassava and their roles in response to cassava bacterial blight </span>. <span class="source" tagx="source" title="source">Genomics, proteomics &amp;amp; bioinformatics</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">172</span>– <span class="lpage" tagx="lpage" title="lpage">181</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gpb.2013.03.001">10.1016/j.gpb.2013.03.001</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Endogenous small-noncoding RNAs and their roles in chilling response and stress acclimation in Cassava </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">634</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-634">10.1186/1471-2164-15-634</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25070534">25070534</a> </span> </span> </li> <li tag="ref"> <a name="b54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonasio</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shiekhattar</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulation of transcription by long noncoding RNAs</span>. <span class="source" tagx="source" title="source">Annual review of genetics</span> <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">433</span>– <span class="lpage" tagx="lpage" title="lpage">455</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev-genet-120213-092323">10.1146/annurev-genet-120213-092323</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Large-scale prediction of long non-coding RNA functions in a coding-non-coding gene co-expression network </span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">3864</span>– <span class="lpage" tagx="lpage" title="lpage">3878</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkq1348">10.1093/nar/gkq1348</a> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21247874">21247874</a> </span> </span> </li> <li tag="ref"> <a name="b56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification of long noncoding RNA genes and their potential association with fecundity and virulence in rice brown planthopper, Nilaparvata lugens </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">749</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1953-y">10.1186/s12864-015-1953-y</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26437919">26437919</a> </span> </span> </li> <li tag="ref"> <a name="b57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide long non-coding RNA screening, identification and characterization in a model microorganism Chlamydomonas reinhardtii </span>. <span class="source" tagx="source" title="source">Scientific reports</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">34109</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep34109">10.1038/srep34109</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27659799">27659799</a> </span> </span> </li> <li tag="ref"> <a name="b58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification and functional prediction of nitrogen-responsive intergenic and intronic long non-coding RNAs in maize (Zea mays L.) </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">350</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-016-2650-1">10.1186/s12864-016-2650-1</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27169379">27169379</a> </span> </span> </li> <li tag="ref"> <a name="b59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shuai</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in Populus trichocarpa </span>. <span class="source" tagx="source" title="source">Journal of experimental botany</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">4975</span>– <span class="lpage" tagx="lpage" title="lpage">4983</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/eru256">10.1093/jxb/eru256</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24948679">24948679</a> </span> </span> </li> <li tag="ref"> <a name="b60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks </span>. <span class="source" tagx="source" title="source">Nature protocols</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">562</span>– <span class="lpage" tagx="lpage" title="lpage">578</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2012.016">10.1038/nprot.2012.016</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036">22383036</a> </span> </span> </li> <li tag="ref"> <a name="b61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dobin</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">STAR: ultrafast universal RNA-seq aligner</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">15</span>– <span class="lpage" tagx="lpage" title="lpage">21</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a> </span> </span> </li> <li tag="ref"> <a name="b62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wakefield</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smyth</span> <span class="given-names" tagx="given-names" title="given-names">G. K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oshlack</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene ontology analysis for RNA-seq: accounting for selection bias</span>. <span class="source" tagx="source" title="source">Genome biology</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">R14</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2010-11-2-r14">10.1186/gb-2010-11-2-r14</a> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20132535">20132535</a> </span> </span> </li> <li tag="ref"> <a name="b63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span>. <span class="mixed-article-title" title="mixed-article-title">KEGG for linking genomes to life and the environment</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span> <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">D480</span>– <span class="lpage" tagx="lpage" title="lpage">484</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm882">10.1093/nar/gkm882</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18077471">18077471</a> </span> </span> </li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors declare no competing financial interests.</p> </div> <div class="fn-type-" title=""> <p> <b>Author Contributions</b> S.X.L. and M.P. devised the study, S.X.L.; X.Y.; N.L. and Z.H.C. conducted the experiments and analyses, all authors contributed to data interpretation and writing of the manuscript. </p> </div> </div> </div>  </body></html>