<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4020777">4020777</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24828743">24828743</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-14-00302</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0096342">10.1371/journal.pone.0096342</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span>: <span class="subject" title="subject">Agricultural Biotechnology</span> <span class="subject" title="subject">Crops</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Transcriptome Analysis</span>: <span class="subject" title="subject">Genome Expression Analysis</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span>: <span class="subject" title="subject">DNA transcription</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Functional Genomics</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span>: <span class="subject" title="subject">Crop Genetics</span> </div>: <span class="subject" title="subject">Mutation</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span>: <span class="subject" title="subject">Plants</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Anatomy</span>: <span class="subject" title="subject">Seeds</span> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
RNA-Seq Profiling of a Defective Seed Coat Mutation in <i>Glycine max</i> Reveals Differential Expression of Proline-Rich and Other Cell Wall Protein Transcripts </div> <div class="alt-title" title="alt-title">
RNA-Seq Profiling of Defective Seed Coat Mutation</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kour</span> <span class="given-names" tagx="given-names" title="given-names">Anupreet</span> </span> <a href="#aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boone</span> <span class="given-names" tagx="given-names" title="given-names">Anne M.</span> </span> <a href="#aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">Lila O.</span> </span> <a href="#aff1" /> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[], <span class="addr-line" title="addr-line">Department of Crop Sciences, University of Illinois, Urbana, Illinois, United States of America </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gijzen</span> <span class="given-names" tagx="given-names" title="given-names">Mark</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Agriculture and Agri-Food Canada, Canada</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">l-vodkin@illinois.edu</span> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: AK LV. Performed the experiments: AK AMB. Analyzed the data: AK AMB LV. Wrote the paper: AK LV. Collected tissue samples and prepared RNA: AK AMB. Analyzed and interpreted sequence data and drafted the manuscript: AK. Performed RNA blots: AMB. Designed initial approach, led and coordinated the project, and edited the manuscript: LV. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2014</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2014-5-5</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e96342</span> <span class="history" title="history"> <span class="received" title="received">received: 2014-1-3</span> <span class="accepted" title="accepted">accepted: 2014-4-4</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Kour et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2014</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The plant cell wall performs a number of essential functions including providing shape to many different cell types and serving as a defense against potential pathogens. The net pattern mutation creates breaks in the seed coat of soybean ( <i>Glycine max</i>) because of ruptured cell walls. Using RNA-Seq, we examined the seed coat transcriptome from three stages of immature seed development in two pairs of isolines with normal or defective seed coat phenotypes due to the net pattern. The genome-wide comparative study of the transcript profiles of these isolines revealed 364 differentially expressed genes in common between the two varieties that were further divided into different broad functional categories. Genes related to cell wall processes accounted for 19% of the differentially expressed genes in the middle developmental stage of 100–200 mg seed weight. Within this class, the cell wall proline-rich and glycine-rich protein genes were highly differentially expressed in both genetic backgrounds. Other genes that showed significant expression changes in each of the isoline pairs at the 100–200 mg seed weight stage were xylem serine proteinase, fasciclin-related genes, auxin and stress response related genes, TRANSPARENT TESTA 1 (TT1) and other transcription factors. The mutant appears to shift the timing of either the increase or decrease in the levels of some of the transcripts. The analysis of these data sets reveals the physiological changes that the seed coat undergoes during the formation of the breaks in the cell wall. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
The research was funded by support from the United Soybean Board and the United States Department of Agriculture (USDA). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Plant cells are surrounded by cell walls which provide tensile strength, mechanical support, protection from insects and pathogens, prevention of water loss, and that participate in cell to cell communication <a href="#pone.0096342-Vorwerk1">[1]</a>, <a href="#pone.0096342-Baskin1">[2]</a>, <a href="#pone.0096342-Somerville1">[3]</a>, <a href="#pone.0096342-Tsukaya1">[4]</a>. The cell walls differ in composition in different plant parts but the basic composition of the cell wall includes cellulose, hemicellulose, pectins, and small proportions of structural proteins including proline-rich and glycine-rich proteins. The seed coat is the outer protective layer of a seed and develops from the integument originally surrounding the ovule and is maternal in origin. The seed coat provides protection of embryo and endosperm from mechanical injuries, insects, bacteria, and fungi, and desiccation of the seed. Seed coats of different species vary in structure and composition including extensive differentiation of cell layers into specialized cell types. These cells may accumulate large quantities of substances including mucilage or pigments depending on plant species. </p> <p>The seed coat, or testa, of the mature soybean ( <i>Glycine max</i>) has been well characterized, and contains features in common with the majority of the legumes: an epidermal layer of palisade cells, or macrosclereids, a sub-epidermal layer of hourglass cells, or osteosclereids, a few layers of parenchyma, and an aleurone layer <a href="#pone.0096342-Corner1">[5]</a>, <a href="#pone.0096342-Miller1">[6]</a>, <a href="#pone.0096342-Williams1">[7]</a>. Soybean seeds begin to form on the plant at the R4 stage when the parent plant has between 13 and 20 leaf nodes <a href="#pone.0096342-Ritchie1">[8]</a>. At this stage, the seed is going through multiple cycles of cell division as well as tissue differentiation. Invertase in the seed coat cleaves sucrose giving rise to hexose which surrounds the embryo of the minuscule soybean seed <a href="#pone.0096342-Bewley1">[9]</a>. The seeds grow at a rapid rate between the R4 and R7 stages as they accumulate carbon, nitrogen, and seed storage proteins. Seeds increase in size from 25 mg to 500 mg fresh weight during the R5 and R6 stages. Towards the end of the R6 stage, nutrient accumulation in the seed begins to decrease. By R7 the seed has amassed almost all the dry weight it will acquire and contains approximately 60% moisture and has begun to yellow. After reaching the maximum fresh weight, near 400–500 mg, the seed readies itself for desiccation and begins to lose total fresh weight. By the R8 stage, most of the pods and seeds have browned and are dry <a href="#pone.0096342-Ritchie1">[8]</a>. </p> <p>Intact seed coats are preferred but a mutation known as the "net pattern" in soybean results in defective seed coats that have ruptured by maturity to expose the cotyledons underneath. The trait has been backcrossed to create isolines in two different genetic backgrounds; however, very little is known about the histology or molecular nature of the trait. Using RNA blots, we have previously reported a developmental delay in the decline of transcripts for a specific proline-rich protein (PRP1) of the cell wall leading to higher levels of this transcript in the defective seed coats at the middle weight range of 100–200 mg seed weight <a href="#pone.0096342-Percy1">[10]</a>. </p> <p>The effect on PRP1 transcript levels was assumed to be a downstream effect of the net pattern mutation. In this report, we present the global gene expression analysis of the net pattern cell wall mutant in soybean using transcriptome analyses of three different stages of seed development at 50–100 mg, 100–200 mg, and 400–500 mg. By comparing RNA-Seq data from the two lines containing the defective seed coat trait compared to their standard counterpart isolines using Bowtie alignments <a href="#pone.0096342-Langmead1">[11]</a> to the soybean genes models from the sequenced soybean genome <a href="#pone.0096342-Schmutz1">[12]</a>, 364 significantly differentially expressed genes in common between the two isolines were revealed, many of which were involved in cell wall processes. These data aid in determining perturbations in the physiological development of the plant cell wall which affects agronomically important traits and is a source of bioenergy <a href="#pone.0096342-PestanaCalsa1">[13]</a>. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="phenotypecomparisonsofisolinepairswithstandardanddefectiveseedcoats" title="sec"> <div class="title" tagx="title" title="title">
Phenotype Comparisons of Isoline Pairs with Standard and Defective Seed Coats</div> <p>The isolines in the background of cultivar Clark are referred to as Clark standard (CS) which exhibits an intact seed coat and Clark defective (CD) that contains the defective seed coat mutation that had been backcrossed into the Clark standard line. Likewise, Harosoy standard (HS) is compared to the Harosoy defective (HD) isoline that contains the defective seed coat mutation. The genotypes and phenotypes of these two isoline pairs is presented in <a href="#pone-0096342-g001">Figure 1</a>. As shown in <a href="#pone-0096342-g001">Figure 1</a>, all the layers of seed coat are ruptured in the mutant isolines. Genetically, mutant isolines contain an unknown gene or set of genes here designated as "Def" for defective seed coat. This abbreviation is not meant to convey an official locus designation. This mutation was transferred during the 1960's by R. L. Bernard of the USDA/ARS by selection of the trait during repetitive backcrossing into both a Clark (with a black seed background) and a Harosoy (yellow seed background). The defective seed coat phenotype is more striking on the black background and is sometimes referred to as a "net" pattern <a href="#pone.0096342-Percy1">[10]</a>. The phenotypes of the two lines along with their alleles controlling seed color <a href="#pone.0096342-Todd1">[14]</a>, <a href="#pone.0096342-Tuteja1">[15]</a>, <a href="#pone.0096342-Gillman1">[16]</a>, <a href="#pone.0096342-Zabala1">[17]</a> are shown in <a href="#pone-0096342-g001">Figure 1</a> and <a href="#pone-0096342-t001">Table 1</a>. The two lines are available from the USDA germplasm collection, Urbana, Illinois, as shown in <a href="#pone-0096342-t001">Table 1</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Comparisons of the Defective Seed Coat Mutation in Soybean.</div> <p>The genotypes and phenotypes of the isolines used in both Clark and Harosoy backgrounds are shown. Representative seed comparing Clark Standard (CS) vs. Clark Defective (CD) and Harosoy Standard (HS) vs. Harosoy Defective (HD) illustrate the defective seed coat mutation exhibiting cracks on seed coat in either a pigmented or non-pigmented genetic background. Refer to <a href="#pone-0096342-t001">Table 1</a> for additional information. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0096342.t001">10.1371/journal.pone.0096342.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary of 12 RNA-Seq Libraries Constructed from Seed Coats of Clark and Harosoy Standard and Defective Seed Coat Isolines at Three Stages of Immature Seed Development. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Isolines</td> <td>Genotype</td> <td>Phenotype</td> <td>USDA #</td> <td>Internal Lab #</td> <td>Seed wt. (mg)</td> <td>Raw reads (M)</td> <td>Sequencing platform</td> <td>Read length (bp)</td> </tr> </thead> <tbody> <tr> <td>CS</td> <td> <i>i R T</i> </td> <td>Self-black</td> <td>PI 547474</td> <td>UC9</td> <td>50–100</td> <td>52.1</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>CS</td> <td> <i>i R T</i> </td> <td>Self-black</td> <td>PI 547474</td> <td>UC9</td> <td>100–200</td> <td>62.3</td> <td>GAII</td> <td>75</td> </tr> <tr> <td>CS</td> <td> <i>i R T</i> </td> <td>Self-black</td> <td>PI 547474</td> <td>UC9</td> <td>400–500</td> <td>50.5</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>CD</td> <td> <i>i R T</i> Def </td> <td>Defective black</td> <td>Clark 5143</td> <td>UC412</td> <td>50–100</td> <td>68.5</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>CD</td> <td> <i>i R T</i> Def </td> <td>Defective black</td> <td>Clark 5143</td> <td>UC412</td> <td>100–200</td> <td>41.1</td> <td>GAII</td> <td>75</td> </tr> <tr> <td>CD</td> <td> <i>i R T</i> Def </td> <td>Defective black</td> <td>Clark 5143</td> <td>UC412</td> <td>400–500</td> <td>31.2</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>HS</td> <td> <i>I r t</i> </td> <td>Yellow</td> <td>PI 548573</td> <td>UC501</td> <td>50–100</td> <td>46.4</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>HS</td> <td> <i>I r t</i> </td> <td>Yellow</td> <td>PI 548573</td> <td>UC501</td> <td>100–200</td> <td>38.7</td> <td>GAII</td> <td>75</td> </tr> <tr> <td>HS</td> <td> <i>I r t</i> </td> <td>Yellow</td> <td>PI 548573</td> <td>UC501</td> <td>400–500</td> <td>49.2</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>HD</td> <td> <i>I r t</i> Def </td> <td>Defective yellow</td> <td>L67–2246</td> <td>UC508</td> <td>50–100</td> <td>49.4</td> <td>HiSeq2000</td> <td>100</td> </tr> <tr> <td>HD</td> <td> <i>I r t</i> Def </td> <td>Defective yellow</td> <td>L67-2246</td> <td>UC508</td> <td>100–200</td> <td>38.2</td> <td>GAII</td> <td>75</td> </tr> <tr> <td>HD</td> <td> <i>I r t</i> Def </td> <td>Defective yellow</td> <td>L67-2246</td> <td>UC508</td> <td>400–500</td> <td>37.7</td> <td>HiSeq2000</td> <td>100</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>The USDA line number and PI accession numbers by which the isolines are searchable in the USDA GRIN (Germplasm Resources Information Network) database are shown. The molecular nature is known for the <i>I</i> alleles <a href="#pone.0096342-Todd1">[14]</a>, <a href="#pone.0096342-Tuteja1">[15]</a> that control distribution of pigment and the <i>R</i> <a href="#pone.0096342-Gillman1">[16]</a>, and <i>T</i> loci <a href="#pone.0096342-Zabala1">[17]</a> that affect seed color phenotype. RNA was extracted from seed coats dissected from immature seed of the indicated fresh weight ranges. The total number of 75- or 100-nt raw reads from each RNA-Seq library is shown. The pictures of each phenotype of the defective seed coat lines are shown in <a href="#pone-0096342-g001">Figure 1</a>. The raw and processed data for the 12 sequencing samples is available as accession series GSE54903 at the National Center for Biotechnology Information. (Abbreviations: CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective). </p> </div> </div> </div> </div> <div class="transcriptomecomparisonsofclarkandharosoystandardseedcoatstodefectiveseedcoatsfromthemutantisolines" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome Comparisons of Clark and Harosoy Standard Seed Coats to Defective Seed Coats from the Mutant Isolines </div> <p>Next-generation high throughput sequencing of the transcriptome (RNA-Seq) was used and the cDNA libraries were constructed from seed coats from 50–100 mg, 100–200 mg and 400–500 mg seed weight stages in standard and mutant isolines of Clark and Harosoy cultivars. Each library was sequenced on the Illumina GAII or HiSeq 2000 platforms. The number of raw sequence reads obtained from all four isolines, at different seed weight stages are presented in <a href="#pone-0096342-t001">Table 1</a>. The 75-bp or 100-bp reads were aligned to 78,773 target Glyma models from the soybean reference genome <a href="#pone.0096342-Schmutz1">[12]</a> using Bowtie <a href="#pone.0096342-Langmead1">[11]</a>. The RNA-Seq data were normalized in reads per kilobase of gene model per million mapped reads (RPKM) considering gene length as parameter <a href="#pone.0096342-Mortazavi1">[18]</a>. </p> <p>As presented in <a href="#pone.0096342.s019">Table S1</a>, 18% and 50% of the genes were expressed in the seed coat of Clark and Harosoy isolines at the cutoff points of above 10 RPKM and 1 RPKM, respectively. For inclusion in further analyses, all genes that showed ≥5 RPKM in either standard or defective isoline as well a ≥2 fold differential expression between standard and defective isolines were included in further functional annotation analyses. An overview of the number of these genes is presented in <a href="#pone-0096342-t002">Table 2</a> which shows that a total of approximately 1,300 out of 78,773 genes in the soybean genome showed differential expression in the seed coats of either the Clark or Harosoy isolines in at least one stage of development. The p-values for differential expression were obtained by analysis with the DESeq package on hit count data from comparisons between Clark and Harosoy standard and defective isolines. The gene models that showed p-values ≤0.05 were considered as significantly differentially expressed. These differentially expressed genes along with gene model numbers, annotations, RPKM values, basemean, and p-values are presented in <a href="#pone.0096342.s020">Tables S2</a>– <a href="#pone.0096342.s022">S4</a>. The graphical presentation of expression levels of these differentially expressed genes in standard and defective isolines are presented in <a href="#pone.0096342.s001">Figures S1</a> &amp;amp; <a href="#pone.0096342.s002">S2</a>. The majority of the differentially expressed seed coat genes were expressed at less than 50 RPKM in both Clark and Harosoy backgrounds. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0096342.t002">10.1371/journal.pone.0096342.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Total Number of Differentially Expressed Genes in the Seed Coats of the Clark and Harosoy Isoline Pairs at Different Seed Weight Stages. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Genotypes</td> <td>Number of differentially expressed genes</td> <td /> </tr> <tr> <td /> <td>50–100 mg</td> <td>100–200 mg</td> <td>400–500 mg</td> <td>Total</td> </tr> </thead> <tbody> <tr> <td>Clark</td> <td>720</td> <td>173</td> <td>417</td> <td>1310</td> </tr> <tr> <td>Harosoy</td> <td>48</td> <td>156</td> <td>1068</td> <td>1272</td> </tr> <tr> <td>Both</td> <td>57</td> <td>265</td> <td>42</td> <td>364</td> </tr> <tr> <td>Opposite</td> <td>9</td> <td>4</td> <td>96</td> <td>109</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>The number of differentially expressed genes in Clark or Harosoy standard versus the defective seed coat isoline that meet the criteria of ≥2 fold differential expression, ≥5 RPKM and p-value ≤0.05 by DESeq analysis. Both: the number of differentially expressed genes shared in common between both the Clark and Harosoy isoline pairs and also the expression levels of these genes are in the same direction <i>i.e.</i> if expression of a particular gene was higher in Clark wild-type and lower in the defective isoline, it was also higher in Harosoy wild type isoline and lower in Harosoy defective isoline. Opposite: If a particular gene showed higher expression in the standard isoline of Clark compared to the Clark defective, but it showed higher expression in the defective Harosoy compared to the standard Harosoy line. </p> </div> </div> </div> </div> <div class="functionalclassificationofgenesdifferentiallyexpressedinseedcoatsofbothclarkandharosoyisolines" title="sec"> <div class="title" tagx="title" title="title">
Functional Classification of Genes Differentially Expressed in Seed Coats of both Clark and Harosoy Isolines </div> <p>As shown in <a href="#pone-0096342-t002">Table 2</a>, a total of 364 genes were differentially expressed in at least one of the three different seed weight stages in both Clark and Harosoy isolines. The detailed information on these genes is presented in <a href="#pone.0096342.s021">Table S3</a> and distribution of these genes into different broad functional categories is presented in <a href="#pone-0096342-g002">Figure 2</a>. Out of these 364 genes, there were 265 that showed significantly different expression in the 100–200 mg weight range while only 57 and 42 genes varied in the 50–100 and 400–500 mg weight ranges, respectively. A representative selection of 29 genes is presented in <a href="#pone-0096342-t003">Tables 3</a> and <a href="#pone-0096342-t004">4</a>. In addition, 248 of these 265 varied only at the 100–200 mg stage and did not show differential expression at the 50–100 or 400–500 mg weight range. Thus, the maximum number of significantly differentially expressed genes in common in both the Clark and Harosoy backgrounds are in the 100–200 mg seed weight range. In the 50–100 mg weight range, there were no differentially expressed genes with cell wall annotations ( <a href="#pone-0096342-g002">Figure 2A</a>), while at the 100–200 mg and 400–500 mg weight range there were 19% and 5% of the differentially expressed genes related to possible cell wall functions, respectively ( <a href="#pone-0096342-g002">Figures 2B and 2C</a>). The 51 cell wall related genes at the 100–200 mg stage were further divided into 10 different categories based on cell wall structural components ( <a href="#pone-0096342-g003">Figure 3</a>). The maximum number (21%) of genes fall under the fasciclin-like arabinogalactan proteins that have been reported to affect tensile strength of the cell wall <a href="#pone.0096342-MacMillan1">[19]</a>. Proline-rich proteins are the second most abundant category at 15%, while cysteine-rich proteins, expansin/extensin-related proteins, and glycine-rich proteins (GRPs) represent 10%, 8%, and 4% of the differentially expressed cell wall proteins in this developmental stage ( <a href="#pone-0096342-g003">Figure 3</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
The Distribution of Genes Differentially Expressed in Standard and Defective Seed Coats of Both Clark and Harosoy Backgrounds During Three Stages of Immature Seed Coat Development. </div> <p>The genes that were differentially expressed in both Clark and Harosoy backgrounds at seed weight ranges of either (A) 50–100 mg (B) 100–200 mg and (C) 400–500 mg. Differential expression is defined as genes with ≥5RPKM expression, ≥2 fold differential expression, and p value ≤0.05. Differential expression of cell wall related genes comprise 19% of the total genes at the 100–200 mg seed weight stage and 5% of the 400–500 mg stage. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
The Distribution of Cell wall Related Genes Differentially Expressed Between Standard and Defective Seed Coats in Both Clark and Harosoy Backgrounds at the 100–200 mg Seed Weight Stage. </div> <p>The distribution of 51 differentially expressed genes into different cell wall structural component categories is shown with genes having annotations related to fasciclin (21%) and protein rich proteins (15%) predominating. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0096342.t003">10.1371/journal.pone.0096342.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Selected Differentially Expressed Genes in the Clark Isoline Pair at the 100–200 mg Seed Weight Stage Using RPKM and DESeq Analysis of RNA-Seq Data. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>a) Overexpressed in Defective</td> <td /> <td>RNA-Seq (RPKM)</td> <td>DESeq (Base Mean)</td> </tr> <tr> <td>Model</td> <td>Annotation</td> <td>CS</td> <td>CD</td> <td>CD/CS</td> <td>CS</td> <td>CD</td> <td>p value</td> </tr> </thead> <tbody> <tr> <td>Glyma13g27050.1</td> <td>unknown</td> <td>4.90</td> <td>68.73</td> <td>14.02</td> <td>567</td> <td>7298</td> <td>2.77E-13</td> </tr> <tr> <td> <b>Glyma09g12200.1</b> <a href="#nt104">*</a> </td> <td> <b>proline rich protein (PRP1)</b> </td> <td>874</td> <td>6181</td> <td>7.07</td> <td>94605</td> <td>613411</td> <td>1.22E-08</td> </tr> <tr> <td>Glyma13g01720.1</td> <td>transparent testa 1</td> <td>1.59</td> <td>9.84</td> <td>6.10</td> <td>130</td> <td>737</td> <td>6.46E-06</td> </tr> <tr> <td>Glyma14g35140.1</td> <td>transparent testa 1</td> <td>1.60</td> <td>9.55</td> <td>5.90</td> <td>112</td> <td>614</td> <td>1.66E-05</td> </tr> <tr> <td>Glyma08g46790.1</td> <td>plant self-incompatibility protein S1</td> <td>17.10</td> <td>82.41</td> <td>4.82</td> <td>655</td> <td>2893</td> <td>1.03E-05</td> </tr> <tr> <td>Glyma11g27320.1</td> <td>unknown</td> <td>43.47</td> <td>205</td> <td>4.72</td> <td>1809</td> <td>7836</td> <td>7.33E-06</td> </tr> <tr> <td>Glyma05g25180.1</td> <td>senescence-associated protein 29</td> <td>8.00</td> <td>36.24</td> <td>4.50</td> <td>579</td> <td>2403</td> <td>2.61E-05</td> </tr> <tr> <td>Glyma08g08200.1</td> <td>senescence-associated protein 29</td> <td>10.60</td> <td>45.29</td> <td>4.20</td> <td>784</td> <td>3070</td> <td>4.27E-05</td> </tr> <tr> <td>Glyma09g41940.1</td> <td>cytochrome P450 monooxygenase</td> <td>3.11</td> <td>12.48</td> <td>4.01</td> <td>635</td> <td>2333</td> <td>0.0001</td> </tr> <tr> <td>Glyma17g09110.1</td> <td>strictosidine synthase protein</td> <td>95.78</td> <td>338</td> <td>3.53</td> <td>10219</td> <td>33034</td> <td>0.0002</td> </tr> <tr> <td>Glyma05g07630.1</td> <td>strictosidine synthase protein</td> <td>105</td> <td>312</td> <td>2.97</td> <td>11557</td> <td>31638</td> <td>0.001</td> </tr> <tr> <td>Glyma04g02660.1</td> <td>GAST1 protein homolog 1</td> <td>421</td> <td>1222</td> <td>2.90</td> <td>22553</td> <td>60062</td> <td>0.002</td> </tr> <tr> <td>Glyma04g02460.2</td> <td>Xylem serine proteinase 1</td> <td>189</td> <td>434</td> <td>2.29</td> <td>42068</td> <td>88490</td> <td>0.017</td> </tr> <tr> <td> <b>b) Overexpressed in Standard</b> </td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <b>Model</b> </td> <td> <b>Annotation</b> </td> <td> <b>CS</b> </td> <td> <b>CD</b> </td> <td> <b>CS/CD</b> </td> <td> <b>CS</b> </td> <td> <b>CD</b> </td> <td> <b>p value</b> </td> </tr> <tr> <td>Glyma12g34160.1</td> <td>glycinin subunit G7</td> <td>40.19</td> <td>1.46</td> <td>27.6</td> <td>2844</td> <td>94.4</td> <td>1.79E-18</td> </tr> <tr> <td>Glyma10g35840.1</td> <td>Unknown</td> <td>523</td> <td>32.38</td> <td>16.16</td> <td>36331</td> <td>2060</td> <td>2.48E-16</td> </tr> <tr> <td>Glyma10g35870.1</td> <td>ADR12 protein</td> <td>729</td> <td>45.18</td> <td>16.13</td> <td>38268</td> <td>2174</td> <td>2.52E-16</td> </tr> <tr> <td>Glyma04g38640.1</td> <td>amino acid permease AAP3</td> <td>25.69</td> <td>1.6</td> <td>16.09</td> <td>4805</td> <td>274</td> <td>2.52E-15</td> </tr> <tr> <td>Glyma10g39170.1</td> <td>lipid transfer family protein</td> <td>21.29</td> <td>2.32</td> <td>9.17</td> <td>3995</td> <td>399</td> <td>4.81E-11</td> </tr> <tr> <td>Glyma06g15410.1</td> <td>calmodulin-binding protein</td> <td>181</td> <td>22.16</td> <td>8.16</td> <td>15411</td> <td>1730</td> <td>8.87E-11</td> </tr> <tr> <td>Glyma07g28610.1</td> <td>glycine rich protein</td> <td>27.31</td> <td>3.85</td> <td>7.08</td> <td>1617</td> <td>209</td> <td>1.75E-08</td> </tr> <tr> <td>Glyma10g03400.1</td> <td>sucrose-binding protein</td> <td>9.58</td> <td>1.75</td> <td>5.48</td> <td>436</td> <td>72.89</td> <td>2.88E-05</td> </tr> <tr> <td>Glyma02g42630.1</td> <td>glycine rich protein</td> <td>30.91</td> <td>5.7</td> <td>5.42</td> <td>1521</td> <td>257</td> <td>6.63E-07</td> </tr> <tr> <td>Glyma15g23830.1</td> <td>proline rich protein</td> <td>144</td> <td>26.61</td> <td>5.41</td> <td>10955</td> <td>1857</td> <td>7.80E-08</td> </tr> <tr> <td>Glyma09g12250.1</td> <td>proline rich protein</td> <td>608</td> <td>115</td> <td>5.31</td> <td>82019</td> <td>14167</td> <td>7.25E-08</td> </tr> <tr> <td> <b>Glyma09g12260.1</b> <a href="#nt104">*</a> </td> <td> <b>proline rich protein (PRP2)</b> </td> <td>809</td> <td>174</td> <td>4.66</td> <td>120651</td> <td>23748</td> <td>5.13E-07</td> </tr> <tr> <td>Glyma09g24410.1</td> <td>Hsp90 protein</td> <td>23.31</td> <td>5.43</td> <td>4.29</td> <td>5297</td> <td>1132</td> <td>3.07E-06</td> </tr> <tr> <td>Glyma05g21680.1</td> <td>auxin-responsive GH3 product</td> <td>8.63</td> <td>2.05</td> <td>4.19</td> <td>1629</td> <td>356</td> <td>1.34E-05</td> </tr> <tr> <td>Glyma13g36400.1</td> <td>seed storage protein</td> <td>239</td> <td>67.03</td> <td>3.57</td> <td>19730</td> <td>5061</td> <td>2.32E-05</td> </tr> <tr> <td>Glyma05g33200.1</td> <td>defensin-like protein</td> <td>192.12</td> <td>54.13</td> <td>3.54</td> <td>10413</td> <td>2689</td> <td>2.83E-05</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>(a) Overexpressed in Clark Defective (CD) or (b) Overexpressed in Clark Standard (CS).</p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>*Indicates Proline Rich Protein (PRP) genes that have had been reported to have major structural functions in the cell wall. </p> </div> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0096342.t004">10.1371/journal.pone.0096342.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
Selected Differentially Expressed Genes in the Harosoy Isoline Pair at the 100–200 mg Seed Weight Stage Using RPKM and DESeq Analysis of RNA-Seq Data. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>a) Overexpressed in defective</td> <td /> <td>RNA-Seq (RPKM)</td> <td>DESeq (Base Mean)</td> </tr> <tr> <td>Model</td> <td>Annotation</td> <td>HS</td> <td>HD</td> <td>HD/HS</td> <td>HS</td> <td>HD</td> <td>p value</td> </tr> </thead> <tbody> <tr> <td>Glyma11g27320.1</td> <td>unknown</td> <td>0.54</td> <td>80</td> <td>80.00</td> <td>16</td> <td>2248</td> <td>3.04E-14</td> </tr> <tr> <td>Glyma17g09110.1</td> <td>strictosidine synthase protein</td> <td>2.33</td> <td>177</td> <td>75.96</td> <td>174</td> <td>12748</td> <td>1.86E-13</td> </tr> <tr> <td>Glyma08g46790.1</td> <td>Plant self-incompatibility protein S1</td> <td>0.26</td> <td>17</td> <td>65.38</td> <td>7</td> <td>438</td> <td>3.53E-08</td> </tr> <tr> <td>Glyma05g07630.1</td> <td>strictosidine synthase protein</td> <td>2.45</td> <td>154</td> <td>62.85</td> <td>189</td> <td>11508</td> <td>1.09E-12</td> </tr> <tr> <td>Glyma05g25180.1</td> <td>senescence-associated protein 29</td> <td>0.64</td> <td>48</td> <td>48</td> <td>32</td> <td>2363</td> <td>3.11E-12</td> </tr> <tr> <td>Glyma13g27050.1</td> <td>unknown</td> <td>0.01</td> <td>47</td> <td>47</td> <td>0.98</td> <td>3655</td> <td>6.87E-22</td> </tr> <tr> <td>Glyma04g02660.1</td> <td>GAST1 protein homolog 1</td> <td>16</td> <td>600</td> <td>37.50</td> <td>608</td> <td>21746</td> <td>9.57E-11</td> </tr> <tr> <td> <b>Glyma09g12200.1</b> <a href="#nt106">*</a> </td> <td> <b>proline rich protein (PRP1)</b> </td> <td>162</td> <td>5890</td> <td>36.35</td> <td>12282</td> <td>430912</td> <td>8.71E-11</td> </tr> <tr> <td>Glyma08g08200.1</td> <td>senescence-associated protein 29</td> <td>1.81</td> <td>64</td> <td>35.35</td> <td>94</td> <td>3192</td> <td>6.58E-10</td> </tr> <tr> <td>Glyma09g41940.1</td> <td>cytochrome P450 monooxygenase</td> <td>1.80</td> <td>47</td> <td>26.11</td> <td>257</td> <td>6537</td> <td>3.35E-09</td> </tr> <tr> <td>Glyma14g35140.1</td> <td>transparent testa 1</td> <td>0.78</td> <td>15</td> <td>15</td> <td>38</td> <td>699</td> <td>2.23E-06</td> </tr> <tr> <td>Glyma13g01720.1</td> <td>transparent testa 1</td> <td>0.75</td> <td>13</td> <td>13</td> <td>43</td> <td>719</td> <td>3.53E-06</td> </tr> <tr> <td>Glyma04g02460.2</td> <td>Xylem serine protease</td> <td>73</td> <td>556</td> <td>7.61</td> <td>11434</td> <td>83564</td> <td>5.84E-05</td> </tr> <tr> <td> <b>b)Overexpressed in Standard</b> </td> <td /> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td> <b>Model</b> </td> <td> <b>Annotation</b> </td> <td> <b>HS</b> </td> <td> <b>HD</b> </td> <td> <b>HS/HD</b> </td> <td> <b>HS</b> </td> <td> <b>HD</b> </td> <td> <b>p value</b> </td> </tr> <tr> <td>Glyma06g15410.1</td> <td>calmodulin-binding protein</td> <td>570</td> <td>0.52</td> <td>570</td> <td>33949</td> <td>30</td> <td>2.32E-24</td> </tr> <tr> <td>Glyma02g42630.1</td> <td>glycine rich protein</td> <td>530</td> <td>0.19</td> <td>530</td> <td>18225</td> <td>6</td> <td>2.21E-26</td> </tr> <tr> <td>Glyma07g28610.1</td> <td>glycine rich protein</td> <td>416</td> <td>0.05</td> <td>416</td> <td>17234</td> <td>2.05</td> <td>3.68E-28</td> </tr> <tr> <td>Glyma09g12250.1</td> <td>proline rich protein</td> <td>2369</td> <td>6</td> <td>395</td> <td>223255</td> <td>551</td> <td>8.49E-21</td> </tr> <tr> <td> <b>Glyma09g12260.1</b> <a href="#nt106">*</a> </td> <td> <b>proline rich protein (PRP2)</b> </td> <td>3438</td> <td>9</td> <td>382</td> <td>358154</td> <td>874</td> <td>7.21E-21</td> </tr> <tr> <td>Glyma15g23830.1</td> <td>proline rich protein</td> <td>423</td> <td>1.45</td> <td>292</td> <td>22525</td> <td>75</td> <td>3.67E-19</td> </tr> <tr> <td>Glyma10g39170.1</td> <td>lipid transfer family protein</td> <td>87</td> <td>2.83</td> <td>31</td> <td>11353</td> <td>359</td> <td>3.67E-10</td> </tr> <tr> <td>Glyma05g33200.1</td> <td>defensin-like protein</td> <td>621</td> <td>26</td> <td>24</td> <td>23517</td> <td>935</td> <td>2.19E-09</td> </tr> <tr> <td>Glyma09g24410.1</td> <td>Hsp90 protein</td> <td>36</td> <td>2.54</td> <td>14</td> <td>5751</td> <td>389</td> <td>3.44E-07</td> </tr> <tr> <td>Glyma04g38640.1</td> <td>amino acid permease 3</td> <td>34</td> <td>2.51</td> <td>14</td> <td>4438</td> <td>318</td> <td>6.35E-07</td> </tr> <tr> <td>Glyma12g34160.1</td> <td>glycinin subunit G7</td> <td>158</td> <td>12</td> <td>13</td> <td>7812</td> <td>576</td> <td>5.94E-07</td> </tr> <tr> <td>Glyma05g21680.1</td> <td>auxin-responsive GH3 product</td> <td>10</td> <td>0.27</td> <td>10</td> <td>1361</td> <td>35</td> <td>1.88E-09</td> </tr> <tr> <td>Glyma13g36400.1</td> <td>seed storage protein</td> <td>401</td> <td>42</td> <td>10</td> <td>23097</td> <td>2382</td> <td>6.99E-06</td> </tr> <tr> <td>Glyma10g03400.1</td> <td>sucrose-binding protein</td> <td>78</td> <td>15</td> <td>5</td> <td>2467</td> <td>454</td> <td>0.0008</td> </tr> <tr> <td>Glyma10g35840.1</td> <td>Unknown</td> <td>45.51</td> <td>11.63</td> <td>3.91</td> <td>2208</td> <td>545</td> <td>0.005</td> </tr> <tr> <td>Glyma10g35870.1</td> <td>ADR12 protein</td> <td>63.83</td> <td>16.65</td> <td>3.83</td> <td>2342</td> <td>590</td> <td>0.006</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>(a) Overexpressed in Harosoy Defective (HD) or (b) Overexpressed in Harosoy Standard (HS). </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>*Indicates Proline Rich Protein (PRP) genes that have had been reported to have major structural functions in the cell wall. </p> </div> </div> </div> <p>A selection of 29 of the 364 differentially expressed genes at the 100–200 mg seed weight stage are shown in <a href="#pone-0096342-t003">Table 3</a> (in Clark background) and <a href="#pone-0096342-t004">Table 4</a> (in Harosoy background) along with their functional annotations, fold-changes in differential expression, and p-values. The data for all 364 genes are shown in <a href="#pone.0096342.s021">Table S3</a>. Six genes in <a href="#pone-0096342-t003">Tables 3</a> and <a href="#pone-0096342-t004">4</a> have cell wall structural protein annotations, including proline-rich proteins PRP1 and PRP2 and glycine-rich proteins. </p> </div> <div class="cellwallrelatedgenesexpressedinbothclarkandharosoyseedcoats" title="sec"> <div class="title" tagx="title" title="title">
Cell Wall Related Genes Expressed in Both Clark and Harosoy Seed Coats</div> <p>As presented in <a href="#pone-0096342-t003">Tables 3</a> &amp;amp; <a href="#pone-0096342-t004">4</a>, PRP1 and PRP2 were two of the highly differentially expressed proline-rich protein genes in both Clark and Harosoy isolines. In the Harosoy isolines, both the RPKM levels and fold differences for these genes were higher as compared to Clark isolines. The expression of the PRP1 was higher earlier in development and declined as shown in <a href="#pone-0096342-g004">Figure 4A</a>. In contrast, PRP2 was expressed only at the later stages of seed development as seen in <a href="#pone-0096342-g004">Figure 4B</a>. The differential expression between the standard and defective seed coats was maximal for both PRP1 and PRP2 at the 100–200 mg weight range in Clark and Harosoy isolines. PRP1 transcripts appeared to decline in abundance faster in the standard seed coats of both Clark and Harosoy isolines leading to higher levels in the defective seed coats whereas PRP2 transcripts were always less abundant in the defective seed coats. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
The Expression Levels of (A) PRP1 and (B) PRP2 Genes in Standard and Defective Seed Coats of either Clark or Harosoy Backgrounds at Three Stages of Seed Development. </div> <p>The blue line in the graphs represents expression level (RPKM) of these genes in seed coats of standard isolines and the red line represents expression levels in the defective seed coats at the 50–100 mg, 100–200 mg, and 400–500 mg seed weight ranges. Maximal differential expression of the genes between the standard and defective isolines was observed at the 100–200 mg seed weight stage. (Abbreviations: CS: Clark standard, CD: Clark defective, HS: Harosoy standard, HD: Harosoy defective). </p>   </div> <p>The expression levels of PRP1 and PRP2 at different seed weight stages were previously demonstrated by <a href="#pone.0096342-Percy1">[10]</a> using RNA blots in the Clark isoline. Our previous RNA blots agree with our present RNA-Seq data showing that PRP1 was expressed more abundantly in early development, whereas PRP2 was present in the older seed coats. These two Glyma models were called as "low confidence" gene models with no annotation by the Glyma gene model version 1.0 at the Phytozome database and as extensin-related genes in Glyma model version 1.1, whereas the non-redundant database of the National Center for Biotechnology Information (NCBI) annotated them as proline-rich proteins. We conducted manual annotations in the following manner to confirm them as the classically defined soybean PRP1 and PRP2 genes which are in NCBI. The nucleotide and amino acid sequences of PRP1 from NCBI (J02746) showed 100% match with Glyma09g12200.1 from the soybean genome ( <a href="#pone.0096342.s003">Figure S3</a>), and the nucleotide sequence of SbPRP2 gene from NCBI (J05208) showed 98.8% similarity with Glyma09g12260.1 ( <a href="#pone.0096342.s004">Figure S4</a>). There was a difference of 10 amino acids between the amino acid sequence of PRP2 in cultivars Wayne (amino acid sequence from NCBI) and Williams (amino acid sequence from the soybean genome). Both PRP1 and PRP2 genes are linked and separation between these genes is approximately 146 kb ( <a href="#pone.0096342.s005">Figure S5</a>). </p> <p>The graphical representations of the expression levels of the other differentially expressed genes shown in <a href="#pone-0096342-t003">Tables 3</a> and <a href="#pone-0096342-t004">4</a> at all three different stages of seed development is presented in <a href="#pone.0096342.s006">Figures S6</a> and <a href="#pone.0096342.s007">S7</a>, respectively, showing that all have the highest differential expression at the mid- developmental stages of 100–200 mg seed weight. Included is the pattern for several other proline and glycine-rich proteins and that of the xylem serine proteinase that is overexpressed in the defective seed coats ( <a href="#pone.0096342.s006">Figure S6</a>-M). In many of these genes, those that are highly expressed in the 50–100 mg seed coats and then decline show a pattern similar to PRP1 in that they are overexpressed in the defective seed coats. Likewise, those that are overexpressed in the standard line are similar to PRP2 in that they generally have higher RPKM expression levels later in development. </p> <p> <a href="#pone-0096342-g003">Figure 3</a> shows that 21% of the cell wall related genes were annotated as fasciclin related. The graphical view of expression of these fasciclin-like arabinogalactan genes in the seed coats of both standard and defective seed coat isolines at three different stages of seed development is presented in <a href="#pone.0096342.s008">Figure S8</a>. All of the genes showed expression patterns with significant differential expression at 100–200mg seed weight stage and their transcript levels were higher in the defective as compared to the standard seed coats, as was the case with PRP1. Thus, all 11 of these genes were overexpressed in the defective seed coat. <a href="#pone.0096342.s009">Figure S9</a> graphs the developmental profiles of the remaining 25 cell wall genes overexpressed in the defective isolines and <a href="#pone.0096342.s010">Figure S10</a> shows the remaining 3 cell wall genes that were overexpressed in the standard seed coats. </p> <p>It is clear from the 68 graphs of the gene expression patterns shown in <a href="#pone.0096342.s006">Figures S6</a>– <a href="#pone.0096342.s010">S10</a> that the 100–200 mg weight range shows the highest differential expression and that many of the genes can be classified as either declining in expression or increasing in expression during seed coat development as is the case for the PRP1 and PRP2 patterns. In many cases, the net pattern mutation appears to increase expression of many of the cell wall related genes in the 100–200 mg weight range as for PRP1. In contrast, many of the genes that are overexpressed in the standard line appear to be increasing during late seed development in both standard and defective lines as does PRP2. </p> </div> <div class="hormoneregulatedgeneswhichweredifferentiallyexpressedinbothclarkandharosoybackgrounds" title="sec"> <div class="title" tagx="title" title="title">
Hormone Regulated Genes which were Differentially Expressed in Both Clark and Harosoy Backgrounds </div> <p>There were 21 hormone regulated genes that were differentially expressed in the seed coat of wild type and defective isolines of both Clark and Harosoy background ( <a href="#pone.0096342.s021">Table S3</a>) including representative auxin and gibberellin regulated genes that are shown in <a href="#pone-0096342-t003">Tables 3</a> &amp;amp; <a href="#pone-0096342-t004">4</a>. One gene model Glyma10g35870.1 showed the best match to the authentic ADR12 (auxin down regulated) gene from NCBI (S58482). It showed approximately 16-fold lower expression in the Clark defective seed coats and 4-fold lower expression expression in Harosoy defective seed coats ( <a href="#pone-0096342-t003">Tables 3</a> &amp;amp; <a href="#pone-0096342-t004">4</a>). Auxin is an important phytohormone that plays several important roles in plant growth and development. The expression level of this gene at different stages of seed development in both standard and defective seed coat isolines is presented in <a href="#pone-0096342-g005">Figure 5A</a> for RNA-Seq data. In the Clark standard isoline, this gene showed higher expression in earlier stages, and there was continuous decrease in the expression level at later stages of seed development. There was significant differential expression at all the seed developmental stages ( <a href="#pone-0096342-g005">Figure 5A</a>, left). In the Harosoy standard isoline, expression of ADR12 increased continuously with developmental stages and also increased in the defective seed coats at the two older stages, although the levels were significantly lower in the defective seed coats ( <a href="#pone-0096342-g005">Figure 5A</a>, right). The expression pattern of ADR12 using RNA blots ( <a href="#pone-0096342-g005">Figure 5B</a>, top) coincides well with the RNA-Seq data in that the defective seed coats have reduced expression especially in the Clark isolines. In addition, the blots confirm that the ADR12 is more highly expressed in standard seed coats of Clark than in Harosoy. As presented in the blots in <a href="#pone-0096342-g005">Figure 5B</a>, bottom panel, there was no differential expression observed from 4–5 days old soybean hypocotyls in standard and defective isolines of Clark and Harosoy. Thus, the differential expression of this gene is likely confined to the seed coats that manifest the defective cell wall structure. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g005"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
The Expression Levels of ADR12 (Glyma10g35870.1) Transcripts in Standard and Defective Seed Coat Isolines as Determined by RNA-Seq and RNA blots. </div> <p>(A) The levels of ADR12 (ADR12-2) transcripts (as RPKM) in seed coats of Clark (left) or Harosoy (right) isolines. The blue line in the graphs represents the expression level (RPKM) of this gene in seed coats of standard isolines and the red line represents expression levels in the defective seed coats at the 50–100 mg, 100–200 mg, and 400–500 mg seed weight ranges. (B, Top) The Northern blots of RNA from 100–200 mg soybean seed coats representing three independent extractions of the Clark and two independent samples of the Harsoy lines. (B, Bottom) The Northern blot of RNA from 4–5 days old soybean hypocotyls extracted from two independent samples of each line. Each gel lane was loaded with 10 ug of total RNA, electrophoresed, stained with ethidium bromide to confirm equal loadings, and blotted. The RNA blots were probed with Gm-r1088-6964, a EST clone representing ADR12 and corresponding to Glyma10g35870.1. Whereas there is differential expression in the seed coats of standard and defective lines, there was no difference in expression of ADR12 in the hypocotyls. (Abbreviations: CS: Clark Standard, CD: Clark defective, HS: Harosoy standard, HD: Harosoy defective). </p>   </div> <p>The expression pattern of two other hormone regulated genes in <a href="#pone-0096342-t003">Tables 3</a> and <a href="#pone-0096342-t004">4</a> that showed significant differential expression in the seed coat of wild type and defective isolines of both Clark and Harosoy backgrounds is presented in <a href="#pone.0096342.s006">Figures S6</a>-L and <a href="#pone.0096342.s007">S7</a>-N. The gene that showed a gibberellin regulated annotation had an expression pattern similar to PRP1 with higher expression in defective isolines as compared to the standard seed coats at 100–200 mg seed weight ( <a href="#pone.0096342.s006">Figure S6</a>-L), while the other gene that had an auxin related annotation showed a different expression pattern with under expression in the defective isoline at the 100–200 mg seed weight stage ( <a href="#pone.0096342.s007">Figure S7</a>-N). </p> </div> <div class="genesdifferentiallyexpressedbetweenstandardanddefectiveisolinesineithertheclarkorharosoybackgrounds" title="sec"> <div class="title" tagx="title" title="title">
Genes Differentially Expressed between Standard and Defective Isolines in Either the Clark or Harosoy Backgrounds </div> <p>Along with the genes that showed significant differential expression in the seed coats of both genetic backgrounds, there were genes that showed significant differential expression in either the Clark or Harosoy isolines. As presented in <a href="#pone-0096342-t002">Table 2</a> and <a href="#pone.0096342.s020">Table S2</a>, there were approximately 1300 genes that showed differential expression in either the Clark or Harosoy background. These genes were divided into different broad functional categories based on functional annotations and presented in <a href="#pone.0096342.s011">Figures S11</a>, <a href="#pone.0096342.s012">S12</a> and <a href="#pone.0096342.s013">S13</a>. The cell wall category is highest at the 100–200 mg stages and comprises 14% in Clark and 44% of the total differentially expressed genes in Harosoy seed coats. Another major category was transcription factor and transcription related genes (up to 11%). There were 109 genes with functional annotations that showed opposite differential expression ( <i>ie</i>., if expression of a particular gene was higher in the standard isoline of Clark, then it was lower in the standard isoline of Harosoy) ( <a href="#pone.0096342.s022">Table S4</a>). </p> </div> <div class="transcriptionfactorgenesthatweredifferentiallyexpressedinstandardanddefectiveseedcoatisolines" title="sec"> <div class="title" tagx="title" title="title">
Transcription Factor Genes that were Differentially Expressed in Standard and Defective Seed Coat Isolines </div> <p>Two zinc finger domain (C2H2 type) containing protein genes that had TT1 (TRANSPARENT TESTA1) annotations were approximately 6-fold and 15-fold more highly expressed in the defective seed coats of the Clark and Harosoy lines, respectively, at the 100–200 mg seed weight stage ( <a href="#pone-0096342-t003">Tables 3</a> &amp;amp; <a href="#pone-0096342-t004">4</a>). The expression level of these transcription factor genes at different stages of development is presented in <a href="#pone-0096342-g006">Figure 6</a>. These genes showed higher expression at the younger stage in both standard and defective isolines in the Clark background and then decreased with progressive seed weight stage. They increased somewhat in the Harosoy line at the 100–200 mg seed stage before declining. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g006"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <div class="title" tagx="title" title="title">
The Expression Levels of TRANSPARENT TESTA 1 (TT1) Genes in Standard or Defective Seed Coats of Clark and Harosoy Backgrounds. </div> <p>Both Glyma14g35140 (Figure A) and Glyma13g01720 (Figure B) gene models had TT1 annotations. The blue line in the graphs represents expression level (RPKM) of these genes in seed coats of standard isolines and the red line represents expression levels in the defective seed coats at the 50–100 mg, 100–200 mg, and 400–500 mg seed weight ranges. There was significant differential expression in these genes at 100–200mg seed weight range. (Abbreviations: CS: Clark standard, CD: Clark defective, HS: Harosoy standard, HD: Harosoy defective). </p>   </div> <p>Along with the two TT1 gene models, there were 14 total transcription factor genes that showed differential expression in seed coats of standard and defective isolines in both of the Clark and Harosoy backgrounds at the 100–200 mg stage ( <a href="#pone-0096342-g007">Figure 7</a>, <a href="#pone.0096342.s021">Table S3</a>). The most prominent category had NAC-related annotations. The other important classes of transcription factor genes were bHLH, zinc finger and ethylene responsive factor genes. The expression of these transcription factor genes at three stages of seed development is presented in <a href="#pone.0096342.s014">Figure S14</a> and <a href="#pone.0096342.s015">S15</a>. There were two transcription factor genes (Myb and bHLH) that showed expression patterns similar to TT1 and PRP1 ( <a href="#pone.0096342.s014">Figure S14</a> A &amp;amp; C). Four of the other transcription factor genes showed expression patterns similar to PRP2 ( <a href="#pone.0096342.s015">Figure S15</a> A, C, E &amp;amp; H). Interestingly, there were no significantly differentially expressed transcription factor genes found in common at the 50–100 mg or 400–500 mg seed weight stages in both backgrounds. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0096342.g007"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <div class="title" tagx="title" title="title">
The Annotations of Differentially Expressed Transcription Factor Genes in Standard and Defective Seed Coats in Common Between Clark and Harosoy Backgrounds at the 100–200mg Seed Weight stage. </div> <p>The classification is shown of 14 significantly differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression and p-value ≤0.05) at the 100–200 mg seed weight stage that are in common between both the Clark and Harosoy backgrounds. </p>   </div> <p> <a href="#pone.0096342.s020">Table S2</a> contains the data for transcription factors that were differentially expressed between standard and defective seed coats in only the Clark or the Harsoy background. At the 50–100 mg seed weight stage, there were 59 transcription factor genes that showed differential expression in the seeds coats of only the Clark isolines with the maximum number of genes being related to the zinc finger category ( <a href="#pone.0096342.s016">Figure S16</a>) whereas in the Harosoy isolines at the 50–100 mg seed weight stage, Glyma15g42810.1, was the only transcription factor gene that showed differential expression. Likewise, <a href="#pone.0096342.s017">Figure S17</a> delineates the functional annotations of the 16 and 12 transcription factors that showed differential expression in the Clark background at the 100–200 or 400–500 mg stages, respectively, and <a href="#pone.0096342.s018">Figure S18</a> shows the 43 and 111 transcription factors that were differentially expressed only in the Harosoy line at the later stages of development. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="manycellwallrelatedproteinsareaffectedbythenetpatterndefectiveseedcoatmutation" title="sec"> <div class="title" tagx="title" title="title">
Many Cell Wall Related Proteins are Affected by the Net Pattern Defective Seed Coat Mutation </div> <p>We provide an overview of the genes that are affected by the "net pattern" mutation which affects the structural integrity of the soybean seed coat. As shown in <a href="#pone-0096342-t002">Table 2</a> and <a href="#pone.0096342.s021">Table S3</a>, a total of 364 genes showed significant differential expression in both Clark and Harosoy isolines that contain the defective seed coat mutation. Approximately 15% of the differentially expressed genes showed cell wall related annotations including proline-rich proteins, glycine-rich proteins, fasciclin-like arabinogalactan proteins, expansins, and extensins. As reported earlier, these are the major classes of proteins that provide tensile strength and structure to the cell wall <a href="#pone.0096342-MacMillan1">[19]</a>, <a href="#pone.0096342-Han1">[20]</a>. Xyloglucan <a href="#pone.0096342-Chou1">[21]</a>, xylulose <a href="#pone.0096342-Lois1">[22]</a> and pectin had also been reported as structural components of the cell wall <a href="#pone.0096342-Bailey1">[23]</a>, <a href="#pone.0096342-Showalter1">[24]</a>, <a href="#pone.0096342-Cosgrove1">[25]</a>, <a href="#pone.0096342-Lamport1">[26]</a>. Structural cell wall GRPs have glycine contents up to 60–70% <a href="#pone.0096342-Ringli1">[27]</a>, <a href="#pone.0096342-Ryser1">[28]</a>, <a href="#pone.0096342-Mangeon1">[29]</a>, <a href="#pone.0096342-Yang1">[30]</a>. In our study, genes with glycine-rich protein annotation had glycine content from 11–40% in their amino acid sequence. </p> <p>Allergens, thaumatin-like proteins, and cysteine-rich domain containing proteins have also been reported to play important roles in various functions related to the cell wall <a href="#pone.0096342-Cosgrove2">[31]</a>, <a href="#pone.0096342-Dafoe1">[32]</a>, <a href="#pone.0096342-Wu1">[33]</a>. The genes with these annotations were differentially expressed in standard and defective isolines in our study and may be involved in the appearance of cracking on the seed coat. Xylem serine proteinase1 was one of the significantly differentially expressed genes at the 100–200 mg seed weight stage in this study. In Arabidopsis, this enzyme has been reported to be involved in xylem formation <a href="#pone.0096342-Zhao1">[34]</a>. </p> <p>While we have concentrated on the 51 cell wall related genes that were significantly differentially expressed in both varieties, we note that there are 920 additional cell wall related genes that did not show differential expression in the seed coats of either of the two isoline pairs. Thus, there are many processes related to the cell wall that are not affected at the transcript level by the net-pattern mutation. </p> </div> <div class="hormoneregulatedgenesaffectedbythenetpatternmutation" title="sec"> <div class="title" tagx="title" title="title">
Hormone Regulated Genes Affected by the Net Pattern Mutation</div> <p>Auxin is a plant growth regulator that is required for cell expansion, division and differentiation. Auxin pathway related genes have been reported to regulate several cell wall related genes in Arabidopsis <a href="#pone.0096342-Evans1">[35]</a>, <a href="#pone.0096342-Overvoorde1">[36]</a>. Likewise gibberellins had been reported to alter the expression of cell wall loosening genes and to work in coordination with auxin and abscisic acid hormone pathways <a href="#pone.0096342-Gou1">[37]</a>. Genes related to the jasmonic acid pathway and ethylene response may also be related to cell wall formation <a href="#pone.0096342-Denness1">[38]</a>, <a href="#pone.0096342-Creelman1">[39]</a>. As presented in the results ( <a href="#pone.0096342.s020">Tables S2</a>– <a href="#pone.0096342.s022">S4</a>), there were differentially expressed genes in the standard and defective isolines that showed ethylene response related, auxin and gibberline hormone related/regulated annotations suggesting an effect of the seed coat mutation on these genes. The role of the auxin downregulated ADR12 transcript that showed high differential expression in the standard and defective seed coats but not in the hypocotyls ( <a href="#pone-0096342-g005">Figure 5</a>) is not known. The predicted peptide form the ADR12 transcript is small at only 71 amino acids. Likely, many of the transcript differences for the affected pathways are confined to the seed coats as is the case for the ADR12 transcript. Such an extensive disruption of cell wall metabolism by the suite of genes affected in the net pattern mutation that leads to producing tears in the walls would likely be detrimental to the growth of the seedlings but is tolerated in the latter stages of seed coat development. </p> </div> <div class="transcriptabundanceandlevelsofsolubleproline-richproteinsarereducedinthedefectiveseedcoatmutation" title="sec"> <div class="title" tagx="title" title="title">
Transcript Abundance and Levels of Soluble Proline-Rich Proteins are Reduced in the Defective Seed Coat Mutation </div> <p>The proline-rich proteins are composed of small tandem repeats such as PPVYK or PPVEK, where the second proline is often hydroxyproline <a href="#pone.0096342-Marcus1">[40]</a>. In our study PRP1 and PRP2 were among the highly differentially expressed genes ( <a href="#pone-0096342-t003">Tables 3</a> and <a href="#pone-0096342-t004">4</a>). PRP1 was previously identified and characterized based on cDNA and amino acid sequence <a href="#pone.0096342-Hong1">[41]</a>, <a href="#pone.0096342-Hong2">[42]</a>, <a href="#pone.0096342-Hong3">[43]</a>, <a href="#pone.0096342-Lindstrom1">[44]</a>. Likewise, PRP2 had also been characterized as a slightly smaller protein than PRP1, and both consist essentially of the repeating decamer PPVYKPPVEK <a href="#pone.0096342-Datta1">[45]</a>. Gene model numbers corresponding to these genes were obtained from the Phytozome database that has genomic sequence from the Williams cultivar of <i>Glycine max</i> <a href="#pone.0096342-Goodstein1">[46]</a>. The Williams PRP2 protein is smaller when compared with the PRP2 in the cultivar Wayne from which it was originally sequenced as presented by amino acid sequence alignment in <a href="#pone.0096342.s004">Figure S4</a>. This corresponds with another study showing that in some soybean varieties both PRP1 and PRP2 proteins are smaller because of in frame deletions in the coding region of units of the tandem decamer repeats. In previous reports, PRP1 and PRP2 had also been reported to be linked, but separated by approximately 13% recombination units <a href="#pone.0096342-Schmidt1">[47]</a>. A BLAST search for these genes in Phytozome showed that they are located on chromosome 9, approximately 146 kb apart ( <a href="#pone.0096342.s005">Figure S5</a>). </p> <p>As described in our results, the expression of PRP1 was dramatically higher in the yellow Harosoy isolines (homozygous for <i>I</i> alleles), as compared to black Clark isolines (homozygous for the <i>i</i> allele) ( <a href="#pone-0096342-g004">Figure 4</a>). Likewise, another study showed that the <i>I</i> locus which controls inhibition of anthocyanin accumulation in the epidermal cells of the soybean seed coat also affects abundance of PRP1 mRNA and protein in the seed coat. Interestingly, an epistatic interaction between the recessive <i>i</i> and <i>t</i> alleles also causes cracking of the pigmented seed coat <a href="#pone.0096342-Nicholas1">[48]</a>. This seed coat cracking is not related to the defective seed coats of the net pattern described in this report which is independent of seed color as shown in <a href="#pone-0096342-g001">Figure 1</a>. However, this genetic interaction implies an interaction of the flavonoid pathway with cell wall structure as the <i>t</i> locus is known to encode a flavonoid 3′ hydroxylase <a href="#pone.0096342-Zabala1">[17]</a>. </p> <p>Our RNA-Seq data ( <a href="#pone-0096342-g004">Figure 4</a>) show excellent agreement with the previous RNA blots <a href="#pone.0096342-Percy1">[10]</a> indicating a delay in the decline of PRP1 transcripts in the defective seed coats leading to higher levels in the defective seed coats at the middle weight range of 100–200 mg. Despite the presence of significant levels of PRP1 transcripts in the defective seed coats, no PRP1 protein was detectable by immunoblotting in defective Clark seed coats at any stage of seed development but it was easily extractable from the standard, non-defective isoline at the same stages <a href="#pone.0096342-Percy1">[10]</a>. However, the similar PRP2 protein was extractable from both standard and defective seed coats. These results implied that a major physiological event in the net pattern defective seed coats may be the irreversible cross-linking of PRP1 into the cell wall occurring in the developing seeds. </p> </div> <div class="transcriptionfactorsaffectedbythenetpatternmutation" title="sec"> <div class="title" tagx="title" title="title">
Transcription Factors Affected by the Net Pattern Mutation</div> <p>Transcription factors are important players for controlling the flow of genetic information from DNA to RNA and ultimately affecting the growth and physiology of the plant. In this study, there were approximately 240 differentially expressed transcription factor genes at different seed weight stages ( <a href="#pone.0096342.s020">Tables S2</a>– <a href="#pone.0096342.s022">S4</a>). These genes were divided into different classes as presented in <a href="#pone-0096342-g007">Figures 7</a>, <a href="#pone.0096342.s016">S16</a>, <a href="#pone.0096342.s017">S17</a>, and <a href="#pone.0096342.s018">S18</a>. The major classes of transcription factors genes which showed differential expression in our study were Myb, NAC, basic helix-loop-helix, WRKY, and zinc finger transcription factors. Transcription factors of these types been reported to affect cell wall integrity in other plant species. MYB domain containing transcription factors are involved in secondary cell wall biosynthesis, pollen wall composition, mucilage deposition, extrusion and lignin deposition in Arabidopsis <a href="#pone.0096342-Zhong1">[49]</a>, <a href="#pone.0096342-Preston1">[50]</a>, <a href="#pone.0096342-Penfield1">[51]</a>, <a href="#pone.0096342-Western1">[52]</a>. The MYB46 and MYB83 transcription factors are thought to regulate secondary wall biosynthesis in Arabidopsis <a href="#pone.0096342-Zhong2">[53]</a>, <a href="#pone.0096342-Zhou1">[54]</a>. A basic helix-loop-helix transcription factor had been reported to play an important role in tapetal cell development <a href="#pone.0096342-Xu1">[55]</a>. The role of a MADS-box transcription factor as RIPENING INHIBITOR (RIN) is in cell wall metabolism and carotenoid biosynthesis <a href="#pone.0096342-Martel1">[56]</a> and a basic leucine zipper domain containing transcription factor had been reported to affect pollen coat patterning <a href="#pone.0096342-Gibalov1">[57]</a>. As demonstrated by Wang <i>et al.,</i> WRKY transcription factors are involved in regulation of downstream genes encoding the NAM, ATAF1/2, and CUC2 (NAC) and CCCH type (C3H) zinc finger TFs that activate secondary wall synthesis <a href="#pone.0096342-Wang1">[58]</a>. NAC domain containing transcription factors were reported to be involved in secondary cell wall biosynthesis and in the regulation of cellulose and hemicellulose biosynthetic genes in addition to those involved in lignin polymerization and signaling in Arabidopsis and Eucalyptus <a href="#pone.0096342-Hussey1">[59]</a>, <a href="#pone.0096342-Wang2">[60]</a>. Rice and maize SWNs (Secondary wall NACs) and MYB46 had been reported as master transcriptional activators of the secondary wall biosynthetic program <a href="#pone.0096342-Zhong3">[61]</a>. The AP2 domain containing family is an ethylene responsive group of transcription factors, and in <i>Arabidopsis thaliana</i>, these were expressed in specific cell types of roots, stems and seeds that undergo suberization <a href="#pone.0096342-Lasserre1">[62]</a>. The TT1 genes were another class of the significantly differentially expressed genes in this study ( <a href="#pone-0096342-g006">Figure 6</a>). These transcription factor genes had been reported to be involved in seed coat pigmentation and integrity in Arabidopsis <a href="#pone.0096342-Sagasser1">[63]</a>. </p> <p>It is likely that the defective seed coat mutation in soybean sets in motion changes in many pathways related to the cell wall. Some of the differential expression may be a temporal shift in developmental appearances of transcripts. Notably, in the case of the TT1 homolog, it appears that the decline in its transcript levels was delayed in the defective seed coats relative to the standard seed coats ( <a href="#pone-0096342-g006">Figure 6</a>), thus leading to significant differential expression at the 100–200 mg seed weight range. This profile is similar to the pattern exhibited by the PRP1 protein ( <a href="#pone-0096342-g004">Figure 4A</a>) in that its decline in expression appears to be slowed. Thus, a developmental delay in the normal expression patterns of multiple genes appeared to be a feature of the effect of this mutation as shown in <a href="#pone.0096342.s006">Figures S6</a>– <a href="#pone.0096342.s010">S10</a> and <a href="#pone.0096342.s014">S14</a>– <a href="#pone.0096342.s015">S15</a>. These shifts may be one of the reasons for significant differential expression at the mid-seed weight range of 100–200 mg and may indicate that the triggering event occurs at or before the 50 mg seed weight range. </p> </div> <div class="summary" title="sec"> <div class="title" tagx="title" title="title">
Summary</div> <p>We present an overview of genes whose expression was affected by the "net pattern" mutation in soybean. Two isolines of different genetic backgrounds were used to understand physiology related to this mutation that causes defective cracks in the seed coats. Twelve samples representing three stages of seed coat development from each of the four lines were subjected to the power of next-generation sequencing to obtain millions of transcript reads that were mapped to all 78,773 high and low confidence gene models of <i>Glycine max</i>. There were approximately 1300 significantly differentially expressed genes affected by this mutation in each isoline pair with 364 in common between the two isolines of different genetic backgrounds. The cell wall structural protein genes including proline-rich proteins (PRP1 and PRP2) and glycine-rich proteins were among the transcripts affected by the mutation as well as a number of transcription factors. Some of the transcript patterns indicate that a developmental shift in timing of gene expression underpins the differential gene expression changes at mid-maturation as shown by graphical presentation of expression patterns for 82 out of 364 genes that were significantly differentially expressed in both the Clark and Harosoy backgrounds. In summary, the mutation appears to set in motion a complex series of events, many manifested at the transcript level, that lead to changes in physiology and ultimately structure of the cell wall. The information on potential candidate genes for this mutation and the regulated genes will not only be helpful in understanding cell wall physiology in soybean, but could also provide a platform for improvement in bioenergy crops like sorghum, maize and <i>Miscanthus</i>. </p> </div> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialsandgeneticnomenclature" title="sec"> <div class="title" tagx="title" title="title">
Plant Materials and Genetic Nomenclature</div> <p>The isolines of <i>Glycine max</i> used in this study were a black seeded Clark line, Clark defective seed coat, Harosoy standard and Harosoy defective seed coat. The defective seed coat mutation was originally found in two PI (plant introduction) lines, PI 339.994 and PI423.730B and these lines were backcrossed to the black seeded Clark line (UC9) to create an isoline UC412 containing this mutation. The same mutation was also backcrossed into the Harosoy line to create an isoline UC508. The L or PI numbers represent the official USDA isoline or plant introduction numbers, respectively. The UC number is an internal number used by our laboratory. Detailed information on these lines is presented in <a href="#pone-0096342-t001">Table 1</a>. </p> </div> <div class="rna-seqmethod" title="sec"> <div class="title" tagx="title" title="title">
RNA-Seq Method</div> <p>For RNA extraction, immature seeds were harvested over the course of several weeks. The individual seeds were pooled and sorted by weight. Following this, each seed was dissected, separating the cotyledon and embryo from the seed coat, and placed in separate 15-ml polypropylene centrifuge tubes (Corning, Acton, MA). All tissue was then frozen in liquid nitrogen and placed in storage at −80°C until it could be lyophilized. RNA of seed coats of seeds from the 50–100 mg, 100–200 mg and 400–500 mg seed weight stages of each isoline was extracted separately using the RNA for 5 ml volumes from ∼30 mg (50–100 mg and 100–200 mg seed weight stage) and ∼70 mg (400–500 mg seed weight stage) dry weight. The modified protocol used here is based on the protocols of McCarty <a href="#pone.0096342-McCarty1">[64]</a>. Extraction of RNA from seed coats with dark pigmentation requires the use of a modified extraction protocol which prevents procyanidins from binding the RNA <a href="#pone.0096342-Todd2">[65]</a>, <a href="#pone.0096342-Wang3">[66]</a>. RNA was extracted from 200 mg of freeze-dried seed coats by using phenol chloroform extraction and lithium chloride precipitation supplemented with PVPP (polyvinylpyrrolidone), polyproline and BSA (bovine serum albumin). Library construction and high-throughput sequencing was carried out using RNA-Seq technology using Illumina GaII and HiSeq2000 instruments by Keck Center, University of Illinois. </p> </div> <div class="rna-seqalignmentanddatanormalization" title="sec"> <div class="title" tagx="title" title="title">
RNA-Seq Alignment and Data Normalization</div> <p>The 75 bp and 100bp reads were mapped to the 78,773 Glyma cDNA gene models (JGI/Phytozome) using Bowtie <a href="#pone.0096342-Langmead1">[11]</a> with up to 3 mismatches allowed and up to 25 alignments. The total number of generated raw reads for Clark standard and defective isolines and for Harosoy isolines at three different stages of seed development is presented in <a href="#pone-0096342-t001">Table 1</a>. RNA-Seq data were normalized in reads per kilobase of gene model per million mapped reads (RPKM), as the RNA-Seq depends on the transcript length since the reads will map to all positions of the transcript <a href="#pone.0096342-Mortazavi1">[18]</a>. </p> <div class="annotationofglymagenemodels" title="sec"> <div class="title" tagx="title" title="title">
Annotation of glyma gene models</div> <p>Coding region gene models were collected from the masked soybean genome from Phytozome version 6.0 and the soybean genome version 1.0 GFF file of the soybean genome <a href="#pone.0096342-Schmutz1">[12]</a> which is available at SoyBase <a href="#pone.0096342-Grant1">[67]</a>. In addition to the PFAM annotations that were downloaded from Phytozome, the 78,773 models (that include both high and low confidence models) were further annotated using BLASTX against the non-redundant (nr) database of the National Center for Biotechnology Information (NCBI), and for trEMBL and Swiss prot of the European Bioinformatics Institute (EBI) as described by Hunt <i>et al</i>. <a href="#pone.0096342-Hunt1">[68]</a>. </p> </div> <div class="deseqdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
DESeq data analysis</div> <p>DESeq (available via Bioconductor) is an R package to analyze count data from high-throughput sequencing data such as RNA-Seq and test for differential expression <a href="#pone.0096342-Anders1">[69]</a>. The basemean value and p values for each Glyma model were analyzed using this package for 78,773 Glyma models using comparison between Clark standard and defective or Harosoy standard and defective isolines. </p> </div> </div> <div class="northernblots" title="sec"> <div class="title" tagx="title" title="title">
Northern Blots</div> <p>Total RNA was extracted from the frozen tissue including 4–5 day old soybean hypocotyls minus their cotyledons and seed coats of 100–200 mg seeds from four lines using a standard phenol chloroform method with lithium chloride precipitation <a href="#pone.0096342-McCarty1">[64]</a> or the modified method for pigmented tissues <a href="#pone.0096342-Todd2">[65]</a>, <a href="#pone.0096342-Wang3">[66]</a>. RNA samples were quantified by spectrophotometer and the integrity was confirmed using agarose gel electrophoresis. RNA was stored at −80°C until further use. For RNA gel blot analysis, 10 µg of total RNA was electrophoresed through 1.2% agarose/1.1% formaldehyde gels <a href="#pone.0096342-Sambrook1">[70]</a> blotted onto nitrocellulose membranes (Schleicher &amp;amp; Schuell, Keene, NH) via capillary action with 10× SSC (1.5 M NaCl and 0.15 M sodium citrate, pH = 7) overnight. After blotting, RNA was cross-linked to the nitrocellulose membranes with UV radiation by a UV cross-linker (Stratagene, La Jolla, CA). Nitrocellulose RNA gel blots were then prehybridized, hybridized, washed, and exposed to Hyperfilm (Amersham, Piscataway,NJ) as described by Todd and Vodkin <a href="#pone.0096342-Todd1">[14]</a>. An ADR12 gene EST was (Gm-r1088-6964) used as probe and labeled with [α- <sup>32</sup>P]dATP by random primer reaction method <a href="#pone.0096342-Feinberg1">[71]</a>. </p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession Numbers</div> <p>The data have been entered into Gene Expression Omnibus at that National Center for Biotechnology Information as Accession Series GSE54903. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p>The Distribution of Expression Levels in RPKMs of Differentially Expressed Genes in the Clark Isoline Pair at Three Different Seed Weight Stages. (A) 50–100 mg in Clark Standard (B) 50–100 mg in Clark Defective (C) 100–200 mg in Clark Standard (D) 100–200 mg in Clark Defective (E) 400–500 mg Clark Standard (F) 400–500 mg Clark Defective. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s001.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p>The Distribution of Expression Levels of Differentially Expressed Genes in the Harosoy Isoline Pair at Three Different Seed Weight Stages. (A) 50–100 mg in Harosy Standard (B) 50–100 mg in Harosoy Defective (C) 100–200 mg in Harosoy Standard (D) 100–200 mg in Harosoy Defective (E) 400–500 mg Harosoy Standard (F) 400–500 mg Harosoy Defective. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s002.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p>The Nucleotide and Amino Acid Sequence Alignment for PRP1. (A) Alignment of PRP1 nucleotide sequence from NCBI (J02746) and PRP1 sequence (Glyma09g12200.1) from Phytozome database (B) Alignment of PRP1 amino acid sequence from NCBI with PRP1 amino acid sequence from Phytozome database. The nucleotide and amino acid sequence from <i>Williams</i> (Phytozome) and <i>Wayne</i> (NCBI) cultivars showed 100% sequence homology. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s003.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S4</span>  <p>The Nucleotide and Amino Acid Sequence Alignment of the PRP2 Gene. (A) Alignment of PRP2 nucleotide sequence from NCBI (J05208) and PRP2 sequence from Phytozome database (Glyma09g12260.1) (B) Alignment of PRP2 amino acid sequence from NCBI with PRP2 amino acid sequence from Phytozome database. There is difference of one tandem repeat in the amino acid sequence. The PRP2 protein is shorter in Williams (Phytozome) as compared to Wayne (NCBI) cultivar. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s004.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S5</span>  <p>The Genome Organization of PRP1 and PRP2. As in the Phytozome database, PRP1 and PRP2 are on chromosome 9 and the distance between these genes is ∼146 kb. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s005.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S6</span>  <p>The Expression Pattern of 13 Selected Differentially Expressed Genes in the Seed Coat of Wildtype and Defective Isolines in Both Clark and Harosoy Backgrounds. The genes over expressed in the defective isolines ( <a href="#pone-0096342-t003">Tables 3a</a>, <a href="#pone-0096342-t004">4a</a> and <a href="#pone.0096342.s021">Table S3</a>). CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s006.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S7</span>  <p>The Expression Pattern of 16 Selected Differentially Expressed Genes in the Seed Coat of Wildtype and Defective Isolines in Both Clark and Harosoy Background. The genes overexpressed in the Standard isolines ( <a href="#pone-0096342-t003">Tables 3b</a>, <a href="#pone-0096342-t004">4b</a> and <a href="#pone.0096342.s021">Table S3</a>). CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s007.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S8</span>  <p>The Expression Pattern of 11 Fasciclin-like Arabinogalactan Gene in the Seed Coat of Wildtype and Defective Isolines in Both Clark and Harosoy Background. These genes showed higher expression in defective isoline as compare to wildtype isoline ( <a href="#pone.0096342.s021">Table S3</a>). CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s008.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S9</span>  <p>The Expression Pattern of Cell Wall Genes Overexpressed in the Seed Coats of Defective Isolines in Both Clark and Harosoy Background. These graphs present data for 25 additional cell wall related genes not previously shown in <a href="#pone.0096342.s006">Figure S6</a> and <a href="#pone.0096342.s007">S7</a>. Their RPKM and p value data are presented in <a href="#pone.0096342.s021">Table S3</a>. CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s009.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S10</span>  <p>The Expression Pattern of Cell Wall Genes Overexpressed in the Standard Seed Coats of Both Clark and Harosoy Background. These graphs present data for 3 additional cell wall related genes not previously shown in <a href="#pone.0096342.s006">Figure S6</a> and <a href="#pone.0096342.s007">S7</a>. Their RPKM and p value data are presented in <a href="#pone.0096342.s021">Table S3</a>. CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s010.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S11</span>  <p>The Distribution of Differentially Expressed Genes in the Seed Coats of Either (A) Clark or (B) Harosoy Backgrounds at the 50–100 mg Seed Weight Stage. The number of differentially expressed genes (≥5RPKM, ≥2 fold differential expression, p-value≤0.05) was (A) 720 genes in Clark isolines and (B) 48 genes in Harosoy isolines. In both backgrounds, there were major categories related to the cell wall. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s011.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S12</span>  <p>The Distribution of Differentially Expressed Genes in Seed Coats of Either (A) Clark or (B) Harosoy Backgrounds at the 100–200 mg Seed Weight Stage. The number of differentially expressed genes (≥5RPKM, ≥2 fold differential expression, p-value ≤0.05) was (A) 173 genes in Clark isolines and (B) 156 genes in Harosoy isolines at 100–200 mg seed weight stage. In both backgrounds, one of the major categories was related to the cell wall. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s012.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S13</span>  <p>The Distribution of Differentially Expressed Genes in Seed Coats of (A) Clark or (B) Harosoy Backgrounds at the 400–500 mg Seed Weight Stage. The number of differentially expressed genes (≥5RPKM, ≥2 fold differential expression, p-value ≤0.05) was (A) 417 genes in Clark isolines and (B) 1068 genes in Harosoy isolines. In both backgrounds, the cell wall related genes occupied 5% of the chart area. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s013.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S14</span>  <p>The Expression Pattern of Differentially Expressed Transcription Factor Genes That Showed Higher Expression in the Seed Coat of Defective Isolines in Both Clark and Harosoy Background. Overexpressed in defective isolines. CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s014.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S15</span>  <p>The Expression Pattern of Differentially Expressed Transcription Factor Genes That Showed Higher Expression in the Seed Coat of Standard Isolines in Both Clark and Harosoy Background. Overexpressed in standard isolines. CS: Clark Standard, CD: Clark Defective, HS: Harosoy Standard, HD: Harosoy Defective. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0096342.s015.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S16</span>  <p>The Distribution of Differentially Expressed Transcription Factor Genes into Different Classes in Clark at the 50–100 mg Seed Weight Stage. The 59 differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression, p-value ≤0.05) were divided into 12 classes based on functional annotations. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s016.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S17</span>  <p>The Distribution of Differentially Expressed Transcription Factor Genes in (A) Clark or (B) Harosoy Backgrounds at the 100–200 mg Seed Weight Stage. (A) The 16 differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression, p-value≤.05) were divided into 9 different classes (B) The 12 differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression, p-value ≤0.05) were divided into 10 different classes based on functional annotation. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s017.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S18</span>  <p>The Distribution of Differentially Expressed Transcription Factor Genes in Different Classes in (A) Clark and (B) Harosoy Backgrounds at the 400–500 mg Seed Weight Stage. (A) The 43 differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression, p-value ≤.05) were divided into 10 different classes (B) The 111 differentially expressed transcription factor genes (≥5RPKM, ≥2 fold differential expression, p-value ≤0.05) were divided into 15 different classes based on functional annotation. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0096342.s018.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p>The Total Number of Expressed Genes in Seed Coats of the Different Isolines. The total number of expressed genes at ≥10 RPKM or ≥1 RPKM in different backgrounds at different seed weight stages are presented in this table. CS: Clark standard, CD: Clark defective, HS: Harosoy Standard and HD: Harosoy defective. </p> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0096342.s019.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p>Significantly Differentially Expressed Genes Found in the Seed Coats of Either Clark or Harosoy Backgrounds at the Indicated Stages of Seed Development. </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0096342.s020.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S3</span>  <p>The 364 Significantly Differentially Expressed Genes Found in Common in Both Clark and Harosoy Backgrounds as Defined in <a href="#pone-0096342-t002">Table 2</a>. </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0096342.s021.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S4</span>  <p>The 109 Significantly Differentially Expressed Genes Common in Both Background But with Expression in Opposite Directions as Described in <a href="#pone-0096342-t002">Table 2</a>. </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0096342.s022.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We are grateful to Dr. Alvaro Hernandez and the staff of Keck center at University of Illinois, Urbana-Champaign for sequencing services. We thank undergraduate assistants Cameron Lowe and Achira Kulasekara for help with data analysis. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0096342-Vorwerk1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vorwerk</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somerville</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somerville</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">The role of plant cell wall polysaccharide composition in disease resistance</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">203</span>– <span class="lpage" tagx="lpage" title="lpage">209</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15063871">15063871</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Baskin1" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baskin</span> <span class="given-names" tagx="given-names" title="given-names">TI</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Anisotropic expansion of the plant cell wall</span>. <span class="source" tagx="source" title="source">Annu Rev Cell Dev Biol</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">203</span>– <span class="lpage" tagx="lpage" title="lpage">222</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16212493">16212493</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Somerville1" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somerville</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bauer</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brininstool</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Facette</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamann</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Toward a systems approach to understanding plant cell walls</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">306</span>: <span class="fpage" tagx="fpage" title="fpage">2206</span>– <span class="lpage" tagx="lpage" title="lpage">2211</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15618507">15618507</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Tsukaya1" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsukaya</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beemster</span> <span class="given-names" tagx="given-names" title="given-names">GT</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Genetics, cell cycle and cell expansion in organogenesis in plants</span>. <span class="source" tagx="source" title="source">J Plant Res</span> <span class="volume" tagx="volume" title="volume">119</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16365786">16365786</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Corner1" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corner</span> <span class="given-names" tagx="given-names" title="given-names">EJH</span> </span> ( <span class="year" tagx="year" title="year">1951</span>) <span class="mixed-article-title" title="mixed-article-title">The leguminous seed</span>. <span class="source" tagx="source" title="source">Phytomorphology</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">117</span>– <span class="lpage" tagx="lpage" title="lpage">150</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Miller1" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">L-AA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">GijzenŒ</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miki</span> <span class="given-names" tagx="given-names" title="given-names">BLA</span> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">Early Development of the Seed Coat of Soybean ( <i>Glycine max</i>) </span>. <span class="source" tagx="source" title="source">Annals of Botany</span> <span class="volume" tagx="volume" title="volume">84</span>: <span class="fpage" tagx="fpage" title="fpage">297</span>– <span class="lpage" tagx="lpage" title="lpage">304</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Williams1" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Williams LF, editor (1950) Structure and genetic characteristics of the soybean. New York: Interscience Publishers. 111–134 p. </span> </li> <li tag="ref"> <a name="pone.0096342-Ritchie1" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ritchie SW, Hanway JJ, Thomspon HE, Benson GO (1996) How a soybean plant develops. IA: Iowa state univ. of science and Tech. Coop. Ext. serv. 53 Ames 53 Ames. </span> </li> <li tag="ref"> <a name="pone.0096342-Bewley1" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bewley JD, Hempel FD, McCormick S, Zambryski P, editors (2000) Reproductive development. In: Biochemistry and Molecular biology of Plants. Rockville, MD: American society of plant physiologists. 988–1043 p. </span> </li> <li tag="ref"> <a name="pone.0096342-Percy1" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Percy</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Philip</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">A defective seed coat pattern (Net) is correlated with the post-transcriptional abundance of soluble proline-rich cell wall proteins </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">603</span>– <span class="lpage" tagx="lpage" title="lpage">613</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10480384">10480384</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Langmead1" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pop</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Ultrafast and memory efficient alignment of short DNA sequences to the human genome</span>. <span class="source" tagx="source" title="source">Genome Biol</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">R25</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174">19261174</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Schmutz1" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="mixed-article-title" title="mixed-article-title">Schmutz J, Cannon S, Schlueter J, Ma J, Mitros T, et al. (2010 ) Genome sequence of the palaeopolyploid soybean </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">463</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913">20075913</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-PestanaCalsa1" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pestana-Calsa</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pacheco</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castro</span> <span class="given-names" tagx="given-names" title="given-names">RCd</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Almeida</span> <span class="given-names" tagx="given-names" title="given-names">RRd</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lira</span> <span class="given-names" tagx="given-names" title="given-names">NPd</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Cell wall, lignin and fatty acid-related transcriptome in soybean: Achieving gene expression patterns for bioenergy legume </span>. <span class="source" tagx="source" title="source">Genet Mol Biol</span> <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">322</span>– <span class="lpage" tagx="lpage" title="lpage">330</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22802717">22802717</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Todd1" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todd</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1996</span>) <span class="mixed-article-title" title="mixed-article-title">Duplications That Suppress and Deletions That Restore Expression from a Chalcone Synthase Multigene Family </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">687</span>– <span class="lpage" tagx="lpage" title="lpage">699</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12239396">12239396</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Tuteja1" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabala</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varala</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hudson</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Endogenous, tissue-specific short interfering RNAs silence the chalcone synthase gene family in Glycine max seed coats </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3063</span>– <span class="lpage" tagx="lpage" title="lpage">3077</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19820189">19820189</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Gillman1" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gillman</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tetlow</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shannon</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bilyeu</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Loss-of-function mutations affecting a specific Glycine max R2R3 MYB transcription factor result in brown hilum and brown seed coats </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21205309">21205309</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Zabala1" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabala</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Cloning of the pleiotropic T locus in soybean and two recessive alleles that differentially affect structure and expression of the encoded flavonoid 3' hydroxylase </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">163</span>: <span class="fpage" tagx="fpage" title="fpage">295</span>– <span class="lpage" tagx="lpage" title="lpage">309</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12586717">12586717</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Mortazavi1" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mortazavi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCue</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaeffer</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wold</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">621</span>– <span class="lpage" tagx="lpage" title="lpage">628</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045">18516045</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-MacMillan1" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MacMillan</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mansfield</span> <span class="given-names" tagx="given-names" title="given-names">SD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stachurski</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evans</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Southerton</span> <span class="given-names" tagx="given-names" title="given-names">SG</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Fasciclin-like arabinogalactan proteins: specialization for stem biomechanics and cell wall architecture in Arabidopsis and Eucalyptus </span>. <span class="source" tagx="source" title="source">Plant J 1</span> <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">689</span>– <span class="lpage" tagx="lpage" title="lpage">703</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Han1" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">YY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">AX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization of a wheat ( <i>Triticum aestivum</i> L.) expansin gene, TaEXPB23, involved in the abiotic stress response and phytohormone regulation </span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span> <span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">58</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22381655">22381655</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Chou1" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chou</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pogorelko</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabotina</span> <span class="given-names" tagx="given-names" title="given-names">OA</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Xyloglucan Xylosyltransferases XXT1, XXT2, and XXT5 and the Glucan Synthase CSLC4 Form Golgi-Localized Multiprotein Complexes </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">159</span>: <span class="fpage" tagx="fpage" title="fpage">1355</span>– <span class="lpage" tagx="lpage" title="lpage">1366</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22665445">22665445</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Lois1" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lois</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campos</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boronat</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Carotenoid biosynthesis during tomato fruit development: Regulatory role of 1-deoxy-D-xylulose 5-phosphate synthase </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">503</span>– <span class="lpage" tagx="lpage" title="lpage">513</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10886770">10886770</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Bailey1" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bailey RW, Chesson A, Monro J (1978) Plant cell wall fractionation and structural analysis. Am J Clin Nutr (10 Suppl): S77–S81. </span> </li> <li tag="ref"> <a name="pone.0096342-Showalter1" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Showalter</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Structure and Function of Plant Cell Wall Proteins</span>. <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">9</span>– <span class="lpage" tagx="lpage" title="lpage">23</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8439747">8439747</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Cosgrove1" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosgrove</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Loosening of plant cell walls by expansins</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">407</span>: <span class="fpage" tagx="fpage" title="fpage">321</span>– <span class="lpage" tagx="lpage" title="lpage">326</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11014181">11014181</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Lamport1" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieliszewski</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Structural proteins of the primary cell wall: extraction, purification, and analysis</span>. <span class="source" tagx="source" title="source">Methods Mol Biol</span> <span class="volume" tagx="volume" title="volume">2011</span>: <span class="fpage" tagx="fpage" title="fpage">209</span>– <span class="lpage" tagx="lpage" title="lpage">219</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Ringli1" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ringli</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keller</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryser</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Glycine-rich proteins as structural components of plant cell walls</span>. <span class="source" tagx="source" title="source">Cell Mol Life Sci</span> <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">1430</span>– <span class="lpage" tagx="lpage" title="lpage">1441</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11693524">11693524</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Ryser1" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryser</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keller</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">1992</span>) <span class="mixed-article-title" title="mixed-article-title">Ultrastructural localization of a bean glycine-rich protein in unlignified primary walls of protoxylem cells </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">773</span>– <span class="lpage" tagx="lpage" title="lpage">783</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12297662">12297662</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Mangeon1" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mangeon</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Junqueira</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sachetto-Martins</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Functional diversity of the plant glycine-rich proteins superfamily</span>. <span class="source" tagx="source" title="source">Plant Signal Behav</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">99</span>– <span class="lpage" tagx="lpage" title="lpage">104</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20009520">20009520</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Yang1" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Min</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Expression profile analysis of genes involved in cell wall regeneration during protoplast culture in cotton by suppression subtractive hybridization and macroarray </span>. <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">3661</span>– <span class="lpage" tagx="lpage" title="lpage">3674</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18775953">18775953</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Cosgrove2" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosgrove</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bedinger</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Durachko</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> ( <span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">Group I allergens of grass pollen as cell wall-loosening agents</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">6559</span>– <span class="lpage" tagx="lpage" title="lpage">6564</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Dafoe1" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dafoe NJ, Gowen BE, Constabel CP (2010) Thaumatin-like proteins are differentially expressed and localized in phloem tissues of hybrid poplar. BMC Plant Biol 26. </span> </li> <li tag="ref"> <a name="pone.0096342-Wu1" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">May</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheung</span> <span class="given-names" tagx="given-names" title="given-names">AY</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">A tobacco gene family for flower cell wall proteins with a proline-rich domain and a cysteine-rich domain </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">6829</span>– <span class="lpage" tagx="lpage" title="lpage">6833</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Zhao1" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kositsup</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beers</span> <span class="given-names" tagx="given-names" title="given-names">EP</span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Exploiting Secondary Growth in Arabidopsis. Construction of Xylem and Bark cDNA Libraries and Cloning of Three Xylem Endopeptidases1 </span>. <span class="source" tagx="source" title="source">Plant Physiology</span> <span class="volume" tagx="volume" title="volume">123</span>: <span class="fpage" tagx="fpage" title="fpage">1185</span>– <span class="lpage" tagx="lpage" title="lpage">1196</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10889267">10889267</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Evans1" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evans</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span> ( <span class="year" tagx="year" title="year">1985</span>) <span class="mixed-article-title" title="mixed-article-title">The action of auxin on plant cell elongation</span>. <span class="source" tagx="source" title="source">CRC Crit Rev Plant Sci</span> <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">317</span>– <span class="lpage" tagx="lpage" title="lpage">365</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11539698">11539698</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Overvoorde1" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Overvoorde</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okushima</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Functional genomic analysis of the AUXIN/INDOLE-3-ACETIC ACID gene family members in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">3282</span>– <span class="lpage" tagx="lpage" title="lpage">3300</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16284307">16284307</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Gou1" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strauss</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Gibberellins regulate lateral root formation in Populus through interactions with auxin and other hormones </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">623</span>– <span class="lpage" tagx="lpage" title="lpage">639</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20354195">20354195</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Denness1" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Denness</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKenna</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Segonzac</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wormit</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madhou</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Cell wall damage-induced lignin biosynthesis is regulated by a reactive oxygen species- and jasmonic acid-dependent process in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">156</span>: <span class="fpage" tagx="fpage" title="fpage">1364</span>– <span class="lpage" tagx="lpage" title="lpage">1374</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21546454">21546454</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Creelman1" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Creelman</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tierney</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mullet</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> ( <span class="year" tagx="year" title="year">1992</span>) <span class="mixed-article-title" title="mixed-article-title">Jasmonic acid/methyl jasmonate accumulate in wounded soybean hypocotyls and modulate wound gene expression </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A 1</span> <span class="volume" tagx="volume" title="volume">89</span>: <span class="fpage" tagx="fpage" title="fpage">4938</span>– <span class="lpage" tagx="lpage" title="lpage">4941</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Marcus1" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marcus</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenberg</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Averyhart-Fullard</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> ( <span class="year" tagx="year" title="year">1991</span>) <span class="mixed-article-title" title="mixed-article-title">Repetive proline-rich proteins in the extracellular matrix of the plant cell</span>. <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="volume" tagx="volume" title="volume">81</span>: <span class="fpage" tagx="fpage" title="fpage">273</span>– <span class="lpage" tagx="lpage" title="lpage">279</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Hong1" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagao</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Key</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> ( <span class="year" tagx="year" title="year">1987</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization and sequence analysis of a developmentally regulated putative cell wall protein gene isolated from soybean </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">8367</span>– <span class="lpage" tagx="lpage" title="lpage">8376</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Hong2" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagao</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Key</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> ( <span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">Developmentally regulated expression of soybean proline-rich cell wall protein genes</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">937</span>– <span class="lpage" tagx="lpage" title="lpage">943</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2535533">2535533</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Hong3" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagao</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Key</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> ( <span class="year" tagx="year" title="year">1990</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization of a proline-rich cell wall protein gene family of soybean. A comparative analysis </span>. <span class="source" tagx="source" title="source">J Biol Chem 15</span> <span class="volume" tagx="volume" title="volume">265</span>: <span class="fpage" tagx="fpage" title="fpage">2470</span>– <span class="lpage" tagx="lpage" title="lpage">2475</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Lindstrom1" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindstrom</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1991</span>) <span class="mixed-article-title" title="mixed-article-title">A soybean cell wall protein is affected by seed color genotype</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">561</span>– <span class="lpage" tagx="lpage" title="lpage">571</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1841717">1841717</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Datta1" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Datta</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marcus</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">Characterization of two soybean repetitive proline-rich proteins and a cognate cDNA from germinated axes </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">945</span>– <span class="lpage" tagx="lpage" title="lpage">952</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2535534">2535534</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Goodstein1" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodstein</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howson</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neupane</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Phytozome: a comparative platform for green plant genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">D1178</span>– <span class="lpage" tagx="lpage" title="lpage">1186</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026">22110026</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Schmidt1" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindstrom</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1994</span>) <span class="mixed-article-title" title="mixed-article-title">Genetic length polymorphisms create size variation in proline-rich proteins of the cell wall </span>. <span class="source" tagx="source" title="source">The Plant Journal</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">177</span>– <span class="lpage" tagx="lpage" title="lpage">186</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7920710">7920710</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Nicholas1" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nicholas</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindstrom</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Variation of proline rich cell wall proteins in soybean lines with anthocyanin mutations</span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">145</span>– <span class="lpage" tagx="lpage" title="lpage">156</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8425044">8425044</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Zhong1" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">A Battery of Transcription Factors Involved in the Regulation of Secondary Cell Wall Biosynthesis in Arabidopsis </span>. <span class="source" tagx="source" title="source">The Plant Cell</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">2763</span>– <span class="lpage" tagx="lpage" title="lpage">2782</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18952777">18952777</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Preston1" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preston</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wheeler</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heazlewood</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parish</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">AtMYB32 is required for normal pollen development in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">979</span>– <span class="lpage" tagx="lpage" title="lpage">995</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15584962">15584962</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Penfield1" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penfield</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meissner</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoue</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carpita</span> <span class="given-names" tagx="given-names" title="given-names">NC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bevan</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">MYB61 is required for mucilage deposition and extrusion in the Arabidopsis seed coat</span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">2777</span>– <span class="lpage" tagx="lpage" title="lpage">2791</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11752387">11752387</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Western1" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Western</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dean</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">WL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samuels</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">MUCILAGE-MODIFIED4 encodes a putative pectin biosynthetic enzyme developmentally regulated by APETALA2, TRANSPARENT TESTA GLABRA1, and GLABRA2 in the Arabidopsis seed coat </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">134</span>: <span class="fpage" tagx="fpage" title="fpage">296</span>– <span class="lpage" tagx="lpage" title="lpage">306</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14701918">14701918</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Zhong2" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">MYB46 and MYB83 bind to the SMRE sites and directly activate a suite of transcription factors and secondary wall biosynthetic genes </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">368</span>– <span class="lpage" tagx="lpage" title="lpage">380</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197883">22197883</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Zhou1" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">ZH</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">MYB58 and MYB63 are transcriptional activators of the lignin biosynthetic pathway during secondary cell wall formation in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">248</span>– <span class="lpage" tagx="lpage" title="lpage">266</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19122102">19122102</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Xu1" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gondwe</span> <span class="given-names" tagx="given-names" title="given-names">MY</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">The ABORTED MICROSPORES regulatory network is required for postmeiotic male reproductive development in <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">91</span>– <span class="lpage" tagx="lpage" title="lpage">107</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20118226">20118226</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Martel1" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martel</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vrebalov</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tafelmeyer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giovannoni</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The tomato MADS-box transcription factor RIPENING INHIBITOR interacts with promoters involved in numerous ripening processes in a COLORLESS NONRIPENING-dependent manner </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">157</span>: <span class="fpage" tagx="fpage" title="fpage">1568</span>– <span class="lpage" tagx="lpage" title="lpage">1579</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21941001">21941001</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Gibalov1" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibalová</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renák</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matczuk</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dupl'áková</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cháb</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">AtbZIP34 is required for Arabidopsis pollen wall patterning and the control of several metabolic pathways in developing pollen </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">70</span>: <span class="fpage" tagx="fpage" title="fpage">581</span>– <span class="lpage" tagx="lpage" title="lpage">601</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19449183">19449183</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Wang1" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avci</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Mutation of WRKY transcription factors initiates pith secondary wall formation and increases stem biomass in dicotyledonous plants </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">107</span>: <span class="fpage" tagx="fpage" title="fpage">22338</span>– <span class="lpage" tagx="lpage" title="lpage">22343</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21135241">21135241</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Hussey1" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hussey</span> <span class="given-names" tagx="given-names" title="given-names">SG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizrachi</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spokevicius</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bossinger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berger</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">SND2, a NAC transcription factor gene, regulates genes involved in secondary cell wall development in Arabidopsis fibers and increases fiber cell area in Eucalyptus </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">173</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Wang2" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">On–Off Switches for Secondary Cell Wall Biosynthesis</span>. <span class="source" tagx="source" title="source">Molecular Plant</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">297</span>– <span class="lpage" tagx="lpage" title="lpage">303</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22138968">22138968</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Zhong3" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reeves</span> <span class="given-names" tagx="given-names" title="given-names">CK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Transcriptional activation of secondary wall biosynthesis by rice and maize NAC and MYB transcription factors </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">1856</span>– <span class="lpage" tagx="lpage" title="lpage">1871</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21908441">21908441</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Lasserre1" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lasserre</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jobet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llauro</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delseny</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">AtERF38 (At2g35700), an AP2/ERF family transcription factor gene from <i>Arabidopsis thaliana</i>, is expressed in specific cell types of roots, stems and seeds that undergo suberization </span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span> <span class="volume" tagx="volume" title="volume">46</span>: <span class="fpage" tagx="fpage" title="fpage">1051</span>– <span class="lpage" tagx="lpage" title="lpage">1061</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18723362">18723362</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Sagasser1" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sagasser</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahlbrock</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weisshaar</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>A. thaliana</i> TRANSPARENT TESTA 1 is involved in seed coat development and defines the WIP subfamily of plant zinc finger proteins </span>. <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">138</span>– <span class="lpage" tagx="lpage" title="lpage">149</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11782451">11782451</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-McCarty1" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarty</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> ( <span class="year" tagx="year" title="year">1986</span>) <span class="mixed-article-title" title="mixed-article-title">A simple method for extraction of RNA from maize tissue</span>. <span class="source" tagx="source" title="source">Maize Genet Coop Newsl</span> <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">61</span>. </span> </li> <li tag="ref"> <a name="pone.0096342-Todd2" /> <span class="label" tagx="label" title="label">65</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todd</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Pigmented Soybean (Glycine max) Seed Coats Accumulate Proanthocyanidins during Development</span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">102</span>: <span class="fpage" tagx="fpage" title="fpage">663</span>– <span class="lpage" tagx="lpage" title="lpage">670</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12231856">12231856</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Wang3" /> <span class="label" tagx="label" title="label">66</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todd</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">1994</span>) <span class="mixed-article-title" title="mixed-article-title">Chalcone synthase mRNA and activity are reduced in yellow soybean seed coats with dominant I alleles </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">739</span>– <span class="lpage" tagx="lpage" title="lpage">748</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8066134">8066134</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Grant1" /> <span class="label" tagx="label" title="label">67</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Grant D, Nelson R, Cannon S, Shoemaker R (2010) SoyBase, the USDA-ARS soybean genetics and genomics database. Nucleic Acids Res (Database issue): D843–846. </span> </li> <li tag="ref"> <a name="pone.0096342-Hunt1" /> <span class="label" tagx="label" title="label">68</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stromvik</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vodkin</span> <span class="given-names" tagx="given-names" title="given-names">LO</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Transcript profiling reveals expression differences in wild-type and glabrous soybean lines </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">145</span>– <span class="lpage" tagx="lpage" title="lpage">160</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22029708">22029708</a> </span> </span> </li> <li tag="ref"> <a name="pone.0096342-Anders1" /> <span class="label" tagx="label" title="label">69</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Anders S, Huber W (2010) Differential expression analysis for sequence count data. Genome Biol 11. </span> </li> <li tag="ref"> <a name="pone.0096342-Sambrook1" /> <span class="label" tagx="label" title="label">70</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sambrook J, Fritsch EF, Maniatis T, editors (1989) Molecular Cloning: A laboratory manual. NY: Cold spring Harbor. </span> </li> <li tag="ref"> <a name="pone.0096342-Feinberg1" /> <span class="label" tagx="label" title="label">71</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feinberg</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogelstein</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> ( <span class="year" tagx="year" title="year">1983</span>) <span class="mixed-article-title" title="mixed-article-title">A technique for radiolabeling DNA restriction endonuclease fragments to high specific activity </span>. <span class="source" tagx="source" title="source">Anal Biochem</span> <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">6</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6312838">6312838</a> </span> </span> </li> </ul> </div> </div>  </body></html>