<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5811567">5811567</a> </span> <span class="publisher-id" title="publisher-id">21193</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-018-21193-4">10.1038/s41598-018-21193-4</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Characterization of Conserved and Novel microRNAs in <i>Lilium lancifolium</i> Thunb. by High-Throughput Sequencing </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">Xiangfeng</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shenkute</span> <span class="given-names" tagx="given-names" title="given-names">Awraris Getachew</span> </span> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Wenhe</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">wwhals@163.com</span> </span> <a href="#Aff1">1</a> <a href="#Aff2">2</a> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Shufa</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">xushufa@caas.cn</span> </span> <a href="#Aff4">4</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1798 6793</span> <span class="institution-id" title="institution-id">grid.411626.6</span> <span class="institution" title="institution">College of Landscape Architecture, </span> <span class="institution" title="institution">Beijing University of Agriculture, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], </span> <span class="citation_author_institution" id="Aff3">[3], </span> <span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0526 1937</span> <span class="institution-id" title="institution-id">grid.410727.7</span> <span class="institution" title="institution">Key Laboratory of Pollinating Insect Biology, Ministry of Agriculture, Institute of Apicultural Research, </span> <span class="institution" title="institution">Chinese Academy of Agricultural Sciences, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-2-2</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-2-2</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">2880</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-6-5</span> <span class="accepted" title="accepted">accepted: 2018-1-31</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>MicroRNAs (miRNAs) are among the class of noncoding small RNA molecules and play a crucial role in post-transcriptional regulation in plants. Although <i>Lilium</i> is one of the most popular ornamental flowers worldwide, however, there is no report on miRNAs identification. In the present study, therefore, miRNAs and their targets were identified from flower, leaf, bulblet and bulb of <i>Lilium lancifolium</i> Thunb. by high-throughput sequencing and bioinformatics analysis. In this study, a total of 38 conserved miRNAs belonging to 17 miRNA families and 44 novel miRNAs were identified. In total, 366 target genes for conserved miRNAs and 415 target genes for novel miRNAs were predicted. The majority of the target genes for conserved miRNAs were transcriptional factors and novel miRNAs targeted mainly protein coding genes. A total of 53 cleavage sites belonging to 6 conserved miRNAs families and 14 novel miRNAs were identified using degradome sequencing. Twenty-three miRNAs were randomly selected, then, their credibility was confirmed using northern blot or stem-loop qRT-PCR. The results from qRT-PCR analysis showed the expression pattern of 4 LL-miRNAs was opposite to their targets. Therefore, our finding provides an important basis to understand the biological functions of miRNAs in <i>Lilium</i>. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>MicroRNAs (miRNAs) are a class of 20–24 nucleotide (nt) noncoding small RNA molecules and play a crucial role in post-transcriptional regulation in animals and plants <sup> <a href="#CR1">1</a>, <a href="#CR2">2</a> </sup>. In plants, microRNA genes are transcribed by RNA polymerase II into primary miRNAs (pri-miRNAs) with a cap and a poly(A) tail. The pri-miRNAs are then processed into hairpin precursors (pre-miRNAs) by a protein complex consisting of the Dicer-like 1 (DCL1), the C2H2-zinc finger protein SERRATE 11(SE), and the double-stranded RNA-binding protein HYPONASTICLEAVES1 (HYL1) <sup> <a href="#CR3">3</a> </sup>. The miRNA duplexes (miRNA/miRNA*) are released from pre-miRNAs by DCL1 and each strand in the miRNA duplex is methylated <sup> <a href="#CR3">3</a> </sup>. The miRNA strand is loaded into the ARGONAUTE (AGO) protein of RNA-induced silencing complex (RISC) to carry out its function <sup> <a href="#CR1">1</a>, <a href="#CR3">3</a> </sup>. Several research evidences revealed that miRNAs play important roles in diverse biological processes including plant growth, development, biotic and abiotic stress responses, and signal transduction <sup> <a href="#CR4">4</a>– <a href="#CR6">6</a> </sup>. </p> <p>The first miRNA, lin-4, was identified from <i>Caenorhabditis elegans</i> in 1993 <sup> <a href="#CR7">7</a> </sup>. In plant, the first miRNAs were identified from <i>Arabidopsis</i> <sup> <a href="#CR8">8</a> </sup>. Following that, some miRNAs have been identified from plants using cloning or bioinformatics prediction <sup> <a href="#CR9">9</a>– <a href="#CR12">12</a> </sup>. The high-throughput sequencing technology was firstly used to identify <i>A.thaliana</i> miRNAs in 2005 <sup> <a href="#CR13">13</a> </sup>. Since then, thousands of miRNAs from different species have been discovered by high-throughput sequencing technology <sup> <a href="#CR14">14</a>– <a href="#CR19">19</a> </sup>. To date, a total of 35,828 mature miRNAs sequences from 223 different species (ranging from viruses to humans) have been identified according to the miRBase database (release 21, June 2014). However, there are only few researches conducted on miRNAs identification in ornamental flowers, including <i>Phalaenopsis aphrodite</i>, <i>Rosa hybrida</i>, <i>Aquilegia coerulea</i> and <i>lotus japonicas</i> <sup> <a href="#CR20">20</a>– <a href="#CR23">23</a> </sup>. </p> <p> <i>Lilium</i> is a genus of Liliaceous perennial bulb plants. Lily ( <i>Lilium</i> spp.), owing to their large and colorful flowers, have become one of the most popular ornamental flowers worldwide <sup> <a href="#CR24">24</a> </sup>. In addition to their ornamental value, some lily species are edible and has long been used as traditional medicine in China and Korea <sup> <a href="#CR25">25</a>– <a href="#CR28">28</a> </sup>. The genus <i>Lilium</i> has more than 100 species worldwide, of which 55 species and 18 varieties are originated in China <sup> <a href="#CR29">29</a> </sup>. <i>Lilium lancifolium</i> Thunb. is a well-known lily species widely distributed in China, and is often planted for landscaping design. It is one of <i>Lilium</i> species formed Asiatic hybrid lilies by interspecific crosses and give them the characteristics of orange flower and raised spot in petal <sup> <a href="#CR30">30</a> </sup>. Several studies have revealed its better capacity for resisting high and low temperature, drought, disease and changing soil salinity than other lilies <sup> <a href="#CR31">31</a>, <a href="#CR32">32</a> </sup>. Therefore, it has been used to produce progeny with desirable stress resistance in lily hybrid breeding <sup> <a href="#CR32">32</a> </sup>. Although several research reports shown that miRNAs play crucial roles in plant growth, development and response to stress in plant, to the best of our knowledge there is currently no report on <i>Lilium</i> miRNAs. In this study, therefore, we employed the high-throughput sequencing and bioinformatics analysis to identify miRNAs and their targets in <i>L. lancifolium</i>. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="constructionandsequencingofsmallrnalibraries" title="sec"> <div class="title" tagx="title" title="title">
Construction and sequencing of small RNA libraries</div> <p>To identify the miRNAs in <i>Lilium</i>, total RNAs were extracted from flower, leaf, bulblet and bulb of <i>L. lancifolium</i> and then used to construct four small RNA libraries (Supplementary Table <a href="#MOESM1">S1</a>). Then the four small RNA libraries were sequenced using Illumina HiSeq. 2500 sequencing platform and analyzed in bioinformatics. A total of 19,025,905 raw reads from flower, 19,636,648 from leaf, 22,776,684 from bulblet, and 18,676,061 from bulb were obtained. After removing adaptors, low quality reads and contaminants, 14,090,897clean reads from flower, 15,279,574 from leaf, 17,560,878 from bulblet and 14,935,394 from bulb were obtained (Table <a href="#Tab1">1</a>). The clean reads and unique reads of four tissues were subjected to analysis of the size distribution as shown in Fig. <a href="#Fig1">1</a>. The majority of the clean reads of small RNAs in four samples were 21 to 26 nt in size. The 21 nt class was the most abundant in flower and bulb, followed by 22, 24 and 26 nt classes (Fig. <a href="#Fig1">1A</a>). In leaf and bulblet 23 nt small RNAs are the most frequent, followed by 26, 22 and 24 nt. However, the 24 nt peak is found to be dominant at a unique read level in all four samples (Fig. <a href="#Fig1">1B</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Statistics of sequencing reads from flower, leaf, bulblet and bulb libraries of <i>L. lancifolium</i>. </p>  <table> <thead> <tr> <th>Samples</th> <th>Raw reads</th> <th>Clean reads (18–30 nt)</th> <th>Unique reads</th> <th>Mapped reads</th> <th>rRNA/tRNA/ snRNA/snoRNA</th> <th>Without annotation</th> </tr> </thead> <tbody> <tr> <td>Flower</td> <td>19,025,905</td> <td>14,090,897</td> <td>1,710,130</td> <td>9,552,373</td> <td>8,453,061</td> <td>5,634,868</td> </tr> <tr> <td>Leaf</td> <td>19,636,648</td> <td>15,279,574</td> <td>1,103,780</td> <td>10,493,835</td> <td>10,263,031</td> <td>5,012,994</td> </tr> <tr> <td>Bulblet</td> <td>22,776,684</td> <td>17,560,878</td> <td>1,394,751</td> <td>12,803,064</td> <td>11,776,773</td> <td>5,764,221</td> </tr> <tr> <td>Bulb</td> <td>18,676,061</td> <td>14,935,394</td> <td>1,965,587</td> <td>88,931,80</td> <td>7,156,402</td> <td>7,774,593</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>The length distribution of the clean and unique reads from flower, leaf, bulblet and bulb of <i>L. lancifolium</i>. ( <b>A</b>) Clean reads; ( <b>B</b>) Unique reads. </p>   </div> <p /> </div> <div class="conservedmirnasinl.lancifolium" title="sec"> <div class="title" tagx="title" title="title">
Conserved miRNAs in <i>L. lancifolium</i> </div> <p>To analyze the population of conserved miRNAs in <i>L. lancifolium</i>, sRNA sequences of the four libraries were compared with known mature miRNAs from other plants and <i>Lilium</i> RNA sequence. A total of 38 conserved miRNAs precursors were identified from four sRNA datasets, as shown in Table <a href="#Tab2">2</a> and Supplementary Table <a href="#MOESM1">S2</a>. All these conserved miRNAs belong to 17 miRNA families and in most miRNA families, more than one precursors were identified. Among them, MIR156 and MIR159 were the largest families identified with five members, followed by MIR166, in which four members were identified. However, several miRNA families possessed only one precursor, including MIR162, MIR167, MIR172, MIR390, MIR399, MIR408 and MIR845. In most miRNA families, at least one conserved miRNA precursor with miRNA* from small RNA sequencing was identified (Supplementary Fig. <a href="#MOESM2">S1</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Conserved miRNAs identified from flower, leaf, bulblet and bulb libraries of <i>L. lancifolium</i>. </p>  <table> <thead> <tr> <th>Family</th> <th>miRNA</th> <th>Number of precursor sequence</th> <th>Sequence (5′-3′)</th> <th>Length (nt)</th> <th>Clean reads</th> </tr> <tr> <th>Bulblet</th> <th>Bulb</th> <th>Flower</th> <th>Leaf</th> </tr> </thead> <tbody> <tr> <td>MIR156</td> <td>LL-miR156a</td> <td>c45184.graph_c0</td> <td>ugacagaagagagugagcac</td> <td>20</td> <td>263</td> <td>29</td> <td>32</td> <td>2</td> </tr> <tr> <td /> <td /> <td>c51027_g1</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>c103364_g2</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>c205656_g1</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR156i</td> <td>c45184.graph_c0</td> <td>ugacagaaagaguagugagca</td> <td>21</td> <td>8</td> <td>1</td> <td>2</td> <td>1</td> </tr> <tr> <td>MIR159</td> <td>LL-miR159a</td> <td>CL2574.Contig1_All</td> <td>uggauugaagggagcucuaca</td> <td>21</td> <td>11</td> <td>13</td> <td>2</td> <td>4</td> </tr> <tr> <td /> <td /> <td>CL2574.Contig4_All</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR159b</td> <td>c117583_g1</td> <td>uuuggauugaagggagcucua</td> <td>21</td> <td>1446</td> <td>969</td> <td>671</td> <td>379</td> </tr> <tr> <td /> <td /> <td>c44060.graph_c0</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR319a</td> <td>c120927_g2</td> <td>uuggacugaagggagcucccu</td> <td>21</td> <td>1</td> <td>2</td> <td>0</td> <td>0</td> </tr> <tr> <td>MIR160</td> <td>LL-miR160a</td> <td>c19797.graph_c0</td> <td>ugccuggcucccuguaugcca</td> <td>21</td> <td>3</td> <td>1</td> <td>8</td> <td>3</td> </tr> <tr> <td /> <td /> <td>c50045_g1</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR160f</td> <td>c99102_g1</td> <td>cugccuggcucccugaaugcc</td> <td>21</td> <td>1</td> <td>18</td> <td>2</td> <td>3</td> </tr> <tr> <td>MIR162</td> <td>LL-miR162a</td> <td>Unigene26825_All</td> <td>ucgauaaaccucugcauccgg</td> <td>21</td> <td>17</td> <td>52</td> <td>50</td> <td>4</td> </tr> <tr> <td>MIR164</td> <td>LL-miR164a</td> <td>c96196_g1</td> <td>uggagaagcagggcacgugca</td> <td>21</td> <td>11</td> <td>0</td> <td>18</td> <td>0</td> </tr> <tr> <td /> <td /> <td>c30311.graph_c0</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>MIR166</td> <td>LL-miR166f</td> <td>Unigene32510_All</td> <td>ucucggaccaggcuucauucc</td> <td>21</td> <td>49</td> <td>40</td> <td>366</td> <td>6</td> </tr> <tr> <td /> <td /> <td>c39808.graph_c0</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>c121414_g2</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR166g</td> <td>c106540_g1</td> <td>ucggaccaggcuucauuccuc</td> <td>21</td> <td>369</td> <td>122</td> <td>259</td> <td>92</td> </tr> <tr> <td>MIR167</td> <td>LL-miR167a</td> <td>c105919_g1</td> <td>ugaagcugccagcaugaucuga</td> <td>21</td> <td>7</td> <td>37</td> <td>1923</td> <td>132</td> </tr> <tr> <td>MIR168</td> <td>LL-miR168a</td> <td>c71740.graph_c0</td> <td>ucgcuuggugcaggucgggaa</td> <td>21</td> <td>40</td> <td>137</td> <td>6</td> <td>4</td> </tr> <tr> <td /> <td /> <td>c24174.graph_c0</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>c105404_g1</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>MIR172</td> <td>LL-miR172a</td> <td>c79803.graph_c0</td> <td>agaaucuugaugaugcugcaa</td> <td>21</td> <td>0</td> <td>0</td> <td>24</td> <td>0</td> </tr> <tr> <td>MIR390</td> <td>LL-miR390a</td> <td>c7928.graph_c0</td> <td>aagcucaggagggauagcgcc</td> <td>21</td> <td>4</td> <td>0</td> <td>16</td> <td>0</td> </tr> <tr> <td>MIR395</td> <td>LL-miR395a</td> <td>c12817.graph_c0</td> <td>ugaaguguuugggggaacucc</td> <td>21</td> <td>208</td> <td>1183</td> <td>105</td> <td>140</td> </tr> <tr> <td /> <td>LL-miR395k</td> <td>c5100.graph_c0</td> <td>ugaagcguuugggggaacucc</td> <td>21</td> <td>0</td> <td>2</td> <td>1</td> <td>1</td> </tr> <tr> <td>MIR396</td> <td>LL-miR396a</td> <td>c103053_g1</td> <td>uuccacagcuuucuugaacug</td> <td>21</td> <td>33</td> <td>123</td> <td>19</td> <td>15</td> </tr> <tr> <td /> <td /> <td>Unigene25901_All</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td>LL-miR396f</td> <td>c30849_g1</td> <td>uuccacggcuuucuugaacua</td> <td>21</td> <td>38</td> <td>78</td> <td>81</td> <td>5</td> </tr> <tr> <td>MIR398</td> <td>LL-miR398b</td> <td>c221661_g1</td> <td>uguguucucaggucaccccug</td> <td>21</td> <td>55</td> <td>70</td> <td>11</td> <td>71</td> </tr> <tr> <td /> <td /> <td>c35683_g1</td> <td /> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>MIR399</td> <td>LL-miR399a</td> <td>c191701_g1</td> <td>ugccaaaggagacuugcccug</td> <td>21</td> <td>2</td> <td>0</td> <td>3</td> <td>5</td> </tr> <tr> <td>MIR408</td> <td>LL-miR408b</td> <td>c121837_g2</td> <td>ugcacugccucuucccuggcu</td> <td>21</td> <td>7</td> <td>0</td> <td>2</td> <td>14</td> </tr> <tr> <td>MIR845</td> <td>LL-miR845</td> <td>c54746.graph_c0</td> <td>cgcucugauaccacuuguugg</td> <td>21</td> <td>1</td> <td>8</td> <td>1</td> <td>0</td> </tr> <tr> <td>MIR2118</td> <td>LL-miR2118e</td> <td>c28582_g1</td> <td>uucccaaugccucucaugccaa</td> <td>22</td> <td>2</td> <td>0</td> <td>10</td> <td>1</td> </tr> <tr> <td /> <td>LL-miR2118a</td> <td>Unigene6944_All</td> <td>uugccgauaccacccauaccga</td> <td>22</td> <td>4</td> <td>6</td> <td>3</td> <td>0</td> </tr> </tbody> </table>  <p /> <p>The sRNA sequencing results indicated that the clean reads of conserved miRNAs ranged from 1 up to more than 1,000 in 4 samples. Of all conserved miRNAs, the clean reads of MIR159, MIR167 and MIR395 exceeded 1,000 in one tissue. The clean reads of four miRNA families (MIR156, MIR166, MIR168 and MIR396) ranged from 100 to 1,000 at least in one tissue. However, the other miRNA families (MIR160, MIR162, MIR164, MIR172, MIR390, MIR398, MIR399, MIR408, MIR845 and MIR2118) had fewer than 100 reads in all four tissues (Table <a href="#Tab2">2</a>). </p> <p>In a broader evolutionary context, <i>L. lancifolium</i> miRNAs were aligned to those of 13 other plants, including 11 monocotyledons ( <i>Aegilops tauschii</i>, <i>Brachypodium distachyon</i>, <i>Elaeis guineensis</i>, <i>Festuca arundinacea</i>, <i>Hordeum vulgare</i>, <i>Oryza sativa</i>, <i>Sorghum bicolor</i>, <i>Saccharum officinarum</i>, <i>Saccharum</i> sp., <i>Triticum aestivum</i> and <i>Zea mays</i>) and 2 dicotyledons ( <i>Arabidopsis thaliana</i> and <i>Glycine max</i>). Of the 17 <i>L. lancifolium</i> miRNA families, 15 were conserved in more than 6 plant species. These miRNAs were considered as well-conserved miRNA families. However, LL-miR845 and LL-miR2118 from <i>L. lancifolium</i> were found in only two and one plant species, respectively (Supplementary Table <a href="#MOESM1">S2</a>). </p> </div> <div class="novelmirnasinl.lancifolium" title="sec"> <div class="title" tagx="title" title="title">
Novel miRNAs in <i>L. lancifolium</i> </div> <p>To identify novel miRNAs that may be specific to <i>L. lancifolium</i>, all unannotated sRNAs were searched against the unigenes from <i>Lilium</i> transcriptome sequencing and EST from NCBI database using miRDeep2. After searching for potential precursors (pre-miRNAs) and predicting their stem-loop hairpin secondary structures, a total of 44 novel miRNAs were identified in four libraries. The novel miRNA sequences ranged in length from 20–24 nt. However, the sequences of most novel miRNAs were 21 nt length and started with a 5′-U (Table <a href="#Tab3">3</a>). The pre-miRNAs ranged in length from 127–1475 nt. The average minimum folding free energy value of the hairpin structures was −152 kcal/mol in <i>L. lancifolium</i> (Supplementary Table <a href="#MOESM1">S3</a>), which is higher than −76.8 kcal/mol found in <i>Arabidopsis</i> <sup> <a href="#CR33">33</a> </sup>. The structures of 44 novel miRNA precursors are shown in Supplementary Fig. <a href="#MOESM2">S2</a>. Nineteen miRNA* sequences from small RNA sequencing were discovered in these novel miRNA precursors. Half of novel miRNAs had more than 10 clean reads in at least one tissue. Only LL-miR09, LL-miR23 and LL-miR25 had more than 100 clean reads in leaf, bulb or flower tissue (Table <a href="#Tab3">3</a>). It is very interesting that two highly similar novel miRNAs, LL-miR07 and LL-miR14, were identified. Only two different bases were found between LL-miR07 and LL-miR14 mature miRNA sequences (Table <a href="#Tab3">3</a>). Moreover, the similarity of precursors between LL-miR07 and LL-miR14 was more than 85%. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Novel miRNAs identified from flower, leaf, bulblet and bulb libraries of <i>L. lancifolium</i>. </p>  <table> <thead> <tr> <th>miRNA</th> <th>Number of precursor sequence</th> <th>Sequence (5′-3′)</th> <th>Length (nt)</th> <th>Clean reads</th> </tr> <tr> <th>Bulblet</th> <th>Bulb</th> <th>Flower</th> <th>Leaf</th> </tr> </thead> <tbody> <tr> <td>LL-miR01</td> <td>c33038.graph_c0</td> <td>uaguaaguuugcagagcagag</td> <td>21</td> <td>0</td> <td>0</td> <td>53</td> <td>0</td> </tr> <tr> <td>LL-miR02</td> <td>c1069.graph_c1</td> <td>cuugugcuucuggacugcucc</td> <td>21</td> <td>0</td> <td>56</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR03</td> <td>c15590.graph_c0</td> <td>aagguauagagucagacacuu</td> <td>20</td> <td>0</td> <td>0</td> <td>9</td> <td>0</td> </tr> <tr> <td>LL-miR04</td> <td>c25608.graph_c0</td> <td>agacgaucgcaccaaacuggcuau</td> <td>24</td> <td>13</td> <td>67</td> <td>1</td> <td>0</td> </tr> <tr> <td>LL-miR05</td> <td>c28639.graph_c0</td> <td>uuuucuaugucacucaauccaa</td> <td>22</td> <td>0</td> <td>2</td> <td>3</td> <td>0</td> </tr> <tr> <td>LL-miR06</td> <td>c30876.graph_c0</td> <td>aguaaguugagaagaguaggagaa</td> <td>24</td> <td>3</td> <td>3</td> <td>0</td> <td>1</td> </tr> <tr> <td>LL-miR07</td> <td>c31471.graph_c0</td> <td>uucacugccaccauccgccugu</td> <td>22</td> <td>1</td> <td>9</td> <td>28</td> <td>1</td> </tr> <tr> <td>LL-miR08</td> <td>c35378.graph_c0</td> <td>cgguugcuuagcuuguacucu</td> <td>21</td> <td>0</td> <td>12</td> <td>1</td> <td>0</td> </tr> <tr> <td>LL-miR09</td> <td>c36638.graph_c0</td> <td>ugcaccuccuccuccuuuucu</td> <td>21</td> <td>56</td> <td>13</td> <td>33</td> <td>244</td> </tr> <tr> <td>LL-miR10</td> <td>CL3742.Contig2_All</td> <td>gguuugaugaaucugagcauc</td> <td>21</td> <td>14</td> <td>19</td> <td>55</td> <td>18</td> </tr> <tr> <td>LL-miR11</td> <td>c53683.graph_c0</td> <td>uuacgugucccuuaaucugacggg</td> <td>24</td> <td>0</td> <td>3</td> <td>0</td> <td>1</td> </tr> <tr> <td>LL-miR12</td> <td>c56352_g1</td> <td>gcucggguuaacggggaagug</td> <td>21</td> <td>9</td> <td>56</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR13</td> <td>c59256_g1</td> <td>uaugaaguuauauagguuguccgg</td> <td>24</td> <td>1</td> <td>6</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR14</td> <td>c79455.graph_c0</td> <td>uuuacugccaccauccgccugc</td> <td>22</td> <td>3</td> <td>12</td> <td>5</td> <td>0</td> </tr> <tr> <td>LL-miR15</td> <td>c94000_g1</td> <td>aagguaagagaaucaacaagaggu</td> <td>24</td> <td>0</td> <td>8</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR16</td> <td>c96112_g1</td> <td>uaggcaacaaauuagagucucu</td> <td>22</td> <td>8</td> <td>8</td> <td>33</td> <td>50</td> </tr> <tr> <td>LL-miR17</td> <td>c99691_g1</td> <td>uuuguauggucuguugaaauu</td> <td>21</td> <td>4</td> <td>0</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR18</td> <td>c109150_g1</td> <td>caggcggcgaggauggggaug</td> <td>21</td> <td>2</td> <td>5</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR19</td> <td>c109175_g1</td> <td>uugcuuagcuuguacucucgc</td> <td>21</td> <td>1</td> <td>4</td> <td>4</td> <td>0</td> </tr> <tr> <td>LL-miR20</td> <td>c110752_g2</td> <td>ugaaaauguagcacuagcacc</td> <td>21</td> <td>1</td> <td>7</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR21</td> <td>c111855_g1</td> <td>uugagaguagagagccaggug</td> <td>21</td> <td>0</td> <td>12</td> <td>1</td> <td>4</td> </tr> <tr> <td>LL-miR22</td> <td>c114504_g1</td> <td>aaaugaugaaucugagccuc</td> <td>20</td> <td>8</td> <td>0</td> <td>10</td> <td>6</td> </tr> <tr> <td>LL-miR23</td> <td>c114692_g4</td> <td>uagaggcgaugaugaugaaau</td> <td>21</td> <td>44</td> <td>795</td> <td>75</td> <td>119</td> </tr> <tr> <td>LL-miR24</td> <td>c117497_g1</td> <td>ugaagacuuggcaaccgacauc</td> <td>22</td> <td>4</td> <td>20</td> <td>2</td> <td>0</td> </tr> <tr> <td>LL-miR25</td> <td>c117720_g1</td> <td>ucugcccugauaugagcuccag</td> <td>22</td> <td>36</td> <td>0</td> <td>137</td> <td>0</td> </tr> <tr> <td>LL-miR26</td> <td>c117786_g3</td> <td>ucugaauagcaaacccaauuc</td> <td>21</td> <td>3</td> <td>5</td> <td>1</td> <td>0</td> </tr> <tr> <td>LL-miR27</td> <td>c166092_g1</td> <td>aaacgaucgauaaaccucugc</td> <td>21</td> <td>0</td> <td>4</td> <td>2</td> <td>0</td> </tr> <tr> <td>LL-miR28</td> <td>c48903.graph_c0</td> <td>aaugagaagacuagugacaagauu</td> <td>24</td> <td>4</td> <td>73</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR29</td> <td>CL711.Contig2_All</td> <td>uucccuucggcugcaaauagc</td> <td>21</td> <td>77</td> <td>25</td> <td>47</td> <td>33</td> </tr> <tr> <td>LL-miR30</td> <td>CL719.Contig1_All</td> <td>uagaggcgaugaugaugaaau</td> <td>21</td> <td>0</td> <td>2</td> <td>0</td> <td>1</td> </tr> <tr> <td>LL-miR31</td> <td>CL1297.Contig2_All</td> <td>aucuuuggccuggagauagagg</td> <td>22</td> <td>0</td> <td>3</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR32</td> <td>c71927_g1</td> <td>ugugccaugcugugugcgucc</td> <td>21</td> <td>2</td> <td>30</td> <td>2</td> <td>0</td> </tr> <tr> <td>LL-miR33</td> <td>CL4047.Contig1_All</td> <td>ugccgggcuaagauacaaggau</td> <td>22</td> <td>1</td> <td>0</td> <td>2</td> <td>1</td> </tr> <tr> <td>LL-miR34</td> <td>HM045458.1</td> <td>ucuauaugacucucggcaacgg</td> <td>22</td> <td>0</td> <td>1</td> <td>25</td> <td>2</td> </tr> <tr> <td>LL-miR35</td> <td>JZ391002</td> <td>uccaaagucagugaggggagc</td> <td>21</td> <td>0</td> <td>9</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR36</td> <td>Unigene13110_All</td> <td>uucgagugacauauggaaacu</td> <td>21</td> <td>1</td> <td>3</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR37</td> <td>Unigene18554_All</td> <td>ucaaucuuuggccuggagauagag</td> <td>24</td> <td>2</td> <td>10</td> <td>4</td> <td>0</td> </tr> <tr> <td>LL-miR38</td> <td>c68386.graph_c0</td> <td>ugggucuccucucauuccaug</td> <td>21</td> <td>9</td> <td>13</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR39</td> <td>Unigene25443_All</td> <td>uucgagugacauauggaaacu</td> <td>21</td> <td>1</td> <td>3</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR40</td> <td>c51021.graph_c0</td> <td>ucaaagacgaaucugagcaua</td> <td>21</td> <td>2</td> <td>6</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR41</td> <td>c56504.graph_c0</td> <td>ucguaucugugguuugcuccu</td> <td>21</td> <td>0</td> <td>1</td> <td>2</td> <td>0</td> </tr> <tr> <td>LL-miR42</td> <td>c59249.graph_c0</td> <td>ugcaguuugguuuguggugug</td> <td>21</td> <td>1</td> <td>3</td> <td>0</td> <td>1</td> </tr> <tr> <td>LL-miR43</td> <td>GW589960</td> <td>cugucgagcuuccauacuggc</td> <td>21</td> <td>0</td> <td>3</td> <td>0</td> <td>0</td> </tr> <tr> <td>LL-miR44</td> <td>JZ391211</td> <td>uggaucuugaaccaaguguuc</td> <td>21</td> <td>0</td> <td>2</td> <td>10</td> <td>0</td> </tr> </tbody> </table>  <p /> </div> <div class="predictionofmirnatargets" title="sec"> <div class="title" tagx="title" title="title">
Prediction of miRNA targets</div> <p>Plant miRNAs play important roles in diverse biological processes by cleaving target mRNAs or suppressing the translation of target genes. In order to understand the biological functions of <i>L. lancifolium</i> miRNAs, TargetFinder and psRNA Target software was used to predict putative target genes of novel and conserved miRNAs. The results from analysis showed that 366 target genes for 17 conserved miRNA families and 415 target genes for 40 novel miRNAs were predicted (Supplementary Tables <a href="#MOESM1">S4</a> and <a href="#MOESM1">S5</a>). The majority of the target genes for conserved miRNAs were transcriptional factors, and many target genes were conserved between <i>Lilium</i> and other plant, such as <i>squamosa promoter-binding proteins</i> ( <i>SPL</i>), <i>MYB</i>, <i>proliferating cell factors</i> ( <i>PCF</i>), <i>auxin response factor</i> ( <i>ARF</i>), <i>DCL1</i>, <i>NAC domain transcription factor</i>, <i>cup-shaped cotyledon 2</i>( <i>CUC2</i>), <i>PHB</i>, <i>AGO1</i>, <i>APETALA</i>, <i>ATP sulfurylases</i>, <i>growth-regulating factors</i> ( <i>GRF</i>), <i>UBC24</i> and <i>blue copper protein</i>, which were involved in various aspects of plant growth and development. However, some predicted target genes of several conserved miRNAs in <i>L. lancifolium</i> were different from those in other plants, including <i>fasciclin-like arabinogalactan protein</i> ( <i>FLA</i>) and <i>homeobox-leucine zipper protein HOX</i> (MIR166), <i>ethylene-responsive transcription factor RAP2-7</i> (MIR172), <i>LRR receptor-like serine/threonine-protein kinase</i> (MIR390), <i>stellacyanin</i> and <i>cucumber peeling cupredoxin</i> (MIR398). Interestingly many target genes for miR845 were predicted in <i>L. lancifolium</i> like <i>patatin protein</i>, <i>phosphatidate cytidylyltransferase</i>, <i>dnaJ protein</i>, and <i>LRR receptor-like serine/threonine-protein kinase</i> (Supplementary Table <a href="#MOESM1">S4</a>). The target genes of novel miRNAs were mainly predicted to be protein coding genes, such as <i>polyphenol oxidase</i>, <i>polycomb group protein FIE2</i>, <i>serine/threonine-protein kinase</i>, <i>pentatricopeptide repeat-containing protein</i>, <i>DRM-type DNA-methyltransferase</i>, <i>ATP-dependent DNA helicase</i>, <i>F-box protein</i> and so on (Supplementary Table <a href="#MOESM1">S5</a>). </p> </div> <div class="validationofmirnasusingnorthenhybridationandstem-looprt-pcr" title="sec"> <div class="title" tagx="title" title="title">
Validation of miRNAs using northen hybridation and stem-loop RT-PCR</div> <p>To confirm the credibility of miRNAs from the high-throughput sequencing and bioinformatics analysis, 4 novel miRNAs (LL-miR7, LL-miR9, LL-miR14 and LL-miR29) were randomly selected and their expressions in flower, leaf, bulblet and bulb of <i>L. lancifolium</i> were detected using northern blot analysis. All of them were detected in at least one tissue (Fig. <a href="#Fig2">2</a>). Two lily cultivars, ‘Brunello’ and ‘White heaven’, were also used for northen hybridization. All 4 novel miRNAs were detected in ‘Brunello’ or ‘White heaven’. In addition, 19 miRNAs (10 conserved and 9 novel miRNAs) were randomly selected for stem-loop RT-PCR analysis. All of these miRNAs were detected from flower, leaf, bulblet and bulb of <i>L. lancifolium</i>. Among them, 7 miRNAs (LL-miR2118, LL-miR5, LL-miR8, LL-miR21, LL-miR32, LL-miR35 and LL-miR43) were found primarily in bulb. Three miRNAs (LL-miR167, LL-miR16 and LL-miR41) were expressed predominantly in flower (Fig. <a href="#Fig3">3</a>). Three miRNAs (LL-miR159, LL-miR156 and LL-miR164) were highly expressed in bulblet or leaf. For the specificity and sensitivity of northern hybridization and stem-loop qRT-PCR <sup> <a href="#CR34">34</a>, <a href="#CR35">35</a> </sup>, the identification of conserved and novel miRNAs in <i>L. lancifolium</i> was effective and credible. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Detection of four novel miRNAs using Northern blotting. Lane 1–4, <i>L. lancifolium</i>; Lane 5,6, Brunello; Lane 7,8, White heaven; Lane1,5,7, flower; Lane 4,6,8, leaf; Lane2, bulblet; Lane3 bulb. U6 RNA is used as a loading control. The full-length blots are presented in Supplementary Fig. S <a href="#MOESM2">3</a>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Validation of miRNAs and targets using qRT-PCR. The X axis represents different tissues. The Y axis represents the relative expression level of miRNAs or targets.The amount of expression of miRNAs and targets was normalized to the level of 5.8S rRNA and 18S rRNA, respectively. Different letters indicate significant differences at P &amp;lt; 0.05 according to Duncan’s multiple range tests. </p>   </div> <p /> </div> <div class="verificationofmirnatargetsbydegradomesequencingandrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
Verification of miRNA targets by degradome sequencing and RT-PCR</div> <p>In order to validate the targets of conserved and novel miRNAs in <i>L. lancifolium</i>, the high-throughput degradome sequencing was used to detect the cleavage site of predicted targets. A total of 683,113,369 raw data were obtained. After removing adaptors, low quality reads and repeat sequences, 14,534,088 clean tags were yielded. A total of 12,194,038 clean tags were mapped perfectly to <i>Lilium</i> RNAs. After bioinformatics analysis, 27 and 26 cleavage sites were identified to 6 conserved miRNAs families and 14 novel miRNAs, respectively (Supplementary Tables <a href="#MOESM1">S4</a> and <a href="#MOESM1">S5</a>). According to Addo-Quaye’s analysis method <sup> <a href="#CR36">36</a> </sup>, all targets identified by degradome sequencing could be classified into five categories: twenty-two cleavage sites with category 0, ten cleavage sites with category 1, nine cleavage sites with category 2, five cleavage sites with category 3, seven cleavage sites with category 4. Most cleavage sites (11/12) with category 3 and 4 were from novel miRNA targets. These targets of conserved miRNAs included <i>SPLs</i>, <i>MYB</i>, <i>GRFs</i>, <i>DCL1</i>, <i>pentatricopeptide repeat-containing protein</i>, <i>Ras-group-related LRR protein</i>, <i>methyltransferase PMT7</i>, <i>chorismatemutase 3</i> and <i>trifunctional UDP-glucose 4,6-dehydratase</i> (Supplementary Table <a href="#MOESM1">S4</a>). Novel miRNAs could cleave <i>P450-like, polyphenol oxidase</i>, <i>auxin response factor 7-like</i>, <i>Kunitz-type trypsin inhibitor</i>, <i>F-box protein</i>, <i>ferredoxin</i>, <i>FLS2</i>, <i>N-alpha-acetyltransferase 16</i>, and so on (Supplementary Table <a href="#MOESM1">S5</a>). </p> <p>To validate the regulation of target expression by miRNA, five targets for two conserved and three novel miRNAs were investigated using qRT-PCR. Among them, LL-miR9 could be detected in flower and bulb using northern blot. <i>Polyphenol oxidase</i>, a predicted target of LL-miR9, has higher expression in bublet and leaf. <i>Phosphatidylinositol-3-phosphatase myotubularin-1</i> to potentially be targeted by LL-miR29 has decreased expression in leaf (Fig. <a href="#Fig3">3</a>). The result agreed with the expression of LL-miR29 from northern blot. In addition, LL-miR156 and LL-miR159 also had the opposite expression pattern with their targets ( <i>SPL16</i> and <i>GAMYB1-like</i>). </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p> <i>Lilium</i> species belong to genus Liliaceous, known as perennial herbaceous flowering plants growing from bulbs and among the most important cut flowers worldwide. Although several research reports shown that miRNAs play crucial roles in plant growth, development, and response to stress in plants, however, there is no report on <i>Lilium</i> miRNAs identification. The high-throughput sequencing is an effective method to screen miRNAs from various plants. Up to date, many miRNAs from ornamental plant, including <i>Lycoris aurea</i>, <i>Nelumbo nucifera</i>, <i>Herbaceous peony</i>, <i>Cymbidium ensifolium</i>, <i>Ginkgo biloba</i>, <i>Prunus mume</i> and rose <sup> <a href="#CR21">21</a>, <a href="#CR37">37</a>– <a href="#CR42">42</a> </sup>, have been identified using this method. To discover miRNAs in <i>Lilium</i> and understand their functions, we constructed and sequenced four small RNA libraries from flower, leaf, bulblet and bulb of <i>L. lancifolium</i>. The result of this research adds our knowledge to understand about the role of miRNAs from Liliaceae, in which a number of genera are popular cultivated plants with ornamental value. </p> <p>The study on miRNAs in <i>Lilium</i> was limited to their large genome (~36 Gb) compared with other plants in which genome has been sequenced, such as <i>Arabidopsis</i> <sup> <a href="#CR43">43</a> </sup>, rice <sup> <a href="#CR44">44</a> </sup>, wheat <sup> <a href="#CR45">45</a> </sup>, Populus <sup> <a href="#CR46">46</a> </sup>. In this study, RNA sequences from <i>Lilium</i> in NCBI database, transcriptome data from <i>Lilium pumilum</i> and an Asiatic hybrid cultivar ‘Easy Dance’ besides to <i>L. lancifolium</i> were also used to predict miRNAs and their targets. As a result a total of 17 conserved miRNA families and 44 novel miRNAs were identified compared with only 9 conserved miRNA families and 17 novel miRNAs using <i>L. lancifolium</i> RNA. Therefore, we believe that deep sequencing of different tissues at various developmental stages could be necessary to fully disclose the miRNA function in <i>Lilium</i>. And in comparison to other close physiologically plants, the number of miRNAs identified from <i>L. lancifolium</i> are less than rice (713), wheat (119), maize (321), <i>Brachypodium distachyon</i> (525) which have been well genomic sequenced, but more than those plants without genome sequencing data, like <i>Elaeis guineensis</i> (6), <i>Festuca arundinacea</i> (15), <i>Saccharum officinarum</i> (16) (miRbase database release 21). </p> <p>The lengths of plant sRNAs usually ranged from 21 nt to 24 nt <sup> <a href="#CR47">47</a> </sup>. Among them, 24 nt sRNA was the most abundant, followed by 21 nt class <sup> <a href="#CR13">13</a>, <a href="#CR48">48</a>, <a href="#CR49">49</a> </sup>. However, our research result shows that 21 nt sRNAs were more abundant in flower and bulb whereas 23 nt sRNA in leaf and bulblet were more abundant than 24 nt sRNA in <i>L. lancifolium</i>. It has been reported that the 21 nt sRNA is the most abundant sRNA species in <i>Populus balsamifer</i>, Chinese Wild <i>Vitis pseudoreticulata</i> and <i>Pinus cordata</i> <sup> <a href="#CR48">48</a>, <a href="#CR50">50</a>, <a href="#CR51">51</a> </sup>. Our results from flower and bulb supported the speculation given by Han and colleagues that the major sRNA species were 21 nt in perennial plants <sup> <a href="#CR51">51</a> </sup>. The 23 nt sRNAs could arising from loci dominated by 24 nt siRNAs <sup> <a href="#CR52">52</a> </sup> and <i>MIR</i> genes <sup> <a href="#CR53">53</a> </sup>, but their biological functions are less understood. Therefore, a large number of 23 nt sRNA may play a special role in leaf and bulblet of <i>L. lancifolium</i>. </p> <p>After analyzing miRNAs identified in this study, we found that the 5′ ends of most miRNAs were U (84%for conserved miRNAs and 70.8% for novel miRNAs) in <i>L. lancifolium</i>. The result was consistent with previous reports that the U at 5′end of miRNA favored the combination with AGO1 <sup> <a href="#CR54">54</a> </sup>. It has been reported that miRNAs had differential accumulation patterns in many plant species, and possessed their own precise regulation processes through the tissue dependent miRNA biogenesis in different plant species <sup> <a href="#CR11">11</a>, <a href="#CR55">55</a> </sup>. In this study, the expression analysis of miRNAs from qRT-PCR and northern blot showed that many conserved and novel miRNAs were tissue biased in <i>L. lancifolium</i>. This result suggested that the miRNAs might play very important roles in development of different tissue. </p> <p>In order to understand the functions of miRNAs in <i>L. lancifolium</i>, a large number of targets of conserved and novel miRNAs were predicted in this research. These conserved miRNAs LL-miR156, LL-miR159, LL-miR319, LL-miR160, LL-miR164, LL-miR166, LL-miR172 and LL-miR396 targeted SPLs, MYBs, PCFs, ARFs, NACs, HD-ZIP III, AP2s and GRFs, respectively. Our result is in agreement with previous research that the targets of conserved miRNAs in plant are mainly transcription factors <sup> <a href="#CR56">56</a> </sup>. These conserved miRNAs and their transcription factor targets in lily might have similar functions on plant growth and development as those in other plant species. However, the LL-miR398 has been predicted to target type I blue copper proteins rather than reported CSD, CoX5b-1 and CCS1, which were involved in responses to environmental stresses <sup> <a href="#CR57">57</a>– <a href="#CR59">59</a> </sup>. In this study, 27 targets of LL-miR156, LL-miR159, LL-miR162, LL-miR390, LL-miR396 and LL-miR2118 have been validated by degradome sequencing. The result from qRT-PCR analysis showed that LL-miR156 and LL-miR159 had the opposite expression pattern with their targets. </p> <p>The novel miRNAs were mainly predicted to be protein coding genes. Among 415 predicted targets, 26 were validated by degradome sequencing. Although some cleavage sites for predicted targets, which were matched very well with novel miRNAs, were not detected using degradome sequencing, it was possible that novel miRNAs regulated them at the level of translation. The novel LL-miR09 and LL-miR35 were predicted to target polyphenol oxidase, and the cleavage site of LL-miR09 has been identified by degradome sequencing. In <i>Populus</i> and <i>Salvia miltiorrhiza</i>, miR1444 and Smi-miR12112 have been reported to regulate a subset of polyphenol oxidases, which have important roles in plant development and response to biotic and abiotic stresses <sup> <a href="#CR60">60</a>– <a href="#CR62">62</a> </sup>. LRR receptor-like serine/threonine-protein kinase FLS2 and ERECTA were predicted as targets of LL-miR21 and miR35, respectively, and the cleavage site of FLS2 has been detected. The FLS2 has been reported to perceive the bacterial elicitor flagellin in <i>Arabidopsis</i> <sup> <a href="#CR63">63</a> </sup>. The ERECTA was involved in the thermo tolerance, stomatal development, plant architecture in <i>Arabidopsis</i> <sup> <a href="#CR64">64</a> </sup>. The serine/threonine-protein kinase ACR4 targeted by LL-miR25 plays important roles in cell division and differentiation in <i>Arabidopsis</i> <sup> <a href="#CR65">65</a> </sup>. The polycomb group protein FIE2, which prevents fertilization-independent seed development in <i>Arabidopsis</i> <sup> <a href="#CR66">66</a> </sup>, is a potential LL-miR08 target. The pentatricopeptide repeat-containing protein, which was potentially targeted by LL-miR18, LL-miR43 and LL-miR44, could affect chloroplast development in <i>Arabidopsis</i> <sup> <a href="#CR67">67</a> </sup>. DRM-type DNA-methyltransferase involved in RNA-directed DNA methylation in <i>Arabidopsis</i>, was predicted as target of miR20 <sup> <a href="#CR68">68</a> </sup>. ATP-dependent DNA helicase DDM1 participated in UV-B induced and oxidative DNA damage repair in <i>Arabidopsis</i> <sup> <a href="#CR69">69</a> </sup>, was potentially targeted by LL-miR36. F-box protein potentially targeted by LL-miR26, LL-miR38 and LL-miR44, has been reported to mediate bouquet formation to promote homologous pairing, synapsis, and recombination in rice meiosis <sup> <a href="#CR70">70</a> </sup>. It is very interesting that LL-miR07 and LL-miR14 owed highly similar mature miRNA and precursor sequences. Therefore, we speculated that LL-miR07 and LL-miR14 might be derived from the same ancestor. Both of them were predicted to target ARF7-like, which regulates lateral root formation, differential growth of hypocotyls in <i>Arabidopsis</i> and fruit set in tomato <sup> <a href="#CR71">71</a>, <a href="#CR72">72</a> </sup>. Since these novel miRNAs were predicted to participate in plant growth, development, biotic and abiotic stress responses, and signal transduction and so on, further studies are recommended to understand the functions of novel miRNAs in <i>Lilium species</i>. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialandrnapreparation" title="sec"> <div class="title" tagx="title" title="title">
Plant material and RNA preparation</div> <p> <i>L. lancifolium</i> was selected as the experiment material, and grown in the greenhouse of Beijing University of Agriculture, Beijing, China. Flowers, leaves, bulblets and bulbs of <i>L. lancifolium</i> were collected and immediately frozen in liquid nitrogen. The frozen samples were then stored at −80 °C for future analysis. The TRIzol® reagent (Invitrogen, USA) was used to extract total RNA from four samples according to the manufacturer’s protocol. Finally, the integrity of total RNA was confirmed using 1% agarose gel electrophoresis and Agilent 2100 Bio analyzer (Agilent Technologies, USA). </p> </div> <div class="smallrnalibraryconstructionandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Small RNA library construction and sequencing</div> <p>Small RNA libraries of flowers, leaves, bulblets and bulbs were constructed using previously described methods <sup> <a href="#CR73">73</a> </sup>. Briefly, small RNAs fragments of 10–30 nt were purified from a 15% denaturing polyacrylamide gel and then ligated with 5′ and 3′ adapters. After being reverse-transcribed by Superscript II reverse transcriptase (Invitrogen) and amplified by PCR, about 20 µg products from each sample were sequenced using Illumina HiSeq. 2500 sequencing platform (Illumina Inc.; San Diego, CA, USA) at the Biomarker Technologies (Beijing, China). </p> </div> <div class="predictionofconservedandnovelmirna" title="sec"> <div class="title" tagx="title" title="title">
Prediction of conserved and novel miRNA</div> <p>After removal of chip adaptor sequences, low quantity reads and contaminations, the clean 18–30 nt small RNAs were mapped to GenBank ( <a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) and Rfam (version 10.1) database ( <a href="http://rfam.sanger.ac.uk">http://rfam.sanger.ac.uk</a>) with a cut-off value of 0.01, and rRNA, tRNA, snRNA, snoRNA were removed to produce filtered small RNAs. All available <i>Lilium</i> RNA sequences were collected to predict miRNAs and their targets, including unigenes from transcriptome sequencing of <i>L. lancifolium</i> (SRA632698), <i>Lilium pumilum</i> (SRA633315), Asiatic hybrid lily cultivars ‘Easy Dance’ (SRA538278), and 3902 ESTs ( <i>Lilium formosanum</i>, <i>Lilium longiflorum</i>, <i>Lilium regale</i>, Lilium hybrid division VII, and <i>Lilium davidii</i> var. willmottiae) downloaded from NCBI. All filtered small RNA, which were aligned against miRbase database (release 21) ( <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>) with no more than two mismatches, were aligned against <i>Lilium</i> RNA sequences. Their flanking sequences were fold with mfold soft <sup> <a href="#CR74">74</a> </sup>. The filtered small RNA in perfect stem-loop structure was considered as conserved miRNAs. The miRDeep2 with modified parameter was used to identify novel miRNAs and check the secondary structures of putative pre-miRNAs <sup> <a href="#CR75">75</a> </sup>. The minimum free energy index (MFEI) was calculated using the equation: MFEI = AMFE/(G + C)%. The adjusted MFE (AMFE) represented the MFE of 100 nucleotides. It was calculated using (MFE/length of RNA sequence) × 100 <sup> <a href="#CR76">76</a> </sup>. </p> </div> <div class="targetpredictionofconservedandnovelmirna" title="sec"> <div class="title" tagx="title" title="title">
Target prediction of conserved and novel miRNA</div> <p> <i>Lilium</i> RNA sequences above for miRNA prediction were also used for target prediction. TargetFinder and psRNA Target were applied to predict the putative targets of conserved and novel miRNA <sup> <a href="#CR77">77</a>, <a href="#CR78">78</a> </sup>. </p> </div> <div class="quantitativereal-timepcranalysisofmirnasandtargets" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR analysis of miRNAs and targets</div> <p>The stem-loop RT-PCR was used to validate the miRNAs from deep sequencing and to analyze their expression patterns. Total RNA of flower, leaf, bulblet, and bulb were extracted using Trizol reagent (Invitrogen) according to the manufacturer’s instruction. Then, total RNA was reverse-transcribed to cDNA using stem-loop RT primer by the PrimeScript RT reagent Kit (TaKaRa, Dalian, China) according to the manufacturer’s protocol. All primers for stem-loop RT-RCR were designed according to the Chen’s report <sup> <a href="#CR35">35</a> </sup> and listed in Supplementary Table <a href="#MOESM1">S6</a>. The qRT-PCR reactions were performed using the SYBR Premix Ex Taq II solution (TaKaRa, Dalian, China) as the following condition: 95 °C for 5 minutes, then 40 cycles of denaturation at 95 °C for 15 s, 60 °C for 30 s, and 72 °C for 30 s. For the qRT-PCR analysis of the miRNA targets, total RNA was used for synthesizing reverse transcripts using PrimeScript RT reagent Kit (Takara, Dalian, China) according to the manufacturer’s instructions. Specific primer pairs for miRNA targets were designed to amplify cDNA (Supplementary Table <a href="#MOESM1">S6</a>). qPCR was performed using SYBR Premix Ex Taq II (TaKaRa, Dalian, China) under the following conditions: 40 cycles at 95 °C for 15 s, 60 °C for 30 s, and 72 °C for 30 s. The qRT-PCR reactions were performed on the BIO-RAD iQ5 (Applied Biosystems, Foster City, CA). Each sample was processed in triplicate, and the relative expression were calculated using 2 <sup>−ΔΔCT</sup> <sup> <a href="#CR79">79</a> </sup>. The 5.8S rRNA <sup> <a href="#CR80">80</a> </sup> and 18S rRNA were used as references to normalize the expression level of miRNAs and their targets. The data were statistically analyzed using SAS Version 9.0 software (SAS Institute, Cary, NC, USA) using Duncan’s multiple range test at the P &amp;lt; 0.05 level of significance. </p> </div> <div class="degradomelibraryconstructionandtargetidentification" title="sec"> <div class="title" tagx="title" title="title">
Degradome library construction and target identification</div> <p>To investigate the potential targets of conserved and novel miRNAs, a degradome library was constructed using mixture of mRNA from flowers, leaves, bulblets and bulbs of <i>L. lancifolium</i> according to the parallel analysis of the RNA ends protocol <sup> <a href="#CR81">81</a> </sup>, and sequenced using Illumina HiSeqTM 2500 sequencing platform (Illumina Inc.; San Diego, CA, USA) at the Beijing Genomics Institute (BGI) (Shenzhen, China). A Public software package, CleaveLand3.0 was used for analyzing sequencing data. All the putative target genes were used as queries to align against <i>Lilium</i> RNA sequences. The true miRNA cleavage sites from background noise were identified using a target plot <sup> <a href="#CR82">82</a> </sup>. </p> </div> <div class="northernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Northern blot analysis</div> <p>Total RNA was isolated from plant tissues using TRIzol reagent (Invitrogen, USA) according to the manufacturer’s instructions. For Northern blots, 40 µg of total RNA was separated by electrophoresis on 17% polyacrylamide gel and electrically transferred to nylon N+ membrane. Blots were hybridized with [ϒ- <sup>32</sup>P] ATP-labeled oligonucleotide probe. Hybridization signal intensity was measured using a PhosphorImager (GE Healthcare). Sequences of the oligonucleotide probes are listed in Supplementary Table <a href="#MOESM1">S6</a>. </p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p>The raw data (Accession Number: SRA633909) in the study can be obtained from SRA database.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Table S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM2_ESM.pdf">LINK</a>  <p>Supplementary Figures S1 and S2</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM3_ESM.xml">LINK</a>  <p>Supplementary Table S2 </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM4_ESM.xml">LINK</a>  <p>Supplementary Table S3 </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM5_ESM.xml">LINK</a>  <p>Supplementary Table S4 </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM6_ESM.xml">LINK</a>  <p>Supplementary Table S5 </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM7_ESM.xml">LINK</a>  <p>Supplementary Table S6 </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2018_21193_MOESM8_ESM.pdf">LINK</a>  <p>Supplementary Figure S3</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at 10.1038/s41598-018-21193-4. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by the National Natural Science Foundation of China (Grant NO. 31640007), the High-level Scientific Research Cultivation Project of BUA (GJB2015004), the Project of Construction of Innovative Teams and Teacher Career Development for Universities and Colleges Under Beijing Municipality (IDHT20150503), the Beijing Municipal Education Commission (CEFF-PXM2017_014207_000043). </p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>X.H. designed this study, performed the lab experiments, analyzed the data and wrote the manuscript. A.G.S. wrote the manuscript. S.X. designed this study, analyzed the data and wrote the manuscript. W.W. designed this study and wrote the manuscript. All authors have reviewed the manuscript. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voinnet</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Origin, Biogenesis, and Activity of Plant MicroRNAs</span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">136</span> <span class="fpage" tagx="fpage" title="fpage">669</span> <span class="lpage" tagx="lpage" title="lpage">687</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2009.01.046">10.1016/j.cell.2009.01.046</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19239888">19239888</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Filipowicz</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharyya</span> <span class="given-names" tagx="given-names" title="given-names">SN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sonenberg</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mechanisms of post-transcriptional regulation by microRNAs: are the answers in sight?</span> <span class="source" tagx="source" title="source">Nat. Rev. Genet.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">2008</span> <span class="fpage" tagx="fpage" title="fpage">102</span> <span class="lpage" tagx="lpage" title="lpage">114</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2290">10.1038/nrg2290</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Small RNAs and Their Roles in Plant Development</span> <span class="source" tagx="source" title="source">Annu. Rev. Cell Dev. Biol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">21</span> <span class="lpage" tagx="lpage" title="lpage">44</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.cellbio.042308.113417">10.1146/annurev.cellbio.042308.113417</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19575669">19575669</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biogenesis, Turnover, and Mode of Action of Plant MicroRNAs</span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">2383</span> <span class="lpage" tagx="lpage" title="lpage">2399</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.113.113159">10.1105/tpc.113.113159</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23881412">23881412</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunkar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">YF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jagadeeswaran</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functions of microRNAs in plant stress responses</span> <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">196</span> <span class="lpage" tagx="lpage" title="lpage">203</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2012.01.010">10.1016/j.tplants.2012.01.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22365280">22365280</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">H-S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNA Directs mRNA Cleavage of the Transcription Factor NAC1 to Downregulate Auxin Signals for Arabidopsis Lateral Root Development </span> <span class="source" tagx="source" title="source">Plant Cell Online</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">1376</span> <span class="lpage" tagx="lpage" title="lpage">1386</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.105.030841">10.1105/tpc.105.030841</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feinbaum</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ambros</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The C. elegans heterochronic gene lin-4encodes small RNAs with antisense comlementarity to lin-14 </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">843</span> <span class="lpage" tagx="lpage" title="lpage">854</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0092-8674(93)90529-Y">10.1016/0092-8674(93)90529-Y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8252621">8252621</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weinstein</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in plants</span> <span class="source" tagx="source" title="source">Plant Signal. Behav.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">1616</span> <span class="lpage" tagx="lpage" title="lpage">1626</span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">BH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">XP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">QL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cobb</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of new plant microRNAs using EST analysis</span> <span class="source" tagx="source" title="source">Cell Res.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">336</span> <span class="lpage" tagx="lpage" title="lpage">360</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/sj.cr.7290302">10.1038/sj.cr.7290302</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15916721">15916721</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Novel and Mechanical Stress-Responsive MicroRNAs in Populus trichocarpa That Are Absent from Arabidopsis </span> <span class="source" tagx="source" title="source">Plant Cell Online</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">2186</span> <span class="lpage" tagx="lpage" title="lpage">2203</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.105.033456">10.1105/tpc.105.033456</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">XF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">YY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of conserved and novel microRNAs and their targets, including a TuMV-induced TIR-NBS-LRR class R gene-derived novel miRNA in Brassica </span> <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">582</span> <span class="fpage" tagx="fpage" title="fpage">2445</span> <span class="lpage" tagx="lpage" title="lpage">2452</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2008.06.011">10.1016/j.febslet.2008.06.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18558089">18558089</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arazi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning and characterization of micro‐RNAs from moss</span> <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">837</span> <span class="lpage" tagx="lpage" title="lpage">848</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02499.x">10.1111/j.1365-313X.2005.02499.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16146523">16146523</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Elucidation of the Small RNA Component of the Transcriptome</span> <span class="source" tagx="source" title="source">Science (80-.).</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">309</span> <span class="fpage" tagx="fpage" title="fpage">1567</span> <span class="lpage" tagx="lpage" title="lpage">1569</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1114112">10.1126/science.1114112</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruby</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Large-Scale Sequencing Reveals 21U-RNAs and Additional MicroRNAs and Endogenous siRNAs in C. elegans </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">127</span> <span class="fpage" tagx="fpage" title="fpage">1193</span> <span class="lpage" tagx="lpage" title="lpage">1207</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2006.10.040">10.1016/j.cell.2006.10.040</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17174894">17174894</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNA Discovery and Profiling in Human Embryonic Stem Cells by Deep Sequencing of Small RNA Libraries </span> <span class="source" tagx="source" title="source">Stem Cells</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">2496</span> <span class="lpage" tagx="lpage" title="lpage">2505</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1634/stemcells.2008-0356">10.1634/stemcells.2008-0356</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18583537">18583537</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burnside</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Deep Sequencing of Chicken microRNAs</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">185</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-9-185">10.1186/1471-2164-9-185</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18430245">18430245</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Novel microRNAs uncovered by deep sequencing of small RNA transcriptomes in bread wheat (Triticum aestivum L.) and Brachypodium distachyon (L.) Beauv </span> <span class="source" tagx="source" title="source">Funct. Integr. Genomics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">499</span> <span class="lpage" tagx="lpage" title="lpage">511</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-009-0128-9">10.1007/s10142-009-0128-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19499258">19499258</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huan</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chong</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Deep sequencing of Brachypodium small RNAs at the global genome level identifies microRNAs involved in cold stress response </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">449</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-10-449">10.1186/1471-2164-10-449</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19772667">19772667</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">C-Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Deep sequencing identifies novel and conserved microRNAs in peanuts (Arachis hypogaea L.) </span> <span class="source" tagx="source" title="source">BMC Plant Biol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-10-3">10.1186/1471-2229-10-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20047695">20047695</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chao</span> <span class="given-names" tagx="given-names" title="given-names">YT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of the microRNA transcriptome of a moth orchid Phalaenopsis aphrodite </span> <span class="source" tagx="source" title="source">Plant Mol. Biol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">529</span> <span class="lpage" tagx="lpage" title="lpage">548</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-013-0150-0">10.1007/s11103-013-0150-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24173913">24173913</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pei, H. <i>et al</i>. Integrative analysis of miRNA and mRNA profiles in response to ethylene in rose petals during flower opening. <i>PLoS One</i> <b>8</b> (2013). </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puzey</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kramer</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of conserved Aquilegia coerulea microRNAs and their targets</span> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">448</span> <span class="fpage" tagx="fpage" title="fpage">46</span> <span class="lpage" tagx="lpage" title="lpage">56</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2009.08.005">10.1016/j.gene.2009.08.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19699282">19699282</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Luis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Two MicroRNAs Linked to Nodule Infection and Nitrogen-Fixing Ability in the Legume Lotus japonicus </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">160</span> <span class="fpage" tagx="fpage" title="fpage">2137</span> <span class="lpage" tagx="lpage" title="lpage">2154</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.112.204883">10.1104/pp.112.204883</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23071252">23071252</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">KEP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Firoozabady</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transformation of floriculture crops</span> <span class="source" tagx="source" title="source">Sci. Hortic. (Amsterdam).</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">83</span> <span class="lpage" tagx="lpage" title="lpage">99</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0304-4238(93)90026-M">10.1016/0304-4238(93)90026-M</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">LingFei</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">FengWang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant regeneration from <i>in vitro</i> cultured leaves of Lanzhou lily (Lilium davidii var. unicolor) </span> <span class="source" tagx="source" title="source">Sci. Hortic. (Amsterdam).</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">119</span> <span class="fpage" tagx="fpage" title="fpage">458</span> <span class="lpage" tagx="lpage" title="lpage">461</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.scienta.2008.08.026">10.1016/j.scienta.2008.08.026</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tzi</span> <span class="given-names" tagx="given-names" title="given-names">BN</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of a novel trypsin inhibitor from fresh lily bulbs</span> <span class="source" tagx="source" title="source">Planta Med.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">546</span> <span class="lpage" tagx="lpage" title="lpage">550</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1055/s-2008-1074502">10.1055/s-2008-1074502</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18401842">18401842</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwon</span> <span class="given-names" tagx="given-names" title="given-names">OK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Anti-inflammatory effects of methanol extracts of the root of Lilium lancifolium on LPS-stimulated Raw264.7 cells </span> <span class="source" tagx="source" title="source">J. Ethnopharmacol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">28</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jep.2010.04.002">10.1016/j.jep.2010.04.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20412846">20412846</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yun</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang</span> <span class="given-names" tagx="given-names" title="given-names">YP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Lilium lancifolium Thunb. extract attenuates pulmonary inflammation and air space enlargement in a cigarette smoke-exposed mouse model </span> <span class="source" tagx="source" title="source">J. Ethnopharmacol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">149</span> <span class="fpage" tagx="fpage" title="fpage">148</span> <span class="lpage" tagx="lpage" title="lpage">156</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jep.2013.06.014">10.1016/j.jep.2013.06.014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23796878">23796878</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Assessment of the genetic diversity and genetic relationships of Lilium in China using ISSR markers </span> <span class="source" tagx="source" title="source">Biochem. Syst. Ecol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">184</span> <span class="lpage" tagx="lpage" title="lpage">189</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bse.2014.03.024">10.1016/j.bse.2014.03.024</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamagishi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">How genes paint lily flowers: Regulation of colouration and pigmentation patterning</span> <span class="source" tagx="source" title="source">Sci. Hortic. (Amsterdam).</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">163</span> <span class="fpage" tagx="fpage" title="fpage">27</span> <span class="lpage" tagx="lpage" title="lpage">36</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.scienta.2013.07.024">10.1016/j.scienta.2013.07.024</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling of the cold response and signaling pathways in Lilium lancifolium</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-203">10.1186/1471-2164-15-203</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24636716">24636716</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">De novo assembly and characterization of stress transcriptome and regulatory networks under temperature, salt and hormone stresses in Lilium lancifolium </span> <span class="source" tagx="source" title="source">Mol. Biol. Rep.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">8231</span> <span class="lpage" tagx="lpage" title="lpage">8245</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11033-014-3725-1">10.1007/s11033-014-3725-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25200436">25200436</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-Throughput Sequencing and Degradome Analysis Identify miRNAs and Their Targets Involved in Fruit Senescence of Fragaria ananassa </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e70959</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0070959">10.1371/journal.pone.0070959</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23990918">23990918</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sempere</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Expression profiling of mammalian microRNAs uncovers a subset of brain-expressed microRNAs with possible roles in murine and human neuronal differentiation </span> <span class="source" tagx="source" title="source">Genome Biol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">R13</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2004-5-3-r13">10.1186/gb-2004-5-3-r13</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15003116">15003116</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Chen, C. <i>et al</i>. Real-time quantification of microRNAs by stem-loop RT-PCR. <i>Nucleic Acids Res</i>. <b>33</b> (2005). </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Addo-Quaye</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eshoo</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Endogenous siRNA and miRNA Targets Identified by Sequencing of the Arabidopsis Degradome</span> <span class="source" tagx="source" title="source">Curr. Biol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">758</span> <span class="lpage" tagx="lpage" title="lpage">762</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2008.04.042">10.1016/j.cub.2008.04.042</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18472421">18472421</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and differential regulation of microRNAs in response to methyl jasmonate treatment in Lycoris aurea by deep sequencing </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">789</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-016-2645-y">10.1186/s12864-016-2645-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27724902">27724902</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bioinformatic Identification and Expression Analysis of <i>Nelumbo nucifera</i> MicroRNA and Their Targets </span> <span class="source" tagx="source" title="source">Appl. Plant Sci.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">1500046</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/apps.1500046">10.3732/apps.1500046</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gong</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of miRNAs responsive to Botrytis cinerea in herbaceous peony (Paeonia lactiflora Pall.) by high-throughput sequencing </span> <span class="source" tagx="source" title="source">Genes (Basel)</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">918</span> <span class="lpage" tagx="lpage" title="lpage">934</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/genes6030918">10.3390/genes6030918</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26393656">26393656</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization and comparative profiling of the small RNA transcriptomes in two phases of flowering in Cymbidium ensifolium </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">622</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1764-1">10.1186/s12864-015-1764-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26289943">26289943</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of MicroRNAs in Ginkgo biloba var. epiphylla mak</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">18</span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and profiling of novel and conserved microRNAs during the flower opening process in Prunus mume via deep sequencing </span> <span class="source" tagx="source" title="source">Mol. Genet. Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">289</span> <span class="fpage" tagx="fpage" title="fpage">169</span> <span class="lpage" tagx="lpage" title="lpage">183</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-013-0800-6">10.1007/s00438-013-0800-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24343764">24343764</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X-J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">N-H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaasterland</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Prediction and identification of Arabidopsis thaliana microRNAs and their mRNA targets</span> <span class="source" tagx="source" title="source">Genome Biol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">R65</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2004-5-9-r65">10.1186/gb-2004-5-9-r65</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15345049">15345049</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sunkar, R., Zhou, X., Zheng, Y., Zhang, W. &amp;amp; Zhu, J.K. Identification of novel and candidate miRNAs in rice by high throughput sequencing. <i>BMC Plant biol</i>. <b>8</b>, 25 (2008). </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of microRNAs from barley (hordeum vulgare L.) by high-throughput sequencing </span> <span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2973</span> <span class="lpage" tagx="lpage" title="lpage">2984</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/ijms13032973">10.3390/ijms13032973</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22489137">22489137</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duan</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-Wide Analysis of MicroRNA Responses to the Phytohormone Abscisic Acid in Populus euphratica </span> <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26858731">26858731</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajagopalan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaucheret</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trejo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A diverse and evolutionarily fluid set of microRNAs in Arabidopsis thaliana</span> <span class="source" tagx="source" title="source">Genes Dev.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">3407</span> <span class="lpage" tagx="lpage" title="lpage">3425</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1476406">10.1101/gad.1476406</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17182867">17182867</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Morin, R. D. <i>et al</i>. Comparative analysis of the small RNA transcriptomes of Pinus contorta and Oryza sativa. 571–584, 10.1101/gr.6897308.1 (2008). </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of maize microRNAs involved in the very early stage of seed germination </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">154</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-154">10.1186/1471-2164-12-154</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21414237">21414237</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barakat</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DiLoreto</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dePamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlson</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Conservation and divergence of microRNAs in Populus</span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">481</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-8-481">10.1186/1471-2164-8-481</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18166134">18166134</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and Characterization of Erysiphe necator-Responsive MicroRNAs in Chinese Wild Vitis pseudoreticulata by High-Throughput Sequencing </span> <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26858731">26858731</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">NR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coruh</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of single non-Templated nucleotides in plant endogenous siRNAs and miRNAs </span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">7395</span> <span class="lpage" tagx="lpage" title="lpage">7405</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw457">10.1093/nar/gkw457</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27207877">27207877</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chellappan</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">siRNAs from miRNA sites mediate DNA methylation of target genes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">6883</span> <span class="lpage" tagx="lpage" title="lpage">6894</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkq590">10.1093/nar/gkq590</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20621980">20621980</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Sorting of Small RNAs into Arabidopsis Argonaute Complexes Is Directed by the 5′ Terminal Nucleotide </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">133</span> <span class="fpage" tagx="fpage" title="fpage">116</span> <span class="lpage" tagx="lpage" title="lpage">127</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2008.02.034">10.1016/j.cell.2008.02.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18342361">18342361</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeong</span> <span class="given-names" tagx="given-names" title="given-names">D-H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Massive Analysis of Rice Small RNAs: Mechanistic Implications of Regulated MicroRNAs and Variants for Differential Target RNA Cleavage </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">4185</span> <span class="lpage" tagx="lpage" title="lpage">4207</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.111.089045">10.1105/tpc.111.089045</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22158467">22158467</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">microRNA biogenesis and function in plants</span> <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">579</span> <span class="fpage" tagx="fpage" title="fpage">5923</span> <span class="lpage" tagx="lpage" title="lpage">5931</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2005.07.071">10.1016/j.febslet.2005.07.071</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16144699">16144699</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Computational identification of plant MicroRNAs and their targets, including a stress-induced miRNA </span> <span class="source" tagx="source" title="source">Mol. Cell</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">787</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molcel.2004.05.027">10.1016/j.molcel.2004.05.027</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15200956">15200956</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonnet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wuyts</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rouze</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van de Peer</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Detection of 91 potential conserved plant microRNAs in Arabidopsis thaliana and Oryza sativa identifies important target genes </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">11511</span> <span class="lpage" tagx="lpage" title="lpage">11516</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0404025101">10.1073/pnas.0404025101</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15272084">15272084</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beauclair</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouché</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNA-directed cleavage and translational repression of the copper chaperone for superoxide dismutase mRNA in Arabidopsis </span> <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">454</span> <span class="lpage" tagx="lpage" title="lpage">462</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2010.04162.x">10.1111/j.1365-313X.2010.04162.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20128885">20128885</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">VL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Conservation and diversity of microRNA-associated copper-regulatory networks in Populus trichocarpa </span> <span class="source" tagx="source" title="source">J. Integr. Plant Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">879</span> <span class="lpage" tagx="lpage" title="lpage">891</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1744-7909.2011.01080.x">10.1111/j.1744-7909.2011.01080.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22013976">22013976</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Origin and evolution of MIR1444 genes in Salicaceae</span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">39740</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep39740">10.1038/srep39740</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28071760">28071760</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of the polyphenol oxidase gene family reveals a novel microRNA involved in posttranscriptional regulation of PPOs in Salvia miltiorrhiza </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">44622</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep44622">10.1038/srep44622</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28304398">28304398</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Gómez</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boller</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FLS2: An LRR Receptor-like Kinase Involved in the Perception of the Bacterial Elicitor Flagellin in Arabidopsis </span> <span class="source" tagx="source" title="source">Mol. Cell</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1003</span> <span class="lpage" tagx="lpage" title="lpage">1011</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1097-2765(00)80265-8">10.1016/S1097-2765(00)80265-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10911994">10911994</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">BKI1 Regulates Plant Architecture through Coordinated Inhibition of the Brassinosteroid and ERECTA Signaling Pathways in Arabidopsis </span> <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">297</span> <span class="lpage" tagx="lpage" title="lpage">308</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molp.2016.11.014">10.1016/j.molp.2016.11.014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27988365">27988365</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization and enzymatic properties of protein kinase ACR4 from Arabidopsis thaliana </span> <span class="source" tagx="source" title="source">Biochem. Biophys. Res. Commun.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">489</span> <span class="fpage" tagx="fpage" title="fpage">270</span> <span class="lpage" tagx="lpage" title="lpage">274</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbrc.2017.05.163">10.1016/j.bbrc.2017.05.163</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28571742">28571742</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohad</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mutations in FIE, a WD polycomb group gene, allow endosperm development without fertilization</span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">407</span> <span class="lpage" tagx="lpage" title="lpage">416</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.11.3.407">10.1105/tpc.11.3.407</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10072400">10072400</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PDM3, a pentatricopeptide repeat-containing protein, affects chloroplast development</span> <span class="source" tagx="source" title="source">J. Exp. Bot.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">267</span> <span class="fpage" tagx="fpage" title="fpage">27</span> <span class="lpage" tagx="lpage" title="lpage">37</span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naumann</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic evidence that DNA methyltransferase DRM2 has a direct catalytic role in RNA-directed DNA methylation in Arabidopsis thaliana </span> <span class="source" tagx="source" title="source">Genetics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">187</span> <span class="fpage" tagx="fpage" title="fpage">977</span> <span class="lpage" tagx="lpage" title="lpage">979</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1534/genetics.110.125401">10.1534/genetics.110.125401</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21212233">21212233</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qüesta</span> <span class="given-names" tagx="given-names" title="given-names">JI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fina</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casati</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DDM1 and ROS1 have a role in UV-B induced- and oxidative DNA damage in A. thaliana</span> <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">12</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2013.00420">10.3389/fpls.2013.00420</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23346092">23346092</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhang, F. <i>et al</i>. The F-box protein ZYGO1 mediates bouquet formation to promote homologous pairing, synapsis, and recombination in rice meiosis. <i>Plant Cell</i> <b>29</b>, tpc.00287.2017 (2017). </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Jong</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolters-Arts</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">García-Martínez</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mariani</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vriezen</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Solanum lycopersicum AUXIN RESPONSE FACTOR 7 (SlARF7) mediates cross-talk between auxin and gibberellin signalling during tomato fruit set and development </span> <span class="source" tagx="source" title="source">J. Exp. Bot.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">617</span> <span class="lpage" tagx="lpage" title="lpage">626</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/jxb/erq293">10.1093/jxb/erq293</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20937732">20937732</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Brassinosteroids regulate the differential growth of arabidopsis hypocotyls through auxin signaling components IAA19 and ARF7 </span> <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">887</span> <span class="lpage" tagx="lpage" title="lpage">904</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/mp/sss123">10.1093/mp/sss123</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23125315">23125315</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-throughput sequencing discovery of conserved and novel MicroRNAs in Chinese cabbage (Brassica rapa L. ssp. pekinensis) </span> <span class="source" tagx="source" title="source">Mol. Genet. Genomics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">287</span> <span class="fpage" tagx="fpage" title="fpage">555</span> <span class="lpage" tagx="lpage" title="lpage">563</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-012-0699-3">10.1007/s00438-012-0699-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22643909">22643909</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuker</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mfold web server for nucleic acid folding and hybridization prediction</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">3406</span> <span class="lpage" tagx="lpage" title="lpage">3415</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkg595">10.1093/nar/gkg595</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12824337">12824337</a> </span> </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhang, Z., Jiang, L., Wang, J., Gu, P. &amp;amp; Chen, M. MTide: an integrated tool for the identification of miRNA—target interaction in plants. <i>Bioinformatics</i> 1–2, 10.1093/bioinformatics/btu633 (2014). </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">BH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">XP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cobb</span> <span class="given-names" tagx="given-names" title="given-names">GP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evidence that miRNAs are different from other RNAs</span> <span class="source" tagx="source" title="source">Cell. Mol. Life Sci.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">246</span> <span class="lpage" tagx="lpage" title="lpage">254</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00018-005-5467-7">10.1007/s00018-005-5467-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16395542">16395542</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Fahlgren, N. <i>et al</i>. High-throughput sequencing of Arabidopsis microRNAs: Evidence for frequent birth and death of MIRNA genes. <i>PLoS One</i> <b>2</b> (2007). </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">PX</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PsRNATarget: A plant small RNA target analysis server</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">155</span> <span class="lpage" tagx="lpage" title="lpage">159</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr319">10.1093/nar/gkr319</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20817928">20817928</a> </span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of Relative Gene Expression Data Using Real-Time Quantitative PCR and the 2−ΔΔCT Method </span> <span class="source" tagx="source" title="source">Methods</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">402</span> <span class="lpage" tagx="lpage" title="lpage">408</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Jia, L. <i>et al</i>. Identification of the conserved and novel miRNAs in mulberry by high-throughput sequencing. <i>PLoS One</i> <b>9</b> (2014). </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">German</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schroth</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Construction of Parallel Analysis of RNA Ends (PARE) libraries for the study of cleaved miRNA targets and the RNA degradome </span> <span class="source" tagx="source" title="source">Nat. Protoc.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">356</span> <span class="lpage" tagx="lpage" title="lpage">362</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2009.8">10.1038/nprot.2009.8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19247285">19247285</a> </span> </span> </li> <li tag="ref"> <a name="CR82" /> <span class="label" tagx="label" title="label">82.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">C-P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of Ice Plant (Mesembryanthemum crystallinum L.) MicroRNAs Using RNA-Seq and Their Putative Roles in High Salinity Responses in Seedlings </span> <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2016.01143">10.3389/fpls.2016.01143</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26858731">26858731</a> </span> </span> </li> </ul> </div> </div>  </body></html>