<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3961409">3961409</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24651267">24651267</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-13-41725</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0092598">10.1371/journal.pone.0092598</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agricultural Methods</span>: <span class="subject" title="subject">Sustainable Agriculture</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Transcriptome Analysis</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span>: <span class="subject" title="subject">Crop Genetics</span> </div>: <span class="subject" title="subject">Gene Identification and Analysis</span> <span class="subject" title="subject">Genomics</span> <span class="subject" title="subject">Molecular Genetics</span> </div>: <span class="subject" title="subject">Plant Science</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Transcriptome Analysis of Salt Tolerant Common Bean ( <i>Phaseolus vulgaris</i> L.) under Saline Conditions </div> <div class="alt-title" title="alt-title">
Common Bean Transcriptome Analysis under Salinity</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiz</span> <span class="given-names" tagx="given-names" title="given-names">Mahmut Can</span> </span> <a href="#aff1" /> <a href="#cor1"> <sup>*</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canher</span> <span class="given-names" tagx="given-names" title="given-names">Balkan</span> </span> <a href="#aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niron</span> <span class="given-names" tagx="given-names" title="given-names">Harun</span> </span> <a href="#aff1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turet</span> <span class="given-names" tagx="given-names" title="given-names">Muge</span> </span> <a href="#aff1" /> </span> </div> <span class="citation_author_institution" id="aff1">[], <span class="addr-line" title="addr-line">Bogazici University Department of Molecular Biology and Genetics, Istanbul, Turkey</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">David D.</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">USDA-ARS-SRRC, United States of America</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">mcanhiz@boun.edu.tr</span> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: MCH MT. Performed the experiments: MCH BC HN. Analyzed the data: MCH BC HN. Contributed reagents/materials/analysis tools: MCH BC HN. Wrote the paper: MCH BC HN MT. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2014</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2014-3-3</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e92598</span> <span class="history" title="history"> <span class="received" title="received">received: 2013-10-11</span> <span class="accepted" title="accepted">accepted: 2014-2-24</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Hiz et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2014</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Salinity is one of the important abiotic stress factors that limit crop production. Common bean, <i>Phaseolus vulgaris</i> L., a major protein source in developing countries, is highly affected by soil salinity and the information on genes that play a role in salt tolerance is scarce. We aimed to identify differentially expressed genes (DEGs) and related pathways by comprehensive analysis of transcriptomes of both root and leaf tissues of the tolerant genotype grown under saline and control conditions in hydroponic system. We have generated a total of 158 million high-quality reads which were assembled into 83,774 all-unigenes with a mean length of 813 bp and N50 of 1,449 bp. Among the all-unigenes, 58,171 were assigned with Nr annotations after homology analyses. It was revealed that 6,422 and 4,555 all-unigenes were differentially expressed upon salt stress in leaf and root tissues respectively. Validation of the RNA-seq quantifications (RPKM values) was performed by qRT-PCR (Quantitative Reverse Transcription PCR) analyses. Enrichment analyses of DEGs based on GO and KEGG databases have shown that both leaf and root tissues regulate energy metabolism, transmembrane transport activity, and secondary metabolites to cope with salinity. A total of 2,678 putative common bean transcription factors were identified and classified under 59 transcription factor families; among them 441 were salt responsive. The data generated in this study will help in understanding the fundamentals of salt tolerance in common bean and will provide resources for functional genomic studies. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This work was financially supported by The Scientific and Technological Research Council of Turkey (TUBITAK 110T922) and Bogazici University Research Funds (BAP 6674). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Soil salinity is one of the most severe abiotic stress factors limiting the productivity of agriculture. Although most plants are glycophytes that are highly sensitive to saline environment, halophytes are plants that naturally grow under saline conditions throughout their life cycle. Salinity effects nearly 20% of all irrigated lands worldwide <a href="#pone.0092598-Munns1">[1]</a> and expected to reach around 50% in the near future <a href="#pone.0092598-Mahajan1">[2]</a>. A soil is considered saline if the electrical conductivity of its saturation (EC) is above 4 dS/m <a href="#pone.0092598-Hide1">[3]</a> which is equivalent to approximately 40 mM NaCl. </p> <p>As a member of grain legumes and a glycophyte crop, common bean ( <i>Phaseolus vulgaris</i> L.) is a major source of human dietary protein, minerals, vitamins, and represents nearly half of the consumed grain legumes worldwide <a href="#pone.0092598-Broughton1">[4]</a>. Common bean is also vital in agriculture as it forms root nodules via symbiotic associations with nitrogen fixing bacteria <a href="#pone.0092598-Broughton1">[4]</a>. Nearly 60% (360 Mt) of the annual fresh bean production in Turkey (FAO: <a href="http://faostat.fao.org/faostat">http://faostat.fao.org/faostat</a>) is highly dominated in Black Sea region where the soil salinity levels can reach up to 2–4 dS/m <a href="#pone.0092598-Madakbas1">[5]</a>. However, it is known that even at 1 dS/m salinity level, the productivity of common bean can be reduced up to 20% <a href="#pone.0092598-Chinnusamy1">[6]</a>.Thus, understanding the fundamentals of salt tolerance in common bean, eventual development of improved varieties and their introduction to saline environments are imperative in agriculture. </p> <p>Although halophytes may use avoidance mechanisms, glycophytes tolerate salinity by minimizing ion disequilibrium and the consequent secondary effects <a href="#pone.0092598-Touchette1">[7]</a>. In other words, tolerance mechanisms require concerted actions of mechano-receptors, ion transport channels, and secondary signal molecules to maintain ion homeostasis as well as cascades of gene activations for hormonal metabolism, signal transduction pathways, and stress responses <a href="#pone.0092598-Huang1">[8]</a>– <a href="#pone.0092598-Silva1">[12]</a>. </p> <p>Considering the multifactorial nature of tolerance responses, development of tolerant plants for the benefit of sustainable crop improvement still awaits accumulation of additional knowledge about the identity of components that are involved in this process. </p> <p>Recent developments in high-throughput approaches to study gene expression profile have emerged as an important tool to understand how plants respond to biotic and abiotic stresses. In the last few years, there have been accumulating reports on RNA-sequencing data and expression profiling on both model plants and agriculturally important crops <a href="#pone.0092598-Seki1">[13]</a>– <a href="#pone.0092598-Postnikova1">[21]</a> to identify genes involved in stress responses. Although, recently such high throughput transcriptome assemblies have been started in legumes <a href="#pone.0092598-Libault1">[22]</a>– <a href="#pone.0092598-Wu1">[30]</a>, there are still only a handful of studies regarding the transcriptome analysis under abiotic stress conditions in these species. These studies include the effects of drought, saline-alkaline conditions and salt stress in gene expression profiling of chickpea <a href="#pone.0092598-Hiremath1">[31]</a>, soybean <a href="#pone.0092598-Fan1">[32]</a>, <i>Medicago truncatula</i> <a href="#pone.0092598-Zahaf1">[33]</a>, and alfa alfa <a href="#pone.0092598-Postnikova1">[21]</a> respectively. However, there is no such transcriptome analysis under abiotic stress available yet for common bean. In combination with continuously generated reference genome sequences for diverse legume species <a href="#pone.0092598-Phytozome1">[34]</a>– <a href="#pone.0092598-Varshney2">[39]</a>, next generation RNA sequence analyses will provide valuable information for both identification and cloning of stress tolerance genes which can be used to improve varieties with enhanced tolerance mechanisms. </p> <p>In this study, we used the Illumina high-throughput RNA-sequencing platform for transcriptomic analysis of a salt tolerant common bean, Ispir genotype. We aimed to identify differentially expressed genes (DEGs) and related pathways by comprehensive analysis of data from both root and leaf tissues of the tolerant genotype grown under saline and control conditions in a hydroponic system. <i>De novo</i> assemblies of the sequencing data, functional annotations of unigenes, and their characterization with gene ontology and metabolic pathway analysis provided potential lists of candidate genes. Functional identification of these candidates using reverse genetics approaches in our ongoing studies will contribute to the understanding of salt tolerance mechanisms. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantgrowthandsalttreatments" title="sec"> <div class="title" tagx="title" title="title">
Plant growth and salt treatments</div> <p>The seeds of salt tolerant "Ispir" variety were kindly supplied by Prof. H. Yildiz Dasgan (Cukurova University Department of Horticulture, Adana, Turkey). The seeds were surface sterilized in a solution containing 5% (v/v) hypochlorite for 5 min and rinsed three times with distilled water. The seeds were germinated in vermiculite containing plug trays at 24/20°C cycle under a 16-h light/8-h dark photoperiod with 300 μmol m <sup>−2</sup> s <sup>−1</sup> light intensity, and 50–60% relative humidity up to the fully expanded foliage stage by daily watering with hydroponic nutrient solution containing 3 mM Ca(NO <sub>3</sub>) <sub>2</sub>, 1 mM MgSO <sub>4</sub>, 0.9 mM K <sub>2</sub>SO <sub>4</sub>, 0.2 mM KH <sub>2</sub>PO <sub>4</sub>, 0.1 mM FeEDTA, 10 nM H <sub>3</sub>BO <sub>3</sub>, 1 nM MnSO <sub>4</sub>, 1 nM ZnSO <sub>4</sub>, 0.1 nM CuSO <sub>4</sub>, and 0.01 nM (NH) <sub>6</sub>Mo <sub>7</sub>O <sub>24</sub> <a href="#pone.0092598-Dasgan1">[40]</a>. Eight seedlings were transferred to two pots (four seedlings each) containing hydroponic nutrient solution that was replenished daily. During salt treatment, the photoperiod was also kept the same as in the germination conditions. Once plants reached trifoliate stage (five days post transfer), one pot was left as control and the other was exposed to gradually increasing NaCl concentrations. In this study, the hydroponics system was preferred to keep the nutrients and NaCl levels under strict control to achieve homogenous growth of the plants. Furthermore to minimize the risk of plasmolysis due to the osmotic shock <a href="#pone.0092598-Shavrukov1">[41]</a> during salt treatments "gradual step acclimation" method was used <a href="#pone.0092598-Sanchez1">[42]</a>, thus salt application was started with 50 mM in the first day, followed by 100 mM in the second day, and reached to 125 mM between days three to five in hydroponic nutrient solution. </p> </div> <div class="samplecollectionandrnaisolation" title="sec"> <div class="title" tagx="title" title="title">
Sample collection and RNA isolation</div> <p>The root and the leaf tissues from salt treated and control plants were sampled at the fifth day of the salt treatment. Samples were frozen in liquid nitrogen and stored at −80°C prior to RNA extractions. Total RNAs were extracted with RNeasy Plant kit (Qiagen, Hilden, Germany) according to the manufacturer's instructions. The concentrations of RNA samples were determined by NanoDrop 1000 spectrophotometer (Thermo Fisher Scientific, Wilmington, DE). RNA quality was assessed by 1% denaturing agarose gel electrophoresis. </p> <p>Initially, RNA samples from root tissues of two control (RC: Root Control; RC1 and RC2) as well as two salt treated plants (RS: Root-Salt treated; RS1 and RS2) were isolated as two biological replicates, and they were pooled as described in <a href="#pone-0092598-g001">Figure 1A and 1B</a>. Forty μg of total RNA from the two biological replicates were sequenced by Illumina HiSeqTM 2000 system (BGI, Shenzen, China). Pearson correlation coefficients between the RPKM (reads per kilobase per million reads) values of the two biological replicates were calculated as 0.99 and 0.97 for control and treated samples respectively ( <a href="#pone-0092598-g001">Figure 1C and 1D</a>). Due to the observation of high correlation within the biological replicates of root tissues, we pooled RNA samples directly from leaf tissues of the same four plants from both control (LC: Leaf Control) and treated samples as a cost effective strategy <a href="#pone.0092598-Liu1">[43]</a>, <a href="#pone.0092598-Everett1">[44]</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Schematic representation of the sample pooling strategy and correlation analysis of root samples RPKM values. </div> <p>Black lines indicate the pooling of leaf and root samples from control plants (A) and salt treated plants (B). Graphs show the correlation analysis within root control (C) and salt treated (D) samples. Nomenclature: LC: Leaf Control; LS: Leaf Salt treated; RC1 and RC2: Root Control 1 and 2; RS1 and RS2: Root Salt treated 1 and 2. </p>   </div> </div> <div class="cdnalibraryconstructionandilluminasequencing" title="sec"> <div class="title" tagx="title" title="title">
cDNA library construction and Illumina sequencing</div> <p>RNA quality and quantity were verified using Nanodrop 1000 spectrophotometer and Agilent 2100 bioanalyzer before cDNA library generation at BGI (Shenzen, China). Total RNAs were treated with DNase I before poly (A+) mRNA enrichment with oligo dT magnetic beads. Poly (A+) RNAs were digested into 200–700 nt fragments by RNA Fragmentation Reagent, and random hexamer primed poly (A+) RNA fragments were transcribed into first-strand cDNAs. Subsequently in the presence of dNTPs, RNase H, and DNA polymerase I, the second strands were synthesized, purified using QiaQuick PCR purification kit (Qiagen, Hilden, Germany), and used for end repair single adenine nucleotide addition. The sequencing adaptors were ligated to the fragments. The paired end library constructions were finalized by size selection with agarose gel electrophoresis and selective enrichment of the cDNA fragments with PCR amplification. The libraries were sequenced on a flow cell using Illumina HiSeq2000 sequencing instrument after quality control with Agilent 2100 bioanalyzer and qPCR to detect fragment size and concentration. </p> </div> <div class="denovoassemblyanddataanalyses" title="sec"> <div class="title" tagx="title" title="title"> <i>De novo</i> assembly and data analyses </div> <p>Subsequent to adapter trimming, the raw data were filtered for reads with more than 5% ambigous bases and/or low quality reads with bases 20% of which has a Phred score less than 10. The clean reads from each library were <i>de novo</i> assembled into contigs with Trinity software <a href="#pone.0092598-Grabherr1">[45]</a> ( <a href="http://sourceforge.net/projects/trinityrnaseq/files">http://sourceforge.net/projects/trinityrnaseq/files</a>) setting k-mer length to 25. This has generated a total of six subtranscriptomes. Contigs of the subtranscriptomes were pooled, clustered, and assembled using the Trinity software to obtain sequences that can no longer be extended on either end, and they were referred as all-unigenes. All-unigene sequences were aligned using BLASTx against NCBI non-redundant (Nr) protein, Swiss-Prot protein, the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway, and Cluster of Orthologous Groups (COG) databases to determine sequence orientations and protein coding region predictions. Proteins with the highest ranks in BLASTx alignment results were used to predict coding region sequences. All-unigenes that could not be aligned to sequences in any of the databases mentioned above were scanned by ESTScan software <a href="#pone.0092598-Iseli1">[46]</a> to predict sequence orientations and coding regions. For annotations, all-unigenes were searched against the Nr database using BLASTx with 10 <sup>−5</sup> as <i>E</i>-value cut-off point and sequences with the highest similarities were retrieved. To obtain Gene Ontology (GO) terms regarding biological process, molecular function and cellular component <a href="#pone.0092598-Ashburner1">[47]</a> descriptions, the resulting BLASTx hits were analyzed by Blast2GO software <a href="#pone.0092598-Conesa1">[48]</a>. The GO annotations were functionally classified by WEGO <a href="#pone.0092598-Ye1">[49]</a> software for gene function distributions of common bean species at macro level. BLASTx analysis against the KEGG pathway database was also performed to assign putative metabolic pathways to all-unigenes. </p> <p>To estimate gene expression levels, six (four from the root libraries, two from the leaf libraries) RPKM values were calculated for every all-unigene by mapping six subtranscriptomes with SOAP2 software <a href="#pone.0092598-Li1">[50]</a>, applying three mismatches as threshold. The clean reads mapped to more than one all-unigene were not used to calculate RPKM. Corrections for false positive and false negative errors were performed by calculating the FDR (false discovery rate) values <a href="#pone.0092598-Benjamini1">[51]</a>. The DEGs were selected by using a FDR-value ≤0.001 and the absolute ratio of log <sub>2</sub> (RPKM-tr/RPKM-cont) ≥1 as threshold values. The GO terms and the KEGG pathways that were enriched within the DEGs were identified by publicly available agriGO <a href="#pone.0092598-Du1">[52]</a> and FatiGO <a href="#pone.0092598-AlShahrour1">[53]</a> software respectively. To analyze the functional significance of enriched terms hypergeometric tests were employed by using the common bean transcriptome as background, setting the FDR and the Adjusted <i>P</i>-values lower than 0.05 for the agriGO and FatiGO software respectively. </p> </div> <div class="qrt-pcranalyses" title="sec"> <div class="title" tagx="title" title="title">
qRT-PCR analyses</div> <p>Quantitative Reverse Transcription PCR (qRT-PCR) analyses were performed for 43 unigenes ( <a href="#pone.0092598.s004">Table S1</a>) using the <i>insulin degrading enzyme</i> ( <i>IDE</i>, Unigene29213) [GenBank: FE702602.1] and the <i>actin-11</i> ( <i>Act-11</i>, CL442.Contig3) [GenBank: CV529679.1] genes of common bean as stably expressed internal references under salt stress <a href="#pone.0092598-Borges1">[54]</a>. Four individual plants from both control and treated groups were used in qRT-PCR analyses as biological replicates. For each biological replicates, three qRT-PCR reactions were performed as technical replicates. Single stranded cDNAs were synthesized from one μg of total RNA using RevertAid First Strand cDNA Synthesis Kit (Thermo Fisher Scientific, Wilmington, DE). Gene-specific primers were designed using the Primer Design module of CLC Main Workbench (version 6.0) software. qRT-PCR analyses were performed by PikoReal 96 Real-time PCR system (Thermo Fisher Scientific, Wilmington, DE) using 10 ng first strand cDNAs and AccuPower 2X Greenstar master mix (Bioneer, Daejon, Korea) with 0.25 pmol forward and reverse primers in a 10 μl solution. Upon initial denaturation at 95°C for five min., 40 cycles of denaturation at 95°C for 10 s and 60 s of annealing, and extension steps were performed at specific temperatures ( <a href="#pone.0092598.s005">Table S2</a>) optimized for each primer pair. The relative expression levels were calculated using the 2 <sup>–ΔΔCt</sup> method for each gene and were normalized to the geometric average of Ct (threshold cycle) values of the internal reference genes. The Student's <i>t</i>-test was applied to determine the significant differences in expression levels between treated and control samples collected at the fifth day of salt treatment. </p> </div> <div class="identificationofputativetranscriptionfactors" title="sec"> <div class="title" tagx="title" title="title">
Identification of putative transcription factors</div> <p>Sequences for Hidden Markov Model (HMM) motifs of the transcription factors (TFs) belonging to <i>Glycine max</i>, <i>Medicago truncatula</i> and <i>Lotus japonicus</i> were acquired from the Legume Transcription Factor Database (Legume TFDB) <a href="#pone.0092598-Mochida1">[55]</a>, <a href="#pone.0092598-Mochida2">[56]</a> and combined to create a single legume motif database consisting of 61 TF families. An all-unigene was accepted as a putative TF if it has shown 90% or more sequence homology ( <i>E</i>-value ≤10 <sup>−10</sup>) with any of the HMM motifs of at least one of the three legume species. The putative TFs within the all-unigenes and the DEGs were categorized based on the TF families and their presence in root or leaf tissues. </p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="denovoassemblyofthesequencingdata" title="sec"> <div class="title" tagx="title" title="title"> <i>De novo</i> assembly of the sequencing data </div> <p>Previous reports have emphasized that during NaCl applications, Na <sup>+</sup> concentrations reach to toxic levels and plants start to react to salt stress rather than osmotic shock only after 24–72 hours gradual step acclimation <a href="#pone.0092598-Shavrukov1">[41]</a>, <a href="#pone.0092598-Sanchez1">[42]</a>. In our studies with NaCl treatment on Ispir variety have also shown that detectable physiological effects of salt stress become significant at the fifth day (72 hours after gradual step acclimation) of treatment in hydroponics cultures (data not shown). Therefore, the sequencing was performed on the leaf and the root RNA samples collected at this time point. The Illumina sequencing was performed separately for each RNA sample (LC, LS, RC1, RC2, RS1, and RS2) and six subtranscriptomes ( <a href="#pone-0092598-g001">Figure 1A and 1B</a>) were generated with 90-bp raw reads. More than 158 million clean reads remained from the six subtranscriptomes ( <a href="#pone-0092598-t001">Table 1</a>) with ∼97% Q20 bases (i.e. percentage of sequences with sequencing error rate lower than 1%) that constituted over 14 GBase (7.05×10 <sup>9</sup> nt in control, and 7.23×10 <sup>9</sup> nt in salt treated samples) of data after quality assessment and data filtering. All clean reads were deposited in the NCBI Short Read Archive (SRA) database and can be accessed with SRP029243 accession number. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0092598.t001">10.1371/journal.pone.0092598.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Statistics of raw data for <i>P.vulgaris</i> L. transcriptome. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Samples</td> <td>Total Reads</td> <td>Total Nucleotides (nt)</td> <td>Q20 percentage</td> <td>GC percentage</td> </tr> </thead> <tbody> <tr> <td>1LC</td> <td>25 907 332</td> <td>2 331 659 880</td> <td>97.02%</td> <td>46.02%</td> </tr> <tr> <td>2RC</td> <td>26 481 724</td> <td>2 383 355 160</td> <td>96.91%</td> <td>46.70%</td> </tr> <tr> <td>3RC</td> <td>25 998 386</td> <td>2 339 854 740</td> <td>96.94%</td> <td>46.97%</td> </tr> <tr> <td>1LS</td> <td>27 191 626</td> <td>2 447 246 340</td> <td>97.12%</td> <td>46.19%</td> </tr> <tr> <td>2RS</td> <td>27 280 462</td> <td>2 455 241 580</td> <td>96.91%</td> <td>47.08%</td> </tr> <tr> <td>3RS</td> <td>25 821 046</td> <td>2 323 894 140</td> <td>96.83%</td> <td>47.30%</td> </tr> </tbody> </table> </div> </div> <p>The clean reads from LC, LS, RC1, RC2, RS1, and RS2 subtranscriptomes were <i>de novo</i> assembled using the Trinity software, which generated 52,858, 51,564, 60,590, 64,986, 59,510, and 58,174 unigenes, respectively ( <a href="#pone-0092598-t002">Table 2</a>). Further assembly of the subtranscriptomes generated the common bean transcriptome consisting of 83,774 all-unigenes. The total length of all-unigenes was 68,147,816 bp with a mean length of 813 bp and N50 of 1,449 bp (i.e 50% of the assembled bases were incorporated into all-unigenes of 1,449 bp or longer) ( <a href="#pone-0092598-t002">Table 2</a>). The lengths of all-unigenes ranged between 150 and 17,022 bp. In total, 55,471 (66.72%) all-unigenes were unique and the remaining 28,303 (33.8%) fell into 5,403 clusters in which the number of unigenes ranged between 2 and 42. Among the 83,774 all-unigenes, 63,011 (71% Nr-annotated) were expressed in the leaf samples whereas 73,523 (69% Nr-annotated) were expressed in the root samples. Randomly distributed clean reads from the six subtranscriptomes evenly covered the common bean transcriptome with an average of ∼20 fold coverage depth ( <a href="#pone.0092598.s001">Figure S1</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0092598.t002">10.1371/journal.pone.0092598.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Statistics analysis for unigene assembly of <i>P. vulgaris L</i>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>Length of Unigenes (bp)</td> <td>Number of unigenes</td> </tr> <tr> <td /> <td>LC</td> <td>RC1</td> <td>RC2</td> <td>LS</td> <td>RS1</td> <td>RS2</td> <td>All</td> </tr> </thead> <tbody> <tr> <td>150–500</td> <td>31659</td> <td>37290</td> <td>41636</td> <td>29684</td> <td>38096</td> <td>37683</td> <td>44558</td> </tr> <tr> <td>500–1000</td> <td>11135</td> <td>12780</td> <td>12836</td> <td>10819</td> <td>12194</td> <td>11878</td> <td>15702</td> </tr> <tr> <td>1000–1500</td> <td>5104</td> <td>5499</td> <td>5514</td> <td>5280</td> <td>4939</td> <td>4612</td> <td>9754</td> </tr> <tr> <td>1500–2000</td> <td>2622</td> <td>2675</td> <td>2678</td> <td>2972</td> <td>2352</td> <td>2202</td> <td>6115</td> </tr> <tr> <td>≥ 2000</td> <td>2338</td> <td>2346</td> <td>2322</td> <td>2809</td> <td>1929</td> <td>1799</td> <td>7645</td> </tr> <tr> <td>Total</td> <td>52858</td> <td>60590</td> <td>64986</td> <td>51564</td> <td>59510</td> <td>58174</td> <td>83774</td> </tr> <tr> <td>Mean length</td> <td>644</td> <td>613</td> <td>585</td> <td>685</td> <td>579</td> <td>568</td> <td>813</td> </tr> <tr> <td>N50 (bp)</td> <td>1016</td> <td>946</td> <td>906</td> <td>1121</td> <td>866</td> <td>843</td> <td>1449</td> </tr> <tr> <td>Total Length (bp)</td> <td>34018974</td> <td>37138056</td> <td>38042659</td> <td>35328894</td> <td>34441515</td> <td>33036044</td> <td>68147816</td> </tr> </tbody> </table> </div> </div> </div> <div class="annotationandclassificationofcommonbeantranscriptome" title="sec"> <div class="title" tagx="title" title="title">
Annotation and classification of common bean transcriptome</div> <p>The BLASTx searches ( <i>E</i>-value ≤ 10 <sup>−5</sup>) revealed that out of 83,774 all-unigenes, 58,171 (69.4%), 44,174 (52.7%), and 28,564 (34.1%) showed significant similarity to the sequences in Nr, Swiss-Prot and KEGG databases respectively. Among the all-unigenes, 43,519 (51.9%) were commonly annotated in Nr and Swiss-Prot; 41,087 (49%) in Nr and KEGG; 27,116 (32.4%) in KEGG and Swiss-Prot, and 27,078 (32.3%) were annotated in all three databases. For the 91% of the Nr -annotated sequences, the <i>E</i>-values were less than or equal to 10 <sup>−10</sup>. In addition, 24,909 (29.7%) all-unigenes did not match significantly to any of the sequences in these databases. </p> <p>More than 46% of all-unigenes smaller than 500 bp had BLASTx hits in the Nr database whereas for those longer than 500 bp the ratio was over 90.4% ( <a href="#pone-0092598-g002">Figure 2A</a>). This has indicated that the longer the all-unigenes were the more they were likely to have Nr annotations and the lower <i>E</i>-values than the shorter all-unigenes ( <a href="#pone-0092598-g002">Figure 2B</a>). The great majority (over 96%) of Nr annotated all-unigenes showed the highest homology to the plant proteins ( <a href="#pone-0092598-g002">Figure 2C</a>). Furthermore, 64.4% were specifically annotated with sequences from legume species ( <a href="#pone-0092598-g002">Figure 2C</a>), which indicated the reliability of our transcriptome analysis. In our data set, we also found few all-unigenes, which were annotated by phytoplankton and plant pathogens proteins, most probably due to the hydroponic growth conditions used in this study. Similar to the observation in an earlier report <a href="#pone.0092598-Huang2">[57]</a>, 96 all-unigenes expressed exclusively in roots were Nr annotated from <i>Phytophthora sojae</i>, which is a soybean pathogen that causes rotting of stem and roots <a href="#pone.0092598-Tyler1">[58]</a>. This observation indicates that our transcriptome analysis detected transcripts even in trace amounts. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Reliability analysis of all-unigene annotations.</div> <p>Histogram of all-unigenes according to the Nr database annotations with 500 bp intervals (A). E-value distributions for Nr annotations based on BLASTx analysis; panel B1 represent the percentage distributions of all-unigenes and panel B2 represent all-unigenes longer than 500 bp. Genera distribution of all-unigenes according to the Nr database annotations (C). </p>   </div> <p>Using WEGO software the 27,959 all-unigenes were assigned to a total of 98,948 GO annotations which fall into three main categories; 19,154 all-unigenes were within 28 terms of the "Biological Process"category, 18,175 all-unigenes were within 15 terms of the "Cellular Component" category, and 22,190 all-unigenes were within 12 terms of the "Molecular Function" category ( <a href="#pone.0092598.s002">Figure S2</a>). </p> <p>All-unigenes were assigned to KEGG pathway and COG databases, a total of 28,564 all-unigenes were annotated in 125 individual pathways ( <a href="#pone.0092598.s006">Table S3</a>) and a total of 23,141 all-unigenes were assigned to 25 functional classes respectively ( <a href="#pone.0092598.s003">Figure S3</a>). </p> </div> <div class="qrt-pcrverificationofrpkmbasedgeneexpression" title="sec"> <div class="title" tagx="title" title="title">
qRT-PCR verification of RPKM based gene expression</div> <p>We performed qRT-PCR analysis for 43 selected all-unigenes ( <a href="#pone.0092598.s004">Table S1</a>) from root and leaf samples with specific primers ( <a href="#pone.0092598.s005">Table S2</a>) to assess the reliability of our sequencing results ( <a href="#pone-0092598-g003">Figure 3</a>). Among these selected all-unigenes, 15 were upregulated (log <sub>2</sub> (RPKM tr/cont) ranged between 1.57 and 12.28), 17 were non-differentially expressed (log <sub>2</sub> (RPKM tr/cont) ranged between −0.77 and 0.89), and 11 were downregulated (log <sub>2</sub> (RPKM tr/cont) ranged between −1.18 and −13.13). The comparison of qRT-PCR and sequencing results revealed a high correlation for the selected unigenes (Pearson r = 0.91, <a href="#pone-0092598-g003">Figure 3</a>). In addition to high correlation between qRT-PCR and RNA-seq results, the production of expected fragment sizes using designed primers have also confirmed the reliability of <i>de novo</i> assembly. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Relative expression level comparisons of qRT-PCR and RNA-seq results for selected all-unigenes. </div> <p>The qRTPCR results of 43 all-unigenes were compared to the RPKM values (log <sub>2</sub> (RPKM-tr/RPKM-cont) ≥1). </p>   </div> </div> <div class="identificationandfunctionalclassificationofdegs" title="sec"> <div class="title" tagx="title" title="title">
Identification and functional classification of DEGs</div> <p>Upon comparison against control groups, all-unigenes with more than or equal to two-fold change (│log <sub>2</sub> (RPKM tr/cont) │≥1) in their gene expression level with a FDR value below 10 <sup>-3</sup> were defined as DEGs. Based on these criteria, the number of DEGs in leaves and roots were 6,422 and 4,555, respectively ( <a href="#pone-0092598-g004">Figure 4</a>; <a href="#pone.0092598.s007">Table S4</a> and <a href="#pone.0092598.s008">S5</a>). Out of 6,422 DEGs, 3,024 (88% Nr-annotated) were upregulated, and 3,398 (88% Nr-annotated) were downregulated in the leaf samples, whereas among the 4,555 DEGs of the roots, 1,237 (89% Nr-annotated) were upregulated, and 3,318 (76% Nr-annotated) were downregulated upon salt treatment ( <a href="#pone-0092598-g004">Figure 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Venn diagram showing the number of DEGs.</div> <p>The sums of the numbers in each circle were indicated within the parantheses of the corresponding title. </p>   </div> <p>In both leaf and root tisssues, the enriched terms in the GO database ( <a href="#pone-0092598-g005">Figure 5</a>, <a href="#pone.0092598.s009">Table S6</a> and <a href="#pone.0092598.s010">S7</a>) were in agreement with the KEGG metabolic pathways ( <a href="#pone-0092598-g006">Figure 6</a>, <a href="#pone.0092598.s011">Table S8</a> and <a href="#pone.0092598.s012">S9</a>). Within the leaf tissues upregulated genes, the terms related with the secondary metabolite metabolism and the membrane transport activity were mostly enriched, however the macromolecular energy metabolism related terms were highly enriched within the downregulated genes ( <a href="#pone-0092598-g005">Figure 5A</a> and <a href="#pone-0092598-g006">6A</a>). Although, the DEGs in the root tissues have also shown a similar pattern with the leaf tissues for the terms related with the secondary metabolite metabolism and the membrane transport activity, the terms related with the macromolecular energy metabolism were only slightly enriched within the upregulated genes. Additionally, in root tissues, the transcription/translation related terms were notably enriched within the downregulated genes ( <a href="#pone-0092598-g005">Figure 5B</a> and <a href="#pone-0092598-g006">6B</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g005"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Gene Ontology functional enrichment analysis of DEGs.</div> <p>Panel A represents the DEGs in leaf tissues and panel B represents the DEGs in root tissues. (FDR-value &amp;lt;0.05). </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0092598.g006"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Pathway enrichment analysis of DEGs.</div> <p>Panel A represents the DEGs in leaf tissues and panel B represents the DEGs in root tissues. (Adjusted- <i>P</i>-value &amp;lt;0.05). </p>   </div> </div> <div class="identificationofputativetranscriptionfactors(tfs)" title="sec"> <div class="title" tagx="title" title="title">
Identification of putative transcription factors (TFs)</div> <p>In this study, 2,678 putative TFs were identified based on the homology analysis of the all-unigenes against a combined legume database which was constructed from <i>Glycine max</i>, <i>Medicago truncatula</i> and <i>Lotus japonicus</i> HMM motifs. The putative TFs were classified under 59 out of the 61 TF families present in this database. The most abundant 10 TF families were AP2_EREBP, bHLH, PHD, HB, (R1)R2R3_Myb, WRKY_Zn, NAC, bZIP, C3H-TypeI, and Myb_related ( <a href="#pone-0092598-t003">Table 3</a>, <a href="#pone.0092598.s013">Table S10</a>). A total of 441(16%) TFs, 331 in the leaf, and 161 in the root tissues ( <a href="#pone-0092598-t003">Table 3</a> and <a href="#pone.0092598.s013">Table S10</a>) identified as salt responsive. The composition of most abundant TF families were preserved within the leaf DEGs except the C3H-Type I family (replaced by C2C2-Zn-CO like) however in the root DEGs, the bZIP, C3H-TypeI, and Myb_related HSF were replaced by the C2C2_Zn-Dof, and C2H2_Zn TF families ( <a href="#pone.0092598.s013">Table S10</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0092598.t003">10.1371/journal.pone.0092598.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Number of total and differentially expressed bean TFs on family basis.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td /> <td>All-unigene</td> <td>DEGs</td> </tr> <tr> <td>TFs</td> <td>All</td> <td>Leaf</td> <td>Root</td> <td>Leaf</td> <td>Root</td> </tr> </thead> <tbody> <tr> <td> <b>AP2_EREBP</b> </td> <td>222</td> <td>163</td> <td>173</td> <td>42</td> <td>25</td> </tr> <tr> <td> <b>bHLH</b> </td> <td>212</td> <td>160</td> <td>171</td> <td>29</td> <td>18</td> </tr> <tr> <td> <b>PHD</b> </td> <td>172</td> <td>157</td> <td>162</td> <td>14</td> <td>8</td> </tr> <tr> <td> <b>HB</b> </td> <td>157</td> <td>115</td> <td>112</td> <td>18</td> <td>7</td> </tr> <tr> <td> <b>(R1)R2R3_Myb</b> </td> <td>154</td> <td>114</td> <td>122</td> <td>28</td> <td>22</td> </tr> <tr> <td> <b>WRKY_Zn</b> </td> <td>144</td> <td>117</td> <td>122</td> <td>20</td> <td>14</td> </tr> <tr> <td> <b>NAC</b> </td> <td>122</td> <td>95</td> <td>104</td> <td>20</td> <td>13</td> </tr> <tr> <td> <b>bZIP</b> </td> <td>111</td> <td>75</td> <td>85</td> <td>14</td> <td>3</td> </tr> <tr> <td> <b>C3H-TypeI</b> </td> <td>97</td> <td>90</td> <td>91</td> <td>7</td> <td>3</td> </tr> <tr> <td> <b>Myb_related</b> </td> <td>94</td> <td>79</td> <td>78</td> <td>16</td> <td>3</td> </tr> <tr> <td> <b>TOTAL</b> </td> <td>2678</td> <td>2141</td> <td>2196</td> <td>331</td> <td>161</td> </tr> </tbody> </table> </div> </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Common bean is an important grain legume that provides 30% of the protein intake in developing countries (FAO: <a href="http://faostat.fao.org/faostat">http://faostat.fao.org/faostat</a>), and the production suffers drastically even in slightly saline soils <a href="#pone.0092598-Chinnusamy1">[6]</a>. Presence of genetic diversity within the species toward salt tolerance <a href="#pone.0092598-Gulduren1">[59]</a> offers a valuable opportunity to identify the key elements that might play a role in salt tolerance. The common bean genotype "Ispir" used in this study is known to be a very old local variety named after Ispir county of Erzurum province where it has been produced by more than 150 years. The variety has been registered in 2008 and patented as "Geographic Trademark" by Turkish patent office ( <a href="http://www.tpe.gov.tr">www.tpe.gov.tr</a>). The ability of the variety to tolerate high salt concentrations even at germination stage without compromising from germination time was well documented <a href="#pone.0092598-Gulduren1">[59]</a>. </p> <p>There exists a substantial amount of reports regarding the physiological responses to salt stress in common bean <a href="#pone.0092598-Gulduren1">[59]</a>, <a href="#pone.0092598-Gama1">[60]</a>. Also there is a considerable collection of ESTs from cDNA libraries obtained from different organs and root nodules of legumes <a href="#pone.0092598-Ramirez1">[61]</a>, <a href="#pone.0092598-Melotto1">[62]</a> including sequences related to abiotic stresses such as phosphorus starvation, rust disease resistance, and drought stress. Within the last few years, there has been increasing number of studies on transcriptome analysis of legume species using high-throughput RNA sequencing approach, among those regarding the effects of salinity were recently reported on <i>Medicago</i> <a href="#pone.0092598-Postnikova1">[21]</a>, <a href="#pone.0092598-Zahaf1">[33]</a>, <i>Glycine</i> <a href="#pone.0092598-Fan1">[32]</a>, <i>Cicer</i> <a href="#pone.0092598-Hiremath1">[31]</a>, <a href="#pone.0092598-Molina1">[63]</a>, and <i>Millettia</i> <a href="#pone.0092598-Huang2">[57]</a>. However, there are only three transcriptomic studies in the literature on common bean conducted by high-throughput RNA-sequencing approach, one of which is a general transcriptome <a href="#pone.0092598-Kalavacharla1">[24]</a>assembly, the other is a data-mining of host resistance gene like sequences <a href="#pone.0092598-Liu2">[64]</a> within the transcriptome and the third one is on the sulfur metabolism in developing seeds <a href="#pone.0092598-Liao1">[65]</a>. Therefore, we aimed to perform a large-scale comparative transcriptome analysis in two different tissues of a salt tolerant common bean variety under salt stress to identify key elements and their related functional pathways, which may play a critical role in stress tolerance responses. </p> <p>In our study, the assembly of the transcriptome relied on the <i>de novo</i> method. The reliability and the sensitivity of our transcriptome were demonstrated by the high percentage of annotations (i.e. Nr-annotations, 69.4%), the ability to detect the trace amounts of transcripts from plant pathogens as well as phytoplankton contaminations (due to hydroponic system), and the correlation between the qRT-PCR results and the RPKM values. Functional annotation of common bean transcriptome also revealed that the highly represented terms in the GO ( <a href="#pone.0092598.s002">Figure S2</a>), KEGG ( <a href="#pone.0092598.s006">Table S3</a>), and COG ( <a href="#pone.0092598.s003">Figure S3</a>) databases were also commonly represented in previous legume transcriptome studies <a href="#pone.0092598-Huang2">[57]</a>, <a href="#pone.0092598-Wang3">[66]</a>, <a href="#pone.0092598-Pang1">[67]</a>. Additionally the two previous common bean transcriptome studies <a href="#pone.0092598-Kalavacharla1">[24]</a>, <a href="#pone.0092598-Liao1">[65]</a> generated comparable number of all-unigenes (94,623 and 77,448 all-unigenes) with our transcriptome assembly (83,774 all-unigenes), although they used pyrosequencing platform. </p> <p>Comparative transcriptional level analysis of salt treated and control samples using RPKM values revealed that 10094 (12%) all-unigenes were salt responsive ( <a href="#pone-0092598-g004">Figure 4</a>, <a href="#pone.0092598.s007">Table S4</a> and <a href="#pone.0092598.s008">S5</a>). Among them 8457 (84%) were Nr-annotated. Although we used the same (or higher) stringent threshold values (│log <sub>2</sub> (RPKM tr/cont) │≥1, FDR ≤0.001) with the previous studies in legumes <a href="#pone.0092598-Ge1">[19]</a>, <a href="#pone.0092598-Postnikova1">[21]</a>, <a href="#pone.0092598-Zahaf1">[33]</a>, a higher number of salt responsive genes were identified in this study except the recently reported study in <i>Milletia pinnata</i> <a href="#pone.0092598-Huang2">[57]</a>. The reasons which may contribute to these disparities could be the differences in sensitivity and resolution between the hybridization <a href="#pone.0092598-Ge1">[19]</a> and sequencing based platforms as well as the differences between the assembly software. Moreover, the differences between salt tolerance characteristics of <i>Milletia pinnata</i>, a transitional species between halophytes and glycophytes <a href="#pone.0092598-Huang2">[57]</a>, and common bean, a glycophyte <a href="#pone.0092598-Chinnusamy1">[6]</a> might be another reason. Additionally, other contributing factors might include the plant growth conditions, direct exposure of plants to high salt concentrations, and the differences in sample collection times. The previous transcriptome studies focused on earlier time points and sudden applications of elevated salt concentrations, whereas our transcriptome analysis was performed using the "gradual step acclimated" plant material and thus reflected the salt responses rather than osmotic shock responses <a href="#pone.0092598-Shavrukov1">[41]</a>. </p> <p>During salt stress within plant, all the major processes such as photosynthesis, protein synthesis, energy and lipid metabolisms are affected <a href="#pone.0092598-Parida1">[68]</a>.Therefore, in order to tolerate this stress, plants require to initiate protective and survival actions by balancing cellular ion concentrations and minimizing ion toxicity through Na <sup>+</sup>/H <sup>+</sup>, K <sup>+</sup> and Cl <sup>-</sup> transporter/antiporter activities to eliminate water and nutrient deficits, regulate osmotic potential changes via synthesis of osmoprotectant and osmoregulants such as sugars, amino acids, amines, and also minimize tissue damage through activation of scavenging pathways to eliminate antioxidant productions <a href="#pone.0092598-Munns1">[1]</a>, <a href="#pone.0092598-Carillo1">[69]</a>, <a href="#pone.0092598-Tomoaki1">[70]</a>. </p> <p>Based on the enrichment analysis of the DEGs, both root and leaf tissue genes were closely related with stress tolerance mechanisms. Major categories of the stress related functions were transmembrane transport activities (such as GO:0031224, GO:0015698, GO:0015103, GO:0015893, ko02010), carbohydrate metabolism (such as GO:0005976, GO:0015925, G0:15926, ko00040, ko00500, ko00052), lipid metabolism (such as GO:0010876, GO:006629, ko00591, ko00565, ko00600), secondary metabolite metabolism and oxidoreductase activity (such as GO:0006720, GO:0016884, GO:0006528 GO:16491 GO:0016679, ko00900, ko00940, ko00944), and transcriptional/translational activities (such as GO:0034728, GO:0022613, GO:0006412, GO:0010467,ko03020, ko03010, ko03013) ( <a href="#pone-0092598-g005">Figure 5</a> and <a href="#pone-0092598-g006">6</a>, <a href="#pone.0092598.s009">Table S6</a>, <a href="#pone.0092598.s010">S7</a>, <a href="#pone.0092598.s011">S8</a>, and <a href="#pone.0092598.s012">S9</a>). </p> <p>Ionic imbalance causes ion toxicity due to replacement of K <sup>+</sup> by Na <sup>+</sup> ions via competitive transport of Na <sup>+</sup> through potassium channels <a href="#pone.0092598-Faal1">[71]</a>. Although cytosolic K <sup>+</sup> is essential as cofactor for several enzyme activities, Na <sup>+</sup> has toxic effects and thus has to be excluded or sequestrated into vacuoles in the cells <a href="#pone.0092598-Faal1">[71]</a>. Therefore to achieve a high K <sup>+</sup>/Na <sup>+</sup> ratio for improved salt tolerance, increased expression of Na <sup>+</sup>/H <sup>+</sup> antiporters with driving force of vacuolar H <sup>+</sup>-ATPases and H <sup>+</sup> pyrophosphatases are important. Importance of these antiporter activities was also implicated once more in our study as the leaf tissues have shown distinctive upregulation in Na <sup>+</sup>/H <sup>+</sup> antiporter (CL4908.Contig3_All) as well as vacuolar H <sup>+</sup>ATPase genes (CL3536.Contig4_All). Additionally, upregulation in the nonselective cyclic nucleotide gated cation channels, <i>CNGC2</i> (CL5050.Contig1_All) in root tissues were striking in our salt tolerant variety, considering the previously implicated role of the CNGC channels in the main pathway of Na <sup>+</sup> entry to roots <a href="#pone.0092598-Tester1">[72]</a>. However, studies in Arabidopsis has shown that when the K <sup>+</sup> level was limited due to elevated Na <sup>+</sup> during stress, activation of nonselective channel <i>At</i>CNGC family members were still crucial to increase K <sup>+</sup> influx to the root cells even at the expense of Na <sup>+</sup> influx <a href="#pone.0092598-Guo1">[73]</a>, <a href="#pone.0092598-Yuen1">[74]</a>. More strikingly, the <i>CNGC2</i> (CL5050.Contig1_All) was shown to be the only member that preferentially conducts K <sup>+</sup> without the transport of Na <sup>+</sup> <a href="#pone.0092598-Leng1">[75]</a>. </p> <p>Similar to Na <sup>+</sup>, Cl <sup>−</sup> is also toxic to the cells, thus Cl <sup>−</sup> homeostasis by endosomal compartmentalization is critical for plant adaptations to salt stress <a href="#pone.0092598-Teakle1">[76]</a>. Although Cl <sup>−</sup> transport genes have been poorly studied, the attention mainly focused on voltage-dependent Cl <sup>−</sup> channels (Cl <sup>−</sup>/H <sup>+</sup> antiporters) localized in endosomal membranes which may function in vacuolar sequestration of Cl <sup>−</sup> to minimize the tissue necrosis and long distance transport in shoots as well as Cl <sup>−</sup> exclusion from roots <a href="#pone.0092598-Samir1">[77]</a>, <a href="#pone.0092598-Teakle2">[78]</a>. Our enrichment analyses have also revealed upregulation of Cl <sup>−</sup> channels and Cl <sup>−</sup>/NO <sub>3</sub> <sup>−</sup> transporter genes in the leaf (CL5256.Contig5_All) and the root tissues (CL5256.Contig6_All) respectively. The actions of Cl <sup>−</sup> channels were correlated with NO <sub>3</sub> <sup>−</sup>, another macronutrient univalent anion in plants, and Cl <sup>−</sup>/NO <sub>3</sub> <sup>−</sup> interactions show analogy to Na <sup>+</sup>/K <sup>+</sup> interactions and selectivity to Na <sup>+</sup> exclusion <a href="#pone.0092598-Teakle2">[78]</a> during salt tolerance. </p> <p>Osmotic potential change in plants is a typical outcome of ionic imbalance during salt stress, which creates domino effect in the activation of multiple metabolic processes. One of these metabolic processes includes increase in the accumulation of highly hydrophilic organic compounds (osmolytes) and hydrophylic proteins <a href="#pone.0092598-Parida1">[68]</a>. Upregulation of both osmolyte biosynthesis enzymes (CL7986.Contig1_All, CL2135.Contig2_All, CL3755.Contig1_All, Unigene14850_All, CL6414.Contig1_All, Unigene27033_All) and the hydrophilic LEA proteins (CL3431.Contig2_All, Unigene12424_All) in both leaf and root transcriptomes were a good indication of their protective role of cytosolic components of cells during osmotic imbalance. Activation of metabolic processes also enhances demands on energy resources, which are supplied by cellular respirations <a href="#pone.0092598-Jacoby1">[79]</a>. Increases in the expression of the genes related with respiration pathway enzymes in both tissues were indicative of such catabolic activities (CL8031.Contig1_All, Unigene25787_All, Unigene20031_All, and CL237.Contig7_All). Catabolic activities bring high risk of oxidative stress in plants which is associated with antioxidant production enzymes <a href="#pone.0092598-MarianaLinsdeOliveira1">[80]</a> to eliminate free oxygen radical accumulation in cells. Several different antioxidant biosynthesis related genes were upregulated within our transcriptome (CL1487.Contig1_All, Unigene1099_All, Unigene20329_All, and Unigene5947_All). During the battle against the osmotic as well as ionic stress in salinity, plant tolerance requires considerable efforts in maintenance of cell structure integrity by reinforcements or reorganizations in membranes, and cell wall components [81 and references therein]. It was not surprising to observe considerable amount of upregulation of genes regarding such activities (CL8348.Contig1_All, Unigene29739_All, Unigene20233_All, and CL7042.Contig1_All) in our transcriptome. </p> <p>When we evaluated the common bean transcriptome under salt stress the results suggested that tolerance responses requires highly obvious efforts regarding ionic and osmotic homeostasis through transmembrane transport activities, mobilization and utilization of energy reserves for protection and preservation of structural integrity via metabolite biosynthesis while avoiding energy consumption for growth related transcriptional/translational activities ( <a href="#pone-0092598-g005">Figure 5</a> and <a href="#pone-0092598-g006">6</a>). </p> <p>Several transcription factors play important roles in translating stress signals into changes in gene expression. Based on the BLASTx analysis against the combined Legume TFDB, we have identified a total of 2,678 TF genes, in our common bean transcriptome ( <a href="#pone-0092598-t003">Table 3</a>, and <a href="#pone.0092598.s013">S10</a>). In an earlier report by Kalavacharla <i>et al</i>. (2011), a similar number of TFs (2,516) were also identified. </p> <p>Almost all of the most abundant TF families identified in our study within the DEGs ( <a href="#pone-0092598-t003">Table 3</a> and <a href="#pone.0092598.s013">Table S10</a>) have been well studied and implicated to play a role in both biotic and abiotic stress responses in plants [82 and references therein]. </p> <p>DREB genes are well studied members of AP2-ERF family, suggested to function in drought, salt, heat, and cold stresses by both ABA dependent or ABA-independent signaling pathways [83 and references therein]. Overexpression of DREB genes in diverse species of plants under various promoters have provided improved tolerance in abiotic stresses. </p> <p>Although the biological function of most bHLH family members has not yet been studied in plants <a href="#pone.0092598-Li2">[84]</a>, increase in the bHLH transcription in tolerant wheat variety during salt stress, and regulation of drought tolerance response in Arabidopsis were elucidated by the bHLH regulated ABA signaling <a href="#pone.0092598-Li2">[84]</a>. Increased focus on bHLH genes might shed light on its involvement in salinity stress as observed from overexpressions of bHLH92 in Arabidopsis <a href="#pone.0092598-Jiang2">[85]</a>, and <i>OrbHLH001</i> in wild rice phloem tissues <a href="#pone.0092598-Chen1">[86]</a> conferring improved salt tolerance. </p> <p>After the first sequencing of NAC family gene RD26 <a href="#pone.0092598-YamaguchiShinozaki1">[87]</a> NAC domain has been characterized based on the consensus sequences from <i>NAM, CUC2, and ATAF1/2</i> proteins <a href="#pone.0092598-Aida1">[88]</a>. Since then differential expressions and involvement of several NAC genes in abiotic stress responses have been demonstrated <a href="#pone.0092598-Nakashima1">[89]</a>. </p> <p>As one of the largest TF families, MYB is involved in several physiological and biochemical processes during abiotic stress responses <a href="#pone.0092598-Dai1">[90]</a>. Among the members, <i>TaMYB33</i> TF, which shows high similarity to R2R3-MYB proteins in rice and maize, <a href="#pone.0092598-LiChao1">[91]</a> has been indicated in salt stress tolerance via ROS detoxification and osmotic balance reconstruction in wheat <a href="#pone.0092598-Qin1">[92]</a>. </p> <p>The regulatory role of WRKY family was also demonstrated in high salinity responses, Arabidopsis <i>WRKY8</i> was predominantly expressed in roots and functions of it was consistent with the changes in Na <sup>+</sup>/K <sup>+</sup> concentrations <a href="#pone.0092598-Chen1">[86]</a>. </p> <p>Several members of PHD finger protein family were observed to respond to abiotic stresses differentially. Especially soybean <i>PHD2</i> was shown to be uniquely expressed in tolerant variety and suspected to provide tolerance by diminishing the oxidative stress through regulation of downstream genes <a href="#pone.0092598-Wei1">[93]</a>. Functional relevance of PHD family as chromatin-mediated transcriptional mediator was suggested through their involvement in activation or repression of <i>ING1</i> <a href="#pone.0092598-Skowyra1">[94]</a>, <i>Pf1</i> <a href="#pone.0092598-Yochum1">[95]</a>, <i>TIF1</i> <a href="#pone.0092598-Venturini1">[96]</a>, and <i>KAP1</i> <a href="#pone.0092598-Schultz1">[97]</a> genes. </p> <p>The Homeobox-zip (HB family) TF members were suggested as excellent candidates to generate stress responses in transgenic plants and cotton root development regulation under salt stress involved <i>GmHB1</i> gene expression <a href="#pone.0092598-Ni1">[98]</a>, <a href="#pone.0092598-Zahur1">[99]</a>. </p> <p>Certain members of C3H-type family TFs ( <i>AtSZF1</i> and <i>AtSZF2</i>) were shown to be involved in salt responses transiently within the first few minutes of salt exposure <a href="#pone.0092598-Sun1">[100]</a>, however in our study the C3H-type TFs were not abundant in salt responsive genes ( <a href="#pone-0092598-t003">Table 3</a>), most probably due to the differences in sample collection time. </p> <p>C2C2-Dof TFs are involved in the photosynthetic gene expression of plants and Dof2-domains play a role as tissue-specific repressors in PEP carboxylase promoters, which suggest their regulatory activities in photosynthesis <a href="#pone.0092598-Saibo1">[101]</a>. The ERF-domain of C2H2-Zn TFs were also shown to be important transcriptional repressors in responses to abiotic stresses, a key role of <i>Zat7</i> protein in defense response during salinity was indicated in Arabidopsis <a href="#pone.0092598-CiftciYilmaz1">[102]</a>. </p> <p>A vast number of HSF family members of TFs were compiled from nine species; major aspects of functions implied their role in gene expression of chaperons for stability, localization of cellular components as well as in regulation of abiotic stress responses <a href="#pone.0092598-Scharf1">[103]</a>. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Our comprehensive transcriptome analysis of a salinity tolerant common bean variety by Illumina sequencing is the first transcriptomic study on the response of common bean under salt stress. The identified all-unigenes have been observed to be involved in similar pathways as in previous reports of legumes according to the GO, KEGG and COG analyses. Substantial upregulation of transmembrane transport activities indicates the efforts of common bean to maintain ionic and osmotic homeostasis. To do so, energy consumption seems to be shifted from growth related transcriptional/translational activities towards maintaining current structural integrity through metabolite biosynthesis. Analysis of TFs among DEGs has implicated well studied TF families with known roles in abiotic and biotic stress as well as those that were not strongly associated with salinity stress previously, such as bHLH family. Although many of the DEGs identified has been annotated in publicly available databases, there were DEGs with dramatic expression differences that has not been annotated or implicated in abiotic stress responses, which are awaiting functional characterization. Overall, transcription profiling and identification of DEGs have provided valuable information for salinity tolerance mechanisms that is indispensable to maximize yield and utilization of arid lands. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p> <b>Random distribution of clean reads</b>. The x-axis describes the number of reads and the y-axis indicates the number of clean reads mapped to relative positions in unigenes for the six subtranscriptomes. The orientations of the genes are in 5′ to 3′direction and the gene lengths are normalized. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0092598.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p> <b>GO annotations of all-unigenes.</b> The annotations were performed with Blast2GO software. The length of each bar indicated the percentage of all unigenes falls under each GO terms. The x-axis is in logarithmic scale. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0092598.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p> <b>COG function prediction of all-unigenes.</b> The possible functions of all-unigenes were predicted by alignment to COG database. Each letter in the x-axis represented the COG categories listed on the right of the graph. </p> <p>(TIFF)</p>  <div class="media" title="media"> <a href="pone.0092598.s003.tiff">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p> <b>List of selected all-unigenes for comparison of qRT-PCR and RNA-seq analysis.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s004.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p> <b>List of primers to amplify the selected all-unigenes for qRT-PCR analysis.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s005.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S3</span>  <p> <b>KEGG pathway annotations for all-unigenes.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s006.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S4</span>  <p> <b>Differentially expressed unigenes in leaf tissues.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s007.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S5</span>  <p> <b>Differentially expressed unigenes in root tissues.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s008.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S6</span>  <p> <b>Gene Ontology funtional enrichment analysis of leaf DEGs.</b> (FDR-value &amp;lt;0.05). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s009.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S7</span>  <p> <b>Gene Ontology functional enrichment analysis of root DEGs.</b> (FDR-value &amp;lt;0.05). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s010.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S8</span>  <p> <b>Pathway enrichment analysis of leaf DEGs.</b> (Adjusted- <i>P</i>-value &amp;lt;0.05). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s011.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S9</span>  <p> <b>Pathway enrichment analysis of root DEGs.</b> (Adjusted- <i>P</i>-value &amp;lt;0.05). </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s012.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S10</span>  <p> <b>The number of genes in TF families.</b> </p> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0092598.s013.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Prof. H. Yildiz Dasgan from Cukurova University Department of Horticulture, Adana, Turkey for Ispir seed supply. We are also grateful to Prof. Kuyas Bugra for assistance with manuscript editing and Dr. Ersen Kavak for his guidance in the statistical analysis. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0092598-Munns1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Munns</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tester</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Mechanisms of salinity tolerance</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">651</span>– <span class="lpage" tagx="lpage" title="lpage">681</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18444910">18444910</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Mahajan1" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahajan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuteja</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Cold, salinity and drought stresses: an overview</span>. <span class="source" tagx="source" title="source">Arch Biochem Biophys</span> <span class="volume" tagx="volume" title="volume">444</span>: <span class="fpage" tagx="fpage" title="fpage">139</span>– <span class="lpage" tagx="lpage" title="lpage">158</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16309626">16309626</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Hide1" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hide JC (1954) Diagnosis and improvement of saline and alkali soils. Washington, D.C.: U.S. Dept. of Agriculture. 160 p. </span> </li> <li tag="ref"> <a name="pone.0092598-Broughton1" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broughton</span> <span class="given-names" tagx="given-names" title="given-names">WJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernandez</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blair</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beebe</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gepts</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Beans ( <i>Phaseolus</i> spp.) - model food legumes </span>. <span class="source" tagx="source" title="source">Plant and Soil</span> <span class="volume" tagx="volume" title="volume">252</span>: <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">128</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Madakbas1" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madakbas</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellialtioglu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Taze fasulyede dayaniklilik islahinin kavrami, mekanizmasi ve kalitimi</span>. <span class="source" tagx="source" title="source">Hasad Bitkisel Uretim</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">80</span>– <span class="lpage" tagx="lpage" title="lpage">86</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Chinnusamy1" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chinnusamy</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jagendorf</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Understanding and improving salt tolerance in plants</span>. <span class="source" tagx="source" title="source">Crop Sci</span> <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">437</span>– <span class="lpage" tagx="lpage" title="lpage">448</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Touchette1" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Touchette</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhodes</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poole</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Salt spray induces osmotic adjustment and tissue rigidity in smooth cordgrass, <i>Spartina alterniflora</i> (Loisel.) </span>. <span class="source" tagx="source" title="source">Estuar Coast</span> <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">917</span>– <span class="lpage" tagx="lpage" title="lpage">925</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Huang1" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lew</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hrywna</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Farnesoid X receptor activates transcription of the phospholipid pump <i>MDR3</i> </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">278</span>: <span class="fpage" tagx="fpage" title="fpage">51085</span>– <span class="lpage" tagx="lpage" title="lpage">51090</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14527955">14527955</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Magnan1" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Magnan</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ranty</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charpenteau</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sotta</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galaud</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Mutations in <i>AtCML9</i>, a calmodulin-like protein from <i>Arabidopsis thaliana</i>, alter plant responses to abiotic stress and abscisic acid </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">575</span>– <span class="lpage" tagx="lpage" title="lpage">589</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18643966">18643966</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Wang1" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wan</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Ethylene and nitric oxide are involved in maintaining ion homeostasis in Arabidopsis callus under salt stress </span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">230</span>: <span class="fpage" tagx="fpage" title="fpage">293</span>– <span class="lpage" tagx="lpage" title="lpage">307</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19455351">19455351</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Xu1" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Wheat cryptochromes: subcellular localization and involvement in photomorphogenesis and osmotic stress responses </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">149</span>: <span class="fpage" tagx="fpage" title="fpage">760</span>– <span class="lpage" tagx="lpage" title="lpage">774</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19052154">19052154</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Silva1" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silva</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geros</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Regulation by salt of vacuolar H <sup>+</sup>-ATPase and H <sup>+</sup>-pyrophosphatase activities and Na <sup>+</sup>/H <sup>+</sup> exchange </span>. <span class="source" tagx="source" title="source">Plant Signal Behav</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">718</span>– <span class="lpage" tagx="lpage" title="lpage">726</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19820346">19820346</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Seki1" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narusaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nanjo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">292</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12164808">12164808</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Wang2" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyazaki</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawai</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deyholos</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galbraith</span> <span class="given-names" tagx="given-names" title="given-names">DW</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Temporal progression of gene expression responses to salt shock in maize roots</span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">873</span>– <span class="lpage" tagx="lpage" title="lpage">891</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13677474">13677474</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Rabbani1" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rabbani</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abe</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katsura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Monitoring expression profiles of rice genes under cold, drought, and high-salinity stresses and abscisic acid application using cDNA microarray and RNA gel-blot analyses </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">1755</span>– <span class="lpage" tagx="lpage" title="lpage">1767</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14645724">14645724</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Jiang1" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deyholos</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Comprehensive transcriptional profiling of NaCl-stressed Arabidopsis roots reveals novel classes of responsive genes </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">25</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17038189">17038189</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Walia1" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walia</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ismail</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Condamine</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Genome-wide transcriptional analysis of salinity stressed japonica and indica rice genotypes during panicle initiation stage </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">63</span>: <span class="fpage" tagx="fpage" title="fpage">609</span>– <span class="lpage" tagx="lpage" title="lpage">623</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17160619">17160619</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Qing1" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qing</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">HF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">HT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Comparative profiles of gene expression in leaves and roots of maize seedlings under conditions of salt stress and the removal of salt stress </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">50</span>: <span class="fpage" tagx="fpage" title="fpage">889</span>– <span class="lpage" tagx="lpage" title="lpage">903</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19264788">19264788</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Ge1" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Global transcriptome profiling of wild soybean ( <i>Glycine soja</i>) roots under NaHCO3 treatment </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">153</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20653984">20653984</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Xu2" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">De novo transcriptome sequencing and comparative analysis of differentially expressed genes in <i>Gossypium aridum</i> under salt stress </span>. <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">525</span>: <span class="fpage" tagx="fpage" title="fpage">26</span>– <span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23651590">23651590</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Postnikova1" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Postnikova</span> <span class="given-names" tagx="given-names" title="given-names">OA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nemchinov</span> <span class="given-names" tagx="given-names" title="given-names">LG</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of the alfalfa root transcriptome in response to salinity stress</span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">1041</span>– <span class="lpage" tagx="lpage" title="lpage">1055</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23592587">23592587</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Libault1" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Libault</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farmer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langley</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">An integrated transcriptome atlas of the crop model <i>Glycine max</i>, and its use in comparative analyses in plants </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">63</span>: <span class="fpage" tagx="fpage" title="fpage">86</span>– <span class="lpage" tagx="lpage" title="lpage">99</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20408999">20408999</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Garg1" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garg</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification </span>. <span class="source" tagx="source" title="source">DNA Res</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">53</span>– <span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129">21217129</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Kalavacharla1" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalavacharla</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thimmapuram</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melmaiee</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Identification and analysis of common bean ( <i>Phaseolus vulgaris</i> L.) transcriptomes by massively parallel pyrosequencing </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">135</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21985325">21985325</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Kaur1" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaur</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cogan</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pembleton</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozuka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savin</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing of lentil based on second-generation technology permits large-scale unigene assembly and SSR marker discovery </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">265</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21609489">21609489</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Moe1" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moe</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chung</span> <span class="given-names" tagx="given-names" title="given-names">J-W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y-I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moon</span> <span class="given-names" tagx="given-names" title="given-names">J-K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ku</span> <span class="given-names" tagx="given-names" title="given-names">J-H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Sequence information on simple sequence repeats and single nucleotide polymorphisms through transcriptome analysis of mungbean </span>. <span class="source" tagx="source" title="source">J Integr Plant Biol</span> <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">63</span>– <span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21205180">21205180</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Franssen1" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franssen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shrestha</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bräutigam</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bornberg-Bauer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Comprehensive transcriptome analysis of the highly complex Pisum sativum genome using next generation sequencing </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">227</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21569327">21569327</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-ParraGonzlez1" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parra-González</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aravena-Abarzúa</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Navarro-Navarro</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Udall</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maughan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Yellow lupin ( <i>Lupinus luteus</i> L.) transcriptome sequencing: molecular marker development and comparative studies </span>. <span class="source" tagx="source" title="source">BMC genomics</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">425</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22920992">22920992</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Zhang1" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterisation of the transcriptome during seed development, and generation of genic-SSR markers in peanut ( <i>Arachis hypogaea</i> L.) </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">90</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22409576">22409576</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Wu1" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matand</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> next-generation sequencing, assembling and annotation of <i>Arachis hypogaea</i> L. Spanish botanical type whole plant transcriptome </span>. <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="volume" tagx="volume" title="volume">126</span>: <span class="fpage" tagx="fpage" title="fpage">1145</span>– <span class="lpage" tagx="lpage" title="lpage">1149</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23338522">23338522</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Hiremath1" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiremath</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farmer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woodward</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kudapa</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Large-scale transcriptome analysis in chickpea ( <i>Cicer arietinum</i> L.), an orphan legume crop of the semi-arid tropics of Asia and Africa </span>. <span class="source" tagx="source" title="source">Plant Biotechnol J</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">922</span>– <span class="lpage" tagx="lpage" title="lpage">931</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21615673">21615673</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Fan1" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">X-D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J-Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">Y-Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Gene expression profiling of soybean leaves and roots under salt, saline-alkali and drought stress by high-throughput Illumina sequencing </span>. <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">512</span>: <span class="fpage" tagx="fpage" title="fpage">392</span>– <span class="lpage" tagx="lpage" title="lpage">402</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23063936">23063936</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Zahaf1" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zahaf</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanchet</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Zelicourt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alunni</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plet</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Comparative transcriptomic analysis of salt adaptation in roots of contrasting <i>Medicago truncatula</i> genotypes </span>. <span class="source" tagx="source" title="source">Mol Plant</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">1068</span>– <span class="lpage" tagx="lpage" title="lpage">1081</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22419822">22419822</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Phytozome1" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Phytozome v9.1 website. Available: <a href="http://www.phytozome.org/commonbean.php">http://www.phytozome.org/commonbean.php</a>. Accessed 3 Feb 2014. </span> </li> <li tag="ref"> <a name="pone.0092598-Sato1" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaneko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asamizu</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Genome structure of the legume, <i>Lotus japonicus</i> </span>. <span class="source" tagx="source" title="source">DNA Res</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">227</span>– <span class="lpage" tagx="lpage" title="lpage">239</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435">18511435</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Schmutz1" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmutz</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlueter</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitros</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the palaeopolyploid soybean</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">463</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913">20075913</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Young1" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debellé</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oldroyd</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geurts</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The Medicago genome provides insight into the evolution of rhizobial symbioses</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">480</span>: <span class="fpage" tagx="fpage" title="fpage">520</span>– <span class="lpage" tagx="lpage" title="lpage">524</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132">22089132</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Varshney1" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varshney</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bharti</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saxena</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Draft genome sequence of pigeonpea ( <i>Cajanus cajan</i>), an orphan legume crop of resource-poor farmers </span>. <span class="source" tagx="source" title="source">Nature biotechnology</span> <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">83</span>– <span class="lpage" tagx="lpage" title="lpage">89</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Varshney2" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varshney</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saxena</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Azam</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Draft genome sequence of chickpea ( <i>Cicer arietinum</i>) provides a resource for trait improvement </span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">240</span>– <span class="lpage" tagx="lpage" title="lpage">246</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23354103">23354103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Dasgan1" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dasgan</span> <span class="given-names" tagx="given-names" title="given-names">HY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koc</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Evaluation of salt tolerance in common bean genotypes by ion regulation and searching for screening parameters </span>. <span class="source" tagx="source" title="source">J Food Agric Environ</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">363</span>– <span class="lpage" tagx="lpage" title="lpage">372</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Shavrukov1" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shavrukov</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Salt stress or salt shock: which genes are we studying?</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">119</span>– <span class="lpage" tagx="lpage" title="lpage">127</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23186621">23186621</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Sanchez1" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lippold</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Redestig</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hannah</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erban</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Integrative functional genomics of salt acclimatization in the model legume <i>Lotus japonicus</i> </span>. <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">973</span>– <span class="lpage" tagx="lpage" title="lpage">987</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18047558">18047558</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Liu1" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">A comparison of RNA-Seq and high-density exon array for detecting differential gene expression between closely related species </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">578</span>– <span class="lpage" tagx="lpage" title="lpage">588</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20864445">20864445</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Everett1" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Everett</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grau</span> <span class="given-names" tagx="given-names" title="given-names">ED</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seeb</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Short reads and nonmodel species: exploring the complexities of next-generation sequence assembly and SNP discovery in the absence of a reference genome </span>. <span class="source" tagx="source" title="source">Mol Ecol Resour</span> <span class="volume" tagx="volume" title="volume">11</span> Suppl 1 <span class="fpage" tagx="fpage" title="fpage">93</span>– <span class="lpage" tagx="lpage" title="lpage">108</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21429166">21429166</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Grabherr1" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yassour</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levin</span> <span class="given-names" tagx="given-names" title="given-names">JZ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span> <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">644</span>– <span class="lpage" tagx="lpage" title="lpage">652</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Iseli1" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Iseli C, Jongeneel CV, Bucher P (1999) ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences. Proc Int Conf Intell Syst Mol Biol: 138–148. </span> </li> <li tag="ref"> <a name="pone.0092598-Ashburner1" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashburner</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ball</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blake</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Botstein</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</span>. <span class="source" tagx="source" title="source">Nat Genet</span> <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">29</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Conesa1" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conesa</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gotz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Gomez</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terol</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Talon</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3674</span>– <span class="lpage" tagx="lpage" title="lpage">3676</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474">16081474</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Ye1" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">WEGO: a web tool for plotting GO annotations</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">W293</span>– <span class="lpage" tagx="lpage" title="lpage">297</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012">16845012</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Li1" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lam</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yiu</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">SOAP2: an improved ultrafast tool for short read alignment</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1966</span>– <span class="lpage" tagx="lpage" title="lpage">1967</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933">19497933</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Benjamini1" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benjamini</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yekutieli</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">The control of the false discovery rate in multiple testing under dependency</span>. <span class="source" tagx="source" title="source">Annals of Statistics</span> <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">1165</span>– <span class="lpage" tagx="lpage" title="lpage">1188</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Du1" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ling</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">agriGO: a GO analysis toolkit for the agricultural community</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">W64</span>– <span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20435677">20435677</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-AlShahrour1" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Shahrour</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Díaz-Uriarte</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dopazo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">FatiGO: a web tool for finding significant associations of Gene Ontology terms with groups of genes </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">578</span>– <span class="lpage" tagx="lpage" title="lpage">580</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14990455">14990455</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Borges1" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borges</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsai</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caldas</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Validation of reference genes for RT-qPCR normalization in common bean during biotic and abiotic stresses </span>. <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">827</span>– <span class="lpage" tagx="lpage" title="lpage">838</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22193338">22193338</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Mochida1" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mochida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakurai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">In silico analysis of transcription factor repertoire and prediction of stress responsive transcription factors in soybean </span>. <span class="source" tagx="source" title="source">DNA Res</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">353</span>– <span class="lpage" tagx="lpage" title="lpage">369</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19884168">19884168</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Mochida2" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mochida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshida</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakurai</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">LegumeTFDB: an integrative database of <i>Glycine max</i>, <i>Lotus japonicus</i> and <i>Medicago truncatula</i> transcription factors </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">290</span>– <span class="lpage" tagx="lpage" title="lpage">291</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933159">19933159</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Huang2" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Transcriptome characterization and sequencing-based identification of salt-responsive genes in <i>Millettia pinnata</i>, a semi-mangrove plant </span>. <span class="source" tagx="source" title="source">DNA Res</span> <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">195</span>– <span class="lpage" tagx="lpage" title="lpage">207</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22351699">22351699</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Tyler1" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyler</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Phytophthora sojae</i>: root rot pathogen of soybean and model oomycete </span>. <span class="source" tagx="source" title="source">Mol Plant Pathol</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20507474">20507474</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Gulduren1" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gulduren</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elkoca</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Salinity tolerance at germination stage of some bean ( <i>Phaseolus vulgaris</i> L.) genotypes collected from north East Anatolia Region and Coruh Valley </span>. <span class="source" tagx="source" title="source">J of Agricultural Faculty of Ataturk Univ</span> <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">29</span>– <span class="lpage" tagx="lpage" title="lpage">41</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Gama1" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gama</span> <span class="given-names" tagx="given-names" title="given-names">PBS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inanaga</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazawa</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Physiological response of common bean ( <i>Phaseolus vulgaris</i> L.) seedlings to salinity stress </span>. <span class="source" tagx="source" title="source">Afr J Biotechnol</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">88</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Ramirez1" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramirez</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanco-Lopez</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silvente</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Medrano-Soto</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Sequencing and analysis of common bean ESTs. Building a foundation for functional genomics </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">137</span>: <span class="fpage" tagx="fpage" title="fpage">1211</span>– <span class="lpage" tagx="lpage" title="lpage">1227</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15824284">15824284</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Melotto1" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melotto</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monteiro-Vitorello</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruschi</span> <span class="given-names" tagx="given-names" title="given-names">AG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camargo</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Comparative bioinformatic analysis of genes expressed in common bean ( <i>Phaseolus vulgaris</i> L.) seedlings </span>. <span class="source" tagx="source" title="source">Genome</span> <span class="volume" tagx="volume" title="volume">48</span>: <span class="fpage" tagx="fpage" title="fpage">562</span>– <span class="lpage" tagx="lpage" title="lpage">570</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16121253">16121253</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Molina1" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Molina C, Zaman-Allah M, Khan F, Fatnassi N, Horres R, <span class="etal" title="etal"> <i>et al.</i> </span>. (2011) The salt-responsive transcriptome of chickpea roots and nodules via deepSuperSAGE. BMC Plant Biology <span class="volume" tagx="volume" title="volume">11</span>.. </span> </li> <li tag="ref"> <a name="pone.0092598-Liu2" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crampton</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todd</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalavacharla</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Identification of expressed resistance gene-like sequences by data mining in 454-derived transcriptomic sequences of common bean ( <i>Phaseolus vulgaris</i> L.) </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">42</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22443214">22443214</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Liao1" /> <span class="label" tagx="label" title="label">65</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liao</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cram</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharpe</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marsolais</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling identifies candidate genes associated with the accumulation of distinct sulfur γ-glutamyl dipeptides in <i>Phaseolus vulgaris</i> and <i>Vigna mungo</i> seeds </span>. <span class="source" tagx="source" title="source">Front Plant Sci</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">60</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23532826">23532826</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Wang3" /> <span class="label" tagx="label" title="label">66</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sha</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shan</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Transcript profile of the response of two soybean genotypes to potassium deficiency</span>. <span class="source" tagx="source" title="source">PloS ONE</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e39856</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22792192">22792192</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Pang1" /> <span class="label" tagx="label" title="label">67</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">C-Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and transcriptome analysis of the desert shrub, <i>Ammopiptanthus mongolicus</i>, during cold acclimation using Illumina/Solexa </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">488</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23865740">23865740</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Parida1" /> <span class="label" tagx="label" title="label">68</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parida</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Salt tolerance and salinity effects on plants: a review</span>. <span class="source" tagx="source" title="source">Ecotoxicol Environ Saf</span> <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">324</span>– <span class="lpage" tagx="lpage" title="lpage">349</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15590011">15590011</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Carillo1" /> <span class="label" tagx="label" title="label">69</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Carillo P, Annunziata MG, Pontecorvo G, Fuggi A, Woodrow P (2011) Salinity stress and salt tolerance. In: Arun S, editor. Abiotic Stress in Plants - Mechanisms and Adaptations. Croatia: InTech. pp. 22–38. </span> </li> <li tag="ref"> <a name="pone.0092598-Tomoaki1" /> <span class="label" tagx="label" title="label">70</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomoaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ichirou</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Salinity tolerance mechanisms in glycophytes: An overview with the central focus on rice plants </span>. <span class="source" tagx="source" title="source">Rice</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">11</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27234237">27234237</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Faal1" /> <span class="label" tagx="label" title="label">71</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faïçal</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khaled</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Ion transporters and abiotic stress tolerance in plants</span>. <span class="source" tagx="source" title="source">ISRN Mol Biol</span> <span class="volume" tagx="volume" title="volume">2012</span>: <span class="fpage" tagx="fpage" title="fpage">ID927436</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Tester1" /> <span class="label" tagx="label" title="label">72</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tester</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davenport</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Na <sup>+</sup> tolerance and Na <sup>+</sup> transport in higher plants </span>. <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">503</span>– <span class="lpage" tagx="lpage" title="lpage">527</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12646496">12646496</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Guo1" /> <span class="label" tagx="label" title="label">73</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">K-M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Babourina</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christopher</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borsics</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rengel</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">The cyclic nucleotide-gated channel, <i>AtCNGC10</i>, influences salt tolerance in Arabidopsis </span>. <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="volume" tagx="volume" title="volume">134</span>: <span class="fpage" tagx="fpage" title="fpage">499</span>– <span class="lpage" tagx="lpage" title="lpage">507</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18823330">18823330</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Yuen1" /> <span class="label" tagx="label" title="label">74</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yuen CYL, Christopher DA (2010) The role of cyclic nucleotide-gated channels in cation nutrition and abiotic stress. In: Demidchik V, Maathuis F, editors. Ion Channels and Plant Stress Responses: Springer Berlin Heidelberg. pp. 137–157. </span> </li> <li tag="ref"> <a name="pone.0092598-Leng1" /> <span class="label" tagx="label" title="label">75</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leng</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mercier</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">B-G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fromm</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berkowitz</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Electrophysiological analysis of cloned cyclic nucleotide-gated ion channels</span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">128</span>: <span class="fpage" tagx="fpage" title="fpage">400</span>– <span class="lpage" tagx="lpage" title="lpage">410</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11842144">11842144</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Teakle1" /> <span class="label" tagx="label" title="label">76</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teakle</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flowers</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Real</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colmer</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Lotus tenuis</i> tolerates the interactive effects of salinity and waterlogging by ‘excluding’ Na <sup>+</sup> and Cl <sup>−</sup> from the xylem </span>. <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">2169</span>– <span class="lpage" tagx="lpage" title="lpage">2180</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17510213">17510213</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Samir1" /> <span class="label" tagx="label" title="label">77</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samir</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sameh</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chedly</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Growth, nitrogen fixation and ion distribution in <i>Medicago truncatula</i> subjected to salt stress </span>. <span class="source" tagx="source" title="source">Plant Soil</span> <span class="volume" tagx="volume" title="volume">312</span>: <span class="fpage" tagx="fpage" title="fpage">59</span>– <span class="lpage" tagx="lpage" title="lpage">67</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Teakle2" /> <span class="label" tagx="label" title="label">78</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teakle</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyerman</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Mechanisms of Cl <sup>-</sup> transport contributing to salt tolerance </span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span> <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">566</span>– <span class="lpage" tagx="lpage" title="lpage">589</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19895402">19895402</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Jacoby1" /> <span class="label" tagx="label" title="label">79</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacoby</span> <span class="given-names" tagx="given-names" title="given-names">RP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">NL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millar</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The role of mitochondrial respiration in salinity tolerance</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">614</span>– <span class="lpage" tagx="lpage" title="lpage">623</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21903446">21903446</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-MarianaLinsdeOliveira1" /> <span class="label" tagx="label" title="label">80</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mariana Lins de Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bety Shiue de</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">João Antônio de Almeida</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rafaela Moura</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jarcilene Silva de</span> <span class="given-names" tagx="given-names" title="given-names">A-C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Photosynthesis and antioxidant activity in <i>Jatropha curcas</i> L. under salt stress </span>. <span class="source" tagx="source" title="source">Braz J Plant Physiol</span> <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">67</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Ghosh1" /> <span class="label" tagx="label" title="label">81</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Abiotic stress responses in plant roots: a proteomics perspective</span>. <span class="source" tagx="source" title="source">Front Plant Sci</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">6</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24478786">24478786</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Lindemose1" /> <span class="label" tagx="label" title="label">82</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindemose</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Shea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skriver</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Structure, function and networks of transcription factors involved in abiotic stress responses </span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">5842</span>– <span class="lpage" tagx="lpage" title="lpage">5878</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23485989">23485989</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Lata1" /> <span class="label" tagx="label" title="label">83</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lata</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prasad</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Role of DREBs in regulation of abiotic stress responses in plants</span>. <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">4731</span>– <span class="lpage" tagx="lpage" title="lpage">4748</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21737415">21737415</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Li2" /> <span class="label" tagx="label" title="label">84</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The bHLH-type transcription factor <i>AtAIB</i> positively regulates ABA response in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">655</span>– <span class="lpage" tagx="lpage" title="lpage">665</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17828375">17828375</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Jiang2" /> <span class="label" tagx="label" title="label">85</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deyholos</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Functional characterization of the Arabidopsis <i>bHLH92</i> transcription factor in abiotic stress </span>. <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="volume" tagx="volume" title="volume">282</span>: <span class="fpage" tagx="fpage" title="fpage">503</span>– <span class="lpage" tagx="lpage" title="lpage">516</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19760256">19760256</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Chen1" /> <span class="label" tagx="label" title="label">86</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chong</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Overexpression of <i>OrbHLH001</i>, a putative helix-loop-helix transcription factor, causes increased expression of <i>AKT1</i> and maintains ionic balance under salt stress in rice </span>. <span class="source" tagx="source" title="source">J Plant Physiol</span> <span class="volume" tagx="volume" title="volume">170</span>: <span class="fpage" tagx="fpage" title="fpage">93</span>– <span class="lpage" tagx="lpage" title="lpage">100</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23083684">23083684</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-YamaguchiShinozaki1" /> <span class="label" tagx="label" title="label">87</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koizumi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urao</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">1992</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and characterization of 9 cDNAs for genes that are responsive to desiccation in <i>Arabidopsis thaliana</i>: Sequence analysis of one cDNA clone that encodes a putative transmembrane channel protein </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">217</span>– <span class="lpage" tagx="lpage" title="lpage">224</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Aida1" /> <span class="label" tagx="label" title="label">88</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujisawa</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tasaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> ( <span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">Genes involved in organ separation in Arabidopsis: an analysis of the cup-shaped cotyledon mutant </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">841</span>– <span class="lpage" tagx="lpage" title="lpage">857</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9212461">9212461</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Nakashima1" /> <span class="label" tagx="label" title="label">89</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takasaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizoi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors in plant abiotic stress responses</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="volume" tagx="volume" title="volume">1819</span>: <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">103</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22037288">22037288</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Dai1" /> <span class="label" tagx="label" title="label">90</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Overexpression of an R1R2R3 MYB gene, <i>OsMYB3R-2</i>, increases tolerance to freezing, drought, and salt stress in transgenic Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">143</span>: <span class="fpage" tagx="fpage" title="fpage">1739</span>– <span class="lpage" tagx="lpage" title="lpage">1751</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17293435">17293435</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-LiChao1" /> <span class="label" tagx="label" title="label">91</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li-Chao</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guang-Yao</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji-Zeng</span> <span class="given-names" tagx="given-names" title="given-names">JIA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiu-Ying</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Cloning and Analysis of Salt Stress-Related Gene <i>TaMYB32</i> in Wheat </span>. <span class="source" tagx="source" title="source">Acta Agron Sin</span> <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">1181</span>– <span class="lpage" tagx="lpage" title="lpage">1187</span>. </span> </li> <li tag="ref"> <a name="pone.0092598-Qin1" /> <span class="label" tagx="label" title="label">92</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Over-expression of <i>TaMYB33</i> encoding a novel wheat MYB transcription factor increases salt and drought tolerance in Arabidopsis </span>. <span class="source" tagx="source" title="source">Mol Biol Rep</span> <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">7183</span>– <span class="lpage" tagx="lpage" title="lpage">7192</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22350156">22350156</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Wei1" /> <span class="label" tagx="label" title="label">93</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">HF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">HW</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Soybean GmPHD-type transcription regulators improve stress tolerance in transgenic Arabidopsis plants </span>. <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e7209</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19789627">19789627</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Skowyra1" /> <span class="label" tagx="label" title="label">94</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skowyra</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeremski</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neznanov</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Differential association of products of alternative transcripts of the candidate tumor suppressor <i>ING1</i> with the <i>mSin3/HDAC1</i> transcriptional corepressor complex </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">276</span>: <span class="fpage" tagx="fpage" title="fpage">8734</span>– <span class="lpage" tagx="lpage" title="lpage">8739</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11118440">11118440</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Yochum1" /> <span class="label" tagx="label" title="label">95</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yochum</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ayer</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Pf1</i>, a novel PHD zinc finger protein that links the TLE corepressor to the mSin3A-histone deacetylase complex </span>. <span class="source" tagx="source" title="source">Mol Cell Biol</span> <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">4110</span>– <span class="lpage" tagx="lpage" title="lpage">4118</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11390640">11390640</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Venturini1" /> <span class="label" tagx="label" title="label">96</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venturini</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">You</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stadler</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galien</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lallemand</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>TIF1gamma</i>, a novel member of the transcriptional intermediary factor 1 family </span>. <span class="source" tagx="source" title="source">Oncogene</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">1209</span>– <span class="lpage" tagx="lpage" title="lpage">1217</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10022127">10022127</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Schultz1" /> <span class="label" tagx="label" title="label">97</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friedman</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rauscher</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Targeting histone deacetylase complexes via KRAB-zinc finger proteins: the PHD and bromodomains of KAP-1 form a cooperative unit that recruits a novel isoform of the Mi-2alpha subunit of <i>NuRD</i> </span>. <span class="source" tagx="source" title="source">Genes Dev</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">428</span>– <span class="lpage" tagx="lpage" title="lpage">443</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11230151">11230151</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Ni1" /> <span class="label" tagx="label" title="label">98</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Novel cotton homeobox gene and its expression profiling in root development and in response to stresses and phytohormones </span>. <span class="source" tagx="source" title="source">Acta Biochim Biophys Sin (Shanghai)</span> <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">78</span>– <span class="lpage" tagx="lpage" title="lpage">84</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18180856">18180856</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Zahur1" /> <span class="label" tagx="label" title="label">99</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zahur</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asif</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeeshan</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mehmood</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malik</span> <span class="given-names" tagx="given-names" title="given-names">MF</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Homeobox leucine zipper proteins and cotton improvement</span>. <span class="source" tagx="source" title="source">Adv Biosci Biotechnol</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">15</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24558636">24558636</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Sun1" /> <span class="label" tagx="label" title="label">100</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The CCCH-type zinc finger proteins <i>AtSZF1</i> and <i>AtSZF2</i> regulate salt stress responses in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="volume" tagx="volume" title="volume">48</span>: <span class="fpage" tagx="fpage" title="fpage">1148</span>– <span class="lpage" tagx="lpage" title="lpage">1158</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17609218">17609218</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Saibo1" /> <span class="label" tagx="label" title="label">101</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saibo</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lourenco</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliveira</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Transcription factors and regulation of photosynthetic and related metabolism under environmental stresses </span>. <span class="source" tagx="source" title="source">Ann Bot</span> <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">609</span>– <span class="lpage" tagx="lpage" title="lpage">623</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19010801">19010801</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-CiftciYilmaz1" /> <span class="label" tagx="label" title="label">102</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ciftci-Yilmaz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morsy</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coutu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krizek</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The EAR-motif of the Cys2/His2-type zinc finger protein Zat7 plays a key role in the defense response of Arabidopsis to salinity stress </span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">282</span>: <span class="fpage" tagx="fpage" title="fpage">9260</span>– <span class="lpage" tagx="lpage" title="lpage">9268</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17259181">17259181</a> </span> </span> </li> <li tag="ref"> <a name="pone.0092598-Scharf1" /> <span class="label" tagx="label" title="label">103</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scharf</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berberich</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ebersberger</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nover</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">The plant heat stress transcription factor (Hsf) family: structure, function and evolution</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="volume" tagx="volume" title="volume">1819</span>: <span class="fpage" tagx="fpage" title="fpage">104</span>– <span class="lpage" tagx="lpage" title="lpage">119</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22033015">22033015</a> </span> </span> </li> </ul> </div> </div>  </body></html>