<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5492850">5492850</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2017.01156">10.3389/fpls.2017.01156</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Divergent Evolutionary Patterns of NAC Transcription Factors Are Associated with Diversification and Gene Duplications in Angiosperm </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">Xiaoli</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#fn003"> <sup>†</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/382752/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">Jing</span> </span> <a href="#aff2"> <sup>2</sup> </a> <a href="#fn003"> <sup>†</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/412821/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nevo</span> <span class="given-names" tagx="given-names" title="given-names">Eviatar</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">Xuegui</span> </span> <a href="#aff4"> <sup>4</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/447187/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Dongfa</span> </span> <a href="#aff5"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">Junhua</span> </span> <a href="#aff4"> <sup>4</sup> </a> <a href="#aff6"> <sup>6</sup> </a> <a href="#fn001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/86020/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department of Agronomy and the Key Laboratory of Crop Germplasm Resource of Zhejiang Province, Zhejiang University </span> <span class="country" tagx="country" title="country">Hangzhou, China</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">Shandong Provincial Key Laboratory of Biophysics, Institute of Biophysics, Dezhou University </span> <span class="country" tagx="country" title="country">Dezhou, China</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">Department of Evolutionary and Environmental Biology, Institute of Evolution, University of Haifa </span> <span class="country" tagx="country" title="country">Haifa, Israel</span> </span> <span class="citation_author_institution" id="aff4">[], <sup>4</sup> <span class="institution" title="institution">Department of Biotechnology, College of Agriculture, Guangdong Ocean University</span> <span class="country" tagx="country" title="country">Zhanjiang, China</span> </span> <span class="citation_author_institution" id="aff5">[], <sup>5</sup> <span class="institution" title="institution">Department of Agronomy, College of Plant Science and Technology, Huazhong Agricultural University </span> <span class="country" tagx="country" title="country">Wuhan, China</span> </span> <span class="citation_author_institution" id="aff6">[], <sup>6</sup> <span class="institution" title="institution">Life Science &amp;amp; Technology Center, and the State Key Lab of Crop Breeding Technology Innovation and Integration, China National Seed Group Co., Ltd. </span> <span class="country" tagx="country" title="country">Wuhan, China</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Michael Eric Schranz, Wageningen University and Research, Netherlands</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Turgay Unver, Izmir International Biomedicine and Genome Institute (iBG-izmir), Turkey; Kui Lin, Beijing Normal University, China </p> </div> <div class="corresp" title="corresp">
*Correspondence: Junhua Peng <span class="email" tagx="email" title="email">junhuapeng@yahoo.com</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Plant Genetics and Genomics, a section of the journal Frontiers in Plant Science </p> </div> <div class="fn-type-other" title="other"> <p>†These authors have contributed equally to this work.</p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-6-6</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">1156</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-3-08</span> <span class="accepted" title="accepted">accepted: 2017-6-15</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Jin, Ren, Nevo, Yin, Sun and Peng</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>NAC (NAM/ATAF/CUC) proteins constitute one of the biggest plant-specific transcription factor (TF) families and have crucial roles in diverse developmental programs during plant growth. Phylogenetic analyses have revealed both conserved and lineage-specific NAC subfamilies, among which various origins and distinct features were observed. It is reasonable to hypothesize that there should be divergent evolutionary patterns of NAC TFs both between dicots and monocots, and among NAC subfamilies. In this study, we compared the gene duplication and loss, evolutionary rate, and selective pattern among non-lineage specific NAC subfamilies, as well as those between dicots and monocots, through genome-wide analyses of sequence and functional data in six dicot and five grass lineages. The number of genes gained in the dicot lineages was much larger than that in the grass lineages, while fewer gene losses were observed in the grass than that in the dicots. We revealed (1) uneven constitution of Clusters of Orthologous Groups (COGs) and contrasting birth/death rates among subfamilies, and (2) two distinct evolutionary scenarios of NAC TFs between dicots and grasses. Our results demonstrated that relaxed selection, resulting from concerted gene duplications, may have permitted substitutions responsible for functional divergence of NAC genes into new lineages. The underlying mechanism of distinct evolutionary fates of NAC TFs shed lights on how evolutionary divergence contributes to differences in establishing NAC gene subfamilies and thus impacts the distinct features between dicots and grasses. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
NAC transcription factors</div> <div class="kwd" title="kwd">
gene duplication</div> <div class="kwd" title="kwd">
evolution rate</div> <div class="kwd" title="kwd">
divergent selection</div> <div class="kwd" title="kwd">
angiosperm</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
National Natural Science Foundation of China <span class="named-content" title="named-content">10.13039/501100001809</span> </div> <span class="award-id" tagx="award-id" title="award-id">31030055</span> <span class="award-id" tagx="award-id" title="award-id">30870233</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Chinese Academy of Sciences <span class="named-content" title="named-content">10.13039/501100002367</span> </div> <span class="award-id" tagx="award-id" title="award-id">KSCX2-YW-Z-0722</span> <span class="award-id" tagx="award-id" title="award-id">XDA05130403</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Gene duplication plays significant roles in evolution of organisms. Polyploidization, also known as whole-genome duplication (WGD), is now recognized as an important evolutionary force in the origin and diversification of plants (De Bodt et al., <a href="#B11">2005</a>; Freeling and Thomas, <a href="#B15">2006</a>; Soltis et al., <a href="#B57">2009</a>; Sarilar et al., <a href="#B49">2013</a>). Most of the flowering plant lineages had experienced one or more rounds of ancient WGD through the evolutionary history (Cui et al., <a href="#B8">2006</a>; Jiao et al., <a href="#B25">2011</a>). WGD results in genetic redundancy, which is acted upon by selection, drift, and mutation, and represents with the emergence of new gene functions (Ohno, <a href="#B39">1970</a>; Zhang, <a href="#B79">2003</a>; Sarilar et al., <a href="#B49">2013</a>). Indeed, gene duplication is an important source of novelty, especially for gene family diversification. Paralogous genes derived from gene duplication have identical sequences and functions but tend to diverge in both regulating and coding regions. The genes retained in duplicates are not evenly distributed among different functional categories. For example, three evident ancient polyploidization events in the <i>Arabidopsis thaliana</i> genome account for over 90% of the duplications in transcription factors (TFs) and signal transducer gene families (Maere et al., <a href="#B33">2005</a>). Therefore, genes relating with transcription regulation, signal transduction and development are preferentially maintained following genome duplications (Seoighe and Gehring, <a href="#B54">2004</a>; De Bodt et al., <a href="#B11">2005</a>; Maere et al., <a href="#B33">2005</a>). </p> <p>TFs and their corresponding <i>cis</i>-regulatory sequences act as molecular switches for gene expression, regulating their temporal and spatial expression. A large portion of the plant genome is dedicated to TFs, with up to 2,500 TFs being coded into 64 gene families in the <i>Arabidopsis</i> genome (Perez-Rodriguez et al., <a href="#B44">2010</a>). In plants, TFs often belong to large gene families, such as the NAC (NAM, ATAF1, 2, and CUC2) TF gene family (Zhao et al., <a href="#B81">2016</a>). The NAC acronym was derived from three genes that were initially discovered to contain highly conserved NAC domains: <i>NAM</i> ( <i>no apical meristem</i>), <i>ATAF1</i> and <i>-2</i>, and <i>CUC2</i> ( <i>cupshaped cotyledon</i>) (Souer et al., <a href="#B58">1996</a>; Aida et al., <a href="#B2">1997</a>; Zhao et al., <a href="#B81">2016</a>; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). Compared to the NAC domains, the C-terminal regions of NAC TFs are rather variable (Ooka et al., <a href="#B41">2003</a>; Zhu et al., <a href="#B86">2012</a>; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) and result in the diversity of the transcriptional activities showed by NAC proteins (Xie et al., <a href="#B71">2000</a>; Yamaguchi et al., <a href="#B73">2008</a>; Jensen et al., <a href="#B24">2010</a>; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). The NAC proteins form one of the largest families of plant-specific TFs, with &amp;lt;100 members discovered in <i>Arabidopsis</i>, rice, and other major angiosperm lineages, which probably expanded via WGD and other gene duplication events (Zhu et al., <a href="#B86">2012</a>). NAC TFs are crucial for plant growth and are associated with diverse biological programs, including various developmental processes (Olsen et al., <a href="#B40">2005</a>), senescence (Uauy et al., <a href="#B64">2006</a>; Yang et al., <a href="#B74">2011</a>), secondary walls or wood formation (Zhong et al., <a href="#B83">2010b</a>), and biotic (Christianson et al., <a href="#B7">2010</a>; Zhu et al., <a href="#B85">2014</a>) and abiotic stress responses (Tran et al., <a href="#B62">2010</a>; Zhu et al., <a href="#B85">2014</a>). </p> <p>There are less than 100 members of NAC TFs s discovered in <i>Arabidopsis</i>, rice, poplar, and other major angiosperm lineages (Ooka et al., <a href="#B41">2003</a>; Zhu et al., <a href="#B86">2012</a>; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), whereas only approximate 30 NAC TFs are present in early land plants (e.g., <i>Selaginella moellendorffii</i>) (Zhu et al., <a href="#B86">2012</a>). Such extensive expansions were presumably mainly obtained via WGD and other small-scale duplication events (Zhu et al., <a href="#B86">2012</a>). Phylogenetic analyses of the NAC TFs of a diverse set of land plants revealed clearly-classified subfamilies (Ooka et al., <a href="#B41">2003</a>; Zhu et al., <a href="#B86">2012</a>). Among these NAC subfamilies, some share the evolutionary history dated before the divergence of dicots and monocots (termed non-lineage-specific subfamilies; e.g., Ia, IIIc, and Vb subfamilies), while some contain exclusively either dicot members or monocot members (termed dicot/monocot-specific subfamilies; e.g., VIa, VIb, and X subfamilies) (Zhu et al., <a href="#B86">2012</a>). As described in great detail by Raven and Johnson ( <a href="#B48">2002</a>), angiosperms were classified into monocots and dicots, between which the differences in morphological and physiological features were observed. The existence of lineage specific subfamilies suggested some various origins of NAC subfamilies such that they should harbor distinct evolutionary features through diverse evolutionary events (e.g., polyploidy) specific to either of the lineages. In addition, albeit the extensive functional diversification, NAC subfamilies could be classified into two major categories according to their functions, the relatively conserved processes and the relatively specific processes (Zhu et al., <a href="#B86">2012</a>). Thus, we would expect underlying factors that drive such functional divergence. </p> <p>Taking all these observations together, it is reasonable for us to come up with the hypothesis that there should be divergent evolutionary patterns of NAC TFs both between dicots and monocots, and among NAC subfamilies as well. And the underlying mechanism should shed lights on how evolutionary divergence contributes to differences in establishing NAC gene subfamilies and thus impacts the distinct morphological and physiological features between dicots and grasses. However, few comprehensive studies have been conducted to prove this point of view. Therefore, to clarify this speculation and to present a picture on the possible evolutionary fates of NAC TFs, and more importantly to address the question of whether NAC genes with identical function evolve through specific mechanisms, we compared gene duplication and loss, evolutionary rate, and selective pattern among NAC subfamilies, as well as that between dicots and monocots. Using six dicot and five grass genomes representing diverse lineages of the flowering plants, evidence supporting our hypothesis was revealed and some subsequent deductions were made. </p> </div> <div class="materials and methods" title="materials and methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="dataretrievalandconstructionofcogs" title="sec"> <div class="title" tagx="title" title="title">
Data retrieval and construction of COGs</div> <p>Protein-coding transcripts of <i>A. thaliana</i> (release TAIR10, The Arabidopsis Genome Initiative), <i>Brassica rapa</i> (release v1.1) (Wang et al., <a href="#B68">2011</a>), <i>Glycine max</i> (release Glyma1.1, Soybean Genome Project, DoE Joint Genome Institute; Li et al., <a href="#B31">2012</a>), <i>Medicago truncatula</i> (release Mt3.5v4, <i>M. truncatula</i> genome sequencing project; Li et al., <a href="#B31">2012</a>), <i>Populus trichocarpa</i> (release Poptr JGIv3.0, The Joint Genome Institute; Li et al., <a href="#B31">2012</a>), <i>Ricinus communis</i> (release v0.1) (Chan et al., <a href="#B6">2010</a>; Li et al., <a href="#B31">2012</a>), <i>Brachypodium distachyon</i> (release Brachy1.2, International Brachypodium Initiative; Li et al., <a href="#B31">2012</a>), <i>Oryza sativa</i> (MSU Release 7.0) (Ouyang et al., <a href="#B42">2007</a>; Li et al., <a href="#B31">2012</a>), <i>Setaria italica</i> (release v2.1, DoE Joint Genome Institute), <i>Sorghum bicolor</i> (release v1.0) (Paterson et al., <a href="#B43">2009</a>; Li et al., <a href="#B31">2012</a>), and <i>Zea mays</i> (Release B73 RefGen_v2, Maize Genome Project) were downloaded from JGI ( <a href="http://www.phytozome.net/">http://www.phytozome.net/</a>) or EnsemblePlants ( <a href="http://plants.ensembl.org/index.html">http://plants.ensembl.org/index.html</a>) to construct a local BLAST database by using BLAST v2.2.27. </p> <p>Multiple query NAC sequences from different lineages were used for each subfamily to obtain comprehensive samples through BLAST searches of our local database. Sequences collected from the same species with DNA identity&amp;gt; 95% and without indels were regarded as possible alleles (Zhang et al., <a href="#B80">2001</a>), thus were removed from the resulting data sets. Sequences shorter than 400 bp, as well as sequences with many ambiguous base calls were excluded from further analyses. All the non-redundant putative NAC protein sequences were manually checked for the NAC domain through CDD ( <a href="http://www.ncbi.nlm.nih.gov/sites/cdd">http://www.ncbi.nlm.nih.gov/sites/cdd</a>; Zhu et al., <a href="#B86">2012</a>), with PFAM NAM domain (PF02365) as guidance. </p> <p>Clusters of Orthologous Groups (COGs; Li et al., <a href="#B31">2012</a>) were generated following the protocol of Li et al. ( <a href="#B31">2012</a>), which was modified from Tatusov et al. ( <a href="#B60">2000</a>) and could be used to detect orthologs among the slowly and rapidly evolving genes. The COGs reflect one-to-many and many-to-many orthologous relationships as well as simple one-to-one relationships (Tatusov et al., <a href="#B60">2000</a>). </p> </div> <div class="sequencealignmentandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence alignment and phylogenetic analysis</div> <p>Protein sequences for each COG were first aligned with ClustalX v2.1 (Larkin et al., <a href="#B30">2007</a>) and extensively adjusted manually with BioEdit ( <a href="http://www.mbio.ncsu.edu/BioEdit/BioEdit.html">http://www.mbio.ncsu.edu/BioEdit/BioEdit.html</a>) when necessary. In order to objectively assess alignment quality, the column scores of each amino acid site were estimated in ClustalX v2.1 (Larkin et al., <a href="#B30">2007</a>) and sites with a column score greater than 12 were retained for tree reconstruction (Shan et al., <a href="#B56">2007</a>). Phylogenetic tree reconstruction was performed with both Maximum Likelihood (ML) and Neighbor Joining (NJ) approaches using the amino acid sequences. The ML analysis was conducted with the program PhyML version 3.0.1 (Guindon and Gascuel, <a href="#B18">2003</a>), using the LG model recommended by ProtTest v2.2 (Abascal et al., <a href="#B1">2005</a>), an estimated gamma distribution parameter, and a Shimodaira-Hasegawa-like approximate likelihood ratio test. ML searches were initiated with a BIONJ tree (Guindon and Gascuel, <a href="#B18">2003</a>). The PHYLIP package version 3.69 (Felsenstein, <a href="#B14">1989</a>) was used to perform 1000 bootstrap replicas of a NJ tree based on a JTT distance matrix. </p> </div> <div class="estimationofgeneduplicationandlossevents" title="sec"> <div class="title" tagx="title" title="title">
Estimation of gene duplication and loss events</div> <p>Gene duplication and loss within each subfamily were analyzed mainly by checking each COG manually. Each COG is normally assumed to have at least two subgroups (the dicot and grass subgroup) and at least six and five members from dicot and grass species, respectively (Li et al., <a href="#B31">2012</a>; Figure <a href="#SM3">S3</a>). If two or more dicot or grass subgroups were detected, or if these subgroups have two or more members from the same species, we deduced that one or more duplication events happened. On the other hand, if there is a member missing from any species within the dicot or grass subgroup, it was deduced to be a gene loss event. Every gene loss was further validated by checking with GenBank (Li et al., <a href="#B31">2012</a>). </p> </div> <div class="molecularevolutionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Molecular evolution analysis</div> <p>A codon sequence alignment of each COG was generated from its corresponding amino acid alignment using the software aa2dna ( <a href="http://www.bio.psu.edu/People/Faculty/Nei/Lab/software.htm">http://www.bio.psu.edu/People/Faculty/Nei/Lab/software.htm</a>). The ratio of non-synonymous substitutions per non-synonymous site ( <i>d</i> <sub>N</sub>) to synonymous substitutions per synonymous site ( <i>d</i> <sub>S</sub>) (ω value) of homologous gene pairs (Li et al., <a href="#B31">2012</a>) was calculated using the likelihood method in Codeml from the PAML package v4.6 (Yang, <a href="#B76">2007</a>). Saturation effects were avoided by discarding the gene pairs for which <i>d</i> <sub>S</sub> &amp;gt; 2 (Ramsay et al., <a href="#B46">2009</a>; Li et al., <a href="#B31">2012</a>). </p> <p>The branch models embedded in Codeml could reveal the ratio variation among branches in the phylogeny and detect positive selection acting on particular lineages (Yang, <a href="#B75">1998</a>; Yang and Nielsen, <a href="#B77">2002</a>), and thus were used to test variation of the ω ratio among different branches in gene trees (Wang et al., <a href="#B66">2015</a>) and were compared with the one-ratio model that assumes a constant ω value across all branches using the likelihood ratio test (LRT) (Li et al., <a href="#B31">2012</a>). In these processes, truncated sequences were removed and tree topologies were manually adjusted (Li et al., <a href="#B31">2012</a>). </p> <p>In order to reveal the molecular evolutionary patterns of the COGs within NAC subfamilies, we further conducted likelihood analysis on the same data sets used in <i>d</i> <sub>N</sub>/ <i>d</i> <sub>S</sub> ratio estimation previously under a nested set of codon-substitution models with Fitmodel v0.5.3 (Guindon et al., <a href="#B19">2004</a>), which allows changes between selection patterns to occur through time. As did in Shan et al. ( <a href="#B55">2009</a>), we obtained the trees and alignments used in the Fitmodel analyses, and the ML estimates for other parameters including branch length, substitution rate ratios (ω <sub>1</sub>, ω <sub>2</sub>, and ω <sub>3</sub>), and equilibrium frequencies for sites in the three rate ratio classes ( <i>p</i> <sub>1</sub>, <i>p</i> <sub>2</sub>, and <i>p</i> <sub>3</sub>). Comparison between models of rate heterogeneity variation across sites without vs. with switching among substitution rate ratio (M3 vs. M3 + S1) was conducted, and the chi-squared test was employed to estimate the significance of difference. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="homologsfrommostofthenacsubfamiliesaredividedintomorethanonecogs" title="sec"> <div class="title" tagx="title" title="title">
Homologs from most of the NAC subfamilies are divided into more than one COGs</div> <p>Our previous analysis based on 837 NAC sequences of land plants demonstrates a 21-subfamily configuration of NAC proteins, some of which are dicot-/monocot-specific (Zhu et al., <a href="#B86">2012</a>). To investigate the divergent evolutionary patterns of NAC genes between dicots and grasses, we focused on those subfamilies that contain both dicot and grass members in the present study. In these 13 non-lineage-specific NAC subfamilies (Ia, Ib, Ic, II, IIIa, IIIb, IIIc, IVb, IVc, IVd, Va, Vb, and VIc subfamilies), 31 COGs were obtained (Figure <a href="#F1">1</a>, Figure <a href="#SM1">S1</a>). Homologs from all subfamilies could be divided into more than one COGs except for IIIc, IVc, Vb, and VIc subfamilies, from which homologs are included in a single COG (Figure <a href="#SM1">S1</a>). The appearance of various COGs within one subfamily suggests more complicated divergences during the evolutionary process of such NAC subfamilies than that of others. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Representative COGs of the Ia subfamily. Ia_1 <b>(A)</b> and Ia_4 <b>(B)</b>. Solid lines show reciprocal BeTs (Best Hits) and broken lines show asymmetrical BeTs. Genes from the same species are adjacent. Each gene ID is indicated, and the prefix "Rco" denotes genes from <i>Ricinus communis</i>. </p>   </div> <p>To unravel the evolutionary relationships among homologs of COGs, we constructed phylogenetic tree for each COG by using both NJ and ML approaches. The NJ analyses show that genes from different dicot or grass species cluster together in compact clades with high support values (mostly &amp;gt;90% bootstrap value; Li et al., <a href="#B31">2012</a>) in all the phylogenetic trees (Figure <a href="#F2">2</a>, Figure <a href="#SM2">S2</a>). The ML analyses yielded identical results (data not shown). Based on the topology of trees, we defined dicot and grass subgroups for each COG, and the relationships among homologous genes in each subgroup are largely consistent with the angiosperm phylogeny (Figure <a href="#F2">2</a>, Figure <a href="#SM2">S2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Phylogenetic relationship of sequences within the COG Ia_1 by NJ method with bootstrap support above 50% shown at the nodes. Square boxes indicate ancient duplication events; the triangles indicate recently occurred species-specific duplication events; and the broken lines indicate absent genes, either lost from those species or not yet sequenced. Gene names, based on previous references, are listed after the gene IDs. </p>   </div> </div> <div class="contrastingchangesinexpansionofthenacgenes" title="sec"> <div class="title" tagx="title" title="title">
Contrasting changes in expansion of the NAC genes</div> <p>NAC proteins were considered to have undergone extensive expansion after the divergence of vascular plants (Zhu et al., <a href="#B86">2012</a>). To well understand how NAC genes have evolved in angiosperms, we estimated the number of NAC genes in the most recent common ancestor (MRCA) of the extant angiosperms including dicots and grasses (Figure <a href="#F3">3</a>, Table <a href="#T1">1</a>). We noticed that there were 341 ancestral NAC genes in the MRCA of angiosperms (Figure <a href="#F3">3A</a>). When the numbers of ancestral genes were compared with those in the MRCA of dicots and grasses, it appeared that the NAC gene family has almost tripled in size since the divergence of dicots and grasses some 145 Mya (Davies et al., <a href="#B9">2004</a>; Anderson et al., <a href="#B3">2005</a>). However, the expansion was uneven among NAC subfamilies and between angiosperm lineages. In several subfamilies (e.g., IIIc), the number of genes remained unchanged, or nearly so, since the dicot/grass split (Figure <a href="#F3">3C</a>, Table <a href="#T1">1</a>). To the contrary, in many other subfamilies, the number had increased dramatically. In the Ic subfamily, for example, there were 86 and 60 genes in the MRCA of dicots and grasses, respectively, far more than the estimated number of genes (55 in total) in the MRCA of angiosperms (Figure <a href="#F3">3B</a>, Table <a href="#T1">1</a>). Dicots and grasses have gained 67 and 42 genes and lost 11 and 7 genes, respectively, in total, since their splits (Figure <a href="#F3">3B</a>). Clearly, the number of genes gained in the dicot lineages was much greater than that gained in the grass lineages (Figure <a href="#F3">3</a>, Table <a href="#T1">1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Evolutionary change of the number of NAC TFs in plants. <b>(A)</b> The overall NAC TFs examined in plants; <b>(B)</b> The Ic subfamily of the NAC TFs examined; <b>(C)</b> The IIIc subfamily of the NAC TFs examined; <b>(D)</b> The IVb subfamily of the NAC TFs examined. The numbers in circles and rectangles represent the numbers of genes in extant and ancestral species, respectively. The numbers with plus and minus signs indicate the number of gene gains and losses, respectively, for each branch (D, dicots and G, grasses). </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Gene duplication and loss in each of the NAC TFs COGs/subfamilies.</p>  <table> <thead> <tr> <th> <b>COG/Subfamily <a href="#TN3"> <sup>*</sup> </a> </b> </th> <th> <b>Gene duplication</b> </th> <th> <b>Gene loss</b> </th> </tr> <tr> <th /> <th> <b>Ancient</b> </th> <th> <b>Recent</b> </th> <th /> <th /> </tr> <tr> <th /> <th> <b>Dicot-specific</b> </th> <th> <b>Grass-specific</b> </th> <th> <b>Dicot species-specific</b> </th> <th> <b>Grass species-specific</b> </th> <th> <b>Dicot-specific</b> </th> <th> <b>Grass-specific</b> </th> </tr> </thead> <tbody> <tr> <td> <b>Ia/NAM/CUC3</b> <sup>(i)</sup> </td> </tr> <tr> <td>Ia_1</td> <td>2</td> <td>1</td> <td>11</td> <td>2</td> <td>5</td> <td>1</td> </tr> <tr> <td>Ia_2</td> <td>–</td> <td>2</td> <td>10</td> <td>4</td> <td>–</td> <td>–</td> </tr> <tr> <td>Ia_3</td> <td>2</td> <td>2</td> <td>18</td> <td>3</td> <td>7</td> <td>–</td> </tr> <tr> <td>Ia_4</td> <td>–</td> <td>–</td> <td>3</td> <td>–</td> <td>–</td> <td>1</td> </tr> <tr> <td>Ia_5</td> <td>–</td> <td>–</td> <td>6</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td> <b>Ib/NAC1</b> <sup>(i)</sup> </td> </tr> <tr> <td>Ib_1</td> <td>–</td> <td>2</td> <td>4</td> <td>1</td> <td>–</td> <td>–</td> </tr> <tr> <td>Ib_2</td> <td>–</td> <td>3</td> <td>6</td> <td>3</td> <td>2</td> <td>4</td> </tr> <tr> <td> <b>Ic/SND</b> <sup>(i)</sup> </td> </tr> <tr> <td>Ic_1</td> <td>3</td> <td>4</td> <td>16</td> <td>3</td> <td>8</td> <td>3</td> </tr> <tr> <td>Ic_2</td> <td>–</td> <td>1</td> <td>3</td> <td>2</td> <td>–</td> <td>–</td> </tr> <tr> <td>Ic_3</td> <td>–</td> <td>1</td> <td>10</td> <td>–</td> <td>–</td> <td>3</td> </tr> <tr> <td>Ic_4</td> <td>1</td> <td>1</td> <td>10</td> <td>1</td> <td>3</td> <td>1</td> </tr> <tr> <td>Ic_5</td> <td>–</td> <td>–</td> <td>4</td> <td>1</td> <td>–</td> <td>–</td> </tr> <tr> <td> <b>II/ONAC4</b> <sup>(i)</sup> </td> </tr> <tr> <td>II_1</td> <td>2</td> <td>1</td> <td>10</td> <td>–</td> <td>7</td> <td>1</td> </tr> <tr> <td>II_2</td> <td>–</td> <td>1</td> <td>5</td> <td>3</td> <td>1</td> <td>–</td> </tr> <tr> <td>II_3</td> <td>1</td> <td>2</td> <td>6</td> <td>1</td> <td>4</td> <td>1</td> </tr> <tr> <td>II_4</td> <td>–</td> <td>–</td> <td>5</td> <td>3</td> <td>–</td> <td>–</td> </tr> <tr> <td> <b>IIIa/TIP</b> <sup>(ii)</sup> </td> </tr> <tr> <td>IIIa_1</td> <td>1</td> <td>1</td> <td>11</td> <td>1</td> <td>2</td> <td>–</td> </tr> <tr> <td>IIIa_2</td> <td>–</td> <td>–</td> <td>7</td> <td>1</td> <td>–</td> <td>–</td> </tr> <tr> <td> <b>IIIb/NAC2</b> <sup>(i)</sup> </td> </tr> <tr> <td>IIIb_1</td> <td>5</td> <td>4</td> <td>28</td> <td>1</td> <td>8</td> <td>1</td> </tr> <tr> <td>IIIb_2</td> <td>–</td> <td>1</td> <td>10</td> <td>1</td> <td>–</td> <td>1</td> </tr> <tr> <td>IIIb_3</td> <td>–</td> <td>–</td> <td>10</td> <td>–</td> <td>2</td> <td>–</td> </tr> <tr> <td>IIIc/ANAC11 <sup>(ii)</sup> </td> <td>–</td> <td>–</td> <td>9</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td> <b>IVb/ANAC34</b> <sup>(i)</sup> <sup>(ii)</sup> </td> </tr> <tr> <td>IVb_1</td> <td>–</td> <td>2</td> <td>4</td> <td>2</td> <td>1</td> <td>–</td> </tr> <tr> <td>IVb_2</td> <td>1</td> <td>1</td> <td>4</td> <td>–</td> <td>4</td> <td>–</td> </tr> <tr> <td>IVc/TERN <sup>(i)</sup> </td> <td>1</td> <td>2</td> <td>13</td> <td>2</td> <td>3</td> <td>–</td> </tr> <tr> <td> <b>IVd/ONAC22</b> <sup>(i)</sup> <sup>(ii)</sup> </td> </tr> <tr> <td>IVd_1</td> <td>–</td> <td>2</td> <td>11</td> <td>1</td> <td>–</td> <td>2</td> </tr> <tr> <td>IVd_2</td> <td>1</td> <td>2</td> <td>11</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td> <b>Va/NAP</b> <sup>(i)</sup> <sup>(ii)</sup> </td> </tr> <tr> <td>Va_1</td> <td>–</td> <td>2</td> <td>6</td> <td>1</td> <td>–</td> <td>2</td> </tr> <tr> <td>Va_2</td> <td>–</td> <td>–</td> <td>7</td> <td>3</td> <td>–</td> <td>–</td> </tr> <tr> <td>Vb/SNAC <sup>(ii)</sup> </td> <td>6</td> <td>6</td> <td>36</td> <td>3</td> <td>13</td> <td>–</td> </tr> <tr> <td>VIc/ONAC6 <sup>(i)</sup> </td> <td>1</td> <td>1</td> <td>10</td> <td>–</td> <td>3</td> <td>1</td> </tr> <tr> <td>TOTAL</td> <td>27</td> <td>45</td> <td>324</td> <td>44</td> <td>74</td> <td>23</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">*</span> <p> <i>(i) and (ii) designate categories of NAC subfamilies, where (i) represents those involved in the relatively conserved processes (such as embryogenesis, cell division, seedling development, floral development, and senescence) and (ii) represents those involved in the relatively specific processes (such as biotic and abiotic stress responses) </i>. </p> </div> </div> </div> </div> <div class="distinctpatternsofgeneduplicationandlossbetweendicotsandgrasses" title="sec"> <div class="title" tagx="title" title="title">
Distinct patterns of gene duplication and loss between dicots and grasses</div> <p>To obtain detailed insights into the different duplication and loss patterns of NAC genes between dicots and grasses, we defined two types of duplication events, (a) the ancient duplications which occurred in the MRCA of dicot (dicot-specific duplication) or grass (grass-specific duplication) before their divergence, and (b) the recent duplications which are specific to a particular organismal lineage (one or few closely related species) of dicot (dicot species-specific duplication) or grass (grass species-specific duplication). We separately examined these two types of duplications in each one of the 31 COGs, and summarized the results in Table <a href="#T1">1</a>, Figure <a href="#F2">2</a>, and Figure <a href="#SM2">S2</a>. </p> <p>Our results demonstrate that most COGs (23/31) have undergone one or more ancient duplications, of which more occurred in grass (45 in total) than in dicot (27 in total) (Table <a href="#T1">1</a>). As to the recent duplications, the highest frequency in grass COGs is 4 times in Ia_2, while it occurred no less than 4 times in nearly all the COGs of dicot (except for Ia_4 and Ic_2), with the highest frequency 36 in Vb COG (subfamily). Given the evidence that recent polyploidization events are often observed in the dicot <i>Glycine max</i>, having the most recent event occurred 13 Mya (Schmutz et al., <a href="#B50">2010</a>), NAC genes in grasses tend to diverge through ancient duplications, most probably from whole genome duplication events in the MRCA of grasses before their divergence, while dicot NAC genes experienced more recent duplication events within specific dicot species. </p> <p>According to their functions, NAC subfamilies were previously classified into two major categories, one involved in the relatively conserved processes (category 1) and the other involved in the relatively specific processes (category 2) (Zhu et al., <a href="#B86">2012</a>). We noticed that subfamilies in category 1 (Ia, Ib, Ic, II, IIIb, IVc, and VIc subfamilies) often experienced more ancient duplications than those from category 2 (IIIa, IIIc, and Vb subfamilies). Moreover, some NAC subfamilies appeared before the divergence of angiosperms (Ia, Ic, IIIc, IVb, IVd, and Vb subfamilies; Zhu et al., <a href="#B86">2012</a>). Therefore, it is not surprising to observe more ancient duplications in these older subfamilies than in the recently diverged ones. </p> <p>We also found that dicot-specific gene losses occurred in nearly all the subfamilies analyzed and majority of the missing members are from <i>Medicago truncatula</i> (Table <a href="#T1">1</a>, Figure <a href="#F2">2</a>, and Figure <a href="#SM2">S2</a>; Li et al., <a href="#B31">2012</a>). By contrast, the grass-specific gene losses only occurred in the Ia, Ib, Ic, II, IIIb, IVd, and VIc subfamilies, all of which contain only category 1 genes, except for subfamily IVd. Obviously, there are fewer gene losses in the grass than that in the dicots (Table <a href="#T1">1</a>). </p> </div> <div class="morerapidevolutionofnacgenesoccurredingrasses" title="sec"> <div class="title" tagx="title" title="title">
More rapid evolution of NAC genes occurred in grasses</div> <p>To understand whether the NAC genes from grasses and dicots are under different evolutionary constraints, we applied two approaches to estimate the ω values of each COG, i.e., the pairwise comparison and the branch model. </p> <p>In the pairwise comparison approach, we calculated the pairwise ω values within each grass or dicot subgroup and compared their averages between these two subgroups. The result showed that the average ω values of 30 grass subgroups (subgroup 3 of Ia_2, subgroups 2 and 3 of Ia_3/II_3, subgroups 1, 2, and 3 of Ib_1, subgroups 3 and 4 of Ib_2, subgroups 1, 2, 4, and 5 of Ic_1, subgroups 1 and 2 of Ic_2/II_2/IIIb_2, subgroups 2 of Ic_3/Ic_4/IVd_1, subgroups 1 of II_4/IIIa_1/IIIa_2/IIIb_3, subgroups 2 and 5 of IIIb_1, and subgroups 1 and 7 of Vb) are significantly higher than that of the corresponding subfamilies composed of the homologs from the dicot; and 11 grass subgroups (subgroup 2 of Ia_2/IVb_1, subgroup 1 of Ia_5/II_1/IVc/Va_1/Vb_2/VIc, subgroup 4 of IIIb_1, and subgroup 1 and 2 of IVb_2) show the opposite result (Figure <a href="#F4">4</a>, Figure <a href="#SM3">S3</a>, and Table <a href="#SM4">S1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Pairwise estimation of ω values in each dicot or grass subgroup within subfamilies. <b>(A)</b> Ia_1-5; <b>(B)</b> Ic_1-5. In each of the subfamilies, a significance test on the difference between dicot and grass subgroup was carried out, and its result, listed in Table <a href="#SM4">S1</a>, is indicated by n.s. (not significant), <sup>*</sup> <i>P</i> &amp;lt; 0.05, and <sup>**</sup> <i>P</i> &amp;lt; 0.01. </p>   </div> <p>In order to examine changes in the evolutionary constraints across different subgroups, we also applied the branch-specific model to the phylogenetic trees of each COG because evolution rate of the most recent common ancestor in the subgroup is not taken into account in the pairwise comparison approach. In this approach, the model assumes two or more ω ratios, which correspond to the number of subgroups (Yang, <a href="#B76">2007</a>; Li et al., <a href="#B31">2012</a>). The result indicated that this model was favored over the one-ratio model by the LRT ( <i>P</i> &amp;lt; 0.05) test in all the phylogenetic trees (Table <a href="#T2">2</a>; Li et al., <a href="#B31">2012</a>). The ω values of the subgroups estimated from branch model confirm most of the result derived from the pairwise comparison approach (Table <a href="#T2">2</a>; Li et al., <a href="#B31">2012</a>). More importantly, when the most recent common ancestor is involved (e.g., subgroup 2 of Ia_2 or subgroup of Ia_5), some of the grass subgroups that exhibited a lower ω value than their dicot subgroups in the pairwise comparison approach demonstrated a higher ω value. It is worth mentioning that homologous NAC genes from nearly all the grass subgroups with a lower ω value than their corresponding dicot subgroups are involved in the relatively conserved biological processes (category 1). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Statistics of the parameters from branch model of PAML.</p>  <table> <thead> <tr> <th> <b>Subfamily/COG</b> </th> <th> <b>Null hypothesis</b> </th> <th> <b>Alternative hypothesis</b> </th> </tr> <tr> <th /> <th> <b>−ln <i>L</i> </b> </th> <th> <b>ω</b> </th> <th> <b>−ln <i>L</i> </b> </th> <th> <b>ω</b> </th> </tr> </thead> <tbody> <tr> <td> <b>Ia/NAM/CUC3</b> </td> </tr> <tr> <td>Ia_1</td> <td>8,386.28</td> <td>0.099</td> <td>8,385.35</td> <td>Dicot = 0.094; Grass1 = 0.120; Grass2 = 0.101</td> </tr> <tr> <td>Ia_2</td> <td>8,992.08</td> <td>0.152</td> <td>8,989.04</td> <td>Dicot = 0.136; Grass1 = 0.192; Grass2 = 0.150; Grass3 = 0.163</td> </tr> <tr> <td>Ia_3</td> <td>7,245.47</td> <td>0.129</td> <td>7,202.71</td> <td>Dicot = 0.087; Grass1 = 0.158; Grass2 = 0.215; Grass3 = 0.328</td> </tr> <tr> <td>Ia_4</td> <td>6,369.06</td> <td>0.174</td> <td>6,364.97</td> <td>Dicot = 0.155; Grass = 0.248</td> </tr> <tr> <td>Ia_5</td> <td>6,161.63</td> <td>0.181</td> <td>6,159.19</td> <td>Dicot = 0.165; Grass = 0.228</td> </tr> <tr> <td> <b>Ib/NAC1</b> </td> </tr> <tr> <td>Ib_1</td> <td>7,596.32</td> <td>0.128</td> <td>7,584.47</td> <td>Dicot = 0.108; Grass1 = 0.113; Grass2 = 0.205; Grass3 = 0.092</td> </tr> <tr> <td>Ib_2</td> <td>7,130.28</td> <td>0.125</td> <td>7,127.05</td> <td>Dicot = 0.111; Grass1 = 0.139; Grass2 = 0.093; Grass3 = 0.162; Grass4 = 0.138</td> </tr> <tr> <td> <b>Ic/SND</b> </td> </tr> <tr> <td>Ic_1</td> <td>7,504.70</td> <td>0.060</td> <td>7,487.05</td> <td>Dicot = 0.049; Grass1 = 0.048; Grass2 = 0.061; Grass3 = 0.033; Grass4 = 0.109; Grass5 = 0.107 </td> </tr> <tr> <td>Ic_2</td> <td>7,064.77</td> <td>0.129</td> <td>7,057.95</td> <td>Dicot = 0.103; Grass1 = 0.148; Grass2 = 0.173</td> </tr> <tr> <td>Ic_3</td> <td>5,570.71</td> <td>0.089</td> <td>5,560.92</td> <td>Dicot = 0.087; Grass1 = 0.197; Grass2 = 0.057</td> </tr> <tr> <td>Ic_4</td> <td>7,536.80</td> <td>0.093</td> <td>7,519.97</td> <td>Dicot = 0.072; Grass1 = 0.120; Grass2 = 0.162</td> </tr> <tr> <td>Ic_5</td> <td>5,354.24</td> <td>0.088</td> <td>5,351.12</td> <td>Dicot = 0.078; Grass = 0.120</td> </tr> <tr> <td> <b>II/ONAC4</b> </td> </tr> <tr> <td>II_1</td> <td>7,819.42</td> <td>0.096</td> <td>7,815.66</td> <td>Dicot = 0.105; Grass1 = 0.062; Grass2 = 0.082</td> </tr> <tr> <td>II_2</td> <td>7,137.84</td> <td>0.220</td> <td>7,126.78</td> <td>Dicot = 0.169; Grass1 = 0.320; Grass2 = 0.252</td> </tr> <tr> <td>II_3</td> <td>5,299.31</td> <td>0.061</td> <td>5,209.24</td> <td>Dicot = 0.019; Grass1 = 0.043; Grass2 = 0.129; Grass3 = 0.236</td> </tr> <tr> <td>II_4</td> <td>9,105.86</td> <td>0.196</td> <td>9,104.73</td> <td>Dicot = 0.186; Grass = 0.223</td> </tr> <tr> <td> <b>IIIa/TIP</b> </td> </tr> <tr> <td>IIIa_1</td> <td>9,158.06</td> <td>0.150</td> <td>9,155.96</td> <td>Dicot = 0.142; Grass1 = 0.195; Grass2 = 0.157</td> </tr> <tr> <td>IIIa_2</td> <td>9,005.51</td> <td>0.196</td> <td>8,985.61</td> <td>Dicot = 0.158; Grass = 0.361</td> </tr> <tr> <td> <b>IIIb/NAC2</b> </td> </tr> <tr> <td>IIIb_1</td> <td>8,008.27</td> <td>0.041</td> <td>7,986.81</td> <td>Dicot = 0.044; Grass1 = 0.021; Grass2 = 0.048; Grass3 = 0.002; Grass4 = 0.051; Grass5 = 0.081 </td> </tr> <tr> <td>IIIb_2</td> <td>10,374.08</td> <td>0.206</td> <td>10,373.51</td> <td>Dicot = 0.200; Grass1 = 0.196; Grass2 = 0.226</td> </tr> <tr> <td>IIIb_3</td> <td>11,422.90</td> <td>0.229</td> <td>11,418.33</td> <td>Dicot = 0.211; Grass = 0.300</td> </tr> <tr> <td>IIIc/ANAC11</td> <td>5,054.30</td> <td>0.115</td> <td>5,032.59</td> <td>Dicot = 0.085; Grass = 0.241</td> </tr> <tr> <td> <b>IVb/ANAC34</b> </td> </tr> <tr> <td>IVb_1</td> <td>7,100.19</td> <td>0.112</td> <td>7,092.41</td> <td>Dicot = 0.121; Grass1 = 0.153; Grass2 = 0.116; Grass3 = 0.079</td> </tr> <tr> <td>IVb_2</td> <td>6,228.24</td> <td>0.124</td> <td>6,220.74</td> <td>Dicot = 0.144; Grass1 = 0.152; Grass2 = 0.081</td> </tr> <tr> <td>IVc/TERN</td> <td>7,762.88</td> <td>0.179</td> <td>7,759.01</td> <td>Dicot = 0.160; Grass1 = 0.196; Grass2 = 0.227; Grass3 = 0.232</td> </tr> <tr> <td> <b>IVd/ONAC22</b> </td> </tr> <tr> <td>IVd_1</td> <td>9,018.96</td> <td>0.138</td> <td>9,018.52</td> <td>Dicot = 0.143; Grass1 = 0.132; Grass2 = 0.143; Grass3 = 0.122</td> </tr> <tr> <td>IVd_2</td> <td>7,229.97</td> <td>0.097</td> <td>7,227.56</td> <td>Dicot = 0.101; Grass1 = 0.109; Grass2 = 0.108; Grass3 = 0.073</td> </tr> <tr> <td> <b>Va/NAP</b> </td> </tr> <tr> <td>Va_1</td> <td>7,207.26</td> <td>0.182</td> <td>7,198.03</td> <td>Dicot = 0.144; Grass1 = 0.230; Grass2 = 0.259</td> </tr> <tr> <td>Va_2</td> <td>6,352.65</td> <td>0.172</td> <td>6,346.95</td> <td>Dicot = 0.143; Grass = 0.226</td> </tr> <tr> <td>Vb/SNAC</td> <td>14,926.52</td> <td>0.043</td> <td>14,895.68</td> <td>Dicot = 0.037; Grass1 = 0.090; Grass2 = 0.018; Grass3 = 0.082; Grass4 = 0.055; Grass5 = 0.087; Grass6 = 0.042; Grass7 = 0.083 </td> </tr> <tr> <td>VIc/ONAC6</td> <td>6,302.10</td> <td>0.180</td> <td>6,300.73</td> <td>Dicot = 0.194; Grass1 = 0.144; Grass2 = 0.167</td> </tr> </tbody> </table> </div> </div> <div class="differentialselectionbetweendicotsandgrasses" title="sec"> <div class="title" tagx="title" title="title">
Differential selection between dicots and grasses</div> <p>To assess the substitution process of these NAC subfamilies, we performed likelihood analyses by using Fitmodel (Guindon et al., <a href="#B19">2004</a>), which takes into account the variability of selection regimes across sites and lineages, and compared the sites under positive selection between the grass and the dicot subgroups within each phylogenetic tree of COG. Table <a href="#T3">3</a> shows that log likelihoods improved significantly ( <i>P</i> &amp;lt; 0.001) as parameters were added to the nested substitution models (Table <a href="#T3">3</a>), suggesting that the M3+S1 model was favored over the M3 model for each of data sets analyzed in our study. The parameter estimates of the M3+S1 model demonstrate that 66% sites ( <i>p</i> <sub>1</sub> + <i>p</i> <sub>2</sub>) are identified to be under purifying selection with ω <sub>1</sub> &amp;lt; 0.023 and ω <sub>2</sub> &amp;lt; 0.724 in all the COGs (Table <a href="#T3">3</a>). On the other hand, 2 to 22% ( <i>p</i> <sub>3</sub>) of sites within the genes in COGs are identified to be under positive selection with ω <sub>3</sub> values considerably larger than 1.0. Among these positive selection sites, the number of sites for genes in COGs of Ib_1, II_2, Va_1, Va_2, and Vb is significantly asymmetric across dicot and grass families, suggesting that the selection mainly occurred either in the dicot or in the grass subclade of these phylogenetic trees. This result provides evidence of divergent selection between dicots and grasses (Table <a href="#T3">3</a>). Since ω <sub>3</sub> values approaching 1.0 are described as indicative of relaxed selection, we cannot discount the possibility that positive selection may also have driven evolution in the COGs of Ia_4, II_1, IIIa_1, IIIb_3, and IVd_2, although no or slightly positive selection sign (ω <sub>3</sub> &amp;lt; 1.096) was observed (Table <a href="#T3">3</a>). Functionally critical (i.e., adaptive) substitutions at sites in the ω <sub>3</sub> class could be identified experimentally. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Likelihood ratio test statistic and parameters estimated from the M3 and M3 + S1 models in Fitmodel. </p>  <table> <thead> <tr> <th> <b>Subfamily/COG</b> </th> <th> <b>M3 Model <a href="#TN1"> <sup>a</sup> </a> (Heterogeneity across sites) </b> </th> <th> <b>M3 + S1 Model (Shifting across branches)</b> </th> <th> <b>Positive sites (M3 + S1 Model)</b> </th> </tr> <tr> <th /> <th> <b>−ln <i>L</i> </b> </th> <th> <b> <i>p</i> <sub>1</sub> </b> <br /><br /> <b>ω <sub>1</sub> </b> </th> <th> <b> <i>p</i> <sub>2</sub> </b> <br /><br /> <b>ω <sub>2</sub> </b> </th> <th> <b> <i>p</i> <sub>3</sub> </b> <br /><br /> <b>ω <sub>3</sub> </b> </th> <th> <b>−ln <i>L</i> </b> </th> <th> <b> <i>p</i> <sub>1</sub> </b> <br /><br /> <b>ω <sub>1</sub> </b> </th> <th> <b> <i>p</i> <sub>2</sub> </b> <br /><br /> <b>ω <sub>2</sub> </b> </th> <th> <b> <i>p</i> <sub>3</sub> </b> <br /><br /> <b>ω <sub>3</sub> </b> </th> <th> <b>Dicots specific</b> </th> <th> <b>Grasses specific</b> </th> </tr> </thead> <tbody> <tr> <td> <b>Ia/NAM/CUC3</b> </td> </tr> <tr> <td>Ia_1</td> <td>7,987.10</td> <td>0.488</td> <td>0.263</td> <td>0.249</td> <td>7,945.83 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.587</td> <td>0.231</td> <td>0.181</td> <td>153, 154, 159, 167</td> <td>30, 172, 173, 177, 202</td> </tr> <tr> <td /> <td /> <td>0.001</td> <td>0.081</td> <td>0.437</td> <td /> <td>&amp;lt;0.001</td> <td>0.103</td> <td>0.671</td> <td /> <td /> </tr> <tr> <td>Ia_2</td> <td>8,695.22</td> <td>0.505</td> <td>0.305</td> <td>0.191</td> <td>8,617.30 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.595</td> <td>0.258</td> <td>0.147</td> <td>33, 155, 166, 170, 176, 177, 180</td> <td>135, 161, 172, 173, 187, 188, 174, 175</td> </tr> <tr> <td /> <td /> <td>0.014</td> <td>0.225</td> <td>0.957</td> <td /> <td>&amp;lt;0.001</td> <td>0.243</td> <td>1.658</td> <td /> <td /> </tr> <tr> <td>Ia_3</td> <td>7,089.74</td> <td>0.548</td> <td>0.377</td> <td>0.075</td> <td>7,029.95 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.614</td> <td>0.308</td> <td>0.079</td> <td>136</td> <td>120, 135</td> </tr> <tr> <td /> <td /> <td>0.024</td> <td>0.242</td> <td>0.997</td> <td /> <td>0.001</td> <td>0.335</td> <td>2.041</td> <td /> <td /> </tr> <tr> <td>Ia_4</td> <td>6,151.02</td> <td>0.472</td> <td>0.286</td> <td>0.242</td> <td>6,140.29 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.513</td> <td>0.246</td> <td>0.241</td> <td>11, 178, 185, 189, 208, 220, 245, 255</td> <td>182, 207, 208, 228</td> </tr> <tr> <td /> <td /> <td>0.009</td> <td>0.235</td> <td>0.925</td> <td /> <td>0.003</td> <td>0.207</td> <td>1.096</td> <td /> <td /> </tr> <tr> <td>Ia_5</td> <td>5,945.22</td> <td>0.518</td> <td>0.233</td> <td>0.249</td> <td>5,903.54 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.665</td> <td>0.276</td> <td>0.059</td> <td>—</td> <td>225</td> </tr> <tr> <td /> <td /> <td>0.014</td> <td>0.281</td> <td>0.863</td> <td /> <td>0.011</td> <td>0.699</td> <td>3.616</td> <td /> <td /> </tr> <tr> <td> <b>Ib/NAC1</b> </td> </tr> <tr> <td>Ib_1</td> <td>7,286.55</td> <td>0.498</td> <td>0.202</td> <td>0.299</td> <td>7,241.32 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.641</td> <td>0.236</td> <td>0.123</td> <td>88, 193</td> <td>94, 126, 141, 145, 170, 180, 183, 191</td> </tr> <tr> <td /> <td /> <td>0.006</td> <td>0.118</td> <td>0.489</td> <td /> <td>0.005</td> <td>0.290</td> <td>1.152</td> <td /> <td /> </tr> <tr> <td>Ib_2</td> <td>6,771.42</td> <td>0.624</td> <td>0.263</td> <td>0.113</td> <td>6,745.78 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.671</td> <td>0.261</td> <td>0.068</td> <td>155, 162</td> <td>159, 181, 166, 178</td> </tr> <tr> <td /> <td /> <td>0.013</td> <td>0.274</td> <td>0.906</td> <td /> <td>0.006</td> <td>0.350</td> <td>2.233</td> <td /> <td /> </tr> <tr> <td> <b>Ic/SND</b> </td> </tr> <tr> <td>Ic_1</td> <td>9,314.31</td> <td>0.592</td> <td>0.318</td> <td>0.091</td> <td>9,266.30 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.735</td> <td>0.221</td> <td>0.044</td> <td>163</td> <td>145, 151</td> </tr> <tr> <td /> <td /> <td>0.007</td> <td>0.111</td> <td>0.488</td> <td /> <td>0.005</td> <td>0.206</td> <td>1.274</td> <td /> <td /> </tr> <tr> <td>Ic_2</td> <td>6,893.55</td> <td>0.537</td> <td>0.334</td> <td>0.128</td> <td>6,873.30 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.589</td> <td>0.282</td> <td>0.129</td> <td>—</td> <td>167, 170, 176, 177, 202, 204</td> </tr> <tr> <td /> <td /> <td>0.009</td> <td>0.235</td> <td>0.979</td> <td /> <td>&amp;lt;0.001</td> <td>0.240</td> <td>1.193</td> <td /> <td /> </tr> <tr> <td>Ic_3</td> <td>5,718.94</td> <td>0.682</td> <td>0.243</td> <td>0.075</td> <td>5,711.67 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.705</td> <td>0.218</td> <td>0.077</td> <td>183</td> <td>162, 164, 171</td> </tr> <tr> <td /> <td /> <td>0.014</td> <td>0.259</td> <td>0.997</td> <td /> <td>0.008</td> <td>0.279</td> <td>1.151</td> <td /> <td /> </tr> <tr> <td>Ic_4</td> <td>7,459.08</td> <td>0.644</td> <td>0.276</td> <td>0.080</td> <td>7,419.25 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.732</td> <td>0.120</td> <td>0.068</td> <td>144, 149, 169, 217</td> <td>199, 200, 140, 142</td> </tr> <tr> <td /> <td /> <td>0.012</td> <td>0.237</td> <td>1.088</td> <td /> <td>0.009</td> <td>0.369</td> <td>1.728</td> <td /> <td /> </tr> <tr> <td>Ic_5</td> <td>5,286.13</td> <td>0.520</td> <td>0.224</td> <td>0.257</td> <td>5,272.27 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.670</td> <td>0.286</td> <td>0.015</td> <td>167</td> <td>—</td> </tr> <tr> <td /> <td /> <td>0.006</td> <td>0.111</td> <td>0.470</td> <td /> <td>0.007</td> <td>0.464</td> <td>2.997</td> <td /> <td /> </tr> <tr> <td> <b>II/ONAC4</b> </td> </tr> <tr> <td>II_1</td> <td>7,535.33</td> <td>0.635</td> <td>0.225</td> <td>0.140</td> <td>7,527.86 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.670</td> <td>0.203</td> <td>0.128</td> <td>16, 146, 207</td> <td>—</td> </tr> <tr> <td /> <td /> <td>0.009</td> <td>0.187</td> <td>0.738</td> <td /> <td>0.005</td> <td>0.204</td> <td>0.867</td> <td /> <td /> </tr> <tr> <td>II_2</td> <td>7,642.77</td> <td>0.227</td> <td>0.498</td> <td>0.235</td> <td>7,606.47 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.379</td> <td>0.400</td> <td>0.220</td> <td>8, 11, 13, 14, 15, 16, 35, 67, 129, 145, 146, 202, 204, 205, 207, 208, 210</td> <td>22, 31, 73, 144, 167, 225</td> </tr> <tr> <td /> <td /> <td>0.009</td> <td>0.225</td> <td>0.784</td> <td /> <td>&amp;lt;0.001</td> <td>0.233</td> <td>1.138</td> <td /> <td /> </tr> <tr> <td>II_3</td> <td>7,027.97</td> <td>0.568</td> <td>0.289</td> <td>0.142</td> <td>7,003.30 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.621</td> <td>0.211</td> <td>0.168</td> <td>2, 206, 227</td> <td>58, 211, 212, 213</td> </tr> <tr> <td /> <td /> <td>0.010</td> <td>0.129</td> <td>0.532</td> <td /> <td>0.002</td> <td>0.147</td> <td>0.633</td> <td /> <td /> </tr> <tr> <td>II_4</td> <td>8,916.62</td> <td>0.411</td> <td>0.399</td> <td>0.190</td> <td>8,781.10 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.673</td> <td>0.285</td> <td>0.042</td> <td>249, 342</td> <td>—</td> </tr> <tr> <td /> <td /> <td>0.025</td> <td>0.265</td> <td>0.860</td> <td /> <td>0.015</td> <td>0.724</td> <td>&amp;gt;&amp;gt;1</td> <td /> <td /> </tr> <tr> <td> <b>IIIa/TIP</b> </td> </tr> <tr> <td>IIIa_1</td> <td>9,189.64</td> <td>0.422</td> <td>0.278</td> <td>0.301</td> <td>9,160.07 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.496</td> <td>0.293</td> <td>0.211</td> <td>161, 163, 165, 175</td> <td>100, 159, 162, 187, 189, 195</td> </tr> <tr> <td /> <td /> <td>0.007</td> <td>0.137</td> <td>0.589</td> <td /> <td>0.002</td> <td>0.186</td> <td>0.852</td> <td /> <td /> </tr> <tr> <td>IIIa_2</td> <td>8,769.59</td> <td>0.290</td> <td>0.352</td> <td>0.358</td> <td>8,726.21 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.349</td> <td>0.315</td> <td>0.337</td> <td>171, 196, 219, 226, 230, 231, 238</td> <td>157, 158, 222, 234, 236, 239, 241</td> </tr> <tr> <td /> <td /> <td>0.006</td> <td>0.113</td> <td>0.615</td> <td /> <td>0.002</td> <td>0.101</td> <td>0.781</td> <td /> <td /> </tr> <tr> <td> <b>IIIb/NAC2</b> </td> </tr> <tr> <td>IIIb_1</td> <td>11,715.75</td> <td>0.666</td> <td>0.309</td> <td>0.025</td> <td>11,609.68 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.685</td> <td>0.266</td> <td>0.049</td> <td>127</td> <td>92</td> </tr> <tr> <td /> <td /> <td>0.010</td> <td>0.109</td> <td>0.562</td> <td /> <td>0.002</td> <td>0.123</td> <td>0.614</td> <td /> <td /> </tr> <tr> <td>IIIb_2</td> <td>9,955.97</td> <td>0.298</td> <td>0.342</td> <td>0.360</td> <td>9,913.68 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.548</td> <td>0.240</td> <td>0.211</td> <td>183, 201, 203, 213, 218, 238</td> <td>176, 185,196, 200, 204, 215</td> </tr> <tr> <td /> <td /> <td>0.011</td> <td>0.133</td> <td>0.682</td> <td /> <td>0.023</td> <td>0.315</td> <td>1.233</td> <td /> <td /> </tr> <tr> <td>IIIb_3</td> <td>11,351.55</td> <td>0.327</td> <td>0.337</td> <td>0.336</td> <td>11,319.73 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.408</td> <td>0.281</td> <td>0.311</td> <td>4, 176, 177, 179, 201, 250, 259, 280, 337, 339</td> <td>—</td> </tr> <tr> <td /> <td /> <td>0.016</td> <td>0.218</td> <td>0.717</td> <td /> <td>0.006</td> <td>0.242</td> <td>0.885</td> <td /> <td /> </tr> <tr> <td>IIIc/ANAC11</td> <td>5,331.14</td> <td>0.482</td> <td>0.337</td> <td>0.180</td> <td>5,280.02 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.578</td> <td>0.246</td> <td>0.176</td> <td>2, 98, 110, 162, 176, 178, 181, 183, 152, 162, 163</td> <td>150, 153, 154, 155, 158, 160, 165, 168, 169, 170, 173, 174, 177, 184</td> </tr> <tr> <td /> <td /> <td>0.007</td> <td>0.160</td> <td>1.051</td> <td /> <td>&amp;lt;0.001</td> <td>0.157</td> <td>1.613</td> <td /> <td /> </tr> <tr> <td> <b>IVb/ANAC34</b> </td> </tr> <tr> <td>IVb_1</td> <td>7,232.44</td> <td>0.587</td> <td>0.238</td> <td>0.174</td> <td>7,188.84 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.655</td> <td>0.209</td> <td>0.136</td> <td>190, 191, 192</td> <td>2, 172, 196, 197, 201</td> </tr> <tr> <td /> <td /> <td>0.011</td> <td>0.224</td> <td>0.823</td> <td /> <td>0.002</td> <td>0.218</td> <td>1.324</td> <td /> <td /> </tr> <tr> <td>IVb_2</td> <td>6,123.96</td> <td>0.404</td> <td>0.399</td> <td>0.197</td> <td>6,086.42 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.583</td> <td>0.286</td> <td>0.131</td> <td>1, 3, 176</td> <td>157, 163, 185</td> </tr> <tr> <td /> <td /> <td>0.003</td> <td>0.122</td> <td>0.587</td> <td /> <td>&amp;lt;0.001</td> <td>0.188</td> <td>1.137</td> <td /> <td /> </tr> <tr> <td>IVc/TERN</td> <td>7,932.92</td> <td>0.581</td> <td>0.298</td> <td>0.121</td> <td>7,898.20 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.641</td> <td>0.260</td> <td>0.098</td> <td>149</td> <td>42</td> </tr> <tr> <td /> <td /> <td>0.038</td> <td>0.400</td> <td>1.115</td> <td /> <td>0.021</td> <td>0.487</td> <td>1.628</td> <td /> <td /> </tr> <tr> <td> <b>IVd/ONAC22</b> </td> </tr> <tr> <td>IVd_1</td> <td>9,087.07</td> <td>0.483</td> <td>0.267</td> <td>0.250</td> <td>9,072.53 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.520</td> <td>0.255</td> <td>0.226</td> <td>181, 187, 197, 210, 220, 234</td> <td>178, 179, 183, 199, 206, 213, 229</td> </tr> <tr> <td /> <td /> <td>0.007</td> <td>0.167</td> <td>0.711</td> <td /> <td>0.002</td> <td>0.173</td> <td>0.834</td> <td /> <td /> </tr> <tr> <td>IVd_2</td> <td>7,912.34</td> <td>0.535</td> <td>0.279</td> <td>0.186</td> <td>7,896.29 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.605</td> <td>0.272</td> <td>0.123</td> <td>143, 50, 126, 146, 147, 148, 149, 150, 154</td> <td>153, 158</td> </tr> <tr> <td /> <td /> <td>0.006</td> <td>0.152</td> <td>0.717</td> <td /> <td>0.002</td> <td>0.191</td> <td>1.026</td> <td /> <td /> </tr> <tr> <td> <b>Va/NAP</b> </td> </tr> <tr> <td>Va_1</td> <td>6,872.05</td> <td>0.488</td> <td>0.308</td> <td>0.204</td> <td>6,833.05 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.566</td> <td>0.285</td> <td>0.148</td> <td>95, 150, 156, 184, 185, 186, 187, 190, 191, 201, 205, 209</td> <td>162, 164, 197, 206, 208</td> </tr> <tr> <td /> <td /> <td>0.007</td> <td>0.239</td> <td>1.030</td> <td /> <td>&amp;lt;0.001</td> <td>0.335</td> <td>1.746</td> <td /> <td /> </tr> <tr> <td>Va_2</td> <td>6,093.17</td> <td>0.466</td> <td>0.282</td> <td>0.252</td> <td>6,066.48 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.544</td> <td>0.252</td> <td>0.203</td> <td>116, 148, 150, 195, 196, 201</td> <td>163, 191, 202, 208, 133, 156, 158, 159, 209</td> </tr> <tr> <td /> <td /> <td>0.006</td> <td>0.177</td> <td>0.892</td> <td /> <td>0.004</td> <td>0.253</td> <td>1.260</td> <td /> <td /> </tr> <tr> <td>Vb/SNAC_11</td> <td>15,736.23</td> <td>0.594</td> <td>0.229</td> <td>0.177</td> <td>15,647.48 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.612</td> <td>0.222</td> <td>0.165</td> <td>52</td> <td>28, 47, 50, 80, 115, 148, 151, 152</td> </tr> <tr> <td /> <td /> <td>0.003</td> <td>0.067</td> <td>0.233</td> <td /> <td>&amp;lt;0.001</td> <td>0.054</td> <td>0.349</td> <td /> <td /> </tr> <tr> <td>VIc/ONAC6</td> <td>6,176.96</td> <td>0.382</td> <td>0.533</td> <td>0.086</td> <td>6,159.23 <a href="#TN2"> <sup>**</sup> </a> </td> <td>0.525</td> <td>0.428</td> <td>0.046</td> <td>114</td> <td>82</td> </tr> <tr> <td /> <td /> <td>0.027</td> <td>0.262</td> <td>0.887</td> <td /> <td>0.006</td> <td>0.395</td> <td>2.033</td> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">a</span> <p> <i>p <sub>i</sub> proportion of sites that fall into the ω <sub>i</sub> site class, i = 1, 2, 3 </i>. </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">**</span> <p> <i>P &amp;lt; 0.01, and the probabilities are obtained by LRT (df = 2) of the models M3 and M3 + S1 </i>. </p> </div> </div> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In the present study, we were seeking evidence of divergent evolutionary patterns among NAC TFs by comparing serial evolutionary features of non-lineage specific NAC subfamilies and uncovered patterns of duplication and diversification in each subfamily. As a result, several distinct evolutionary features were observed, (1) uneven constitution of COGs and contrasting birth/death rates among subfamilies, (2) distinct patterns of gene duplication and loss between dicots and grasses, (3) a relatively higher evolution rate in grass than in dicot in most subfamilies analyzed, and (4) asymmetrical selection pattern between dicots and grasses. Clearly, with these findings, a general picture has emerged for the mode, tempo, and consequence of the evolution of NAC TFs during the angiosperm evolution, and several relevant conclusions can now be addressed. </p> <div class="differentscenariosfornactfsevolutionindicotsandgrasses" title="sec"> <div class="title" tagx="title" title="title">
Different scenarios for NAC TFs evolution in dicots and grasses</div> <p>We have demonstrated that in a given COG the copy number of grass genes is always less than that of dicot genes. This phenomenon might attribute to two observations. First, dicots underwent more recent duplications. Comparative analyses have revealed up to two additional rounds of recent genome duplication in <i>B. rapa</i> after its divergence from <i>A. thaliana</i>: one triplication that arose approximately 13 Mya (Rana et al., <a href="#B47">2004</a>; Mun et al., <a href="#B34">2009</a>), and one allopolyploidization (Mun et al., <a href="#B34">2009</a>; Wang et al., <a href="#B68">2011</a>). A very recent WGD was also observed in <i>G. max</i> (Schmutz et al., <a href="#B50">2010</a>). Second, Ohno ( <a href="#B39">1970</a>) pointed out that following gene duplications a relaxed constraint at some sites was expected. This is very much concordant with our observations that NAC genes always show lower ω values in dicot species within one COG/subfamily. This conclusion is favored when we consider only the COGs/subfamilies with contrasting numbers of recent duplications. Meanwhile, some studies have also showed that genes having low evolution rate tend to stay in the species after gene duplication and might be more easily sub-functionalized (Davis and Petrov, <a href="#B10">2004</a>; Brunet et al., <a href="#B5">2006</a>; Semon and Wlofe, <a href="#B51">2008</a>). </p> <p>The above observations guaranteed dicots recruited more members of NAC TFs than monocots but usually with a lower evolution rate within certain COGs/subfamilies. Therefore, two diverse evolutionary scenarios can be proposed, the relatively more copies of NAC TFs in dicots might interact with each other as part of a network through sub-functionalization, on the other hand, with high evolution rate, grass NAC TFs were always accompanied by diversifying selection (positive selection). In fact, the poplar PtrWND TFs (Ic subfamily) interact with each other and perform a diverse combination of heterodimers together with their downstream TFs to form a transcriptional network, thus activating the secondary wall biosynthetic program during wood formation (Zhong et al., <a href="#B84">2011</a>). In contrast with dicots, the frequent diversifying selection in grass history and following relaxed selection constraints might imply the increased allelic diversity at some sites and increased opportunity for novel gene interactions. For instance, our another study on allelic variation of <i>NAM-B1</i> gene (Va subfamily) through 41 populations of wild emmer wheat revealed great diversity in <i>NAM-B1</i> locus both between and within populations. Meanwhile, the polymorphism of <i>NAM-B1</i> gene was significantly correlated with multiple eco-environmental factors (unpublished data). Thus, it is possible that the <i>NAM-B1</i> divergence may contribute to fitness as an adaptive evolution strategy to novel ecological challenges. </p> </div> <div class="newly-bornnactfscontributetotheevolutioncomplexityofthehigherplants" title="sec"> <div class="title" tagx="title" title="title">
Newly-born NAC TFs contribute to the evolution complexity of the higher plants</div> <p>Several NAC subfamilies possessed stable copy number, but many others had more dynamic birth-and-death evolution, much higher birth rate than death rate (Table <a href="#T1">1</a>, Figure <a href="#F3">3</a>). Moreover, in some subfamilies, the estimated birth rates are obviously higher than that for other well-known rapidly-duplicating genes, such as Type I MADS-box genes (Nam et al., <a href="#B36">2004</a>), SKP1-like genes (Kong et al., <a href="#B29">2007</a>), and F-box genes (Xu et al., <a href="#B72">2009</a>). The obviously high birth rates elicited the question why plants have maintained so many new genes during the evolution process. </p> <p>The possible explanation is that members of the rapidly-duplicating gene subfamilies interact with variable and/or changing targets. As shown previously, the C-terminal regions of NAC proteins, essential for target recognition, are highly variable even within the same subfamily (Ooka et al., <a href="#B41">2003</a>; Zhu et al., <a href="#B86">2012</a>). The most direct evidence could be addressed in Vb subfamily. Vb subfamily is one of the three most ancient subfamilies (Ia, Ic, and Vb subfamilies) and had been established since the origin of early-diverged land plant <i>Physcomitrella patens</i> (Zhu et al., <a href="#B86">2012</a>). Our results showed that all of the members in Vb subfamily were included in one COG, suggesting an extraordinarily high birth rate from a single ancestral sequence (Figure <a href="#F3">3D</a>, Table <a href="#T1">1</a>). The unusual increase of the gene number in Vb subfamily might be because of their function in balancing the cross talk between stress-related networks or pathogen-borne targets (see discussion below) in a precisely controlled fashion because almost all the members in this family contribute to response to biotic and abiotic stresses. </p> <p>Coincidentally, the NAC TFs from Ic subfamily were found to function as master switches in the secondary wall biosynthesis, one of the pivotal features that vascular plants have evolved to enable themselves to conquer dry land around 430 Mya (Raven and Johnson, <a href="#B48">2002</a>; Zhong et al., <a href="#B82">2010a</a>). Genetic and functional analyses of secondary wall NACs (SWNs) in diverse species have indicated that the SWN-mediated activation of the secondary wall biosynthetic process is a conserved mechanism (Zhong et al., <a href="#B83">2010b</a>) in both seedless vascular plants (e.g., <i>S. moellendorffii</i>) and seed plants, including gymnosperms and angiosperms (Zhong et al., <a href="#B83">2010b</a>). Our results demonstrate that the SWN homologs exist in both herbaceous plants, such as <i>Arabidopsis</i>, and woody poplar; and the number of SWNs in poplar is nearly twice of <i>Arabidopsis</i>. This increase in the number of SWNs in poplar was probably due to a salicoid duplication event, a WGD event that occurred near the emergence of Salix and Populus lineages 60-65 Mya (Tuskan et al., <a href="#B63">2006</a>). Given the fact that homologs of SWNs in poplar display a different transcriptional program regulating the biosynthesis of secondary walls during wood formation apart from their <i>Arabidopsis</i> counterparts (Zhong et al., <a href="#B84">2011</a>), this might imply that woody angiosperm plant lineages have co-opted the pre-existing ancestral NAC genes, through duplication and specialization, into new functions to coordinate the activation of the secondary wall biosynthesis in wood formation. </p> <p>However, several studies suggest that the number of genes in a family initially depends on their functional requirement. However, after the number reached a sufficient level, the number can fluctuate occasionally (Nei, <a href="#B37">2007</a>; Nei et al., <a href="#B38">2008</a>). In our case, NAC TFs share relatively identical function within the same subfamily (Zhu et al., <a href="#B86">2012</a>). Although some recently duplicated NAC genes were found to be functionally redundant (Hasson et al., <a href="#B21">2011</a>; Zhong et al., <a href="#B84">2011</a>), which suggests that small gene number changes do not necessarily change in the physiological requirement, we still cannot exclude the possibility that the gain or loss of some NAC genes, at least in some subfamilies, were due to random events. </p> </div> <div class="highevolutionratesdrivenactfsintoversatilefunctions" title="sec"> <div class="title" tagx="title" title="title">
High evolution rates drive NAC TFs into versatile functions</div> <p>The whole-genome expression profiling studies in <i>Arabidopsis</i> and rice have revealed NAC genes to be induced by at least one type of abiotic stress such as salinity, drought, cold, or abscisic acid (Fang et al., <a href="#B13">2008</a>; Kawaura et al., <a href="#B26">2008</a>), whereas in several other species, NAC genes were also shown to be induced by jasmonic acid, salicylic acid, and ethylene (Tran et al., <a href="#B61">2004</a>; Nakashima et al., <a href="#B35">2007</a>; Xia et al., <a href="#B70">2010</a>; Yoshii et al., <a href="#B78">2010</a>). Meanwhile, NAC proteins were also proven to have an affirmative role in the regulation of plant defense against different pathogens as well as wounding and insect feeding (Lin et al., <a href="#B32">2007</a>; Wang et al., <a href="#B67">2009</a>; Xia et al., <a href="#B70">2010</a>). Members involved in these stress-response processes include all the COGs of IIIa, IIIc, IVb, IVc, IVd, Va, and Vb subfamilies and part of COGs in Ia, Ib, II, and IIIb subfamilies. </p> <p>Among the stress-related NAC subfamilies, the Vb subfamily possessed the smallest ω value, with &amp;gt;9 sites evolving under relaxed selection, suggesting that members of this subfamily are more conserved than others. It was shown that members of Vb subfamily tend to function at the convergence point of abiotic stress signaling and pathogen defense that was induced by multiple stresses (Fujita et al., <a href="#B16">2004</a>; Delessert et al., <a href="#B12">2005</a>; Jensen et al., <a href="#B23">2008</a>; Puranik et al., <a href="#B45">2012</a>), suggesting members of Vb subfamily are key factors involved in cross talk between the abiotic- and biotic-stress cellular network (Fujita et al., <a href="#B17">2009</a>). Thus, it makes sense to conclude that changes in one key element of a specific network can be constrained by its interactions with other elements, which lead to a relatively low evolution rate and less variable function of members in the Vb subfamily. On the contrary, members from other stress-related subfamilies (e.g., II, IIIa, IIIb, and IVb subfamilies) have integrated responses to environmental stresses into modulation of plant development processes, such as lateral root development (He et al., <a href="#B22">2005</a>; Hao et al., <a href="#B20">2011</a>; Puranik et al., <a href="#B45">2012</a>; Wei et al., <a href="#B69">2016</a>), seed germination (Kim et al., <a href="#B28">2008</a>; Seo et al., <a href="#B52">2010</a>; Wei et al., <a href="#B69">2016</a>), flowering (Kim et al., <a href="#B27">2007</a>; Wei et al., <a href="#B69">2016</a>), and senescence (Balazadeh et al., <a href="#B4">2010</a>; Seo and Park, <a href="#B53">2011</a>; Yang et al., <a href="#B74">2011</a>). Such versatility of NAC functions might result in greater ω values and more sites under positive selection in order to ensure plants' longevity, survival, and reproductive success under various environmental stresses. </p> <p>Another example emerges from the evolutionary process of <i>CUCs</i>, all of which belong to Ia subfamily. After the <i>CUC3</i> gave birth to <i>CUC1/2</i> through gene duplication event at the base of extant seed plants (Vialette-Guiraud et al., <a href="#B65">2011</a>), the novel <i>CUCs</i> evolved differentially since the initial duplication. <i>CUC2</i> did not diverge much from the ancestral gene, while <i>CUC1</i> evolved more profoundly from its ancestor through neo-functionalization (Hasson et al., <a href="#B21">2011</a>). Meanwhile, Ia subfamily genes appear to have obtained novel functions in defense responses while retaining their ancestral function in the establishment of the shoot organ boundaries (Aida et al., <a href="#B2">1997</a>; Yamaguchi et al., <a href="#B73">2008</a>; Takeda et al., <a href="#B59">2011</a>). </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study, we reported divergent evolutionary patterns of NAC TFs, both among subfamilies and between dicots and grasses. Observations also showed that the distinct fates of NAC TFs are associated with plant lineage diversification and gene duplication events. Considering our results along with functional data, we hypothesize that rapidly born NAC TFs and relaxed selection in certain NAC COGs/subfamilies may have permitted substitutions that are responsible for functional divergence into new lineages. Meanwhile, the high evolution rates have increased complexity of interaction of NAC TFs following the concerted duplications from a MRCA of all the extant angiosperms. </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author contributions</div> <p>XJ and JR were in charge of data collection, analyses, and manuscript drafting and finalizing. EN and XY worked on revising and finalizing the manuscript. JP took care of the whole project, i.e., applying for grants, designing the experiments, outlining and finalizing the manuscript. All relevant authors and institutions have approved the submission for publication of this manuscript; and all persons entitled to authorship have been so named and have agreed to the submitted version of the manuscript. </p> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Dr. Tingting Zhu, the former Ph.D. student of JP, for her great help in data collection and analysis, and drafting the manuscript. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding.</b> This research was supported by the China National Science Foundation (NSFC) Grant Nos. 31030055 and 30870233 (to JP), China National Special Program for Development of Transgenic Plant and Animal New Cultivars (to JP), Chinese Academy of Sciences under the Important Directional Program of Knowledge Innovation Project Grant No. KSCX2-YW-Z-0722, the CAS Strategic Priority Research Program Grant No. XDA05130403 (to JP), and the "973" National High-Tech Research Program Grant No. 2011AA10A106, and the Taishan Scholars Program of Shandong province of China. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <p>The Supplementary Material for this article can be found online at: <a href="http://journal.frontiersin.org/article/10.3389/fpls.2017.01156/full#supplementary-material">http://journal.frontiersin.org/article/10.3389/fpls.2017.01156/full#supplementary-material</a> </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table1.DOCX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Presentation1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Presentation2.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Presentation3.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abascal</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zardoya</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">ProtTest: selection of best-fit models of protein evolution</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">2104</span>– <span class="lpage" tagx="lpage" title="lpage">2105</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti263">10.1093/bioinformatics/bti263</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15647292">15647292</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aida</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukaki</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujisawa</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tasaka</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Genes involved in organ separation in Arabidopsis: an analysis of the cupshaped cotyledon mutant </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">841</span>– <span class="lpage" tagx="lpage" title="lpage">857</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.9.6.841">10.1105/tpc.9.6.841</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9212461">9212461</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bremer</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friis</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Dating phylogenetically basal eudicots using rbcL sequences and multiple fossil reference points </span>. <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="volume" tagx="volume" title="volume">92</span>, <span class="fpage" tagx="fpage" title="fpage">1737</span>– <span class="lpage" tagx="lpage" title="lpage">1748</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.92.10.1737">10.3732/ajb.92.10.1737</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21646091">21646091</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balazadeh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siddiqui</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allu</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matallana-Ramirez</span> <span class="given-names" tagx="given-names" title="given-names">L. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caldana</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mehrnia</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">A gene regulatory network controlled by the NAC transcription factor ANAC092/AtNAC2/ORE1 during salt promoted senescence </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">62</span>, <span class="fpage" tagx="fpage" title="fpage">250</span>– <span class="lpage" tagx="lpage" title="lpage">264</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2010.04151.x">10.1111/j.1365-313X.2010.04151.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20113437">20113437</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunet</span> <span class="given-names" tagx="given-names" title="given-names">F. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roest Crollius</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paris</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aury</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibert</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaillon</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Gene loss and evolutionary rates following whole-genome duplication in teleost fishes</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">1808</span>– <span class="lpage" tagx="lpage" title="lpage">1816</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msl049">10.1093/molbev/msl049</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16809621">16809621</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chan</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crabtree</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lorenzi</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orvis</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puiu</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Draft genome sequence of the oilseed species Ricinus communis</span>. <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">951</span>– <span class="lpage" tagx="lpage" title="lpage">956</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.1674">10.1038/nbt.1674</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20729833">20729833</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christianson</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llewellyn</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">I. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">ATAF NAC transcription factors: regulators of plant stress signaling</span>. <span class="source" tagx="source" title="source">Plant Signal. Behav.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">428</span>– <span class="lpage" tagx="lpage" title="lpage">432</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.5.4.10847">10.4161/psb.5.4.10847</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20118664">20118664</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">P. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindsay</span> <span class="given-names" tagx="given-names" title="given-names">B. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doyle</span> <span class="given-names" tagx="given-names" title="given-names">J. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Widespread genome duplications throughout the history of flowering plants</span>. <span class="source" tagx="source" title="source">Genome Res.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">738</span>– <span class="lpage" tagx="lpage" title="lpage">749</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.4825606">10.1101/gr.4825606</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16702410">16702410</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davies</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barraclough</span> <span class="given-names" tagx="given-names" title="given-names">T. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chase</span> <span class="given-names" tagx="given-names" title="given-names">M. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savolainen</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Darwin's abominable mystery: insights from a supertree of the angiosperms</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">101</span>, <span class="fpage" tagx="fpage" title="fpage">1904</span>– <span class="lpage" tagx="lpage" title="lpage">1909</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0308127100">10.1073/pnas.0308127100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14766971">14766971</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petrov</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Preferential duplication of conserved proteins in eukaryotic genomes</span>. <span class="source" tagx="source" title="source">PLoS Biol.</span> <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">E55</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pbio.0020055">10.1371/journal.pbio.0020055</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15024414">15024414</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Bodt</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maere</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van de Peer</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Genome duplication and the origin of angiosperms</span>. <span class="source" tagx="source" title="source">Trends Ecol. Evol.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">591</span>– <span class="lpage" tagx="lpage" title="lpage">597</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tree.2005.07.008">10.1016/j.tree.2005.07.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16701441">16701441</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delessert</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kazan</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">I. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Der Straeten</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manners</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">The transcription factor ATAF2 represses the expression of pathogenesis-related genes in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">745</span>– <span class="lpage" tagx="lpage" title="lpage">757</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02488.x">10.1111/j.1365-313X.2005.02488.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16115070">16115070</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">You</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Systematic sequence analysis and identification of tissue-specific or stress-responsive genes of NAC transcription factor family in rice </span>. <span class="source" tagx="source" title="source">Mol. Genet. Genomics</span> <span class="volume" tagx="volume" title="volume">280</span>, <span class="fpage" tagx="fpage" title="fpage">535</span>– <span class="lpage" tagx="lpage" title="lpage">546</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-008-0386-6">10.1007/s00438-008-0386-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18813955">18813955</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felsenstein</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1989</span>). <span class="mixed-article-title" title="mixed-article-title">PHYLIP-phylogeny inference package (version 3.6.9)</span>. <span class="source" tagx="source" title="source">Cladistics</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">164</span>– <span class="lpage" tagx="lpage" title="lpage">166</span>. </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freeling</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Gene-balanced duplications, like tetraploidy, provide predictable drive to increase morphological complexity </span>. <span class="source" tagx="source" title="source">Genome Res.</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">805</span>– <span class="lpage" tagx="lpage" title="lpage">814</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.3681406">10.1101/gr.3681406</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16818725">16818725</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiratsu</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohme-Takagi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">A dehydration induced NAC protein, RD26, is involved in a novel ABA-dependent stress-signaling pathway </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">863</span>– <span class="lpage" tagx="lpage" title="lpage">876</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2004.02171.x">10.1111/j.1365-313X.2004.02171.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15341629">15341629</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Stress physiology of higher plants: cross-talk between abiotic and biotic stress signaling</span>, in <span class="source" tagx="source" title="source">Plant Stress Biology: From Genomics to Systems Biology</span>, ed <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirt</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley-VCH Verlag GmbH &amp;amp; Co. KgaA</span>), <span class="fpage" tagx="fpage" title="fpage">65</span>– <span class="lpage" tagx="lpage" title="lpage">89</span>. </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guindon</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gascuel</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <span class="source" tagx="source" title="source">Syst. Biol.</span> <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">696</span>– <span class="lpage" tagx="lpage" title="lpage">704</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1080/10635150390235520">10.1080/10635150390235520</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136">14530136</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guindon</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodrigo</span> <span class="given-names" tagx="given-names" title="given-names">A. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dyer</span> <span class="given-names" tagx="given-names" title="given-names">K. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Modeling the site-specific variation of selection patterns along lineages</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">101</span>, <span class="fpage" tagx="fpage" title="fpage">12957</span>– <span class="lpage" tagx="lpage" title="lpage">12962</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0402177101">10.1073/pnas.0402177101</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15326304">15326304</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">Y. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">Q. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Soybean NAC transcription factors promote abiotic stress tolerance and lateral root formation in transgenic plants </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">68</span>, <span class="fpage" tagx="fpage" title="fpage">302</span>– <span class="lpage" tagx="lpage" title="lpage">313</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2011.04687.x">10.1111/j.1365-313X.2011.04687.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21707801">21707801</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasson</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plessis</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blein</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adroher</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grigg</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsiantis</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution and diverse roles of the CUP-SHAPED COTYLEDON genes in Arabidopsis leaf development </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">54</span>– <span class="lpage" tagx="lpage" title="lpage">68</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.110.081448">10.1105/tpc.110.081448</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21258003">21258003</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">X. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">W. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">S. Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">AtNAC2, a transcription factor downstream of ethylene and auxin signaling pathways, is involved in salt stress response and lateral root development </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">903</span>– <span class="lpage" tagx="lpage" title="lpage">916</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2005.02575.x">10.1111/j.1365-313X.2005.02575.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16359384">16359384</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hagedorn</span> <span class="given-names" tagx="given-names" title="given-names">P. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Torres-Zabala</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grant</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rung</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collinge</span> <span class="given-names" tagx="given-names" title="given-names">D. B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptional regulation by an NAC (NAM-ATAF1, 2- CUC2) transcription factor attenuates ABA signalling for efficient basal defense towards <i>Blumeria graminis</i> f. sp. hordei in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">867</span>– <span class="lpage" tagx="lpage" title="lpage">880</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03646.x">10.1111/j.1365-313X.2008.03646.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18694460">18694460</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kjaersgaard</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galberg</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Shea</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skriver</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis thaliana</i> NAC transcription factor family: structure-function relationships and determinants of ANAC019 stress signalling </span>. <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="volume" tagx="volume" title="volume">426</span>, <span class="fpage" tagx="fpage" title="fpage">183</span>– <span class="lpage" tagx="lpage" title="lpage">196</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20091234">10.1042/BJ20091234</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19995345">19995345</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wickett</span> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ayyampalayam</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chanderbali</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landherr</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ralph</span> <span class="given-names" tagx="given-names" title="given-names">P. E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Ancestral polyploidy in seed plants and angiosperms</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">473</span>, <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">102</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature09916">10.1038/nature09916</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21478875">21478875</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawaura</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mochida</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogihara</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis for identification of salt-responsive genes in common wheat</span>. <span class="source" tagx="source" title="source">Funct. Integr. Genomics</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">277</span>– <span class="lpage" tagx="lpage" title="lpage">286</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-008-0076-9">10.1007/s10142-008-0076-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18320247">18320247</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">A membrane-associated NAC transcription factor regulates salt-responsive flowering via FLOWERING LOCUS T in Arabidopsis </span>. <span class="source" tagx="source" title="source">Planta</span> <span class="volume" tagx="volume" title="volume">226</span>, <span class="fpage" tagx="fpage" title="fpage">647</span>– <span class="lpage" tagx="lpage" title="lpage">654</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00425-007-0513-3">10.1007/s00425-007-0513-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17410378">17410378</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">A membrane-bound NAC transcription factor NTL8 regulates gibberellic acid-mediated salt signaling in Arabidopsis seed germination </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">77</span>– <span class="lpage" tagx="lpage" title="lpage">88</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03493.x">10.1111/j.1365-313X.2008.03493.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18363782">18363782</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landherr</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frohlich</span> <span class="given-names" tagx="given-names" title="given-names">M. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Pamphilis</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Patterns of gene duplication in the plant SKP1 gene family in angiosperms: evidence for multiple mechanisms of rapid gene birth </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">873</span>– <span class="lpage" tagx="lpage" title="lpage">885</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2007.03097.x">10.1111/j.1365-313X.2007.03097.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17470057">17470057</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larkin</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blackshields</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">N. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chenna</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGettigan</span> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McWilliam</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Clustal W and Clustal X version 2.0</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">2947</span>– <span class="lpage" tagx="lpage" title="lpage">2948</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btm404">10.1093/bioinformatics/btm404</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Q. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunwell</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Divergent evolutionary pattern of starch biosynthetic pathway genes in grasses and dicots </span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">3227</span>– <span class="lpage" tagx="lpage" title="lpage">3236</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mss131">10.1093/molbev/mss131</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22586327">22586327</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Rice gene OsNAC19 encodes a novel NAC-domain transcription factor and responds to infection by <i>Magnaporthe grisea</i> </span>. <span class="source" tagx="source" title="source">Plant Sci.</span> <span class="volume" tagx="volume" title="volume">172</span>, <span class="fpage" tagx="fpage" title="fpage">120</span>– <span class="lpage" tagx="lpage" title="lpage">130</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plantsci.2006.07.019">10.1016/j.plantsci.2006.07.019</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maere</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Bodt</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raes</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casneuf</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Montagu</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuiper</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Modeling gene and genome duplications in eukaryotes</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">102</span>, <span class="fpage" tagx="fpage" title="fpage">5454</span>– <span class="lpage" tagx="lpage" title="lpage">5459</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0501102102">10.1073/pnas.0501102102</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15800040">15800040</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mun</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwon</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seol</span> <span class="given-names" tagx="given-names" title="given-names">Y. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide comparative analysis of the <i>Brassica rapa</i> gene space reveals genome shrinkage and differential loss of duplicated genes after whole genome triplication </span>. <span class="source" tagx="source" title="source">Genome Biol.</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">R111</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/gb-2009-10-10-r111">10.1186/gb-2009-10-10-r111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19821981">19821981</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">L. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Nguyen</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todaka</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Functional analysis of a NAC-type transcription factor OsNAC6 involved in abiotic and biotic stress responsive gene expression in rice </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">51</span>, <span class="fpage" tagx="fpage" title="fpage">617</span>– <span class="lpage" tagx="lpage" title="lpage">630</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2007.03168.x">10.1111/j.1365-313X.2007.03168.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17587305">17587305</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nam</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Type I MADS-box genes have experienced faster birth-and-death evolution than type II MADS-box genes in angiosperms </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">101</span>, <span class="fpage" tagx="fpage" title="fpage">1910</span>– <span class="lpage" tagx="lpage" title="lpage">1915</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0308430100">10.1073/pnas.0308430100</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14764899">14764899</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">The new mutation theory of phenotypic evolution</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">104</span>, <span class="fpage" tagx="fpage" title="fpage">12235</span>– <span class="lpage" tagx="lpage" title="lpage">12242</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0703349104">10.1073/pnas.0703349104</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17640887">17640887</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niimura</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nozawa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">The evolution of animal chemosensory receptor gene repertoires: roles of chance and necessity </span>. <span class="source" tagx="source" title="source">Nature Rev. Genet.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">951</span>– <span class="lpage" tagx="lpage" title="lpage">963</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2480">10.1038/nrg2480</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19002141">19002141</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohno</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1970</span>). <span class="source" tagx="source" title="source">Evolution by Gene Duplication</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Heidelberg</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>. </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsen</span> <span class="given-names" tagx="given-names" title="given-names">A. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ernst</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leggio</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skriver</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors: structurally distinct, functionally diverse</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">87</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2004.12.010">10.1016/j.tplants.2004.12.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15708345">15708345</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ooka</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satoh</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doi</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagata</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otomo</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murakami</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Comprehensive analysis of NAC family genes in <i>Oryza sativa</i> and <i>Arabidopsis thaliana</i> </span>. <span class="source" tagx="source" title="source">DNA Res.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">239</span>– <span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/10.6.239">10.1093/dnares/10.6.239</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15029955">15029955</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouyang</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamilton</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Childs</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">The TIGR rice genome annotation resource: improvements and new features</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">D883</span>– <span class="lpage" tagx="lpage" title="lpage">D887</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkl976">10.1093/nar/gkl976</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17145706">17145706</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paterson</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowers</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruggmann</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubchak</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimwood</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gundlach</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The <i>Sorghum bicolor</i> genome and the diversification of grasses </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">457</span>, <span class="fpage" tagx="fpage" title="fpage">551</span>– <span class="lpage" tagx="lpage" title="lpage">556</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature07723">10.1038/nature07723</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423">19189423</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perez-Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ria-o-Pachón</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corrêa</span> <span class="given-names" tagx="given-names" title="given-names">L. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kersten</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mueller-Roeber</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">PlnTFDB: updated content and new features of the plant transcription factor database</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">D822</span>– <span class="lpage" tagx="lpage" title="lpage">D827</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkp805">10.1093/nar/gkp805</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19858103">19858103</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puranik</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahu</span> <span class="given-names" tagx="given-names" title="given-names">P. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prasad</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">NAC proteins: regulation and role in stress tolerance</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">369</span>– <span class="lpage" tagx="lpage" title="lpage">381</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2012.02.004">10.1016/j.tplants.2012.02.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22445067">22445067</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramsay</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rieseberg</span> <span class="given-names" tagx="given-names" title="given-names">L. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ritland</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The correlation of evolutionary rate with pathway position in plant terpenoid biosynthesis</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">1045</span>– <span class="lpage" tagx="lpage" title="lpage">1053</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msp021">10.1093/molbev/msp021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19188263">19188263</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rana</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boogaart</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Neill</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hynes</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bent</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macpherson</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Conservation of the microstructure of genome segments in <i>Brassica napus</i> and its diploid relatives </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">725</span>– <span class="lpage" tagx="lpage" title="lpage">733</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2004.02244.x">10.1111/j.1365-313X.2004.02244.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15546355">15546355</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raven</span> <span class="given-names" tagx="given-names" title="given-names">P. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">G. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="source" tagx="source" title="source">Biology</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">McGraw Hill</span>. </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarilar</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palacios</span> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rousselet</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ridel</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falque</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eber</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Allopolyploidy has a moderate impact on restructuring at three contrasting transposable element insertion sites in resynthesized <i>Brassica napus</i> allotetraploids </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">198</span>, <span class="fpage" tagx="fpage" title="fpage">593</span>– <span class="lpage" tagx="lpage" title="lpage">604</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.12156">10.1111/nph.12156</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23384044">23384044</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmutz</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">S. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlueter</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitros</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nelson</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the palaeopolyploid soybean</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">463</span>, <span class="fpage" tagx="fpage" title="fpage">178</span>– <span class="lpage" tagx="lpage" title="lpage">183</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature08670">10.1038/nature08670</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913">20075913</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Semon</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wlofe</span> <span class="given-names" tagx="given-names" title="given-names">K. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Preferential subfunctionalization of slow-evolving genes after allopolyploydization in <i>Xenopus laevis</i> </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">105</span>, <span class="fpage" tagx="fpage" title="fpage">8333</span>– <span class="lpage" tagx="lpage" title="lpage">8338</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0708705105">10.1073/pnas.0708705105</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18541921">18541921</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Y. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Proteolytic processing of an Arabidopsis membrane-bound NAC transcription factor is triggered by cold induced changes in membrane fluidity </span>. <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="volume" tagx="volume" title="volume">427</span>, <span class="fpage" tagx="fpage" title="fpage">359</span>– <span class="lpage" tagx="lpage" title="lpage">367</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20091762">10.1042/BJ20091762</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20156199">20156199</a> </span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Signaling linkage between environmental stress resistance and leaf senescence in Arabidopsis</span>. <span class="source" tagx="source" title="source">Plant Signal. Behav.</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1564</span>– <span class="lpage" tagx="lpage" title="lpage">1566</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.4161/psb.6.10.17003">10.4161/psb.6.10.17003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21921691">21921691</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seoighe</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gehring</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Genome duplication led to highly selective expansion of the <i>Arabidopsis thaliana</i> proteome </span>. <span class="source" tagx="source" title="source">Trends Genet.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">461</span>– <span class="lpage" tagx="lpage" title="lpage">464</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tig.2004.07.008">10.1016/j.tig.2004.07.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15363896">15363896</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shan</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zahn</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guindon</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wall</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of plant MADS box transcription factors: evidence for shifts in selection associated with early angiosperm diversification and concerted gene duplications </span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">2229</span>– <span class="lpage" tagx="lpage" title="lpage">2244</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msp129">10.1093/molbev/msp129</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19578156">19578156</a> </span> </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shan</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Patterns of gene duplication and functional diversification during the evolution of the AP1/SQUA subfamily of plant MADS-box genes </span>. <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">26</span>– <span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2007.02.016">10.1016/j.ympev.2007.02.016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17434760">17434760</a> </span> </span> </li> <li tag="ref"> <a name="B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albert</span> <span class="given-names" tagx="given-names" title="given-names">V. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leebens-Mack</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">C. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paterson</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Polyploidy and angiosperm diversification</span>. <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="volume" tagx="volume" title="volume">96</span>, <span class="fpage" tagx="fpage" title="fpage">336</span>– <span class="lpage" tagx="lpage" title="lpage">348</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.0800079">10.3732/ajb.0800079</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21628192">21628192</a> </span> </span> </li> <li tag="ref"> <a name="B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Souer</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Houwelingen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kloos</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mol</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koes</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">The no apical meristem gene of petunia is required for pattern formation in embryos and flowers and is expressed at meristem and primordial boundaries </span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">85</span>, <span class="fpage" tagx="fpage" title="fpage">159</span>– <span class="lpage" tagx="lpage" title="lpage">170</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0092-8674(00)81093-4">10.1016/S0092-8674(00)81093-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8612269">8612269</a> </span> </span> </li> <li tag="ref"> <a name="B59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takeda</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanano</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kariya</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimizu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsui</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">CUP-SHAPED COTYLEDON 1 transcription factor activates the expression of LSH4 and LSH3, two members of the ALOG gene family, in shoot organ boundary cells </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">1066</span>– <span class="lpage" tagx="lpage" title="lpage">1077</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2011.04571.x">10.1111/j.1365-313X.2011.04571.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21435050">21435050</a> </span> </span> </li> <li tag="ref"> <a name="B60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tatusov</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galperin</span> <span class="given-names" tagx="given-names" title="given-names">M. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Natale</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koonin</span> <span class="given-names" tagx="given-names" title="given-names">E. V.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">The COG database: a tool for genome-scale analysis of protein functions and evolution</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">33</span>– <span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/28.1.33">10.1093/nar/28.1.33</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592175">10592175</a> </span> </span> </li> <li tag="ref"> <a name="B61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">L. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuma</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">S. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and functional analysis of Arabidopsis stress-inducible NAC transcription factors that bind to a drought responsive cis-element in the early responsive to dehydration stress 1 promoter </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">2481</span>– <span class="lpage" tagx="lpage" title="lpage">2498</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.104.022699">10.1105/tpc.104.022699</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15319476">15319476</a> </span> </span> </li> <li tag="ref"> <a name="B62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">L. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishiyama</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Potential utilization of NAC transcription factors to enhance abiotic stress tolerance in plants by biotechnological approach </span>. <span class="source" tagx="source" title="source">GM Crops</span> <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">32</span>– <span class="lpage" tagx="lpage" title="lpage">39</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.4161/gmcr.1.1.10569">10.4161/gmcr.1.1.10569</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21912210">21912210</a> </span> </span> </li> <li tag="ref"> <a name="B63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuskan</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Difazio</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansson</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grigoriev</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellsten</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">The genome of black cottonwood, <i>Populus trichocarpa</i> (Torr. &amp;amp; Gray) </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">313</span>, <span class="fpage" tagx="fpage" title="fpage">1596</span>– <span class="lpage" tagx="lpage" title="lpage">1604</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1128691">10.1126/science.1128691</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872">16973872</a> </span> </span> </li> <li tag="ref"> <a name="B64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uauy</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Distelfeld</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahima</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blechl</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubcovsky</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">A NAC gene regulating senescence improves grain protein, zinc, and iron content in wheat </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">314</span>, <span class="fpage" tagx="fpage" title="fpage">1298</span>– <span class="lpage" tagx="lpage" title="lpage">1301</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1133649">10.1126/science.1133649</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17124321">17124321</a> </span> </span> </li> <li tag="ref"> <a name="B65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vialette-Guiraud</span> <span class="given-names" tagx="given-names" title="given-names">A. C. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adam</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finet</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jasinski</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jouannic</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scutt</span> <span class="given-names" tagx="given-names" title="given-names">C. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Insights from ANA-grade angiosperms into the early evolution of CUP-SHAPED COTYLEDON genes </span>. <span class="source" tagx="source" title="source">Ann. Bot.</span> <span class="volume" tagx="volume" title="volume">107</span>, <span class="fpage" tagx="fpage" title="fpage">1511</span>– <span class="lpage" tagx="lpage" title="lpage">1519</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/aob/mcr024">10.1093/aob/mcr024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21320879">21320879</a> </span> </span> </li> <li tag="ref"> <a name="B66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Positive selection drives neofunctionalization of the UbiA prenyltransferase gene family </span>. <span class="source" tagx="source" title="source">Plant Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">383</span>– <span class="lpage" tagx="lpage" title="lpage">394</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-015-0285-2">10.1007/s11103-015-0285-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25605655">25605655</a> </span> </span> </li> <li tag="ref"> <a name="B67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Basnayake</span> <span class="given-names" tagx="given-names" title="given-names">B. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Virk</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis ATAF1, a NAC transcription factor, is a negative regulator of defense responses against necrotrophic fungal and bacterial pathogens </span>. <span class="source" tagx="source" title="source">Mol. Plant Microbe Interact.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1227</span>– <span class="lpage" tagx="lpage" title="lpage">1238</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1094/MPMI-22-10-1227">10.1094/MPMI-22-10-1227</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19737096">19737096</a> </span> </span> </li> <li tag="ref"> <a name="B68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The genome of the mesopolyploid crop species <i>Brassica rapa</i> </span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">1035</span>– <span class="lpage" tagx="lpage" title="lpage">1039</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ng.919">10.1038/ng.919</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998">21873998</a> </span> </span> </li> <li tag="ref"> <a name="B69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-wide investigation of the NAC transcription factor family in melon ( <i>Cucumis melo</i> L.) and their expression analysis under salt stress </span>. <span class="source" tagx="source" title="source">Plant Cell Rep.</span> <span class="volume" tagx="volume" title="volume">35</span>, <span class="fpage" tagx="fpage" title="fpage">1827</span>– <span class="lpage" tagx="lpage" title="lpage">1839</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-016-1997-8">10.1007/s00299-016-1997-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27229006">27229006</a> </span> </span> </li> <li tag="ref"> <a name="B70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deng</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">G. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">TaNAC8, a novel NAC transcription factor gene in wheat, responds to stripe rust pathogen infection and abiotic stresses </span>. <span class="source" tagx="source" title="source">Physiol. Mol. Plant Pathol.</span> <span class="volume" tagx="volume" title="volume">74</span>, <span class="fpage" tagx="fpage" title="fpage">394</span>– <span class="lpage" tagx="lpage" title="lpage">402</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pmpp.2010.06.005">10.1016/j.pmpp.2010.06.005</a> </span> </span> </li> <li tag="ref"> <a name="B71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frugis</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colgan</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">N. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Arabidopsis NAC1 transduces auxin signal downstream of TIR1 to promote lateral root development </span>. <span class="source" tagx="source" title="source">Genes Dev.</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">3024</span>– <span class="lpage" tagx="lpage" title="lpage">3036</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.852200">10.1101/gad.852200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11114891">11114891</a> </span> </span> </li> <li tag="ref"> <a name="B72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masatoshi</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution of F-box genes in plants: different modes of sequence divergence and their relationships with functional diversification </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span> <span class="volume" tagx="volume" title="volume">106</span>, <span class="fpage" tagx="fpage" title="fpage">835</span>– <span class="lpage" tagx="lpage" title="lpage">840</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0812043106">10.1073/pnas.0812043106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19126682">19126682</a> </span> </span> </li> <li tag="ref"> <a name="B73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubo</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukuda</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demura</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">VASCULARRELATED NAC-DOMAIN7 is involved in the differentiation of all types of xylem vessels in Arabidopsis roots and shoots </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">652</span>– <span class="lpage" tagx="lpage" title="lpage">664</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2008.03533.x">10.1111/j.1365-313X.2008.03533.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18445131">18445131</a> </span> </span> </li> <li tag="ref"> <a name="B74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">S. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">The Arabidopsis NAC transcription factor VNI2 integrates abscisic acid signals into leaf senescence via the COR/RD genes </span>. <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">2155</span>– <span class="lpage" tagx="lpage" title="lpage">2168</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.111.084913">10.1105/tpc.111.084913</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21673078">21673078</a> </span> </span> </li> <li tag="ref"> <a name="B75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution </span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">568</span>– <span class="lpage" tagx="lpage" title="lpage">573</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025957">10.1093/oxfordjournals.molbev.a025957</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9580986">9580986</a> </span> </span> </li> <li tag="ref"> <a name="B76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">1586</span>– <span class="lpage" tagx="lpage" title="lpage">1591</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msm088">10.1093/molbev/msm088</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a> </span> </span> </li> <li tag="ref"> <a name="B77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages </span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">908</span>– <span class="lpage" tagx="lpage" title="lpage">917</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a004148">10.1093/oxfordjournals.molbev.a004148</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12032247">12032247</a> </span> </span> </li> <li tag="ref"> <a name="B78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshii</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamazaki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rakwal</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kishi-Kaboshi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyao</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirochika</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The NAC transcription factor RIM1 of rice is a new regulator of jasmonate signaling</span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">804</span>– <span class="lpage" tagx="lpage" title="lpage">815</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2009.04107.x">10.1111/j.1365-313X.2009.04107.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20015061">20015061</a> </span> </span> </li> <li tag="ref"> <a name="B79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Evolution by gene duplication: an update</span>. <span class="source" tagx="source" title="source">Trends Ecol. Evol.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">292</span>– <span class="lpage" tagx="lpage" title="lpage">298</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0169-5347(03)00033-8">10.1016/S0169-5347(03)00033-8</a> </span> </span> </li> <li tag="ref"> <a name="B80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaut</span> <span class="given-names" tagx="given-names" title="given-names">B. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vision</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Gene duplication and evolution</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">293</span>: <span class="fpage" tagx="fpage" title="fpage">1551</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.293.5535.1551a">10.1126/science.293.5535.1551a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11550701">11550701</a> </span> </span> </li> <li tag="ref"> <a name="B81" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">GhNAC12, a nuetral candidate gene, leads to early aging in cotton ( <i>Gossypium hirsutum</i> L) </span>. <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">268</span>– <span class="lpage" tagx="lpage" title="lpage">274</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2015.10.042">10.1016/j.gene.2015.10.042</a> </span> </span> </li> <li tag="ref"> <a name="B82" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">Z. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010a</span>). <span class="mixed-article-title" title="mixed-article-title">Global analysis of direct targets of secondary wall NAC master switches in Arabidopsis</span>. <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">1087</span>– <span class="lpage" tagx="lpage" title="lpage">1103</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/mp/ssq062">10.1093/mp/ssq062</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20935069">20935069</a> </span> </span> </li> <li tag="ref"> <a name="B83" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">Z. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010b</span>). <span class="mixed-article-title" title="mixed-article-title">Evolutionary conservation of the transcriptional network regulating secondary cell wall biosynthesis </span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">625</span>– <span class="lpage" tagx="lpage" title="lpage">632</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2010.08.007">10.1016/j.tplants.2010.08.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20833576">20833576</a> </span> </span> </li> <li tag="ref"> <a name="B84" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhong</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">Z. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Dissection of the transcriptional program regulating secondary wall biosynthesis during wood formation in poplar </span>. <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="volume" tagx="volume" title="volume">157</span>, <span class="fpage" tagx="fpage" title="fpage">1452</span>– <span class="lpage" tagx="lpage" title="lpage">1468</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.111.181354">10.1104/pp.111.181354</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21908685">21908685</a> </span> </span> </li> <li tag="ref"> <a name="B85" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">A new tomato NAC (NAM/ATAF1/2/CUC2) transcription factor, SlNAC4, functions as a positive regulator of fruit ripening and carotenoid accumulation </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol.</span> <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">119</span>– <span class="lpage" tagx="lpage" title="lpage">135</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pct162">10.1093/pcp/pct162</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24265273">24265273</a> </span> </span> </li> <li tag="ref"> <a name="B86" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nevo</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analyses unravel the evolutionary history of NAC proteins in plants</span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">1833</span>– <span class="lpage" tagx="lpage" title="lpage">1848</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1558-5646.2011.01553.x">10.1111/j.1558-5646.2011.01553.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22671550">22671550</a> </span> </span> </li> </ul> </div> </div>  </body></html>