<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4773123">4773123</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0149495">10.1371/journal.pone.0149495</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-15-26125</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene regulation</span>: <span class="subject" title="subject">MicroRNAs</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">MicroRNAs</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Anatomy</span>: <span class="subject" title="subject">Seeds</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span>: <span class="subject" title="subject">Medicinal Plants</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene Expression</span>: <span class="subject" title="subject">Gene Regulation</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span>: <span class="subject" title="subject">Plants</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Gene Expression</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span>: <span class="subject" title="subject">Plant Genomics</span> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
MicroRNA from <i>Moringa oleifera</i>: Identification by High Throughput Sequencing and Their Potential Contribution to Plant Medicinal Value </div> <div class="alt-title" title="alt-title">
MicroRNAs in <i>Moringa</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pirrò</span> <span class="given-names" tagx="given-names" title="given-names">Stefano</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff005"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanella</span> <span class="given-names" tagx="given-names" title="given-names">Letizia</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kenzo</span> <span class="given-names" tagx="given-names" title="given-names">Maurice</span> </span> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montesano</span> <span class="given-names" tagx="given-names" title="given-names">Carla</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minutolo</span> <span class="given-names" tagx="given-names" title="given-names">Antonella</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Potestà</span> <span class="given-names" tagx="given-names" title="given-names">Marina</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sobze</span> <span class="given-names" tagx="given-names" title="given-names">Martin Sanou</span> </span> <a href="#aff003"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canini</span> <span class="given-names" tagx="given-names" title="given-names">Antonella</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cirilli</span> <span class="given-names" tagx="given-names" title="given-names">Marco</span> </span> <a href="#aff006"> <sup>6</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muleo</span> <span class="given-names" tagx="given-names" title="given-names">Rosario</span> </span> <a href="#aff006"> <sup>6</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colizzi</span> <span class="given-names" tagx="given-names" title="given-names">Vittorio</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff005"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galgani</span> <span class="given-names" tagx="given-names" title="given-names">Andrea</span> </span> <a href="#aff004"> <sup>4</sup> </a> <a href="#aff005"> <sup>5</sup> </a> <a href="#cor001">*</a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Department of Biology, University of Rome "Tor Vergata", Rome, Italy</span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">SOCOPOMO S/C 46 P.A. de Lingang, Dschang, Cameroon</span> </span> <span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">University of Dschang, Dschang, Cameroon</span> </span> <span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Centro di Servizi Interdipartimentale, Stazione per la Tecnologia Animale, University of Rome‘‘Tor Vergata", Rome, Italy </span> </span> <span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Mir-Nat s.r.l., Rome, Italy</span> </span> <span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Department of Agricultural and Forest Science, University of Tuscia, Viterbo, Italy</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Yun</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Kunming University of Science and Technology, CHINA</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-conflict" title="conflict"> <p> <b>Competing Interests: </b>VC, SP and AG are partners in a Startup company: Mir-Nat s. r. l., Rome, Italy, who provided research materials for this study. This does not alter the authors' adherence to PLOS ONE policies on sharing data and materials. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: AG SP VC. Performed the experiments: MP CM AM. Analyzed the data: SP LZ. Contributed reagents/materials/analysis tools: VC AC. Wrote the paper: AG SP RM MC. Collected biological samples: MK SS. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">galgani@scienze.uniroma2.it</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-3-3</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0149495</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-6-15</span> <span class="accepted" title="accepted">accepted: 2016-2-2</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Pirrò et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0149495.pdf" title="self-uri">pone.0149495.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Moringa oleifera</i> is a widespread plant with substantial nutritional and medicinal value. We postulated that microRNAs (miRNAs), which are endogenous, noncoding small RNAs regulating gene expression at the post-transcriptional level, might contribute to the medicinal properties of plants of this species after ingestion into human body, regulating human gene expression. However, the knowledge is scarce about miRNA in <i>Moringa</i>. Furthermore, in order to test the hypothesis on the pharmacological potential properties of miRNA, we conducted a high-throughput sequencing analysis using the Illumina platform. A total of 31,290,964 raw reads were produced from a library of small RNA isolated from <i>M</i>. <i>oleifera</i> seeds. We identified 94 conserved and two novel miRNAs that were validated by qRT-PCR assays. Results from qRT-PCR trials conducted on the expression of 20 <i>Moringa</i> miRNA showed that are conserved across multiple plant species as determined by their detection in tissue of other common crop plants. <i>In silico</i> analyses predicted target genes for the conserved miRNA that in turn allowed to relate the miRNAs to the regulation of physiological processes. Some of the predicted plant miRNAs have functional homology to their mammalian counterparts and regulated human genes when they were transfected into cell lines. To our knowledge, this is the first report of discovering <i>M</i>. <i>oleifera</i> miRNAs based on high-throughput sequencing and bioinformatics analysis and we provided new insight into a potential cross-species control of human gene expression. The widespread cultivation and consumption of <i>M</i>. <i>oleifera</i>, for nutritional and medicinal purposes, brings humans into close contact with products and extracts of this plant species. The potential for miRNA transfer should be evaluated as one possible mechanism of action to account for beneficial properties of this valuable species. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
The authors received no specific funding for this work. Mir-Nat s. r. l. provided support in the form of salaries for authors SP, VC and AG and provided support in the form of research materials, but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the ‘author contributions’ section. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="Data has been deposited in NCBI/GEO public database under accession number GSE70423." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Data has been deposited in NCBI/GEO public database under accession number GSE70423.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p> <i>Moringa oleifera</i> Lam, a naturalized species from the monogenus family <i>Moringaceae</i>, is one of the best known, most widely distributed and most useful nutritional and medicinal plants [ <a href="#pone.0149495.ref001">1</a>– <a href="#pone.0149495.ref003">3</a>]. Several organs of the <i>Moringa</i> tree are edible (e.g., pods, seeds, flowers and leaves) and are used in many countries (including many regions of Africa) for their high nutritional value [ <a href="#pone.0149495.ref001">1</a>– <a href="#pone.0149495.ref002">2</a>]. Almost all tissues of this plant can be used in the treatment of inflammation or infectious diseases along with cardiovascular, gastrointestinal, haematological and neoplastic diseases [ <a href="#pone.0149495.ref002">2</a>– <a href="#pone.0149495.ref004">4</a>]. The leaves are a source of natural antioxidants [ <a href="#pone.0149495.ref005">5</a>], vitamins A, B and C, minerals, proteins and essential amino acids [ <a href="#pone.0149495.ref001">1</a>– <a href="#pone.0149495.ref002">2</a>]. The Italian Ministry of Health in compliance with the European Pharmaceutical Plant legislation has included <i>Moringa</i> seeds in the "List of Plant and Vegetal Integrators". </p> <p>In 2014, Jung IL reported a tumor suppressor activity in in mammalian cells treated with cold water-soluble extract of <i>M</i>. <i>oleifera</i> leaves [ <a href="#pone.0149495.ref006">6</a>]. The author found an abnormal ribosomal RNA (rRNA) pattern and down-regulation of many genes and proteins involved in cell transformation and proliferation in mammalian cells treated with this extract. He concluded that the cold water-soluble extract of <i>M</i>. <i>oleifera</i> induced rRNA degradation. In 2015, Tian and coworkers reported a high-quality draft genome sequence of <i>M</i>. <i>oleifera</i> and compared the genome to related woody plant genomes in order to clarify the derivation of this species [ <a href="#pone.0149495.ref007">7</a>]. </p> <p>Plant miRNAs are a class of 18–24 nucleotide (nt) small, non-coding RNA that negatively regulate specific messenger RNA (mRNA). MiRNAs operate in a sequence-specific manner and silence specific protein-coding genes at the post-transcriptional level by targeting the 3’ untranslated region (3’UTR) of mRNA [ <a href="#pone.0149495.ref008">8</a>]. This process causes mRNA cleavage and decreases protein translation [ <a href="#pone.0149495.ref009">9</a>– <a href="#pone.0149495.ref011">11</a>]. In general, miRNAs are key regulators of development, stress response, growth and other important physiological processes [ <a href="#pone.0149495.ref012">12</a>– <a href="#pone.0149495.ref013">13</a>]. </p> <p>In 2012, Zhang and collaborators demonstrated that <i>osa</i>- <i>miR-168a</i> and other exogenous miRNAs that are abundant in rice plants can be acquired by mice through food intake, as evidenced by their presence in sera or tissues of the mammalian. <i>In vitro</i> and <i>in vivo</i> functional studies showed that these exogenous miRNAs are able to inhibit mammalian gene expression in the liver, demonstrating the first case of cross-kingdom regulation [ <a href="#pone.0149495.ref014">14</a>]. More recently, oral administered cocktails of endogenous tumor suppressor miRNAs, exhibiting characteristics of plant miRNAs, reduced tumor burden in a mouse model of colon cancer [ <a href="#pone.0149495.ref015">15</a>]. These observations show that plant miRNAs are absorbed in the mammalian digestive tract and can target mammalian genes. Moreover, they suggest the hypothesis that engineered edible plants producing mammalian tumor suppressor miRNAs might be a new treatment modality for cancer. Such treatment might be an effective, nontoxic, and inexpensive chemo-preventive strategy for human. </p> <p>Recently, Shu and collaborators presented an integrative study where comparative analysis and computational prediction have been applied to assess the cross-species transportation of miRNAs, particularly focusing on inferring the likelihood of exogenous miRNA in human circulation [ <a href="#pone.0149495.ref016">16</a>]. </p> <p>This work demonstrated the data-driven computational analysis is highly promising approach to study novel molecular characteristics of deliverable miRNAs allowing to bypass the complex mechanistic details. </p> <p>Several miRNAs discovery methods including computational prediction, cloning strategies and others have been used [ <a href="#pone.0149495.ref017">17</a>– <a href="#pone.0149495.ref019">19</a>], even though these methods demand an increased rigor in miRNA annotations [ <a href="#pone.0149495.ref020">20</a>]. High-throughput sequencing technologies have contributed markedly to the expansion of knowledge about the miRNA universe in eukaryotic cells. These technologies have revealed a number of newly evolved and species-specific miRNAs that were previously unknown [ <a href="#pone.0149495.ref021">21</a>– <a href="#pone.0149495.ref022">22</a>]. </p> <p>With the effort to define medicinal factors found in this interesting plant species, we searched for understanding better genome expression and gene regulation by micro RNA (miRNA) in <i>M</i>. <i>oleifera</i> Lam. We recognize that seeds contain all information about the tissues that will develop in the adult plant; therefore, we have programmed to analyze the miRNA populations obtained from seeds. In this work, we have identified novel miRNAs and their potential target genes from <i>M</i>. <i>oleifera</i> seeds using Illumina platform technologies. We found 94 conserved miRNAs and two novel ones, and some of them were validated by qRT-PCR. Target genes were predicted by <i>in silico</i> and results indicate that <i>mol</i>-miRNAs are putatively involved in many physiological process. A selected number of miRNAs were compared to other crop species plants, resulting to be conserved across multiple species. By taking advantage of a recently developed web-application based on an algorithm that compares plant and mammalian miRNAs ( <a href="http://160.80.35.140/MirCompare">http://160.80.35.140/MirCompare</a>), we have identified a few <i>M</i>. <i>oleifera miRNAs</i> with functional homologies to mammalian ones. We conducted a preliminary analysis to investigate potential human gene regulation by the plant miRNA mimics. The reported analysis increases the information on plant miRNAs currently available and improves the knowledge on to the molecular mechanisms associated to nutritional and medicinal activities of this plant species. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterials,samplecollectionandrnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Plant materials, sample collection and RNA extraction</div> <p>Seeds collected from mature pods, before they split open and fall to the ground, harvested from <i>M</i>. <i>oleifera</i> trees grown in the District of Dschang in West Cameroon, without use of chemical fertilizer, by the Cooperative of Medical Plant Producers SOCOPOMO. Seeds were stored at -80°C until used. Seeds were germinated in a greenhouse at the Department of Biology, University of "Tor Vergata", Rome, by placing them on paper soaked in sterile water. Leaves, stems and roots were collected at one month from the beginning of germination, immediately frozen in liquid nitrogen and subsequently stored at -80°C until used. Tissues from other species ( <i>Solanum tuberosa</i>, <i>Olea europaea</i> and <i>Medicago sativa</i>) grown under natural field conditions, were also stored at -80°C until used. Total RNA was extracted from plant tissues using the mirVana kit (Ambion, USA) according to the manufacturer’s protocol. The assessment of RNA quality and quantity were evaluated by Agilent 2100 Bioanalyzer (Agilent Technologies, Waldbroon, Germany) and by spectrophotometry (SmartSpec Plus, Bio-Rad, USA), respectively. </p> </div> <div class="constructingandsequencingsmallrnalibraries" title="sec"> <div class="title" tagx="title" title="title">
Constructing and sequencing small RNA libraries</div> <p>Quality control for next generation sequencing experiments was performed by Genomix4life S.r.l. (Baronissi, Salerno, Italy). Individual indexed libraries were prepared from 1 μg of purified RNA using the TruSeq SmallRNA Sample Prep Kit (Illumina, USA) according to the manufacturer’s instructions. Libraries were quantified using the Agilent 2100 Bioanalyzer (Agilent Technologies, Waldbroon, Germany) and pooled so that each index-tagged sample was present in equimolar amounts with final concentrations of the pooled samples adjusted to 2 nM. The pooled samples were subject to cluster generation and sequencing using an Illumina HiSeq 2500 System (Illumina, USA) in a 1x50 single read format at a final concentration of 10 pmol. The raw sequence files generated underwent quality control analyses using FastQC [ <a href="#pone.0149495.ref023">23</a>]. This method offers a powerful means for quantitative and qualitative profiling small RNA populations in different plant species, for which limited genome information is available, such as <i>M</i>. <i>oleifera</i>. </p> </div> <div class="sequencedataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence data analysis</div> <p>Several bioinformatics tools have been developed to identify conserved miRNAs and discover new ones, starting from high-throughput sequencing [ <a href="#pone.0149495.ref024">24</a>– <a href="#pone.0149495.ref027">27</a>]. The biggest limitation of these algorithms is that genome sequencing is required to execute the analysis. The only published algorithm able to discover novel miRNAs in species without sequenced genome is miReader [ <a href="#pone.0149495.ref028">28</a>]. Jha and coworkers also presented an approach for identifying novel miRNAs in <i>Miscanthus giganteus</i>, whose genome has not been sequenced yet [ <a href="#pone.0149495.ref028">28</a>]. Although the <i>M</i>. <i>oleifera</i> genome was already sequenced in 2015 [ <a href="#pone.0149495.ref007">7</a>], the raw data are not yet available, therefore we proceeded with a genome-independent strategy based on Jha’s work. </p> <p> <a href="#pone.0149495.s001">S1 Fig</a> summarizes the analysis workflow performed with small RNA raw data. This analysis was performed to identify a pool of RNA fragments with lengths between 18 and 24 nucleotides that would be added to the subsets of known or uncharacterized miRNAs. These data have been deposited in NCBI/GEO public database under accession number GSE70423. </p> <div class="rawdatafiltering" title="sec"> <div class="title" tagx="title" title="title">
Raw data filtering</div> <p>Starting from a total of 31,290,964 reads the Illumina small-RNA adapter was first clipped using FASTX-Toolkit ( <a href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a>). Sequences shorter than 15nt after trimming were discarded and we obtained a total of 22,737,895 reads. Sequencing artifacts were removed and the remaining reads were trimmed on the 3p and 5p ends in order to remove low-quality bases. At the end of this procedure BLASTn [ <a href="#pone.0149495.ref029">29</a>] v 2.2.30 with parameters "-task megablast, -perc_identity 100" was used to compare the remaining reads with the Rfam database (Rfam 11.0) [ <a href="#pone.0149495.ref030">30</a>]. This stringent step removed non-coding RNA (rRNA, tRNA, snRNA, snoRNA) and degraded fragments of mRNA. Reads with lengths less than 15 nucleotides were discarded. After the alignment process, a total of 10,415,180 reads were identified as possible miRNA sequences. </p> </div> <div class="identificationofknownandputativenovelmirnas" title="sec"> <div class="title" tagx="title" title="title">
Identification of known and putative novel miRNAs</div> <p>The 10,415,180 reads highlighted as possible miRNAs were aligned against a specific-plant miRBASE (Release 21) with a low-redundancy rate ( <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>) using BLASTn v 2.2.30 [ <a href="#pone.0149495.ref029">29</a>] with parameters "-task blastn-short, -perc_identity 100". Sequences with homology rate equal to 100% were considered conserved <i>M</i>. <i>oleifera</i> miRNAs. In the alignment process all reads with abundance values lower than 10 copies were discarded resulting in a total of 303,872 reads. </p> <p>In order to identify uncharacterized putative novel miRNAs, all the discarded reads, generated from the alignment with the wide libraries of miRNAs present in the repertory of miRBase bank, have been selected by a homemade custom script and were considered as candidate novel miRNAs. </p> <p>Subsequently, the miReader algorithm [ <a href="#pone.0149495.ref028">28</a>] has been used to identify a strict number of uncharacterized miRNAs without the support of a reference genome. Although, this algorithm is less qualitatively informative (i.e. prediction of secondary structures) than others algorithms, it can be considered the most effective quantitative analysis with unknown genome, as is the case of <i>M</i>. <i>oleifera</i>. </p> </div> <div class="qualitycontrols" title="sec"> <div class="title" tagx="title" title="title">
Quality controls</div> <p>Quality controls were performed using FastQC software [ <a href="#pone.0149495.ref023">23</a>], before ( <a href="#pone.0149495.s003">S1 File</a>) and after ( <a href="#pone.0149495.s003">S1 File</a>) removing the sequencing adapter. Figures A and B in <a href="#pone.0149495.s003">S1 File</a> show the quality scores across all bases and sequences in more detail, highlighting the validity of sequencing and analysis processes. The Illumina small-RNA adapter inside the sequenced sample is clearly visible. (Figure C in <a href="#pone.0149495.s003">S1 File</a>). In Figures A and B in <a href="#pone.0149495.s004">S2 File</a> is visible how sequencing quality was preserved through the filtering process, the complete removal of the small-RNA adapter (Figure C in <a href="#pone.0149495.s004">S2 File</a>) and the removal of all ncRNAs that do not belong to miRNAs. </p> </div> <div class="predictingconservedmirnasacrossotherplantspecies" title="sec"> <div class="title" tagx="title" title="title">
Predicting conserved miRNAs across other plant species</div> <p>In order to evaluate the conservation rate for <i>M</i>. <i>oleifera</i> miRNAs across all others plant species, we developed a custom script, which retrieves the sequence information for plant organisms stored inside MirBase repository and evaluates the overlapping rate in terms of sequence homology between <i>M</i>. <i>oleifera</i> and all other plant miRNAs. </p> </div> </div> <div class="targetprediction" title="sec"> <div class="title" tagx="title" title="title">
Target prediction</div> <div class="predictingmrnatargetsinplantsforbothknownanduncharacterizedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Predicting mRNA targets in plants for both known and uncharacterized miRNAs</div> <p> <a href="#pone.0149495.s002">S2 Fig</a> shows the process to predict gene target for known and novel-putative miRNA. Starting from the list of 74 known, psRNATarget [ <a href="#pone.0149495.ref031">31</a>] was used to predict target genes in the plant. To reduce the false-positive rate we set the maximum expectation value to 2.0 and forced the system to select only the best five predictions for each miRNA. After the target prediction analysis, the three gene sets were submitted to PlantGSEA [ <a href="#pone.0149495.ref032">32</a>] to obtain a smaller number of gene clusters that are related to plant molecular processes. </p> </div> <div class="predictingknownmirnawithputativerolesinhumangeneregulation" title="sec"> <div class="title" tagx="title" title="title">
Predicting known miRNA with putative roles in human gene regulation</div> <p>Several <i>Moringa</i> miRNAs were further analyzed to investigate their possible roles in human gene regulation. MirCompare ( <a href="http://160.80.35.140/MirCompare">http://160.80.35.140/MirCompare</a>) is a web-application based on an algorithm that compares libraries of miRNAs belonging to organisms from plant and animal kingdoms in order to find cross-kingdom regulation. The main settings for MirCompare analysis are: 1) r-value: the best comparison rate between each miRNA couple; 2) seed-region stringency: the minimum number of matches, related to the seed region. </p> <p>MirCompare was used with r-value = 0.55 and seed-region threshold = 5 to identify possible <i>Moringa</i> miRNAs with functional homologies to mammalian miRNAs. The combinatorial miRNA target prediction (COMIR) web tool [ <a href="#pone.0149495.ref033">33</a>], which combines four popular scoring schemes (miRanda, PITA, TargetScan and mirSVR), was used to compute the potential of a gene to be targeted by a set of miRNAs. Therefore, COMIR tool generated a list of genes that might be regulated by <i>M</i>. <i>oleifera</i> miRNAs. Subsequently, the identified genes were submitted to Gene Ontology enrichment analysis to understand the pathways, in which miRNAs might putatively play a regulating role. </p> </div> <div class="transfection" title="sec"> <div class="title" tagx="title" title="title">
Transfection</div> <p>We studied biological activity of the synthetic <i>mol-miR168a</i> (UCGCUUGGUGCAGGUCGGGAC) that was transfected into the hepatocellular carcinoma cell line (HEP-G2) by the lipofectamin method (Hi-Fect, Qiagen, USA) according with manufacture’s instruction (MiRNA mimic and inhibitor experiments protocols®, Qiagen, USA). Synthetic <i>mol-miR168a</i> tagged with fluorescein isothiocyanate (FITC) was used also as control of transfection. Cells were harvested 72h after transfection and characterized for the efficiency of miRNA uptake by EVOS FLoid cell imaging station (LifeTechnologies, USA) analysis, and for the effect of miRNAs on specific target genes by Western Blot assay. </p> </div> <div class="westernblotanalysis" title="sec"> <div class="title" tagx="title" title="title">
Western blot analysis</div> <p>Aliquots of 3x10 <sup>6</sup> transfected cells were suspended in buffers for cytoplasmic or nuclear protein extraction, and further processed for western blot analysis as already described [ <a href="#pone.0149495.ref034">34</a>]. Primary antibodies included the rabbit monoclonal antibody <i>SIRT1</i> and the goat monoclonal antibody human <i>beta-actin</i> (Santa Cruz biotechnology, CA, USA). Secondary antibodies included anti-goat and anti-rabbit IgG chain-specific antibodies that were conjugated to peroxidase (Calbiochem, Merck Millipore, Darmstadt, Germany). Western blot analysis for each sample was quantified by densitometry analysis (TINA software) </p> <p>Data analysis was performed using GraphPad Prism version 6.00 (GraphPad Software, La Jolla, CA, USA). Statistical probabilities were expressed as p &amp;lt;0.05 (*) or p &amp;lt; 0.01 (**). Comparison of means of <i>SIRT-1</i> protein expression in response to <i>mol-miR168a</i> transfection was carried out using t-Test analysis. </p> </div> </div> <div class="experimentalvalidationofconservedmirnasbyquantitativereal-timepcr(qrt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Experimental validation of conserved miRNAs by Quantitative Real-Time PCR (qRT-PCR)</div> <p>The expression level of different miRNAs was validated using poly(A)-tailed qRT-PCR method. According to the manufacturer’s protocols (Exiqon A/S, Vedbaek, Denmark), a poly-A tail was added to the mature miRNA templates (20 ng). cDNA was synthesized using a poly-T primer with a 3’ degenerate anchor and a 5’ universal tag at 42°C for 60 min followed by heat-inactivated for 5 min at 95°C. To provide a control for quality of the cDNA synthesis reaction and the PCR, RNA spike-in ( <i>UniSp6</i>) was added to the sample prior to cDNA synthesis. The cDNA template was then amplified using miRNA-specific and LNA™-enhanced forward and reverse primers. SYBR®Green was used for detection. The reactions were carried out in a Rotor-Gene®Q 72-Well Rotor (Qiagen, USA) with the following amplification conditions: activation/denaturation at 95°C for 10 min followed by 40 cycles of denaturation at 95°C for 10s, annealing and extension together at 60°C for 60 sec. Finally, melting analyses were performed to confirm the absence of false-positive peaks. qRT-PCR was performed only for conserved miRNAs. All reactions were performed in triplicate for each sample. Two controls (no template control and no Reverse Transcription control) were included in all reactions. Relative expression levels of miRNAs were quantified by using the 2 <sup>-Δ ΔCt</sup> method and <i>miR159</i> was used as the internal control miRNA. To determine significant differences among samples or miRNAs we applied a One Way ANOVA analysis using GraphPad Prism version 6.00. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="analyzingsmallrnapopulations" title="sec"> <div class="title" tagx="title" title="title">
Analyzing small RNA populations</div> <p>sRNAs from <i>M</i>. <i>oleifera</i> seeds with 5’-phosphate and 3’-hydroxyl groups were identified by high-throughput Illumina TruSeq smallRNA sequencing (Illumina HiSeq 1500). cDNA libraries were constructed from seeds of <i>M</i>. <i>oleifera</i> plants after removing the 5 bp adapter sequence and filtering out low quality ‘‘n" sequences. One small RNA library was constructed with about 30 million reads. Reads cleaned from adapters, ranging from 14 to 51 nts in length, were filtered with the Rfam database [ <a href="#pone.0149495.ref030">30</a>]. Looking at the sequence distribution after the filtering process, the majority of reads were 18 to 25 nts in length. The main size groups were 21 nt and 24 nt, respectively ( <a href="#pone.0149495.g001">Fig 1</a>). These results were consistent with previous studies in other plant species where 24 nt small RNAs were the most abundant [ <a href="#pone.0149495.ref019">19</a>]. The amounts of 21 nt and 24 nt small RNAs were approximately 18.15% and 16.06%, respectively. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Length distribution of small RNA populations.</div> <p>Sequence length distributions of sRNAs in <i>M</i>. <i>oleifera</i> seeds filtered by RFAM database. </p>   </div> </div> <div class="identifyingconservedm.oleifera-specificmirnas" title="sec"> <div class="title" tagx="title" title="title">
Identifying conserved <i>M</i>. <i>oleifera</i>-specific miRNAs </div> <p>Conserved families of miRNAs are present in many plant species due to their important regulatory role played. The highly conserved nature gives the opportunity to identify miRNAs in plant species for which genome sequence information is partially or fully available, or previously undefined. To identify the conserved miRNAs in <i>M</i>. <i>oleifera</i> seed plants, the massive dataset was compared to known plant mature miRNAs against miRBASE (Release 21), using BLASTn. The sequence analysis revealed the presence of 94 miRNAs belonging to 40 conserved families ( <a href="#pone.0149495.t001">Table 1</a>). Deeper investigations conducted by miRBASE also identify secondary structures for most of the known <i>M</i>. <i>oleifera</i> miRNA sequences ( <a href="#pone.0149495.s005">S1 Table</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t001">10.1371/journal.pone.0149495.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Known microRNAs in <i>M</i>. <i>oleifera</i> seeds. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>miRNA family</th> <th>miRNA members</th> <th>miRNA sequence</th> <th>miRNA* sequence</th> <th>Read Counts</th> </tr> <tr> <th>miR</th> <th>miR*</th> </tr> </thead> <tbody> <tr> <td> <i>mol-miR156</i> </td> <td> <i>mol-miR156</i> </td> <td>CUGACAGAAGAGAGUGAGCAC</td> <td /> <td>2459</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR156d</i> </td> <td /> <td>GCUCUCUAUGCUUCUGUCAUCA</td> <td /> <td>23</td> </tr> <tr> <td /> <td> <i>mol-miR156f</i> </td> <td>CUGACAGAAGAGAGUGAGCA</td> <td>CUCACUUCUCUUUCUGUCAAUC</td> <td>86</td> <td>34</td> </tr> <tr> <td /> <td> <i>mol-miR156g</i> </td> <td>CGACAGAAGAGAGUGAGCAC</td> <td /> <td>77</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR156h</i> </td> <td>UGACAGAAGAAAGAGAGCAC</td> <td>GCUCUCUUUCCUUCUGCCACC</td> <td>33</td> <td>NA</td> </tr> <tr> <td /> <td> <i>mol-miR156j</i> </td> <td>GUUGACAGAAGAGAGUGAGCAC</td> <td /> <td>2546</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR156q</i> </td> <td>UGACAGAAGAGAGUGAGCACU</td> <td /> <td>2476</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR156t</i> </td> <td>UUGACAGAAGAGAGAGAGCAC</td> <td /> <td>111</td> <td /> </tr> <tr> <td> <i>mol-miR157</i> </td> <td> <i>mol-miR157a</i> </td> <td>UUGACAGAAGAUAGAGAGCAC</td> <td>GCUCUCUAGCCUUCUGUCAUCA</td> <td>316</td> <td>NA</td> </tr> <tr> <td /> <td> <i>mol-miR157d</i> </td> <td>UGACAGAAGAUAGAGAGCAC</td> <td>GCUCUCUAUGCUUCUGUCAUC</td> <td>NA</td> <td>23</td> </tr> <tr> <td> <i>mol-miR159</i> </td> <td> <i>mol-miR159</i> </td> <td>AGCUCCCUUCGAUCCAAUC</td> <td>CUUGGAUUGAAGGGAGCUCU</td> <td>NA</td> <td>48</td> </tr> <tr> <td /> <td> <i>mol-miR159a</i> </td> <td>UUUGGAUUGAAGGGAGCUCUA</td> <td /> <td>11908</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR159b</i> </td> <td /> <td>UUUGGAUUGAAGGGAGCUCUU</td> <td /> <td>2555</td> </tr> <tr> <td /> <td> <i>mol-miR159b</i>. <i>1</i> </td> <td /> <td>UUUGGAUUGAAGGGAGCUCUG</td> <td /> <td>2492</td> </tr> <tr> <td /> <td> <i>mol-miR159c</i> </td> <td /> <td>UUUGGAUUGAAGGGAGCUCCU</td> <td /> <td>1917</td> </tr> <tr> <td /> <td> <i>mol-miR159d</i> </td> <td>AUUGGAUUGAAGGGAGCUCCG</td> <td /> <td>21</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR159f</i> </td> <td>CUUGGAUUGAAGGGAGCUCUA</td> <td /> <td>442</td> <td /> </tr> <tr> <td> <i>mol-miR160</i> </td> <td> <i>mol-miR160h</i> </td> <td>UGCCUGGCUCCCUGUAUGCCAUU</td> <td /> <td>25</td> <td /> </tr> <tr> <td> <i>mol-miR162</i> </td> <td> <i>mol-miR162</i> </td> <td /> <td>UCGAUAAACCUCUGCAUCCAG</td> <td /> <td>158</td> </tr> <tr> <td /> <td> <i>mol-miR162a</i> </td> <td>UCGAUAAACCUCUGCAUCCA</td> <td /> <td>16</td> <td /> </tr> <tr> <td> <i>mol-miR164</i> </td> <td> <i>mol-miR164a</i> </td> <td>UGGAGAAGCAGGGCACGUGAA</td> <td /> <td>13</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR164c</i> </td> <td>UGGAGAAGCAGGGCACGUGCG</td> <td /> <td>13</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR164d</i> </td> <td>UGGAGAAGCAGGGCACGUGCA</td> <td /> <td>373</td> <td /> </tr> <tr> <td> <i>mol-miR165</i> </td> <td> <i>mol-miR165a</i> </td> <td>UCGGACCAGGCUUCAUCCCCC</td> <td /> <td>368</td> <td /> </tr> <tr> <td> <i>mol-miR166</i> </td> <td> <i>mol-miR166</i> </td> <td>CCGGACCAGGCUUCAUCCCAG</td> <td /> <td>14</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166b</i> </td> <td>UCGGACCAGGCUUCAUUCCCUU</td> <td /> <td>2367</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166e</i> </td> <td>GGACCAGGCUUCAUUCCCC</td> <td /> <td>5541</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166h</i> </td> <td>UCGGACCAGGCUUCAUUCCCGU</td> <td /> <td>2923</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166i</i> </td> <td>UCGGACCAGGCUUCAUUCCCCC</td> <td /> <td>65243</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166j</i> </td> <td>UCCGGACCAGGCUUCAUUCCC</td> <td /> <td>546</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR166k</i> </td> <td>GGAUUGUUGUCUGGCUCGGUG</td> <td>UCGGACCAGGCUUCAAUCCCU</td> <td>NA</td> <td>240</td> </tr> <tr> <td /> <td> <i>mol-miR166u</i> </td> <td>UCUCGGACCAGGCUUCAUUC</td> <td /> <td>3738</td> <td /> </tr> <tr> <td> <i>mol-miR167</i> </td> <td> <i>mol-miR167</i> </td> <td>UCAAGCUGCCAGCAUGAUCUA</td> <td>AGAUCAUGUGGCAGUUUCACC</td> <td>27</td> <td>514</td> </tr> <tr> <td /> <td> <i>mol-miR167a</i> </td> <td>UGAAGCUGCCAGCAUGAUCUC</td> <td /> <td>3881</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR167b</i> </td> <td>UGAAGCUGCCAGCAUGAUCUA</td> <td /> <td>3834</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR167c</i> </td> <td>UGAAGCUGCCAGCAUGAUCUGG</td> <td /> <td>3908</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR167c</i> </td> <td>UAAGCUGCCAGCAUGAUCUUG</td> <td>UAGGUCAUGCUGGUAGUUUCACC</td> <td>271</td> <td>NA</td> </tr> <tr> <td /> <td> <i>mol-miR167d</i> </td> <td>UGAAGCUGCCAGCAUGAUCUGA</td> <td /> <td>4007</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR167h</i> </td> <td>UGAAGCUGCCAGCAUGAUCUUA</td> <td>AGAUCAUGUGGCAGUUUCACC</td> <td>18563</td> <td>NA</td> </tr> <tr> <td /> <td> <i>mol-miR167i</i> </td> <td>UCAUGCUGGCAGCUUCAACUGGU</td> <td /> <td>586</td> <td /> </tr> <tr> <td> <i>mol-miR168</i> </td> <td> <i>mol-miR168</i> </td> <td>AUUCAGUUGAUGCAAGGCGGGAUC</td> <td /> <td>91</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR168a</i> </td> <td>UCGCUUGGUGCAGGUCGGGAC</td> <td /> <td>165</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR168c</i> </td> <td>UCGCUUGGUGCAGGUCGGGAC</td> <td>CCCGCCUUGCAUCAACUGAAU</td> <td>NA</td> <td>91</td> </tr> <tr> <td /> <td> <i>mol-miR168d</i> </td> <td>UCGCUUGGUGCAGGUCGGGAA</td> <td>CCCGCCUUGCAUCAACUGAAU</td> <td>517</td> <td>NA</td> </tr> <tr> <td> <i>mol-miR169</i> </td> <td> <i>mol-miR169d</i> </td> <td>UAGCCAAGGAUGACUUGCCU</td> <td /> <td>22</td> <td /> </tr> <tr> <td> <i>mol-miR170</i> </td> <td> <i>mol-miR170</i> </td> <td>UAUUGGCCUGGUUCACUCAGA</td> <td>UGAUUGAGCCGUGUCAAUAUC</td> <td>NA</td> <td>77</td> </tr> <tr> <td> <i>mol-miR171</i> </td> <td> <i>mol-miR171a</i> </td> <td>UGAUUGAGCCGUGCCAAUAU</td> <td /> <td>423</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR171c</i> </td> <td>UAUUGACGCGGUUCAAUUCGA</td> <td>UGAUUGAGCCGUGCCAAUAUC</td> <td>NA</td> <td>503</td> </tr> <tr> <td /> <td> <i>mol-miR171d</i> </td> <td /> <td>UUGAGCCGUGCCAAUAUCACG</td> <td /> <td>410</td> </tr> <tr> <td> <i>mol-miR172</i> </td> <td> <i>mol-miR172m</i> </td> <td>GGAGCAUCAUCAAGAUUCACA</td> <td>AGAAUCUUGAUGAUGCUGCAG</td> <td>NA</td> <td>68</td> </tr> <tr> <td> <i>mol-miR319</i> </td> <td> <i>mol-miR319</i> </td> <td>UUGGACUGAAGGGAGCUCCC</td> <td /> <td>1204</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR319e</i> </td> <td>UUUGGACUGAAGGGAGCUCCU</td> <td /> <td>2843</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR319g</i> </td> <td>UUGGACUGAAGGGAGCUCCUUC</td> <td /> <td>1240</td> <td /> </tr> <tr> <td> <i>mol-miR390</i> </td> <td> <i>mol-miR390a</i> </td> <td>AAGCUCAGGAGGGAUAGCGCC</td> <td>CGCUAUCCAUCCUGAGUUUCA</td> <td>1128</td> <td>82</td> </tr> <tr> <td /> <td> <i>mol-miR390d</i> </td> <td>AAGCUCAGGAGGGAUAGCGCC</td> <td>CGCUAUCCAUCCUGAGUUUUA</td> <td>NA</td> <td>15</td> </tr> <tr> <td /> <td> <i>mol-miR390e</i> </td> <td>AGCUCAGGAGGGAUAGCGCC</td> <td>CGCUAUCUAUCCUGAGCUCCA</td> <td>203</td> <td>NA</td> </tr> <tr> <td> <i>mol-miR393</i> </td> <td> <i>mol-miR393a</i> </td> <td>CAUCCAAAGGGAUCGCAUUGA</td> <td /> <td>591</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR393b</i> </td> <td>UCCAAAGGGAUCGCAUUGAUC</td> <td /> <td>1730</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR393c</i> </td> <td>UCCAAAGGGAUCGCAUUGAUCU</td> <td>AUCAGUGCAAUCCCUUUGGAAU</td> <td>11813</td> <td>NA</td> </tr> <tr> <td /> <td> <i>mol-miR393h</i> </td> <td>UUCCAAAGGGAUCGCAUUGAUC</td> <td /> <td>9974</td> <td /> </tr> <tr> <td> <i>mol-miR394</i> </td> <td> <i>mol-miR394b</i> </td> <td>UUGGCAUUCUGUCCACCUCC</td> <td>CUGUUGGUCUCUCUUUGUAA</td> <td>1147</td> <td>NA</td> </tr> <tr> <td> <i>mol-miR395</i> </td> <td> <i>mol-miR395a</i> </td> <td>CUGAAGUGUUUGGGGGAACUC</td> <td /> <td>84</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR395d</i> </td> <td>UGAAGUGUUUGGGGGAACUUU</td> <td /> <td>26</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR395g</i> </td> <td>UUGAAGUGUUUGGGGGAACUC</td> <td /> <td>43</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR395h</i> </td> <td>AUGAAGUGUUUGGGGGAACUU</td> <td /> <td>26</td> <td /> </tr> <tr> <td> <i>mol-miR396</i> </td> <td> <i>mol-miR396a</i> </td> <td>UUCCACAGCUUUCUUGAACGU</td> <td /> <td>265</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR396c</i> </td> <td>UUCCACAGCUUUCUUGAACUU</td> <td /> <td>7917</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR396e</i> </td> <td>UUCCACAGGCUUUCUUGAACUG</td> <td /> <td>188</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR396g</i> </td> <td>UCCCACAGCUUUAUUGAACUG</td> <td>GUUCAAGAAAGCUGUGGAAGA</td> <td>12</td> <td>265</td> </tr> <tr> <td /> <td> <i>mol-miR396h</i> </td> <td>UCCACAGCUUUCUUGAACUG</td> <td /> <td>419</td> <td /> </tr> <tr> <td> <i>mol-miR397</i> </td> <td> <i>mol-miR397a</i> </td> <td>UCAUUGAGUGCAGCGUUGAUG</td> <td /> <td>529</td> <td /> </tr> <tr> <td> <i>mol-miR398</i> </td> <td> <i>mol-miR398a</i> </td> <td>GGGUUGAUUUGAGAACAUAUG</td> <td>UAUGUUCUCAGGUCGCCCCUG</td> <td>NA</td> <td>23</td> </tr> <tr> <td /> <td> <i>mol-miR398c</i> </td> <td>UGUGUUCUCAGGUCGCCCCUG</td> <td /> <td>3285</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR398f</i> </td> <td>GGUGUUCUCAGGUCGCCCCUG</td> <td /> <td>115</td> <td /> </tr> <tr> <td> <i>mol-miR399</i> </td> <td> <i>mol-miR399a</i> </td> <td>CGCCAAAGGAGAGUUGCCCUU</td> <td /> <td>119</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR399d</i> </td> <td>UGCCAAAGGAGAGUUGCCCUU</td> <td /> <td>76</td> <td /> </tr> <tr> <td> <i>mol-miR403</i> </td> <td> <i>mol-miR403</i> </td> <td>UUAGAUUCACGCACAAACUCG</td> <td /> <td>5277</td> <td /> </tr> <tr> <td /> <td> <i>mol-miR403a</i> </td> <td>UUAGAUUCACGCACAAACUUG</td> <td /> <td>324</td> <td /> </tr> <tr> <td> <i>mol-miR408</i> </td> <td> <i>mol-miR408</i> </td> <td>CAGGGAUGAGGCAGAGCAUGG</td> <td>CUGCACUGCCUCUUCCCUGGC</td> <td>NA</td> <td>27</td> </tr> <tr> <td> <i>mol-miR530</i> </td> <td> <i>mol-miR530</i> </td> <td>UCUGCAUUUGCACCUGCACCU</td> <td /> <td>950</td> <td /> </tr> <tr> <td> <i>mol-miR535</i> </td> <td> <i>mol-miR535a</i> </td> <td>UGACAACGAGAGAGAGCACGC</td> <td /> <td>347</td> <td /> </tr> <tr> <td> <i>mol-miR827</i> </td> <td> <i>mol-miR827</i> </td> <td>UUUGUUGAUUGACAUCUAUAC</td> <td>UUAGAUGACCAUCAACAAACG</td> <td>22</td> <td>NA</td> </tr> <tr> <td> <i>mol-miR858</i> </td> <td> <i>mol-miR858b</i> </td> <td>UUCGUUGUCUGUUCGACCUUG</td> <td /> <td>34</td> <td /> </tr> <tr> <td> <i>mol-miR894</i> </td> <td> <i>mol-miR894</i> </td> <td>CGUUUCACGUCGGGUUCACC</td> <td /> <td>396</td> <td /> </tr> <tr> <td> <i>mol-miR1310</i> </td> <td> <i>mol-miR1310</i> </td> <td>AGGCAUCGGGGGCGCAACGCCC</td> <td /> <td>1170</td> <td /> </tr> <tr> <td> <i>mol-miR1511</i> </td> <td> <i>mol-miR1511</i> </td> <td>CGUGGUAUCAGAGUCAUGUUA</td> <td>ACCUGGCUCUGAUACCAUAAC</td> <td>1279</td> <td>NA</td> </tr> <tr> <td> <i>mol-miR1515</i> </td> <td> <i>mol-miR1515</i> </td> <td>UCAUUUUUGCGUGCAAUGAUCC</td> <td /> <td>61</td> <td /> </tr> <tr> <td> <i>mol-miR3711</i> </td> <td> <i>mol-miR3711</i> </td> <td>UGGCGCUAGAAGGAGGGCCU</td> <td /> <td>271</td> <td /> </tr> <tr> <td> <i>mol-miR5139</i> </td> <td> <i>mol-miR5139</i> </td> <td>AAACCUGGCUCUGAUACCA</td> <td /> <td>1612</td> <td /> </tr> <tr> <td> <i>mol-miR5559</i> </td> <td> <i>mol-miR5559</i> </td> <td>UACUUGGUGAAUUGUUGGAUC</td> <td /> <td>322</td> <td /> </tr> <tr> <td> <i>mol-miR6300</i> </td> <td> <i>mol-miR6300</i> </td> <td>GUCGUUGUAGUAUAGUGG</td> <td /> <td>10523</td> <td /> </tr> <tr> <td> <i>mol-miR6478</i> </td> <td> <i>mol-miR6478</i> </td> <td>CCGACCUUAGCUCAGUUGGUG</td> <td /> <td>8520</td> <td /> </tr> <tr> <td> <i>mol-miR7751</i> </td> <td> <i>mol-miR7751</i> </td> <td>AUCUUCCUCGUGGACAAGCGGUAG</td> <td>UUUGGUGCACCCGGCUGGAGAUGG</td> <td>NA</td> <td>1683</td> </tr> <tr> <td> <i>mol-miR8155</i> </td> <td> <i>mol-miR8155</i> </td> <td>UAACCUGGCUCUGAUACCA</td> <td /> <td>1612</td> <td /> </tr> </tbody> </table> </div> </div> <p>The most abundant miRNA families were <i>miR166</i>, <i>miR156</i> and <i>miR167</i> with eight members each. Among the other families, in <i>miR159</i>, seven members, and in <i>miR396</i> five members were present. The remaining miRNA families had less than five members. As reported in <a href="#pone.0149495.g002">Fig 2</a>, the sequence length distribution had two major peaks at 21 and 22nts with high levels of reads from 20 to 22 nucleotides. Low levels of 24 nucleotide fragments could be due to the high stringent filters applied. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Length distribution of miRNA populations.</div> <p>Sequence length distributions of known and unknown miRNAs in <i>M</i>. <i>oleifera</i> seeds. </p>   </div> </div> <div class="predictinguncharacterizedm.oleiferamirnas" title="sec"> <div class="title" tagx="title" title="title">
Predicting uncharacterized <i>M</i>. <i>oleifera</i> miRNAs </div> <p>All data set was analysed by BLASTn and 21,065 sequences were identified as available for novel miRNA identification with miReader software. To avoid spurious contamination of mRNA fragments or sequencing artifacts, this filtering process was forced to discard all reads shorter than 18nts nucleotides. The length distribution of the resulted sequences is shown in <a href="#pone.0149495.g002">Fig 2</a>. The distribution had a peak at 18 nts decreasing as the sequence length increases. The processed set of sequences was sub-sequentially provided to miReader and, after the analysis, only two uncharacterized miRNA duplexes were predicted as novel uncharacterized miRNAs ( <a href="#pone.0149495.t002">Table 2</a>). This result was not surprising since only the non-conserved miRNAs are normally expressed at low level, only in specific cell-types or under specific organ development and growth conditions [ <a href="#pone.0149495.ref035">35</a>]. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t002">10.1371/journal.pone.0149495.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Putative novel miRNAs list.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>miRNA members</th> <th>miRNA sequence</th> <th>miRNA* sequence</th> <th>Read Counts</th> </tr> <tr> <th>miR</th> <th>miR*</th> </tr> </thead> <tbody> <tr> <td> <i>mol-miR1p</i> </td> <td>CCGUCUCGCCCGGACCCUG</td> <td>CGACGCGGAUCGCGACGG</td> <td>12</td> <td>99</td> </tr> <tr> <td> <i>mol-miR2p</i> </td> <td>CUAUACCCGGCCGUUGGGGC</td> <td>ACCGCAUAGCGCAGUGGAU</td> <td>129</td> <td>53</td> </tr> </tbody> </table> </div> </div> </div> <div class="identifyingknownmirnaconservationratesamongplantspecies" title="sec"> <div class="title" tagx="title" title="title">
Identifying known miRNA conservation rates among plant species</div> <p>In order to evaluate the conservation rate for the <i>M</i>. <i>oleifera</i> known miRNAs within the plant kingdom an in-house ruby script was used. In <a href="#pone.0149495.t003">Table 3</a>, the top 20 conserved miRNAs together with their conservation rates and their abundance in <i>M</i>. <i>oleifera</i> seeds are reported. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t003">10.1371/journal.pone.0149495.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Most highly conserved miRNAs among selected plants.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>miRNA name</th> <th>Sequence</th> <th>Conservation Rate</th> <th>Read Counts</th> </tr> </thead> <tbody> <tr> <td> <i>mol-miR166i</i> </td> <td>UCGGACCAGGCUUCAUUCCCCC</td> <td>147</td> <td>65,243</td> </tr> <tr> <td> <i>mol-miR156</i> </td> <td>CUGACAGAAGAGAGUGAGCAC</td> <td>136</td> <td>2,459</td> </tr> <tr> <td> <i>mol-miR160h</i> </td> <td>UGCCUGGCUCCCUGUAUGCCAUU</td> <td>91</td> <td>25</td> </tr> <tr> <td> <i>mol-miR395a</i> </td> <td>CUGAAGUGUUUGGGGGAACUC</td> <td>90</td> <td>84</td> </tr> <tr> <td> <i>mol-miR171c-3p</i> </td> <td>UGAUUGAGCCGUGCCAAUAUC</td> <td>77</td> <td>503</td> </tr> <tr> <td> <i>mol-miR164d</i> </td> <td>UGGAGAAGCAGGGCACGUGCA</td> <td>73</td> <td>373</td> </tr> <tr> <td> <i>mol-miR167b</i> </td> <td>UGAAGCUGCCAGCAUGAUCUA</td> <td>66</td> <td>3,834</td> </tr> <tr> <td> <i>mol-miR157a-5p</i> </td> <td>UUGACAGAAGAUAGAGAGCAC</td> <td>63</td> <td>316</td> </tr> <tr> <td> <i>mol-miR390a-5p</i> </td> <td>AAGCUCAGGAGGGAUAGCGCC</td> <td>54</td> <td>1,128</td> </tr> <tr> <td> <i>mol-miR169d</i> </td> <td>UAGCCAAGGAUGACUUGCCU</td> <td>51</td> <td>22</td> </tr> <tr> <td> <i>mol-miR394b-5p</i> </td> <td>UUGGCAUUCUGUCCACCUCC</td> <td>44</td> <td>1,147</td> </tr> <tr> <td> <i>mol-miR396c</i> </td> <td>UUCCACAGCUUUCUUGAACUU</td> <td>40</td> <td>7,917</td> </tr> <tr> <td> <i>mol-miR167d</i> </td> <td>UGAAGCUGCCAGCAUGAUCUGA</td> <td>38</td> <td>4.007</td> </tr> <tr> <td> <i>mol-miR162</i> </td> <td>UCGAUAAACCUCUGCAUCCAG</td> <td>34</td> <td>158</td> </tr> <tr> <td> <i>mol-miR319e</i> </td> <td>UUUGGACUGAAGGGAGCUCCU</td> <td>33</td> <td>2,843</td> </tr> <tr> <td> <i>mol-miR403</i> </td> <td>UUAGAUUCACGCACAAACUCG</td> <td>31</td> <td>5,277</td> </tr> <tr> <td> <i>mol-miR171d</i> </td> <td>UUGAGCCGUGCCAAUAUCACG</td> <td>30</td> <td>410</td> </tr> <tr> <td> <i>mol-miR168d-5p</i> </td> <td>UCGCUUGGUGCAGGUCGGGAA</td> <td>25</td> <td>517</td> </tr> <tr> <td> <i>mol-miR393b</i> </td> <td>UCCAAAGGGAUCGCAUUGAUC</td> <td>24</td> <td>1,730</td> </tr> <tr> <td> <i>mol-miR159a</i> </td> <td>UUUGGAUUGAAGGGAGCUCUA</td> <td>23</td> <td>11,908</td> </tr> </tbody> </table> </div> </div> <p>Moreover, we drew a plot for better understanding the relationship between miRNA conservation rate and abundance in the seed tissues of <i>Moringa</i>. A logarithmic scale was used to obtain a linear plot since the range of abundance values was too high. As shown in <a href="#pone.0149495.g003">Fig 3</a>, <i>mol-miR166i</i> was the most conserved miRNA throughout plant species and the most abundant miRNA in <i>Moringa</i> seed tissues. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Correlation analysis.</div> <p>Correlation analysis between the conservation rate through plants and abundance in <i>M</i>. <i>oleifera</i> samples. </p>   </div> </div> <div class="validatingexpressionofconservedmirnasamongtheorganofm.oleiferaandcropplantsspecies" title="sec"> <div class="title" tagx="title" title="title">
Validating expression of conserved miRNAs among the organ of <i>M</i>. <i>oleifera</i> and crop plants species </div> <p>To validate the representative expression patterns of the miRNAs, we performed qRT-PCR analysis of eight conserved miRNAs on different organ tissues of <i>M</i>. <i>oleifera</i> and on three crop plant species. The data collected demonstrated that the expression patterns were similar between the two analytical tools (Illumina sequencing and qRT-PCR) for six of the eight miRNAs, whereas <i>mol-miR156f-5p</i> showed different expression pattern as detected between the two molecular tools ( <a href="#pone.0149495.g004">Fig 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Quantitative RT-PCR analysis of mature miRNAs in <i>M</i>. <i>oleifera</i> seeds. </div> <p>A quantitative analysis of plant miRNAs in <i>M</i>. <i>oleifera</i> seeds by qRT-PCR. The levels of 8 plant miRNAs detected by Illumina sequencing and confirmed by qRT-PCR. All qRT-PCR reactions were prepared in triplicate for each sample. </p>   </div> <p>As illustrated in <a href="#pone.0149495.g005">Fig 5</a>, <i>mol-miR168d-5p</i> and <i>mol-miR156f-5p</i> were more abundant than other miRNAs in seed, leaf and root tissues. Vice versa, <i>mol-miR164c</i> showed higher expression levels in stem tissues. However, apart from <i>mol-miR164c</i> other miRNAs detected by qRT-PCR had higher expression in seeds compared to the other organ tissues. Similar results were obtained by assessing conserved <i>mol-miR168d-5p</i>, <i>mol-miR166j</i> and <i>mol-miR156t</i> in multiple plant tissues. As shown ( <a href="#pone.0149495.g006">Fig 6A</a>) <i>mol-miR168d-5p</i> was the most abundant among well-expressed miRNAs, except for <i>M</i>. <i>sativa</i> sprout and <i>O</i>. <i>europaea</i> leaf. Moreover, when comparing <i>mol-miR156t</i> expression levels in different tissues, including zygotic embryo, and in seeds treated in different way, we found that it was mainly expressed in dry seed tissues rather than other tissues ( <a href="#pone.0149495.g006">Fig 6B</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Quantitative RT-PCR analysis of mature miRNAs in <i>M</i>. <i>oleifera</i> different tissues. </div> <p>The relative levels of 6 plant miRNAs detected by qRT-PCR in different <i>M</i>. <i>oleifera</i> tissues. The expression level of <i>miR390d-3p</i> was set as control and taken as 1, and expression level in all other miRNAs was quantified relative to it. <i>MiR159</i> was used as an endogenous control. Each value represents the mean of three different determination. Significant differences at P&amp;lt;0.05 (One Way ANOVA) between miRNAs are indicated with different letters. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Quantitative RT-PCR analysis of mature miRNAs in different edible plants and in different <i>M</i>. <i>oleifera</i> tissues. </div> <p>(A) Quantitative analysis of plant <i>mol-miR168d-5p</i>, <i>mol-miR166j</i> and <i>mol-miR156t</i> in different edible plants. (B) Quantitative analysis of <i>mol-miR156t</i> in different tissue and in different seed conditions. Data are reported as the mean of three different experiments. Significant differences at P&amp;lt;0.05 (One Way ANOVA) between samples are indicated with asterisks or line. </p>   </div> </div> <div class="predictingtargetgenesforconservedanduncharacterizedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Predicting target genes for conserved and uncharacterized miRNAs</div> <p>The system of prediction analysis was forced to select the five best predictions for each miRNA, and from the resulting data, 48 putative target genes were identified as potentially regulated by known miRNAs ( <a href="#pone.0149495.t004">Table 4</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t004">10.1371/journal.pone.0149495.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
List of target genes for known <i>M</i>. <i>oleifera</i> miRNAs. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>miRNA name</th> <th>Target Gene</th> <th>Entrez ID</th> <th>Gene Aligned Sequence</th> <th>Mechanism of regulation</th> </tr> </thead> <tbody> <tr> <td> <i>mol-miR393c-5p</i> </td> <td> <i>AFB3</i> </td> <td>837838</td> <td>AACAAUGCGAUCCCUUUGGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR393c-5p</i> </td> <td> <i>AFB2</i> </td> <td>822296</td> <td>AACAAUGCGAUCCCUUUGGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156</i> </td> <td> <i>SPL3</i> </td> <td>817948</td> <td>UGCUUACUCUCUUCUGUCAG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156q</i> </td> <td> <i>SPL10</i> </td> <td>839626</td> <td>AGUGCUCUCUCUCUUCUGUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156j</i> </td> <td> <i>SPL11</i> </td> <td>839625</td> <td>GUGCUCUCUCUCUUCUGUCAAC</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156j</i> </td> <td> <i>SPL10</i> </td> <td>839626</td> <td>GUGCUCUCUCUCUUCUGUCAAC</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR159a</i> </td> <td> <i>MYB101</i> </td> <td>817807</td> <td>UAGAGCUUCCUUCAAACCAAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR159a</i> </td> <td> <i>DUO1</i> </td> <td>825217</td> <td>UGGAGCUCCAUUCGAUCCAAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR159a</i> </td> <td> <i>AtMYB104</i> </td> <td>817236</td> <td>UGGAGCUCCCUUCAUUCCAAG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR397a</i> </td> <td> <i>LAC2</i> </td> <td>817462</td> <td>AUCAAUGCUGCACUCAAUGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR397a</i> </td> <td> <i>IRX12</i> </td> <td>818386</td> <td>GUCAACGCUGCACUUAAUGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR397a</i> </td> <td> <i>LAC17</i> </td> <td>836124</td> <td>AUCAAUGCUGCACUUAAUGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR166j</i> </td> <td> <i>PHV</i> </td> <td>839928</td> <td>GGGAUGAAGCCUGGUCCGGA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR393a</i> </td> <td> <i>AFB3</i> </td> <td>837838</td> <td>CAAUGCGAUCCCUUUGGAUG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR393a</i> </td> <td> <i>TIR1</i> </td> <td>825473</td> <td>CAAUGCGAUCCCUUUGGAUG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR393a</i> </td> <td> <i>AFB2</i> </td> <td>822296</td> <td>CAAUGCGAUCCCUUUGGAUG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR393a</i> </td> <td> <i>GRH1</i> </td> <td>828045</td> <td>CCAUGCGAUCCCUUUGGAUG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR171a</i> </td> <td> <i>HAM1</i> </td> <td>836582</td> <td>AUAUUGGCGCGGCUCAAUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR171a</i> </td> <td> <i>HAM2</i> </td> <td>830834</td> <td>AUAUUGGCGCGGCUCAAUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR171a</i> </td> <td> <i>HAM3</i> </td> <td>828208</td> <td>AUAUUGGCGCGGCUCAAUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156t</i> </td> <td> <i>SPL10</i> </td> <td>839626</td> <td>GUGCUCUCUCUCUUCUGUCAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156t</i> </td> <td> <i>SPL2</i> </td> <td>834345</td> <td>GUGCUCUCUCUCUUCUGUCAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR396e</i> </td> <td> <i>AtGRF4</i> </td> <td>824457</td> <td>CCGUUCAAGAAAGCCUGUGGAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR396e</i> </td> <td> <i>AtGRF3</i> </td> <td>818213</td> <td>CCGUUCAAGAAAGCCUGUGGAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR396e</i> </td> <td> <i>AtGRF1</i> </td> <td>816815</td> <td>GUUCAAGAAAGCCUGUGGAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR396e</i> </td> <td> <i>AtGRF2</i> </td> <td>829930</td> <td>GUUCAAGAAAGCCUGUGGAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR396e</i> </td> <td> <i>AtGRF9</i> </td> <td>819156</td> <td>GUUCAAGAAAGCUUGUGGAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR172m-3p</i> </td> <td> <i>AP2</i> </td> <td>829845</td> <td>CUGCAGCAUCAUCAGGAUUCU</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR172m-3p</i> </td> <td> <i>TOE2</i> </td> <td>836134</td> <td>UGCAGCAUCAUCAGGAUUCU</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR156h-5p</i> </td> <td> <i>SPL10</i> </td> <td>839626</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR156h-5p</i> </td> <td> <i>SPL13A</i> </td> <td>21393429</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR156h-5p</i> </td> <td> <i>SPL2</i> </td> <td>834345</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR156f-5p</i> </td> <td> <i>SPL3</i> </td> <td>817948</td> <td>UGCUUACUCUCUUCUGUCAG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR160h</i> </td> <td> <i>ARF17</i> </td> <td>844120</td> <td>UGGCAUGCAGGGAGCCAGGCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR160h</i> </td> <td> <i>ARF10</i> </td> <td>817382</td> <td>GGAAUACAGGGAGCCAGGCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR827-5p</i> </td> <td> <i>NLA</i> </td> <td>839559</td> <td>UGUUUGUUGAUGGUCAUCUAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR395g</i> </td> <td> <i>AST68</i> </td> <td>830882</td> <td>AGUUCUCCCAAACACUUCAA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR395h</i> </td> <td> <i>APS4</i> </td> <td>834400</td> <td>GAGUUCCUCCAAACACUUCAU</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR395d</i> </td> <td> <i>APS4</i> </td> <td>834400</td> <td>AGAGUUCCUCCAAACACUUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR395d</i> </td> <td> <i>AST68</i> </td> <td>830882</td> <td>AAGUUCUCCCAAACACUUCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR166</i> </td> <td> <i>PHV</i> </td> <td>839928</td> <td>UUGGGAUGAAGCCUGGUCCGG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR166</i> </td> <td> <i>PHB</i> </td> <td>818036</td> <td>UUGGGAUGAAGCCUGGUCCGG</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR157a-5p</i> </td> <td> <i>SPL10</i> </td> <td>839626</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR157a-5p</i> </td> <td> <i>SPL13A</i> </td> <td>21393429</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR157a-5p</i> </td> <td> <i>SPL2</i> </td> <td>834345</td> <td>GUGCUCUCUCUCUUCUGUCA</td> <td>Translation</td> </tr> <tr> <td> <i>mol-miR164c</i> </td> <td> <i>CUC1</i> </td> <td>820748</td> <td>GCACGUGUCCUGUUUCUCCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR164c</i> </td> <td> <i>CUC2</i> </td> <td>835478</td> <td>GCACGUGUCCUGUUUCUCCA</td> <td>Cleavage</td> </tr> <tr> <td> <i>mol-miR164a</i> </td> <td> <i>ANAC080</i> </td> <td>830661</td> <td>UUUACGUGCCCUGCUUCUCCA</td> <td>Cleavage</td> </tr> </tbody> </table> </div> </div> <p>Despite research of targets for all putative novel miRNA, only 3 target genes were selected ( <a href="#pone.0149495.g007">Fig 7</a>) that involve <i>mol-miR2p-5p</i>, the most abundant uncharacterized miRNA highlighted by the miReader algorithm. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g007"> <span class="label" tagx="label" title="label">Fig 7</span> </a> </div>   <div class="title" tagx="title" title="title">
List of target genes for putative novel <i>M</i>. <i>oleifera</i> miRNAs. </div>   </div> <p>Enrichment analysis with gene ontology terms on both target gene sets highlighted the relevance of these genes for plant organisms. As shown in <a href="#pone.0149495.g008">Fig 8</a>, most genes regulated by known miRNAs are involved in biological pathways localized to root, meristem and seed. Furthermore, these miRNAs participate in fundamental processes such as the development of anatomical (root, leaf) and reproductive structure, maintenance (flower, meristem, pollen and stamen), embryonic, and post-embryonic development. Genes putatively regulated by unknown miRNAs are involved in cellular metabolism of macromolecules, in particular macromolecules affecting protein transport and biosynthesis. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g008"> <span class="label" tagx="label" title="label">Fig 8</span> </a> </div>   <div class="title" tagx="title" title="title">
Known miRNA target functions.</div> <p>The miRNA target genes were classified according to the biological process.</p>   </div> </div> <div class="bioinformaticspredictionofhumangenetargetsform.oleiferamirnas" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics prediction of human gene targets for <i>M</i>. <i>oleifera</i> miRNAs </div> <p>To investigate whether <i>M</i>. <i>oleifera</i> miRNAs might regulate human gene expression we used MirCompare software to search homologous human miRNAs. The most and conserved <i>mol-miR166i</i> resulted functional homologies with <i>hsa-miR6503-3p</i> ( <a href="#pone.0149495.t005">Table 5</a>) that is involved in regulating inflammation [ <a href="#pone.0149495.ref036">36</a>], and <i>mol-miR393c</i> was homologous to <i>hsa-miR548ah-5p</i> that is involved in immune tolerance [ <a href="#pone.0149495.ref037">37</a>]. Further, <i>mol-mir168a</i> showed sequence homology with <i>hsa-miR579</i>, a human miRNA that normally regulates <i>TNFα</i> expression during endotoxin tolerance [ <a href="#pone.0149495.ref038">38</a>]. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t005">10.1371/journal.pone.0149495.t005</a> <span class="label" tagx="label" title="label">Table 5</span>  <div class="title" tagx="title" title="title">
Comparing <i>M</i>. <i>oleifera</i> miRNAs with their human counterparts using MirCompare software. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Human miRNA name</th> <th> <i>M</i>. <i>oleifera</i> miRNA name </th> <th>r-value</th> <th>Sequence alignment details</th> </tr> </thead> <tbody> <tr> <td> <i>hsa-miR6503-3p</i> </td> <td> <i>mol-miR166i</i> </td> <td>0.67</td> <td>-GGAC-AGG-U-CA--CC-CC</td> </tr> <tr> <td> <i>hsa-miR548ah-5p</i> </td> <td> <i>mol-miR393c</i> </td> <td>0.65</td> <td>-AAAG-GAU-GCA-UG-U--</td> </tr> <tr> <td> <i>hsa-miR3940-5p</i> </td> <td> <i>mol-miR159a</i> </td> <td>0.65</td> <td>-UGG-UUG--G-G-GCUCU-</td> </tr> <tr> <td> <i>hsa-miR579</i> </td> <td> <i>mol-miR168a</i> </td> <td>0.57</td> <td>UC--UUGGU--A---CG-GA-</td> </tr> <tr> <td> <i>hsa-miR4534</i> </td> <td> <i>mol-miR159a</i> </td> <td>0.71</td> <td>GGAU-GA-G-G-G-UCU</td> </tr> <tr> <td> <i>hsa-miR1306-3p</i> </td> <td> <i>mol-miR6478</i> </td> <td>0.78</td> <td>AC-UU-GCUC-G-UGGUG</td> </tr> <tr> <td> <i>hsa-miR4703-3p</i> </td> <td> <i>mol-miR6300</i> </td> <td>0.72</td> <td>GU-GUUGUA-U-UA-UG-</td> </tr> <tr> <td> <i>hsa-miR5008-5p</i> </td> <td> <i>mol-miR6300</i> </td> <td>0.67</td> <td>G-C-UUG--G-A-AGUGG</td> </tr> <tr> <td> <i>hsa-miR4273</i> </td> <td> <i>mol-miR398c</i> </td> <td>0.67</td> <td>GUGUUCUC-G-U-G-C--</td> </tr> </tbody> </table> </div> </div> <p>COMIR software [ <a href="#pone.0149495.ref033">33</a>] predicted human genes that could be regulated by <i>M</i>. <i>oleifera</i> miRNAs. A subset of genes relevant to leukemia and acute myeloid ( <a href="#pone.0149495.t006">Table 6</a>), which have key roles in the regulating cellular pathways such as apoptosis, cell cycle and protein degradation, have been found. Based on these results we identified <i>mol-miR168a</i> homologous to <i>hsa-miR579</i>, a human miRNA with many target genes including <i>SIRT1</i>. The complete report of predicted genes ranked by COMIR score (always upper than 0.9) is given in <a href="#pone.0149495.s006">S2 Table</a>. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0149495.t006">10.1371/journal.pone.0149495.t006</a> <span class="label" tagx="label" title="label">Table 6</span>  <div class="title" tagx="title" title="title">
Predicting gene target in humans using COMIR software.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>miRNA name</th> <th>Ensemble Gene ID</th> <th>Entrez ID</th> <th>Target Gene</th> <th>COMIR Score</th> </tr> </thead> <tbody> <tr> <td> <i>mol-miR166i</i> </td> <td>ENSG00000082701</td> <td>2932</td> <td> <i>GSK3B</i> </td> <td>0.9019</td> </tr> <tr> <td /> <td>ENSG00000064393</td> <td>28996</td> <td> <i>HIPK2</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000156113</td> <td>3778</td> <td> <i>KCNMA1</i> </td> <td>0.9038</td> </tr> <tr> <td /> <td>ENSG00000263162</td> <td>8924 100653292</td> <td> <i>HERC2</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000169213</td> <td>5865</td> <td> <i>RAB3B</i> </td> <td>0.9075</td> </tr> <tr> <td /> <td>ENSG00000171105</td> <td>3643</td> <td> <i>INSR</i> </td> <td>0.9015</td> </tr> <tr> <td /> <td>ENSG00000078142</td> <td>5289</td> <td> <i>PIK3C3</i> </td> <td>0.9044</td> </tr> <tr> <td /> <td>ENSG00000263162</td> <td>8924 100653292</td> <td> <i>HERC2</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000119547</td> <td>9480</td> <td> <i>ONECUT2</i> </td> <td>0.9075</td> </tr> <tr> <td> <i>mol-miR393c</i> </td> <td>ENSG00000178662</td> <td>80034</td> <td> <i>CSRNP3</i> </td> <td>0.9075</td> </tr> <tr> <td /> <td>ENSG00000102908</td> <td>10725</td> <td> <i>NFAT5</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000128585</td> <td>4289</td> <td> <i>MKLN1</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000145907</td> <td>10146</td> <td> <i>G3BP1</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000009413</td> <td>5980</td> <td> <i>REV3L</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000010244</td> <td>7756</td> <td> <i>ZNF207</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000100354</td> <td>23112</td> <td> <i>TNRC6B</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000143190</td> <td>5451</td> <td> <i>pou2f1</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000173611</td> <td>286205</td> <td> <i>Scai</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000100731</td> <td>22990</td> <td> <i>pcnx</i> </td> <td>0.9071</td> </tr> <tr> <td> <i>mol-miR159a</i> </td> <td>ENSG00000171435</td> <td>283455</td> <td> <i>KSR2</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000119547</td> <td>9480</td> <td> <i>ONECUT2</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000153721</td> <td>154043</td> <td> <i>CNKSR3</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000261115</td> <td>1.01E+08</td> <td> <i>TMEM178B</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000158445</td> <td>3745</td> <td> <i>KCNB1</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000077157</td> <td>4660</td> <td> <i>PPP1R12B</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000196090</td> <td>11122</td> <td> <i>PTPRT</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000055609</td> <td>58508</td> <td> <i>KMT2C</i> </td> <td>0.9068</td> </tr> <tr> <td /> <td>ENSG00000132549</td> <td>157680</td> <td> <i>VPS13B</i> </td> <td>0.9065</td> </tr> <tr> <td /> <td>ENSG00000148019</td> <td>84131</td> <td> <i>CEP78</i> </td> <td>0.9062</td> </tr> <tr> <td> <i>mol-miR168a</i> </td> <td>ENSG00000096717</td> <td>23411</td> <td> <i>SIRT1</i> </td> <td>0.9087</td> </tr> <tr> <td /> <td>ENSG00000178562</td> <td>940</td> <td> <i>CD28</i> </td> <td>0.9099</td> </tr> <tr> <td /> <td>ENSG00000134352</td> <td>3572</td> <td> <i>IL6ST</i> </td> <td>0.9237</td> </tr> <tr> <td /> <td>ENSG00000118689</td> <td>2309</td> <td> <i>FOXO3</i> </td> <td>0.9190</td> </tr> <tr> <td /> <td>ENSG00000106799</td> <td>7046</td> <td> <i>TGFBR1</i> </td> <td>0.9115</td> </tr> <tr> <td /> <td>ENSG00000169967</td> <td>10746</td> <td> <i>MAP3K2</i> </td> <td>0.9118</td> </tr> <tr> <td /> <td>ENSG00000175595</td> <td>2072</td> <td> <i>ERCC4</i> </td> <td>0.9110</td> </tr> <tr> <td /> <td>ENSG00000149311</td> <td>472</td> <td> <i>ATM</i> </td> <td>0.9187</td> </tr> <tr> <td /> <td>ENSG00000149948</td> <td>8091</td> <td> <i>HMGA2</i> </td> <td>0.9119</td> </tr> <tr> <td /> <td>ENSG00000007372</td> <td>5080</td> <td> <i>PAX6</i> </td> <td>0.9189</td> </tr> <tr> <td> <i>mol-miR6478</i> </td> <td>ENSG00000134313</td> <td>57498</td> <td> <i>KIDINS220</i> </td> <td>0.9061</td> </tr> <tr> <td /> <td>ENSG00000106261</td> <td>7586</td> <td> <i>ZKSCAN1</i> </td> <td>0.9058</td> </tr> <tr> <td /> <td>ENSG00000134909</td> <td>9743</td> <td> <i>ARHGAP32</i> </td> <td>0.9055</td> </tr> <tr> <td /> <td>ENSG00000136709</td> <td>55339 84826</td> <td> <i>WDR33</i> </td> <td>0.9055</td> </tr> <tr> <td /> <td>ENSG00000112706</td> <td>3617</td> <td> <i>IMPG1</i> </td> <td>0.9051</td> </tr> <tr> <td /> <td>ENSG00000107331</td> <td>20</td> <td> <i>ABCA2</i> </td> <td>0.9049</td> </tr> <tr> <td /> <td>ENSG00000088808</td> <td>23368</td> <td> <i>PPP1R13B</i> </td> <td>0.9029</td> </tr> <tr> <td /> <td>ENSG00000167654</td> <td>85300</td> <td> <i>ATCAY</i> </td> <td>0.9028</td> </tr> <tr> <td /> <td>ENSG00000189339</td> <td>728661</td> <td> <i>SLC35E2B</i> </td> <td>0.9026</td> </tr> <tr> <td /> <td>ENSG00000180370</td> <td>5062</td> <td> <i>PAK2</i> </td> <td>0.9025</td> </tr> <tr> <td> <i>mol-miR6300</i> </td> <td>ENSG00000178567</td> <td>9852</td> <td> <i>EPM2AIP1</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000213699</td> <td>54978</td> <td> <i>SLC35F6</i> </td> <td>0.9075</td> </tr> <tr> <td /> <td>ENSG00000197818</td> <td>23315</td> <td> <i>SLC9A8</i> </td> <td>0.9072</td> </tr> <tr> <td /> <td>ENSG00000183751</td> <td>10607</td> <td> <i>TBL3</i> </td> <td>0.9071</td> </tr> <tr> <td /> <td>ENSG00000206190</td> <td>57194</td> <td> <i>ATP10A</i> </td> <td>0.9071</td> </tr> <tr> <td /> <td>ENSG00000166206</td> <td>2562</td> <td> <i>GABRB3</i> </td> <td>0.9071</td> </tr> <tr> <td /> <td>ENSG00000198000</td> <td>55035</td> <td> <i>NOL8</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000172380</td> <td>55970</td> <td> <i>GNG12</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000152443</td> <td>284309</td> <td> <i>ZNF776</i> </td> <td>0.9067</td> </tr> <tr> <td /> <td>ENSG00000133703</td> <td>3845</td> <td> <i>KRAS</i> </td> <td>0.9066</td> </tr> <tr> <td> <i>mol-miR398c</i> </td> <td>ENSG00000055609</td> <td>58508</td> <td> <i>KMT2C</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000164684</td> <td>619279</td> <td> <i>ZNF704</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000145012</td> <td>4026</td> <td> <i>LPP</i> </td> <td>0.9076</td> </tr> <tr> <td /> <td>ENSG00000151914</td> <td>667</td> <td> <i>DST</i> </td> <td>0.9075</td> </tr> <tr> <td /> <td>ENSG00000158258</td> <td>64084</td> <td> <i>CLSTN2</i> </td> <td>0.9074</td> </tr> <tr> <td /> <td>ENSG00000110436</td> <td>6506</td> <td> <i>SLC1A2</i> </td> <td>0.9073</td> </tr> <tr> <td /> <td>ENSG00000064393</td> <td>28996</td> <td> <i>HIPK2</i> </td> <td>0.9073</td> </tr> <tr> <td /> <td>ENSG00000118482</td> <td>23469</td> <td> <i>PHF3</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000143970</td> <td>55252</td> <td> <i>ASXL2</i> </td> <td>0.907</td> </tr> <tr> <td /> <td>ENSG00000135968</td> <td>9648</td> <td> <i>GCC2</i> </td> <td>0.907</td> </tr> </tbody> </table> </div> </div> </div> <div class="syntheticmol-mir168atransfectionandproteinmodulation" title="sec"> <div class="title" tagx="title" title="title">
Synthetic <i>mol-miR168a</i> transfection and protein modulation </div> <p>In these preliminary studies, we tested whether an exogenous miRNA derived from <i>M</i>. <i>oleifera</i> might be functional in human cell line and we verified its ability to inhibit post transcriptional mRNA expression. In particular, transfecting Hepatoma cell line G2 with the synthetic FITC <i>mol-miR168a</i> was run. By 72 hours after transfection with FITC ( <a href="#pone.0149495.g009">Fig 9A</a>, green cells, panel b), synthetic <i>mol-miR168a</i> increased the number of fluorescent cells by EVOS FLoid cell imaging station analysis (about 60–70% of HEPG2 cells resulted significantly and positively transfected). The transfection of synthetic <i>mol-miR168a</i>, a plant miRNA that show sequences homology with <i>hsa-miR579</i>, determined a significant decrease of <i>SIRT1</i> protein level in comparison with HF and control samples ( <a href="#pone.0149495.g009">Fig 9B and 9C</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0149495.g009"> <span class="label" tagx="label" title="label">Fig 9</span> </a> </div>   <div class="title" tagx="title" title="title">
Experimental Validation of miRNA-mRNA human gene regulation.</div> <p>Histograms represent the mean values and ± the SD values. A) Presence of green cells in HepG2 cell lines (panel b) 72hrs after transfection with the synthetic FITC <i>mol-miR168a (EVOS FLoid cell imaging station</i>, <i>LifeTechnologies)</i>. Left panel: HepG2 cells transfected only with the lipofectamin (HF); right panel: HepG2 cells transfected with lipofectamin (HF) and synthetic FITC <i>mol-miR168a</i>. B) One representative experiment (out of three) of Western Blot analysis of <i>SIRT-1</i> expression in HepG2 cells after synthetic <i>mol-miR168a</i> transfection; <i>β-actin</i> expression, run on the same gel, indicates that an equal amount of protein was loaded for each sample. C) Western blot analysis for each sample was quantified by densitometry analysis and values were expressed as OD-Bkg/mm <sup>2</sup>*10 <sup>3</sup>. Histograms represent mean values ± S.D. from three independent experiments performed on HepG2 cells. Statistical comparison of means using t-Test provided the following results: control 72h versus HF, not significant (NS); synthetic <i>mol-miR168a</i> versus control and versus HF, p&amp;lt;0.001. </p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Medicinal plants are studied to identify new therapeutic agents and understand their mechanisms of action against a variety of human diseases. Medicinal plant compounds are known to have high biocompatibility, low toxicity, and potential biological activity [ <a href="#pone.0149495.ref039">39</a>]. In particular, <i>M</i>. <i>oleifera</i> is commonly known and used for its health benefits [ <a href="#pone.0149495.ref002">2</a>]. For centuries and in many cultures around the world, <i>M</i>. <i>oleifera</i> has been used to treat human diseases [ <a href="#pone.0149495.ref001">1</a>] and is an example of traditional medicine that is increasingly popular among African countries, due in part to their poor economic conditions. A recent collaboration between universities in Italy and Cameroon has the objective of investigating the anti-oxidant and anti-tumor properties of <i>M</i>. <i>oleifera</i> [ <a href="#pone.0149495.ref004">4</a>]. The incredible <i>Moringa’s</i> usage as medicinal, claimed by real-life experience, is now slowly being confirmed by scientific experiences. </p> <p>Here we address a new aspect of plant biology that may have substantial impact on understanding of medicinal activity. The miRNA expressed in plants may comprise an independent category of medicinal agents capable of providing beneficial effects for human consumers of valuable plants. </p> <p>Recently, Shahzad and collaborators [ <a href="#pone.0149495.ref003">3</a>] reported a study of DNA markers for genetic diversity and population structure in worldwide collections of <i>M</i>. <i>oleifera</i>. However, no study on the breadth of miRNA has yet been reported for <i>M</i>. <i>oleifera</i>, therefore, the knowledge about molecular mechanisms for compounds produced by <i>M</i>. <i>oleifera</i> is still limited. Here, we describe the identification and characterization of conserved and possibly novel miRNAs from seed of <i>M</i>. <i>oleifera</i> plants grown in Cameroon. We have used bioinformatics tools to understand their potential roles in diverse biological processes and their possible role in human gene regulation. </p> <p>Since the discovery of sRNAs (miRNAs and siRNAs) as regulators of gene expression in <i>C</i>. <i>elegans</i> [ <a href="#pone.0149495.ref040">40</a>] and, more recently, the discovery of plant miRNAs [ <a href="#pone.0149495.ref041">41</a>– <a href="#pone.0149495.ref042">42</a>], the study on miRNAs has become an important and integral topic in functional genomic research. </p> <p>In plants, miRNAs regulate gene expression at the post-transcriptional level by degrading or repressing translation of target mRNAs [ <a href="#pone.0149495.ref043">43</a>]. A high number of experimental and computational studies have indicated that mature miRNAs are evolutionarily conserved in plants. miRNA-mediated gene regulation has an ancient phylogenetic origin and plays an important regulatory role in physiological processes [ <a href="#pone.0149495.ref043">43</a>], many aspects of plant growth, development and environmental adaptability. </p> <p>High-throughput sequencing based on Illumina technology has become a good approach for identification and expression analysis of miRNAs in several plant species, like <i>Arabidopsis</i> [ <a href="#pone.0149495.ref044">44</a>] and other plant species [ <a href="#pone.0149495.ref045">45</a>– <a href="#pone.0149495.ref046">46</a>], although the presence of biases introduced during the construction of sRNA libraries, primarily derived from the adaptor ligation steps [ <a href="#pone.0149495.ref047">47</a>], requires a carefully attention. </p> <p>In this study, we identified 94 conserved miRNAs belonging to 40 families, two uncharacterized miRNA duplexes with 48 and 3 targets gene for conserved and uncharacterized miRNAs respectively. As in earlier studies [ <a href="#pone.0149495.ref045">45</a>– <a href="#pone.0149495.ref046">46</a>], the majority of highly conserved miRNAs from <i>Moringa</i>-seed that were predicted by our analysis resulted to be evolutionarily conserved across plant species and to have high levels of expression. For example, <i>miR166</i>, <i>miR393</i>, <i>miR167</i>, <i>miR396</i>, <i>miR159</i> and <i>miR156</i> families are well conserved among other plant species [ <a href="#pone.0149495.ref048">48</a>] and have a fundamental role in plant biology. Other miRNA families, such as <i>miR408</i> or <i>miR1515</i> were present at lower abundance. <i>mol-miR166</i> families had the highest number of reads (80,612); in particular, <i>mol-miR166i</i> and <i>mol-miR166</i> were the most and the least abundant miRNAs in these families, respectively. Results indicate that different members of the same miRNA family have differing levels of expression. </p> <p>The most and least conserved miRNA families may have evolved to play different roles in plant biology. For example, <i>miR156</i>, <i>miR159</i>, <i>miR166</i> and <i>miR160</i> target <i>SPL</i>, <i>MYB</i>, <i>PHV</i> and <i>ARF genes</i> respectively. These transcription factors are important to plant growth and development. The <i>miR159</i> was reported to target <i>MYB101</i> and <i>MYB33</i> transcription factors, which are positive regulators of <i>ABA</i> signaling during <i>Arabidopsis</i> seed germination. Indeed, <i>miR159</i> may play a role in seed germination [ <a href="#pone.0149495.ref049">49</a>]. <i>MiR156</i> plays crucial role in the control of juvenile-to-adult transition in plants by targeting the <i>SQUAMOSA PROMOTER BINDING PROTEIN LIKE</i> ( <i>SPL</i>) plant-specific transcription factors. <i>SPLs</i> affect processes like leaf development, shoot maturation, phase change and flowering in plants [ <a href="#pone.0149495.ref050">50</a>– <a href="#pone.0149495.ref052">52</a>]. A recent study showed that <i>miR156</i> regulates shoot regenerative capacity and a gradual increase in <i>miR156</i> leads to a decline in shoot regenerative capacity for old plants [ <a href="#pone.0149495.ref053">53</a>]. <i>Auxin response factors</i> were found as a predicted target of <i>miR160</i>, while research studies assessed that this miRNA targets <i>ARF10</i>, <i>ARF16</i> and <i>ARF17</i> and regulates various aspects of plant development in <i>Arabidopsis</i> [ <a href="#pone.0149495.ref054">54</a>]. <i>MiR166/165</i> is an example of well-studied plant miRNAs implicated in various aspects of plant development. The <i>miR166/165</i> negatively regulates its targets <i>Class III Homeodomain Leucine-Zipper</i> ( <i>HD-ZIPIII</i>) <i>transcription factors</i> that in turn regulate the polarity establishment in leaves and vasculature and radial patterning of root. The majority of <i>HD-ZIPIII</i> gene family members consisting of <i>PHABULOSA</i> ( <i>PHB</i>), <i>PHAVOLUTA</i> ( <i>PHV</i>), <i>REVOLUTA</i> ( <i>REV</i>), <i>ATHB8</i>, and <i>ATHB15</i>, are conserved in several land plants including bryophytes, lycopods and seed plants [ <a href="#pone.0149495.ref055">55</a>– <a href="#pone.0149495.ref058">58</a>]. <i>MiR393</i> is a conserved miRNA family discovered in many plants [ <a href="#pone.0149495.ref041">41</a>]. In <i>A</i>. <i>thaliana</i>, four F-box genes <i>TIR1 (TRANSPORT INHIBITOR RESPONSE PROTEIN1)</i>, <i>AFB1</i>, <i>AFB2</i> and <i>AFB3</i> ( <i>AUXIN SIGNALING F-BOX</i>) were validated as <i>miR393</i> targets [ <a href="#pone.0149495.ref059">59</a>]. In rice, overexpression of <i>miR393</i> negatively regulates mRNAs of <i>TIR1</i> and <i>AFB2</i> [ <a href="#pone.0149495.ref060">60</a>]. <i>TIR1</i> and <i>AFB2</i> interact with <i>IAA</i> ( <i>INDOLE-3-ACETIC ACID</i>) proteins, probably releasing the activities of <i>ARFs</i> ( <i>AUXIN RESPONSE FACTORS</i>) and increasing resistance to auxin. The change in auxin response consequently affects diverse aspects of plant growth and development, such as flag leaf inclination, primary root growth, crown root initiation and seed development. </p> <p>In this study, three possible novel miRNA targets were found: <i>mol-miR2p-5p</i> could affect target genes involved in macromolecule metabolism, in particular cellular protein transport and biosynthesis. In <i>Arabidopsis thaliana</i>, the <i>ATPTR1</i> gene seems to be involved in long-distance transport of di- and tri-peptides during seed germination [ <a href="#pone.0149495.ref061">61</a>] while <i>DUF579</i> affects xylan biosynthesis and modulates cell wall biosynthesis [ <a href="#pone.0149495.ref062">62</a>]. Further research on possible targets in other plants may provide important evidence to facilitate the understanding of these novel miRNA functions. Additional studies into novel <i>M</i>. <i>oleifera</i> miRNAs may shed light on their roles in <i>M</i>. <i>oleifera</i> biological processes. </p> <p>The final phase of our bioinformatics analysis focused on the potential for human gene regulation by the most conserved <i>M</i>. <i>oleifera</i> miRNAs. The possibility of identifying plant miRNAs able to regulate human genome expression may be highly important in future studies on the nutritional value and medical usage of food. The combined use of MirCompare and COMIR software on the massive collection of data has identified a small number of human genes that might be regulated by <i>M</i>. <i>oleifera</i> miRNAs. These gene targets include cell-cycle regulation and signaling through the <i>p53</i> pathway; genes related to some classes of cancers including leukemia, acute myeloid and lipoma. For instance, <i>Sirtuins</i> have important roles in cell cycle, apoptosis, metabolic regulation and inflammation. The human genome encodes seven <i>Sirtuin</i> isoforms <i>SIRT1</i>- <i>SIRT7</i> with varying intracellular distribution; a number of studies reported evidence for their roles in a spectrum of disease like cancer, diabetes, obesity and neurodegenerative diseases [ <a href="#pone.0149495.ref063">63</a>]. Recent evidence suggests that genomic stability requires cooperation of <i>p53</i> and <i>SIRT1</i> [ <a href="#pone.0149495.ref064">64</a>]. Our transfection experiments showed that <i>mol-miR168a</i> identified by MirCompare-COMIR software inhibited translation of <i>SIRT1 mRNA</i> in cancer cells. We focused our attention on <i>mol-miR168a</i> because this is the first plant miRNA involved in cross-kingdom activity ( <i>osa-miR168a</i> shown high sequence homology with <i>mol-miR168a</i>) [ <a href="#pone.0149495.ref014">14</a>] and it has functional homology with <i>hsa-miR579</i>, a putative regulator of the <i>SIRT1</i> gene. </p> <p>In conclusion, we have identified a population of <i>Moringa</i>-specific miRNAs that could help our understanding of the regulatory role of miRNAs in this plant. Our results demonstrate that the differentially expressed miRNAs and predictions for their target genes provides a basis for further understanding <i>M</i>. <i>oleifera</i> seed miRNAs and biological processes in which they are involved. Further studies are necessary to search for more miRNAs that are novel and to validate their targets by expression analysis during seed development stage. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Sequence analysis workflow.</div> <p>Schematization of the process for the identification of known and novel miRNAs.</p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0149495.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Workflow of miRNA target prediction.</div> <p>Plant targets were predicted using PsRNA Target. Enrichment analysis for all the predicted targets were conducted using Plant GSEA. MirCompare was used to predict cross-kingdom interaction targets in human. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0149495.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 File</span>  <div class="title" tagx="title" title="title">
Quality control analysis before the removal of the sequencing adapter.</div> <p>Quality scores across all bases ( <b>Figure A in S1 File</b>). Quality score distribution over all sequences ( <b>Figure B in S1 FIle</b>). Percentage of adapter sequence ( <b>Figure C in S1 File</b>). </p> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0149495.s003.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 File</span>  <div class="title" tagx="title" title="title">
Quality control analysis after the removal of the sequencing adapter.</div> <p>Quality scores across all bases ( <b>Figure A in <a href="#pone.0149495.s003">S1 File</a> </b>). Quality score distribution over all sequences ( <b>Figure B in <a href="#pone.0149495.s003">S1 File</a> </b>). Percentage of adapter sequence ( <b>Figure C in <a href="#pone.0149495.s003">S1 File</a> </b>). </p> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0149495.s004.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Known microRNAs in <i>M</i>. <i>oleifera</i> juvenile seed, with additional detailed information. </div> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0149495.s005.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
Complete prediction report of gene target in humans, using COMIR software.</div> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0149495.s006.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Genomix4Life for sequence experiments and Flavia del Gallo for continuous support. We thank Giulia Cappelli and Francesca Mariani for focusing our attention to cross kingdom concept. We thank Ivana Matic and Alberto Calderone for the first editing of the draft and are fully grateful to David C Pauza, University of Maryland, for his suggestion and improving of the final manuscript editing. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0149495.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anwar</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Latif</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashraf</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilani</span> <span class="given-names" tagx="given-names" title="given-names">AH</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Moringa oleifera: a food plant with multiple medicinal uses</span>. <span class="source" tagx="source" title="source">Phytother Res</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">17</span>– <span class="lpage" tagx="lpage" title="lpage">25</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1002/ptr.2023">10.1002/ptr.2023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17089328">17089328</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdull</span> <span class="given-names" tagx="given-names" title="given-names">Razis AF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ibrahim</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kntayya</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Health benefits of Moringa oleifera</span>. <span class="source" tagx="source" title="source">Asian Pac J Cancer Prev</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">8571</span>– <span class="lpage" tagx="lpage" title="lpage">8576</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.7314/apjcp.2014.15.20.8571">10.7314/apjcp.2014.15.20.8571</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25374169">25374169</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shahzad</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaskani</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korban</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genetic diversity and population structure of Moringa oleifera</span>. <span class="source" tagx="source" title="source">Conserv Genet</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1161</span>– <span class="lpage" tagx="lpage" title="lpage">1172</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s10592-013-0503-x">10.1007/s10592-013-0503-x</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canini</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Antioxidant extracts of African medicinal plants induce cell cycle arrest and differentiation in B16F10 melanoma cells </span>. <span class="source" tagx="source" title="source">Int J Oncol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="comment">doi: <a href="http://dx.doi.org/10.3892/ijo.2013.2001">10.3892/ijo.2013.2001</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asma</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farooq</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maleeha</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ammara</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Antioxidant Activity of Different Solvent Extracts of Moringa oleifera Leaves under Accelerated Storage of Sunflower Oil </span>. <span class="source" tagx="source" title="source">Asian J Plant Sci</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">630</span>– <span class="lpage" tagx="lpage" title="lpage">635</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.3923/ajps.2005.630.635">10.3923/ajps.2005.630.635</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">IL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Soluble Extract from Moringa oleifera Leaves with a New Anticancer Activity</span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e95492</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0095492">10.1371/journal.pone.0095492</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24748376">24748376</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">High quality reference genome of drumstick tree (Moringa oleifera Lam.), a potential perennial crop </span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">627</span>– <span class="lpage" tagx="lpage" title="lpage">638</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s11427-015-4872-x">10.1007/s11427-015-4872-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26032590">26032590</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNAs: Target Recognition and Regulatory Functions</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">215</span>– <span class="lpage" tagx="lpage" title="lpage">233</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.cell.2009.01.002">10.1016/j.cell.2009.01.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19167326">19167326</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schommer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrington</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Control of leaf morphogenesis by microRNAs</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">425</span>: <span class="fpage" tagx="fpage" title="fpage">257</span>– <span class="lpage" tagx="lpage" title="lpage">263</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nature01958">10.1038/nature01958</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12931144">12931144</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brodersen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakvarelidze-Achard</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruun-Rasmussen</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunoyer</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamamoto</span> <span class="given-names" tagx="given-names" title="given-names">YY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sieburth</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Widespread Translational Inhibition by Plant miRNAs and siRNAs</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">320</span>: <span class="fpage" tagx="fpage" title="fpage">1185</span>– <span class="lpage" tagx="lpage" title="lpage">1190</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1126/science.1159151">10.1126/science.1159151</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18483398">18483398</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delannoy</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sormani</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Floris</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brodersen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crete</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Biochemical Evidence for Translational Repression by Arabidopsis MicroRNAs</span>. <span class="source" tagx="source" title="source">PLANT CELL ONLINE</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1762</span>– <span class="lpage" tagx="lpage" title="lpage">1768</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.108.063412">10.1105/tpc.108.063412</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pulido</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laufs</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Co-ordination of developmental processes by small RNAs during leaf development</span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">1277</span>– <span class="lpage" tagx="lpage" title="lpage">1291</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/jxb/erp397">10.1093/jxb/erp397</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20097843">20097843</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunkar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y-F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jagadeeswaran</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Functions of microRNAs in plant stress responses</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">196</span>– <span class="lpage" tagx="lpage" title="lpage">203</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.tplants.2012.01.010">10.1016/j.tplants.2012.01.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22365280">22365280</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hou</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Exogenous plant MIR168a specifically targets mammalian LDLRAP1: evidence of cross-kingdom regulation by microRNA </span>. <span class="source" tagx="source" title="source">Cell Res</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">107</span>– <span class="lpage" tagx="lpage" title="lpage">126</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/cr.2011.158">10.1038/cr.2011.158</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21931358">21931358</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mlotshwa</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruss</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MacArthur</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endres</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hofseth</span> <span class="given-names" tagx="given-names" title="given-names">LJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A novel chemopreventive strategy based on therapeutic microRNAs produced in plants</span>. <span class="source" tagx="source" title="source">Cell Res</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">521</span>– <span class="lpage" tagx="lpage" title="lpage">524</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/cr.2015.25">10.1038/cr.2015.25</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25721325">25721325</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiang</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zempleni</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Computational Characterization of Exogenous MicroRNAs that Can Be Transferred into Human Circulation </span>. <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, editor. <span class="source" tagx="source" title="source">PLOS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0140587</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0140587">10.1371/journal.pone.0140587</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26528912">26528912</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mendes</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freitas</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sagot</span> <span class="given-names" tagx="given-names" title="given-names">M-F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Current tools for the identification of miRNA genes and their targets</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">2419</span>– <span class="lpage" tagx="lpage" title="lpage">2433</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkp145">10.1093/nar/gkp145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19295136">19295136</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thiebaut</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grativol</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carnavale-Bottino</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojas</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanurdzic</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farinelli</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Computational identification and analysis of novel sugarcane microRNAs</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">290</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2164-13-290">10.1186/1471-2164-13-290</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22747909">22747909</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yanik</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turktas</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dundar</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernandez</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dorado</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Unver</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification of alternate bearing-associated microRNAs (miRNAs) in olive (Olea europaea L.) </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">10</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2229-13-10">10.1186/1471-2229-13-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23320600">23320600</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyers</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Axtell</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baulcombe</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Criteria for Annotation of Plant MicroRNAs</span>. <span class="source" tagx="source" title="source">PLANT CELL ONLINE</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">3186</span>– <span class="lpage" tagx="lpage" title="lpage">3190</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.108.064311">10.1105/tpc.108.064311</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahlgren</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sullivan</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapman</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cumbie</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montgomery</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Computational and analytical framework for small RNA profiling by high-throughput sequencing </span>. <span class="source" tagx="source" title="source">RNA</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">992</span>– <span class="lpage" tagx="lpage" title="lpage">1002</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1261/rna.1473809">10.1261/rna.1473809</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19307293">19307293</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pantaleo</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szittya</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moxon</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miozzi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moulton</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dalmay</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of grapevine microRNAs and their targets using high-throughput sequencing and degradome analysis </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2010</span>; no–no. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313x.2010.04208.x">10.1111/j.1365-313x.2010.04208.x</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Andrew ss. FastQC A Quality Control tool for High Throughput Sequence Data. 2010; Available: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a> </span> </li> <li tag="ref"> <a name="pone.0149495.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Evans</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhagwate</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Middha</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bockol</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">CAP-miRSeq: a comprehensive analysis pipeline for microRNA sequencing data</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">423</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2164-15-423">10.1186/1471-2164-15-423</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24894665">24894665</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackenberg</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Ezpeleta</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aransay</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span>. <span class="mixed-article-title" title="mixed-article-title">miRanalyzer: an update on the detection and analysis of microRNAs in high-throughput sequencing experiments </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">W132</span>– <span class="lpage" tagx="lpage" title="lpage">W138</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkr247">10.1093/nar/gkr247</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21515631">21515631</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">miRDeepFinder: a miRNA analysis tool for deep sequencing of plant small RNAs</span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">75</span>– <span class="lpage" tagx="lpage" title="lpage">84</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s11103-012-9885-2">10.1007/s11103-012-9885-2</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gan</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Modai</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sukacheov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dror</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halperin</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">miRNAkey: a software for microRNA deep sequencing analysis</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2615</span>– <span class="lpage" tagx="lpage" title="lpage">2616</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btq493">10.1093/bioinformatics/btq493</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20801911">20801911</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jha</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shankar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">miReader: Discovering novel miRNAs in species without sequenced genome</span>. <span class="source" tagx="source" title="source">PloS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span> Available: <a href="http://dx.plos.org/10.1371/journal.pone.0066857">http://dx.plos.org/10.1371/journal.pone.0066857</a> </span> </li> <li tag="ref"> <a name="pone.0149495.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gish</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myers</span> <span class="given-names" tagx="given-names" title="given-names">EW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span>. <span class="source" tagx="source" title="source">J Mol Biol</span>. <span class="year" tagx="year" title="year">1990</span>; <span class="volume" tagx="volume" title="volume">215</span>: <span class="fpage" tagx="fpage" title="fpage">403</span>– <span class="lpage" tagx="lpage" title="lpage">410</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2">10.1016/s0022-2836(05)80360-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burge</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daub</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eberhardt</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tate</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barquist</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nawrocki</span> <span class="given-names" tagx="given-names" title="given-names">EP</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Rfam 11.0: 10 years of RNA families</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">D226</span>– <span class="lpage" tagx="lpage" title="lpage">D232</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gks1005">10.1093/nar/gks1005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23125362">23125362</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">PX</span> </span>. <span class="mixed-article-title" title="mixed-article-title">psRNATarget: a plant small RNA target analysis server</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">W155</span>– <span class="lpage" tagx="lpage" title="lpage">W159</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkr319">10.1093/nar/gkr319</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21622958">21622958</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>. <span class="mixed-article-title" title="mixed-article-title">PlantGSEA: a gene set enrichment analysis toolkit for plant community</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">W98</span>– <span class="lpage" tagx="lpage" title="lpage">W103</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkt281">10.1093/nar/gkt281</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23632162">23632162</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coronnello</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benos</span> <span class="given-names" tagx="given-names" title="given-names">PV</span> </span>. <span class="mixed-article-title" title="mixed-article-title">ComiR: combinatorial microRNA target prediction tool</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">W159</span>– <span class="lpage" tagx="lpage" title="lpage">W164</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkt379">10.1093/nar/gkt379</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23703208">23703208</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minutolo</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grelli</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marino-Merlo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cordero</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brandi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macchi</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">D(−)lentiginosine-induced apoptosis involves the intrinsic pathway and is p53-independent</span>. <span class="source" tagx="source" title="source">Cell Death Dis</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e358</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/cddis.2012.97">10.1038/cddis.2012.97</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22833097">22833097</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donaire</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pedrola</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de la Rosa</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Llave</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">High-Throughput Sequencing of RNA Silencing-Associated Small RNAs in Olive (Olea europaea L.) </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e27916</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0027916">10.1371/journal.pone.0027916</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22140484">22140484</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Z-X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ke</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Y-C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">X-B</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">MicroRNA expression profile of mature dendritic cell in chronic rhinosinusitis</span>. <span class="source" tagx="source" title="source">Inflamm Res</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">885</span>– <span class="lpage" tagx="lpage" title="lpage">893</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s00011-015-0870-5">10.1007/s00011-015-0870-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26337346">26337346</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Expression profile and clinical significance of miRNAs at different stages of chronic hepatitis B virus infection </span>. <span class="source" tagx="source" title="source">Int J Clin Exp Med</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">5611</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26131144">26131144</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El Gazzar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCall</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNAs Distinguish Translational from Transcriptional Silencing during Endotoxin Tolerance </span>. <span class="source" tagx="source" title="source">J Biol Chem</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">285</span>: <span class="fpage" tagx="fpage" title="fpage">20940</span>– <span class="lpage" tagx="lpage" title="lpage">20951</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1074/jbc.m110.115063">10.1074/jbc.m110.115063</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20435889">20435889</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">WKK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">ZJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chan</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">CCM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">2,3′,4,4′,5′-Pentamethoxy-trans-stilbene, a resveratrol derivative, inhibits colitis-associated colorectal carcinogenesis in mice </span>. <span class="source" tagx="source" title="source">Br J Pharmacol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">160</span>: <span class="fpage" tagx="fpage" title="fpage">1352</span>– <span class="lpage" tagx="lpage" title="lpage">1361</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1476-5381.2010.00785.x">10.1111/j.1476-5381.2010.00785.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20590626">20590626</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feinbaum</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ambros</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14 </span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1993</span>; <span class="volume" tagx="volume" title="volume">75</span>: <span class="fpage" tagx="fpage" title="fpage">843</span>– <span class="lpage" tagx="lpage" title="lpage">854</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/0092-8674(93)90529-y">10.1016/0092-8674(93)90529-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8252621">8252621</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref041" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messing</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>. <span class="mixed-article-title" title="mixed-article-title">CARPEL FACTORY, a Dicer Homolog, and HEN1, a Novel Protein, Act in microRNA Metabolism in Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">Curr Biol</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1484</span>– <span class="lpage" tagx="lpage" title="lpage">1495</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/s0960-9822(02)01017-5">10.1016/s0960-9822(02)01017-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12225663">12225663</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref042" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNAs in plants</span>. <span class="source" tagx="source" title="source">Genes Dev</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">1616</span>– <span class="lpage" tagx="lpage" title="lpage">1626</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1101/gad.1004402">10.1101/gad.1004402</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12101121">12101121</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref043" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Small RNAs and Their Roles in Plant Development</span>. <span class="source" tagx="source" title="source">Annu Rev Cell Dev Biol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">21</span>– <span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1146/annurev.cellbio.042308.113417">10.1146/annurev.cellbio.042308.113417</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19575669">19575669</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref044" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fahlgren</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kasschau</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapman</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sullivan</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cumbie</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">High-Throughput Sequencing of Arabidopsis microRNAs: Evidence for Frequent Birth and Death of MIRNA Genes </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">e219</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0000219">10.1371/journal.pone.0000219</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17299599">17299599</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref045" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jia</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of the Conserved and Novel miRNAs in Mulberry by High-Throughput Sequencing</span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e104409</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0104409">10.1371/journal.pone.0104409</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25118991">25118991</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref046" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marshall</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryan</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hornyik</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification and Characterization of miRNA Transcriptome in Potato by High-Throughput Sequencing </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e57233</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0057233">10.1371/journal.pone.0057233</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23437348">23437348</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref047" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuchs</span> <span class="given-names" tagx="given-names" title="given-names">RT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhuang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robb</span> <span class="given-names" tagx="given-names" title="given-names">GB</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Bias in Ligation-Based Small RNA Sequencing Library Construction Is Determined by Adaptor and RNA Structure </span>. <span class="source" tagx="source" title="source">PLOS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0126049</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0126049">10.1371/journal.pone.0126049</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25942392">25942392</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref048" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones-Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Conservation and divergence in plant microRNAs</span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">3</span>– <span class="lpage" tagx="lpage" title="lpage">16</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s11103-011-9829-2">10.1007/s11103-011-9829-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21996939">21996939</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref049" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chua</span> <span class="given-names" tagx="given-names" title="given-names">N-H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">ABA induction of miR159 controls transcript levels of two MYB factors during Arabidopsis seed germination </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">592</span>– <span class="lpage" tagx="lpage" title="lpage">606</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02980.x">10.1111/j.1365-313x.2006.02980.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17217461">17217461</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref050" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cardon</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohmann</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nettesheim</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saedler</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huijser</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Functional analysis of the Arabidopsis thaliana SBP-box gene SPL3: a novel gene involved in the floral transition </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">367</span>– <span class="lpage" tagx="lpage" title="lpage">377</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1046/j.1365-313x.1997.12020367.x">10.1046/j.1365-313x.1997.12020367.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9301089">9301089</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref051" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genomic Organization, Differential Expression, and Interaction of SQUAMOSA Promoter-Binding-Like Transcription Factors and microRNA156 in Rice </span>. <span class="source" tagx="source" title="source">PLANT Physiol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">142</span>: <span class="fpage" tagx="fpage" title="fpage">280</span>– <span class="lpage" tagx="lpage" title="lpage">293</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.106.084475">10.1104/pp.106.084475</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16861571">16861571</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref052" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palatnik</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riester</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schommer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmid</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weigel</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Specific Effects of MicroRNAs on the Plant Transcriptome</span>. <span class="source" tagx="source" title="source">Dev Cell</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">517</span>– <span class="lpage" tagx="lpage" title="lpage">527</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.devcel.2005.01.018">10.1016/j.devcel.2005.01.018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15809034">15809034</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref053" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">T-Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lian</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dolezal</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">C-M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">An Intrinsic MicroRNA Timer Regulates Progressive Decline in Shoot Regenerative Capacity in Plants </span>. <span class="source" tagx="source" title="source">Plant Cell Online</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">349</span>– <span class="lpage" tagx="lpage" title="lpage">360</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.114.135186">10.1105/tpc.114.135186</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref054" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mallory</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNA-Directed Regulation of Arabidopsis AUXIN RESPONSE FACTOR17 Is Essential for Proper Development and Modulates Expression of Early Auxin Response Genes </span>. <span class="source" tagx="source" title="source">PLANT CELL ONLINE</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">1360</span>– <span class="lpage" tagx="lpage" title="lpage">1375</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.105.031716">10.1105/tpc.105.031716</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref055" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Floyd</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Gene regulation: Ancient microRNA target sequences in plants</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">428</span>: <span class="fpage" tagx="fpage" title="fpage">485</span>– <span class="lpage" tagx="lpage" title="lpage">486</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/428485a">10.1038/428485a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15057819">15057819</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref056" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rhoades</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhart</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">LP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burge</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartel</span> <span class="given-names" tagx="given-names" title="given-names">DP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Prediction of Plant MicroRNA Targets</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">110</span>: <span class="fpage" tagx="fpage" title="fpage">513</span>– <span class="lpage" tagx="lpage" title="lpage">520</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/s0092-8674(02)00863-2">10.1016/s0092-8674(02)00863-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12202040">12202040</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref057" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otsuga</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeGuzman</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prigge</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drews</span> <span class="given-names" tagx="given-names" title="given-names">GN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span>. <span class="mixed-article-title" title="mixed-article-title">REVOLUTA regulates meristem initiation at lateral positions</span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">236</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1046/j.1365-313x.2001.00959.x">10.1046/j.1365-313x.2001.00959.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11169198">11169198</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref058" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McConnell</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emery</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eshed</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bao</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barton</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Role of PHABULOSA and PHAVOLUTA in determining radial patterning in shoots</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">411</span>: <span class="fpage" tagx="fpage" title="fpage">709</span>– <span class="lpage" tagx="lpage" title="lpage">713</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/35079635">10.1038/35079635</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11395776">11395776</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref059" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z-H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bao</span> <span class="given-names" tagx="given-names" title="given-names">M-L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Y-Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Y-J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X-H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J-H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Regulation of auxin response by miR393-targeted transport inhibitor response protein 1 is involved in normal development in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">77</span>: <span class="fpage" tagx="fpage" title="fpage">619</span>– <span class="lpage" tagx="lpage" title="lpage">629</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s11103-011-9838-1">10.1007/s11103-011-9838-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22042293">22042293</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref060" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bian</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Distinctive expression patterns and roles of the miRNA393/TIR1 homolog module in regulating flag leaf inclination and primary and crown root growth in rice (Oryza sativa) </span>. <span class="source" tagx="source" title="source">New Phytol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">196</span>: <span class="fpage" tagx="fpage" title="fpage">149</span>– <span class="lpage" tagx="lpage" title="lpage">161</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1469-8137.2012.04248.x">10.1111/j.1469-8137.2012.04248.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22846038">22846038</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref061" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dietrich</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hammes</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thor</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suter-Grotemeyer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flückiger</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slusarenko</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">AtPTR1, a plasma membrane peptide transporter expressed during seed germination and in vascular tissue of Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">40</span>: <span class="fpage" tagx="fpage" title="fpage">488</span>– <span class="lpage" tagx="lpage" title="lpage">499</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02224.x">10.1111/j.1365-313x.2004.02224.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15500465">15500465</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref062" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cocuron</span> <span class="given-names" tagx="given-names" title="given-names">J-C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ralph</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilkerson</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The DUF579 domain containing proteins IRX15 and IRX15-L affect xylan synthesis in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">66</span>: <span class="fpage" tagx="fpage" title="fpage">387</span>– <span class="lpage" tagx="lpage" title="lpage">400</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313x.2010.04475.x">10.1111/j.1365-313x.2010.04475.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21288268">21288268</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref063" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poulose</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raju</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Sirtuin regulation in aging and injury</span>. <span class="source" tagx="source" title="source">Biochim Biophys Acta BBA—Mol Basis Dis</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">1852</span>: <span class="fpage" tagx="fpage" title="fpage">2442</span>– <span class="lpage" tagx="lpage" title="lpage">2455</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.bbadis.2015.08.017">10.1016/j.bbadis.2015.08.017</a> </span> </span> </li> <li tag="ref"> <a name="pone.0149495.ref064" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonfloni</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iannizzotto</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiani</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bellusci</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ciccone</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diederich</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">P53 and Sirt1: Routes of metabolism and genome stability</span>. <span class="source" tagx="source" title="source">Biochem Pharmacol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">149</span>– <span class="lpage" tagx="lpage" title="lpage">156</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.bcp.2014.08.034">10.1016/j.bcp.2014.08.034</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25218422">25218422</a> </span> </span> </li> </ul> </div> </div>  </body></html>