<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4917099">4917099</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0157871">10.1371/journal.pone.0157871</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-16-12118</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flowering Plants</span>: <span class="subject" title="subject">Onions</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Agriculture</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Crop Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Crops</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vegetables</span>: <span class="subject" title="subject">Onions</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vegetables</span>: <span class="subject" title="subject">Onions</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Transcriptome Analysis</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Transcriptome Analysis</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Databases</span>: <span class="subject" title="subject">Sequence Databases</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Databases</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Databases</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span>: <span class="subject" title="subject">Database Searching</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Assays and Analysis Techniques</span>: <span class="subject" title="subject">Phylogenetic Analysis</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Assays and Analysis Techniques</span>: <span class="subject" title="subject">Phylogenetic Analysis</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Sequence Assembly Tools</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Sequence Assembly Tools</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database Searching</span>: <span class="subject" title="subject">Sequence Similarity Searching</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Ontologies</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Ontologies</span> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Identification of an NAC Transcription Factor Family by Deep Transcriptome Sequencing in Onion ( <i>Allium cepa</i> L.) </div> <div class="alt-title" title="alt-title">
Identification of Onion NAC Transcription Factor</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Xia</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Shouwei</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Siyuan</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">Qiuzhong</span> </span> <a href="#aff001" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Touming</span> </span> <a href="#cor001">*</a> <a href="#aff001" /> </span> </div> <span class="citation_author_institution" id="aff001">[], <span class="addr-line" title="addr-line">Institute of Bast Fiber Crops and Center of Southern Economic Crops, Chinese Academy of Agricultural Sciences, Changsha, China </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Min</span> <span class="given-names" tagx="given-names" title="given-names">Xiang Jia</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Youngstown State University, UNITED STATES</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-conflict" title="conflict"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: ST TL. Performed the experiments: ST XZ. Analyzed the data: SZ. Contributed reagents/materials/analysis tools: QD. Wrote the paper: TL. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">liutouming@gmail.com</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-6-6</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0157871</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-3-23</span> <span class="accepted" title="accepted">accepted: 2016-6-6</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Zheng et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0157871.pdf" title="self-uri">pone.0157871.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Although onion has been used extensively in the past for cytogenetic studies, molecular analysis has been lacking because the availability of genetic resources is limited. NAM, ATAF, and CUC (NAC) transcription factors (TFs) are plant-specific proteins, and they play key roles in plant growth, development, and stress tolerance. However, none of the onion NAC (CepNAC) genes had been identified thus far. In this study, the transcriptome of onion leaves was analyzed by Illumina paired-end sequencing. Approximately 102.9 million clean sequence reads were produced and used for <i>de novo</i> assembly, which generated 117,189 non-redundant transcripts. Of these transcripts, 39,472 were annotated for their function. In order to mine the CepNAC TFs, CepNAC genes were searched from the transcripts assembled, resulting in the identification of all 39 CepNAC genes. These 39 CepNAC proteins were subjected to phylogenetic analysis together with 47 NAC proteins of known function that were previously identified in other species. The results showed that they can be divided into five groups (NAC-I–V). Interestingly, the NAC-IV and -V groups were found to be likely related to the processes of secondary wall synthesis and stress response, respectively. The transcriptome analysis generated a substantial amount of transcripts, which will aid immensely in identifying important genes and accelerating our understanding of onion growth and development. Moreover, the discovery of 39 CepNAC TFs and the identification of the sequence conservation between them and NAC proteins published will provide a basis for further characterization and validation of their functions in the future. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">the Agricultural Science and Technology Innovation Program of China (ASTIP)</span> </div> <span class="award-id" tagx="award-id" title="award-id">CAAS-ASTIP-IBFC</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">Qiuzhong</span> </span> </span> </div> <div class="funding-statement" title="funding-statement">
This work was supported by Technology Innovation Program of China (grant number: CAAS-ASTIP-IBFC), <a href="http://www.caas.cn/en/research/research_program/index.shtml">http://www.caas.cn/en/research/research_program/index.shtml</a>. The funder had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="Sequencing reads and transcripts assembled are deposited in the NCBI Sequence Read Archive (SRA) and Transcriptome Shotgun Assembly (TSA) Database, under accession number SRA347079 and GEOY00000000, respectively. The other relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Sequencing reads and transcripts assembled are deposited in the NCBI Sequence Read Archive (SRA) and Transcriptome Shotgun Assembly (TSA) Database, under accession number SRA347079 and GEOY00000000, respectively. The other relevant data are within the paper and its Supporting Information files. </p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Transcription factors (TFs) are proteins that regulate gene expression by binding specific DNA sequences in the regulatory region of target genes, thereby modulating RNA polymerase activity for the initiation of transcription. NAM, ATAF, and CUC (NAC) TFs are one kind of proteins that only exist in plant. The name of this family was originally derived from the names of three proteins: no apical meristem (NAM), ATAF1-2, and CUC2 (cup-shaped cotyledon) [ <a href="#pone.0157871.ref001">1</a>, <a href="#pone.0157871.ref002">2</a>]. NAC TFs have a highly conserved DNA-binding domain in the N-terminal region and a variable functional domain in the C-terminal region [ <a href="#pone.0157871.ref003">3</a>, <a href="#pone.0157871.ref004">4</a>]. Generally, this variable C-terminal domain can function as transcriptional activator or repressor [ <a href="#pone.0157871.ref005">5</a>– <a href="#pone.0157871.ref007">7</a>]. The NAC family is a large gene family and comprises a large number of protein members in plants, and at least 151, 117, 163, and 101 non-redundant NAC TF protein members have been identified so far in rice, <i>Arabidopsis</i>, <i>Populus</i>, and soybean genomes, respectively [ <a href="#pone.0157871.ref008">8</a>– <a href="#pone.0157871.ref010">10</a>]. </p> <p>At present, there are at least 50 NAC genes that have been functionally characterized [ <a href="#pone.0157871.ref011">11</a>, <a href="#pone.0157871.ref012">12</a>, <a href="#pone.0157871.ref013">13</a>], and they are involved in many aspects of plant growth and development, including embryo development, seed germination, lateral root development, maintenance of the shoot apical meristem, flower formation, plant immune response, cell division and growth, senescence, and formation of secondary walls [ <a href="#pone.0157871.ref003">3</a>, <a href="#pone.0157871.ref011">11</a>]. Moreover, NAC TFs were also found to participate in regulating of plant tolerance to abiotic/biotic stress [ <a href="#pone.0157871.ref014">14</a>, <a href="#pone.0157871.ref015">15</a>]. Therefore, NAC family is multifunctional proteins family with various roles in regulating plant growth and development. </p> <p>Onion ( <i>Allium cepa</i>) is one of the most important vegetable crops of the <i>Allium</i> family; it is known for its nutritional and medicinal properties and has been widely planted around the world with more than 5000 years [ <a href="#pone.0157871.ref016">16</a>]. Onion is a diploid (2n = 16) plant and has a large nuclear genome of approximately 16.4 Gb per 1C, which is about 38 and 130 times larger than the rice and <i>Arabidopsis</i> genomes, respectively [ <a href="#pone.0157871.ref017">17</a>, <a href="#pone.0157871.ref018">18</a>]. Molecular analysis in onion has been hindered by the limited availability of genetic resources, as a result of the onion’s large nuclear genome, which presents challenges for gene identification. Therefore, while the onion has been used extensively in previous studies for cytogenetic analysis, the molecular analysis of onion genes has been lacking. In recent years, the rise of next generation sequencing (NGS) technologies has offered a powerful and cost-efficient tool for high-throughput sequence determination. Transcriptome analysis based on this technology has led to the discovery of new genes in hundreds of species [ <a href="#pone.0157871.ref019">19</a>– <a href="#pone.0157871.ref023">23</a>]. However, no studies to date have identified NAC family genes in onion. </p> <p>In the present study, we have <i>de novo</i> assembled and characterized the transcriptome of onion leaves using Illumina paired-end sequencing technology and searched onion NAC (CepNAC) genes from the assembled transcripts. Furthermore, we have performed phylogenetic analysis of the CepNAC TFs identified in this study along with 47 NAC proteins of known function that were previously identified in other species ( <a href="#pone.0157871.s003">S1 Table</a>). The discovery of these <i>CepNAC</i> genes will provide a basis for further understanding of their function in onion growth and development. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="plantmaterialandrnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Plant material and RNA extraction</div> <p>The study included a local variety of onion (Chalinghuangpi) that produces yellow-skin bulbs and was collected from Chaling, Hunan, China. The onion was planted in the experimental field of the Institute of Bast Fiber Crops, Chinese Academy of Agricultural Sciences, Changsha, China, on Sept. 20, 2014. On Mar. 10, 2015, leaves tissues of three individuals were sampled. The tissue sample was immediately frozen in liquid nitrogen, and then stored at −80°C until use. E.Z.N.A. Plant RNA Kit (OMEGA Bio-Tek, USA) was used to extract the total RNA according to the manufacturer’s protocol. </p> </div> <div class="cdnalibraryconstruction,sequencing,andassembly" title="sec"> <div class="title" tagx="title" title="title">
cDNA library construction, sequencing, and assembly</div> <p>Illumina sequencing was performed at Novogene Bioinformatics Technology Co., Ltd, Beijing, China ( <a href="http://www.novogene.cn">www.novogene.cn</a>). RNA of onion leaves was used to construct the cDNA libraries with fragment lengths of approximately 250 bp (±25 bp). Paired-end sequencing was then performed using the Illumina sequencing platform (HiSeq™ 2500) according to the manufacturer’s instructions (Illumina, San Diego, CA). After trimming adapter sequences and filtering low-quality reads, the clean reads were used to search against NCBI non-redundant protein sequences (NR) database by BLAST (Basic Local Alignment Search Tool) with an E-value threshold of 10 <sup>−1</sup>, and the reads that showed significant similarity with other species might be the sequences derived from other species, thereby being removed. Thereafter, the remaining clean reads were used to <i>de novo</i> assemble the onion transcriptome using Trinity [ <a href="#pone.0157871.ref024">24</a>]. Sequencing reads and transcripts assembled are deposited in the NCBI Sequence Read Archive (SRA) and Transcriptome Shotgun Assembly (TSA) Database, under accession number SRA347079 and GEOY00000000, respectively. </p> </div> <div class="geneannotationandclassification" title="sec"> <div class="title" tagx="title" title="title">
Gene annotation and classification</div> <p>To annotate the transcripts of onion transcriptome using bioinformatics approachs, each of the assembled transcripts was searched against seven public databases, including KEGG Orthology (KO) database, protein family (PFAM) database, eukaryotic orthologous groups (KOG) database, NR database, NCBI nucleotide sequences (NT) database, Swiss-Prot protein database, and Gene Ontology (GO) database. WEGO software was used to perform GO functional classification of all transcripts to visualize gene functions distribution. Transcripts annotated in the KOG database were classified according to the COG group that each transcript was assigned to, whereas transcripts annotated from the KO database were grouped according to the KEGG pathway that gene was assigned to. </p> </div> <div class="identificationofnacgenesandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Identification of NAC genes and phylogenetic analysis</div> <p>In order to identify the NAC genes, the keywords ‘‘NAC," ‘‘no apical meristem," or ‘‘NAM" were used as the query to search against annotations of the onion transcripts assembled. ORFs of identified NAC genes were screened using the findorf software. All non-redundant genes identified were further validated as NAC genes by manually checking for their conserved NAM domain. In addition, information regarding 47 functionally characterized NAC genes was collected from previous studies conducted in other plant species, and can be found in <a href="#pone.0157871.s003">S1 Table</a>. We performed the phylogenetic analysis on these 47 NAC proteins of known function, together with the CepNAC proteins identified in this study. Clustal X (version 1.83) program [ <a href="#pone.0157871.ref025">25</a>] was used to perform multiple sequence alignments of full-length protein sequences, including the highly conserved N-terminal NAM domain and the more divergent C-terminal activation domain. Neighbor-joining (NJ) method was used to construct an unrooted phylogenetic tree by using MEGA 4.0 [ <a href="#pone.0157871.ref026">26</a>], and the bootstrap analysis was performed with 1000 replicates. NAC proteins of known function that had closest evolutionary relationship to each CepNAC protein were ascertained based on the phylogenetic tree, and their percentage similarity analyzed using the BLASTP program. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="illuminapaired-endsequencinganddenovoassembly" title="sec"> <div class="title" tagx="title" title="title">
Illumina paired-end sequencing and <i>de novo</i> assembly </div> <p>RNA extracted from onion leaves was used to construct a library of cDNA sequences. In all 102.9 million clean sequence reads with lengths of 125 bp were sequenced from the library. Next, these sequence reads were used for the <i>de novo</i> assembly using Trinity software. Sequences that were not extended on either end were defined as transcripts. Eventually, a total of 117,189 non-redundant transcripts were assembled. The average length and total length of these <i>de novo</i> assembled transcripts were 632 bp and 74.03 Mb, respectively ( <a href="#pone.0157871.t001">Table 1</a>). The length was between 500 and 1000 for 21,559 (18.4%) transcripts, between 1001 and 2000 bp for 13,322 (11.4%) transcripts, more than 2000 bp for 6,008 (5.1%) transcripts, and no more than 500 bp for all remaining transcripts ( <a href="#pone.0157871.g001">Fig 1</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0157871.t001">10.1371/journal.pone.0157871.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary of transcriptomes assembled by Illumina sequencing.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <tbody> <tr> <td>Clear reads</td> <td>102,874,740</td> </tr> <tr> <td>Clean bases (Gb)</td> <td>12.86</td> </tr> <tr> <td>Transcript number</td> <td>117,189</td> </tr> <tr> <td>Total length of transcript (Mb)</td> <td>74.03</td> </tr> <tr> <td>Average length of transcript (bp)</td> <td>632</td> </tr> <tr> <td>Transcript number annotated</td> <td>39,472 (33.68%)</td> </tr> </tbody> </table> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Length distribution of assembled transcripts.</div>   </div> </div> <div class="functionalannotation" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation</div> <p>To validate and annotate these assembled transcripts, sequence similarity searches were performed against in seven public databases. Of 117,189 transcripts, 32,919 (28.1%) showed significant similarity to known proteins in the NCBI non-redundant protein sequence (NR) database, 11,141 (9.5%) in the NCBI nucleotide sequence (NT) database, 10,741 (9.2%) in the KEGG orthology (KO) database, 22,543 (19.2%) in the SwissProt database, 24,578 (21.0%) in the protein family (PFAM) database, 24,672 (21.1%) in the Gene Ontology (GO) database, and 11,175 (9.5%) in the eukaryotic orthologous groups (KOG) database ( <a href="#pone.0157871.g002">Fig 2</a>). Finally, 39,472 (33.7%) transcripts showed similarity to known proteins in at least one of these seven databases, and were achieved for annotating function. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Transcript numbers annotated in the seven public databases searched.</div>   </div> </div> <div class="functionalclassification" title="sec"> <div class="title" tagx="title" title="title">
Functional classification</div> <p>Functional classification of assembled transcripts was performed by BLAST searching against GO, KOG and KEGG databases. The results of the GO analysis showed matches to known proteins of 56 GO classes for all 24,672 transcripts, with a total of 125,547 functional terms. The majority of these GO functional terms consisted of assignments to the biological process ontology (59,019; 47.0%), followed by cellular component (37,756; 30.1%) and molecular function (28,772; 22.9%) ontologies ( <a href="#pone.0157871.g003">Fig 3</a>). KOG functional classification resulted in the assignment of 11,175 to all 25 KOG categories. Some transcripts were assigned to several KOG categories, which led to a total of 12,517 sequences being classified into 25 KOG categories ( <a href="#pone.0157871.g004">Fig 4</a>). In addition, on using BLASTx with an E-value threshold of 10 <sup>−5</sup> against the KEGG database, 10,741 transcript sequences matched a total of 12,035 functional terms in the database and were assigned to 32 KEGG categories ( <a href="#pone.0157871.g005">Fig 5</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Gene ontology classifications of assembled transcripts; the red, blue, and green histogram bar represented three main categories: biological process, molecular function, and cellular component. </div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Histogram presentation of clusters of eukaryotic ortholog groups (KOG) classification.</div>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Classifications of transcripts according to the matching KEGG pathway; A, B, C, D, and E represent five main categories: cellular processes, environmental information processing, genetic information processing, metabolism, and organismal systems, respectively. </div>   </div> </div> <div class="identificationofthecepnacgenes" title="sec"> <div class="title" tagx="title" title="title">
Identification of the CepNAC genes</div> <p>The <i>NAC</i> genes of onion were discovered by searching annotated sequences among the assembled transcripts. A total of 39 <i>CepNAC</i> genes were identified, and were named <i>CepNAC01</i> through <i>CepNAC39</i> ( <a href="#pone.0157871.t002">Table 2</a>). Among these 39 genes, 35 contained the full-length open reading frame (ORF), and the other 4 genes were partial ( <a href="#pone.0157871.t002">Table 2</a>). The proteins encoded by these 4 genes with partial sequence contained the complete NAM domain. The DNA sequences of <i>CepNAC</i> genes and their predicted protein sequences are shown in <a href="#pone.0157871.s002">S1 File</a>. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0157871.t002">10.1371/journal.pone.0157871.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
The information of NAC genes of onion.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Gene <a href="#t002fn001"> <sup>a</sup> </a> </th> <th>Gene length</th> <th>PSC <a href="#t002fn002"> <sup>b</sup> </a> </th> <th>PEC <a href="#t002fn002"> <sup>b</sup> </a> </th> <th>Predicted protein</th> <th>The most similarity gene</th> <th>Group</th> </tr> <tr> <th /> <th>(bp)</th> <th>(bp)</th> <th>(bp)</th> <th>(aa)</th> <th>Gene</th> <th>RS <a href="#t002fn003"> <sup>c</sup> </a>(%) </th> <th /> </tr> </thead> <tbody> <tr> <td> <i>CepNAC01</i> * </td> <td>995</td> <td>1</td> <td>993</td> <td>331</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC02</i> </td> <td>1018</td> <td>30</td> <td>845</td> <td>272</td> <td> <i>ANAC036</i> </td> <td>47</td> <td>III</td> </tr> <tr> <td> <i>CepNAC03</i> </td> <td>1349</td> <td>125</td> <td>1078</td> <td>318</td> <td> <i>ENAC1</i> </td> <td>44</td> <td>III</td> </tr> <tr> <td> <i>CepNAC04</i> </td> <td>1289</td> <td>56</td> <td>796</td> <td>247</td> <td> <i>OsNAC4</i> </td> <td>80</td> <td>V</td> </tr> <tr> <td> <i>CepNAC05</i> </td> <td>1291</td> <td>135</td> <td>1034</td> <td>300</td> <td> <i>ORE1</i> </td> <td>67</td> <td>II</td> </tr> <tr> <td> <i>CepNAC06</i> </td> <td>2081</td> <td>71</td> <td>1822</td> <td>584</td> <td> <i>RIM1</i> </td> <td>61</td> <td>III</td> </tr> <tr> <td> <i>CepNAC07</i> </td> <td>3456</td> <td>802</td> <td>1299</td> <td>166</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC08</i> </td> <td>1204</td> <td>74</td> <td>973</td> <td>300</td> <td> <i>OsNAC10</i> </td> <td>56</td> <td>V</td> </tr> <tr> <td> <i>CepNAC09</i> </td> <td>3028</td> <td>44</td> <td>2371</td> <td>776</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC10</i> </td> <td>3167</td> <td>90</td> <td>1097</td> <td>336</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC11</i> </td> <td>1122</td> <td>93</td> <td>989</td> <td>299</td> <td> <i>NAC1</i> </td> <td>47</td> <td>II</td> </tr> <tr> <td> <i>CepNAC12</i> </td> <td>2373</td> <td>198</td> <td>2138</td> <td>647</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC13</i> </td> <td>1130</td> <td>103</td> <td>954</td> <td>284</td> <td> <i>NAC1</i> </td> <td>46</td> <td>II</td> </tr> <tr> <td> <i>CepNAC14</i> </td> <td>1201</td> <td>49</td> <td>1026</td> <td>326</td> <td /> <td /> <td>V</td> </tr> <tr> <td> <i>CepNAC15</i> </td> <td>1689</td> <td>34</td> <td>1416</td> <td>461</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC16</i> </td> <td>1569</td> <td>340</td> <td>1074</td> <td>245</td> <td> <i>OsNAC52</i> </td> <td>58</td> <td>V</td> </tr> <tr> <td> <i>CepNAC17</i> </td> <td>2027</td> <td>935</td> <td>1864</td> <td>310</td> <td> <i>PtrWND6B</i> </td> <td>51</td> <td>IV</td> </tr> <tr> <td> <i>CepNAC18</i> * </td> <td>768</td> <td>86</td> <td>766</td> <td>227</td> <td> <i>RIM1</i> </td> <td>60</td> <td>III</td> </tr> <tr> <td> <i>CepNAC19</i> </td> <td>1417</td> <td>250</td> <td>978</td> <td>243</td> <td /> <td /> <td /> </tr> <tr> <td> <i>CepNAC20</i> </td> <td>1878</td> <td>97</td> <td>1338</td> <td>414</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC21</i>* </td> <td>675</td> <td>2</td> <td>673</td> <td>224</td> <td /> <td /> <td /> </tr> <tr> <td> <i>CepNAC22</i> </td> <td>1040</td> <td>121</td> <td>855</td> <td>245</td> <td /> <td /> <td>II</td> </tr> <tr> <td> <i>CepNAC23</i> </td> <td>1265</td> <td>50</td> <td>610</td> <td>187</td> <td> <i>OsNAC4</i> </td> <td>82</td> <td>V</td> </tr> <tr> <td> <i>CepNAC24</i> </td> <td>1095</td> <td>215</td> <td>931</td> <td>239</td> <td /> <td /> <td>II</td> </tr> <tr> <td> <i>CepNAC25</i> </td> <td>1163</td> <td>372</td> <td>1091</td> <td>240</td> <td /> <td /> <td>II</td> </tr> <tr> <td> <i>CepNAC26</i> </td> <td>2046</td> <td>134</td> <td>1774</td> <td>547</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC27</i> </td> <td>1217</td> <td>245</td> <td>1018</td> <td>258</td> <td /> <td /> <td>II</td> </tr> <tr> <td> <i>CepNAC28</i> </td> <td>1143</td> <td>84</td> <td>551</td> <td>156</td> <td> <i>ENAC1</i> </td> <td>61</td> <td>III</td> </tr> <tr> <td> <i>CepNAC29</i> </td> <td>1618</td> <td>40</td> <td>489</td> <td>150</td> <td /> <td /> <td>V</td> </tr> <tr> <td> <i>CepNAC30</i> </td> <td>1239</td> <td>64</td> <td>1215</td> <td>384</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC31</i> </td> <td>2037</td> <td>451</td> <td>1713</td> <td>421</td> <td> <i>FSQ6</i> </td> <td>41</td> <td>III</td> </tr> <tr> <td> <i>CepNAC32</i> </td> <td>2120</td> <td>157</td> <td>1953</td> <td>599</td> <td> <i>NTL4</i> </td> <td>32</td> <td>III</td> </tr> <tr> <td> <i>CepNAC33</i> </td> <td>1513</td> <td>268</td> <td>1143</td> <td>292</td> <td /> <td /> <td /> </tr> <tr> <td> <i>CepNAC34</i> </td> <td>1648</td> <td>118</td> <td>1407</td> <td>430</td> <td /> <td /> <td>I</td> </tr> <tr> <td> <i>CepNAC35</i> </td> <td>802</td> <td>30</td> <td>608</td> <td>193</td> <td> <i>XND1</i> </td> <td>51</td> <td /> </tr> <tr> <td> <i>CepNAC36</i> </td> <td>1172</td> <td>298</td> <td>948</td> <td>217</td> <td /> <td /> <td /> </tr> <tr> <td> <i>CepNAC37</i> </td> <td>2577</td> <td>201</td> <td>2180</td> <td>660</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC38</i> </td> <td>3414</td> <td>296</td> <td>3109</td> <td>938</td> <td /> <td /> <td>III</td> </tr> <tr> <td> <i>CepNAC39</i>* </td> <td>517</td> <td>32</td> <td>517</td> <td>162</td> <td> <i>NAM</i> </td> <td>81</td> <td>V</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>a</sup> the asterisk indicated that the gene with partial cDNA sequence. </p> </div> <div class="fn-type-" title=""> <p> <sup>b</sup> PSC and PEC represented the positions of starting and ending codon in cDNA sequence. </p> </div> <div class="fn-type-" title=""> <p> <sup>c</sup> RS indicated the ratio of similarity of the protein sequence. </p> </div> <div class="fn-type-" title=""> <p>^#</p> </div> </div> </div> </div> <div class="phylogeneticanalysisofcepnacproteinsandfunctionallycharacterizednacproteinsidentifiedinotherspecies" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis of CepNAC proteins and functionally characterized NAC proteins identified in other species </div> <p>All 39 CepNAC protein sequences identified in this study, together with 47 functionally characterized NAC proteins of other species, were subjected to phylogenetic analysis. The results suggested that the 86 NAC proteins can be classified into five groups (I–V, <a href="#pone.0157871.g006">Fig 6</a>). A total of 6, 7, 13, 1, and 7 CepNAC members were classified into NAC-I, II, III, IV, and V, respectively ( <a href="#pone.0157871.t002">Table 2</a>, <a href="#pone.0157871.g006">Fig 6</a>). Moreover, a total of 8, 9, 9, and 16 NAC proteins of known function were assigned to the NAC-II, III, IV and V subfamilies, respectively, whereas none of the NAC TFs reported fell into the NAC-I group ( <a href="#pone.0157871.g006">Fig 6</a>). In addition, five CepNAC members and five NAC proteins of known function were not assigned to any phylogenetic group ( <a href="#pone.0157871.g006">Fig 6</a>). Interestingly, all 9 NAC TFs reported to be in the NAC-IV subfamily function as secondary wall synthesis regulators, and all 16 NAC TFs reported to be in the NAC-V group function as stress tolerance modulators ( <a href="#pone.0157871.g006">Fig 6</a>; <a href="#pone.0157871.s003">S1 Table</a>). This suggests that the NAC-IV and V subfamilies might be functionally related to the processes of secondary wall synthesis and stress response, respectively. On the other hand, the function of NAC TFs reported in the NAC-II and III groups was very diverse ( <a href="#pone.0157871.s003">S1 Table</a>). Seventeen CepNAC TFs showed highest homology to the functionally characterized NAC proteins reported in other species, with a sequence similarity ratio ranging from 32% to 82% ( <a href="#pone.0157871.t002">Table 2</a>). For 7 of those CepNAC proteins, the similarity ratio with the NAC proteins of known function was more than 60%. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0157871.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Phylogenetic tree of 39 CepNAC proteins and 47 NAC proteins published.</div> <p>Neighbor-joining method was used for constructing this unrooted tree by using the MEGA 4 program. Bootstrap values from 1000 replicates are indicated at each node, and are displayed in <a href="#pone.0157871.s001">S1 Fig</a>. </p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="characterizationofthetranscriptomeofonionleaves" title="sec"> <div class="title" tagx="title" title="title">
Characterization of the transcriptome of onion leaves</div> <p>Because of its cost-efficiency, high throughput and speed, NGS technology offers several advantages over traditional Sanger sequencing, which is costly, time-consuming, and laborious. NGS has been proven a powerful tool in several researching areas, including re-sequencing, micro-RNA expression profiling, DNA methylation, and especially <i>de novo</i> transcriptome sequencing for non-model organisms [ <a href="#pone.0157871.ref027">27</a>]. Based on this technology, the transcriptomes of hundreds of species, including non-model organisms, were characterized [ <a href="#pone.0157871.ref022">22</a>], and a large number of expressed genes have been discovered, thereby accelerating our understanding of the complexity of expression, regulation and networks of gene in model and non-model organisms. </p> <p>Onion is an economically important crop. By Sanger sequencing for the cDNA library, a total of 11,008 unique ESTs obtained from this crop [ <a href="#pone.0157871.ref028">28</a>], which first provided an insight into onion expressed genes. In a recent study, Rajkumar et al. <i>de novo</i> assembled and characterized the red bulb onion transcriptome, generating a total of 293,475 genes with a total length of 281 Mb [ <a href="#pone.0157871.ref016">16</a>]. Obviously, the number of genes assembled in the above study is enormous, far greater than that of other <i>Allium</i> species, such as garlic (135,360 transcripts), Chinese chive (60,031 transcripts), and Welsh onion (53,837 transcripts) [ <a href="#pone.0157871.ref029">29</a>– <a href="#pone.0157871.ref031">31</a>]. The transcriptome characterized in this previous study was also huge (almost two times the genome size of <i>Arabidopsis</i>). Probably, there are some redundant genes in the previous onion transcriptome. In addition, as only the onion bulb was used for transcriptome analysis [ <a href="#pone.0157871.ref016">16</a>], genes that are specifically expressed in other tissues such as leaf were not identified. </p> <p>In the present study, we have described the transcriptome analysis in yellow bulb onion leaves, and identified 117,189 non-redundant transcripts. However, among these transcripts, there were only 33.7% of them achieved for functional annotation. This low ratio might be caused by the large size of onion transcriptome. Our result revealed that onion had a transcriptome of about 74 MB. Because of its large size, it is difficult to assemble for the onion transcriptome. Therefore, although that the average sequencing depth of coverage in the transcriptome was more than 170 folds in this study, many small sequence fragments still were generated, and they were difficult for annotating the function. Actually, a similar low ratio for the annotated transcripts had been observed in onion transcriptome in previous study [ <a href="#pone.0157871.ref016">16</a>]. There were substantial amount of transcripts obtained in this study, which will immensely help us to explore the major genes and accelerate our understanding in the onion growth and development. </p> </div> <div class="identificationof39cepnactfs" title="sec"> <div class="title" tagx="title" title="title">
Identification of 39 CepNAC TFs</div> <p>There are 8,133 NAC genes that have been identified from 74 species in the PlantTFDB database [ <a href="#pone.0157871.ref032">32</a>]. Previous studies revealed that NAC TFs are involved in many aspects of plant growth and development, including modulation of senescence, secondary wall synthesis, stress tolerance, and embryogenesis [ <a href="#pone.0157871.ref011">11</a>, <a href="#pone.0157871.ref014">14</a>, <a href="#pone.0157871.ref015">15</a>]. However, no study to date has identified NAC TFs family members in onion. In the present study, a total of 39 CepNAC TFs were identified for the first time. Sequence analysis revealed marked homology of 17 CepNAC TFs to NAC proteins of known function reported in other species. Among those, 7 showed more than 60% sequence similarity with the NAC proteins previously reported. The high homology in protein sequence will provide further insights into the molecular mechanisms in which CepNAC TFs are involved. </p> <p>Phylogenetic analysis was performed for 39 CepNAC TFs and 47 functionally characterized NAC proteins reported in previous studies. Among these 47 NAC proteins of known function, 10 were found to be involved in secondary wall synthesis. The results of the phylogenetic analysis revealed that 9 out of 10 NAC genes involved in secondary wall synthesis and 1 CepNAC TF (CepNAC17) were assigned to the NAC-IV subfamily, suggesting that CepNAC17 TFs share a conserved sequence with NAC TFs responsible for secondary wall synthesis. Moreover, CepNAC35 showed 51% homology with XND1, a regulator of secondary wall synthesis in <i>Arabidopsis</i> [ <a href="#pone.0157871.ref033">33</a>]. Most likely, CepNAC17 and CepNAC35 have a potential role in secondary wall synthesis in onion. In addition, 16 NAC proteins of known function fell into the NAC-V subfamily. Interestingly, all these 16 NAC TFs were found to be involved in the regulation of stress tolerance ( <a href="#pone.0157871.s003">S1 Table</a>), suggesting that this NAC group might be a stress-responsive NAC subfamily. In light of our results, this stress-responsive NAC subfamily would contain 7 CepNAC members. </p> <p>In conclusion, 39 CepNAC genes were identified for the first time in this study. Analysis of sequence conservation revealed that 2 CepNAC TFs had high homology with NAC TFs of other species that functioned as secondary wall synthesis regulators, and phylogenetic analysis found that 7 CepNAC TFs were contained in the stress-responsive NAC subfamily. The identification of the CepNAC TFs reported in this study will provide a basis for further characterization and validation of their function in the future. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Rectangular phylogenetic tree of NAC proteins with bootstrap values.</div> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0157871.s001.TIF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 File</span>  <div class="title" tagx="title" title="title">
cDNA sequences of 39 <i>CepNAC</i> identified genes and their predicted protein products. </div> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0157871.s002.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Details of the 47 functionally characterized NAC genes previously reported in other species. </div> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0157871.s003.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We acknowledge the Novogene Bioinformatics Institute for the assistance with original data processing and related bioinformatics analysis. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0157871.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujisawa</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tasaka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genes involved in organ separation in Arabidopsis: an analysis of the cup-shaped cotyledon mutant </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">841</span>– <span class="lpage" tagx="lpage" title="lpage">857</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9212461">9212461</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Souer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Houwelingen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kloos</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mol</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koes</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The no apical meristem gene of Petunia is required for pattern formation in embryos and flowers and is expressed at meristem and primordia boundaries </span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">85</span>: <span class="fpage" tagx="fpage" title="fpage">159</span>– <span class="lpage" tagx="lpage" title="lpage">170</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8612269">8612269</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ernst</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leggio</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skriver</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors: structurally distinct, functionally diverse</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">87</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15708345">15708345</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ernst</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olsen</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larsen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leggio</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Structure of the conserved domain of ANAC, a member of the NAC family of transcription factors </span>. <span class="source" tagx="source" title="source">EMBO Rep</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">297</span>– <span class="lpage" tagx="lpage" title="lpage">303</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15083810">15083810</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Overexpressing a NAM, ATAF, and CUC (NAC) transcription factor enhances drought resistance and salt tolerance in rice </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">103</span>: <span class="fpage" tagx="fpage" title="fpage">12987</span>– <span class="lpage" tagx="lpage" title="lpage">12992</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16924117">16924117</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A membrane-associated NAC transcription factor regulates salt-responsive flowering via FLOWERING LOCUS T in Arabidopsis </span>. <span class="source" tagx="source" title="source">Planta</span> <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">226</span>: <span class="fpage" tagx="fpage" title="fpage">647</span>– <span class="lpage" tagx="lpage" title="lpage">654</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17410378">17410378</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakuma</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujita</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruyama</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation and functional analysis of Arabidopsis stress inducible NAC transcription factors that bind to a drought responsive cis-element in the early responsive to dehydration stress 1 promoter </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">2481</span>– <span class="lpage" tagx="lpage" title="lpage">2498</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15319476">15319476</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Comprehensive Analysis of NAC Domain Transcription Factor Gene Family in Populus trichocarpa</span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">145</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2229-10-145">10.1186/1471-2229-10-145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20630103">20630103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinheiro</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marques</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Costa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reis</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alves</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carvalho</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Complete inventory of soybean NAC transcription factors: Sequence conservation and expression analysis uncover their distinct roles in stress response </span>. <span class="source" tagx="source" title="source">Gene</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">444</span>: <span class="fpage" tagx="fpage" title="fpage">10</span>– <span class="lpage" tagx="lpage" title="lpage">23</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.gene.2009.05.012">10.1016/j.gene.2009.05.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19497355">19497355</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nuruzzaman</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manimekalai</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharoni</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satoh</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kondoh</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ooka</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide analysis of NAC transcription factor family in rice</span>. <span class="source" tagx="source" title="source">Gene</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">465</span>: <span class="fpage" tagx="fpage" title="fpage">30</span>– <span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.gene.2010.06.008">10.1016/j.gene.2010.06.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20600702">20600702</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of 32 full-length NAC transcription factors in ramie ( <i>Boehmeria nivea</i> L. Gaud) and characterization of the expression pattern of these genes </span>. <span class="source" tagx="source" title="source">Mol Genet Genomics</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">289</span>: <span class="fpage" tagx="fpage" title="fpage">675</span>– <span class="lpage" tagx="lpage" title="lpage">684</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s00438-014-0842-4">10.1007/s00438-014-0842-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24691727">24691727</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puranik</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prasad</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NAC proteins: regulation and role in stress tolerance</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">369</span>– <span class="lpage" tagx="lpage" title="lpage">381</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.tplants.2012.02.004">10.1016/j.tplants.2012.02.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22445067">22445067</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xing</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Two NAC transcription factors from <i>Caragana intermedia</i> altered salt tolerance of the transgenic Arabidopsis </span>. <span class="source" tagx="source" title="source">BMC Plant Biol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">208</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/s12870-015-0591-5">10.1186/s12870-015-0591-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26297025">26297025</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohtani</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toyooka</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wakazaki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Contribution of NAC transcription factors to plant adaptation to land</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">343</span>: <span class="fpage" tagx="fpage" title="fpage">1505</span>– <span class="lpage" tagx="lpage" title="lpage">1508</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1126/science.1248417">10.1126/science.1248417</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24652936">24652936</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takasaki</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">unya Mizoi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NAC transcription factors in plant abiotic stress responses</span>. <span class="source" tagx="source" title="source">Biochimica et Biophysica Acta</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">1819</span>: <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">103</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.bbagrm.2011.10.005">10.1016/j.bbagrm.2011.10.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22037288">22037288</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajkumar</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramagoni</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anchoju</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vankudavath</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Syed</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome analysis of <i>Allium cepa</i> L. (Onion) bulb to identify allergens and epitopes </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0135387</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0135387">10.1371/journal.pone.0135387</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26284934">26284934</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arumuganathan</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Earle</span> <span class="given-names" tagx="given-names" title="given-names">ED</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Nuclear DNA content of some important plant species</span>. <span class="source" tagx="source" title="source">Plant Mol Biol Rep</span>. <span class="year" tagx="year" title="year">1991</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">208</span>– <span class="lpage" tagx="lpage" title="lpage">218</span>. </span> </li> <li tag="ref"> <a name="pone.0157871.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCallum</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samantha</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masayoshi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yanbo</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sjaak van</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pither-joyce</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">AlliumMap-A comparative genomics resource for cultivated Allium vegetables</span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">168</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2164-13-168">10.1186/1471-2164-13-168</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22559261">22559261</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of transcriptome using Illumina paired-end sequencing and identification of <i>CesA</i> gene in ramie ( <i>Boehmeria nivea</i> L. Gaud) </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/1471-2164-14-125">10.1186/1471-2164-14-125</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23442184">23442184</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome comparison reveals the patterns of selection in domesticated and wild ramie ( <i>Boehmeria nivea L</i>. Gaud) </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">86</span>: <span class="fpage" tagx="fpage" title="fpage">85</span>– <span class="lpage" tagx="lpage" title="lpage">92</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s11103-014-0214-9">10.1007/s11103-014-0214-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24934879">24934879</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lou</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing and metabolite analysis reveals the role of delphinidin metabolism in flower colour in grape hyacinth </span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">3157</span>– <span class="lpage" tagx="lpage" title="lpage">3164</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/jxb/eru168">10.1093/jxb/eru168</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24790110">24790110</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mutz</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heilkenbrinker</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lonne</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walter</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stahl</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis using next-generation sequencing</span>. <span class="source" tagx="source" title="source">Curr Opin Biotechnol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">22</span>– <span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.copbio.2012.09.004">10.1016/j.copbio.2012.09.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23020966">23020966</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koenig</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiménez-Gómez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimura</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fulopa</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chitwooda</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Headlanda</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">110</span>: <span class="fpage" tagx="fpage" title="fpage">E2655</span>– <span class="lpage" tagx="lpage" title="lpage">E2662</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1073/pnas.1309606110">10.1073/pnas.1309606110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23803858">23803858</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yassour</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amit</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Full length transcriptome assembly from RNA-Seq data without a reference genome</span>. <span class="source" tagx="source" title="source">Nature Biotechnol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">644</span>– <span class="lpage" tagx="lpage" title="lpage">652</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plewniak</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeanmougin</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">4876</span>– <span class="lpage" tagx="lpage" title="lpage">4882</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9396791">9396791</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudley</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">24</span>: <span class="fpage" tagx="fpage" title="fpage">1596</span>– <span class="lpage" tagx="lpage" title="lpage">1599</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738">17488738</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metzker</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Sequencing technologies-the next generation</span>. <span class="source" tagx="source" title="source">Nature reviews genetics</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">46</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nrg2626">10.1038/nrg2626</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19997069">19997069</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhl</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheung</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zewdie</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCallum</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A unique set of 11,008 onion expressed sequence tags reveals expressed sequence and genomic differences between the monocot orders <i>Asparagales</i> and <i>Poales</i> </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">114</span>– <span class="lpage" tagx="lpage" title="lpage">125</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14671025">14671025</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mei</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Large-scale development of expressed sequence tag-derived simple sequence repeat markers by deep transcriptome sequencing in garlic ( <i>Allium sativum</i> L.) </span>. <span class="source" tagx="source" title="source">Mol Breeding</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">204</span>. </span> </li> <li tag="ref"> <a name="pone.0157871.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>. <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and annotation of the Chinese chive ( <i>Allium tuberosum</i> Rottler ex Spr.) transcriptome using the Illumina platform </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0133312</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0133312">10.1371/journal.pone.0133312</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26204518">26204518</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of the Welsh onion ( <i>Allium fistulosum</i> L.) transcriptome using Illumina technology </span>. <span class="source" tagx="source" title="source">Mol GenetGenomics</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">291</span>: <span class="fpage" tagx="fpage" title="fpage">647</span>– <span class="lpage" tagx="lpage" title="lpage">659</span>. </span> </li> <li tag="ref"> <a name="pone.0157871.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2013</span>;: <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkt1016">10.1093/nar/gkt1016</a> </span> </span> </li> <li tag="ref"> <a name="pone.0157871.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Avci</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grant</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haigler</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beers</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">XND1, a member of the NAC domain family in Arabidopsis thaliana, negatively regulates lignocellulose synthesis and programmed cell death in xylem </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">53</span>, <span class="fpage" tagx="fpage" title="fpage">425</span>– <span class="lpage" tagx="lpage" title="lpage">436</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18069942">18069942</a> </span> </span> </li> </ul> </div> </div>  </body></html>