<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4373911">4373911</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25806518">25806518</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0120385">10.1371/journal.pone.0120385</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-14-43484</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genetic and Molecular Characterization of Submergence Response Identifies <i>Subtol6</i> as a Major Submergence Tolerance Locus in Maize </div> <div class="alt-title" title="alt-title">
Molecular Responses of Maize during Submergence</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">Malachy T.</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Proctor</span> <span class="given-names" tagx="given-names" title="given-names">Christopher A.</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dou</span> <span class="given-names" tagx="given-names" title="given-names">Yongchao</span> </span> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">Aaron J.</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phansak</span> <span class="given-names" tagx="given-names" title="given-names">Piyaporn</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff003"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kruger</span> <span class="given-names" tagx="given-names" title="given-names">Greg R.</span> </span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Chi</span> </span> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walia</span> <span class="given-names" tagx="given-names" title="given-names">Harkamal</span> </span> <a href="#aff001"> <sup>1</sup> </a> <a href="#cor001">*</a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">University of Nebraska-Lincoln, Department of Agronomy and Horticulture, Lincoln, Nebraska, United States of America </span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">University of Nebraska-Lincoln, School of Biological Sciences, Lincoln, Nebraska, United States of America </span> </span> <span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Nakhon Phanom University, Muang District, Thailand</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">Mingliang</span> </span> <span class="role" tagx="role" title="role">Academic Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">China Agricultural University, CHINA</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: MC HW CZ. Performed the experiments: MC CP PP AS. Analyzed the data: MC CP CZ YD. Contributed reagents/materials/analysis tools: HW CZ GK. Wrote the paper: MC CP HW CZ. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">hwalia2@unl.edu</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-3-3</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2015</span> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0120385</span> <span class="history" title="history"> <span class="received" title="received">received: 2014-9-26</span> <span class="accepted" title="accepted">accepted: 2015-1-21</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Campbell et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <a class="self-uri" href="pone.0120385.pdf" title="self-uri">pone.0120385.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Maize is highly sensitive to short term flooding and submergence. Early season flooding reduces germination, survival and growth rate of maize seedlings. We aimed to discover genetic variation for submergence tolerance in maize and elucidate the genetic basis of submergence tolerance through transcriptional profiling and linkage analysis of contrasting genotypes. A diverse set of maize nested association mapping (NAM) founder lines were screened, and two highly tolerant (Mo18W and M162W) and sensitive (B97 and B73) genotypes were identified. Tolerant lines exhibited delayed senescence and lower oxidative stress levels compared to sensitive lines. Transcriptome analysis was performed on these inbreds to provide genome level insights into the molecular responses to submergence. Tolerant lines had higher transcript abundance of several fermentation-related genes and an unannotated <i>Pyrophosphate-Dependent Fructose-6-Phosphate 1-Phosphotransferase</i> gene during submergence. A coexpression network enriched for <i>CBF</i> ( <i>C-REPEAT/DRE BINDING FACTOR</i>: <i>C-REPEAT/DRE BINDING FACTOR</i>) genes, was induced by submergence in all four inbreds, but was more activated in the tolerant Mo18W. A recombinant inbred line (RIL) population derived from Mo18W and B73 was screened for submergence tolerance. A major QTL named <i>Subtol6</i> was mapped to chromosome 6 that explains 22% of the phenotypic variation within the RIL population. We identified two candidate genes ( <i>HEMOGLOBIN2</i> and <i>RAV1</i>) underlying <i>Subtol6</i> based on contrasting expression patterns observed in B73 and Mo18W. Sources of tolerance identified in this study ( <i>Subtol6</i>) can be useful to increase survival rate during flooding events that are predicted to increase in frequency with climate change. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
Support was provided by the Life Sciences Competition award to HW and CZ (2011–2013), University of Nebraska, Lincoln. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="The raw Illumina data are publicly available through the National Center for Biotechnology Information Gene Expression Omnibus (GSE63429)." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The raw Illumina data are publicly available through the National Center for Biotechnology Information Gene Expression Omnibus (GSE63429). </p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The frequency and intensity of extreme precipitation events (drought and flooding) are predicted to increase due to changes in climatic patterns. These changes in precipitation patterns will impact many maize-producing regions of the world, which receive considerable rainfall during early growing season [ <a href="#pone.0120385.ref001">1</a>, <a href="#pone.0120385.ref002">2</a>]. Maize is highly sensitive to waterlogging and submergence (complete inundation) during germination and seedling stages and leads to poor seedling establishment, stunted growth, and delayed development. Most of the previous research has focused on waterlogging, while submergence tolerance remains less characterized in maize [ <a href="#pone.0120385.ref003">3</a>– <a href="#pone.0120385.ref010">10</a>]. </p> <p>At a cellular level, hypoxic conditions during submergence reduce protein translation and induce selectively translated proteins involved in glycolysis and the fermentative pathways [ <a href="#pone.0120385.ref011">11</a>]. The shift to anaerobic respiration allows the plants to continue producing ATP, albeit at a considerably lower rate, while carbon reserves are still available [ <a href="#pone.0120385.ref012">12</a>– <a href="#pone.0120385.ref014">14</a>]. As part of a metabolic adaptation to hypoxia, several ATP consuming biochemical pathways are inhibited [ <a href="#pone.0120385.ref015">15</a>]. Furthermore, survival through prolonged inundation hypoxia also involves the use of inorganic pyrophosphate (PPi) as an alternative energy source and induction of enzymes that reduce reactive oxygen species (ROS) or cytoplasmic acidosis [ <a href="#pone.0120385.ref012">12</a>– <a href="#pone.0120385.ref014">14</a>, <a href="#pone.0120385.ref016">16</a>, <a href="#pone.0120385.ref017">17</a>]. </p> <p>The molecular and physiological mechanisms involved in the submergence responses in plants have largely been characterized in two tolerant species, rice ( <i>Oryza sativa</i>) and <i>Rumex palustris</i> [ <a href="#pone.0120385.ref018">18</a>– <a href="#pone.0120385.ref023">23</a>]. One of the early responses to submergence involves the differential regulation of a suite of transcription factors belonging to the ethylene response factor (ERFs) gene family [ <a href="#pone.0120385.ref020">20</a>, <a href="#pone.0120385.ref021">21</a>, <a href="#pone.0120385.ref024">24</a>– <a href="#pone.0120385.ref028">28</a>]. Rice <i>SUB1A</i>, a member of the ERF gene family, dramatically improves submergence tolerance in controlled and field conditions. <i>SUB1A</i> limits shoot elongation during submergence by repressing gibberellic acid (GA) levels and modulating GA signaling [ <a href="#pone.0120385.ref020">20</a>– <a href="#pone.0120385.ref022">22</a>]. In striking contrast, two ERFs, <i>SNORKEL1</i> and <i>SNORKEL2</i>, confer submergence adaptation in deepwater rice by inducing rapid internode elongation through enhanced GA responses [ <a href="#pone.0120385.ref025">25</a>]. </p> <p>Given the economic significance of submergence tolerance for maize, our current knowledge of the submergence responses in maize is limited when compared to rice and <i>Rumex palustris</i>. The extent of genetic variation for submergence tolerance in maize is largely unexplored. Utilizing the natural variation in maize for submergence tolerance studies has become more tractable due to availability of genetic resources such the Nested Association Mapping (NAM) populations and high-density marker information [ <a href="#pone.0120385.ref029">29</a>]. The aims of this study were to (i) assess the natural variation for submergence tolerance in maize germplasm by screening the diverse maize founder lines, (ii) examine the molecular responses to submergence through transcriptional profiling of a subset of tolerant and sensitive inbreds, and (iii) elucidate the genetic basis of submergence tolerance through QTL analysis. We show that substantial variation for submergence tolerance exists in maize NAM population germplasm, and identify a major QTL associated with submergence tolerance. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="maizeinbredsexhibitsignificantphenotypicdiversityforsubmergencetolerance" title="sec"> <div class="title" tagx="title" title="title">
Maize inbreds exhibit significant phenotypic diversity for submergence tolerance</div> <p>A panel of 23 genetically diverse maize Nested Association Mapping (NAM) parental lines, representing five subpopulations, and B73 were screened for submergence tolerance for a period of 48 h or 96 h. The NAM founder lines were selected to represent maize diversity and include more than 50% tropical lines and 9 temperate lines [ <a href="#pone.0120385.ref029">29</a>]. Submerged plants were scored for visual damage using a scoring system specifically developed for this study (see <a href="#sec013">Material and Methods</a> section; <a href="#pone.0120385.s001">S1 Fig</a> and <a href="#pone.0120385.s002">S2 Fig</a>). A parallel set of non-submerged plants was maintained as controls. The scoring range was set from 0–10 and lower scores correspond to higher tolerance based on visual phenotyping. Based on the submergence tolerance scoring system, we observed considerable variation among maize NAM parents ( <a href="#pone.0120385.g001">Fig. 1</a>). Our visual scoring of leaf 1 (oldest leaf), 2 and 3 (youngest leaf), identified several genotypes (B73, B97, CML52, Il14H, NC350 and NC358) with high sensitivity after 48 h of submergence. In contrast, at least 12 of the NAM parents had minimal or no visual damage from 48 h of submergence followed by 7 d recovery period ( <a href="#pone.0120385.g001">Fig. 1A</a>). Scores based on 96 h of submergence, followed by 7 d recovery period were more reliable in identifying the highly tolerant inbreds. The identified tolerant lines (CML228, KY21, M162W and Mo18W) exhibited no or minimal signs of stress after 48 h and 96 h of submergence in the three replicates ( <a href="#pone.0120385.g001">Fig. 1</a>). The four tolerant genotypes tested had either minor or no chlorosis on the lower leaves, while newer leaves remained turgid and green after 48 h and 96 h of submergence ( <a href="#pone.0120385.s003">S3 Fig</a>). These results indicate that most maize inbreds are highly sensitive to even short duration of submergence. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0120385.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Phenotyping of NAM parents after 48 h and 96 h of complete inundation.</div> <p>Plants were completely submerged for 48 h (a) and 96 h (b) and scored daily for 7 d after removing from submergence. Plants were scored on a 0 to10 scale with a score of 0 showing no visual damage and 10 indicating complete senescence. Graphed are the scores of leaves 1 to 3 (1: oldest leaf, 3: youngest leaf) at 7 d. Genotypes with no senescence or other damage from submergence (e.g. Ky21 at 48 h) do not have a bar graph representation. Genotypes in which the leaf received the same score in each replicate have no error bars. Bars represent standard error, where n = 9 for this experiment. Genotypes were separated into subfamilies based on the classification of McMullen et al. [ <a href="#pone.0120385.ref029">29</a>]. NSS: non-stiff stalk. </p>   </div> </div> <div class="submergencetolerantgenotypesmaintainlowerreactiveoxygenspecieslevels" title="sec"> <div class="title" tagx="title" title="title">
Submergence tolerant genotypes maintain lower reactive oxygen species levels</div> <p>Because stress-associated senescence and loss of turgor in the sensitive genotypes are typical symptoms of disrupted cellular processes, we selected two tolerant (Mo18W and M162W) and two highly sensitive genotypes (B73 and B97) for estimating reactive oxygen species (ROS) levels in submerged plants ( <a href="#pone.0120385.g002">Fig. 2A</a>). Based on Nitro Blue Tetrazolium (NBT) and Diaminobenzidine tetrahydrochloride (DAB) staining assays, which measure superoxide and hydrogen peroxide respectively, we observed that tolerant genotypes generally had lower levels of H <sub>2</sub>O <sub>2</sub> and superoxide in leaves after 96 h of complete submergence compared to sensitive genotypes ( <a href="#pone.0120385.g002">Fig. 2A</a>). B73 and B97 showed more intense DAB and NBT staining after 96 h of submergence suggesting that higher levels of superoxide and hydrogen peroxide were produced in the leaves. The lower ROS levels in submergence tolerant inbreds could be due to lower ROS production and/or more effective ROS scavenging. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0120385.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Detection of ROS and expression of ROS marker genes in inbreds with contrasting levels of submergence tolerance. </div> <p>(a.) Detection of H <sub>2</sub>O <sub>2</sub> and superoxide in selected submergence tolerant and sensitive maize genotypes. A subset of four nested association mapping panel parents were selected based on visual leaf scoring for ROS staining assays after submergence. Hydrogen peroxide was detected using stain 3,3′-diaminobenzidine tetrahydrochloride (DAB), while superoxide was detected using nitroblue tetrazolium (NBT). Both assays were performed after 96 h of submergence. In B97, the third leaf was barely emerged and not used for the assay. (b-d) Real-time quantitative PCR of ROS genes. ROS genes were selected based on the study by Gadjev, Vanderauwera and Gechev [ <a href="#pone.0120385.ref030">30</a>] and showed a significant induction in response to at least two stresses that induce ROS formation in Arabidopsis. (b.) <i>ALTERNATIVE OXIDASE 1a</i> ( <i>AOX1a</i>); (c.) <i>WRKY6</i> (d.) <i>CYTOCHROME P81 D8</i> ( <i>CYP81D8</i>). All sample were collected at 24 h and 72 h after submergence. Expression levels are relative to shoot tissue of control plants at 24 h. Letters above bars indicate nonsignificant differences in expression determined using Tukey’s HSD ( <i>p</i> &amp;lt; 0.05). Statistical tests were performed separately for each time point. Plots without letters (ie 2c 24 h) indicate that all comparisons are significantly different. Error bars display standard error where <i>n</i> = 3. </p>   </div> <p>To validate the results of NBT and DAB staining assays, we examined the expression of known ROS marker genes in the four inbreds after 24 and 72 h of submergence using quantitative real-time PCR (qPCR). The ROS marker genes were from derived from an Arabidopsis dataset for four oxidative stress treatments (methyl viologen, <i>Alternaria alternata</i> toxin, 3-aminotriazole, and ozone) [ <a href="#pone.0120385.ref030">30</a>]. Genes were selected if they showed an induction in at least two of the four stresses and had corresponding maize orthologs. The relative gene expression was measured using real-time quantitative PCR (qPCR) for <i>ALTERNATIVE OXIDASE 1a</i> ( <i>AOX1a</i>; GRMZM2G125669). Both sensitive inbreds displayed a considerably higher induction of <i>AOX1a</i> in response to submergence when compared to M162W at 24h ( <a href="#pone.0120385.g002">Fig. 2B</a>; <i>p</i> &amp;lt; 0.05). While at 72 h after submergence, the expression of <i>AOX1a</i> was significantly higher in both B73 and B97 compared to tolerant inbreds ( <a href="#pone.0120385.g002">Fig. 2B</a>; <i>p</i> &amp;lt; 0.05). Maize <i>WRKY6</i> ortholog (GRMZM5G871347) showed contrasting expression patterns in response to submergence in maize ( <a href="#pone.0120385.g002">Fig. 2C</a>; <i>p</i> &amp;lt; 0.05). The expression of <i>WRKY6</i> was induced in both B73 and B97 by more than two-fold at 24 and 72 h after submergence relative to control samples, while in tolerant inbreds the expression of <i>WRKY6</i> was down-regulated at both time points relative to controls. A third marker gene, <i>CYP81D8</i> (GRMZM2G087875) was expressed at significantly lower levels in both tolerant lines at 24 h after submergence compared to sensitive lines ( <a href="#pone.0120385.g002">Fig. 2D</a>; <i>p</i> &amp;lt; 0.05). While at 72 h after submergence, <i>CYP81D1</i> showed significantly lower expression in Mo18W when compared to both sensitive inbreds. Collectively, the staining assays and quantitative expression analysis suggest that the relatively lower survival and poor recovery of the sensitive inbreds could partly be due to higher ROS levels and downstream transcriptome changes that accompany oxidative stresses. </p> </div> <div class="sensitiveandtolerantmaizeinbredshavedistincttranscriptomedynamicsduringsubmergence" title="sec"> <div class="title" tagx="title" title="title">
Sensitive and tolerant maize inbreds have distinct transcriptome dynamics during submergence</div> <p>We performed RNAseq analysis using shoot tissue sampled at 24 and 72 h after submergence for the sensitive genotypes (B73 and B97) and tolerant genotypes (M162W and Mo18W). Non-submerged shoot samples were collected at 24 h as controls. We sampled at 24 h to capture the early responses to short-term submergence. Because, by 96 h, the tolerant genotypes were clearly distinguishable from the sensitive genotypes based on ROS staining assays, we reasoned that the 72 h time point could provide insights on the inducible tolerance responses before the visual phenotypes appear. The complete set of differentially expressed genes is provided as <a href="#pone.0120385.s018">S1 File</a> (FDR &amp;lt; 0.001). </p> <p>Submergence resulted in a major shift in the transcriptome of all four maize genotypes as indicated by a large number of responsive genes ( <a href="#pone.0120385.t001">Table 1</a>). Two numeric features emerged from submergence responsive differential expression analysis. The tolerant M162W had almost half the number of differentially expressed genes at 24 h relative to the other genotypes. The three replicates for M162W had a high Spearman correlation ( <i>R</i> <sup> <i>2</i> </sup> = 0.95–0.98) suggesting that a smaller set of differentially expressed genes was not due to variation among the three biological replicates. The number of differentially expressed genes in M162W at 72 h was comparable to other genotypes. These results suggested that short-term submergence had a delayed effect on M162W transcriptome relative to other genotypes. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0120385.t001">10.1371/journal.pone.0120385.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary of differential expression analysis.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th>B73</th> <th>B97</th> <th>Mo18W</th> <th>M162W</th> <th>Commonly Expressed in Sensitive Lines</th> <th>Commonly Expressed in Tolerant Lines</th> </tr> <tr> <th> </th> <th>Up</th> <th>Down</th> <th>Up</th> <th>Down</th> <th>Up</th> <th>Down</th> <th>Up</th> <th>Down</th> <th>Up</th> <th>Down</th> <th>Up</th> <th>Down</th> </tr> </thead> <tbody> <tr> <td> <b>24 h</b> </td> <td>3603</td> <td>3018</td> <td>2390</td> <td>2168</td> <td>2292</td> <td>2123</td> <td>1344</td> <td>575</td> <td>1969</td> <td>1304</td> <td>1041</td> <td>425</td> </tr> <tr> <td> <b>72 h</b> </td> <td>3389</td> <td>2262</td> <td>2439</td> <td>1816</td> <td>3222</td> <td>3068</td> <td>3490</td> <td>2801</td> <td>1820</td> <td>1096</td> <td>1899</td> <td>1745</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Number of differentially expressed transcripts from comparisons between submerged and control plants. All transcripts showed significant differences in expression (FDR &amp;lt;0.001). </p> </div> </div> </div> <p>Despite the phenotypic similarities observed in the tolerant lines, very few genes displayed similar expression patterns after 24 h of submergence ( <a href="#pone.0120385.s004">S4 Fig</a>). In contrast, the transcriptional response of the sensitive lines was mostly shared. Approximately 50% of the genes that were differentially expressed in response to submergence in B73 at 24 h displayed similar expression patterns in B97 at 24 h after submergence ( <a href="#pone.0120385.t001">Table 1</a>). The overlapping expression patterns observed in sensitive lines suggest that they may share a similar response to submergence. </p> </div> <div class="tolerantlinesexhibithigherexpressionofhypoxia-associatedenergyhomeostasisgenes" title="sec"> <div class="title" tagx="title" title="title">
Tolerant lines exhibit higher expression of hypoxia-associated energy homeostasis genes </div> <p>We explored the possibility that unannotated genes may be contributing to submergence response in maize ( <a href="#pone.0120385.s019">S2 File</a>). A notable finding from this analysis was a transcript (TCONS_00005048) with similarity to the alpha subunit of <i>PYROPHOSPHATE-DEPENDENT FRUCTOSE-6-PHOSPHATE 1-PHOSPHOTRANSFERASE (PFP)</i> protein in maize. Although the expression of <i>PFP</i> was strongly reduced by submergence in all lines, the transcript abundance of <i>PFP</i> was higher in the tolerant lines during submergence due to a relatively gradual decline compared to the rapid drop in transcript levels observed in the sensitive inbreds. The expression of this <i>PFP</i> transcript was confirmed using qPCR, indicating that it is a bona fide transcript ( <a href="#pone.0120385.g003">Fig. 3A</a>). Several other genes participating in ethanolic fermentation downstream of <i>PFP</i> also had higher transcript abundance in the tolerant lines after 24 and 72 h of submergence relative to controls. For instance, a <i>PYRUVATE DECARBOXYLASE3</i> gene ( <i>PDC3</i>; GRMZM2G087186) showed more than a 3 and 4-fold induction in M162W and Mo18W, respectively after 24 h of submergence when compared to control plants ( <a href="#pone.0120385.g003">Fig. 3B</a>). The transcript abundance of <i>PDC3</i> continued to increase significantly in tolerant lines during submergence, showing a 4.8 and 6-fold induction in Mo18W and M162W, respectively after 72 h of submergence. A similar trend in <i>PDC3</i> expression was also observed in B73 during submergence; however, the expression in response to submergence was considerably lower relative to the tolerant lines. A gene encoding <i>ALCOHOL DEHYDROGENASE1</i> ( <i>ADH1</i>; GRMZM2G442658) showed more than a seven-fold induction in Mo18W and M162W after 24 h of submergence ( <a href="#pone.0120385.g003">Fig. 3C</a>). The expression increased further after 72 h of submergence, with more than a 9.5 and eight-fold up-regulation observed in Mo18W and M162W, respectively. Although <i>ADH1</i> was induced by submergence in the sensitive lines, the induction was considerably lower relative to the tolerant lines. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0120385.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Real-time quantitative PCR of <i>PYROPHOSPHATE-DEPENDANT FRUCTOSE-6-PHOSPHATE 1-PHOSPHOTRANSERASE</i> ( <i>PFP</i>) and anaerobic glycolysis genes during submergence. </div> <p>(a.) The expression of an unannotated <i>PYROPHOSPHATE-DEPENDANT FRUCTOSE-6-PHOSPHATE 1-PHOSPHOTRANSERASE</i> ( <i>PFP</i>) gene in submerged shoot tissue during submergence. (b.) <i>PYRUVATE DECARBOXYLASE3</i> ( <i>PDC3</i>) expression in shoot tissues during submergence. (c.) <i>ALCOHOL DEHYDROGENASE1</i> ( <i>ADH1</i>) transcript levels in submerged shoot tissue. All sample were collected at 24 h and 72 h after submergence. Expression levels are relative to shoot tissue of control plants at 24 h. Letters above bars indicate nonsignificant differences in expression determined using Tukey’s HSD ( <i>p</i> &amp;lt; 0.05). Statistical tests were performed separately for each timepoint. Plots without letters (ie 3a 72 h and 3c 72 h) indicate that all comparisons are significantly different. Error bars display standard error where <i>n</i> = 3. </p>   </div> </div> <div class="oxidativestressassociatedgenefamiliesaredifferentiallyregulatedduringsubmergenceinmaize" title="sec"> <div class="title" tagx="title" title="title">
Oxidative stress associated gene families are differentially regulated during submergence in maize </div> <p>To identify pathways that may be affected during submergence, gene ontology (GO) enrichment analysis was conducted on differentially expressed gene lists ( <a href="#pone.0120385.s021">S4 File</a>). Transcripts associated with protein phosphorylation and ubiquination, were enriched among up-regulated genes in all in accessions at 24 and 72 h after submergence ( <a href="#pone.0120385.s021">S4 File</a>). Several notable features were observed in genes that showed similar expression patterns in both tolerant and sensitive lines. Transcripts associated with trehalose metabolism were enriched among up-regulated transcripts in Mo18W at 24 and 72 h after submergence, suggesting that this may be a unique component of the transcriptional response to submergence in this inbred ( <a href="#pone.0120385.s021">S4 File</a>; GO:0005991, GO:0005992). Genes involved in apoptosis and cell death were overrepresented ( <i>p</i> &amp;lt; 0.05) in up-regulated genes in sensitive lines, suggesting that processes associated with senescence were triggered as part of an early response to submergence in these accessions ( <a href="#pone.0120385.s005">S5C Fig</a>; GO:0006915, GO:0012501, GO:0008219, GO:0016265). Several notable gene families with role in senescence were down-regulated in the tolerant lines when compared to one or more sensitive line during submergence. Redox homeostasis related transcripts, such as <i>Gluathione S-Transferase</i> ( <i>GST</i>) and <i>Alternative Oxidase</i> genes ( <i>AOX</i>) were expressed at significantly higher levels in sensitive lines compared to tolerant varieties ( <a href="#pone.0120385.s006">S6A Fig</a>). Both GST and <i>AOX</i> are critical for redox homeostasis. <i>AOX</i> competes with cytochrome bc1 complex in mitochondria, reducing the production of superoxide radicals [ <a href="#pone.0120385.ref031">31</a>, <a href="#pone.0120385.ref032">32</a>]. </p> <p>We performed transcription factor enrichment analysis and found significant enrichment for several transcription factor families such as <i>ERF</i>, <i>NAC</i> ( <i>NAM</i>, <i>ATAF</i>, and <i>CUC</i>) and <i>WRKY</i> across genotypes in response to submergence ( <i>p</i>&amp;lt;0.05; <a href="#pone.0120385.t002">Table 2</a>; <a href="#pone.0120385.s012">S1 Table</a>). Although these transcription factor families are involved in many diverse processes, several orthologous genes known to be involved with oxidative stress responses and ethylene signaling were expressed at significantly higher levels in sensitive inbreds relative to tolerant lines. Besides the genes that respond to submergence, genotypic comparisons showed contrasting expression patterns between the tolerant and sensitive lines for these transcription factor families ( <a href="#pone.0120385.t003">Table 3</a>; <a href="#pone.0120385.s013">S2 Table</a>). We observed that the <i>NAC</i> family transcription factors were significantly enriched among down-regulated genes in all comparisons between tolerant and sensitive lines, except comparisons between Mo18W and B97 (p &amp;lt;0.05; <a href="#pone.0120385.t003">Table 3</a>). A gene similar to <i>AtNAC029</i>, which is induced during leaf senescence in Arabidopsis, displayed considerably lower expression in all tolerant lines when compared to sensitive inbreds at both 24 and 72 h after submergence ([ <a href="#pone.0120385.ref033">33</a>]; <a href="#pone.0120385.s006">S6B Fig</a>). Furthermore, SUPPRESSOR <i>OF GAMMA RADIATION 1</i> ( <i>SOG1</i>) (GRMZM2G078954) in Arabidopsis showed considerably lower levels of expression in Mo18W when compared to both sensitive lines at 24 and 72h after submergence. In Arabidopsis, <i>SOG1</i> is a central player in DNA damage response, inducing hundreds of genes involved in DNA repair [ <a href="#pone.0120385.ref034">34</a>, <a href="#pone.0120385.ref035">35</a>]. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0120385.t002">10.1371/journal.pone.0120385.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Transcription factor enrichment analysis from comparisons between treatments.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th /> <th>24 h</th> <th>72 h</th> </tr> <tr> <th /> <th /> <th> <i>AP2-ERF</i> </th> <th> <i>NAC</i> </th> <th> <i>WRKY</i> </th> <th> <i>AP2-ERF</i> </th> <th> <i>NAC</i> </th> <th> <i>WRKY</i> </th> </tr> </thead> <tbody> <tr> <td> <b>B73</b> </td> <td> <b>Down</b> </td> <td>4.0%</td> <td>1.9%</td> <td>3.8%</td> <td>3.3%</td> <td>1.3%</td> <td>3.8%</td> </tr> <tr> <td> <b>Up</b> </td> <td>13.9% <a href="#t002fn001">***</a> </td> <td>15.2% <a href="#t002fn001">***</a> </td> <td>12.5% <a href="#t002fn001">**</a> </td> <td>13.1% <a href="#t002fn001">***</a> </td> <td>17.1% <a href="#t002fn001">***</a> </td> <td>13.1% <a href="#t002fn001">**</a> </td> </tr> <tr> <td> <b>B97</b> </td> <td> <b>Down</b> </td> <td>2.2%</td> <td>1.3%</td> <td>1.3%</td> <td>2.2%</td> <td>0.6%</td> <td>1.3%</td> </tr> <tr> <td> <b>Up</b> </td> <td>10.6% <a href="#t002fn001">***</a> </td> <td>10.1% <a href="#t002fn001">***</a> </td> <td>11.3% <a href="#t002fn001">***</a> </td> <td>9.9% <a href="#t002fn001">***</a> </td> <td>12.7% <a href="#t002fn001">***</a> </td> <td>9.4% <a href="#t002fn001">**</a> </td> </tr> <tr> <td> <b>Mo18W</b> </td> <td> <b>Down</b> </td> <td>2.2%</td> <td>1.3%</td> <td>2.5%</td> <td>4.7%</td> <td>5.1%</td> <td>4.4%</td> </tr> <tr> <td> <b>Up</b> </td> <td>10.6% <a href="#t002fn001">***</a> </td> <td>10.8% <a href="#t002fn001">***</a> </td> <td>8.8% <a href="#t002fn001">*</a> </td> <td>11.3% <a href="#t002fn001">***</a> </td> <td>13.9% <a href="#t002fn001">***</a> </td> <td>7.5%</td> </tr> <tr> <td> <b>M162W</b> </td> <td> <b>Down</b> </td> <td>0.7%</td> <td>0.0%</td> <td>3.1% <a href="#t002fn001">*</a> </td> <td>4.4%</td> <td>1.9%</td> <td>3.1%</td> </tr> <tr> <td> <b>Up</b> </td> <td>10.9% <a href="#t002fn001">***</a> </td> <td>5.1% <a href="#t002fn001">***</a> </td> <td>3.8%</td> <td>14.6% <a href="#t002fn001">***</a> </td> <td>16.5% <a href="#t002fn001">***</a> </td> <td>14.4% <a href="#t002fn001">***</a> </td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Subset of enriched transcription factors that were significantly enriched among differentially expressed genes. Differentially expressed genes were identified from comparisons between submerged and control plants at 24 and 72 h (FDR &amp;lt; 0.001). Asterisks denote statistical significance as determined using hypergeometric test: * p≤0.05, ** p≤0.01 and *** p≤0.001. <i>NAC</i>: <i>NAM</i>, <i>ATAF</i>, <i>CUC1</i>. </p> </div> </div> </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0120385.t003">10.1371/journal.pone.0120385.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Transcription factor enrichment analysis from comparisons between genotypes.</div> <p>Subset of enriched transcription factor identified from comparisons between genotypes at 24 and 72 h after submergence. Differentially expressed genes were identified from comparisons between submerged and control plants at 24 and 72 h (FDR &amp;lt; 0.001). Asterisks denote statistical significance as determined using hypergeometric test: * <i>p</i>≤0.05, ** <i>p</i>≤0.01 and *** <i>p</i>≤0.001. <i>NAC</i>: <i>NAM</i>, <i>ATAF</i>, <i>CUC1</i>. </p>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th /> <th /> <th>24 h</th> <th>72 h</th> </tr> <tr> <th /> <th /> <th> <i>AP2-ERF</i> </th> <th> <i>NAC</i> </th> <th> <i>WRKY</i> </th> <th> <i>AP2-ERF</i> </th> <th> <i>NAC</i> </th> <th> <i>WRKY</i> </th> </tr> </thead> <tbody> <tr> <td> <b>Mo18W vs B73</b> </td> <td> <b>Down</b> </td> <td>6.2%</td> <td>8.2% *</td> <td>5.6%</td> <td>7.7%</td> <td>11.4% **</td> <td>13.1% ***</td> </tr> <tr> <td> <b>Up</b> </td> <td>3.2% *</td> <td>0.6%</td> <td>1.3%</td> <td>1.5%</td> <td>0.6%</td> <td>0.6%</td> </tr> <tr> <td> <b>Mo18W vs B97</b> </td> <td> <b>Down</b> </td> <td>3.3%</td> <td>5.1%</td> <td>5.0%</td> <td>3.6%</td> <td>10.1% ***</td> <td>12.5% ***</td> </tr> <tr> <td> <b>Up</b> </td> <td>2.9%</td> <td>1.3%</td> <td>2.5%</td> <td>1.8%</td> <td>3.2%</td> <td>1.3%</td> </tr> <tr> <td> <b>M162W vs B73</b> </td> <td> <b>Down</b> </td> <td>5.1%</td> <td>8.9% **</td> <td>8.8% **</td> <td>2.9%</td> <td>5.1% *</td> <td>1.9%</td> </tr> <tr> <td> <b>Up</b> </td> <td>1.5%</td> <td>0.0%</td> <td>0.6%</td> <td>3.3%</td> <td>3.2%</td> <td>2.5%</td> </tr> <tr> <td> <b>M162W vs B97</b> </td> <td> <b>Down</b> </td> <td>3.6%</td> <td>7.0% **</td> <td>6.9% **</td> <td>6.2% *</td> <td>9.5% ***</td> <td>3.8%</td> </tr> <tr> <td> <b>Up</b> </td> <td>4.4%</td> <td>1.3%</td> <td>2.5%</td> <td>2.2%</td> <td>3.8%</td> <td>1.9%</td> </tr> </tbody> </table> </div> </div> <p> <i>WRKYs</i> were enriched among the down-regulated genes in M162W when compared to both sensitive inbreds at 24 h after submergence. However, at 72 h a significant enrichment was observed among the down-regulated genes of Mo18W when compared to the sensitive lines ( <a href="#pone.0120385.t003">Table 3</a>). One notable WRKY gene, which is similar to <i>WRKY53</i> (GRMZM2G063880) in Arabidopsis, showed considerable higher expression in both sensitive inbreds when compared to M162W and Mo18W at both 24h and 72h after submergence respectively ( <a href="#pone.0120385.s006">S6B Fig</a>). In Arabidopsis, <i>WRKY53</i> has been shown to be a positive regulator of leaf senescence [ <a href="#pone.0120385.ref036">36</a>– <a href="#pone.0120385.ref038">38</a>]. Two putative WRKY genes that are similar to <i>AtWRKY33</i> (GRMZM2G169966, GRMZM2G148087) showed lower expression in M162W at 24 and Mo18W at 72h when compared to both sensitive lines ( <a href="#pone.0120385.s006">S6B Fig</a>). <i>WRKY33</i> is involved in ethylene-mediated thermotolerance and was proposed to regulate ethylene signaling via a positive feedback mechanism [ <a href="#pone.0120385.ref039">39</a>]. More recently it was shown that <i>WRKY33</i> regulates the expression of the ethylene biosynthesis gene, <i>ACS6</i> [ <a href="#pone.0120385.ref040">40</a>]. Collectively, our analysis of <i>WRKY</i> and <i>NAC</i> factors suggested that expression patterns of several of these genes, with orthologs known to be associated with senescence correlated with submergence sensitivity response. Surprisingly, some of these transcription factors and senescence-associated transcriptional signatures were triggered within 24 h of submergence in the sensitive maize genotypes. </p> </div> <div class="erfsareanintegralcomponentoftheearlysubmergenceresponseinmaize" title="sec"> <div class="title" tagx="title" title="title">
ERFs are an integral component of the early submergence response in maize</div> <p>We found that ERFs were significantly enriched among up-regulated genes in response to submergence ( <i>p</i> &amp;lt; 0.001). Eighteen ERFs were up-regulated in all four lines at 24 h after submergence, while at 72 h, 19 ERF genes were up-regulated in response to submergence ( <a href="#pone.0120385.t002">Table 2</a>; <a href="#pone.0120385.s012">S1 Table</a>). We also found a notable ERF that was induced only in the tolerant lines at 24 h. The expression of maize ortholog of <i>REDOX-RESPONSIVE TRANSCRIPTION FACTOR</i> ( <i>RRTF1</i>; GRMZM2G138396; <a href="#pone.0120385.s007">S7A Fig</a>) was induced by approximately 7- and 4.6-fold in Mo18W and M162W respectively. <i>RRTF1</i> is a key member of a regulatory network required for redox homeostasis in Arabidopsis and could play a role in ameliorating the redox imbalance associated with submergence in the tolerant inbreds [ <a href="#pone.0120385.ref041">41</a>]. </p> </div> <div class="inductionofb-2erfsbysubmergenceisconservedacrossspecies" title="sec"> <div class="title" tagx="title" title="title">
Induction of B-2 ERFs by submergence is conserved across species</div> <p>Given the significant differential response of maize ERFs during submergence, we extended our expression analysis across multiple species for the maize submergence responsive ERFs. We mined the published submergence/hypoxia transcriptome datasets from Arabidopsis and rice for orthologous <i>ERFs</i> that were differentially regulated during maize submergence [ <a href="#pone.0120385.ref022">22</a>, <a href="#pone.0120385.ref042">42</a>– <a href="#pone.0120385.ref043">43</a>] (GSE41103; GSE18930;, GSE24077). Results from the cross-species comparisons are summarized in <a href="#pone.0120385.s014">S3 Table</a>. One notable feature to emerge from the <i>ERF</i> family analysis was the expression of seven <i>ERF</i> gene family members corresponding to the <i>ERF</i> clade B-2 (Arabidopsis group VII <i>ERFs</i>; <a href="#pone.0120385.s007">S7A Fig</a>, <a href="#pone.0120385.s008">S8 Fig</a>; <a href="#pone.0120385.s022">S5 File</a>). These transcription factors were characterized by strong up-regulation at both time points in the four maize genotypes ( <a href="#pone.0120385.t002">Table 2</a>; <a href="#pone.0120385.s012">S1 Table</a>). Arabidopsis and rice orthologs of this subgroup were also strongly induced in response to submergence, indicating a conserved response across these three species. The orthologs in this clade play important roles in hypoxia and submergence tolerance, and include the <i>HYPOXIA-RESPONSE ERFs</i> ( <i>HRE1</i> and <i>HRE2</i>) in Arabidopsis, and the rice <i>SUB1A</i> gene [ <a href="#pone.0120385.ref026">26</a>, <a href="#pone.0120385.ref027">27</a>]. Collectively, our maize data and reports from Arabidopsis and rice suggest that the <i>ERF</i> gene family and specifically, the subgroup B-2 exhibited a conserved response to submergence across multiple species. While the transcriptional patterns of B-2 ERFs are conserved across species during submergence, members of the B-2 ERF subfamily have also been reported be regulated at the post-translational level [ <a href="#pone.0120385.ref027">27</a>, <a href="#pone.0120385.ref028">28</a>]. </p> </div> <div class="cbftranscriptionalnetworkissubmergenceinducibleinmaize" title="sec"> <div class="title" tagx="title" title="title">
CBF transcriptional network is submergence inducible in maize</div> <p>We identified four submergence-responsive <i>ERF</i> genes that are orthologous to the Arabidopsis <i>C-REPEAT/DRE BINDING FACTOR2</i> ( <i>CBF2</i>; At4g25470) and another <i>ERF</i> orthologous to <i>C-REPEAT/DRE BINDING FACTOR3</i> ( <i>CBF3</i>; At4g25480). These <i>CBF</i> genes were up-regulated at almost all time points in the four genotypes indicating that the maize <i>CBF</i> pathway was triggered by submergence. Strikingly, the induction of <i>CBF2</i> genes (GRMZM2G069146 and GRMZM2G069126) was higher in the tolerant Mo18W compared to other genotypes after 24 and 72 h of submergence. Arabidopsis <i>CBF2</i> gene is known to regulate a suite of downstream genes. To identify genes that are co-expressed with the two maize <i>CBF2</i> orthologs, we utilized the Maize Coexpression Browser (COB v1.02; <a href="http://csbio.cs.umn.edu/COB/">http://csbio.cs.umn.edu/COB/</a>), which generates co-expression networks from a maize developmental atlas dataset. Using the <i>CBF2</i> orthologs as seeds, two clusters were generated consisting of 23 and 17 genes from GRMZM2G069146 and GRMZM2G069126, respectively ( <a href="#pone.0120385.g004">Fig. 4A</a>; <a href="#pone.0120385.s009">S9 Fig</a>). Approximately 61% of the genes (14 of 23) populating the GRMZM2G069146 network showed significant expression differences between control and submerged plants after 24 h ( <a href="#pone.0120385.g004">Fig. 4B</a>). While in the GRMZM2G069126 cluster only 4 of 17 genes showed significant differences between control and submerged samples for at least one genotype. Notably, half of these genes in the larger GRMZM2G069146 cluster corresponded to <i>ERFs</i>, four of which encoded putative <i>CBF</i> genes. The presence of multiple <i>CBF</i> genes in this cluster is not surprising, considering that many <i>CBF</i> genes are co-expressed and regulate different aspects of the cold response in Arabidopsis [ <a href="#pone.0120385.ref044">44</a>, <a href="#pone.0120385.ref045">45</a>]. However, higher up-regulation of these genes in Mo18W suggested that <i>CBFs</i> could be an important component of the submergence tolerance response in this particular inbred. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0120385.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title"> <i>CBF</i> ( <i>C-REPEAT/DRE BINDING FACTOR</i>) co-expression cluster is induced by submergence in maize. </div> <p>(a.) <i>CBF</i> co-expression network generated form genes differentially expressed at 24 h in comparisons between submerged and control plants. Each gene in the network is represented as a circle. (b.) Genes highlighted in the heatmap are represented by grey circles, while that with a red outline indicates the seed gene for the network. The heatmap highlights the expression of a subset of genes in B73, B97, Mo18W and M162W after 24 h of submergence. (c.) Expression of highlighted genes at 72 h after submergence. All genes showed significant differences in expression between submerged and control plants (FDR &amp;lt;0.001). The scale indicates log <sub>2</sub> fold-change. Up-regulation indicates higher expression in submerged samples. <i>CBF2</i>: <i>C-REPEAT/DRE BINDING FACTOR2</i>; <i>CBF3</i>: <i>C-REPEAT/DRE BINDING FACTOR3</i>; <i>ERF7</i>: <i>ETHYLENE RESPONSE FACTOR7</i>; <i>RRTF1</i>: <i>REDOX REPONSIVE TRANSCRIPTION FACTOR1</i> </p>   </div> </div> <div class="subtol6,amajorqtlonchromosome6isassociatedwithsubmergencetolerance" title="sec"> <div class="title" tagx="title" title="title">
Subtol6, a major QTL on chromosome 6 is associated with submergence tolerance</div> <p>We selected the RIL family derived from the cross between the tolerant Mo18W and sensitive B73 lines in the NAM population to study the genetic basis of submergence tolerance in Mo18W. To this end we screened 166 RILs for submergence tolerance using the protocol described earlier for screening the founder lines ( <a href="#pone.0120385.g005">Fig. 5A</a>) [ <a href="#pone.0120385.ref029">29</a>]. QTL analysis was performed with a publically available set of 1478 markers [ <a href="#pone.0120385.ref046">46</a>]. A major QTL, that we named <i>Submergence Tolerance 6</i> ( <i>Subtol6)</i> was detected on chromosome 6. The <i>Subtol6</i> region spans ∼ 20 Mb (141–162 Mb) with the most significant marker located at ∼162 Mb ( <a href="#pone.0120385.g005">Fig. 5B</a>). This QTL explains 22% of the total phenotypic variation for mean leaf score. To determine whether <i>Subtol6</i> is associated with submergence tolerance or sensitivity, we examined the mean leaf score for RILs harboring the Mo18W and B73 alleles at this location. RILs harboring the Mo18W allele for <i>Subtol6</i> were more tolerant than RILs with the B73 allele, suggesting that this region from Mo18W contains alleles conferring submergence tolerance in Mo18W relative to B73 ( <a href="#pone.0120385.g005">Fig. 5C</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0120385.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title"> <i>Subtol6</i> is associated with submergence tolerance in maize. </div> <p>(a.) Distribution of mean leaf scores for Mo18W x B73 RIL mapping population and parental inbreds. Leaf score was averaged across leaves 1 to 3. Scores for parental inbreds are indicated with red arrows. (b.) Interval mapping results for QTL identified in the Mo18W x B73 RIL family from the NAM population. LOD threshold was determined using 10,000 permutations. (c.) Observed genotypes of chromosome 6 for a subset of tolerant and sensitive RILs. Markers with the B73 allele are represented by open ovals, while Mo18W alleles are represented by closed ovals. (d.) Expression of a subset of genes that fall within the Bayes credible interval for the most significant marker. Expression data was derived from RNA sequencing of B73 and Mo18W after 24 h and 72 h after submergence. All genes between ∼141.4 to 162.8 Mb on chromosome 6 and showing significant differences in expression in comparisons between B73 and Mo18W were considered (FDR &amp;lt;0.001). The scale indicates log <sub>2</sub> fold-change. Up-regulation indicates higher expression Mo18W relative to B73. <i>TH9</i>: <i>THIOREDOXIN9</i> (GRMZM2G701204); <i>WRKY33</i> (GRMZM2G169966); <i>HB2</i>: <i>HEMOGLOBIN2</i> (GRMZM2G168898); <i>GSTU8</i>: <i>GLUTATHIONE S-TRANSFERASE8</i> (GRMZM2G097989); <i>RAV1</i>: <i>RELATED TO ABI3/VP1 1</i> (GRMZM2G059939); <i>ERF4</i>: <i>ETHYLENE RESPONSE FACTOR4</i> (GRMZM2G020150) </p>   </div> <p>To identify candidate genes for submergence tolerance in the <i>Subtol6</i> region, we mined the submergence transcriptome data for B73 and Mo18W from our RNAseq analysis. All genes falling within the 95% Bayes credible interval around the most significant marker (141–162 Mb on chromosome 6) were considered for expression analysis. Sixty-seven genes within this region showed significant differences (FDR &amp;lt; 0.001) in transcript abundance between Mo18W and B73 under control, or after 24 and 72 h of submergence ( <a href="#pone.0120385.s020">S3 File</a>). Six of these differentially expressed genes underlying the <i>Subtol6</i> region have been previously reported to be involved in abiotic stress responses, hypoxia and senescence/oxidative stress ( <a href="#pone.0120385.g005">Fig. 5D</a>). Two notable candidate genes are <i>RELATED TO ABA-INSENSITIVE3(ABI3)/VIVIPAROUS1</i> ( <i>RAV1;</i> GRMZM2G059939) and <i>HEMOGLOBIN2</i> ( <i>HB2;</i> GRMZM2G168898). <i>RAV1</i>, an <i>ERF</i> located at approximately 162.6 Mb, was expressed at significantly lower levels in Mo18W when compared to B73 in control and submerged conditions ( <a href="#pone.0120385.s010">S10A Fig</a>). The position of <i>RAV1</i> is in close proximity to the most significant marker for <i>Subtol6</i> at ∼162 Mb. Arabidopsis <i>RAV1</i> is known to positively regulate leaf senescence [ <a href="#pone.0120385.ref047">47</a>]. The second candidate gene, <i>HEMOGLOBIN2</i> ( <i>HB2</i>), which encodes for a non-symbiotic hemoglobin protein was identified at approximately 159.1 Mb on chromosome 6. <i>HB2</i> was expressed at higher level in Mo18W relative to B73 in both control and submerged conditions ( <a href="#pone.0120385.s010">S10B Fig</a>). Hemoglobin proteins in maize have been associated with repressing ROS levels and maintaining energy status of maize cells during hypoxic conditions [ <a href="#pone.0120385.ref048">48</a>– <a href="#pone.0120385.ref051">51</a>]. </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We have characterized the genetic variation for submergence tolerance in the maize NAM parent diversity panel and identified two inbreds with significantly higher submergence tolerance during the early seedling stage. These results have provided a means to map and characterize a major submergence tolerance QTL ( <i>Subtol6</i>) using RILs derived from Mo18W and B73. Interestingly, the RILs harboring the B73 allele at <i>Subtol6</i>, were considerably more sensitive to submergence than those with the Mo18W allele. Since B73 is widely used in many U.S. breeding programs, this suggests that there is a need for improving submergence tolerance in lines that have retained alleles associated with B73 submergence sensitivity. </p> <p>The submergence tolerant lines identified from our experiments displayed delayed senescence in response to complete submergence. Lower ROS levels were observed in the tolerant lines during submergence. An increase in ROS has been reported in plants exposed to submergence and hypoxic conditions [ <a href="#pone.0120385.ref052">52</a>, <a href="#pone.0120385.ref053">53</a>]. One of the likely candidate genes that we propose for <i>Subtol6</i> is <i>HB2</i>. This is based on the expression differences between the parent inbreds (Mo18W and B73) and previously reported role of <i>HB2</i> in plant survival in low oxygen or low ATP conditions [ <a href="#pone.0120385.ref048">48</a>– <a href="#pone.0120385.ref051">51</a>]. A recent report has linked lower <i>HB2</i> levels with the triggering of programmed cell death, mediated by higher ROS levels in maize embryogenic tissue [ <a href="#pone.0120385.ref051">51</a>]. Our observations that the tolerant Mo18W has higher <i>HB2</i> transcript abundance, lower ROS levels and minimal senescence after submergence stress present <i>HB2</i> as an attractive candidate for <i>Subtol6</i>. </p> <p>The second candidate gene underlying the <i>Subtol6</i> that we propose is the maize ortholog of <i>AtRAV1</i>, a member of the ERF gene family. Maize <i>RAV1</i> transcript abundance was lower in Mo18W relative to B73 under control and submerged conditions. Although, <i>RAV1</i> expression is induced in both Mo18W and B73, the up-regulation in B73 is higher than Mo18W in response to submergence. The differential expression pattern of <i>RAV1</i> between Mo18W and B73 is noteworthy because overexpression of <i>RAV1</i> in Arabidopsis is known to lead to early senescence. In wildtype Arabidopsis plants, <i>RAV1</i> expression is induced during onset of senescence [ <a href="#pone.0120385.ref047">47</a>]. This is consistent with the increased senescence observed in B73 relative to Mo18W after submergence stress. Although our selection of candidates for <i>Subtol6</i> is based on gene expression level differences, it is possible that the genetic basis of the <i>Subtol6</i> gene(s) could be due to sequence polymorphism in the underlying gene that has comparable transcript abundance during submergence and in control conditions. It is also possible that Mo18W may have additional gene(s) not present in the reference genome based on B73 as was the case with the rice <i>Sub1</i> locus. Future characterization of the <i>Subtol6</i> locus will address these possibilities and elucidate the casual gene underlying <i>Subtol6</i>. </p> <p>Several <i>CBF/DREB</i> genes were induced in response to submergence in maize. This response was only observed in maize and is missing in rice and Arabidopsis. <i>CBF</i> genes act together in a transcriptional network and are expressed in response to a variety of conditions such as low temperature, dehydration, mechanical agitation, and senescence [ <a href="#pone.0120385.ref054">54</a>]. Notably, the induction of the CBF transcriptional network was stronger in the tolerant Mo18W, indicating that a <i>CBF</i>-mediated response may be more active in this line. In our study, submergence-induced senescence was delayed considerably in Mo18W compared to B73 and B97. Studies by Sharabi-Schwager et al. have shown that the ectopic expression of <i>CBF</i> genes negatively regulates senescence in Arabidopsis [ <a href="#pone.0120385.ref055">55</a>, <a href="#pone.0120385.ref056">56</a>]. Although further studies are needed to determine the underlying cause of delayed senescence in Mo18W, the higher expression of the <i>CBF</i> transcriptional network highlights an interesting feature of the submergence response of this inbred and merits further exploration. </p> <p>Although the tolerant lines have very different genetic backgrounds, at the molecular level, several interesting features were shared. Both tolerant inbreds had higher transcript abundance of ethanol fermentation related genes, specifically <i>PDC3</i> and <i>ADH1</i> compared to sensitive inbreds. In hypoxic conditions, PDC catalyzes the conversion of pyruvate to acetaldehyde, which is then converted to ethanol by ADH providing NAD <sup>+</sup> for glycolysis [ <a href="#pone.0120385.ref016">16</a>]. An unannotated transcript showing homology to the alpha subunit of <i>PYROPHOSPHATE-DEPENDENT FRUCTOSE-6-PHOSPHATE 1-PHOSPHOTRANSFERASE (PFP)</i> protein in maize was relatively less repressed in both tolerant lines compared to sensitive lines during submergence. During anoxic conditions the levels of ATP declines dramatically, and pathways that consume ATP are replaced by those that utilize PPi [ <a href="#pone.0120385.ref013">13</a>, <a href="#pone.0120385.ref014">14</a>, <a href="#pone.0120385.ref057">57</a>, <a href="#pone.0120385.ref058">58</a>]. In rice seedlings, the activity of PFP increased during prolonged oxygen deprivation, and the activity of <i>PFP</i> was dependent on protein levels rather than protein kinetics [ <a href="#pone.0120385.ref012">12</a>]. Furthermore, higher glycolytic rates, and higher protein levels and activity of <i>PFP</i> were reported in anoxia tolerant rice [ <a href="#pone.0120385.ref018">18</a>]. Although the exact function of <i>PFP</i> in plants is not entirely clear, it is thought that fructose-6-phosphate could be phosphorylated by <i>PFP</i> rather than phosphofructose kinase during anoxia. This mechanism could increase the ATP yield of glycolysis by as much as 50% during anaerobiosis [ <a href="#pone.0120385.ref012">12</a>]. To date no studies have explored the importance of <i>PFP</i> in hypoxia/submergence tolerance in maize, however the expression pattern of <i>PFP</i> in tolerant genotypes in our study presents an intriguing possibility for a role in submergence tolerance in maize. </p> <p>Complete submergence of maize during an early vegetative stage is an event that could contribute to substantial seedling leaf senescence with resultant yield losses. Precipitation events leading to maize submergence are likely to increase in frequency with the predicted climatic shifts in precipitation patterns. Despite the economic implications of this situation, very few studies have explored submergence tolerance in maize [ <a href="#pone.0120385.ref007">7</a>, <a href="#pone.0120385.ref059">59</a>]. Here, we have identified a major QTL for submergence tolerance and characterized the molecular responses of a diverse set of maize inbreds to submergence. Future work will focus on identification and functional characterization of the major <i>Subtol6</i> gene conferring submergence tolerance in Mo18W. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="post-submergencevisualscoringofnamparents" title="sec"> <div class="title" tagx="title" title="title">
Post-submergence visual scoring of NAM parents</div> <p>Initial phenotypic screening for submergence tolerance was conducted using twenty-three genetically diverse maize ( <i>Zea mays</i>) inbreds from the Nested Association Mapping (NAM) parent panel, representing five subfamilies, and B73 [ <a href="#pone.0120385.ref029">29</a>]. The genotypes used in this study and their corresponding subfamilies are listed in <a href="#pone.0120385.s015">S4 Table</a>. Seeds of each genotype were sown directly in soil and grown in greenhouse conditions. Temperature was maintained between 26.7 to 32.2°C during the day and 26.1 to 29.4°C during the night. The relative humidity ranged from 35% to 70%. Supplemental lighting was supplied by 1000-watt metal halide bulbs (Philips Lighting Co., Somerset, NJ, USA) from 0700 to 1800 hrs. Light intensity was measured at approximately 1200 hrs one day prior to submergence with LI-250A light meter and ranged from approximately 378 ± 62.4 μmol m <sup>−2</sup> s <sup>−1</sup> (LI-COR Biosciences, Lincoln, NE, USA). Pots were irrigated twice a day with approximately 50 ml of water. Half-strength Hoagland solution (50 ml) was applied once the first true leaf was visible to ensure that plants had adequate nutrition throughout the duration of the experiment [ <a href="#pone.0120385.ref060">60</a>]. Uniform seedlings (3 seedlings per inbred) were transferred to 416 L tubs at the V2 stage (three leaf stage) for submergence treatment. The water in the tubs was allowed to reach near ambient room temperatures (24–27°C) 24 to 48 h prior to submerging the plants. Plants were arranged in a complete randomized design, and were arranged towards the center to avoid shading by the walls of the tub. Plants were scored using a visual scoring system for the duration of one week post-submergence ( <a href="#pone.0120385.s001">S1 Fig</a>). During the recovery period the temperatures maintained between 26.7 to 32.2°C during the day and 26.1 to 29.4°, and the relative humidity ranged from 35% to 70%. Supplemental lighting was supplied by 1000-watt metal halide bulbs (Philips Lighting Co., Somerset, NJ, USA) from 0700 to 1800 hrs. The duration of submergence was 48 or 96 h. Each leaf was scored visually on a 0 to 10 scale with 0 being completely healthy and 10 indicating complete senescence. Three independent biological replicates of the submergence experiments were conducted during May and August. Samples were collected from three biological replicates for each genotype for scoring and RNA extraction. For QTL mapping, a single RIL family (Mo18w x B73) consisting of 180 lines from the NAM population was selected for phenotypic screening. Four independent biological replicates of the submergence experiments described above were conducted in the months of November, December, May, and June. Pots were submerged for 48 h and arranged in an alpha lattice design with two adjacent tubs comprising one replication. Phenotypic screening of RILs was performed at 48 h rather than 96 h due to the high sensitivity to submergence observed in many lines. A large portion of the RIL population experienced complete senescence when seedlings were submerged for longer than 48 h. Leaf senescence was scored on the first three leaves at 24 h after submergence. </p> </div> <div class="visualscoringforleafh2o2andsuperoxide" title="sec"> <div class="title" tagx="title" title="title">
Visual scoring for leaf H <sub>2</sub>O <sub>2</sub> and Superoxide </div> <p>Plants at the V2 stage were submerged for a period of 96 h. To detect superoxide production, leaves were excised immediately after removing from submergence and placed in a 0.5 mg/mL nitroblue tetrazolium (NBT; Sigma-Aldrich, St. Louis, MO, USA) solution in 10mM potassium phosphate buffer (pH 7.6) and were incubated at 25ºC for 3h in complete darkness. For H <sub>2</sub>O <sub>2</sub> visualization, leaf tissue was collected as described above, immersed in 1 mg/mL 3,3-diaminobenzidine tetrahydrochloride (DAB; Sigma-Aldrich, St. Louis, MO, USA) in 50 mM tris-acetate buffer (pH 5.0), and were incubated at 25ºC for 24 h in complete darkness. For each assay, the leaves were boiled in a 95% ethanol solution until all chlorophyll was removed, and were rehydrated in a 40% glycerol solution for 16 h at 25ºC. </p> </div> <div class="rnaisolation" title="sec"> <div class="title" tagx="title" title="title">
RNA Isolation</div> <p>For submergence experiments, plants were removed from submergence tubs and the shoot tissue was cut approximately 1 cm above the soil line and snap frozen in liquid nitrogen. This process was completed within 10 seconds. RNA was extracted using the TRIzol method and quantified using Nanodrop8000 spectrophotometer (ThermoScientific, Wilmington, DE, USA). RNA cleanup was performed according to RNeasy MinElute Cleanup kit protocol (Qiagen, Valencia, CA, USA). RNA was eluted with 15 μl RNase-free water after on-column DNase1 treatment using the protocol described by the manufacturer. </p> </div> <div class="rnasequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA sequencing</div> <p>RNA sequencing was done using Illumina HiSeq 2000 sequencer. Prior to library construction, RNA integrity was analyzed using Agilent 2100 Bioanalyzer (Agilent Technologies, Palo Alto, CA, USA). RNAseq libraries were prepared using PolyA+ mRNA isolated from each RNA sample following standard Illumina RNASeq protocol. cDNA synthesis and library construction was done according to the manufactures guidelines. The 36 samples were distributed evenly across six lanes of the Illumina HiSeq2000 flow cell. On average, approximately 34,308,889 101 bp single-end reads were produced in each lane. The total number of reads obtained is provided as <a href="#pone.0120385.s016">S5 Table</a>. The raw data are publicly available through the National Center for Biotechnology Information Gene Expression Omnibus (GSE63429). </p> </div> <div class="rna-seqdifferentialexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
RNA-seq differential expression analysis</div> <p>Short reads were mapped against the B73 genome (ZmB73_RefGen_v2 from maizesequence.org) using Bowtie, allowing up to two base mismatches per read [ <a href="#pone.0120385.ref061">61</a>]. Reads mapped to multiple locations were discarded. Detailed mapping information for each sample is provided in <a href="#pone.0120385.s017">S6 Table</a>. Numbers of reads in genes were counted by the HTSeq-count tool using B73 gene annotations (ZmB73_5b_FGS; <a href="http://ftp.maizesequence.org/current/filtered-set/ZmB73_5b_FGS.gff.gz">http://ftp.maizesequence.org/current/filtered-set/ZmB73_5b_FGS.gff.gz</a>) and the "union" resolution mode was used [ <a href="#pone.0120385.ref062">62</a>]. For pair-wise comparisons, the edgeR package with TMM normalization method was used to analyze the numbers of reads aligned to genes and to identify differential expressed genes [ <a href="#pone.0120385.ref063">63</a>, <a href="#pone.0120385.ref064">64</a>]. A threshold value for fold-change of differential expression was set at log <sub>2</sub> (fold-change) &amp;gt; 1 (two-fold actual value) and adjusted <i>p</i>-values &amp;lt; 0.001 for the null hypothesis. </p> </div> <div class="identificationofunannotatedtranscripts" title="sec"> <div class="title" tagx="title" title="title">
Identification of unannotated transcripts</div> <p>Short reads were mapped against the B73 genome (ZmB73_RefGen_v2) using TopHat, allowing up to two base mismatches per read [ <a href="#pone.0120385.ref065">65</a>, <a href="#pone.0120385.ref066">66</a>]. Reads mapped to multiple locations were discarded. All mapped reads were analyzed with Cufflinks package to discover novel transcripts [ <a href="#pone.0120385.ref066">66</a>]. First, mapped reads for one replicate were assembled by Cufflinks to find expressed transcripts, and results of all replicates for the same genotype were merged together using the "cuffmerge" program. Second, these discovered transcripts were compared with B73 gene annotation (ZmB73_5a_WGS) with the "cuffcompare" program to find unannotated transcripts, which were considered as novel transcripts. Finally, the sequences of novel transcripts were extracted and blasted against the non-redundant (NR) library to determine if they are coding or non-coding genes [ <a href="#pone.0120385.ref067">67</a>]. Transcripts were considered to be potential protein coding genes if the e-value was less than 1 x 10 <sup>–8</sup>. Differential expression analysis for unannotated transcripts was performed using edgeR as described above. The nucleotide sequence for <i>PFP1</i> (TCONS_00005048) showed 99.8% identity to that described by Guo et al [ <a href="#pone.0120385.ref068">68</a>] (JQ522972; <a href="#pone.0120385.s011">S11 Fig</a>). </p> </div> <div class="transcriptionfactorandgeneontologyenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Transcription Factor and Gene Ontology Enrichment Analysis</div> <p>To test whether transcription factor families were significantly enriched among differentially expressed genes the hypergeometric test was performed for each transcription factor family. Gene annotations were obtained from the Plant Transcription Factor DatabaseV3.0 [ <a href="#pone.0120385.ref069">69</a>]. The total number of genes annotated for each transcription factor family is listed in <a href="#pone.0120385.s012">S1 Table</a> and <a href="#pone.0120385.s013">S2 Table</a>. </p> <p>Gene ontology enrichment analysis was conducted on sets of genes that were up- and down-regulated in response to submergence with respect to control plants. GO annotations for the complete proteome of B73 (Release 4a.53) were retrieved from AgriGO (Zea mays V5a; <a href="http://bioinfo.cau.edu.cn/agriGO/">http://bioinfo.cau.edu.cn/agriGO/</a>). Enrichment analysis was carried out using the Bioconductor package TopGO in R [ <a href="#pone.0120385.ref070">70</a>]. Only the "biological function" category was used for analysis. Enrichment tests were performed using the "classic" algorithm with Fisher’s exact test. Results of the GO enrichment analysis are provided as <a href="#pone.0120385.s021">S4 File</a>. </p> </div> <div class="phylogeneticanalysisoferftranscriptionfactors" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Analysis of ERF transcription factors</div> <p>To determine phylogenetic relationships between maize, rice and Arabidopsis ERF genes multiple sequence alignment of the AP2 domain was performed using ClustalOMEGA (V1.2.1) with the following settings:—iterations 3—max-guidetree-iterations 3—max-hmm-iterations 3 [ <a href="#pone.0120385.ref071">71</a>] The AP2 domain(s) sequence was obtained from Plant Transcription Factor DatabaseV3.0 [ <a href="#pone.0120385.ref069">69</a>]. In some cases where multiple isoforms were reported for a single gene, only the longest isoform was selected. Sequences with more than one AP2 domain were excluded from the analysis. In total 238 maize proteins, 111 Arabidopsis and 104 rice proteins were used for alignment. Construction of the phylogenetic tree was performed using FastTree2 with Jones-Taylor-Thorton model of amino acid evolution and the "CAT" approximation was used to account for variation in rates across sites [ <a href="#pone.0120385.ref072">72</a>]. Maize genes were classified into ERF subfamilies based on similarity to Arabidopsis and rice ERF family members as reported by Nakano et al. [ <a href="#pone.0120385.ref073">73</a>]. A complete phylogenetic tree is provided as <a href="#pone.0120385.s022">S5 File</a>. </p> </div> <div class="microarrayanalysis" title="sec"> <div class="title" tagx="title" title="title">
Microarray Analysis</div> <p>Microarray analysis for rice ( <i>Oryza sativa</i>) and Arabidopsis ( <i>Arabidopsis thaliana</i>) data was carried out as described by Placido et al. [ <a href="#pone.0120385.ref074">74</a>]. Briefly, the Bioconductor "affy" and "limma" packages were used for reading CEL files and normalizing microarray data [ <a href="#pone.0120385.ref075">75</a>]. The raw intensity values for the GeneChip arrays were background corrected, log <sub>2</sub> transformed, and quantile normalized with robust multiarray average within the R package affy [ <a href="#pone.0120385.ref076">76</a>]. The statistical computation and assessment of differential expression were done with the empirical Bayes function, which moderates the SE values of estimated log fold changes toward a pooled SD value. P values were adjusted to the Benjamini and Hochberg method (BH) to control the false discovery rate. The top table function was used to list the genes according to the specified <i>p-value</i> of 0.05, log <sub>2</sub> fold change of 1, and <i>p-value</i> adjust = BH. </p> </div> <div class="cdnasynthesisandquantitativepcr" title="sec"> <div class="title" tagx="title" title="title">
cDNA synthesis and quantitative PCR</div> <p>First strand cDNA synthesis was performed using SuperScript VILO for real-time quantitative PCR (qPCR) (Invitrogen Corp., Carlsbad, CA, USA). First-strand cDNA was generated for the two-step quantitative real-time PCR (qRT-PCR) by following the protocol from Superscript VILO cDNA synthesis kit. An amount of 2 μg of RNA was used in the 20 μL reaction mixture. For the qPCR reaction, 3 μL of the diluted cDNA (1:20) was used in the 15 μL reaction mixture. In the qPCR reaction volume, 7.5 μL of LightCycler 480 SYBR Green I Mastermix was used (Roche Diagnostics, Indianapolis, IN, USA). The qRT-PCR was carried out using Roche LightCycle 480 II with the following parameter settings (Roche Diagnostics, Indianapolis, IN, USA): 95°C pre-incubation for 5 min, amplification was done for 40 cycles at 95°C for 20 sec and 60°C for 15 sec and extension at 72°C for 15 sec; the melting curve was set-up for 95°C, 65°C, 97°C; cooling was set-up at 40°C for 30 sec. We used two independent tissue samples, each sample having two technical replicates. Primer sequences are provided in <a href="#pone.0120385.s017">S6 Table</a>. Ubiquitin-conjugating enzyme (GRMZM2G102471) was used as an internal control and showed stable expression patterns in all samples. Relative expression was determined using the delta-delta Ct method [ <a href="#pone.0120385.ref077">77</a>]. </p> </div> <div class="qtlanalysis" title="sec"> <div class="title" tagx="title" title="title">
QTL Analysis</div> <p>We conducted QTL analysis on mean leaf senescence score for three leaves from a single plant following submergence. On hundred sixty-six RILs common across 4 independent biological replicates were included in the analysis. The genetic map consisted of 1478 SNPs scored using genotype by sequencing (GBS). Data is from the Panzea database labeled AllZeaGBSv2.3 ( <a href="http://www.panzea.org/lit/data_sets.html#genos">http://www.panzea.org/lit/data_sets.html#genos</a>). Missing data was previously phased and imputed using the algorithm of Swarts et al. [ <a href="#pone.0120385.ref046">46</a>]. Interval mapping was implemented with the R/qtl package and percent phenotypic variation was determined using the "fitqtl" function [ <a href="#pone.0120385.ref078">78</a>, <a href="#pone.0120385.ref079">79</a>]. The 5% significance level corresponded to a logarithm of odds threshold of 3.2 and were determined using 10,000 permutations of the phenotypic data. QTL intervals were further examined for significance by determining a 95% Bayes credible interval. Potential candidate genes within this credible interval underlying the major QTL on chromosome 6 were identified. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Graphical summary of visual scoring system used for phenotypic evaluation of submergence and dark-induced senescence. </div> <p>Leaves were scored visually on a 0–10 scale, with 0 indicating no visual stress symptoms and 10 indicating complete senescence. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s001.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Photograph describing leaf designations used for visual scoring.</div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s002.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Fig</span>  <div class="title" tagx="title" title="title">
Photographs showing tolerant (top and bottom left) and sensitive (top and bottom right pictures) inbreds. </div> <p>Photographs were taken on day 5 of recovery after 96 h of submergence.</p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s003.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Fig</span>  <div class="title" tagx="title" title="title">
Venn diagrams showing differentially expressed genes at 24h and 72h after submergence.</div> <p>All genes showed significant differences in expression between submerged and control plants (p &amp;lt;0.001). Upregulation indicates higher expression in submerged samples. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s004.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S5 Fig</span>  <div class="title" tagx="title" title="title">
Gene Ontology (GO) enrichment analysis of differentially expressed transcripts.</div> <p>(A, C). A subset of GO terms that were significantly enriched among genes showing similar expression patterns in tolerant or sensitive inbreds. Genes that are down-regulated in response to submergence are shown in panel A, while those that were up-regulated are shown in panel C. (B, D) A subset of GO terms that were significantly enriched among genes showing similar expression patterns in all inbreds. GO categories that are enriched among transcripts that are down-regulated in response to submergence are displayed in panel B, while those that are up-regulated in response to submergence are shown in panel D. All genes including in GO enrichment analysis showed significant differences in expression (FDR &amp;lt; 0.001). Scale indicates Log10 p-values determined using Fisher’s Exact test. Full results are provided as <a href="#pone.0120385.s021">S4 File</a>. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s005.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S6 Fig</span>  <div class="title" tagx="title" title="title">
Expression of oxidative stress-related transcripts in comparisons between genotypes.</div> <p>(A). A subset of genes that are associated with oxidative stress responses and showed significant differences in expression between genotypes (FDR &amp;lt; 0.001). (B) Transcription factor genes belonging to families showing significant enrichment among significantly differentially expressed genes. These transcription factors were selected as they have been reported to be involved in leaf senescence in Arabidopsis. All genes including in GO enrichment analysis showed significant differences in expression (FDR &amp;lt; 0.001). Scale indicates Log2 fold change. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s006.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S7 Fig</span>  <div class="title" tagx="title" title="title">
Expression of a subset of ERF transcription factors.</div> <p>(A) ERFs displaying significant differences in response to submergence. (B) ERFs displaying significant differences between inbreds. All genes showed significant differences in expression between submerged and control plants (FDR &amp;lt; 0.001). The scale indicates log2 fold-change. Up-regulation indicates higher expression in submerged samples. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s007.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S8 Fig</span>  <div class="title" tagx="title" title="title">
An unrooted phylogenetic tree of maize, rice and Arabidopsis group B-2 (VII) ERF proteins.</div> <p>The amino acid sequence of AP2 domains were obtained from PlantTFDB [ <a href="#pone.0120385.ref069">69</a>] and aligned using ClustalOMEGA [ <a href="#pone.0120385.ref070">70</a>] and the phylogenetic tree was generated using the Jones-Taylor-Thorton model of amino acid evolution implemented in FastTree2 [ <a href="#pone.0120385.ref071">71</a>]. Maize genes were classified into ERF subfamilies based on similarity to Arabidopsis and rice ERF family members as reported by Nakano et al. [ <a href="#pone.0120385.ref072">72</a>]. Common names are provided for B-2 ERF proteins in Arabidopsis and rice. The protein sequences for rice were obtained from MSU Rice Genome Annotation Project Release 6. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s008.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S9 Fig</span>  <div class="title" tagx="title" title="title">
CBF (C-REPEAT/DRE BINDING FACTOR) co-expression clusters.</div> <p>Each gene in the network is represented by a gray circle. The top cluster was seeded with GRMZM2G069146, while the cluster on the bottom was seeded with GRMZM2G069126. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s009.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S10 Fig</span>  <div class="title" tagx="title" title="title">
Quantitative real-time PCR of a subset of <i>Subtol6</i> candidates. </div> <p>(a) Expression of <i>RELATED TO ABA-INSENSITIVE3(ABI3)/VIVIPAROUS1</i> in B73 and Mo18W. (b) Expression of <i>HEMOGLOBIN2 (HB2</i>; GRMZM2G168898) in B73 and Mo18W. Statistical significance was determined using a Student’s T-Test. Asterisks represent statistically significant differences between genotypes within each treatment: *** p &amp;lt; 0.001, ** p &amp;lt; 0.01. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s010.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S11 Fig</span>  <div class="title" tagx="title" title="title">
Alignment of <i>PFP1</i> (TCONS_00005048) and JQ522972.1. </div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s011.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Transcription factor enrichment analysis from differentially expressed genes of each genotype at 24 h and 72 h after submergence. </div> <p>Comparisons were made between submerged and control samples.</p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s012.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
Transcription factor enrichment analysis for comparisons between genotypes at 24 h and 72 h after submergence. </div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s013.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Table</span>  <div class="title" tagx="title" title="title">
Summary of comparative analysis of maize, Arabidopsis and rice submergence transcriptome analysis. </div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s014.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 Table</span>  <div class="title" tagx="title" title="title">
Maize varieties and subfamily classification [ <a href="#pone.0120385.ref029">29</a>]. </div> <p>NA: not included in the classification by McMullen et al., [ <a href="#pone.0120385.ref029">29</a>]. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s015.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S5 Table</span>  <div class="title" tagx="title" title="title">
Detailed mapping information for all 36 RNA-seq samples.</div> <p>Mapping was done using Bowtie and Tophat as described in Materials and Methods.</p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s016.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S6 Table</span>  <div class="title" tagx="title" title="title">
List of Primers</div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0120385.s017.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 File</span>  <div class="title" tagx="title" title="title">
Genes displaying significant differences between treatments and genotypes (FDR &amp;lt; 0.001).</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0120385.s018.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 File</span>  <div class="title" tagx="title" title="title">
Full list of unannotated genes.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0120385.s019.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 File</span>  <div class="title" tagx="title" title="title">
Genes within <i>Subtol6</i> displaying significant differences (FDR &amp;lt; 0.001) in transcript abundance between Mo18W and B73 under control, or after 24 and 72 h of submergence. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0120385.s020.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S4 File</span>  <div class="title" tagx="title" title="title">
Gene ontology (GO) enrichment analysis of differentially expressed genes.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0120385.s021.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S5 File</span>  <div class="title" tagx="title" title="title">
Phylogenetic tree of ERF transcription factors in rice, maize and Arabidopsis.</div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0120385.s022.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0120385.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosenzweig</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tubiello</span> <span class="given-names" tagx="given-names" title="given-names">FN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldberg</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mills</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bloomfield</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> (2002) <span class="mixed-article-title" title="mixed-article-title">Increased crop damage in the US from excess precipitation under climate change</span>. <span class="source" tagx="source" title="source">Glob Environ Chang</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">197</span>– <span class="lpage" tagx="lpage" title="lpage">202</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zaidi</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rafique</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rai</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">NN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> (2004) <span class="mixed-article-title" title="mixed-article-title">Tolerance to excess moisture in maize (Zea mays L.): susceptible crop stages and identification of tolerant genotypes </span>. <span class="source" tagx="source" title="source">F Crop Res</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">90</span>: <span class="fpage" tagx="fpage" title="fpage">189</span>– <span class="lpage" tagx="lpage" title="lpage">202</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abiko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kotula</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shiono</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malik</span> <span class="given-names" tagx="given-names" title="given-names">AI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colmer</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakazono</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> (2012) <span class="mixed-article-title" title="mixed-article-title">Enhanced formation of aerenchyma and induction of a barrier to radial oxygen loss in adventitious roots of Zea nicaraguensis contribute to its waterlogging tolerance as compared with maize (Zea mays ssp. mays) </span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">1618</span>– <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-3040.2012.02513.x">10.1111/j.1365-3040.2012.02513.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22471697">22471697</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumari</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ware</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Characterization of miRNAs in response to short-term waterlogging in three inbred lines of Zea mays </span>. <span class="source" tagx="source" title="source">PloS One</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e39786</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0039786">10.1371/journal.pone.0039786</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22768123">22768123</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mano</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muraki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujimori</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takamizo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kindiger</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of QTL controlling adventitious root formation during flooding conditions in teosinte (Zea mays ssp. huehuetenangensis) seedlings </span>. <span class="source" tagx="source" title="source">Euphytica</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">142</span>: <span class="fpage" tagx="fpage" title="fpage">33</span>– <span class="lpage" tagx="lpage" title="lpage">42</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mano</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Omori</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Flooding tolerance in interspecific introgression lines containing chromosome segments from teosinte (Zea nicaraguensis) in maize (Zea mays subsp. mays) </span>. <span class="source" tagx="source" title="source">Annals of Botany</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">1125</span>– <span class="lpage" tagx="lpage" title="lpage">39</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1093/aob/mct160">10.1093/aob/mct160</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23877074">23877074</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osman</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic QTL analysis and candidate gene mapping for waterlogging tolerance at maize seedling stage </span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e79305</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0079305">10.1371/journal.pone.0079305</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24244474">24244474</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mapping of QTL associated with waterlogging tolerance during the seedling stage in maize </span>. <span class="source" tagx="source" title="source">Annals of Botany</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">99</span>: <span class="fpage" tagx="fpage" title="fpage">1067</span>– <span class="lpage" tagx="lpage" title="lpage">81</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17470902">17470902</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saab</span> <span class="given-names" tagx="given-names" title="given-names">IN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sachs</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A flooding-induced xyloglucan endo-transglycosylase homolog in maize is responsive to ethylene and associated with aerenchyma </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">385</span>– <span class="lpage" tagx="lpage" title="lpage">91</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8819334">8819334</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thirunavukkarasu</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hossain</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shiriga</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittal</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide expression of transcriptomes and their co-expression pattern in subtropical maize ( <i>Zea mays</i> L.) under waterlogging stress </span>. <span class="source" tagx="source" title="source">PloS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e70433</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0070433">10.1371/journal.pone.0070433</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23936429">23936429</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sachs</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Subbaiah</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saab</span> <span class="given-names" tagx="given-names" title="given-names">IN</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Anaerobic gene expression and flooding tolerance in maize</span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">15</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mertens</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larondelle</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hers</span> <span class="given-names" tagx="given-names" title="given-names">HG</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Induction of pyrophosphate:fructose 6-phosphate 1-phosphotransferase by anoxia in rice seedlings </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">1990</span>; <span class="volume" tagx="volume" title="volume">93</span>: <span class="fpage" tagx="fpage" title="fpage">584</span>– <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16667507">16667507</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mertens</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Pyrophosphate-dependent phosphofructokinase, an anaerobic glycolytic enzyme?</span> <span class="source" tagx="source" title="source">FEBS Lett</span>. <span class="year" tagx="year" title="year">1991</span>; <span class="volume" tagx="volume" title="volume">285</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">5</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1648508">1648508</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plaxton</span> <span class="given-names" tagx="given-names" title="given-names">WC</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The Organization and Regulation of Plant Glycolysis</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Physiol Plant Mol Biol</span>. <span class="year" tagx="year" title="year">1996</span>; <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">185</span>– <span class="lpage" tagx="lpage" title="lpage">214</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012287">15012287</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geigenberger</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Response of plant metabolism to too little oxygen</span>. <span class="source" tagx="source" title="source">Curr Opin Plant Biol</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">247</span>– <span class="lpage" tagx="lpage" title="lpage">256</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12753974">12753974</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailey-Serres</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibbs</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holdsworth</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Licausi</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Making sense of low oxygen sensing</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">129</span>– <span class="lpage" tagx="lpage" title="lpage">38</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.tplants.2011.12.004">10.1016/j.tplants.2011.12.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22280796">22280796</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailey-Serres</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brinton</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Waterproofing crops: effective flooding survival strategies</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">160</span>: <span class="fpage" tagx="fpage" title="fpage">1698</span>– <span class="lpage" tagx="lpage" title="lpage">709</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.112.208173">10.1104/pp.112.208173</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23093359">23093359</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibbs</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morrell</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valdez</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Setter</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenway</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Regulation of alcoholic fermentation in coleoptiles of two rice cultivars differing in tolerance to anoxia </span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">785</span>– <span class="lpage" tagx="lpage" title="lpage">96</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10938871">10938871</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Millenaar</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Plant Movement. Submergence-Induced Petiole Elongation in Rumex palustris Depends on Hyponastic Growth </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">282</span>– <span class="lpage" tagx="lpage" title="lpage">291</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12746533">12746533</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailey-Serres</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Submergence tolerance conferred by Sub1A is mediated by SLR1 and SLRL1 restriction of gibberellin responses in rice </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">16814</span>– <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1073/pnas.0807821105">10.1073/pnas.0807821105</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18936491">18936491</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">K-H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">Y-S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walia</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canlas</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The submergence tolerance regulator Sub1A mediates stress-responsive expression of AP2/ERF transcription factors </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">152</span>: <span class="fpage" tagx="fpage" title="fpage">1674</span>– <span class="lpage" tagx="lpage" title="lpage">92</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.109.152157">10.1104/pp.109.152157</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20107022">20107022</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Folsom</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jikamaru</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronald</span> <span class="given-names" tagx="given-names" title="given-names">PC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walia</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">SUB1A-mediated submergence tolerance response in rice involves differential regulation of the brassinosteroid pathway </span>. <span class="source" tagx="source" title="source">New Phytol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">198</span>: <span class="fpage" tagx="fpage" title="fpage">1060</span>– <span class="lpage" tagx="lpage" title="lpage">70</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/nph.12202">10.1111/nph.12202</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23496140">23496140</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Veen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mustroph</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barding</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vergeer-van</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Welschen-Evertman</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pedersen</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Two Rumex Species from Contrasting Hydrological Niches Regulate Flooding Tolerance through Distinct Mechanisms </span>. <span class="source" tagx="source" title="source">Plant Cell</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">4691</span>– <span class="lpage" tagx="lpage" title="lpage">707</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1105/tpc.113.119016">10.1105/tpc.113.119016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24285788">24285788</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canlas</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maghirang-Rodriguez</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heuer</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Sub1A is an ethylene-response-factor-like gene that confers submergence tolerance to rice </span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">442</span>: <span class="fpage" tagx="fpage" title="fpage">705</span>– <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16900200">16900200</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hattori</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagai</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furukawa</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">X-J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawano</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakakibara</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The ethylene response factors SNORKEL1 and SNORKEL2 allow rice to adapt to deep water</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">460</span>: <span class="fpage" tagx="fpage" title="fpage">1026</span>– <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nature08258">10.1038/nature08258</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19693083">19693083</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Licausi</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Dongen</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giuntoli</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santaniello</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geigenberger</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">HRE1 and HRE2, two hypoxia-inducible ethylene response factors, affect anaerobic responses in Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">Plant J</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">302</span>– <span class="lpage" tagx="lpage" title="lpage">15</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313X.2010.04149.x">10.1111/j.1365-313X.2010.04149.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20113439">20113439</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Licausi</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kosmacz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weits</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giuntoli</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giorgi</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voesenek</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Oxygen sensing in plants is mediated by an N-end rule pathway for protein destabilization</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">479</span>: <span class="fpage" tagx="fpage" title="fpage">419</span>– <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nature10536">10.1038/nature10536</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22020282">22020282</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibbs</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isa</span> <span class="given-names" tagx="given-names" title="given-names">NM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gramuglia</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bassel</span> <span class="given-names" tagx="given-names" title="given-names">GW</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Homeostatic response to hypoxia is regulated by the N-end rule pathway in plants</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">479</span>: <span class="fpage" tagx="fpage" title="fpage">415</span>– <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nature10534">10.1038/nature10534</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22020279">22020279</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McMullen</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kresovich</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Villeda</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradbury</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic properties of the maize nested association mapping population</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">325</span>: <span class="fpage" tagx="fpage" title="fpage">737</span>– <span class="lpage" tagx="lpage" title="lpage">40</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1126/science.1174320">10.1126/science.1174320</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19661427">19661427</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gadjev</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vanderauwera</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gechev</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Transcriptomic footprints disclose specificity of reactive oxygen species signaling in Arabidopsis </span>. <span class="source" tagx="source" title="source">The Plant Journal</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">141</span>: <span class="fpage" tagx="fpage" title="fpage">436</span>– <span class="lpage" tagx="lpage" title="lpage">445</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Apel</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirt</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Reactive oxygen species: metabolism, oxidative stress, and signal transduction</span>. <span class="source" tagx="source" title="source">Annu Rev Plant Biol</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">373</span>– <span class="lpage" tagx="lpage" title="lpage">399</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15377225">15377225</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lenaz</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The mitochondrial production of reactive oxygen species: mechanisms and implications in human pathology </span>. <span class="source" tagx="source" title="source">IUBMB Life</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">159</span>– <span class="lpage" tagx="lpage" title="lpage">164</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11798028">11798028</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">C-M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Signaling linkage between environmental stress resistance and leaf senescence in Arabidopsis</span>. <span class="source" tagx="source" title="source">Plant Signal Behav</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">1564</span>– <span class="lpage" tagx="lpage" title="lpage">1566</span> <span class="comment">doi: <a href="http://dx.doi.org/10.4161/psb.6.10.17003">10.4161/psb.6.10.17003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21921691">21921691</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furukawa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curtis</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tominey</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duong</span> <span class="given-names" tagx="given-names" title="given-names">YH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilcox</span> <span class="given-names" tagx="given-names" title="given-names">BWL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aggoune</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A shared DNA-damage-response pathway for induction of stem-cell death by UVB and by gamma irradiation </span>. <span class="source" tagx="source" title="source">DNA Repair (Amst)</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">940</span>– <span class="lpage" tagx="lpage" title="lpage">948</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1016/j.dnarep.2010.06.006">10.1016/j.dnarep.2010.06.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20634150">20634150</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshiyama</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conklin</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huefner</span> <span class="given-names" tagx="given-names" title="given-names">ND</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Britt</span> <span class="given-names" tagx="given-names" title="given-names">AB</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Suppressor of gamma response 1 (SOG1) encodes a putative transcription factor governing multiple responses to DNA damage </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">106</span>: <span class="fpage" tagx="fpage" title="fpage">12843</span>– <span class="lpage" tagx="lpage" title="lpage">12848</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1073/pnas.0810304106">10.1073/pnas.0810304106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19549833">19549833</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ay</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irmler</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uhlemann</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reuter</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Humbeck</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Epigenetic programming via histone methylation at WRKY53 controls leaf senescence in Arabidopsis thaliana </span>. <span class="source" tagx="source" title="source">The Plant Journal</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">333</span>– <span class="lpage" tagx="lpage" title="lpage">346</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1365-313X.2008.03782.x">10.1111/j.1365-313X.2008.03782.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19143996">19143996</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laun</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmermann</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zentgraf</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Targets of the WRKY53 transcription factor and its role during leaf senescence in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Mol Biol</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">853</span>– <span class="lpage" tagx="lpage" title="lpage">867</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15604721">15604721</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zentgraf</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The antagonist function of Arabidopsis WRKY53 and ESR/ESP in leaf senescence is modulated by the jasmonic and salicylic acid equilibrium </span>. <span class="source" tagx="source" title="source">The Plant Cell</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">819</span>– <span class="lpage" tagx="lpage" title="lpage">830</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17369373">17369373</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis thaliana WRKY25, WRKY26, and WRKY33 coordinate induction of plant thermotolerance</span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">233</span>: <span class="fpage" tagx="fpage" title="fpage">1237</span>– <span class="lpage" tagx="lpage" title="lpage">1252</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1007/s00425-011-1375-2">10.1007/s00425-011-1375-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21336597">21336597</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mao</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Dual-level regulation of ACC synthase activity by MPK3/MPK6 cascade and its downstream WRKY transcription factor during ethylene induction in Arabidopsis </span>. <span class="source" tagx="source" title="source">PLoS Genetics</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1002767</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pgen.1002767">10.1371/journal.pgen.1002767</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22761583">22761583</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref041" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khandelwal</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elvitigala</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quatrano</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis transcriptome reveals control circuits regulating redox homeostasis and the role of an AP2 transcription factor </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">148</span>: <span class="fpage" tagx="fpage" title="fpage">2050</span>– <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.108.128488">10.1104/pp.108.128488</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18829981">18829981</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref042" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mustroph</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oosumi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanetti</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Cross-kingdom comparison of transcriptomic adjustments to low-oxygen stress highlights conserved and plant-specific responses </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">152</span>: <span class="fpage" tagx="fpage" title="fpage">1484</span>– <span class="lpage" tagx="lpage" title="lpage">1500</span>. <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.109.151845">10.1104/pp.109.151845</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20097791">20097791</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref043" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mustroph</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sasidharan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vashisht</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pedersen</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oosumi</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the submergence response of the Arabidopsis thaliana ecotype Columbia </span>. <span class="source" tagx="source" title="source">New Phytol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">190</span>: <span class="fpage" tagx="fpage" title="fpage">457</span>– <span class="lpage" tagx="lpage" title="lpage">71</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03590.x">10.1111/j.1469-8137.2010.03590.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21231933">21231933</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref044" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fowler</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomashow</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis transcriptome profiling indicates that multiple regulatory pathways are activated during cold acclimation in addition to the CBF cold response pathway </span>. <span class="source" tagx="source" title="source">The Plant Cell</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1675</span>– <span class="lpage" tagx="lpage" title="lpage">1690</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12172015">12172015</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref045" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novillo</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Medina</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salinas</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Arabidopsis CBF1 and CBF3 have a different function than CBF2 in cold acclimation and define different gene classes in the CBF regulon </span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">104</span>: <span class="fpage" tagx="fpage" title="fpage">21002</span>– <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18093929">18093929</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref046" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swarts</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romero-Navarro</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romay-Alvarez</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hearne</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">FSFHap (Full-Sib Family Haplotype Imputation) and FILLIN (Fast, Inbred Line Library ImputatioN) optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants </span>. <span class="source" tagx="source" title="source">The Plant Genome</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">12</span>. </span> </li> <li tag="ref"> <a name="pone.0120385.ref047" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woo</span> <span class="given-names" tagx="given-names" title="given-names">HR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">I-J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The RAV1 transcription factor positively regulates leaf senescence in Arabidopsis</span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">3947</span>– <span class="lpage" tagx="lpage" title="lpage">57</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1093/jxb/erq206">10.1093/jxb/erq206</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20826506">20826506</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref048" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sowa</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duff</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guy</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Altering hemoglobin levels changes energy status in maize cells under hypoxia</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span>. <span class="year" tagx="year" title="year">1998</span>; <span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">10317</span>– <span class="lpage" tagx="lpage" title="lpage">21</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9707645">9707645</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref049" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perazzolli</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dominici</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romero-Puertas</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zago</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeier</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sonoda</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Arabidopsis nonsymbiotic hemoglobin AHb1 modulates nitric oxide bioactivity</span>. <span class="source" tagx="source" title="source">The Plant Cell</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">2785</span>– <span class="lpage" tagx="lpage" title="lpage">2794</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15367716">15367716</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref050" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manac’h-Little</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Igamberdiev</span> <span class="given-names" tagx="given-names" title="given-names">AU</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Hemoglobin expression affects ethylene production in maize cell cultures</span>. <span class="source" tagx="source" title="source">Plant Physiol Biochem</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">485</span>– <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15914016">15914016</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref051" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wally</span> <span class="given-names" tagx="given-names" title="given-names">OSD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dionisio</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ayele</span> <span class="given-names" tagx="given-names" title="given-names">BT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jami</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Hemoglobin Control of Cell Survival/Death Decision Regulates in Vitro Plant Embryogenesis</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">165</span>: <span class="fpage" tagx="fpage" title="fpage">810</span>– <span class="lpage" tagx="lpage" title="lpage">825</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24784758">24784758</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref052" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blokhina</span> <span class="given-names" tagx="given-names" title="given-names">OB</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chirkova</span> <span class="given-names" tagx="given-names" title="given-names">TV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagerstedt</span> <span class="given-names" tagx="given-names" title="given-names">KV</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Anoxic stress leads to hydrogen peroxide formation in plant cells</span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">1179</span>– <span class="lpage" tagx="lpage" title="lpage">90</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11432936">11432936</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref053" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukao</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailey-Serres</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Plant responses to hypoxia—is survival a balancing act?</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">449</span>– <span class="lpage" tagx="lpage" title="lpage">56</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15337495">15337495</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref054" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zarka</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cook</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomashow</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Cold induction of Arabidopsis CBF genes involves multiple ICE (inducer of CBF expression) promoter elements and a cold-regulatory circuit that is desensitized by low </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">910</span>– <span class="lpage" tagx="lpage" title="lpage">918</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14500791">14500791</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref055" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharabi-Schwager</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lers</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samach</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guy</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porat</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Overexpression of the CBF2 transcriptional activator in Arabidopsis delays leaf senescence and extends plant longevity </span>. <span class="source" tagx="source" title="source">J Exp Bot</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">261</span>– <span class="lpage" tagx="lpage" title="lpage">73</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1093/jxb/erp300">10.1093/jxb/erp300</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19854800">19854800</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref056" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharabi-Schwager</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Samach</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porat</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Overexpression of the CBF2 transcriptional activator in Arabidopsis suppresses the responsiveness of leaf tissue to the stress hormone ethylene </span>. <span class="source" tagx="source" title="source">Plant Biol (Stuttg)</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">630</span>– <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1111/j.1438-8677.2009.00255.x">10.1111/j.1438-8677.2009.00255.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20636906">20636906</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref057" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mustroph</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boamfa</span> <span class="given-names" tagx="given-names" title="given-names">EI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laarhoven</span> <span class="given-names" tagx="given-names" title="given-names">LJJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harren</span> <span class="given-names" tagx="given-names" title="given-names">FJM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albrecht</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimm</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Organ-specific analysis of the anaerobic primary metabolism in rice and wheat seedlings. I: Dark ethanol production is dominated by the shoots </span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">225</span>: <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16845530">16845530</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref058" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mustroph</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boamfa</span> <span class="given-names" tagx="given-names" title="given-names">EI</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laarhoven</span> <span class="given-names" tagx="given-names" title="given-names">LJJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harren</span> <span class="given-names" tagx="given-names" title="given-names">FJM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pörs</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimm</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Organ specific analysis of the anaerobic primary metabolism in rice and wheat seedlings II: light exposure reduces needs for fermentation and extends survival during anaerobiosis </span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">225</span>: <span class="fpage" tagx="fpage" title="fpage">139</span>– <span class="lpage" tagx="lpage" title="lpage">52</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16802177">16802177</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref059" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Functional genomics of maize submergence tolerance and cloning of the related gene Sicyp51 </span>. <span class="source" tagx="source" title="source">Science in China Series C: Life …</span> <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">48</span>: <span class="fpage" tagx="fpage" title="fpage">337</span>– <span class="lpage" tagx="lpage" title="lpage">45</span>. </span> </li> <li tag="ref"> <a name="pone.0120385.ref060" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hoagland DR, Arnon DI. The water-culture method for growing plants without soil. Circ. 347. Univ. of Calif.Agric. Exp. Station, Berkley; 1950 </span> </li> <li tag="ref"> <a name="pone.0120385.ref061" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pop</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <span class="source" tagx="source" title="source">Genome Biology</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">R25</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25">10.1186/gb-2009-10-3-r25</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174">19261174</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref062" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Anders S. HTSeq: Analysing high-throughput sequencing data with Python. 2010. Available: <a href="http://www-huber.embl.de/users/anders/HTSeq/">http://www-huber.embl.de/users/anders/HTSeq/</a> </span> </li> <li tag="ref"> <a name="pone.0120385.ref063" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smyth</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span>. <span class="mixed-article-title" title="mixed-article-title">EdgeR: a Bioconductor package for differential expression analysis of digital gene expression data </span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">139</span>– <span class="lpage" tagx="lpage" title="lpage">40</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btp616">10.1093/bioinformatics/btp616</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">19910308</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref064" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oshlack</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A scaling normalization method for differential expression analysis of RNA-seq data</span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">R25</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1186/gb-2010-11-3-r25">10.1186/gb-2010-11-3-r25</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20196867">20196867</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref065" /> <span class="label" tagx="label" title="label">65</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pachter</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">TopHat: discovering splice junctions with RNA-Seq</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1105</span>– <span class="lpage" tagx="lpage" title="lpage">11</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btp120">10.1093/bioinformatics/btp120</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445">19289445</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref066" /> <span class="label" tagx="label" title="label">66</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertea</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mortazavi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Baren</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation </span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">511</span>– <span class="lpage" tagx="lpage" title="lpage">5</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/nbt.1621">10.1038/nbt.1621</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464">20436464</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref067" /> <span class="label" tagx="label" title="label">67</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruitt</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tatusova</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maglott</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">D501</span>– <span class="lpage" tagx="lpage" title="lpage">4</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15608248">15608248</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref068" /> <span class="label" tagx="label" title="label">68</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronhovde</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soundararajan</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elthon</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Pyrophosphate-dependent fructose-6-phosphate 1-phosphotransferase induction and attenuation of Hsp gene expression during endosperm modification in quality protein maize </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">158</span>: <span class="fpage" tagx="fpage" title="fpage">917</span>– <span class="lpage" tagx="lpage" title="lpage">929</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.111.191163">10.1104/pp.111.191163</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22158678">22158678</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref069" /> <span class="label" tagx="label" title="label">69</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pérez-Rodríguez</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riaño-Pachón</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corrêa</span> <span class="given-names" tagx="given-names" title="given-names">LGG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kersten</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mueller-Roeber</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">PlnTFDB: updated content and new features of the plant transcription factor database</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">38</span>: <span class="fpage" tagx="fpage" title="fpage">D822</span>– <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1093/nar/gkp805">10.1093/nar/gkp805</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19858103">19858103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref070" /> <span class="label" tagx="label" title="label">70</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Alexa A, Rahnenfuhrer J. TopGO: topGO: Enrichment analysis for Gene Ontology. R package version 2.14.0; 2010 </span> </li> <li tag="ref"> <a name="pone.0120385.ref071" /> <span class="label" tagx="label" title="label">71</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sievers</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilm</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dineen</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karplus</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega </span>. <span class="source" tagx="source" title="source">Mol Syst Biol</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">539</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1038/msb.2011.75">10.1038/msb.2011.75</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835">21988835</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref072" /> <span class="label" tagx="label" title="label">72</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">MN</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dehal</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arkin</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">FastTree 2—approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e9490</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref073" /> <span class="label" tagx="label" title="label">73</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakano</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujimura</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinshi</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-Wide Analysis of the ERF Gene Family</span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">140</span>: <span class="fpage" tagx="fpage" title="fpage">411</span>– <span class="lpage" tagx="lpage" title="lpage">432</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16407444">16407444</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref074" /> <span class="label" tagx="label" title="label">74</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Placido</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">MT</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Folsom</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kruger</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baenziger</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Introgression of novel traits from a wild wheat relative improves drought adaptation in wheat </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">161</span>: <span class="fpage" tagx="fpage" title="fpage">1806</span>– <span class="lpage" tagx="lpage" title="lpage">19</span> <span class="comment">doi: <a href="http://dx.doi.org/10.1104/pp.113.214262">10.1104/pp.113.214262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23426195">23426195</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref075" /> <span class="label" tagx="label" title="label">75</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smyth</span> <span class="given-names" tagx="given-names" title="given-names">GK</span> </span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Limma: linear models for microarray data</span> In: <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gentleman</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carey</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudoit</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irizarry</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, eds, <span class="source" tagx="source" title="source">Bioinformatics and Computational Biology Solutions Using R and Bioconductor</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>, pp <span class="fpage" tagx="fpage" title="fpage">397</span>– <span class="lpage" tagx="lpage" title="lpage">420</span>; <span class="year" tagx="year" title="year">2005</span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref076" /> <span class="label" tagx="label" title="label">76</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Irizarry</span> <span class="given-names" tagx="given-names" title="given-names">RA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hobbs</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collin</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beazer-Barclay</span> <span class="given-names" tagx="given-names" title="given-names">YD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Antonellis</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scherf</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data </span>. <span class="source" tagx="source" title="source">Biostatistics</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">249</span>– <span class="lpage" tagx="lpage" title="lpage">264</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12925520">12925520</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref077" /> <span class="label" tagx="label" title="label">77</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method </span>. <span class="source" tagx="source" title="source">Methods</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">402</span>– <span class="lpage" tagx="lpage" title="lpage">8</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="pone.0120385.ref078" /> <span class="label" tagx="label" title="label">78</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Development Core Team. R: A Language and Environment for Statistical Computing. Vienna, Austria; 2009 </span> </li> <li tag="ref"> <a name="pone.0120385.ref079" /> <span class="label" tagx="label" title="label">79</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broman</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Churchill</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">R/qtl: QTL mapping in experimental crosses</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">889</span>– <span class="lpage" tagx="lpage" title="lpage">890</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12724300">12724300</a> </span> </span> </li> </ul> </div> </div>  </body></html>