<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Genomics</span> <span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5217398">5217398</a> </span> <span class="publisher-id" title="publisher-id">3421</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-016-3421-8">10.1186/s12864-016-3421-8</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Transcriptome profiling reveals the genetic basis of alkalinity tolerance in wheat</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">Chen</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">haiyangxingchen@163.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quan</span> <span class="given-names" tagx="given-names" title="given-names">Tai-Yong</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">quanty@sdu.edu.cn</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Zhong-Yi</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">Zhongyi.Li@csiro.au</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">Kang-Li</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">1107871429@qq.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">Li</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">yanli1949@163.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">Yu</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">liangyu_79@163.com</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dai</span> <span class="given-names" tagx="given-names" title="given-names">Jiu-Lan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">daijiulan@sdu.edu.cn</span> </span> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">Guang-Min</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">Xiagm@sdu.edu.cn</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Shu-Wei</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">liushuwei@126.com</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], </span> <span class="citation_author_institution" id="Aff2">[2], </span> <span class="citation_author_institution" id="Aff3">[3], </span> <span class="citation_author_institution" id="Aff4">[4], </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-1-1</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-1-1</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">24</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-2-29</span> <span class="accepted" title="accepted">accepted: 2016-12-14</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Soil alkalinity shows significant constraints to crop productivity; however, much less attention has been paid to analyze the effect of soil alkalinity on plant growth and development. Shanrong No. 4 (SR4) is an alkalinity tolerant bread wheat cultivar selected from an asymmetric somatic hybridization between the bread wheat cultivar Jinan 177 (JN177) and tall wheatgrass ( <i>Thinopyrum ponticum</i>), which is a suitable material for studying alkalinity tolerant associate genes. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The growth of SR4 plant seedlings was less inhibited than that of JN177 when exposed to alkalinity stress conditions. The root cytosolic Na <sup>+</sup>/K <sup>+</sup> ratio in alkalinity stressed SR4 was lower than in JN177, while alkalinity stressed SR4 contained higher level of nutrient elements than in JN177. SR4 plant seedlings accumulated less malondialdehyde (MDA) and reactive oxygen species (ROS), it also showed higher activity of ROS scavenging enzymes than JN177 under alkalinity stress. The root intracellular pH decreased in both alkalinity stressed JN177 and SR4, however, it was much lower in SR4 than in JN177 under alkalinity stress. The transcriptomes of SR4 and JN177 seedlings exposed to alkalinity stress were analyzed by digital gene expression tag profiling method. Alkalinity stress conditions up- and down-regulated a large number of genes in the seedling roots that play the functions in the categories of transcription regulation, signal transduction and protein modification. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>SR4 expresses a superior tolerance to alkaline stress conditions which is due to its strong absorbing ability for nutrient ions, a strong regulating ability for intracellular and rhizosphere pH and a more active ROS scavenging ability. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (doi:10.1186/s12864-016-3421-8) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Alkali tolerance</div> <div class="kwd" title="kwd">
Digital gene expression</div> <div class="kwd" title="kwd">
pH</div> <div class="kwd" title="kwd">
Reactive oxygen species</div> <div class="kwd" title="kwd">
Wheat</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001809</span> <span class="institution" title="institution">National Natural Science Foundation of China</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">31271707</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">National Basic Research 973 Program of China</span> </div> <span class="award-id" tagx="award-id" title="award-id">2012CB114204</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Fundamental Research Funds of Shandong University</span> </div> <span class="award-id" tagx="award-id" title="award-id">2014JC048</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Young Scholars Program of Shandong University</span> </div> <span class="award-id" tagx="award-id" title="award-id">2016WLJH39</span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Soil salinity and alkalinity both represent significant constraints to crop productivity. Nearly 830 million hectares of the land are affected by salt–alkali stress, and over 434 million hectares are alkalinized soils [ <a href="#CR1">1</a>]. Alkaline salt stress and neutral salt stress are two distinct kinds of stresses for plants and should be called alkali stress and salt stress. Salt stress in the soil generally involves osmotic stress and ion injury, while alkali stress has an added high-pH effect. The high-pH directly affects the absorption of mineral elements and interferes with ionic balance. Thus, the effects of soil alkalinization on plants resulting from high concentrations of NaHCO <sub>3</sub> and Na <sub>2</sub>CO <sub>3</sub> may be more destructive than the effects of soil salinization induced by the accumulation of neutral salts, such as NaCl and Na <sub>2</sub>SO <sub>4</sub> [ <a href="#CR2">2</a>, <a href="#CR3">3</a>]. The plant response to salinity stress has been exhaustively researched, but much less attention has been paid to analysing the effect of soil alkalinity on plant growth and development. The identification of alkali–responsive genes may provide valuable information on understanding the genetic basis of plants to alkali stress and improve alkali tolerance of crops by genetic engineering. </p> <p>Wheat is an important crop in the world because of its high production (678 million tons in 2009, FAO), wide geographical range and high proportion of human consumption. Wheat is a glycophyte which is relatively sensitive to both salinity and alkalinity stress. However, two wheat lines, named as Shanrong No. 4 (SR4) and Shanrong No. 3 (SR3) were reported as salinity tolerant lines due to their more tolerant of salinity than JN177 [ <a href="#CR4">4</a>, <a href="#CR5">5</a>], which were selected from the derivatives of an asymmetric somatic hybridization between the salinity sensitive line, bread wheat cultivar Jinan 177 (JN177) and the highly tolerant related species, tall wheatgrass ( <i>Thinopyrum ponticum</i>) [ <a href="#CR4">4</a>, <a href="#CR5">5</a>]. Transcriptomic and proteomic experiments showed that imposing salinity stress induced the differential up- or down-regulation of many genes present in SR3, and some of these, when expressed as transgenes, produced a marked positive impact on the plant’s level of salinity tolerance [ <a href="#CR6">6</a>– <a href="#CR11">11</a>]. Although field trials showed that SR3 out-performs SR4 in pH neutral saline soils, the reverse was the case in alkaline pH saline soils (unpublished data). Thus, while SR3 had been registered as a salinity tolerant cultivar, SR4 was recommended for cultivation on high pH soils which is, therefore, a suitable material for identifying alkalinity tolerant associate genes. </p> <p>Genome scale analysis of gene expression profiles is a powerful method to understand biological functions, such as cell differentiation, development and stress responses. With the increasing availability of sequencing data, expression profiling has been widely used in recent years to identify genes that are involved in the adaptive responses to drought and other abiotic stresses. High-throughput transcriptome sequencing and digital gene expression (DGE) tag profiling are efficient and cost-effective methods for characterizing non-model organisms without the need for a reference genome [ <a href="#CR12">12</a>, <a href="#CR13">13</a>]. DGE is particularly useful for identifying differentially expressed genes, thanks to its capacity to detect unknown and low abundance transcripts, and to generate absolute (rather than relative) transcript abundances [ <a href="#CR14">14</a>]. </p> <p>Recently, some studies for alkaline-stress responses have been reported in <i>Glycine soja</i>, <i>Leymus chinensis</i> and <i>Tamarix hispida</i> [ <a href="#CR15">15</a>– <a href="#CR18">18</a>]; however, there is still little information about the alkaline-stress responses of common wheat. The present study focuses on the short-term and more sustained responses of the wheat root to alkalinity stress, by contrasting the SR4 transcriptome with that of JN177 using DGE profiling method. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="theplantresponsetoalkalinitystress" title="sec"> <div class="title" tagx="title" title="title">
The plant response to alkalinity stress</div> <p>The growth of seedlings and their roots of wheat line, SR4 was less inhibited than that of wheat cultivar JN177 when they were grown in the presence of the alkalinity stress. The growth of the seedlings and roots were similar between SR4 and JN177 when they were grown in the absent of the alkalinity stress (Fig. <a href="#Fig1">1a</a> and <a href="#Fig1">b</a>). However, the shoot dry weight and root length were reduced to ~22 and ~35% respectively in SR4, while they were reduced to ~45 and 53% respectively in JN177 (Fig. <a href="#Fig1">1c–f</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Growth of seedlings and roots of JN177 and SR4 under non-alkalinity and alkalinity stress conditions. <b>a</b>- <b>d</b> Three week old JN177 and SR4 seedlings grown under non-alkalinity stress ( <b>a</b>, <b>b</b>) and under 100 mM alkali salts ( <b>c</b>, <b>d</b>). Bar: 1 cm length. <b>e</b>- <b>f</b> The effect of alkalinity stress on shoot dry weight ( <b>e</b>) and root length ( <b>f</b>). Data are given in the form mean ± s.d. The double asterisks represent significant difference determined by the Student’s <i>t</i>-test at <i>P</i> &amp;lt; 0.01 (**) </p>   </div> <p /> <p>The high-salt environments in alkaline soil can break the ion homeostasis of plant cells; moreover, the high-pH caused by alkali stress directly affects the absorption of mineral elements. To understanding how SR4 adapt to overcome alkali stress, we analyzed the ion contents in roots of SR4 and JN177. The alkaline medium induced an increased Na <sup>+</sup> and decreased K <sup>+</sup> contents, overall lowering the K <sup>+</sup>⁄Na <sup>+</sup> ratio in the roots of both wheats. The effect of the stress on root Na <sup>+</sup> content was more marked for JN177 than for SR4, and vice versa for root K <sup>+</sup> content (Fig. <a href="#Fig2">2a</a> and <a href="#Fig2">b</a>). SR4 line maintained a higher root K <sup>+</sup>/Na <sup>+</sup> ratio than that in JN177 cultivar. Root Ca <sup>2+</sup>, Mg <sup>2+</sup> and total Fe concentrations were increased by the alkalinity stress in both cultivars, but SR4 maintained a higher level of root Ca <sup>2+</sup>, Mg <sup>2+</sup> and total Fe content than that of JN177 (Fig. <a href="#Fig2">2c–e</a>). In contrast, alkali stress had little effect on the levels of Zn <sup>2+</sup> and Cl <sup>−</sup> in both cultivars (Fig. <a href="#Fig2">2f</a> and <a href="#Fig2">g</a>). Alkali stress significantly decreased the levels of Mn <sup>2+</sup>, NH <sub>4</sub> <sup>+</sup>, SO <sub>4</sub> <sup>2−</sup>, NO <sub>3</sub> <sup>−</sup> and H <sub>2</sub>PO <sub>4</sub> <sup>−</sup> in roots of both cultivars, and the effect of the stress on root Mn <sup>2+</sup>, NH <sub>4</sub> <sup>+</sup>, NO <sub>3</sub> <sup>−</sup> and H <sub>2</sub>PO <sub>4</sub> <sup>−</sup> content was more marked for JN177 than for SR4 (Fig. <a href="#Fig2">2h–l</a>), indicating SR4 might absorb more nutrient elements than JN177 under alkali stress. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Content of selected ions of seedling roots of JN177 and SR4 under non-alkalinity and alkalinity stress conditions. Root content of ( <b>a</b>) Na <sup>+</sup>, ( <b>b</b>) K <sup>+</sup>, ( <b>c</b>) Ca <sup>2+</sup>, ( <b>d</b>) Mg <sup>2+</sup>, ( <b>e</b>) Fe, ( <b>f</b>) Zn <sup>2+</sup>, ( <b>g</b>) Cl <sup>−</sup>, ( <b>h</b>) Mn <sup>2+</sup>, ( <b>i</b>) NH <sub>4</sub> <sup>+</sup>, ( <b>j</b>) SO <sub>4</sub> <sup>2−</sup>, ( <b>k</b>) NO <sub>3</sub> <sup>−</sup> and ( <b>l</b>) H <sub>2</sub>PO <sub>4</sub> <sup>−</sup>. Data are given in the form mean ± s.d. The asterisk represents significant difference determined by the Student’s <i>t</i>-test at <i>P</i> &amp;lt; 0.05 (*) </p>   </div> <p /> <p>The high-pH caused by alkali stress can also directly affect the internal pH of the plant. To investigate how SR4 adapt to overcome the high-pH under alkali stress, we detected the intracellular pH in roots of both wheats by using pH indicator 2′7′-bis(2-carboxyethyl)-5(6)-carboxyfluorescein (BCECF). Under alkalinity stress, the BCECF fluorescence was decreased in root of both SR4 and JN177 compared to non-stressed condition, however, the fluorescence intensity was much lower in SR4 than in JN177 under alkalinity stress (Fig. <a href="#Fig3">3a–d</a>). Since the BCECF fluorescence intensity was proportional to the intracellular pH (Fig. <a href="#Fig3">3e</a>), we calculated the intracellular pH in root of both SR4 and JN177 according to BCECF fluorescence intensity. Results indicated that the intracellular pH fell about 0.1 and 0.2 pH unit in JN177 and SR4, respectively, under alkalinity stress (Fig. <a href="#Fig3">3f</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>The intracellular pH of seedling roots of JN177 and SR4 under non-alkalinity and alkalinity stress conditions. <b>a</b>- <b>d</b> Fluorescence images for JN177 and SR4 under non-alkalinity stress ( <b>a</b>, <b>b</b>) and under 100 mM alkali salts ( <b>c</b>, <b>d</b>). <b>e</b> The fluorescence intensity is proportional to pH. <b>f</b> The intracellular pH of seedling roots of JN177 and SR4. Data are given in the form mean ± s.d </p>   </div> <p /> <p>Environmental stresses including alkali stress can induce the production of toxic reactive oxygen species (ROS), which if not controlled, can pose a threat to cells. Thus, plants have developed antioxidant mechanisms to eliminate superfluous ROS under alkaline-saline stress, in which superoxide dismutase (SOD), ascorbate peroxidase (APX), glutathione peroxidase (GPX), peroxidase (POD) and catalase (CAT) are most important antioxidative enzymes. To assess whether SR4 can effectively scavenge ROS under alkali stress, we detected the ROS content and activity of major ROS scavenging enzymes in roots of both wheats. ROS can degrade polyunsaturated lipids to form malondialdehyde (MDA), which is generally taken as a marker of oxidative stress, thus we also detected the content of MDA in roots of both SR4 and JN177. Both O <sub>2</sub> <sup>•−</sup> radicals and H <sub>2</sub>O <sub>2</sub> was raised by alkali stress in wheats, and both cultivars accumulated similar level of ROS in the absence of alkalinity stress, but, in the presence of stress, SR4 accumulated ROS less than JN177 did (Fig. <a href="#Fig4">4a–c</a>). Similarly, root MDA content was raised by the stress, but less so in SR4 than in JN177 (Fig. <a href="#Fig4">4d</a>). With respect to the activities of the ROS scavenging enzymes SOD, APX, GPX, CAT and POD, the activity of CAT was consistently higher in SR4 than in JN177 (Fig. <a href="#Fig4">4e</a>); the activities of SOD, APX and GPX were inhibited by the stress, but the activity level of SOD and APX were consistently higher in SR4 than in JN177 (Fig. <a href="#Fig4">4f–h</a>); while the activity of POD was unaffected by the stress in SR4, but its activity was increased in JN177 (Fig. <a href="#Fig4">4i</a>). These results were fully consistent with the alkalinity tolerance profiles which showed that SR4 displaying a superior tolerance of alkalinity stress than JN177. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Content of ROS and activities of ROS scavenging enzymes of seedling roots of JN177 and SR4 under non-alkalinity and alkalinity stress conditions. <b>a</b> Root O <sub>2</sub> <sup>•-</sup> content in three leaf stage seedlings grown in the absence (CK) or presence of 100 mM alkali salts. <b>b</b> The relative O <sub>2</sub> <sup>•-</sup> content in the roots shown in ( <b>a</b>) calculated from mean pixel density. <b>c</b>, <b>d</b> Root content of H <sub>2</sub>O <sub>2</sub> ( <b>c</b>) and MDA ( <b>d</b>). <b>e</b>- <b>i</b> The activity of the ROS scavenging-associated enzymes in the seedling roots including catalase (CAT) ( <b>e</b>), superoxide dismutase (SOD) ( <b>f</b>), ascorbate peroxidise (APX) ( <b>g</b>), glutathione peroxidise (GPX) ( <b>h</b>) and peroxidase (POD) ( <b>i</b>). Data are given in the form mean ± s.d. The asterisk and double asterisks represent significant difference determined by the Student’s <i>t</i>-test at <i>P</i> &amp;lt; 0.05 (*) and <i>P</i> &amp;lt; 0.01 (**), respectively </p>   </div> <p /> </div> <div class="transcriptomeprofilingsofwheatseedlingrootsunderalkalinitystress" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome profilings of wheat seedling roots under alkalinity stress</div> <p>A total of 35,671,144 unfiltered tags were recovered from the six libraries (Table <a href="#Tab1">1</a>), with the number of sequences from each sample ranged from 5.7 to 6.2 million. After data filtering, a total of 33,994,682 clean tags were obtained for further analysis. The cleaned tags of each sample ranged from 5.4 to 5.9 million. About 50% of the clean tags can be mapped to the reference sequences from NCBI unigene databases, and most of them were uniquely mapped to gene tags (Table <a href="#Tab1">1</a>). All these unambiguous tags were mapped to over 20,000 unigenes (Table <a href="#Tab1">1</a>). The clean tags were also mapped to the wheat genomic DNA sequences from the IWGSC website ( <a href="ftp://ftpmips.helmholtz-muenchen.de/plants/wheat/IWGSC/genePrediction_v2.2/">ftp://ftpmips.helmholtz-muenchen.de/plants/wheat/IWGSC/genePrediction_v2.2/</a>). Although the percentage of the clean tags mapped to IWGSC wheat genome sequences (about 70%) were higher than that mapped to NCBI unigene databases (50%), the percentage of unambiguous tags mapped to the IWGSC wheat genome sequences (only 21–23%) were much less than that of NCBI unigene databases (42–46%) (Additional file <a href="#MOESM1">1</a>: Tables S1). Since only unambiguous tags can be used for expression analysis, thus, the NCBI unigene databases were used for wheat RNA-seq tags annotation. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Statistics analysis of the six DGE tag libraries constructed from the seedling roots of JN177 and SR4 mapping to NCBI unigene databases </p>  <table> <thead> <tr> <th>Mapping to NCBI unigene databases</th> <th>JNCK</th> <th>JN0.5</th> <th>JN24</th> <th>SRCK</th> <th>SR0.5</th> <th>SR24</th> </tr> </thead> <tbody> <tr> <td>Raw data</td> <td>6234376</td> <td>5975104</td> <td>5763389</td> <td>5954683</td> <td>5897848</td> <td>5845744</td> </tr> <tr> <td>Total number of clean tags</td> <td>5943058</td> <td>5712643</td> <td>5487894</td> <td>5692072</td> <td>5633349</td> <td>5525666</td> </tr> <tr> <td>Clean tags mapped to gene</td> <td>3184388</td> <td>2806047</td> <td>2666921</td> <td>2837152</td> <td>2745121</td> <td>2771868</td> </tr> <tr> <td>% of clean tags mapped to gene</td> <td>53.58%</td> <td>49.12%</td> <td>48.60%</td> <td>49.84%</td> <td>48.73%</td> <td>50.16%</td> </tr> <tr> <td>Unambiguous tags mapped to gene</td> <td>2741617</td> <td>2430920</td> <td>2317889</td> <td>2460457</td> <td>2379007</td> <td>2388369</td> </tr> <tr> <td>% of unambiguous tags mapped to gene</td> <td>46.13%</td> <td>42.55%</td> <td>42.24%</td> <td>43.23%</td> <td>42.23%</td> <td>43.22%</td> </tr> <tr> <td>Unambiguous tag-mapped genes</td> <td>21552</td> <td>20185</td> <td>20483</td> <td>20663</td> <td>20199</td> <td>20648</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p> <i>Note: JN</i> JN177, <i>SR</i> SR4, <i>CK</i> plants not subjected to alkalinity stress, 0.5 and 24: plants subjected to alkalinity stress for 0.5 h and 24 h, respectively </p> </div>  <p /> <p>The DGE analysis identified a set of 2,619 and 3028 genes respectively as being transcriptionally altered in SR4 and JN177 through the exposure to alkalinity stress conditions (Additional file <a href="#MOESM2">2</a>: Tables S2 and Additional file <a href="#MOESM3">3</a>: Table S3). To verify whether the DGE output represented the true variation of the transcripts, twelve genes were randomly chosen for the qRT-PCR amplification. The results were clearly showed that the qRT-PCR data were consistent with the DGE output (Fig. <a href="#Fig5">5</a>). To evaluate the biological functions of alkaline stress responsive genes, GO enrichment analysis were conducted. In all, 13 GO categories were over-represented in SR4 ( <i>P</i> &amp;lt; 0.01, FDR &amp;lt; 0.05) (Fig. <a href="#Fig6">6a</a>), while 14 GO categories were over-represented in JN177 ( <i>P</i> &amp;lt; 0.01, FDR &amp;lt; 0.05) (Fig. <a href="#Fig6">6b</a>). The GO categories over-represented in both wheat cultivars were much similar, with most of them for the metabolising different compounds, responsing to stress and iron transport. In SR4 there were also GO categories for response to ROS, epigenetic regulation of gene expression and protein targeting, while RNA modification was over-represented in JN177. In SR4, a group of 62 differential expressed genes were associated with signal transduction and 61 genes in transcription regulation, which included genes encoding calcium sensors, protein kinases, alternative splicing factors and various transcription factors. Another 22 genes were associated with protein modification, among which were proteins involved in phosphorylation, small ubiquitin-related modifier (SUMO) modification, ubiquitin modification and ubiquitin-like modification. There were 37 genes encoding ion transport proteins, 27 genes encoding enzymes participating in lipid metabolism, organic acid metabolism and phenylpropanoid metabolic processes, 23 genes associated with ROS scavenging, 22 genes encoding chaperones, five genes involved in DNA damage repair and five genes involved in chromatin structure remodeling. In JN177, 20 differential expressed genes were associated with RNA modification, including those for pre-mRNA processing factor, RNA helicase and pre-mRNA-splicing factor. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>qRT-PCR-based validation of the transcription expression of twelve genes selected from the DGE analysis. <b>a</b> Ca <sup>2+</sup>/H <sup>+</sup>-exchanging protein. <b>b</b> AAA-type ATPase. <b>c</b> NRT1/PTR-like transporter. <b>d</b> Auxin response factor 6-like. <b>e</b> Auxin induced protein PCNT115. <b>f</b> Ascorbate peroxidase. <b>g</b> Ethylene responsive factor 5a. <b>h</b> DNA damage-binding protein 1a. <b>i</b> DNA repair protein RAD23. <b>j</b> PP2A regulatory subunit TAP46. <b>k</b> MADS-box transcription factor TaAGL4. <b>l</b> MADS-box transcription factor TaAGL33. The qRT-PCR data are given in the form mean ± s.d. ( <i>n</i> = 3). The asterisk and double asterisks represent significant difference determined by the Student’s <i>t</i>-test at <i>P</i> &amp;lt; 0.05 (*) and <i>P</i> &amp;lt; 0.01 (**), respectively </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Functional categories of genes responding to alkalinity stress and of those in the full reference set in SR4 and JN177. There are 13 functional categories in SR4 ( <b>a</b>) and 14 functional categories in JN177 ( <b>b</b>). Functional categories for gene group that are significantly differentially expressed (shown by <i>green bars</i>) and for full reference gene set ( <i>blue bars</i>) are shown with their percentages </p>   </div> <p /> <p>Exposured the seedlings to alkali stress for 0.5 h, 844 and 676 genes were up-regulated, while 774 and 1165 genes were down-regulated, respectively in SR4 and JN177. Under the 24 h exposure, 856 and 655 genes were up-regulated, while 722 and 1244 genes were down-regulated, respectively in SR4 and JN177 (Fig. <a href="#Fig7">7</a>). Of these differential expressed genes under two conditions, 254 genes were up- and 280 genes down-regulated commonly in both treatments in SR4, while 247 genes were up- and 440 genes down-regulated commonly in both treatments in JN177. A set of 33 and 8 genes were up-regulated in the short exposure treatment, but down-regulated in the long one, while 10 and 17 other genes behaved in the opposite way, respectively in SR4 and JN177. The transcription of a major proportion of the genes was altered either under only the short exposure (557 and 421 up-, 484 and 708 down-regulated respectively in SR4 and JN177), or under only the long exposure (592 and 391 up-, 409 and 796 down-regulated respectively in SR4 and JN177) (Fig. <a href="#Fig7">7</a>). Among the genes affected by the stress at both time point in SR4, ten encoded chaperones and were strongly induced, six encoded heat shock proteins, ten encoded transporter proteins (including four nitrate transporters, a plasma membrane H <sup>+</sup>-ATPase and a vacuolar H <sup>+</sup>-ATPase), nine encoded glutathione transferase and four encoded enzymes involved in phenylpropanoid synthesis. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>The relationship of differential expressed genes under alkalinity stress conditions treated for 0.5 h and for 24 h. <b>a</b> differential expressed genes in SR4, <b>b</b> differential expressed genes in JN177. Four ovals represent the four types of differential expressed gene groups which are indicated above the figure. The numbers within one circle denote the number of genes up- or down-regulated by alkali stress condition at one time point. The numbers within multiple circles denote the number of genes up- or down-regulated by alkali stress condition at multiple time points </p>   </div> <p /> <p>Most of the genes specifically regulated by the short alkalinity stress treatment in SR4 encoded proteins involved in signal transduction (34 genes), transcription regulation (36 genes), ion transport (14 genes) and molecular chaperoning (12 genes). The 34 signal transduction genes included eight genes encoding calmodulin, calcineurin B-like proteins (CBL), CBL-interacting protein kinase or calcium/calmodulin-dependent protein kinase, five genes encoding mitogen-activated protein kinases, ten genes encoding receptor like kinases, three genes encoding jasmonic acid-related proteins and a homolog of the gibberellin receptor GID1. The 36 regulated transcription regulation genes contained six genes encoding alternative splicing regulators, six genes encoding zinc finger transcription factors, five genes encoding members of the MYB transcription factor family and four genes encoding NAM, ATAF1,2, CUC2 (NAC) transcription factors. The regulated transporter genes were one gene encoding the vacuolar proton-inorganic pyrophosphatase (H <sup>+</sup>-PPase), one gene encoding Ca <sup>2+</sup>/H <sup>+</sup>-exchanging protein and one genes encoding transporters for NO <sub>3</sub> <sup>−</sup>, NH <sub>4</sub> <sup>+</sup>, K <sup>+</sup>, Fe <sup>3+</sup>, Mg <sup>2+</sup> and Si. Of the 12 chaperones, ten genes encoded small (&amp;lt;40 kDA) heat shock proteins. </p> <p>The genes regulated by the long exposure time to alkali stress in SR4 were genes encoding proteins associated with the cellular response to nutrition status, ion transport, redox homeostasis balance, histone modification and cellular metabolism. There were 16 genes encoding transporters (four NO <sub>3</sub> <sup>−</sup> transporters, three aquaporins, two NH <sub>4</sub> <sup>+</sup> transporters, two SO <sub>4</sub> <sup>2−</sup> transporters, two Fe-phytosiderophore transporters and one each of K <sup>+</sup>, H <sub>2</sub>PO <sub>4</sub> <sup>−</sup>, Fe <sup>3+</sup> and Zn <sup>2+</sup> transporters). In addition, there were 12 genes encoding ROS scavenging enzymes, including SOD, thioredoxin and glutathione transferase. Genes encoding a ROS responsive DNA damage repair-related poly (ADP-ribose) polymerase, an RCD1-like protein [ <a href="#CR11">11</a>] and two DNA damage repair proteins were up-regulated, while the set of genes repressed included four genes encoding various histone variants and one gene encoding histone deacetylase 19-like protein. There were 31 genes involved in cellular metabolism were specifically induced, including 12 genes encoding enzymes involved in lipid metabolism, nine genes encoding enzymes in carbohydrate metabolism and five genes encoding enzymes in flavonoid synthesis. Two gibberellin associated genes were up-regulated: one encoded a gibberellin 3-oxidase and the other was a homolog of <i>SPINDLY</i>, a negative regulator of GA signalling. Details of all of these regulated genes are listed in Additional file <a href="#MOESM4">4</a>: Table S4 and Additional file <a href="#MOESM5">5</a>: Table S5. </p> </div> <div class="thedifferentialresponseofsr4andjn177toalkalinitystress" title="sec"> <div class="title" tagx="title" title="title">
The differential response of SR4 and JN177 to alkalinity stress</div> <p>In the absence of alkaline stress, 1,721 genes (591 up- and 1130 down-regulated) showed differentially transcribed in SR4 compared to that in JN177 (Fig. <a href="#Fig8">8</a>). The most well populated GO categories were those involved in transport, transcription regulation, signal transduction, epigenetic modification, cellular metabolism and molecular chaperoning. Among the 31 genes involved in transport, seven genes encoding NO <sub>3</sub> <sup>−</sup> transporters, six genes encoding aquaporins, two genes encoding NH <sub>4</sub> <sup>+</sup> transporters, two genes encoding vacuolar H <sup>+</sup>-ATPases, and one gene encoding a plasma memebrane H <sup>+</sup>-ATPase. Among the 28 transcription factor genes, 21 genes were more abundantly transcribed in SR4 than that in JN177. Similarly, nine genes encoding protein kinases were up-regulated in SR4, as well as one gene encoding a jasmonate-induced protein and one gene encoding JAZ protein. There were 17 genes involved in carbohydrate metabolism which were differentially transcribed in SR4 and JN177. The abundance of seven transcripts associated with flavonoid synthesis was less in SR4 than in JN177. All eight differentially transcribed genes involved in epigenetic regulation were down-regulated in SR4; these included three genes encoding histone variants, two genes encoding DNA methyltransferases, one gene encoding histone deacetylase, one gene encoding DNA-(apurinic or apyrimidinic site) lyase and one gene encoding chromatin assembly factor protein. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Differential expressed transcripts in SR4 and JN177 under both non- alkalinity and alkalinity stress conditions. <b>a</b> Genes transcribed more abundantly in SR4 than in JN177, <b>b</b> vice versa </p>   </div> <p /> <p>Following the short alkalinity stress treatment, the abundance of 364 transcripts was higher and that of 315 was lower in SR4 compared to in JN177; the numbers under the long exposure were 703 and 497, respectively (Fig. <a href="#Fig8">8</a>, Additional file <a href="#MOESM6">6</a>: Table S6). The majority of genes encoding transporters (for NO <sub>3</sub> <sup>−</sup>, NH <sub>4</sub> <sup>+</sup>, H <sub>2</sub>PO <sub>4</sub> <sup>−</sup>, Fe <sup>3+</sup> and some other ions) were more highly transcribed in SR4 than in JN177. A set of 13 genes encoding various transcription factors were up-regulated in SR4 in response to the stress. As also was the case in the absence of the stress, the abundance of transcripts was more abundant in SR4 than in JN177 including the genes encoding a plasma membrane H <sup>+</sup>-ATPase, a jasmonate-induced protein and a JAZ protein. Most of the 20 regulated genes encoding ROS scavenging enzymes were more highly transcribed in SR4 than in JN177, including genes encoding SOD, glutathione transferase and dehydroascorbate reductase. The transcription of an <i>RCD1-like</i> homolog was also more abundant in SR4 than in JN177. Although the seven genes involved in epigenetic regulation which were differentially transcribed in SR4 than JN177 in the absence of stress were down-regulated in SR4, six of the seven were more strongly induced by the 24 h stress episode in SR4 than JN177. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="sr4hasgreatertolerancetoalkalinestressthanjn177" title="sec"> <div class="title" tagx="title" title="title">
SR4 has greater tolerance to alkaline stress than JN177</div> <p>Root and shoot lengths are both sensitive indices of the plant response to abiotic stress. While the alkalinity stress imposed here was sufficiently strong to compromise the growth of both the wheat seedling shoot and root, SR4 was clearly better able to tolerate with the stress than was JN177 (Fig. <a href="#Fig1">1</a>). The ability of plants challenged with alkalinity stress to take up K <sup>+</sup> is typically weakened, which also leads to the over- uptaking of the toxic ion Na <sup>+</sup>. The ability of a plant to maintain K <sup>+</sup>/Na <sup>+</sup> ratio homeostasis has been suggested as a diagnostic of tolerance to both salinity and alkalinity stress. When plants were exposed to alkalinity stress, the low tolerant JN177 plants were less able to maintain their K <sup>+</sup>/Na <sup>+</sup> ratio than were the SR4 ones. In addition, less MDA was generated in SR4 than in JN177 roots, which implied that a reduced level of plasma membrane damage caused by lipid peroxidation. </p> </div> <div class="sr4possessesahighabilityinabsorbingnutrientionsunderalkalistress" title="sec"> <div class="title" tagx="title" title="title">
SR4 possesses a high ability in absorbing nutrient ions under alkali stress</div> <p>The cellular response to alkalinity stress is much less well studied than that to salinity stress. It has been suggested that a high pH environment can inhibit the plant’s capacity to take up NO <sub>3</sub> <sup>−</sup> and H <sub>2</sub>PO <sub>4</sub> <sup>−</sup>, with consequences for the nutritional status of the plant [ <a href="#CR19">19</a>, <a href="#CR20">20</a>]. The transcriptome analysis revealed that a number of NO <sub>3</sub> <sup>−</sup>, H <sub>2</sub>PO <sub>4</sub> <sup>−</sup> and SO <sub>4</sub> <sup>2−</sup> transporters were much more strongly up-regulated in SR4 plants under alkalinity stress condition than in JN177 plants. The qRT-PCR analysis also indicated that a nitrate transporter gene <i>NRT1</i> was induced in SR4 under alkalinity stress (Fig. <a href="#Fig5">5c</a>). Moreover, the NO <sub>3</sub> <sup>−</sup> and H <sub>2</sub>PO <sub>4</sub> <sup>−</sup> content were much higher in SR4 than in JN177 under alkali stress (Fig. <a href="#Fig2">2k−l</a>). The implication is that the genetic make-up of SR4 allows it, when exposed to alkalinity stress, to better maintain an adequate supply of key anions than can JN177. A high soil pH has a significant impact on the availability of a number of important cations (notably Ca <sup>2+</sup>, Mg <sup>2+</sup>, Zn <sup>2+</sup> and Fe <sup>3+</sup>), with further negative consequences for the ion homeostasis and nutritional status [ <a href="#CR19">19</a>, <a href="#CR21">21</a>]. Graminaceous plants acquire Fe <sup>3+</sup> by releasing phytosiderophores able to chelate the ion. Of note is that the constitutive expression in barley of an Fe phytosiderophore transporter gene has a marked ameliorative effect on the plant’s tolerance of alkaline soil [ <a href="#CR22">22</a>]. The present analysis revealed that two putative Fe phytosiderophore transporter genes were up-regulated by alkalinity stress in SR4 and that their transcript abundance was higher than in JN177. In addition, a number of other cation transporter genes were induced by alkalinity stress in SR4, and two of these (encoding a Mn <sup>2+</sup> transporter and a putative Zn <sup>2+</sup> transporter) were differentially up-regulated in alkalinity stressed SR4. Physiological analysis also indicated that SR4 contained higher levels of Ca <sup>2+</sup>, Mg <sup>2+</sup>, total Fe, Mn <sup>2+</sup> and NH <sub>4</sub> <sup>+</sup> than JN177 under alkali stress (Fig. <a href="#Fig2">2c–e, h</a> and <a href="#Fig2">i</a>). Taken together, we suggest that SR4 may have a higher ability to absorb both nutrient anions and cations than JN177 under alkaline condition, which may help to explain the better performance of this cultivar upon alkalinity. </p> </div> <div class="sr4controlstheintracellularandrhizospherephlevelsmoreeffectivelythanjn177" title="sec"> <div class="title" tagx="title" title="title">
SR4 controls the intracellular and rhizosphere pH levels more effectively than JN177</div> <p>In addition to its impeding nutrient uptake, a high soil pH can also directly affect the internal pH of the plant [ <a href="#CR19">19</a>]. Part of the resistance mechanism against alkalinity stress involves the expenditure of energy to control both the intracellular and rhizosphere pH. Eukaryotic cells have evolved for this purpose various ion pumps, such as plasma membrane proton translocating ATPases (PM H <sup>+</sup>-ATPases), vacuolar H <sup>+</sup>-ATPases (V-ATPases) and proton pyrophosphatase (H <sup>+</sup>-PPases). V-ATPases form part of the cellular machinery responsible for regulating intracellular pH [ <a href="#CR23">23</a>– <a href="#CR25">25</a>], while H <sup>+</sup>-PPases act to complement V-ATPase transport activity through the acidification of intracellular compartments [ <a href="#CR26">26</a>]. The DGE analysis revealed a number of genes encoding V-ATPase or H <sup>+</sup>-PPase being induced by alkalinity stress; some of these were more abundantly transcribed in SR4 than in JN177. Measurement of intracellular pH also indicated that the root intracellular pH decreased in both alkalinity stressed JN177 and SR4, moreover, it was much lower in SR4 than in JN177 under alkalinity stress (Fig. <a href="#Fig3">3</a>), which coincide with the DGE data. The implication is that SR4 may be better able than JN177, through its use of various V-ATPases and H <sup>+</sup>-PPases, to maintain intracellular pH homeostasis when challenged with alkalinity stress. </p> <p>The plasma membrane H <sup>+</sup>-ATPases of plant cells establish the pH and membrane potential gradients across the plasma membrane [ <a href="#CR27">27</a>], and some recent research has shown that plasma membrane H <sup>+</sup>-ATPase appears to mediate proton secretion during the adaptation of the root to alkaline stress [ <a href="#CR28">28</a>]. The protein kinase PKS5 (CIPK11) inhibits the plasma membrane H <sup>+</sup>-ATPase by preventing interaction with 14-3-3 protein, loss-of function <i>pks5</i> mutant plants are more tolerant of high external pH due to extrusion of protons to the extracellular space [ <a href="#CR28">28</a>]. Moreover, a chaperone J3 (DnaJ homolog 3) activates the plasma membrane H <sup>+</sup>-ATPase by repressing PKS5 [ <a href="#CR29">29</a>]. In tomato, the product of <i>TFT4</i> (which encodes a 14-3-3 signaling protein) integrates proton efflux and the PKS5-J3 pathway as part of the root’s alkalinity stress response [ <a href="#CR30">30</a>]. The present transcriptomic analysis identified the differential up-regulation of one gene in SR4 encoding a plasma membrane H <sup>+</sup>-ATPase gene, both under non-stressed and alkalinity stressed conditions. In addition there were three differentially regulated genes encoding a DnaJ homolog chaperone, as well as one encoding a 14-3-3-like protein. This suggests that the differential transcription may enable SR4 to secrete protons more effectively into the rhizosphere than can JN177, and thereby improve its ability to withstand stress. </p> </div> <div class="sr4tissuehasamoreeffectiverosscavengingabilitythanjn177" title="sec"> <div class="title" tagx="title" title="title">
SR4 tissue has a more effective ROS scavenging ability than JN177</div> <p>Plants have evolved a number of ROS removal mechanisms to avoid excessive cellular damage when high levels of ROS are present [ <a href="#CR31">31</a>]. SOD converts O <sub>2</sub> <sup>•−</sup> radicals to H <sub>2</sub>O <sub>2</sub> [ <a href="#CR32">32</a>], which is in turn decomposed by APX, GPX and CAT [ <a href="#CR31">31</a>]. Under alkalinity stress, SR4 showed higher SOD, CAT and APX activity than JN177 (Fig. <a href="#Fig4">4e–g</a>), and the transcript abundance of most of the 20 genes involved in ROS scavenging was also higher in SR4. Direct measurement of ROS showed that the SR4 accumulated less ROS than JN177 (Fig. <a href="#Fig4">4a–c</a>). All these results suggested that SR4 had a powerful ability to prevent the over accumulation of ROS and to protect the cells against ROS caused by alkali stress. It had been concluded that the enhanced salinity tolerance of SR3, the sister line of SR4, also governed by a more active ROS scavenging system than JN177 [ <a href="#CR6">6</a>, <a href="#CR9">9</a>]. It seems like that the higher ROS scavenging ability is the common feature of both SR4 and SR3 in defending salt-alkali stress. </p> <p>The ROS responsive <i>RCD1</i>-like gene <i>TaSRO1</i> has been shown to be an important component of the enhanced salinity tolerance of the SR4 sister line SR3, partly due to its ability to regulate ROS homeostasis and partly to maintain DNA integrity [ <a href="#CR11">11</a>]. The same gene, along with two others associated with DNA damage repair, was also induced by alkalinity stress, with a stronger level of induction in SR4 than in JN177. It suggested that SR4 might also have stronger ability to maintain DNA integrity under alkali stress, like SR3. </p> </div> <div class="involvementofepigeneticmodificationinthealkalinitystressresponses" title="sec"> <div class="title" tagx="title" title="title">
Involvement of epigenetic modification in the alkalinity stress responses</div> <p>Since the abiotic stress response involves changes to the transcription of large numbers of genes, global regulation mechanisms are likely to be triggered by the stress. One such mechanism could operate through alterations in the chromatin, involving both non-sequence changes to the DNA (eg through cytosine methylation) and covalent modifications to the histone (eg acetylation, methylation, phosphorylation, ubiquitinylation) or even the replacement of canonical histones by specific variants [ <a href="#CR33">33</a>]. Histone variants are known to affect the plant response to both low and high temperature stress, and also are induced by P starvation [ <a href="#CR34">34</a>– <a href="#CR36">36</a>]. The incorporation of histone variants depends in part on the action of histone chaperones and chromatin remodeling factors [ <a href="#CR37">37</a>, <a href="#CR38">38</a>]. Some aspects of transcription during episodes of abiotic stress have been shown to be affected by the methylation variants present within specific genes in a range of plant species [ <a href="#CR39">39</a>– <a href="#CR41">41</a>]. The somatic hybridization process (as used to generate SR4, see [ <a href="#CR5">5</a>]) has been shown to induce wholesale alterations in cytosine methylation patterns, and some of these have been implicated in underlying the heightened salinity tolerance displayed by SR3 [ <a href="#CR42">42</a>]. Both cytosine methylation and histone deacetylation variants are believed to be responsible for regulating the level of transcription of certain salinity stress responsive genes in SR3 [ <a href="#CR42">42</a>]. The present transcriptomic analysis identified eight genes involved in epigenetic regulation (encoding various histone variants, a DNA methyltransferase, a histone deacetylase and a chromatin assembly factor protein) which were differentially transcribed between SR4 and JN177, and six of these genes were up-regulated by the imposition of alkalinity stress for 24 h and showed higher expression level in SR4 than in JN177. Whether epigenetic modifications are involved in alkali stress responses and its contribution to the alkali tolerance in SR4 will require further investigation. </p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Alkali stress was normally closely related to salt stress in soil; however, alkali stress and salt stress are two distinct kinds of stresses for plants. Salt stress consists of the low water potentials and ion toxicities factors, while alkali stress has an added high-pH effect, which is the major feature of alkalinity stress. The high pH surrounding the roots could direct some ions to precipitate, which may destroy the nutrient supply and ion balance around the roots. Moreover, a high soil pH can also directly affect the internal pH of the plant. Bread wheat cultivar SR4 expresses a superior tolerance to alkaline stress than its parent cultivar JN177. This study showed that this superiority are due to its capability in taking up nutritionally important anions and cations, in having a more active ROS scavenging ability and also probably in regulating both the plant’s internal pH and that of its rhizosphere. Since higher ROS scavenging ability was also involved in the salinity tolerance of another somatic introgression wheat cultivar SR3, considering the extra high pH effect of alkalinity stress compared with salinity stress, the high absorbing ability for nutrient ions and the good regulating ability for intracellular and rhizosphere pH might be more important for alkali tolerance of wheat. All of the evidences provide clues for uncovering the tolerance of SR4 to the alkalinity stress and broaden our understanding of the mechanism of wheat to alkalinity. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="plantmaterialsandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and growth conditions</div> <p>The bread wheat ( <i>Triticum aestivum</i>) cultivar Shanrong No. 4 (SR4) used in this study is a derivative of an asymmetric somatic hybrid between bread wheat cultivar JN177 and tall wheatgrass ( <i>Th. ponticum</i>) created by Dr. Guangmin Xia, and the detail information about asymmetric somatic hybridization was described in Xia’s report [ <a href="#CR4">4</a>]. JN177 and SR4 grains were germinated on moist filter paper for two days at 23°C and then transferred to half-strength Hoagland’s liquid medium under a 16 h photoperiod, 50% relative humidity and 300 μmol m <sup>−2</sup>s <sup>−1</sup> photon flux density light. Alkalinity stress was applied after nine days by adding 100 mM mixed salts (NaHCO <sub>3</sub>:Na <sub>2</sub>CO <sub>3</sub> in a molar ration at 9:1, pH 8.90). The solution was changed daily to avoid anoxia. After ten days of exposure to the alkalinity stress, measurements were made for shoot length, root length and seedling fresh and dry weight from 20 seedlings of each cultivar, with triple replicates. Image J software (National Institutes of Health) was used to measure the length of the roots of the two wheat cultivars under non-alkaline and alkaline stress conditions. At the three leaf stage (21 d after germination), the hydroponic solution was supplemented with 100 mM mixed salts (NaHCO <sub>3</sub>:Na <sub>2</sub>CO <sub>3</sub> in a molar ration at 9:1, pH 8.90). After 24 h exposure to alkaline stress, the roots of seedlings in the alkalinity treatment and control were harvested for detecting the contents of MDA, O <sub>2</sub> <sup>•−</sup>, H <sub>2</sub>O <sub>2</sub>, ions and the activities of APX, GPX, POD, SOD and CAT. The experiment was replicated three times. Seedlings used as a source of RNA extraction were grown to the three leaf stage in half strength Hoagland’s liquid medium. After either a 0.5 h or a 24 h exposure to 100 mM of the mixed alkali salts (NaHCO <sub>3</sub>:Na <sub>2</sub>CO <sub>3</sub> by 9:1 molar ratio, pH 8.90), the roots of both treated and non-treated seedlings were snap-frozen in liquid nitrogen and stored at −80°C until required. Each treatment was replicated three times. </p> </div> <div class="physiologicalindicatorsofstress" title="sec"> <div class="title" tagx="title" title="title">
Physiological indicators of stress</div> <p>The root MDA contents were assessed following Liu et al. [ <a href="#CR9">9</a>]. The root ion contents were determined following Yang et al. [ <a href="#CR19">19</a>]. POD activity was measured following Ranieri et al. [ <a href="#CR43">43</a>]. The root H <sub>2</sub>O <sub>2</sub> content and activities of SOD, GPX and CAT were assayed using commercial kits provided by Beyotime (Beyotime, Shanghai, China). The activity of APX was assayed using commercial kit provided by Solarbio (Solarbio, Beijing, China). To monitor the accumulation of O <sub>2</sub> <sup>•−</sup> radicals, roots were infiltrated with 10 mL nitro blue tetrazolium (NBT) solution (0.5 mg mL <sup>−1</sup> NBT supplied in 10 mM potassium phosphate buffer, pH 7.8) for 1 h in the dark, then staining reaction was stopped by adding an excess of 70% (v/v) methanol. The roots were rinsed in fresh 70% (v/v) methanol and then observed and photographed under a stereomicroscope. The strength of the NBT signal was estimated from a digital image of the stained tissue with ImageJ software (National Institutes of Health). The polymerization product after NBT staining can reflect the O <sub>2</sub> <sup>•−</sup> content, thus the relative superoxide content of each sample can be represented by the average pixel density in the root area of the photograph. </p> </div> <div class="intracellularph" title="sec"> <div class="title" tagx="title" title="title">
Intracellular pH</div> <p>The intracellular pH was monitored with BCECF. The roots of JN177 and SR4 were incubated for 30 min in the dark at 22–24°C in incubation buffer (pH 6.2) containing 10 mM MOPS, 50 mM KCl, 1 mM MgCl <sub>2</sub>, 0.05% pluronic F-127 and 10 μM BCECF-AM. Loading was terminated by rinsing the roots three times in incubation buffer lacking the dye. BCECF-AM loaded into the cytoplasm was hydrolyzed to BCECF by esterase. The intracellular pH was monitored by measuring the fluorescence of BCECF with a laser scan confocal microscope (LSM700, Zesis, Germany). The excitation wavelength was 488 nm and emission wavelength was 525 nm. The pH showed a linear relationship with fluorescence intensity over the pH ranges 6.4-7.6. For pH calibration, the wheat roots were incubated in the dark in calibration buffer (pH6.6, 7.0, 7.2, 7.4 and 7.6) containing 10 mM MOPS, 130 mM KCl, 1 mM MgCl <sub>2</sub>, 10 mM NaCl, 0.05% pluronic F-127, 10 μg/mL nigericin and 10 μM BCECF-AM. </p> </div> <div class="rnasequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA sequencing</div> <p>Tag libraries were constructed from root mRNAs extracted from alkalinity-stressed (both 0.5 h and 24 h exposure times) JN177 and SR4 plants, as well as from the non-stressed roots of both cultivars. Magnetic beads coated with polyT were used to isolate the mRNA fraction of a 30 μg aliquot of total RNA. Double strand cDNA synthesized from this mRNA was digested with <i>Nla</i>III, an enzyme which cleaves at the sequence CATG. <i>Nla</i>III-restricted 3′ cDNA fragments were purified by Oligo(dT) beads, then the Illumina adaptor 1 is ligated to the sticky 5′ end, which retains the CATG site. The fragments were further digested with <i>Mme</i>I (this enzyme cleaves 17 bp downstream of the CATG site), then ligated to Illumina adaptor 2 to construct a 21 bp tag library with different adaptors at both ends. After linear PCR amplification based on the primer pair corresponding to the adaptors, the amplified samples were purified by polyacrylamide gel electrophoresis. The quality of the constructed library was analyzed by Agilent 2100 Bioanaylzer and ABI StepOnePlus Real-Time PCR System. The tag library was finally sequenced using a HiSeq 2000 device (Illumina). The short-read sequence data from the six libraries are respectively deposited in NCBI Sequence Read Archive (SRA, <a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) with accession number SRR3223630, SRR3223636, SRR3223646, SRR3223672, SRR3223674 and SRR3223675. Adaptor tags (the raw sequences containing 3′ adaptor sequences), empty reads (reads with only 3′ adaptor sequences but no Tags), low-quality tags (tags with ambiguous base calls), tags of unusual length (longer or shorter than 21 nt) and nonredundant tags (tags with a copy number less than 2) were removed from the raw data, yielding a dataset consisting of clean tags. Subsequently, the clean tags were classified according to their copy number in the library, their percentage of total clean tags was calculated, and the saturation of the library was analyzed. </p> </div> <div class="differentialtranscription" title="sec"> <div class="title" tagx="title" title="title">
Differential transcription</div> <p>The frequency of each tag in the various cDNA libraries was used to infer differential transcription. To identify genes involved, the reference sequences from NCBI wheat EST collection ( <a href="http://ftp.ncbi.nih.gov/repository/UniGene/Triticum_aestivum/Ta.seq.uniq.gz">http://ftp.ncbi.nih.gov/repository/UniGene/Triticum_aestivum/Ta.seq.uniq.gz</a>) or IWGSC wheat genomic DNA sequences ( <a href="ftp://ftpmips.helmholtz-muenchen.de/plants/wheat/IWGSC/genePrediction_v2.2/">ftp://ftpmips.helmholtz-muenchen.de/plants/wheat/IWGSC/genePrediction_v2.2/</a>) were searched for CATG sites, all the possible CATG + 17 nt sequences were used as a reference tag library for following clean tags mapping. All clean tags were mapped to the reference tag library with perfect match or allowing a 1 nt mismatch (there might be SNPs between different varieties) [ <a href="#CR44">44</a>], conducting by a Perl program through string matching. Tags identifying more than one gene were removed, and the number of remaining unambiguous tags was normalized to TPM (number of transcripts per million clean tags). To identify differentially expressed genes (DEGs), an algorithm to identify differential transcript accumulation between different samples (alkalinity vs no alkalinity treatment, SR4 vs JN177) was used according to Audic and Claverie [ <a href="#CR45">45</a>]. A false discovery rate (FDR) threshold of 0.001 and a log2 ratio threshold of 1 were applied as the criteria for declaring differential transcription [ <a href="#CR46">46</a>]. Gene ontology (GO) assignments and enrichment analysis was conducted to assign biological process, cellular component and molecular function through hypergeometric test to map all differentially expressed genes to terms in GO database ( <a href="http://amigo.geneontology.org/amigo">http://amigo.geneontology.org/amigo</a>), looking for significantly enriched GO terms in DEGs comparing to the genome background. </p> <div class="quantitativereal-timepcr(qrt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR (qRT-PCR)</div> <p>Total root RNA extracted using the TRIzol reagent (Invitrogen) provided the template for a qRT-PCR assay. The cDNA was synthesized from 2 μg DNase-treated total RNA based on M-MLV reverse transcriptase (TaKaRa) in accordance with the manufacturer’s protocol. Each 10 μL qRT-PCR contained 5 μL SYBR Green, 5 μM of each primer and 1 μL template. The reference sequence was a segment of the wheat <i>Actin</i> gene (AB181991). Relative expression level was calculated using the 2 <sup>-ΔΔCt</sup> formula. For each of the three biological replicates, three technical replicates were included. All relevant primer sequences are listed in Additional file <a href="#MOESM7">7</a>: Table S7. </p> </div> </div> </div> </div> <div class="back" title="back"> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="additionalfiles" title="sec"> <div class="title" tagx="title" title="title">
Additional files</div> <p> </p><div class="media" title="media"> <a href="12864_2016_3421_MOESM1_ESM.doc">LINK</a> <span class="label" tagx="label" title="label">Additional file 1: Table S1.</span>  <p>Statistics analysis of the six DGE tag libraries constructed from the seedling roots of JN177 and SR4 mapped to wheat genomic DNA sequences. (DOC 32 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM2_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 2: Table S2.</span>  <p>Differentially expressed genes in SR4 after alkalinity stress. Total number of genes differentially up- and down-regulated in roots under 100 mM alkaline stress treatment compared with the sample without alkalinity stress ( <i>P</i> &amp;lt; 0.05, <i>q</i> &amp;lt; 0.15). (XLS 1142 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM3_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 3: Table S3.</span>  <p>Differentially expressed genes in JN177 after alkalinity stress. Total number of genes differentially up- and down-regulated in roots under 100 mM alkaline stress treatment compared with the sample without alkalinity stress ( <i>P</i> &amp;lt; 0.05, <i>q</i> &amp;lt; 0.15). (XLS 1340 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM4_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 4: Table S4.</span>  <p>Differentially expressed genes at different time point of alkalinity treatment in SR4. (XLS 4177 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM5_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 5: Table S5.</span>  <p>Differentially expressed genes at different time point of alkalinity treatment in JN177. (XLS 1417 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM6_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 6: Table S6.</span>  <p>Differentially expressed genes between SR4 and JN177 under non-alkalinity or alkalinity stress. (XLS 1230 kb) </p>  </div> <div class="media" title="media"> <a href="12864_2016_3421_MOESM7_ESM.xls">LINK</a> <span class="label" tagx="label" title="label">Additional file 7: Table S7.</span>  <p>The primer sequences for qRT-PCR analysis. (XLS 40 kb)</p>  </div> <p /> </div> </div> </div> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
ABA</div> <div class="def" title="def"> <p>Abscisic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
APX</div> <div class="def" title="def"> <p>Ascorbate peroxidase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CAT</div> <div class="def" title="def"> <p>Catalase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CBL</div> <div class="def" title="def"> <p>Calcineurin B-like proteins</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CDPK</div> <div class="def" title="def"> <p>Calcium/calmodulin-dependent protein kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CIPK</div> <div class="def" title="def"> <p>CBL-interacting protein kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DGE</div> <div class="def" title="def"> <p>Digital gene expression</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DHAR</div> <div class="def" title="def"> <p>Dehydroascorbate reductase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GA</div> <div class="def" title="def"> <p>Gibberellic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p>Gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GPX</div> <div class="def" title="def"> <p>Glutathione peroxidase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HSP</div> <div class="def" title="def"> <p>Heat shock protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
JAZ</div> <div class="def" title="def"> <p>Jasmonate ZIM domain-containing protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MAPK</div> <div class="def" title="def"> <p>Mitogen-activated protein kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MDA</div> <div class="def" title="def"> <p>Malondialdehyde</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MYB</div> <div class="def" title="def"> <p>Myeloblastosis</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NAC</div> <div class="def" title="def"> <p>NAM, ATAF1,2, CUC2</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NBT</div> <div class="def" title="def"> <p>Nitro blue tetrazolium</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
POD</div> <div class="def" title="def"> <p>Peroxidase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
qRT-PCR</div> <div class="def" title="def"> <p>Quantitative real-time PCR</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RLK</div> <div class="def" title="def"> <p>Receptor like kinase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ROS</div> <div class="def" title="def"> <p>Reactive oxygen species</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SOD</div> <div class="def" title="def"> <p>Superoxide dismutase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SUMO</div> <div class="def" title="def"> <p>Small ubiquitin-related modifier</p> </div> </div> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Dr. Austin Cape for careful reading and feedback.</p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>This research was supported by the Natural Science Foundation of China (31271707), the funds of the National Basic Research 973 Program of China (2012CB114204), the Young Scholars Program of Shandong University (2016WLJH39) and the Fundamental Research Funds of Shandong University (2014JC048). </p> </div> <div class="availabilityofdataandmaterials" title="sec"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>The raw sequence data from the six libraries are respectively deposited in NCBI Sequence Read Archive (SRA, <a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) with accession number SRR3223630, SRR3223636, SRR3223646, SRR3223672, SRR3223674 and SRR3223675. All supporting data are included as additional supplementary files. </p> </div> <div class="authors’contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>SWL, ZYL and GMX designed and supervised the research. CM, TYQ, LY, JLD and YL performed DGE data analysis; CM and KLC performed physiological and expression analysis. SWL, CM, ZYL and GMX wrote the manuscript. All authors read and approved the final manuscript. </p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="consentforpublication" title="sec"> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> </div> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ban</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Generation and analysis of expressed sequence tags from a NaHCO <sub>3</sub>-treated <i>Limonium bicolor</i> cDNA library </span> <span class="source" tagx="source" title="source">Plant Physiol Biochem</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">977</span> <span class="lpage" tagx="lpage" title="lpage">986</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2008.06.001">10.1016/j.plaphy.2008.06.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18640047">18640047</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheng</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of various salt-alkaline mixed stress conditions on sunflower seedlings and analysis of their stress factors </span> <span class="source" tagx="source" title="source">Environ Exp Bot</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">8</span> <span class="lpage" tagx="lpage" title="lpage">21</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.envexpbot.2004.05.003">10.1016/j.envexpbot.2004.05.003</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of various salt-alkaline mixed stresses on <i>Aneurolepidium chinense</i> (Trin.) Kitag </span> <span class="source" tagx="source" title="source">Plant Soil</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">271</span> <span class="fpage" tagx="fpage" title="fpage">15</span> <span class="lpage" tagx="lpage" title="lpage">26</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11104-004-1307-z">10.1007/s11104-004-1307-z</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiang</span> <span class="given-names" tagx="given-names" title="given-names">FN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Asymmetric somatic hybridization between wheat ( <i>Triticum aestivum</i> L.) and <i>Agropyron elongatum</i> (Host) Nevishi </span> <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">299</span> <span class="lpage" tagx="lpage" title="lpage">305</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00122-003-1247-7">10.1007/s00122-003-1247-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12845444">12845444</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The place of asymmetric somatic hybridization in wheat breeding</span> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">595</span> <span class="lpage" tagx="lpage" title="lpage">603</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-013-1552-9">10.1007/s00299-013-1552-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24370665">24370665</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">ZY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A proteomic study of the response to salinity and drought stress in an introgression strain of bread wheat </span> <span class="source" tagx="source" title="source">Mol Cell Proteomics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">2676</span> <span class="lpage" tagx="lpage" title="lpage">2686</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/mcp.M900052-MCP200">10.1074/mcp.M900052-MCP200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19734139">19734139</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ai</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">XL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>TaCHP</i>: a wheat zinc finger protein gene downregulated by abscisic acid and salinity stress plays a positive role in stress tolerance </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">154</span> <span class="fpage" tagx="fpage" title="fpage">211</span> <span class="lpage" tagx="lpage" title="lpage">221</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.110.161182">10.1104/pp.110.161182</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20639406">20639406</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">YN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional analysis of <i>TaDi19A</i>, a salt-responsive gene in wheat </span> <span class="source" tagx="source" title="source">Plant Cell Environ</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">117</span> <span class="lpage" tagx="lpage" title="lpage">129</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3040.2009.02080.x">10.1111/j.1365-3040.2009.02080.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19895399">19895399</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">GM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A transcriptomic analysis reveals the nature of salinity tolerance of a wheat introgression line </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">78</span> <span class="fpage" tagx="fpage" title="fpage">159</span> <span class="lpage" tagx="lpage" title="lpage">169</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-011-9854-1">10.1007/s11103-011-9854-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22089973">22089973</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quan</span> <span class="given-names" tagx="given-names" title="given-names">TY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">KQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">LT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Wheat oxophytodienoate reductase gene <i>TaOPR1</i> confers salinity tolerance via enhancement of abscisic acid signaling and reactive oxygen species scavenging </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">161</span> <span class="fpage" tagx="fpage" title="fpage">1217</span> <span class="lpage" tagx="lpage" title="lpage">1228</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.112.211854">10.1104/pp.112.211854</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23321418">23321418</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">ST</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A wheat SIMILAR TO RCD-ONE gene enhances seedling growth and abiotic stress resistance by modulating redox homeostasis and maintaining genomic integrity </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">164</span> <span class="lpage" tagx="lpage" title="lpage">180</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.113.118687">10.1105/tpc.113.118687</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24443520">24443520</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerstein</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snyder</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span> <span class="source" tagx="source" title="source">Nat Rev Genet</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">57</span> <span class="lpage" tagx="lpage" title="lpage">63</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrg2484">10.1038/nrg2484</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19015660">19015660</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Surget-Groba</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montoya-Burgos</span> <span class="given-names" tagx="given-names" title="given-names">JI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization of de novo transcriptome assembly from next-generation sequencing data</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">1432</span> <span class="lpage" tagx="lpage" title="lpage">1440</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.103846.109">10.1101/gr.103846.109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20693479">20693479</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">SQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mo</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">QW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jia</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">XC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Aberrant host immune response induced by highly virulent PRRSV identified by digital gene expression tag profiling </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">544</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-11-544">10.1186/1471-2164-11-544</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20929578">20929578</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Global transcriptome profiling of wild soybean ( <i>Glycine soja</i>) roots under NaHCO <sub>3</sub> treatment </span> <span class="source" tagx="source" title="source">BMC Plant Biol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">153</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-10-153">10.1186/1471-2229-10-153</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20653984">20653984</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ge</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lv</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bai</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cai</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alkaline-stress response in <i>Glycine soja</i> leaf identifies specific transcription factors and ABA-mediated signaling factors </span> <span class="source" tagx="source" title="source">Funct Integr Genomics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">369</span> <span class="lpage" tagx="lpage" title="lpage">379</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10142-010-0191-2">10.1007/s10142-010-0191-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20938706">20938706</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Transcriptome exploration in <i>Leymus chinensis</i> under saline–alkaline treatment using 454 pyrosequencing </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e53632</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0053632">10.1371/journal.pone.0053632</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23365637">23365637</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">CQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">LQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">YC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comprehensive transcriptional profiling of NaHCO <sub>3</sub>-stressed <i>Tamarix hispida</i> roots reveals networks of responsive genes </span> <span class="source" tagx="source" title="source">Plant Mol Biol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">145</span> <span class="lpage" tagx="lpage" title="lpage">157</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-013-0124-2">10.1007/s11103-013-0124-2</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chong</span> <span class="given-names" tagx="given-names" title="given-names">JN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">DL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Osmotic adjustment and ion balance traits of an alkali resistant halophyte <i>Kochia sieversiana</i> during adaptation to salt and alkali conditions </span> <span class="source" tagx="source" title="source">Plant Soil</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">294</span> <span class="fpage" tagx="fpage" title="fpage">263</span> <span class="lpage" tagx="lpage" title="lpage">276</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11104-007-9251-3">10.1007/s11104-007-9251-3</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">LX</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">XM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of saline and alkaline stress on germination, seedling growth, and ion balance in wheat </span> <span class="source" tagx="source" title="source">Agron J</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">1252</span> <span class="lpage" tagx="lpage" title="lpage">1260</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2134/agronj2010.0022">10.2134/agronj2010.0022</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of ion balance and nitrogen metabolism in old and young leaves of alkali-stressed rice plants </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e37817</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0037817">10.1371/journal.pone.0037817</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22655071">22655071</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Galera</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sudhakar</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pelacho</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Capell</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christou</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Constitutive expression of a barley Fe phytosiderophore transporter increases alkaline soil tolerance and results in iron partitioning between vegetative and storage tissues under stress </span> <span class="source" tagx="source" title="source">Plant Physiol Biochem</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">46</span> <span class="lpage" tagx="lpage" title="lpage">53</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2012.01.009">10.1016/j.plaphy.2012.01.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22316602">22316602</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nanda</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gukovskaya</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tseng</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grinstein</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Activation of vacuolar-type proton pumps by protein kinase C. Role in neutrophil pH regulation </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">267</span> <span class="fpage" tagx="fpage" title="fpage">22740</span> <span class="lpage" tagx="lpage" title="lpage">22746</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1331065">1331065</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wadsworth</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Rossum</span> <span class="given-names" tagx="given-names" title="given-names">GDV</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Role of vacuolar adenosine triphosphatase in the regulation of cytosolic pH in hepatocytes</span> <span class="source" tagx="source" title="source">J Membr Biol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">142</span> <span class="fpage" tagx="fpage" title="fpage">21</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00233380">10.1007/BF00233380</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7707351">7707351</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diakov</span> <span class="given-names" tagx="given-names" title="given-names">TT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kane</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Regulation of vacuolar proton-translocating ATPase activity and assembly by extracellular pH </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">285</span> <span class="fpage" tagx="fpage" title="fpage">23771</span> <span class="lpage" tagx="lpage" title="lpage">23778</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M110.110122">10.1074/jbc.M110.110122</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20511227">20511227</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pérez-Castiñeira</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernández</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drake</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Serrano</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A plant proton-pumping inorganic pyrophosphatase functionally complements the vacuolar ATPase transport activity and confers bafilomycin resistance in yeast </span> <span class="source" tagx="source" title="source">Biochem J</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">437</span> <span class="fpage" tagx="fpage" title="fpage">269</span> <span class="lpage" tagx="lpage" title="lpage">278</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20110447">10.1042/BJ20110447</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21612578">21612578</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmgren</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant plasma membrane H <sup>+</sup>-ATPases: Powerhouses for nutrient uptake </span> <span class="source" tagx="source" title="source">Annu Rev Plant Physiol Plant Mol Biol</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">817</span> <span class="lpage" tagx="lpage" title="lpage">845</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.52.1.817">10.1146/annurev.arplant.52.1.817</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11337417">11337417</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuglsang</span> <span class="given-names" tagx="given-names" title="given-names">AT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cuin</span> <span class="given-names" tagx="given-names" title="given-names">TA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">QS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kristiansen</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Arabidopsis</i> protein kinase PKS5 inhibits the plasma membrane H <sup>+</sup>-ATPase by preventing interaction with 14-3-3 protein </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1617</span> <span class="lpage" tagx="lpage" title="lpage">1634</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.105.035626">10.1105/tpc.105.035626</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483306">17483306</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">YQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">YX</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">CG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">FY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The <i>Arabidopsis</i> chaperone J3 regulates the plasma membrane H <sup>+</sup>-ATPase through interaction with the PKS5 kinase </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">1313</span> <span class="lpage" tagx="lpage" title="lpage">1332</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.109.069609">10.1105/tpc.109.069609</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20418496">20418496</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jia</span> <span class="given-names" tagx="given-names" title="given-names">LG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">WM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baluška</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kronzucker</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Tomato 14-3-3 protein TFT4 modulates proton efflux, basipetal auxin transport and PKS5-J3 pathway in root growth response to alkaline stress </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">163</span> <span class="fpage" tagx="fpage" title="fpage">1817</span> <span class="lpage" tagx="lpage" title="lpage">1828</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.224758">10.1104/pp.113.224758</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24134886">24134886</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Oxidative stress, antioxidants and stress tolerance</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">405</span> <span class="lpage" tagx="lpage" title="lpage">410</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1360-1385(02)02312-9">10.1016/S1360-1385(02)02312-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12234732">12234732</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fridovich</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Superoxide radical and superoxide dismutases</span> <span class="source" tagx="source" title="source">Annu Rev Biochem</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">64</span> <span class="fpage" tagx="fpage" title="fpage">97</span> <span class="lpage" tagx="lpage" title="lpage">112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.bi.64.070195.000525">10.1146/annurev.bi.64.070195.000525</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7574505">7574505</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">AW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Histone variants and chromatin assembly in plant abiotic stress responses</span> <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="year" tagx="year" title="year">1819</span> <span class="volume" tagx="volume" title="volume">2012</span> <span class="fpage" tagx="fpage" title="fpage">343</span> <span class="lpage" tagx="lpage" title="lpage">348</span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deal</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Topp</span> <span class="given-names" tagx="given-names" title="given-names">CN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKinney</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meagher</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Repression of flowering in <i>Arabidopsis</i> requires activation of <i>FLOWERING LOCUS C</i> expression by the histone variant H2A.Z </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">74</span> <span class="lpage" tagx="lpage" title="lpage">83</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.048447">10.1105/tpc.106.048447</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17220196">17220196</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">SV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wigge</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">H2A.Z-containing nucleosomes mediate the thermosensory response in <i>Arabidopsis</i> </span> <span class="source" tagx="source" title="source">Cell</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">140</span> <span class="fpage" tagx="fpage" title="fpage">136</span> <span class="lpage" tagx="lpage" title="lpage">147</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2009.11.006">10.1016/j.cell.2009.11.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20079334">20079334</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deal</span> <span class="given-names" tagx="given-names" title="given-names">RB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagarajan</span> <span class="given-names" tagx="given-names" title="given-names">VK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poling</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raghothama</span> <span class="given-names" tagx="given-names" title="given-names">KG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Histone H2A.Z regulates the expression of several classes of phosphate starvation response genes but not as a transcriptional activator </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">152</span> <span class="fpage" tagx="fpage" title="fpage">217</span> <span class="lpage" tagx="lpage" title="lpage">225</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.109.145532">10.1104/pp.109.145532</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19897606">19897606</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Konev</span> <span class="given-names" tagx="given-names" title="given-names">AY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tribus</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">SY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Podhraski</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emelyanov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">CHD1 motor protein is required for deposition of histone variant H3.3 into chromatin in vivo </span> <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">317</span> <span class="fpage" tagx="fpage" title="fpage">1087</span> <span class="lpage" tagx="lpage" title="lpage">1090</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1145339">10.1126/science.1145339</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17717186">17717186</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawatsubashi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murata</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujiki</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ito</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A histone chaperone, DEK, transcriptionally coactivates a nuclear receptor</span> <span class="source" tagx="source" title="source">Genes Dev</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">159</span> <span class="lpage" tagx="lpage" title="lpage">170</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gad.1857410">10.1101/gad.1857410</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20040570">20040570</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aina</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sgorbati</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santagostino</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labra</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghiani</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Citterio</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Specific hypomethylation of DNA is induced by heavy metals in white clover and industrial hemp </span> <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">121</span> <span class="fpage" tagx="fpage" title="fpage">472</span> <span class="lpage" tagx="lpage" title="lpage">480</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1399-3054.2004.00343.x">10.1111/j.1399-3054.2004.00343.x</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wada</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyamoto</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kusano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Association between up-regulation of stress-responsive genes and hypomethylation of genomic DNA in to tobacco plants </span> <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">271</span> <span class="fpage" tagx="fpage" title="fpage">658</span> <span class="lpage" tagx="lpage" title="lpage">666</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-004-1018-4">10.1007/s00438-004-1018-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15148604">15148604</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sano</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Abiotic-stress induces demethylation and transcriptional activation of a gene encoding a glycerophosphodiesterase-like protein in tobacco plants </span> <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">277</span> <span class="fpage" tagx="fpage" title="fpage">589</span> <span class="lpage" tagx="lpage" title="lpage">600</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-007-0209-1">10.1007/s00438-007-0209-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17273870">17273870</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">LN</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Induced and constitutive DNA methylation in a salinity tolerant wheat introgression line </span> <span class="source" tagx="source" title="source">Plant Cell Physiol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">1354</span> <span class="lpage" tagx="lpage" title="lpage">1365</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcu059">10.1093/pcp/pcu059</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24793752">24793752</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ranieri</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castagna</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lorenzini</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soldatini</span> <span class="given-names" tagx="given-names" title="given-names">GF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Changes in thylakoid protein patterns and antioxidant levels in two wheat cultivars with different sensitivity to sulphur dioxide </span> <span class="source" tagx="source" title="source">Environ Exp Bot</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">125</span> <span class="lpage" tagx="lpage" title="lpage">135</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0098-8472(96)01040-4">10.1016/S0098-8472(96)01040-4</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome expression profile analysis of the immature maize embryo during dedifferentiation</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e32237</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0032237">10.1371/journal.pone.0032237</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22448216">22448216</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Audic</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Claverie</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The significance of digital gene expression profiles</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">986</span> <span class="lpage" tagx="lpage" title="lpage">995</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9331369">9331369</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benjamini</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yekutieli</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The control of the false discovery rate in multiple testing under dependency</span> <span class="source" tagx="source" title="source">Ann Stat</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1165</span> <span class="lpage" tagx="lpage" title="lpage">1188</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1214/aos/1013699998">10.1214/aos/1013699998</a> </span> </span> </li> </ul> </div> </div>  </body></html>